<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>yield</title>
	<style type="text/css">
		.gene {
			background-color: #CCCCFF
		}

		.trait {
			background-color: yellow
		}

		body {
			margin: 30px
		}
	</style>
	<SCRIPT>
		function isHidden(oDiv) {
			var vDiv = document.getElementById(oDiv);
			vDiv.style.display = (vDiv.style.display == 'none') ? 'block' : 'none';
		}
	</SCRIPT>
</head>

<body>
	<h1>yield</h1>
	<br/>
	<div>
		

			
			<h2>文献标题:Extracellular Enzyme Composition and Functional Characteristics of Aspergillus niger An-76 Induced by Food Processing Byproducts and Based on Integrated Functional-omics.</h2>
			<h3>PMID:29334221</h3>
				

				
			<p>
				作者：Liu L, Gong W, Sun X, Chen G, Wang L.
				<br/> 
				年份：2018 Jan 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Byproducts of food processing can be utilized for the production of high-value-added enzyme cocktails. In this study, we utilized integrated functional-omics technology to analyze composition and functional characteristics of extracellular enzymes produced by Aspergillus niger grown on food processing byproducts. The results showed that oligosaccharides constituted by arabinose, xylose, and glucose in wheat bran were able to efficiently induce the production of extracellular enzymes of A. niger. Compared with other substrates, wheat bran was more effective at inducing the secretion of β-glucosidases from <span class="gene">GH1</span> and <span class="gene">GH3</span> families, as well as &gt;50% of proteases from A1-family aspartic proteases. Compared with proteins induced by single wheat bran or soybean dregs, the protein <span class="trait">yield</span> induced by their mixture was doubled, and the time required to reach peak enzyme activity was shortened by 25%. This study provided a technical platform for the complex formulation of various substrates and functional analysis of extracellular enzymes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Novel mechanisms for organic acid-mediated aluminium tolerance in roots and leaves of two contrasting soybean genotypes</h2>
			<h3>PMID:29302304</h3>
				

				
					<h3>PMCID:5739043</h3>
					<h4>期刊:AoB Plants</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Shou-Cheng </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  <xref ref-type="aff" rid="AF0003"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chu </surname>  <given-names> Shu-Juan </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yu-Min </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ji </surname>  <given-names> Ya-Jing </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Dong-Qing </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Gui-Hua </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Rong-Wu </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Tang </surname>  <given-names> Cheng-Yi </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  <xref ref-type="corresp" rid="c1"/>  <!--<email> tangchengyi_nju@163.com </email> --> </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qi </surname>  <given-names> Jin-Liang </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  <xref ref-type="corresp" rid="c1"/>  <!--<email> qijl@nju.edu.cn </email> --> </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Yang </surname>  <given-names> Yong-Hua </given-names>  </name>  <xref ref-type="aff" rid="AF0001"/>  <xref ref-type="aff" rid="AF0002"/>  <xref ref-type="corresp" rid="c1"/>  <!--<email> yangyh@nju.edu.cn </email> --> </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Aluminium (Al) toxicity is one of the most important limiting factors for crop <span class="trait">yield</span> in acidic soils. However, the mechanisms that confer Al tolerance still remain largely unknown. To understand the molecular mechanism that confers different tolerance to Al, we performed global transcriptome analysis to the roots and leaves of two contrasting soybean genotypes, BX10 (Al-tolerant) and BD2 (Al-sensitive) under 0 and 50 μM Al3+ treatments, respectively. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes analyses revealed that the expression levels of the genes involved in lipid/carbohydrate metabolism and jasmonic acid (JA)-mediated signalling pathway were highly induced in the roots and leaves of both soybean genotypes. The gene encoding enzymes, including pyruvate kinase, phosphoenolpyruvate carboxylase, ATP-citrate lyase and glutamate-oxaloacetate transaminase 2, associated with organic acid metabolism were differentially expressed in the BX10 roots. In addition, the genes involved in citrate transport were differentially expressed. Among these genes, <span class="gene">FRD3b</span> was down-regulated only in BD2, whereas the other two multidrug and toxic compound extrusion genes were up-regulated in both soybean genotypes. These findings confirmed that BX10 roots secreted more citrate than BD2 to withstand Al stress. The gene encoding enzymes or regulators, such as lipoxygenase, 12-oxophytodienoate reductase, acyl-CoA oxidase and jasmonate ZIM-domain proteins, involved in JA biosynthesis and signalling were preferentially induced in BD2 leaves. This finding suggests that the JA defence response was activated, possibly weakening the growth of aerial parts because of excessive resource consumption and ATP biosynthesis deficiency. Our results suggest that the Al sensitivity in some soybean varieties could be attributed to the low level of citrate metabolism and exudation in the roots and the high level of JA-mediated defence response in the leaves.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Al-induced citrate and malate exudation are critical for Al <span class="trait">resistance</span> in soybeans ( <xref rid="CIT0062" ref-type="bibr"> Yang and Zheng 2006 </xref> ; <xref rid="CIT0028" ref-type="bibr"> Liang <italic> et al. </italic> 2013 </xref> ). Thus, the induction of genes encoding citrate transporters is required for Al-stimulated citrate efflux ( <xref rid="CIT0062" ref-type="bibr"> Yang and Zheng 2006 </xref> ). Citrate can be secreted into rhizosphere and/or vascular system for Al chelation. The function of <span class="gene">MATE</span> proteins as mediators of citrate transport is conserved in different plant species ( <xref rid="CIT0035" ref-type="bibr"> Magalhaes <italic> et al. </italic> 2007 </xref> ; <xref rid="CIT0068" ref-type="bibr"> Zhou <italic> et al. </italic> 2013 </xref> ). <span class="gene">Glyma13g27300,</span> a <span class="gene">MATE</span> gene that is considered as a candidate gene involved in Al tolerance in soybeans ( <xref rid="CIT0031" ref-type="bibr"> Liu <italic> et al. </italic> 2016 </xref> ), was significantly up-regulated in our study. In addition, the down-regulation of another <span class="gene">MATE</span> gene, <italic> <span class="gene">GmFRD3b</span> </italic> , received our attention. Previous report revealed that both <italic> <span class="gene">GmFRD3a</span> </italic> and <italic> <span class="gene">GmFRD3b</span> </italic> are involved in Fe deficiency response in soybeans ( <xref rid="CIT0043" ref-type="bibr"> Rogers <italic> et al. </italic> 2009 </xref> ; <xref rid="CIT0049" ref-type="bibr"> Takanashi <italic> et al. </italic> 2014 </xref> ). Further study showed that <italic> <span class="gene">GmFRD3a</span> </italic> is up-regulated in response to Al ( <xref rid="CIT0063" ref-type="bibr"> You <italic> et al. </italic> 2011 </xref> ). In the present study, we found that <italic> <span class="gene">GmFRD3a</span> </italic> was not induced in these two soybean genotypes, instead, <italic> <span class="gene">GmFRD3b</span> </italic> was down-regulated in the roots of BD2 but not in BX10. In <italic> Arabidopsis </italic> , AtFRD3 functions in transporting citrate into root vasculature where it forms a complex with iron and then transfers to shoots ( <xref rid="CIT0042" ref-type="bibr"> Rogers and Guerinot 2002 </xref> ; <xref rid="CIT0015" ref-type="bibr"> Green and Rogers 2004 </xref> ; <xref rid="CIT0011" ref-type="bibr"> Durrett <italic> et al. </italic> 2007 </xref> ). Moreover, the overexpression of <italic> AtFRD3 </italic> contributes to Al tolerance ( <xref rid="CIT0011" ref-type="bibr"> Durrett <italic> et al. </italic> 2007 </xref> ). Therefore, the up-regulation of glyma13g27300 and glyma02g31370 in both genotypes and the down-regulation of <italic> <span class="gene">GmFRD3b</span> </italic> in BD2 roots suggest that the enhanced citrate exudation is a common mechanism, whereas the sensitive soybeans increase the supply of citrate for exudation by inhibiting the internal flow of citrate into xylem. We suggest that the decreased expression of <italic> <span class="gene">GmFRD3b</span> </italic> in BD2 may suppress citrate loading into root xylem. As a result, a relatively large amount of toxic Al would be translocated into the aerial portions, accumulate and cause Al toxicity in the leaves.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Identification of <italic> Chalcone Reductase </italic> Gene Family in Soybean: Insight into Root-Specific <italic> <span class="gene">GmCHR</span> </italic> s and <italic> Phytophthora sojae </italic> <span class="trait">Resistance</span></h2>
			<h3>PMID:29270182</h3>
				

				
					<h3>PMCID:5725808</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sepiol </surname>  <given-names> Caroline J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/459179/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Jaeju </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/470485/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dhaubhadel </surname>  <given-names> Sangeeta </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190408/overview"/>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 11 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max [L.] Merr) is one of the main grain legumes worldwide. Soybean farmers lose billions of dollars' worth of <span class="trait">yield</span> annually due to root and stem rot disease caused by the oomycete Phytophthora sojae. Many strategies have been developed to combat the disease, however, these methods have proven ineffective in the long term. A more cost effective and durable approach is to select a trait naturally found in soybean that can increase <span class="trait">resistance.</span> One such trait is the increased production of phytoalexin glyceollins in soybean. Glyceollins are isoflavonoids, synthesized via the legume-specific branch of general phenylpropanoid pathway. The first key enzyme exclusively involved in glyceollin synthesis is chalcone reductase (CHR) which coacts with chalcone synthase for the production of isoliquiritigenin, the precursor for glyceollin biosynthesis. Here we report the identification of 14 putative CHR genes in soybean where 11 of them are predicted to be functional. Our results show that <span class="gene">GmCHRs</span> display tissue-specific gene expression, and that only root-specific <span class="gene">GmCHRs</span> are induced upon P. sojae infection. Among 4 root-specific <span class="gene">GmCHRs,</span> <span class="gene">GmCHR2A</span> is located near a QTL that is linked to P. sojae <span class="trait">resistance</span> suggesting <span class="gene">GmCHR2A</span> as a novel locus for partial <span class="trait">resistance</span> that can be utilized for <span class="trait">resistance</span> breeding.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> [L.] Merr) is one of the main grain legumes worldwide. Soybean farmers lose billions of dollars&#x02019; worth of <span class="trait">yield</span> annually due to root and stem rot disease caused by the oomycete <italic> Phytophthora sojae </italic> . Many strategies have been developed to combat the disease, however, these methods have proven ineffective in the long term. A more cost effective and durable approach is to select a trait naturally found in soybean that can increase <span class="trait">resistance.</span> One such trait is the increased production of phytoalexin glyceollins in soybean. Glyceollins are isoflavonoids, synthesized via the legume-specific branch of general phenylpropanoid pathway. The first key enzyme exclusively involved in glyceollin synthesis is chalcone reductase (CHR) which coacts with chalcone synthase for the production of isoliquiritigenin, the precursor for glyceollin biosynthesis. Here we report the identification of 14 putative <italic> CHR </italic> genes in soybean where 11 of them are predicted to be functional. Our results show that <italic> <span class="gene">GmCHR</span> </italic> s display tissue-specific gene expression, and that only root-specific <italic> <span class="gene">GmCHR</span> </italic> s are induced upon <italic> P. sojae </italic> infection. Among 4 root-specific <italic> <span class="gene">GmCHR</span> </italic> s, <italic> <span class="gene">GmCHR2A</span> </italic> is located near a QTL that is linked to <italic> P. sojae </italic> <span class="trait">resistance</span> suggesting <italic> <span class="gene">GmCHR2A</span> </italic> as a novel locus for partial <span class="trait">resistance</span> that can be utilized for <span class="trait">resistance</span> breeding.
					</p>
				
					<p>&nbsp;&nbsp;
						The traditional strategies to reduce the incidence of <italic> P. sojae </italic> infection such as soil treatment with calcium application ( <xref rid="B52" ref-type="bibr"> Sugimoto et al., 2010 </xref> ), fungicide application ( <xref rid="B3" ref-type="bibr"> Anderson and Buzzell, 1982 </xref> ) and improved soil drainage and tillage ( <xref rid="B60" ref-type="bibr"> Workneh et al., 1998 </xref> ) have not only proven to be ineffective, but also found to place selective pressures on <italic> P. sojae </italic> leading to <span class="trait">resistance</span> ( <xref rid="B33" ref-type="bibr"> Li et al., 2010 </xref> ). An alternative approach to this problem is selecting a cultivar of soybean with optimal <span class="trait">resistance</span> to <italic> P. sojae </italic> infection. <span class="trait">Resistance</span> to pathogen infection in soybeans can either be complete or partial. Complete <span class="trait">resistance</span> or race-specific <span class="trait">resistance</span> is conferred by single dominant <span class="trait">resistance</span> to <italic> P. sojae </italic> ( <italic> Rps </italic> ) genes, which counteract the virulence genes within <italic> P. sojae </italic> . This relationship is similar to effector-triggered immune response in other pathosystems ( <xref rid="B51" ref-type="bibr"> Sugimoto et al., 2012 </xref> ). To date, there are 27 <italic> Rps </italic> genes and more than 200 identified races of <italic> P. sojae </italic> ( <xref rid="B41" ref-type="bibr"> Sahoo et al., 2017 </xref> ). However, the rapid evolution of <italic> P. sojae </italic> races continues to diversify and as a result the new races become ineffective to the <italic> Rps </italic> genes leading to disease susceptibility. Partial <span class="trait">resistance,</span> referred to as field <span class="trait">resistance,</span> is more durable and a broad-spectrum non-race-specific trait conferred by several minor genes which involves various defense components. Cultivars with this type of <span class="trait">resistance</span> contain fewer damaged roots than completely susceptible cultivars, show delayed disease progression, and are effective against all races of <italic> P. sojae </italic> ( <xref rid="B44" ref-type="bibr"> Schmitthenner, 1985 </xref> ). Some key traits such as increased levels of soybean root suberin and increased production of phytoalexin glyceollins have been correlated with strong partial <span class="trait">resistance</span> in soybean ( <xref rid="B53" ref-type="bibr"> Thomas et al., 2007 </xref> ; <xref rid="B37" ref-type="bibr"> Lygin et al., 2013 </xref> ). Isoflavonoids act as phytoalexins in soybean that play a role as plant&#x02019;s basal and innate response to biotic and abiotic stress. When isoflavonoid production was compromised by down regulating the biosynthetic genes, <italic> chalcone reductase </italic> ( <italic> CHR </italic> ) and <italic> isoflavone synthase </italic> ( <italic> <span class="gene">IFS</span> </italic> ), it reduced the plants&#x02019; ability to fight off the pathogen attack ( <xref rid="B49" ref-type="bibr"> Subramanian et al., 2005 </xref> ; <xref rid="B20" ref-type="bibr"> Graham et al., 2007 </xref> ; <xref rid="B36" ref-type="bibr"> Lozovaya et al., 2007 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we performed a genome-wide search of <italic> AKR </italic> gene family members and identified 14 <italic> CHR </italic> genes in soybean. The phylogenetic history of the <span class="gene">GmCHRs,</span> and their sequence analysis suggested 11 <span class="gene">GmCHRs</span> as catalytically active for its role. Transcript analysis revealed that 4 root-specific <italic> <span class="gene">GmCHR</span> </italic> s are induced upon stress. Some of the stress induced <italic> <span class="gene">GmCHR</span> </italic> s locate in the QTL regions that are linked to <italic> P. sojae </italic> <span class="trait">resistance.</span> These <italic> <span class="gene">GmCHR</span> </italic> genes together with other minor genes and <italic> Rps </italic> gene can be stacked in the same cultivar for future resistant breeding.
					</p>
				
					<p>&nbsp;&nbsp;
						The QTLs and QTL markers from the year 2003 to 2016 corresponding to <italic> P. sojae </italic> <span class="trait">resistance</span> were extracted from the SoyBase and Soybean Breeder&#x02019;s Toolbox <sup> <xref ref-type="fn" rid="fn07"> 7 </xref> </sup> . To ensure no QTLs or QTL marker were missed in the search; a literature search was also conducted. Relative positions of <italic> <span class="gene">GmCHRs</span> </italic> , and QTL markers were mapped onto the chromosomes. QTLs regions in base pairs were noted from the <italic> G. max </italic> genome assembly on Soybase.org.
					</p>
				
					<p>&nbsp;&nbsp;
						To determine QTLs and QTL markers that are linked to <italic> P. sojae </italic> <span class="trait">resistance,</span> a survey of soybean database and literature search were conducted. A search in the &#x02018;SoyBase and Soybean Breeder&#x02019;s Toolbox&#x02019; from the year 2003 to 2016 identified 77 QTLs that are linked to <italic> P. sojae </italic> <span class="trait">resistance</span> in soybean (Supplementary Table <xref ref-type="supplementary-material" rid="SM3"> S2 </xref> ). These 77 QTLs were extensively researched for <italic> <span class="gene">GmCHR</span> </italic> loci, parental lines and LOD scores. Four QTLs, Phytoph 10-3 ( <xref rid="B21" ref-type="bibr"> Han et al., 2008 </xref> ), Phytoph 14-3 ( <xref rid="B32" ref-type="bibr"> Lee et al., 2013 </xref> ), Phytoph 8-2 ( <xref rid="B55" ref-type="bibr"> Tucker et al., 2010 </xref> ) and Phytoph 15-5 ( <xref rid="B58" ref-type="bibr"> Wang et al., 2012 </xref> ) were found which contain <italic> <span class="gene">GmCHR</span> </italic> loci. Highlights of the QTLs included: (1) Phytoph 10-3 flanks <italic> <span class="gene">GmCHR2A</span> </italic> locus 2 megabase pairs and has an LOD score of 29.7; (2) Phytoph 14-3 and Phytoph 15-5 contain <italic> <span class="gene">GmCHR18</span> </italic> locus within the marker interval with lower LOD score; (3) Phytoph 8-2 contains <italic> <span class="gene">GmCHR20</span> </italic> locus that stretches over 31 megabase pairs with the LOD score of 4.5 ( <bold> Table <xref ref-type="table" rid="T3"> 3 </xref> </bold> ). These details suggest that Phytoph 10-3 containing <italic> <span class="gene">GmCHR2A</span> </italic> is involved in <italic> P. sojae </italic> <span class="trait">resistance</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Characteristics of QTLs linked to <italic> P. sojae </italic> <span class="trait">resistance</span> that contain <italic> <span class="gene">GmCHRs.</span> </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						An additional literature search was conducted for QTL markers linked to <italic> P. sojae </italic> <span class="trait">resistance</span> in soybean. This process identified over 500 QTL markers (Supplementary Table <xref ref-type="supplementary-material" rid="SM4"> S3 </xref> ). The markers that share the same chromosome as <italic> <span class="gene">GmCHR</span> </italic> were separated, and exact locations of the QTL markers were noted. A total of 8 QTL markers were found to flank <italic> <span class="gene">GmCHR2A,</span> <span class="gene">GmCHR16A,</span> <span class="gene">GmCHR18</span> </italic> and <italic> <span class="gene">GmCHR20</span> </italic> loci ( <xref rid="B21" ref-type="bibr"> Han et al., 2008 </xref> ; <xref rid="B55" ref-type="bibr"> Tucker et al., 2010 </xref> ; <xref rid="B57" ref-type="bibr"> Wang et al., 2010 </xref> ). The details on the QTL marker are summarized in <bold> Table <xref ref-type="table" rid="T4"> 4 </xref> </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						Characteristics of QTL markers linked to <italic> P. sojae </italic> <span class="trait">resistance</span> that flank <span class="gene">GmCHRs.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Since several QTL markers and QTLs linked to <italic> P. sojae </italic> <span class="trait">resistance</span> contained <italic> <span class="gene">GmCHR</span> </italic> gene family members, we evaluated the parental cultivars of QTL Phytoph 10-3 for root-specific <italic> <span class="gene">GmCHR</span> </italic> gene expression. Root tissues of 2-week old seedlings of soybean cv. Conrad and OX760-6 were used for gene expression analysis. As shown in <bold> Figure <xref ref-type="fig" rid="F7"> 7A </xref> </bold> , expression of <italic> <span class="gene">GmCHR2A,</span> <span class="gene">GmCHR14</span> </italic> and <italic> <span class="gene">GmCHR18</span> </italic> were significantly higher in roots of Conrad ( <italic> P. sojae </italic> resistant cultivar) compared to the OX760-6 ( <italic> P. sojae </italic> susceptible cultivar). No difference in the expression of <italic> <span class="gene">GmCHR20</span> </italic> was observed between Conrad and OX760-6. When the etiolated hypocotyls of Conrad and OX760-6 were treated with AgNO <sub> 3 </sub> , and expression of root-specific <italic> <span class="gene">GmCHR</span> </italic> s were monitored, only the expression of <italic> <span class="gene">GmCHR2A</span> </italic> was induced in response to AgNO <sub> 3 </sub> treatment and its transcript accumulation was higher in Conrad compared to OX760-6 at later time points ( <bold> Figure <xref ref-type="fig" rid="F7"> 7B </xref> </bold> ). No difference in the level of other root-specific <italic> <span class="gene">GmCHRs</span> </italic> were observed in Conrad and OX760-6 both in control and AgNO <sub> 3- </sub> treated samples. Furthermore, expression of <italic> <span class="gene">GmCHR14</span> </italic> and <italic> <span class="gene">GmCHR18</span> </italic> were not induced in response to AgNO <sub> 3 </sub> treatment (not shown).
					</p>
				
					<p>&nbsp;&nbsp;
						The legume-specific enzyme CHR together with CHS, converts 1 molecule of <italic> p </italic> -coumaroyl-CoA and 3 molecules of malonyl-CoA to isoliquiritigenin, the building block of two core isoflavone aglycones, glycitein and daidzein, where daidzein serves as a precursor for the production of phytoalexin glyceollins in soybean. Many studies have reported the expression patterns of genes during infection and the heritability of <span class="trait">resistance,</span> however, little is known about the importance of <italic> CHR </italic> , the first key enzyme, which directs the flux to the production of phytoalexin glyceollins in soybean. Here we report the genome-wide identification of <italic> <span class="gene">GmCHRs</span> </italic> in soybean, investigate their subcellular location, and tissue-specific and pathogen induced gene expression. Our results demonstrate that the root-specific <italic> <span class="gene">GmCHRs</span> </italic> are induced upon pathogen infection, and are located near QTLs and QTL markers linked to <italic> P. sojae </italic> <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						To evaluate if <italic> <span class="gene">GmCHR</span> </italic> family members respond differently upon pathogen infection, their expression levels were studied by RT-PCR at various time points after <italic> P. sojae </italic> infection. Interestingly, the expression levels of only 4 root-specific <italic> <span class="gene">GmCHRs,</span> <span class="gene">GmCHR2A,</span> <span class="gene">GmCHR14,</span> <span class="gene">GmCHR18</span> </italic> and <span class="gene">GmCHR</span> <italic> 20 </italic> , were induced upon infection suggesting that they have a role in defense against <italic> P. sojae </italic> infection ( <bold> Figure <xref ref-type="fig" rid="F5"> 5A </xref> </bold> ). Upon infection, <italic> P. sojae </italic> releases elicitors which stimulate the plant defense response ( <xref rid="B29" ref-type="bibr"> Jones and Dangl, 2006 </xref> ). As a result, the plant induces the expression of <span class="trait">resistance</span> and defense related genes to counteract infection. Studies have shown the upregulation or induction of <italic> CHR </italic> s at the infection site during <italic> Fusarium </italic> attack in soybeans ( <xref rid="B26" ref-type="bibr"> Iqbal et al., 2005 </xref> ), cadmium treatment in <italic> Medicago truncatula </italic> ( <xref rid="B2" ref-type="bibr"> Aloui et al., 2012 </xref> ), and <italic> Colletotrichum falcatum </italic> infection in sugarcane ( <xref rid="B46" ref-type="bibr"> Selvaraj et al., 2014 </xref> ). Several studies have used the AgNO <sub> 3 </sub> treatment to mimic pathogen infection and induce phytoalexin production in soybeans ( <xref rid="B38" ref-type="bibr"> Moy et al., 2004 </xref> ; <xref rid="B30" ref-type="bibr"> Kube&#x00161; et al., 2014 </xref> ). The mechanism of this &#x0201c;elicitor effect&#x0201d; has not been identified yet. The quantitative analysis of root-specific <italic> <span class="gene">GmCHRs</span> </italic> in response to AgNO <sub> 3 </sub> demonstrated a significant increase in transcript accumulation of <italic> <span class="gene">GmCHR2A,</span> <span class="gene">GmCHR14</span> </italic> and <italic> <span class="gene">GmCHR18</span> </italic> ( <bold> Figure <xref ref-type="fig" rid="F5"> 5C </xref> </bold> ). These root-specific <italic> <span class="gene">GmCHRs</span> </italic> respond to the AgNO <sub> 3 </sub> treatment as early as 12 h which coincides with findings from ( <xref rid="B1" ref-type="bibr"> Alkharouf et al., 2006 </xref> ). Changes in gene expression within roots upon <italic> Heterodera glycines </italic> (the soybean cyst nematode) attack was investigated using a 6000 gene microarray ( <xref rid="B1" ref-type="bibr"> Alkharouf et al., 2006 </xref> ). It was found that <italic> CHR </italic> (Genbank BM108162) was induced as soon as 6 and 12 h upon infection.
					</p>
				
					<p>&nbsp;&nbsp;
						Several studies have suggested that <italic> Rps </italic> genes ( <xref rid="B15" ref-type="bibr"> Dorrance and Schmitthenner, 2000 </xref> ; <xref rid="B42" ref-type="bibr"> Sandhu et al., 2005 </xref> ), isoflavonoid biosynthetic genes ( <xref rid="B49" ref-type="bibr"> Subramanian et al., 2005 </xref> ; <xref rid="B20" ref-type="bibr"> Graham et al., 2007 </xref> ), and genes involved in suberin production ( <xref rid="B40" ref-type="bibr"> Ranathunge et al., 2008 </xref> ) contribute to <italic> P. sojae </italic> <span class="trait">resistance</span> in soybean. Furthermore, molecular markers linked to <italic> Rps </italic> genes or QTLs and QTL markers linked to <italic> P. sojae </italic> <span class="trait">resistance</span> have been identified in soybean. However, little has been reported linking the candidate genes with the QTLs and the phenotype. Here, we have identified a total of 8 QTL markers and 4 QTLs which flank or are approximate to <italic> <span class="gene">GmCHR2A,</span> <span class="gene">GmCHR18</span> </italic> and <italic> <span class="gene">GmCHR20</span> </italic> . Among the QTLs, Phytoph 14-3 covers most of the chromosome 18 ( <bold> Table <xref ref-type="table" rid="T3"> 3 </xref> </bold> ). Even though QTL regions can generally span several megabase pairs, and can contain several hundreds to thousands of genes ( <xref rid="B16" ref-type="bibr"> Dupuis and Siegmund, 1999 </xref> ), further fine-mapping experiments are required to pinpoint the spanning regions ( <xref rid="B54" ref-type="bibr"> Touzet et al., 1995 </xref> ; <xref rid="B24" ref-type="bibr"> Holtan and Hake, 2003 </xref> ). Statistical association or validation studies can confirm the co-segregation of genes with a QTL. Based on these parameters, QTL Phytoph 10-3 was selected as the most reliable QTL found in this study as it is confined to a specific region on the chromosome and contains the highest LOD score. Even though the expression of root-specific <italic> <span class="gene">GmCHRs,</span> <span class="gene">GmCHR2A,</span> <span class="gene">GmCHR14</span> </italic> and <italic> <span class="gene">GmCHR18</span> </italic> were significantly higher in the roots of the parental cultivars Conrad (resistant cultivar) as compared to OX760-6 (susceptible cultivar) used in the QTL study by <xref rid="B21" ref-type="bibr"> Han et al. (2008) </xref> , only expression of <italic> <span class="gene">GmCHR2A</span> </italic> was induced upon stress and was higher in the cultivar Conrad as compared to OX760-6 ( <bold> Figure <xref ref-type="fig" rid="F7"> 7B </xref> </bold> ). This result together with the expression of root-specific <italic> <span class="gene">GmCHRs</span> </italic> ( <bold> Figure <xref ref-type="fig" rid="F4"> 4 </xref> </bold> ), suggests that in addition to <span class="gene">GmCHRs,</span> there are other yet unidentified factors that play an important role in partial <span class="trait">resistance</span> in soybean. Identification of each component is crucial to understand the <span class="trait">resistance</span> mechanism so that genes encoding those factors could be stacked together into a single cultivar for <span class="trait">resistance</span> breeding.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean peroxidase immobilized onto silica-coated superparamagnetic iron oxide nanoparticles: Effect of silica layer on the enzymatic activity.</h2>
			<h3>PMID:29169120</h3>
				

				
			<p>
				作者：Donadelli JA(1), García Einschlag FS(1), Laurenti E(2), Magnacca G(3), Carlos L(4).
				<br/> 
				年份：2018 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Peroxidase immobilization onto magnetic supports is considered an innovative strategy for the development of technologies that involves enzymes in wastewater treatment. In this work, magnetic biocatalysts were prepared by immobilization of soybean peroxidase (SBP) onto different silica-coated superparamagnetic iron oxide nanoparticles. The obtained magnetic biocatalysts were tested for the degradation of malachite green (MG), a pollutant often found in industrial wastewaters and with significant drawbacks for the human and environmental health. A deep physicochemical characterization of the materials was performed by means of X-ray diffraction (XRD), Fourier Transform Infrared Spectroscopy (FTIR), High Resolution-Transmission Electron Microscope (HR-TEM) and magnetization measurements among others techniques. Results showed high immobilization <span class="trait">yield</span> of <span class="gene">SBP</span> onto nanomaterials with excellent properties for magnetic recoverability. A partial loss of activity with respect to free <span class="gene">SBP</span> was observed, compatible with the modification of the conformational structure of the enzyme after immobilization. The structural modification depended on the amount (and thickness) of silica present in the hybrid materials and the activity <span class="trait">yield</span> of 43% was obtained for the best biocatalyst. Thermal stability and reusability capacity were also evaluated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Increasing <span class="trait">drought</span> and diminishing benefits of elevated carbon dioxide for soybean yields across the US Midwest.</h2>
			<h3>PMID:29110424</h3>
				

				
			<p>
				作者：Jin Z(1), Ainsworth EA(2)(3), Leakey ADB(2), Lobell DB(1).
				<br/> 
				年份：2017 Nov 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Elevated atmospheric CO2 concentrations ([CO2 ]) are expected to increase <span class="gene">C3</span> crop <span class="trait">yield</span> through the CO2 fertilization effect (CFE) by stimulating photosynthesis and by reducing stomatal conductance and transpiration. The latter effect is widely believed to lead to greater benefits in dry rather than wet conditions, although some recent experimental evidence challenges this view. Here we used a process-based crop model, the Agricultural Production Systems sIMulator (APSIM), to quantify the contemporary and future CFE on soybean in one of its primary production area of the US Midwest. APSIM accurately reproduced experimental data from the Soybean Free-Air CO2 Enrichment site showing that the CFE declined with increasing <span class="trait">drought</span> stress. This resulted from greater radiation use efficiency (RUE) and above-ground biomass production at elevated [CO2 ] that outpaced gains in transpiration efficiency (TE). Using an ensemble of eight climate model projections, we found that <span class="trait">drought</span> frequency in the US Midwest is projected to increase from once every 5 years currently to once every other year by 2050. In addition to directly driving <span class="trait">yield</span> loss, greater <span class="trait">drought</span> also significantly limited the benefit from rising [CO2 ]. This study provides a link between localized experiments and regional-scale modeling to highlight that increased <span class="trait">drought</span> frequency and severity pose a formidable challenge to maintaining soybean <span class="trait">yield</span> progress that is not offset by rising [CO2 ] as previously anticipated. Evaluating the relative sensitivity of RUE and TE to elevated [CO2 ] will be an important target for future modeling and experimental studies of climate change impacts and adaptation in <span class="gene">C3</span> crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Confirmation of the pleiotropic control of leaflet shape and number of seeds per pod by the <span class="gene">Ln</span> gene in induced soybean mutants.</h2>
			<h3>PMID:29085246</h3>
				

				
			<p>
				作者：Sayama T(1)(2), Tanabata T(3), Saruta M(2)(4), Yamada T(1), Anai T(5), Kaga A(1), Ishimoto M(1).
				<br/> 
				年份：2017 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Most soybean cultivars possess broad leaflets; however, a recessive allele on the <span class="gene">Ln</span> locus is known to cause the alteration of broad to narrow leaflets. The recessive allele <span class="gene">ln</span> has also been considered to increase the number of seeds per pod (NSP) and has the potential to improve <span class="trait">yield.</span> Recently, <span class="gene">Gm-JAG1</span> <span class="gene">(Glyma20g25000),</span> a gene controlling Ln, has been shown to complement leaf shape and silique length in Arabidopsis mutants. However, whether <span class="gene">Gm-JAG1</span> is responsible for those traits in soybean is not yet known. In this study, we investigated the pleiotropic effect of soybean <span class="gene">Ln</span> gene on leaflet shape and NSP by using two independent soybean <span class="gene">Gm-jag1</span> mutants and four <span class="gene">ln</span> near isogenic lines (NILs). The leaflet shape was evaluated using a leaf image analysis software, SmartLeaf, which was customized from SmartGrain. The leaflets of both the <span class="gene">Gm-jag1</span> mutants were longer and narrower than those of the wild-type plants. Interestingly, the image analysis results clarified that the perimeter of the mutant leaflets did not change, although their leaflet area decreased. Furthermore, one mutant line with narrow leaflets showed significantly higher NSP than that in the wild (or Ln) genotype, indicating that soybean <span class="gene">Ln</span> gene pleiotropically controls leaflet shape and NSP.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Osmoprotective functions conferred to soybean plants via inoculation with Sphingomonas sp. LK11 and exogenous trehalose.</h2>
			<h3>PMID:28942839</h3>
				

				
			<p>
				作者：Asaf S(1), Khan AL(2), Khan MA(1), Imran QM(1), Yun BW(1), Lee IJ(3).
				<br/> 
				年份：2017 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Osmotic stress induced by <span class="trait">drought</span> can hinder the growth and <span class="trait">yield</span> of crop plants. To understand the eco-physiological role of osmoprotectants, the combined utilization of endophytes and osmolytes (trehalose) can be an ideal strategy used to overcome the adverse effects of <span class="trait">drought.</span> Hence, in the present study, we aimed to investigate the role of Sphingomonas sp. LK11, which produces phytohormones and synthesizes trehalose, in improving soybean plant growth under drought-induced osmotic stress (-0.4, -0.9, and -1.2MPa). The results showed that the inoculation of soybean plants with Sphingomonas sp. LK11 significantly increased plant length, dry biomass, photosynthetic pigments, glutathione, amino acids (proline, glycine, and glutamate), and primary sugars as compared to control plants under varying <span class="trait">drought</span> stresses. Trehalose applied to the plant with or without endophyte-inoculation also showed similar plant growth-promoting attributes under stress. Stress exposure significantly enhanced endogenous jasmonic (JA) and abscisic (ABA) acid contents in control plants. In contrast, Sphingomonas sp. LK11-inoculation significantly lowered ABA and JA levels in soybean plants, but these phytohormones increased in response to combined treatments during stress. The drought-induced osmotic stress <span class="trait">resistance</span> associated with Sphingomonas sp. LK11 and trehalose was also evidenced by increased mRNA gene expression of soybean dehydration responsive element binding protein (DREB)-type transcription factors <span class="gene">(GmDREBa</span> and <span class="gene">GmDREB2)</span> and the MYB (myeloblastosis) transcription factor <span class="gene">(GmMYBJ1)</span> as compared to the control. In conclusion, our findings demonstrated that inoculation with this endophyte and trehalose improved the negative effects of drought-induced osmotic stress, and it enhanced soybean plant growth and tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Control of Glyphosate-Resistant Common Ragweed ( <italic> Ambrosia artemisiifolia </italic> L.) in Glufosinate-Resistant Soybean [ <italic> Glycine max </italic> (L.) Merr]</h2>
			<h3>PMID:28868065</h3>
				

				
					<h3>PMCID:5563374</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Barnes </surname>  <given-names> Ethann R. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/461806/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Knezevic </surname>  <given-names> Stevan Z. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sikkema </surname>  <given-names> Peter H. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lindquist </surname>  <given-names> John L. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jhala </surname>  <given-names> Amit J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/426074/overview"/>  </contrib>
				<br/> 
				年份：<day> 04 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Common ragweed emerges early in the season in Nebraska, USA and is competitive with soybean; therefore, preplant herbicides are important for effective control. Glyphosate has been used as a preplant control option; however, confirmation of glyphosate-resistant (GR) common ragweed in Nebraska necessitates evaluating other herbicide options. The objectives of this study were to (1) evaluate the efficacy of preplant (PP) herbicides followed by (fb) glufosinate alone or in tank-mixture with imazethapyr, acetochlor, or S-metolachlor applied post-emergence (POST) for control of <span class="gene">GR</span> common ragweed in glufosinate-resistant soybean; (2) their effect on common ragweed density, biomass, and soybean <span class="trait">yield;</span> and (3) the partial economics of herbicide programs. A field experiment was conducted in a grower's field infested with <span class="gene">GR</span> common ragweed in Gage County, Nebraska, USA in 2015 and 2016. Preplant herbicide programs containing glufosinate, paraquat, 2,4-D, dimethenamid-P, cloransulam-methyl, or high rates of flumioxazin plus chlorimuron-ethyl provided 90-99% control of common ragweed at 21 d after treatment (DAT). The aforementioned PP herbicides fb a POST application of glufosinate alone or in tank-mixture with imazethapyr, acetochlor, or S-metolachlor controlled <span class="gene">GR</span> common ragweed 84-98% at soybean harvest, reduced common ragweed density (≤20 plants m-2) and biomass by ≥93%, and secured soybean <span class="trait">yield</span> 1,819-2,158 kg ha-1. The PP fb POST herbicide programs resulted in the highest gross profit margins (US$373-US$506) compared to PP alone (US$91) or PRE fb POST programs (US$158). The results of this study conclude that effective and economical control of <span class="gene">GR</span> common ragweed in glufosinate-resistant soybean is achievable with PP fb POST herbicide programs.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Glyphosate is a broad-spectrum, systemic, post-emergence (POST) herbicide (Duke and Powles, <xref rid="B12" ref-type="bibr"> 2008 </xref> ) first marketed in 1974 (Franz et al., <xref rid="B14" ref-type="bibr"> 1997 </xref> ). In 1996, <span class="gene">GR</span> soybean was first commercialized in the United States (Wiesbrook et al., <xref rid="B42" ref-type="bibr"> 2001 </xref> ), and as of 2016, commercially grown <span class="gene">GR</span> crops include alfalfa ( <italic> Medicago sativa </italic> L.), canola ( <italic> Brassica napus </italic> L.), corn ( <italic> Zea mays </italic> L.), cotton ( <italic> Gossypium hirsutum </italic> L.), soybean ( <italic> Glycine max </italic> L.), and sugarbeet ( <italic> Beta vulgaris </italic> L.) (Duke and Powles, <xref rid="B13" ref-type="bibr"> 2009 </xref> ). With the commercialization of <span class="gene">GR</span> crops, POST application of glyphosate increased dramatically (Dill, <xref rid="B11" ref-type="bibr"> 2005 </xref> ), resulting in the evolution of <span class="gene">GR</span> weeds. As of 2016, glyphosate <span class="trait">resistance</span> has been reported in 37 weed species globally, including 16 species in the United States (Heap, <xref rid="B20" ref-type="bibr"> 2017 </xref> ). Missouri was the first state to confirm <span class="gene">GR</span> common ragweed in 2004 (Pollard, <xref rid="B29" ref-type="bibr"> 2007 </xref> ; Heap, <xref rid="B20" ref-type="bibr"> 2017 </xref> ), and since then, <span class="gene">GR</span> common ragweed has been confirmed in 15 states in the United States and in Ontario, Canada (Heap, <xref rid="B20" ref-type="bibr"> 2017 </xref> ). <span class="gene">GR</span> common ragweed has been recently confirmed as the sixth <span class="gene">GR</span> weed in Nebraska, USA (Ganie and Jhala, <xref rid="B16" ref-type="bibr"> 2017 </xref> ). In response to widespread adoption of <span class="gene">GR</span> corn and soybean, and the effective, broad-spectrum, and affordable weed control with glyphosate, no-tillage, and reduced tillage production systems increased as the use of glyphosate replaced pre-plant tillage (Givens et al., <xref rid="B18" ref-type="bibr"> 2009 </xref> ). No-till soybean production reduces soil erosion and operating cost while providing comparable yields to conventional tillage systems (Stougaard et al., <xref rid="B34" ref-type="bibr"> 1984 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						There has been no study published in the scientific literature about control of <span class="gene">GR</span> common ragweed in glufosinate-resistant soybean. The objectives of this study were to evaluate the efficacy of PP herbicides fb glufosinate alone or in tank-mixture with acetochlor, imazethapyr, or <italic> S </italic> -metolachlor for control of <span class="gene">GR</span> common ragweed in glufosinate-resistant soybean, their effect on soybean injury and <span class="trait">yield,</span> and the economics of herbicide programs. The hypothesis for this study was that a PP application of an effective herbicide fb glufosinate will provide effective control of <span class="gene">GR</span> common ragweed in glufosinate-resistant soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Year-by-treatment interactions for <span class="gene">GR</span> common ragweed control, density, biomass, and soybean <span class="trait">yield</span> were not significant; therefore, data were combined. Average daily temperatures during the study were similar to the 30-year average (Table <xref ref-type="table" rid="T2"> 2 </xref> ). May and June of 2015 received higher precipitation (36.2 cm) than the 30-year average (18.6 cm); however, the 2016 growing season received similar precipitation to the 30-year average (16.3 cm in May and June; Table <xref ref-type="table" rid="T2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Preplant herbicide was followed by PRE and/or POST herbicide application for season-long control of <span class="gene">GR</span> common ragweed. A follow up application after PP was needed to avoid poor control and potential <span class="trait">yield</span> reduction. Glufosinate applied POST alone or in a tank-mixture with imazethapyr, acetochlor, or <italic> S </italic> -metolachlor controlled <span class="gene">GR</span> common ragweed 84&#x02013;98%. Tharp and Kells ( <xref rid="B36" ref-type="bibr"> 2002 </xref> ) reported that PRE herbicide followed by glufosinate controlled common ragweed, redroot pigweed ( <italic> Ameranthus retroflexus </italic> L.), and common lambsquarters &#x02265;92% at 28 DAT. Tharp and Kells ( <xref rid="B36" ref-type="bibr"> 2002 </xref> ) also reported that glufosinate tank-mixed with <italic> S </italic> -metolachlor or acetochlor controlled common ragweed &#x02265;99% at 28 DAT. Although a PP application fb sequential POST applications provided 99% control at 14 DALPOST, statistically it was comparable to PP fb single POST programs indicating that an effective PP herbicide fb a single POST application of glufosinate controlled <span class="gene">GR</span> common ragweed &#x0003e;90% and that a second POST application is not needed.
					</p>
				
					<p>&nbsp;&nbsp;
						Single application of PP herbicides in glufosinate-resistant soybean were unable to protect soybean <span class="trait">yield</span> potential, indicating that common ragweed can be extremely competitive in soybean fields if not controlled, or if controlled with only PP herbicide application without additional follow up treatments. Kaur et al. ( <xref rid="B24" ref-type="bibr"> 2014 </xref> ) reported 100% soybean <span class="trait">yield</span> reduction when <span class="gene">GR</span> giant ragweed was not controlled. Similarly, Jhala et al. ( <xref rid="B22" ref-type="bibr"> 2014 </xref> ) reported that a PP alone treatment resulted in 100% soybean <span class="trait">yield</span> reduction due to giant ragweed competition later in the season compared with PP fb POST programs. Furthermore, Aulakh and Jhala ( <xref rid="B3" ref-type="bibr"> 2015 </xref> ) reported that a single POST herbicide application was ineffective in protecting soybean <span class="trait">yield</span> potential.
					</p>
				
					<p>&nbsp;&nbsp;
						Results of this study conclude that PP herbicide options are available for early season control of <span class="gene">GR</span> common ragweed; however, a follow-up POST application of glufosinate alone or in tank-mixture is needed to achieve season-long control. Most of the PP herbicides tested in this study provided effective control (&#x0003e;95%) of common ragweed during soybean emergence and establishment. Furthermore, glufosinate can be used as an effective POST herbicide for control of <span class="gene">GR</span> common ragweed and can be tank-mixed with other herbicides such as <italic> S </italic> -metolachlor or acetochlor depending on the weed species present in the field. A recent update to the glufosinate (Liberty) label in the USA allows a maximum cumulative rate of 1,783 g ai ha <sup> &#x02212;1 </sup> per growing season. Two applications, each of 656&#x02013;881 g ai ha <sup> &#x02212;1 </sup> , could be made POST in glufosinate-resistant soybean before flowering (Anonymous, <xref rid="B2" ref-type="bibr"> 2017 </xref> ). Soybean yields were reduced when a PP application was not made compared to PP fb POST programs, primarily due to early season common ragweed competition. PP fb PRE fb POST programs did not decrease density, improve biomass reduction, or increase soybean <span class="trait">yield</span> compared to PP fb POST programs, suggesting that three time herbicide application is not necessary for controlling <span class="gene">GR</span> common ragweed or protecting soybean <span class="trait">yield.</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Robust RNAi-mediated <span class="trait">resistance</span> to infection of seven potyvirids in soybean expressing an intron hairpin <span class="gene">NIb</span> RNA.</h2>
			<h3>PMID:28840434</h3>
				

				
			<p>
				作者：Yang X(1), Niu L(1), Zhang W(1), He H(1), Yang J(1), Xing G(1), Guo D(1), Du Q(1), Qian X(1), Yao Y(1), Li Q(2), Dong Y(3).
				<br/> 
				年份：2017 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Viral pathogens, such as soybean mosaic virus (SMV), are a major constraint in soybean production and often cause significant <span class="trait">yield</span> loss and quality deterioration. Engineering <span class="trait">resistance</span> by RNAi-mediated gene silencing is a powerful strategy for controlling viral diseases. In this study, a 248-bp inverted repeat of the replicase (nuclear inclusion b, NIb) gene was isolated from the SMV SC3 strain, driven by the leaf-specific rbcS2 promoter from Phaseolus vulgaris, and introduced into soybean. The transgenic lines had significantly lower average disease indices (ranging from 2.14 to 12.35) than did the non-transformed (NT) control plants in three consecutive generations, exhibiting a stable and significantly enhanced <span class="trait">resistance</span> to the SMV SC3 strain under field conditions. Furthermore, seed mottling did not occur in transgenic seeds, whereas the NT plants produced ~90% mottled seeds. Virus <span class="trait">resistance</span> spectrum screening showed that the greenhouse-grown transgenic lines exhibited robust <span class="trait">resistance</span> to five SMV strains (SC3, SC7, SC15, SC18, and a recombinant SMV), bean common mosaic virus, and watermelon mosaic virus. Nevertheless, no significantly enhanced <span class="trait">resistance</span> to bean pod mottle virus (BPMV, Comovirus) was observed in the transgenic lines relative to their NT counterparts. Consistent with the results of <span class="trait">resistance</span> evaluation, the accumulation of each potyvirid (but not of BPMV) was significantly inhibited in the transgenic plants relative to the NT controls as confirmed by quantitative real-time (qRT-PCR) and double antibody sandwich enzyme-linked immunosorbent assay (DAS-ELISA). These results demonstrate that robust RNAi-mediated <span class="trait">resistance</span> to multiple potyvirids in soybean was conferred by expressing an intron hairpin SMV <span class="gene">NIb</span> RNA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine-mapping and identification of a novel locus Rsc15 underlying soybean <span class="trait">resistance</span> to Soybean mosaic virus.</h2>
			<h3>PMID:28825113  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rui R(1)(2), Liu S(1)(2), Karthikeyan A(1)(2), Wang T(1)(2), Niu H(1)(2), Yin J(1)(2), Yang Y(1)(2), Wang L(1)(2), Yang Q(1)(2), Zhi H(3)(4), Li K(5)(6).
				<br/> 
				年份：2017 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Rsc15, a novel locus underlying soybean <span class="trait">resistance</span> to SMV, was fine mapped to a 95-kb region on chromosome 6. The Rsc15- mediated <span class="trait">resistance</span> is likely attributed to the gene <span class="gene">GmPEX14</span> , the relative expression of which was highly correlated with the accumulation of H 2 O 2 along with the activities of POD and CAT during the early stages of SMV infection in RN-9. Soybean mosaic virus (SMV) causes severe <span class="trait">yield</span> losses and seed quality deterioration in soybean [Glycine max (L.) Merr.] worldwide. A series of single dominant SMV <span class="trait">resistance</span> genes have been identified on respective soybean chromosomes 2, 13 and 14, while one novel locus, Rsc15, underlying <span class="trait">resistance</span> to the virulent SMV strain SC15 from soybean cultivar RN-9 has been recently mapped to a 14.6-cM region on chromosome 6. However, candidate gene has not yet been identified within this region. In the present study, we aimed to fine map the Rsc15 region and identify candidate gene(s) for this invaluable locus. High-resolution fine-mapping revealed that the Rsc15 gene was located in a 95-kb genomic region which was flanked by the two simple sequence repeat (SSR) markers SSR_06_17 and BARCSOYSSR_06_0835. Allelic sequence comparison and expression profile analysis of candidate genes inferred that the gene <span class="gene">Glyma.06g182600</span> (designated as <span class="gene">GmPEX14)</span> was the best candidate gene attributing for the <span class="trait">resistance</span> of Rsc15, and that genes encoding receptor-like kinase (RLK) (i.e., <span class="gene">Glyma.06g175100</span> and <span class="gene">Glyma.06g184400)</span> and serine/threonine kinase (STK) (i.e., <span class="gene">Glyma.06g182900</span> and <span class="gene">Glyma.06g183500)</span> were also potential candidates. High correlations were established between the relative expression level of <span class="gene">GmPEX14</span> and the hydrogen peroxide (H2O2) concentration and activities of catalase (CAT) and peroxidase (POD) during the early stages of SMV-SC15 infection in RN-9. The results of the present study will be useful in marker-assisted breeding for SMV <span class="trait">resistance</span> and will lead to further understanding of the molecular mechanisms of host <span class="trait">resistance</span> against SMV.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Interaction of 2,4-D or Dicamba with Glufosinate for Control of Glyphosate-Resistant Giant Ragweed ( <italic> Ambrosia trifida </italic> L.) in Glufosinate-Resistant Maize ( <italic> Zea mays </italic> L.)</h2>
			<h3>PMID:28740503</h3>
				

				
					<h3>PMCID:5502257</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ganie </surname>  <given-names> Zahoor A. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/431434/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jhala </surname>  <given-names> Amit J. </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/426074/overview"/>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glyphosate-resistant (GR) giant ragweed is a problematic broadleaf weed in crops including maize and soybean in the Midwestern United States. Commercialization of crops with 2,4-D or dicamba and glufosinate <span class="trait">resistance</span> will allow post-emergence (POST) applications of these herbicides. Therefore, information is needed on how 2,4-D/dicamba will interact with glufosinate in various rate combinations. The objectives of this study were to evaluate the interaction of glufosinate plus 2,4-D and/or dicamba for control of <span class="gene">GR</span> giant ragweed, and to determine their effect on <span class="gene">GR</span> giant ragweed density, biomass, maize injury, and <span class="trait">yield.</span> Field experiments were conducted in 2013 and 2014 in a field infested with <span class="gene">GR</span> giant ragweed in Nebraska, United States. The treatments included POST applications of glufosinate (450 or 590 g ai ha-1), 2,4-D, or dicamba at 280 or 560 g ae ha-1 applied alone and in tank-mixtures in glufosinate-resistant maize. The results showed that dicamba applied alone resulted in 56 to 62% and 73 to 83% control at 14 and 28 days after treatment (DAT), respectively, and ≥95% control at 60 DAT or at harvest compared to 17 to 30% and 57 to 73% control with 2,4-D applied alone at 280 and 560 g ai ha-1, respectively. Glufosinate tank-mixed with 2,4-D and/or dicamba consistently provided ≥89% control of <span class="gene">GR</span> giant ragweed, except that control with glufosinate plus 2,4-D varied from 80 to 92% at 60 DAT and at harvest. The comparison between the observed and expected control (determined by Colby's equation) suggested an additive interaction between glufosinate and 2,4-D or dicamba for control of <span class="gene">GR</span> giant ragweed. Contrast analysis also indicated that <span class="gene">GR</span> giant ragweed control with glufosinate plus 2,4-D or dicamba was either consistently higher or comparable with individual herbicides excluding 2,4-D applied alone. Herbicide programs, excluding 2,4-D at 280 g ae ha-1, resulted in ≥80% reduction in <span class="gene">GR</span> giant ragweed density. Tank-mixing glufosinate with 2,4-D or dicamba showed an additive effect and will be an additional tool with two effective modes of action for the management of <span class="gene">GR</span> giant ragweed in maize.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Previous research has shown that a specific rate of the constituent active ingredients is needed for herbicide mixtures with synergistic interaction. For example, <xref rid="B35" ref-type="bibr"> Hugie et al. (2008) </xref> reported that a threshold mesotrione rate was needed to attain synergism between mesotrione and atrazine. Therefore, research is needed to determine the effect of individual herbicide rates on the type of interaction (additive, synergistic, or antagonistic) between glufosinate with 2,4-D and/or dicamba. The objectives of this study were to evaluate the efficacy of 2,4-D and/or dicamba plus glufosinate for control of <span class="gene">GR</span> giant ragweed, and to determine the effect of herbicide rate combinations on the type of interaction between 2,4-D and/or dicamba plus glufosinate on giant ragweed control, density, biomass, maize injury, and <span class="trait">yield.</span> We hypothesized that an additive interaction between synthetic auxins (2,4-D or dicamba) and glufosinate for <span class="gene">GR</span> giant ragweed control will be achieved when constituent herbicides are used at the labeled rate in tank-mixture.
					</p>
				
					<p>&nbsp;&nbsp;
						Field experiments were conducted at Clay Center (40.52&#x000b0;N, 98.05&#x000b0;W) and David City (41.25&#x000b0;N, 97.13&#x000b0;W), Nebraska in 2013 and 2014, respectively, in growers&#x02019; fields infested with <span class="gene">GR</span> giant ragweed. Giant ragweed biotypes from these sites were confirmed resistant to glyphosate in 2011, with 14-fold <span class="trait">resistance</span> compared to glyphosate-susceptible biotypes included for comparison ( <xref rid="B52" ref-type="bibr"> Rana et al., 2013 </xref> ). The density of <span class="gene">GR</span> giant ragweed at these sites varied from 18 to 30 plants m <sup> -2 </sup> . The soil type at Clay Center was fine, smectitic, mesic Vertic Argiaquolls (Butler series) with a silt loam texture (17% sand, 58% silt, 25% clay), 2.5% organic matter, and a pH of 6.5. The soil type at David City was fine, smectitic, mesic Udic Argiustolls (Hastings series) with a silty clay loam texture (18% sand, 50% silt, 32% clay), 2.1% organic matter, and a pH of 5.4. The experiment was arranged in a randomized complete block design with 18 treatments and four replications. The treatments included POST applications of glufosinate (450 or 590 g ai ha <sup> -1 </sup> ) (Liberty 280, Bayer Crop Science, Research Triangle Park, NC, United States), 2,4-D amine (280 or 560 g ae ha <sup> -1 </sup> ) (2,4-D Amine, Winfield Solutions, LLC, St Paul, MN, United States), and dicamba (280 or 560 g ae ha <sup> -1 </sup> ) (Clarity, BASF Corporation, Research Triangle Park, NC, United States) alone and in tank-mixtures ( <bold> Table <xref ref-type="table" rid="T1"> 1 </xref> </bold> ). Treatment with no herbicide application served as a non-treated control for comparison. Maize seeds (Cv. &#x0201c;Pioneer 1151AM&#x0201d; in 2013 and &#x0201c;Mycogen 2V709&#x0201d; in 2014) with <span class="trait">resistance</span> to both glyphosate and glufosinate were planted on May 16, 2013 and May 17, 2014. The seeds were planted 3 cm deep at a density of 79,000 seeds ha <sup> -1 </sup> . Individual plots were 3 m wide and 9 m long with four maize rows spaced 76 cm apart.
					</p>
				
					<p>&nbsp;&nbsp;
						Results of this study indicated that tank-mixing glufosinate with 2,4-D or dicamba showed an additive interaction and provided an effective POST option for the control of <span class="gene">GR</span> giant ragweed in maize and secured optimum <span class="trait">yield.</span> Although results of this study reported excellent control of giant ragweed with 2,4-D/dicamba tank-mixed with glufosinate, a diverse weed management program should be adopted by growers, because relying on these herbicides, particularly applied alone, may result in the evolution of resistant weeds. For example, 2,4-D-resistant common waterhemp in eastern Nebraska ( <xref rid="B9" ref-type="bibr"> Bernards et al., 2012 </xref> ) and dicamba-resistant kochia in western Nebraska ( <xref rid="B19" ref-type="bibr"> Crespo et al., 2014 </xref> ) have been confirmed. Similarly, glufosinate <span class="trait">resistance</span> has been reported in few weed species including goosegrass, Italian ryegrass [ <italic> Lolium perenne </italic> L. ssp. <italic> multiflorum </italic> (Lam.) Husnot], and perennial ryegrass ( <italic> Lolium perenne </italic> L.) ( <xref rid="B37" ref-type="bibr"> Jalaludin et al., 2017 </xref> ). Thus, over-reliance on 2,4-D, glufosinate, or dicamba should be avoided and a diversity of herbicide chemistries must be maintained by using herbicide tank-mixtures with multiple effective modes of action, along with non-chemical weed control methods including crop rotation, tillage, competitive cultivars, weed seed destruction, and cover crops, among others ( <xref rid="B50" ref-type="bibr"> Norsworthy et al., 2012 </xref> ).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Canadian Potential Healthcare and Societal Cost Savings from Consumption of Pulses: A Cost-Of-Illness Analysis</h2>
			<h3>PMID:28737688</h3>
				

				
					<h3>PMCID:5537906</h3>
					<h4>期刊:Nutrients</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Abdullah </surname>  <given-names> Mohammad M. H. </given-names>  </name>  <xref ref-type="aff" rid="af1-nutrients-09-00793"> 1 </xref>  <xref ref-type="aff" rid="af2-nutrients-09-00793"> 2 </xref>  <xref ref-type="author-notes" rid="fn1-nutrients-09-00793"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marinangeli </surname>  <given-names> Christopher P. F. </given-names>  </name>  <xref ref-type="aff" rid="af3-nutrients-09-00793"> 3 </xref>  <xref ref-type="author-notes" rid="fn1-nutrients-09-00793"> &#x02020; </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jones </surname>  <given-names> Peter J. H. </given-names>  </name>  <xref ref-type="aff" rid="af1-nutrients-09-00793"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Carlberg </surname>  <given-names> Jared G. </given-names>  </name>  <xref ref-type="aff" rid="af4-nutrients-09-00793"> 4 </xref>  <xref rid="c1-nutrients-09-00793" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 7 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Consumption of dietary pulses, including beans, peas and lentils, is recommended by health authorities across jurisdictions for their nutritional value and effectiveness in helping to prevent and manage major diet-related illnesses of significant socioeconomic burden. The aim of this study was to estimate the potential annual healthcare and societal cost savings relevant to rates of reduction in complications from type 2 diabetes (T2D) and incidence of cardiovascular disease (CVD) following a low glycemic index (GI) or high fiber diet that includes pulses, or 100 g/day pulse intake in Canada, respectively. A four-step cost-of-illness analysis was conducted to: (1) estimate the proportions of individuals who are likely to consume pulses; (2) evaluate the reductions in established risk factors for T2D and CVD; (3) assess the percent reduction in incidences or complications of the diseases of interest; and (4) calculate the potential annual savings in relevant healthcare and related costs. A low GI or high fiber diet that includes pulses and 100 g/day pulse intake were shown to potentially <span class="trait">yield</span> Can$6.2 (95% <span class="gene">CI</span> $2.6-$9.9) to Can$62.4 (95% <span class="gene">CI</span> $26-$98.8) and Can$31.6 (95% <span class="gene">CI</span> $11.1-$52) to Can$315.5 (95% <span class="gene">CI</span> $110.6-$520.4) million in savings on annual healthcare and related costs of T2D and CVD, respectively. Specific provincial/territorial analyses suggested annual T2D and CVD related cost savings that ranged from up to Can$0.2 million in some provinces to up to Can$135 million in others. In conclusion, with regular consumption of pulse crops, there is a potential opportunity to facilitate T2D and CVD related socioeconomic cost savings that could be applied to Canadian healthcare or re-assigned to other priority domains. Whether these potential cost savings will be offset by other healthcare costs associated with longevity and diseases of the elderly is to be investigated over the long term.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Influence of supplemental canola or soybean oil on milk <span class="trait">yield,</span> fatty acid profile and postpartum weight changes in grazing dairy goats.</h2>
			<h3>PMID:28728359</h3>
				

				
			<p>
				作者：Lerma-Reyes I(1), Mendoza-Martínez GD(2), Rojo-Rubio R(3), Mejia M(4), García-Lopez JC(5), Lee-Rangel HA(1).
				<br/> 
				年份：2018 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						OBJECTIVE: This experiment was designed to evaluate the effect of supplementation with soybean or canola oil on milk production and the composition of long chain fatty acids as well as weight changes in the goats and their kids. METHODS: Thirty nine mulitparous crossed Alpine×Nubian goats (initial body weight [BW] 43.5±1.7 kg) from the day of parturition were assigned to the treatments: grazing control (n = 15); grazing plus 20 mL/goat/d of supplemental soybean oil (n = 12); and grazing plus 20 mL/goat/d of supplemental canola oil (n = 12) from November 26, 2014 to March 7, 2015. The planned contrasts were: <span class="gene">CI</span> (control vs supplemented with oils); CII (soybean vs canola oil) to compare the treatment effects. RESULTS: The vegetable oil supplementation reduced weight losses in lactating goats (CI: -0.060 vs 0.090 kg/d; p = 0.03) but did not improve milk production or affect kids' growth. The content of C4, C6, C8, C10, C11, C14, and C18:1n9t in the milk was increased (p&lt;0.05) with respect to control. However, C12, C14, C16, C18, C18:1n9c, C18:2n6c, and C18:3n3 were reduced (p&lt;0.05) in supplemented goats. Conjugated linoleic acid (CLA) was increased (p&lt;0.05) in goats supplemented with oils compared to the control group. CONCLUSION: Supplementation with 20 mL/d of soybean or canola oil did not affect milk production or kids' performance; however, it increased CLA concentration and reduced the reduced weight losses in lactating goats.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular characterization of the acquisition of longevity during seed maturation in soybean</h2>
			<h3>PMID:28700604</h3>
				

				
					<h3>PMCID:5507495</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Pereira Lima </surname>  <given-names> Juliana Joice </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Buitink </surname>  <given-names> Julia </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lalanne </surname>  <given-names> David </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rossi </surname>  <given-names> Rubiana Falopa </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pelletier </surname>  <given-names> Sandra </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> da Silva </surname>  <given-names> Edvaldo Aparecido Amaral </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0003-1414-8690 </contrib-id>  <name>  <surname> Leprince </surname>  <given-names> Olivier </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed longevity, defined as the ability to remain alive during storage, is an important agronomic factor. Poor longevity negatively impacts seedling establishment and consequently crop <span class="trait">yield.</span> This is particularly problematic for soybean as seeds have a short lifespan. While the economic importance of soybean has fueled a large number of transcriptome studies during embryogenesis and seed filling, the mechanisms regulating seed longevity during late maturation remain poorly understood. Here, a detailed physiological and molecular characterization of late seed maturation was performed in soybean to obtain a comprehensive overview of the regulatory genes that are potentially involved in longevity. Longevity appeared at physiological maturity at the end of seed filling before maturation drying and progressively doubled until the seeds reached the dry state. The increase in longevity was associated with the expression of genes encoding protective chaperones such as heat shock proteins and the repression of nuclear and chloroplast genes involved in a range of chloroplast activities, including photosynthesis. An increase in the raffinose family oligosaccharides (RFO)/sucrose ratio together with changes in RFO metabolism genes was also associated with longevity. A gene co-expression network analysis revealed 27 transcription factors whose expression profiles were highly correlated with longevity. Eight of them were previously identified in the longevity network of Medicago truncatula, including homologues of ERF110, HSF6AB, NFXL1 and members of the <span class="gene">DREB2</span> family. The network also contained several transcription factors associated with auxin and developmental cell fate during flowering, organ growth and differentiation. A transcriptional transition occurred concomitant with seed chlorophyll loss and detachment from the mother plant, suggesting the activation of a post-abscission program. This transition was enriched with AP2/EREBP and WRKY transcription factors and genes associated with growth, germination and post-transcriptional processes, suggesting that this program prepares the seed for the dry quiescent state and germination.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Furthermore, an overrepresentation was found of processes related to water deprivation and salt stress ( <xref ref-type="table" rid="pone.0180282.t004"> Table 4 </xref> ), which included genes involved in RFO metabolism such as galactinol synthase <span class="gene">(Glyma.03g229800,</span> <span class="gene">Glyma.20g094500),</span> raffinose synthase <span class="gene">(Glyma.06g179200).</span> In the GO term &#x0201c;response to water deprivation&#x0201d;; we noticed two Arabidopsis homologues of a RING domain ligase, namely RGLG1 and RGLG2, that have a ubiquitin E3 ligase activity and mediate the transcription of AtERF53 in response to <span class="trait">drought</span> [ <xref rid="pone.0180282.ref046" ref-type="bibr"> 46 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The network analysis revealed 27 TF whose transcript profiles are correlated with P50 ( <xref ref-type="fig" rid="pone.0180282.g006"> Fig 6 </xref> , <xref ref-type="supplementary-material" rid="pone.0180282.s005"> S4 Table </xref> ), thereby forming a longevity module similar to that found in developing Arabidopsis and <italic> M </italic> . <italic> truncatula </italic> [ <xref rid="pone.0180282.ref023" ref-type="bibr"> 23 </xref> , <xref rid="pone.0180282.ref024" ref-type="bibr"> 24 </xref> ]. Indeed, the homologues of 7 soybean genes were also present in the list of the 9 TF belonging to Medicago longevity module [ <xref rid="pone.0180282.ref024" ref-type="bibr"> 24 </xref> ], namely three homologues of an ERF110 of Arabidopsis <span class="gene">(Glyma.06g148400,</span> <span class="gene">Glyma.04g217400,</span> <span class="gene">Glyma.08g145300),</span> a homologue of the ETHYLENE INSENSITIVE PROTEIN 3 family <span class="gene">(Glyma.05g180300),</span> two homologues of the <span class="gene">DREB2</span> family <span class="gene">(Glyma.07g156200,</span> <span class="gene">Glyma.14g056200),</span> and a homologue of NF-X-like1 gene <span class="gene">(Glyma.09g173000).</span> The implication of two of these genes in longevity has been demonstrated [ <xref rid="pone.0180282.ref055" ref-type="bibr"> 55 </xref> , <xref rid="pone.0180282.ref024" ref-type="bibr"> 24 </xref> ]. A <span class="gene">DREB2</span> from sunflower enhanced seed longevity of tobacco when ectopically over-expressed with a heat shock factor, HaHSFA9 [ <xref rid="pone.0180282.ref055" ref-type="bibr"> 55 </xref> ]. In Arabidopsis, seeds of <italic> nfxl1 </italic> exhibited impaired acquisition of longevity during maturation [ <xref rid="pone.0180282.ref024" ref-type="bibr"> 24 </xref> ]. When overexpressed in vegetative tissues, NFXL1 induces a higher survival upon salt stress, <span class="trait">drought</span> and high light intensity [ <xref rid="pone.0180282.ref056" ref-type="bibr"> 56 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The genetic architecture of water-soluble protein content and its genetic relationship to total protein content in soybean.</h2>
			<h3>PMID:28698580</h3>
				

				
			<p>
				作者：Zhang D(1), Lü H(2), Chu S(2), Zhang H(3), Zhang H(4), Yang Y(5), Li H(2), Yu D(6).
				<br/> 
				年份：2017 Jul 11
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Water-soluble protein content (WSPC) is a critical factor in both soybean protein quality and functionality. However, the underlying genetic determinants are unclear. Here, we used 219 soybean accessions and 152 recombinant inbred lines genotyped with high-density markers and phenotyped in multi-environments to dissect the genetic architectures of WSPC and protein content (PC) using single- and multi-locus genome-wide association studies. In the result, a total of 32 significant loci, including 10 novel loci, significantly associated with WSPC and PC across multi-environments were identified, which were subsequently validated by linkage mapping. Among these loci, only four exhibited pleiotropic effects for PC and WSPC, explaining the low correlation coefficient between the two traits. The largest-effect WSPC-specific loci, GqWSPC8, was stably identified across all six environments and tagged to a linkage disequilibrium block comprising two promising candidate genes AAP8 and 2 S albumin, which might contribute to the high level of WSPC in some soybean varieties. In addition, two genes, <span class="gene">Glyma.13G123500</span> and <span class="gene">Glyma.13G194400</span> with relatively high expression levels at seed development stage compared with other tissues were regarded as promising candidates associated with the PC and WSPC, respectively. Our results provide new insights into the genetic basis of WSPC affecting soybean protein quality and <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tissue-Specific Regulation of <span class="gene">Gma-miR396</span> Family on Coordinating Development and Low Water Availability Responses</h2>
			<h3>PMID:28694817</h3>
				

				
					<h3>PMCID:5483475</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Weican </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/400748/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Yonggang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xiaowei </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xingchao </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Yuanyuan </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Nan </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xiuming </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Huan </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yao </surname>  <given-names> Na </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cui </surname>  <given-names> Xiyan </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jameel </surname>  <given-names> Aysha </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Fawei </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/451424/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Haiyan </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previously, it was reported that miR396s interact with growth-regulating factors (GRFs) to modulate plant growth, development, and stress <span class="trait">resistance.</span> In soybean, 11 gma-miR396 precursors (Pre-miR396a-k) were found, and 24 <span class="gene">GmGRFs</span> were predicted as targets of seven mature gma-miR396s (gma-miR396a/b/c/e/h/i/k). To explore the roles of the miR396-GRF module in low water availability response of soybean, we analyzed the expression of Pre-miR396a-k, and found that Pre-miR396a/i/bdgk/e/h were up-regulated in leaves and down-regulated in roots; on the contrary, <span class="gene">GmGRF5/6/7/8/15/17/21</span> were down-regulated in leaves and <span class="gene">GmGRF1/2/17/18/19/20/21/22/23/24</span> were up-regulated in roots of low water potential stressed soybean. Any one of gma-miR396a/b/c/e/h/i/k was able to interact with 20 <span class="gene">GmGRFs</span> <span class="gene">(GmGRF1/2/6-11/13-24),</span> confirming that this module represents a multi-to-multi network interaction. We generated Arabidopsis plants over-expressing each of the 11 gma-miR396 precursors (Pre-miR396a-k), and seven of them (miR396a/b/c/e/h/i/k-OE transgenic Arabidopsis) showed altered development. The low water availability of miR396a/b/c/e/h/i/k-OE was enhanced in leaves but reduced in seeds and roots. Contrary to previous reports, miR396a/b/c/i-OE seedlings showed lower survival rate than WT when recovering after rewatering under soil drying. In general, we believe our findings are valuable to understand the role of gma-miR396 family in coordinating development and low water availability responses, and can provide potential strategies and directions for soybean breeding programs to improve seed <span class="trait">yield</span> and plant <span class="trait">drought</span> tolerance.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> ) is one of the most important legume crops, and the most drought-sensitive crop plant in the legume family ( <xref rid="B10" ref-type="bibr"> Clement et al., 2008 </xref> ). To date, 11 <italic> Glycine max </italic> gma-miR396 precursors (Pre-miR396a&#x02013;k) have been identified, and 24 <italic> Glycine max GRFs </italic> ( <italic> <span class="gene">GmGRF1&#x02013;GmGRF24</span> </italic> ) were predicted as the targets of seven gma-miR396a/b/c/e/h/i/k. These miR396s and <italic> <span class="gene">GmGRFs</span> </italic> were identified from sequencing data and annotated by an automated computational analysis ( <xref rid="B35" ref-type="bibr"> Schmutz et al., 2010 </xref> ; <xref rid="B23" ref-type="bibr"> Kulcheski et al., 2011 </xref> ), but their functions have not been confirmed experimentally. In this study, we confirmed that 20 <italic> <span class="gene">GmGRFs</span> </italic> are targeted by seven gma-miR396s (gma-miR396a/b/c/e/h/i/k) in soybean using two experimental methods. We verified the function of the gma-miR396 gene family by over-expressing Pre-miR396a&#x02013;k in <italic> Arabidop </italic> sis (miR396a/b/c/d/e/f/g/h/i/k-OE), seven gma-miR396 precursors (Pre-miR396a/b/c/e/h/i/k) affected the development of miR396a/b/c/e/h/i/k-OE transgenic <italic> Arabidopsis </italic> plants, which exhibited small leaves, short roots, and decreased seed <span class="trait">yield.</span> Interestingly, Pre-miR396a/i/bdgk/e/h showed opposite regulation responses in leaves and roots of low water potential stressed soybean. The low water availability of miR396a/b/c/e/h/i/k-OE transgenic plants differed among tissues, enhanced in the small leaves, but reduced in the short roots and transgenic seeds. Contrary to previous reports, miR396a/b/c/i-OE transgenic seedlings in soil drying response displayed more lower survival rate than that of wild type (WT). Together, these data contribute to our understanding of the tissue-specific regulation of the gma-miR396 family in coordinating development and low water availability responses. This information provides directions for soybean breeding programs to improve seed <span class="trait">yield</span> and <span class="trait">drought</span> tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						We also explored the regulatory mechanism of the miR396&#x02013; <italic> GRF </italic> regulatory module in soybean in low water availability response. Under low water potential stress, soybean seedlings formed smaller leaves and longer roots (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> S1 </xref> ), both are important adaptive traits when the plants response to low water potential stress ( <xref rid="B39" ref-type="bibr"> Verslues et al., 2006 </xref> ). Meanwhile, we detected up-regulated expression of Pre-miR396a/i/bdgk/e/h and down-regulated expression of <italic> <span class="gene">GmGRF5/6/7/8/15/17/21</span> </italic> in leaves of soybean under low water potential stress; conversely, in roots of drought-stressed soybean, there was down-regulated expression of Pre-miR396a/i/bdgk/e/h and up-regulated expression of <italic> <span class="gene">GmGRF1/2/17/18/19/20/21/23/24.</span> </italic> Furthermore, we further verified the function of <italic> Glycine max </italic> miR396-family genes by over-expressing Pre-miR396a&#x02013;k in <italic> Arabidopsis </italic> , and found miR396a/b/c/e/h/i/k-OE <italic> Arabidopsis </italic> seedlings all exhibited shorter roots and smaller leaves. Taken together, our results suggested that Pre-miR396a/b/c/e/h/i/k are positive regulatory factors in leaves of soybean in low water availability response. They probably inhibit leaf growth by targeting <italic> <span class="gene">GmGRFs</span> </italic> to decrease water loss as an adaptation to the low water availability environment. In contrast, Pre-miR396a/b/c/e/h/i/k functioned as negative regulatory factors in roots during the soybean seedling response to low water availability stress, root growth was promoted by enhancing targeted <italic> GRFs </italic> expression to increase the water absorption from soil to adapt to the low water availability environment. Overall, our results illustrate the tissue-specific regulation of the gma-miR396 family in coordinating development and low water availability response. This information will provide potential strategies and directions for soybean breeding programs to improve <span class="trait">drought</span> tolerance.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Over-expression of <span class="gene">GmSN1</span> enhances virus <span class="trait">resistance</span> in Arabidopsis and soybean.</h2>
			<h3>PMID:28656325</h3>
				

				
			<p>
				作者：He H(1), Yang X(2), Xun H(1), Lou X(1), Li S(1), Zhang Z(1), Jiang L(1), Dong Y(2), Wang S(3), Pang J(4), Liu B(1).
				<br/> 
				年份：2017 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: <span class="gene">GmSN1</span> enhances virus <span class="trait">resistance</span> in plants most likely by affecting the expression of signal transduction and immune response genes. Soybean mosaic virus (SMV) infection causes severe symptom and leads to massive <span class="trait">yield</span> loss in soybean (Glycine max). By comparative analyzing gene expression in the SMV-resistant soybean cultivar Rsmv1 and the susceptible cultivar Ssmv1 at a transcriptome level, we found that a subgroup of Gibberellic Acid Stimulated Transcript (GAST) genes were down-regulated in SMV inoculated Ssmv1 plants, but not Rsmv1 plants. Sequence alignment and phylogenetic analysis indicated that one of the GAST genes, <span class="gene">GmSN1,</span> was closely related to Snakin-1, a well-characterized potato microbial disease <span class="trait">resistance</span> gene. When over-expressed in Arabidopsis and soybean, respectively, under the control of the 35S promoter, <span class="gene">GmSN1</span> enhanced turnip mosaic virus <span class="trait">resistance</span> in the transgenic Arabidopsis plants, and SMV <span class="trait">resistance</span> in the transgenic soybean plants, respectively. Transcriptome analysis results showed that the up-regulated genes in the 35S:GmSN1 transgenic Arabidopsis plants were largely enriched in functional terms including "signal transduction" and "immune response". Real-time PCR assay indicated that the expression of <span class="gene">GmAKT2,</span> a potassium channel gene known to enhance SMV <span class="trait">resistance</span> when over-expressed in soybean, was elevated in the 35S:GmSN1 transgenic soybean plants. Taken together, our results suggest that <span class="gene">GmSN1</span> enhances virus <span class="trait">resistance</span> in plants most likely by affecting the expression of signal transduction and immune response genes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide identification of Major Intrinsic Proteins in <italic> Glycine soja </italic> and characterization of <span class="gene">GmTIP2;1</span> function under salt and water stress</h2>
			<h3>PMID:28646139</h3>
				

				
					<h3>PMCID:5482899</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Da-yong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Kumar </surname>  <given-names> Manoj </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wan </surname>  <given-names> Qun </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Yi-hong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhao-Long </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Xiao-Lan </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Jin-Biao </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-9180-0924 </contrib-id>  <name>  <surname> Pandey </surname>  <given-names> Girdhar K. </given-names>  </name>  <address>  <email> gkpandey@south.du.ac.in </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Shao </surname>  <given-names> Hong-Bo </given-names>  </name>  <address>  <email> shaohongbochu@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0017 5204 </institution-id>  <institution-id institution-id-type="GRID"> grid.454840.9 </institution-id>  <institution> Salt-soil Agricultural Center, Institute of Agricultural Resources and Environment,  </institution>  <institution> Jiangsu Academy of Agricultural Sciences,  </institution>  </institution-wrap> Zhongling Street No.50, Nanjing, 210014 China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2109 4999 </institution-id>  <institution-id institution-id-type="GRID"> grid.8195.5 </institution-id>  <institution> Department of Plant Molecular Biology,  </institution>  <institution> University of Delhi South Campus,  </institution>  </institution-wrap> Benito Juarez Road, Dhaula Kuan, New Delhi, 110021 India  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI">  0000 0001 0038 6319 </institution-id>  <institution-id institution-id-type="GRID"> grid.458469.2 </institution-id>  <institution> Key Laboratory of Biogeography and Bioresources in Arid Land,  </institution>  <institution> Xinjiang Institute of Ecology and Geography, Chinese Academy of Sciences Urumqi,  </institution>  </institution-wrap> Urumqi, China  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1791 6031 </institution-id>  <institution-id institution-id-type="GRID"> grid.443649.8 </institution-id>  <institution> JLCBE,  </institution>  <institution> Yancheng Teachers University,  </institution>  </institution-wrap> Xiwang Avenue 1, Yancheng, 224002 China  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0498 924X </institution-id>  <institution-id institution-id-type="GRID"> grid.10706.30 </institution-id>  <institution> School of Biotechnology,  </institution>  <institution> Jawaharlal Nehru University,  </institution>  </institution-wrap> New Delhi, 110067 India  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In different plant species, aquaporins (AQPs) facilitate water movement by regulating root hydraulic conductivity under diverse stress conditions such as salt and water stresses. To improve survival and <span class="trait">yield</span> of crop plants, a detailed understanding of stress responses is imperative and required. We used Glycine soja genome as a tool to study AQPs, considering it shows abundant genetic diversity and higher salt environment tolerance features and identified 62 Gs AQP genes. Additionally, this study identifies major aquaporins responsive to salt and <span class="trait">drought</span> stresses in soybean and elucidates their mode of action through yeast two-hybrid assay and BiFC. Under stress condition, the expression analysis of AQPs in roots and leaves of two contrasting ecotypes of soybean revealed diverse expression patterns suggesting complex regulation at transcriptional level. Based on expression analysis, we identify <span class="gene">GmTIP2;1</span> as a potential candidate involved in salinity and <span class="trait">drought</span> responses. The overexpression of <span class="gene">GmTIP2;1</span> in Saccharomyces cerevisiae as well as in-planta enhanced salt and <span class="trait">drought</span> tolerance. We identified that <span class="gene">GmTIP2;1</span> forms homodimers as well as interacts with <span class="gene">GmTIP1;7</span> and <span class="gene">GmTIP1;8.</span> This study augments our knowledge of stress responsive pathways and also establishes <span class="gene">GmTIP2;1</span> as a new stress responsive gene in imparting salt stress tolerance in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Overexpression of <italic> <span class="gene">GmTIP2;1</span> </italic> , <italic> <span class="gene">GmTIP1;7</span> </italic> and <italic> <span class="gene">GmTIP1;8</span> </italic> improved salt and <span class="trait">drought</span> tolerance in <italic> Saccharomyces cerevisiae </italic> and soybean. ( <bold> a </bold> ) Various combination of overexpressing yeast cells were plated on 30% PEG 6000 and 5&#x02009;M NaCl into SC-Ura medium. According to the survival patterns of the yeast cells, it is visible that overexpression of <span class="gene">GmTIP2;1</span> improved salt and <span class="trait">drought</span> tolerance in <italic> Saccharomyces cerevisiae </italic> . ( <bold> b </bold> , <bold> c </bold> ) Growth curves depict analysis of the growth of yeast strains overexpressing the indicated proteins in YPDA medium supplemented with NaCl (0.5&#x02009;M) and PEG (10%). X-axis shows time and Y-axis shows OD600 values. OD were measured every 2&#x02009;h. ( <bold> d </bold> ) Phenotypic analysis of the hairy root transformed transgenic plants treated by 200&#x02009;mM NaCl solution for seven days. The photograph was documented after 7 days of growth of these respective seedlings in hydroponic media containing 200&#x02009;mM NaCl. <bold> (e) </bold> Real-time RT-PCR analysis of <italic> <span class="gene">GmTIP2;1</span> </italic> , <italic> <span class="gene">GmTIP1;7</span> </italic> and <italic> <span class="gene">GmTIP1;8</span> </italic> under <span class="trait">drought</span> stress: cultivated soybean seedlings were treated with 30% PEG 6000 for 0, 2, 4 and 8&#x02009;h (P0&#x02013;8)&#x000a0;and relative expression of <italic> <span class="gene">GmTIP2;1</span> </italic> , <italic> <span class="gene">GmTIP1;7</span> </italic> and <italic> <span class="gene">GmTIP1;8</span> </italic> were plotted compared to control.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Metabolomic Profiling of Soybeans ( <italic> Glycine max </italic> L.) Reveals the Importance of Sugar and Nitrogen Metabolism under <span class="trait">Drought</span> and Heat Stress</h2>
			<h3>PMID:28587097</h3>
				

				
					<h3>PMCID:5489793</h3>
					<h4>期刊:Plants</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Das </surname>  <given-names> Aayudh </given-names>  </name>  <xref ref-type="aff" rid="af1-plants-06-00021"> 1 </xref>  <xref ref-type="aff" rid="af2-plants-06-00021"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rushton </surname>  <given-names> Paul J. </given-names>  </name>  <xref ref-type="aff" rid="af1-plants-06-00021"> 1 </xref>  <xref ref-type="aff" rid="af3-plants-06-00021"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rohila </surname>  <given-names> Jai S. </given-names>  </name>  <xref ref-type="aff" rid="af1-plants-06-00021"> 1 </xref>  <xref ref-type="aff" rid="af4-plants-06-00021"> 4 </xref>  <xref rid="c1-plants-06-00021" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 22 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is an important crop that is continually threatened by abiotic stresses, especially <span class="trait">drought</span> and heat stress. At molecular levels, reduced yields due to <span class="trait">drought</span> and heat stress can be seen as a result of alterations in metabolic homeostasis of vegetative tissues. At present an incomplete understanding of abiotic stress-associated metabolism and identification of associated metabolites remains a major <span class="gene">gap</span> in soybean stress research. A study with a goal to profile leaf metabolites under control conditions (28/24 °C), <span class="trait">drought</span> [28/24 °C, 10% volumetric water content (VWC)], and heat stress (43/35 °C) was conducted in a controlled environment. Analyses of non-targeted metabolomic data showed that in response to <span class="trait">drought</span> and heat stress, key metabolites (carbohydrates, amino acids, lipids, cofactors, nucleotides, peptides and secondary metabolites) were differentially accumulated in soybean leaves. The metabolites for various cellular processes, such as glycolysis, the tricarboxylic acid (TCA) cycle, the pentose phosphate pathway, and starch biosynthesis, that regulate carbohydrate metabolism, amino acid metabolism, peptide metabolism, and purine and pyrimidine biosynthesis, were found to be affected by <span class="trait">drought</span> as well as heat stress. Computationally based regulatory networks predicted additional compounds that address the possibility of other metabolites and metabolic pathways that could also be important for soybean under <span class="trait">drought</span> and heat stress conditions. Metabolomic profiling demonstrated that in soybeans, keeping up with sugar and nitrogen metabolism is of prime significance, along with phytochemical metabolism under <span class="trait">drought</span> and heat stress conditions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Environmental stresses, such as <span class="trait">drought</span> and heat, are serious threats to soybean cultivation in the US and in other soybean growing countries, causing multi-billion dollar losses every year. Stress in plants can arise as a reduction in growth and development that could be the result of disrupted metabolic homeostasis which requires the temporal readjustment of several metabolic pathways [ <xref rid="B1-plants-06-00021" ref-type="bibr"> 1 </xref> ]. Under abiotic stress it is a general trend that growth and <span class="trait">yield</span> of crops are reduced below optimum levels [ <xref rid="B2-plants-06-00021" ref-type="bibr"> 2 </xref> ]. Deryng et al. (2014) showed that by the year 2080, soybean yields at global level may reduce by a quarter ( <italic> &#x00394;Y </italic> = 15.3 &#x000b1; 26.5% versus 20.4 &#x000b1; 22.1% without heat stress) as a result of heat stress [ <xref rid="B3-plants-06-00021" ref-type="bibr"> 3 </xref> ]. The growth, development and yields of soybean are significantly affected by <span class="trait">drought</span> and heat stress; thus, in mid-south United States, an alternative production method, i.e., the Early Soybean Production System, is in practice to help farmers to shift the planting and harvest times by incorporating maturity group III&#x02013;V in soybean cultivation to minimize late season <span class="trait">drought</span> stress effects on <span class="trait">yield</span> [ <xref rid="B4-plants-06-00021" ref-type="bibr"> 4 </xref> ]. Further, it has also been shown that abiotic stresses are directly related to changes in various cellular metabolic pathways [ <xref rid="B4-plants-06-00021" ref-type="bibr"> 4 </xref> , <xref rid="B5-plants-06-00021" ref-type="bibr"> 5 </xref> , <xref rid="B6-plants-06-00021" ref-type="bibr"> 6 </xref> , <xref rid="B7-plants-06-00021" ref-type="bibr"> 7 </xref> , <xref rid="B8-plants-06-00021" ref-type="bibr"> 8 </xref> ], including carbohydrate, amino acid, and peptide metabolism [ <xref rid="B9-plants-06-00021" ref-type="bibr"> 9 </xref> ]. Hence, abiotic stresses that alter the metabolomic profiles of soybeans leaf tissues (source) could reduce its <span class="trait">yield</span> (sink) potential. In the past, genetic and agronomic advancements have enhanced the <span class="trait">drought</span> tolerance of crop plants, and have resulted in stabilizing yields over time [ <xref rid="B10-plants-06-00021" ref-type="bibr"> 10 </xref> ], but now with advancements in mass spectrometry (MS) technology, it is imperative to study the global metabolomic responses of soybeans to <span class="trait">drought</span> and heat stresses for developing metabolomic markers, utilizing metabolic pathways, and assisting soybean breeding programs. Non-targeted metabolomics aided by the advancements of <span class="gene">MS</span> technologies is capable of profiling diverse classes of metabolites, and thus has become a platform of choice to study metabolites perturbed by environmental cues [ <xref rid="B5-plants-06-00021" ref-type="bibr"> 5 </xref> ], and are continually adding significant understanding to our knowledge of molecular mechanisms that could lead to predictive biology in the future by taking a systems biology approach [ <xref rid="B6-plants-06-00021" ref-type="bibr"> 6 </xref> ]. Recently, Rabara et al. [ <xref rid="B7-plants-06-00021" ref-type="bibr"> 7 </xref> ] compared two metabolomic profiles of soybean and tobacco to elucidate unique signature metabolites during progressive dehydration treatment, and showed that 4-hydroxy-2-oxoglutaric acid in tobacco roots and coumestrol in soybean roots show the highest tissue-specific accumulations. Previously, in a similar progressive <span class="trait">drought</span> stress study in <italic> Medicago truncatula </italic> , Zhang et al. [ <xref rid="B11-plants-06-00021" ref-type="bibr"> 11 </xref> ] meticulously combined transcriptomic and metabolomic data of roots and shoots from control, mild-, moderate-, severely-stressed, and re-watered plants, and provided insights into the regulation of metabolic pathways operating under different levels of <span class="trait">drought</span> stress in plants. The study concluded that myo-inositol and proline had striking regulatory profiles indicating involvement in the <italic> Medicago </italic> <span class="trait">drought</span> tolerance mechanism.
					</p>
				
					<p>&nbsp;&nbsp;
						Metabolomics is being used to measure differential abundance of metabolites at a global level; and has successfully been used in many crops such as soybean, rice, corn, and barley [ <xref rid="B13-plants-06-00021" ref-type="bibr"> 13 </xref> , <xref rid="B14-plants-06-00021" ref-type="bibr"> 14 </xref> , <xref rid="B15-plants-06-00021" ref-type="bibr"> 15 </xref> , <xref rid="B16-plants-06-00021" ref-type="bibr"> 16 </xref> , <xref rid="B17-plants-06-00021" ref-type="bibr"> 17 </xref> ]. In Arabidopsis, metabolic profiling has elucidated the temporal dynamics associated with heat stress [ <xref rid="B18-plants-06-00021" ref-type="bibr"> 18 </xref> ]. Earlier integrated metabolomic-proteomics studies have shown comprehensive exploration of soybean roots and hypocotyls under abiotic [ <xref rid="B19-plants-06-00021" ref-type="bibr"> 19 </xref> ] and wheat biotic stresses [ <xref rid="B20-plants-06-00021" ref-type="bibr"> 20 </xref> ]. Similar metabolomic studies were performed in grapes for <span class="trait">drought</span> and salinity stresses to define alterations in the metabolic pathways related to gluconeogenesis, energy metabolism and nitrogen assimilation [ <xref rid="B21-plants-06-00021" ref-type="bibr"> 21 </xref> ]. At present an incomplete understanding of abiotic stress-associated metabolism and identification of associated metabolites remains a major <span class="gene">gap</span> in soybean stress research. In addition, metabolomic studies of economically important crops, such as soybean that is being used for food as well as feed, are of special interest, and are advantageous for genomics-assisted selection for plant breeders to develop improved germplasm and release novel climate resilient varieties in the future [ <xref rid="B22-plants-06-00021" ref-type="bibr"> 22 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Based on metabolomic profiling and using a MapMan-derived metabolic pathway overview ( <xref ref-type="app" rid="app1-plants-06-00021"> Figure S6 </xref> ), we proposed a model ( <xref ref-type="fig" rid="plants-06-00021-f008"> Figure 8 </xref> ), which explains the heat stress-related changes in soybeans. The present study showed that in response to <span class="trait">drought</span> stress, soybeans experience reduced growths to alleviate the stress effects. In addition, we suggested that during <span class="trait">drought,</span> soybeans <span class="gene">try</span> to activate a tolerance pathways to ensure an adjusted metabolism, which is likely one of the soybeans defense strategies, similar to other plants, for combating <span class="trait">drought</span> stress [ <xref rid="B82-plants-06-00021" ref-type="bibr"> 82 </xref> , <xref rid="B83-plants-06-00021" ref-type="bibr"> 83 </xref> ]. By using compounds that were detected by GC- and LC-MS as input data, we also built regulatory networks, addressing related compounds that were not detected but might be affected in response to abiotic stress ( <xref ref-type="app" rid="app1-plants-06-00021"> Figures S6 and S7 </xref> ). Likewise, we also found that heat stress affects the synthesis of phytochemicals, such as daidzein, daidzin, formononetin, glycitin, syringic acid, genistein and, genistin.
					</p>
				
					<p>&nbsp;&nbsp;
						The following are available online at <uri xlink:href="www.mdpi.com/2223-7747/6/2/21/s1"> www.mdpi.com/2223-7747/6/2/21/s1 </uri> , Figure S1: Categorization of identified metabolites in seven broad groups. <italic> Y </italic> axis has number of metabolites; <italic> X </italic> axis has 7 different broad categories of identified metabolites. Green color <span class="gene">bar</span> represents number of metabolites that were found increased in abundance; golden color <span class="gene">bar</span> represents number of metabolites decreased in abundance compared to the control condition, Figure S2: Effect of stress on metabolites involved in starch and carbohydrate metabolism (A) Box plots showing the effect of <span class="trait">drought</span> and heat stress on the expression of maltose, galactitol, sucrose, and mannitol. (B) Heat map showing the effect of <span class="trait">drought</span> and heat stress on the abundance of metabolites that are involved in starch and carbohydrate metabolism, Figure S3: Metscape network shows the effect of abiotic stress on sugar metabolism. Red colored nodes indicate input metabolites from GC/LC-MS results, green squares indicate enzymes, grey squares indicate pathway number from KEGG database and other hexagons indicate other predicted metabolites that might be associated with the reaction, Figure S4: A proposed model by combining proteomic and metabolomic studies in the realm of nitrogen metabolism. Figure shows how glutamate and glutamine metabolite level is regulated in response to <span class="trait">drought</span> and heat stress based on differential expression of glutamine synthetase. (Based on Das et al., 2016 [ <xref rid="B49-plants-06-00021" ref-type="bibr"> 49 </xref> ], and this study), Figure S5: Differential expression of phytochemicals in response to <span class="trait">drought</span> stress, Figure S6: Metabolic pathways regulation in response to heat stress derived from MapMan. 93 data points were mapped and 72 data points are visible in this figure. Arabidopsis ATH1_TAIR9 was used as mapping template, Figure S7: A fully connected network of all carbohydrate derived metabolites detected in response to heat stress. It is a compound-gene based network. Red color nodes indicate input data and yellow color nodes indicate related compounds with input compounds. This network comprises of 110 nodes and 217 edges, Figure S8: Network elucidating all amino acid derived metabolites detected in response to <span class="trait">drought</span> and heat stress. It is a compound-gene based network. Red color nodes indicate input data and yellow color nodes indicate related compounds with input compounds. This network comprises of 244 nodes and 502 edges, Table S1: Ratio for differential expression, fold change and mean values of metabolites in response to <span class="trait">drought</span> and heat stress compared to the control, Table S2: Mean and <italic> p </italic> -values of differentially expressed carbohydrates and amino acids. (A) Summary of differentially expressed carbohydrates during <span class="trait">drought</span> and heat stress compared to the control, along with its mean and <italic> p </italic> -values. A value of <italic> p </italic> &#x02264; 0.05 was considered statistically significant (red font) for the two group comparisons. (B) Summary of differentially expressed amino acids during <span class="trait">drought</span> and heat stress compared to the control, along with its mean and <italic> p </italic> -values. A value of <italic> p </italic> &#x02264; 0.05 was considered statistically significant (red font) for the two group comparisons.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Silencing of Soybean Raffinose Synthase Gene Reduced Raffinose Family Oligosaccharides and Increased True Metabolizable Energy of Poultry Feed</h2>
			<h3>PMID:28559898</h3>
				

				
					<h3>PMCID:5432567</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Valentine </surname>  <given-names> Michelle F. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/419282/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> De Tar </surname>  <given-names> Joann R. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mookkan </surname>  <given-names> Muruganantham </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/426379/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Firman </surname>  <given-names> Jeffre D. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Zhanyuan J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/411571/overview"/>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 4 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] is the number one oil and protein crop in the United States, but the seed contains several anti-nutritional factors that are toxic to both humans and livestock. RNA interference technology has become an increasingly popular technique in gene silencing because it allows for both temporal and spatial targeting of specific genes. The objective of this research is to use RNA-mediated gene silencing to down-regulate the soybean gene raffinose synthase 2 (RS2), to reduce total raffinose content in mature seed. Raffinose is a trisaccharide that is indigestible to humans and monogastric animals, and as monogastric animals are the largest consumers of soy products, reducing raffinose would improve the nutritional quality of soybean. An RNAi construct targeting <span class="gene">RS2</span> was designed, cloned, and transformed to the soybean genome via Agrobacterium-mediated transformation. Resulting plants were analyzed for the presence and number of copies of the transgene by PCR and Southern blot. The efficiency of mRNA silencing was confirmed by real-time quantitative PCR. Total raffinose content was determined by HPLC analysis. Transgenic plant lines were recovered that exhibited dramatically reduced levels of raffinose in mature seed, and these lines were further analyzed for other phenotypes such as development and <span class="trait">yield.</span> Additionally, a precision-fed rooster assay was conducted to measure the true metabolizable energy (TME) in full-fat soybean meal made from the wild-type or transgenic low-raffinose soybean lines. Transgenic low-raffinose soy had a measured TME of 2,703 kcal/kg, an increase as compared with 2,411 kcal/kg for wild-type. As low digestible energy is a major limiting factor in the percent of soybean meal that can be used in poultry diets, these results may substantiate the use of higher concentrations of low-raffinose, full-fat soy in formulated livestock diets.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [ <italic> Glycine max </italic> (L.) Merr.] is the number one oil and protein crop in the United States, but the seed contains several anti-nutritional factors that are toxic to both humans and livestock. RNA interference technology has become an increasingly popular technique in gene silencing because it allows for both temporal and spatial targeting of specific genes. The objective of this research is to use RNA-mediated gene silencing to down-regulate the soybean gene <italic> raffinose synthase 2 </italic> ( <italic> <span class="gene">RS2</span> </italic> ), to reduce total raffinose content in mature seed. Raffinose is a trisaccharide that is indigestible to humans and monogastric animals, and as monogastric animals are the largest consumers of soy products, reducing raffinose would improve the nutritional quality of soybean. An RNAi construct targeting <italic> <span class="gene">RS2</span> </italic> was designed, cloned, and transformed to the soybean genome via <italic> Agrobacterium-mediated </italic> transformation. Resulting plants were analyzed for the presence and number of copies of the transgene by PCR and Southern blot. The efficiency of mRNA silencing was confirmed by real-time quantitative PCR. Total raffinose content was determined by HPLC analysis. Transgenic plant lines were recovered that exhibited dramatically reduced levels of raffinose in mature seed, and these lines were further analyzed for other phenotypes such as development and <span class="trait">yield.</span> Additionally, a precision-fed rooster assay was conducted to measure the true metabolizable energy (TME) in full-fat soybean meal made from the wild-type or transgenic low-raffinose soybean lines. Transgenic low-raffinose soy had a measured TME of 2,703 kcal/kg, an increase as compared with 2,411 kcal/kg for wild-type. As low digestible energy is a major limiting factor in the percent of soybean meal that can be used in poultry diets, these results may substantiate the use of higher concentrations of low-raffinose, full-fat soy in formulated livestock diets.
					</p>
				
					<p>&nbsp;&nbsp;
						Raffinose biosynthesis in developing soybean seeds is catalyzed by raffinose synthase 2 (RS2), which is encoded by <span class="gene">Glyma06g18890</span> (Dierking and Bilyeu, <xref rid="B8" ref-type="bibr"> 2008 </xref> ). <span class="gene">RS2</span> catalyzes the reaction: sucrose + galactinol <inline-graphic xlink:href="fpls-08-00692-i0001.jpg"/> raffinose + myo-inositol. The subsequent conversions of raffinose to stachyose, and stachyose to verbascose produce the spectrum of raffinose family oligosaccharides, though verbascose content in soybean seeds is negligible (Kumar et al., <xref rid="B22" ref-type="bibr"> 2010 </xref> ). <span class="gene">RS2</span> is considered the committed step in RFO biosynthesis, and, as a result, down-regulation of this gene should lead to increased sucrose, and decreased raffinose and stachyose. Successful soybean breeding efforts to reduce raffinose content in mature seeds have lowered raffinose content from about 1&#x02013;1.5% in wild-type (WT) to nearly undetectable levels in mutant lines. Two major mutations in the soybean gene <italic> <span class="gene">RS2</span> </italic> have been identified and associated with the low-raffinose phenotype (Kerr and Sebastian, <xref rid="B19" ref-type="bibr"> 2000 </xref> ; Dierking and Bilyeu, <xref rid="B9" ref-type="bibr"> 2009a </xref> ; Bilyeu and Wiebold, <xref rid="B2" ref-type="bibr"> 2016 </xref> ). The two mutants showed relatively variable low raffinose phenotypes across environments. The absolute quantities of sucrose, raffinose, and stachyose are highly dependent on location and planting date. Agronomic traits such as emergence, <span class="trait">yield,</span> maturity, seed protein, and oil content were not affected in one mutant (W331) (Neus et al., <xref rid="B24" ref-type="bibr"> 2005 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Transgenic (MM3-2-15) and WT plants were grown side-by-side in a field plot to increase seeds. Ten randomly chosen plants for each WT and transgenic were leaf-painted for absence of <italic> <span class="gene">bar</span> </italic> gene and, as expected, all leaves were susceptible. Mid-mature seeds corresponding to roughly stage R5/R6 were analyzed for <italic> <span class="gene">RS2</span> </italic> and <italic> <span class="gene">RS3</span> </italic> gene expression by qRT-PCR. No visible difference in plant growth or maturation was observed throughout the growing season (Figure <xref ref-type="supplementary-material" rid="SM5"> S4 </xref> ). Total seed yields for MM3-2-15 and WT were 6.44 and 5.77 kg. This result contradicts the <span class="trait">yield</span> reduction observed in greenhouse-grown plants. However, the field results are likely more representative of the true <span class="trait">yield</span> potential of transgenic, low-raffinose plants. To determine if the reduction in carbohydrate content affected seed fat, crude protein, or amino acid composition, samples of field-grown seed were analyzed. There were no substantial differences between the transgenic and WT lines in moisture content (6.13 and 6.14%, respectively), crude fat (19.18 and 19.79%, respectively), ash (5.21 and 5.27%, respectively), crude protein (37.26 and 36.14%, respectively), or for any of the amino acids. Crude fiber appeared reduced in transgenic (6.56%) compared to WT (7.07%). This result was similar to that by Chen et al. ( <xref rid="B5" ref-type="bibr"> 2013 </xref> ), so it is possible that the alterations in sucrose and oligosaccharide also affected total fiber content.
					</p>
				
					<p>&nbsp;&nbsp;
						We have achieved increased true metabolizable energy of poultry feed in soybean seeds by silencing soybean <italic> <span class="gene">RS2</span> </italic> through RNAi. In the current work, results for transgene co-integration, transgene inheritance, T1 seedling vigor, carbohydrate phenotype, and <span class="trait">yield</span> were somewhat unexpected and deserve additional examination. Of the 24 <italic> <span class="gene">bar</span> </italic> -positive events generated from transformation using the pMU2T-bar-RS2 plasmid, nine also had the <span class="gene">RS2</span> transgene of interest. This 37.5% co-integration rate is lower than the 47% rate reported for rice and tobacco in Komari et al. ( <xref rid="B20" ref-type="bibr"> 1996 </xref> ) and the 70% rate for soybean (Xing et al., <xref rid="B34" ref-type="bibr"> 2000 </xref> ). Of the nine co-integration events, four passed both transgenes to the progeny, and marker-free progeny were observed in all four lines by T3 generation. Thus, 4 out of 24 events, or 16.6% produced transgene-positive, marker-free progeny. Interestingly, Matthews et al. ( <xref rid="B23" ref-type="bibr"> 2001 </xref> ) also reported 16% of transformed barley events producing transgene-positive, marker-free progeny using a &#x0201c;twin T-DNA&#x0201d; vector where the two T-DNA borders are immediately adjacent to one another. This information is useful to future researchers wishing to obtain marker-free transgenic events, as it shows that about one in six events produced will <span class="trait">yield</span> progeny with the desired genotype using the pMU2T-bar plasmid.
					</p>
				
					<p>&nbsp;&nbsp;
						This work led to a low-raffinose line of marker-free transgenic soybean. For commercial transgenic crops, <span class="trait">yield</span> parity is one of the most important traits, and though total seed <span class="trait">yield</span> was significantly reduced in the greenhouse <span class="trait">yield</span> study, <span class="trait">yield</span> was comparable between WT and transgenic plants under the field conditions. These lines are likely not commercial products, but this work is separate confirmation of the contribution of <span class="gene">RS2</span> gene to raffinose accumulation in soybean seeds. The lines described in Bilyeu and Wiebold ( <xref rid="B2" ref-type="bibr"> 2016 </xref> ) were from non-elite lines, and it would require several generations of backcrossing to obtain <span class="trait">yield</span> parity with modern commercial lines. The transformation approach illustrated in this present study provides a proof that low raffinose family could be obtained in elite soybean variety by transformation. Future experiments are needed to evaluate the inheritance of transgenes through several more generations, as well as the carbohydrate phenotype over generations and environmental conditions. Also, replicated, controlled experiments are needed to precisely measure agronomic traits such as emergence, stress tolerance, maturation, and <span class="trait">yield</span> for the transgenic plants. The line MM3-2-15 is certainly a strong candidate for use in improving soybean carbohydrate quality.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of Isoflavone Biosynthesis by miRNAs in Two Contrasting Soybean Genotypes at Different Seed Developmental Stages</h2>
			<h3>PMID:28450878</h3>
				

				
					<h3>PMCID:5390031</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Gupta </surname>  <given-names> Om P. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/405729/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nigam </surname>  <given-names> Deepti </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/319709/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dahuja </surname>  <given-names> Anil </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/124307/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kumar </surname>  <given-names> Sanjeev </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vinutha </surname>  <given-names> T. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sachdev </surname>  <given-names> Archana </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Praveen </surname>  <given-names> Shelly </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/344056/overview"/>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Owing to the presence of nutritionally important, health-promoting bioactive compounds, especially isoflavones, soybean has acquired the status of a functional food. miRNAs are tiny riboregulator of gene expression by either decreasing and/or increasing the expression of their corresponding target genes. Despite several works on identification and functional characterization of plant miRNAs, the role of miRNAs in the regulation of isoflavones metabolism is still a virgin field. In the present study, we identified a total of 31 new miRNAs along with their 245 putative target genes from soybean seed-specific ESTs using computational approach. The Kyoto Encyclopedia of Genes and Genomes pathway analyses indicated that miRNA putatively regulates metabolism and genetic information processing. Out of that, a total of 5 miRNAs <span class="gene">(Gma-miRNA12,</span> <span class="gene">Gma-miRNA24,</span> <span class="gene">Gma-miRNA26,</span> <span class="gene">Gma-miRNA28,</span> and <span class="gene">Gma-miRNA29)</span> were predicted and validated for their probable role during isoflavone biosynthesis. We also validated their five target genes using RA-PCR, which is as good as 5'RLM-RACE. Temporal regulation [35 days after flowering, 45, 55, and 65 DAF] of miRNAs and their targets showed differential expression schema. Differential expression of <span class="gene">Gma-miR26</span> and <span class="gene">Gma-miRNA28</span> along with their corresponding target genes <span class="gene">(Glyma.10G197900</span> and <span class="gene">Glyma.09G127200)</span> showed a direct relationship with the total isoflavone content. Therefore, understanding the miRNA-based genetic regulation of isoflavone pathway would assist in selection and manipulation to get high-performing soybean genotypes with better isoflavone <span class="trait">yield.</span>
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Host <span class="trait">Resistance</span> and Chemical Control for Management of Sclerotinia Stem Rot of Soybean in Ohio.</h2>
			<h3>PMID:28398874  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Huzar-Novakowiski J(1), Paul PA(1), Dorrance AE(1).
				<br/> 
				年份：2017 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Recent outbreaks of Sclerotinia stem rot (SSR) of soybean in Ohio, along with new fungicides and cultivars with <span class="trait">resistance</span> to this disease, have led to a renewed interest in studies to update disease management guidelines. The effect of host <span class="trait">resistance</span> (in moderately resistant [MR] and moderately susceptible [MS] cultivars) and chemical control on SSR and <span class="trait">yield</span> was evaluated in 12 environments from 2014 to 2016. The chemical treatments evaluated were an untreated check, four fungicides (boscalid, picoxystrobin, pyraclostrobin, and thiophanate-methyl), and one herbicide (lactofen) applied at soybean growth stage R1 (early flowering) alone or at R1 followed by a second application at R2 (full flowering). SSR developed in 6 of 12 environments, with mean disease incidence in the untreated check of 2.5 to 41%. The three environments with high levels of SSR (disease incidence in the untreated check &gt;20%) were used for further statistical analysis. There were significant effects (P &lt; 0.05) of soybean cultivar and chemical treatment on SSR levels. Significantly lower levels of SSR were observed in MR cultivars. Both boscalid and lactofen reduced SSR but did not increase <span class="trait">yield.</span> Pyraclostrobin increased SSR compared with the untreated check in the three environments with high levels of disease. In the six fields where SSR did not develop, chemical treatment did not increase <span class="trait">yield,</span> nor was the <span class="trait">yield</span> from the MR cultivar significantly different from the <span class="gene">MS</span> cultivar. For Ohio, MR cultivars alone were effective for management of SSR in soybean fields where this disease has historically occurred.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Foliar and Seed Application of Amino Acids Affects the Antioxidant Metabolism of the Soybean Crop</h2>
			<h3>PMID:28377778</h3>
				

				
					<h3>PMCID:5359285</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Teixeira </surname>  <given-names> Walqu&#x000ed;ria F. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/390527/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fagan </surname>  <given-names> Evandro B. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/419408/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Soares </surname>  <given-names> Lu&#x000ed;s H. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/420036/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Umburanas </surname>  <given-names> Renan C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/420365/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reichardt </surname>  <given-names> Klaus </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neto </surname>  <given-names> Durval D. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/403905/overview"/>  </contrib>
				<br/> 
				年份：<day> 23 </day>  <month> 2 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In recent years, the application of natural substances on crops has been intensified in order to increase the <span class="trait">resistance</span> and <span class="trait">yield</span> of the soybean crop. Among these products are included plant biostimulants that may contain algae extracts, amino acids, and plant regulators in their composition. However, there is little information on the isolated effect of each of these constituents. The objective of this research was to evaluate the effect of the application of isolated amino acids on the antioxidant metabolism of the soybean crop. Experiments were carried out in a greenhouse and in the field with the application of the amino acids glutamate, phenylalanine, cysteine, glycine in seed treatment, and foliar application at V4 growth stage. Antioxidant metabolism constituents evaluated were superoxide dismutase, catalase, peroxidase, hydrogen peroxide content, proline, and lipid peroxidation. In addition, <span class="trait">resistance</span> enzymes as polyphenol oxidase and phenylalanine ammonia-lyase (PAL) were evaluated. In both experiments, the use of cysteine, only in seed treatment and in both seed treatment and foliar application increased the activity of the enzyme <span class="gene">PAL</span> and catalase. Also in both experiments, the use of phenylalanine increased the activity of the enzyme <span class="gene">PAL</span> when the application was carried out as foliar application or both in seed treatment and foliar application. In the field experiment, the application of glutamate led to an increase in the activity of the catalase and <span class="gene">PAL</span> enzymes for seed treatment and foliar application. The use of the set of amino acids was only efficient in foliar application, which led to a greater activity of the enzymes peroxidase, PAL, and polyphenol oxidase. The other enzymes as well as lipid peroxidation and hydrogen peroxide presented different results according to the experiment. Therefore, glutamate, cysteine, phenylalanine, and glycine can act as signaling amino acids in soybean plants, since small doses are enough to increase the activity of the antioxidant enzymes.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genetic Architecture of Phenomic-Enabled Canopy Coverage in <italic> Glycine max </italic></h2>
			<h3>PMID:28363978</h3>
				

				
					<h3>PMCID:5499164</h3>
					<h4>期刊:Genetics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Xavier </surname>  <given-names> Alencar </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hall </surname>  <given-names> Benjamin </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hearst </surname>  <given-names> Anthony A. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cherkauer </surname>  <given-names> Keith A. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rainey </surname>  <given-names> Katy M. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Department of Agronomy, Purdue University, West Lafayette, Indiana 47907 </aff>  <aff id="aff2">  <label> &#x02020; </label> Department of Agriculture and Biological Engineering, Purdue University, West Lafayette, Indiana 47907 </aff>
				<br/> 
				年份：<day> 03 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Digital imagery can help to quantify seasonal changes in desirable crop phenotypes that can be treated as quantitative traits. Because limitations in precise and functional phenotyping restrain genetic improvement in the postgenomic era, imagery-based phenomics could become the next breakthrough to accelerate genetic gains in field crops. Whereas many phenomic studies focus on exploratory analysis of spectral data without obvious interpretative value, we used field images to directly measure soybean canopy development from phenological stage <span class="gene">V2</span> to R5. Over 3 years, we collected imagery using ground and aerial platforms of a large and diverse nested association panel comprising 5555 lines. Genome-wide association analysis of canopy coverage across sampling dates detected a large quantitative trait locus (QTL) on soybean (Glycine max, L. Merr.) chromosome 19. This QTL provided an increase in <span class="trait">yield</span> of 47.3 kg ha-1 Variance component analysis indicated that a parameter, described as average canopy coverage, is a highly heritable trait (h2 = 0.77) with a promising genetic correlation with grain <span class="trait">yield</span> (0.87), enabling indirect selection of <span class="trait">yield</span> via canopy development parameters. Our findings indicate that fast canopy coverage is an early season trait that is inexpensive to measure and has great potential for application in breeding programs focused on <span class="trait">yield</span> improvement. We recommend using the average canopy coverage in multiple trait schemes, especially for the early stages of the breeding pipeline (including progeny rows and preliminary <span class="trait">yield</span> trials), in which the large number of field plots makes collection of grain <span class="trait">yield</span> data challenging.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:GmILPA1, Encoding an APC8-like Protein, Controls Leaf Petiole Angle in Soybean.</h2>
			<h3>PMID:28336772  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gao J(1), Yang S(2), Cheng W(1), Fu Y(1), Leng J(1), Yuan X(1), Jiang N(1), Ma J(2), Feng X(2).
				<br/> 
				年份：2017 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Leaf petiole angle (LPA) is an important plant architectural trait that affects canopy coverage, photosynthetic efficiency, and ultimately productivity in many legume crops. However, the genetic basis underlying this trait remains unclear. Here, we report the identification, isolation, and functional characterization of Glycine max Increased Leaf Petiole Angle1 <span class="gene">(GmILPA1),</span> a gene encoding an APC8-like protein, which is a subunit of the anaphase-promoting complex/cyclosome in soybean (Glycine max). A gamma ray-induced deletion of a fragment involving the fourth exon of <span class="gene">GmILPA1</span> and its flanking sequences led to extension of the third exon and formation of, to our knowledge, a novel 3'UTR from intronic and intergenic sequences. Such changes are responsible for enlarged LPAs that are associated with reduced motor cell proliferation in the <span class="gene">Gmilpa1</span> mutant. <span class="gene">GmILPA1</span> is mainly expressed in the basal cells of leaf primordia and appears to function by promoting cell growth and division of the pulvinus that is critical for its establishment. <span class="gene">GmILPA1</span> directly interacts with <span class="gene">GmAPC13a</span> as part of the putative anaphase-promoting complex. <span class="gene">GmILPA1</span> exhibits variable expression levels among varieties with different degrees of LPAs, and expression levels are correlated with the degrees of the LPAs. Together, these observations revealed a genetic mechanism modulating the plant petiole angle that could pave the way for modifying soybean plant architecture with optimized petiole angles for enhanced <span class="trait">yield</span> potential.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Natural variation at the soybean J locus improves adaptation to the tropics and enhances <span class="trait">yield.</span></h2>
			<h3>PMID:28319089  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lu S(1)(2), Zhao X(1)(2), Hu Y(3)(4), Liu S(4)(5), Nan H(1), Li X(1)(4), Fang C(1)(4), Cao D(1)(2), Shi X(1), Kong L(1)(4), Su T(1)(4), Zhang F(1)(4), Li S(1)(4), Wang Z(5), Yuan X(1), Cober ER(6), Weller JL(7), Liu B(1)(2), Hou X(3), Tian Z(5), Kong F(1)(2).
				<br/> 
				年份：2017 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a major legume crop originating in temperate regions, and photoperiod responsiveness is a key factor in its latitudinal adaptation. Varieties from temperate regions introduced to lower latitudes mature early and have extremely low grain yields. Introduction of the long-juvenile (LJ) trait extends the vegetative phase and improves <span class="trait">yield</span> under short-day conditions, thereby enabling expansion of cultivation in tropical regions. Here we report the cloning and characterization of J, the major classical locus conferring the LJ trait, and identify J as the ortholog of Arabidopsis thaliana EARLY FLOWERING 3 (ELF3). J depends genetically on the legume-specific flowering repressor E1, and J protein physically associates with the <span class="gene">E1</span> promoter to downregulate its transcription, relieving repression of two important FLOWERING LOCUS T (FT) genes and promoting flowering under short days. Our findings identify an important new component in flowering-time control in soybean and provide new insight into soybean adaptation to tropical regions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Enhancing soybean photosynthetic CO2 assimilation using a cyanobacterial membrane protein, ictB.</h2>
			<h3>PMID:28273517  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hay WT(1), Bihmidine S(2), Mutlu N(3), Hoang KL(4), Awada T(2), Weeks DP(3), Clemente TE(5), Long SP(6).
				<br/> 
				年份：2017 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean <span class="gene">C3</span> photosynthesis can suffer a severe loss in efficiency due to photorespiration and the lack of a carbon concentrating mechanism (CCM) such as those present in other plant species or cyanobacteria. Transgenic soybean (Glycine max cv. Thorne) plants constitutively expressing cyanobacterial ictB (inorganic carbon transporter B) gene were generated using Agrobacterium-mediated transformation. Although more recent data suggest that ictB does not actively transport HCO3-/CO2, there is nevertheless mounting evidence that transformation with this gene can increase higher plant photosynthesis. The hypothesis that expression of the ictB gene would improve photosynthesis, biomass production and seed <span class="trait">yield</span> in soybean was tested, in two independent replicated greenhouse and field trials. Results showed significant increases in photosynthetic CO2 uptake (Anet) and dry mass in transgenic relative to wild type (WT) control plants in both the greenhouse and field trials. Transgenic plants also showed increased photosynthetic rates and biomass production during a <span class="trait">drought</span> mimic study. The findings presented herein demonstrate that ictB, as a single-gene, contributes to enhancement in various <span class="trait">yield</span> parameters in a major commodity crop and point to the significant role that biotechnological approaches to increasing photosynthetic efficiency can play in helping to meet increased global demands for food.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of cyanobacterial FBP/SBPase in soybean prevents <span class="trait">yield</span> depression under future climate conditions.</h2>
			<h3>PMID:28204603</h3>
				

				
			<p>
				作者：Köhler IH(1)(2), Ruiz-Vera UM(1), VanLoocke A(3), Thomey ML(1)(2), Clemente T(4), Long SP(4), Ort DR(1)(2)(5), Bernacchi CJ(1)(2)(5).
				<br/> 
				年份：2017 Jan 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Predictions suggest that current crop production needs to double by 2050 to meet global food and energy demands. Based on theory and experimental studies, overexpression of the photosynthetic enzyme sedoheptulose-1,7-bisphosphatase (SBPase) is expected to enhance <span class="gene">C3</span> crop photosynthesis and yields. Here we test how expression of the cyanobacterial, bifunctional fructose-1,6/sedoheptulose-1,7-bisphosphatase (FBP/SBPase) affects carbon assimilation and seed <span class="trait">yield</span> (SY) in a major crop (soybean, Glycine max). For three growing seasons, wild-type (WT) and FBP/SBPase-expressing (FS) plants were grown in the field under ambient (400 μmol mol−1) and elevated (600 μmol mol−1) CO2 concentrations [CO2] and under ambient and elevated temperatures (+2.7 °C during daytime, +3.4 °C at night) at the SoyFACE research site. Across treatments, FS plants had significantly higher carbon assimilation (4–14%), Vc,max (5–8%), and Jmax (4–8%). Under ambient [CO2], elevated temperature led to significant reductions of SY of both genotypes by 19–31%. However, under elevated [CO2] and elevated temperature, FS plants maintained SY levels, while the WT showed significant reductions between 11% and 22% compared with plants under elevated [CO2] alone. These results show that the manipulation of the photosynthetic carbon reduction cycle can mitigate the effects of future high CO2 and high temperature environments on soybean <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Genome-Wide Association Study for Agronomic Traits in Soybean Using SNP Markers and SNP-Based Haplotype Analysis</h2>
			<h3>PMID:28152092</h3>
				

				
					<h3>PMCID:5289539</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0001-6468-9394 </contrib-id>  <name>  <surname> Contreras-Soto </surname>  <given-names> Rodrigo Iv&#x000e1;n </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Mora </surname>  <given-names> Freddy </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Oliveira </surname>  <given-names> Marco Ant&#x000f4;nio Rott </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Higashi </surname>  <given-names> Wilson </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Scapim </surname>  <given-names> Carlos Alberto </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schuster </surname>  <given-names> Ivan </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Mapping quantitative trait loci through the use of linkage disequilibrium (LD) in populations of unrelated individuals provides a valuable approach for dissecting the genetic basis of complex traits in soybean (Glycine max). The haplotype-based genome-wide association study (GWAS) has now been proposed as a complementary approach to intensify benefits from LD, which enable to assess the genetic determinants of agronomic traits. In this study a GWAS was undertaken to identify genomic regions that control 100-seed weight (SW), plant height (PH) and seed <span class="trait">yield</span> (SY) in a soybean association mapping panel using single nucleotide polymorphism (SNP) markers and haplotype information. The soybean cultivars (N = 169) were field-evaluated across four locations of southern Brazil. The genome-wide haplotype association analysis (941 haplotypes) identified eleven, seventeen and fifty-nine SNP-based haplotypes significantly associated with SY, SW and PH, respectively. Although most marker-trait associations were environment and trait specific, stable haplotype associations were identified for SY and SW across environments (i.e., haplotypes <span class="gene">Gm12_Hap12).</span> The haplotype block 42 on Chr19 <span class="gene">(Gm19_Hap42)</span> was confirmed to be associated with PH in two environments. These findings enable us to refine the breeding strategy for tropical soybean, which confirm that haplotype-based GWAS can provide new insights on the genetic determinants that are not captured by the single-marker approach.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The haplotype block 42, associated to PH on Chr19 <span class="gene">(Gm19_Hap42),</span> is a region containing the <italic> Determinate stem 1 </italic> gene (Dt1; <span class="gene">Glyma19g37890)</span> at 18.6 kb upstream of the peak SNP ss715635425 (Chr19_45000827; Table E in <xref ref-type="supplementary-material" rid="pone.0171105.s001"> S1 File </xref> and <xref ref-type="table" rid="pone.0171105.t005"> Table 5 </xref> ), which has been previously associated with PH and days to maturity in soybean [ <xref rid="pone.0171105.ref004" ref-type="bibr"> 4 </xref> ] ( <xref ref-type="fig" rid="pone.0171105.g005"> Fig 5 </xref> ). In addition, other marker-yield associations have been previously identified at this region, seed <span class="trait">yield</span> 11&#x02013;6, Plant height 13&#x02013;8 and Plant height 4&#x02013;2 [ <xref rid="pone.0171105.ref038" ref-type="bibr"> 38 </xref> , <xref rid="pone.0171105.ref039" ref-type="bibr"> 39 </xref> ] and associated with <span class="gene">Dt1</span> [ <xref rid="pone.0171105.ref040" ref-type="bibr"> 40 </xref> ] ( <xref ref-type="fig" rid="pone.0171105.g005"> Fig 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						For SY in Cascavel, the haplotypes TAAT <span class="gene">(Gm12_Hap12a)</span> and TAAC <span class="gene">(Gm12_Hap12b)</span> showed significant differences with the haplotype CGGT <span class="gene">(Gm12_Hap12c).</span> <span class="gene">Gm12_Hap12a</span> and <span class="gene">Gm12_Hap12b</span> had a mean value of 2567 kg ha <sup> -1 </sup> and 2381 kg ha <sup> -1 </sup> , respectively, while the haplotype <span class="gene">Gm12_Hap12c</span> yielded a mean of 1929 kg ha <sup> -1 </sup> , a <span class="trait">yield</span> 19% and 25% lower than the haplotypes <span class="gene">Gm12_Hap12a</span> and <span class="gene">Gm12_Hap12b,</span> respectively ( <xref ref-type="table" rid="pone.0171105.t003"> Table 3 </xref> ). For SW, in Palotina, the same haplotypes <span class="gene">(Gm12_Hap12a</span> and <span class="gene">Gm12_Hap12b)</span> showed statistical differences with <span class="gene">Gm12_Hap12c.</span> In average, the haplotypes <span class="gene">Gm12_Hap12a</span> and <span class="gene">Gm12_Hap12b</span> had values of 11.4 g/100 seeds and 11.5 g/100 seeds, while the haplotype <span class="gene">Gm12_Hap12c</span> yielded 10.5 g/100 seeds (respectively 8% and 9% lower SW than <span class="gene">Gm12_Hap12a</span> and <span class="gene">Gm12_Hap12b).</span> In Primavera do Leste, the same haplotypes did not show statistical differences for SW. These haplotypes had the following frequencies in this association mapping panel: 30% for <span class="gene">Gm12_Hap12a,</span> 44% for <span class="gene">Gm12_Hap12b</span> and 26% for <span class="gene">Gm12_Hap12c,</span> and explained together a phenotypic variation of 12.1% for SY in Cascavel; 31.2% and 21.9% for SW in Palotina and Primavera do Leste, respectively (Tables <xref ref-type="table" rid="pone.0171105.t003"> 3 </xref> and <xref ref-type="table" rid="pone.0171105.t004"> 4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Yield</span> QTLs identified on chromosome 12 are of particular interest because they showed consistent effects across locations (Palotina, Primavera do Leste and Cascavel). Zhang et al. [ <xref rid="pone.0171105.ref005" ref-type="bibr"> 5 </xref> ] recently reported a close SNP (ss715613104) as effectively associated to SW in soybean. Furthermore, the following SSR markers: satt568, satt442 and satt192, which are linked to seed protein [ <xref rid="pone.0171105.ref036" ref-type="bibr"> 36 </xref> ] and seed glycitein [ <xref rid="pone.0171105.ref037" ref-type="bibr"> 37 </xref> ], respectively, have been co-localized near to the haplotype block identified on chromosome 12. One of the primary advantages of GWAS is the high mapping resolution. This feature enables GWAS to further narrow down the chromosomal region of putative QTLs and predict causal genes [ <xref rid="pone.0171105.ref005" ref-type="bibr"> 5 </xref> ]. Biologically important genes were identified on this haplotype block region <span class="gene">(Gm12_Hap12).</span> The gene <span class="gene">Glyma12g075700,</span> which encodes a ribonuclease III protein, represents an uncharacterized protein associated to BT097697 code in soybase [ <xref rid="pone.0171105.ref046" ref-type="bibr"> 46 </xref> ]. <span class="gene">Glyma12g075600</span> is another gene located near to <span class="gene">Gm12_Hap12,</span> which encodes a protein for senescence regulator in soybean (i.e., annotated as a U-box domain-containing protein 13-like; <ext-link ext-link-type="uri" xlink:href="http://phytozome.jgi.doe.gov/"> phytozome.jgi.doe.gov/ </ext-link> ). Importantly, its homolog in <italic> Arabidopsis thaliana </italic> regulates the expression of proteins associated with leaf senescence [ <xref rid="pone.0171105.ref047" ref-type="bibr"> 47 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The SNP at 45 Mb on Chr19 associated with PH has been previously reported by Lee et al. [ <xref rid="pone.0171105.ref038" ref-type="bibr"> 38 </xref> ] and Specht et al. [ <xref rid="pone.0171105.ref039" ref-type="bibr"> 39 </xref> ], which has QTLs associated with Seed <span class="trait">Yield</span> 11&#x02013;6, Plant height 4&#x02013;2 and Plant Height 13&#x02013;8. Zhang et al. [ <xref rid="pone.0171105.ref004" ref-type="bibr"> 4 </xref> ] also reported this SNP, which was strongly associated to PH and days to maturity. In fact, this result indicated that some causal gene(s) might exist in this genomic region. These associated markers may be useful for aggregation of causal genes of interest to improve soybean <span class="trait">yield.</span> Furthermore, in this region some markers have been reported near to the <span class="gene">Dt1</span> gene <span class="gene">(Glyma19g37890)</span> [ <xref rid="pone.0171105.ref004" ref-type="bibr"> 4 </xref> ]. <span class="gene">Dt1</span> is homologous to <italic> Arabidopsis </italic> terminal flower 1, and plays a predominant role in determining stem growth habit in soybean [ <xref rid="pone.0171105.ref048" ref-type="bibr"> 48 </xref> ]. Stem growth habit is an important discriminant trait for soybean cultivars classifying it in two major categories, determinate and indeterminate. Given the high relationship between plant growth habit, plant height and seed <span class="trait">yield</span> in soybean, our result is highly consistent with the result of Zhang et al. [ <xref rid="pone.0171105.ref004" ref-type="bibr"> 4 </xref> ], who determined that the locus harboring <span class="gene">Dt1</span> was strongly associated with PH.
					</p>
				
					<p>&nbsp;&nbsp;
						In conclusion, with the aid of the haplotype block map constructed by Song et al. [ <xref rid="pone.0171105.ref003" ref-type="bibr"> 3 </xref> ] and our haplotype block results, we efficiently tested SNPs and SNP-based haplotypes for optimized association analyses. Importantly, various haplotypes were significantly associated with SY (11), SW (17) and PH (59), of which some were located in/or near regions where QTLs for <span class="trait">yield</span> and yield-related traits have been previously mapped by either linkage or GWAS analysis. Moreover, new haplotypes-trait associations have been identified in this study (as the case of <span class="gene">Gm12_Hap12:</span> <span class="gene">Gm12_Hap12a</span> and <span class="gene">Gm12_Hap12b),</span> which could be used as putative regions for further research efforts focusing on the genetic basis of soybean <span class="trait">yield</span> and <span class="trait">yield</span> components. These haplotypes showed the best performance in comparison with the <span class="gene">Gm12_Hap12c</span> haplotype, and depended upon both geographic location and traits.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Whole genome-wide transcript profiling to identify differentially expressed genes associated with seed field emergence in two soybean low phytate mutants</h2>
			<h3>PMID:28100173</h3>
				

				
					<h3>PMCID:5242038</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Fengjie </given-names>  </name>  <address>  <email> fjyuanhz@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Xiaomin </given-names>  </name>  <address>  <email> 18658165151@wo.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dong </surname>  <given-names> Dekun </given-names>  </name>  <address>  <email> dongdekun@yahoo.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Qinghua </given-names>  </name>  <address>  <email> tsingyang2009@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fu </surname>  <given-names> Xujun </given-names>  </name>  <address>  <email> xujunfu@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Shenlong </given-names>  </name>  <address>  <email> zwsddz@zaas.org </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhu </surname>  <given-names> Danhua </given-names>  </name>  <address>  <email> danhua163@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9883 3553 </institution-id>  <institution-id institution-id-type="GRID"> grid.410744.2 </institution-id>  <institution/>  <institution> Institute of Crop Science and Nuclear Technology Utilization, Zhejiang Academy of Agricultural Sciences,  </institution>  </institution-wrap> Hangzhou, 310021 China  </aff>
				<br/> 
				年份：<day> 16 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Seed germination is important to soybean (Glycine max) growth and development, ultimately affecting soybean <span class="trait">yield.</span> A lower seed field emergence has been the main hindrance for breeding soybeans low in phytate. Although this reduction could be overcome by additional breeding and selection, the mechanisms of seed germination in different low phytate mutants remain unknown. In this study, we performed a comparative transcript analysis of two low phytate soybean mutants (TW-1 and TW-1-M), which have the same mutation, a 2 bp deletion in <span class="gene">GmMIPS1,</span> but show a significant difference in seed field emergence, TW-1-M was higher than that of TW-1 . RESULTS: Numerous genes analyzed by RNA-Seq showed markedly different expression levels between TW-1-M and TW-1 mutants. Approximately 30,000-35,000 read-mapped genes and ~21000-25000 expressed genes were identified for each library. There were ~3900-9200 differentially expressed genes (DEGs) in each contrast library, the number of up-regulated genes was similar with down-regulated genes in the mutant TW-1and TW-1-M. Gene ontology functional categories of DEGs indicated that the ethylene-mediated signaling pathway, the abscisic acid-mediated signaling pathway, response to hormone, ethylene biosynthetic process, ethylene metabolic process, regulation of hormone levels, and oxidation-reduction process, regulation of flavonoid biosynthetic process and regulation of abscisic acid-activated signaling pathway had high correlations with seed germination. In total, 2457 DEGs involved in the above functional categories were identified. Twenty-two genes with 20 biological functions were the most highly up/down- regulated (absolute value Log2FC &gt;5) in the high field emergence mutant TW-1-M and were related to metabolic or signaling pathways. Fifty-seven genes with 36 biological functions had the greatest expression abundance (FRPM &gt;100) in germination-related pathways. CONCLUSIONS: Seed germination in the soybean low phytate mutants is a very complex process, which involves a series of physiological, morphological and transcriptional changes. Compared with TW-1, TW-1-M had a very different gene expression profile, which included genes related to plant hormones, antioxidation, anti-stress and energy metabolism processes. Our research provides a molecular basis for understanding germination mechanisms, and is also an important resource for the genetic analysis of germination in low phytate crops. Plant hormone- and antioxidation-related genes might strongly contribute to the high germination rate in the TW-1-M mutant.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Seed germination is important to soybean ( <italic> Glycine max </italic> ) growth and development, ultimately affecting soybean <span class="trait">yield.</span> A lower seed field emergence has been the main hindrance for breeding soybeans low in phytate. Although this reduction could be overcome by additional breeding and selection, the mechanisms of seed germination in different low phytate mutants remain unknown. In this study, we performed a comparative transcript analysis of two low phytate soybean mutants (TW-1 and TW-1-M), which have the same mutation, a 2&#x000a0;bp deletion in <italic> <span class="gene">GmMIPS1</span> </italic> , but show a significant difference in seed field emergence, TW-1-M was higher than that of TW-1 .
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Identification of regulatory networks and hub genes controlling soybean seed set and size using RNA sequencing analysis</h2>
			<h3>PMID:28087653</h3>
				

				
					<h3>PMCID:5429000</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Du </surname>  <given-names> Juan </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To understand the gene expression networks controlling soybean seed set and size, transcriptome analyses were performed in three early seed developmental stages, using two genotypes with contrasting seed size. The two-dimensional data set provides a comprehensive and systems-level view on dynamic gene expression networks underpinning soybean seed set and subsequent development. Using pairwise comparisons and weighted gene coexpression network analyses, we identified modules of coexpressed genes and hub genes for each module. Of particular importance are the discoveries of specific modules for the large seed size variety and for seed developmental stages. A large number of candidate regulators for seed size, including those involved in hormonal signaling pathways and transcription factors, were transiently and specifically induced in the early developmental stages. The soybean homologs of a brassinosteroid signaling receptor kinase, a brassinosteroid-signaling kinase, were identified as hub genes operating in the seed coat network in the early seed maturation stage. Overexpression of a candidate seed size regulatory gene, <span class="gene">GmCYP78A5,</span> in transgenic soybean resulted in increased seed size and seed weight. Together, these analyses identified a large number of potential key regulators controlling soybean seed set, seed size, and, consequently, <span class="trait">yield</span> potential, thereby providing new insights into the molecular networks underlying soybean seed development.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In seed coat samples harvested at the S3 stage, 310 and 222 genes were found to be significantly up- or down-regulated, respectively, in the <span class="gene">V1</span> genotype compared with their expression in <span class="gene">V2</span> ( <xref ref-type="fig" rid="F3"> Fig. 3A </xref> ). The up- or down-regulated DEGs are shown in <xref ref-type="table" rid="T6"> Tables 6 </xref> and <xref ref-type="table" rid="T7"> 7 </xref> , respectively. Highly expressed genes include genes encoding a TGACG motif-binding factor 4, G-box binding factor 3, MYB domain protein 116, and a cohort of hormone signal transduction genes encoding indole-3-acetic acid 6, BRI1-associated receptor kinase, and an ethylene-forming enzyme. Ubiquitin-mediated proteolysis has been reported to regulate seed size ( <xref rid="CIT0032" ref-type="bibr"> Li and Li, 2014 </xref> ; <xref rid="CIT0044" ref-type="bibr"> Orozco-Arroyo <italic> et al. </italic> , 2015 </xref> ). In our study, a number of ubiquitin-mediated proteolysis candidate genes were found to be significantly enriched in seed coat at the early seed maturation stage (S3-1) in V1. Cytochrome <span class="gene">P450</span> family genes have been reported to positively regulate seed <span class="trait">yield</span> in rice and soybean ( <xref rid="CIT0062" ref-type="bibr"> Wang <italic> et al. </italic> , 2015 </xref> ; <xref rid="CIT0067" ref-type="bibr"> Yang <italic> et al. </italic> , 2013 </xref> ). In our study, several candidate cytochrome <span class="gene">P450</span> family genes exhibited high expression levels in the seed coat of the large seed cultivar, V1. Interestingly, a glutathione peroxidase gene was specifically expressed in V1S3-1, but not in the other samples of <span class="gene">V1</span> or any sample of V2. Two leucine-rich repeat protein kinase genes showed over four-fold increased expression in V1S3-1 in comparison with V2S3-1.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Root xylem plasticity to improve water use and <span class="trait">yield</span> in water-stressed soybean</h2>
			<h3>PMID:28064176</h3>
				

				
					<h3>PMCID:5428998</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Prince </surname>  <given-names> Silvas J. </given-names>  </name>  <xref ref-type="author-notes" rid="fn-0001"> * </xref>  <xref ref-type="author-notes" rid="fn-0002">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We tested the hypothesis that increasing the number of metaxylem vessels would enhance the efficiency of water uptake in soybean (Glycine max) and decrease the <span class="trait">yield</span> <span class="gene">gap</span> in water-limited environments. A panel of 41 soybean accessions was evaluated in greenhouse, rainout shelter, and rain-fed field environments. The metaxylem number influenced the internal capture of CO2 and improved stomatal conductance, enhancing water uptake/use in soybeans exposed to stress during the reproductive stage. We determined that other root anatomical features, such as cortex cell area and the percentage of stele that comprised cortical cells, also affected seed <span class="trait">yield</span> under similar growth parameters. Seed <span class="trait">yield</span> was also impacted by pod retention rates under <span class="trait">drought</span> stress (24-80 pods/plant). We surmise that effective biomass allocation, that is, the transport of available photosynthates to floral structures at late reproductive growth stages (R6-R7), enables <span class="trait">yield</span> protection under <span class="trait">drought</span> stress. A mesocosm study of contrasting lines for <span class="trait">yield</span> under <span class="trait">drought</span> stress and root anatomical features revealed that increases in metaxylem number as an adaptation to <span class="trait">drought</span> in the high-yielding lines improved root hydraulic conductivity, which reduced the metabolic cost of exploring water in deeper soil strata and enhanced water transport. This allowed the maintenance of shoot physiological processes under water-limited conditions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Multiple studies in maize ( <italic> Zea mays </italic> ) have reported that root cortical cells and cortical aerenchyma confer improved plant performance in nutrient-limited soils, including those with limited phosphorus ( <xref rid="CIT0013" ref-type="bibr"> <span class="gene">Fan</span> <italic> et al. </italic> , 2003 </xref> ; <xref rid="CIT0023" ref-type="bibr"> Lynch, 2011 </xref> ), nitrogen ( <xref rid="CIT0039" ref-type="bibr"> Saengwilai <italic> et al. </italic> , 2014 </xref> ), and water availability ( <xref rid="CIT0048" ref-type="bibr"> Zhu <italic> et al. </italic> , 2010 </xref> ; <xref rid="CIT0018" ref-type="bibr"> Jaramillo <italic> et al. </italic> , 2013 </xref> ; <xref rid="CIT0010" ref-type="bibr"> Chimungu <italic> et al </italic> ., 2015 <italic> b </italic> </xref> ). Root cortical cell size (CCS) and file number have been associated with deeper soil exploration and greater water acquisition, and thus enhanced <span class="trait">drought</span> tolerance in maize ( <xref rid="CIT0007" ref-type="bibr"> Chimungu <italic> et al. </italic> , 2014 <italic> a </italic> , <italic> b </italic> </xref> ; <xref rid="CIT0010" ref-type="bibr"> Chimungu <italic> et al. </italic> , 2015 <italic> b </italic> </xref> ), providing higher biomass and grain yields ( <xref rid="CIT0007" ref-type="bibr"> Chimungu <italic> et al. </italic> , 2014 <italic> a </italic> , <italic> b </italic> </xref> ). Of the studies root phenes, stele diameter was observed to be a better predictor of root penetrative ability than root diameter ( <xref rid="CIT0009" ref-type="bibr"> Chimungu <italic> et al </italic> ., 2015 <italic> a </italic> </xref> ). A recent comparative study of rice ( <italic> Oryza sativa </italic> ) and wheat ( <italic> Triticum aestivum </italic> ) by <xref rid="CIT0019" ref-type="bibr"> Kadam <italic> et al. </italic> , 2015 </xref> revealed that stele and xylem number/diameter were more responsive to water-deficit conditions in wheat than in rice owing to xylem developmental plasticity. Similar interspecies variation in xylem diameter was observed in a <span class="trait">drought</span> study across a single genotype of six different legume crops compared with pearl millet ( <italic> Pennisetum glaucum </italic> ) ( <xref rid="CIT0032" ref-type="bibr"> Purushothaman <italic> et al. </italic> , 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A recent study in different rain-fed environments varying in soil type and climatic factors revealed that soybean genotypes with better modulation of shoot and root traits were more productive upon exposure to <span class="trait">drought</span> stress in the reproductive stage ( <xref rid="CIT0030" ref-type="bibr"> Prince <italic> et al. </italic> , 2016 </xref> ). This study also detailed the natural genetic ability of exotic (landraces not of US origin) species to gain access to water and nutrients by enhancing their lateral root angle, a functional trait with the ability to improve grain production in water-limited environments. However, in soybean there is a <span class="trait">yield</span> gap, which is the difference between the existing <span class="trait">yield</span> potential of a given variety and on-farm <span class="trait">yield</span> patterns ( <xref rid="CIT0044" ref-type="bibr"> Van Roekel and Purcell, 2014 </xref> ). This <span class="trait">yield</span> <span class="gene">gap</span> changes dramatically with water availability during a cropping season ( <xref rid="CIT0015" ref-type="bibr"> Grassini <italic> et al. </italic> , 2014 <italic> a </italic> </xref> ; <xref rid="CIT0016" ref-type="bibr"> Grassini <italic> et al. </italic> , 2015 </xref> ). Identifying genotypes with effective water uptake and use efficiency in both rain-fed and irrigated fields is critical to narrow this <span class="trait">yield</span> gap.
					</p>
				
					<p>&nbsp;&nbsp;
						Various root anatomical phenes have been reported to be associated with plant performance and productivity under water-limited conditions in several cereal crops ( <xref rid="CIT0007" ref-type="bibr"> Chimungu <italic> et al. </italic> , 2014 <italic> a </italic> , <italic> b </italic> </xref> ; <xref rid="CIT0010" ref-type="bibr"> Chimungu <italic> et al. </italic> , 2015 <italic> b </italic> </xref> ; <xref rid="CIT0019" ref-type="bibr"> Kadam <italic> et al. </italic> , 2015 </xref> ) and a few in legume crops ( <xref rid="CIT0028" ref-type="bibr"> Pe&#x000f1;a-Valdivia <italic> et al. </italic> , 2010 </xref> ; <xref rid="CIT0032" ref-type="bibr"> Purushothaman <italic> et al. </italic> , 2013 </xref> ). To the best of our knowledge, none of the previous studies correlated reduced MX with enhanced <span class="trait">yield</span> performance under <span class="trait">drought</span> stress. These water-conducting xylem elements play a vital role not only in the uptake but also in the transport of nutrients and water within the plant system ( <xref rid="CIT0042" ref-type="bibr"> Steudle and Peterson, 1998 </xref> ). In a recent study, <xref rid="CIT0019" ref-type="bibr"> Kadam <italic> et al. </italic> (2015) </xref> demonstrated the plasticity of root elements in wheat: stele and xylem properties are key traits that improve water-use efficiency in wheat over that in rice. This study has emphasized the need to establish the relationships between root morphological and anatomical trait plasticity and <span class="trait">yield</span> under stress in soybean. The present study has established the relationship between the plasticity of MX and xylem diameter with improved <span class="trait">yield</span> under water-deficit stress. Other root anatomical trait such as <span class="gene">CCS</span> may also play a beneficial role in enhancing <span class="trait">yield</span> under <span class="trait">drought</span> stress. A study by <xref rid="CIT0007" ref-type="bibr"> Chimungu <italic> et al. </italic> (2014 <italic> a </italic> </xref> ) showed that maize lines with increased <span class="gene">CCS</span> had improved soil exploration, water acquisition, growth, and development under <span class="trait">drought.</span> Therefore, more study is required to provide further validation with soybean lines with contrasting CCA and to clarify the relationship of CCA with enhanced <span class="trait">yield.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						The three HYS lines identified in this study within the soybean NAM panel could facilitate the development of soybean lines with different drought-adaptive mechanisms to stabilize the seed <span class="trait">yield</span> under water-limited conditions. PI 427136 among the HYS group exhibits superior constitutive seedling root architectural traits and a reduction of leaf area, allowing for efficient water uptake and use during periods of water limitation. Similar modulation of shoot and root traits has been reported previously for other exotic soybean PIs ( <xref rid="CIT0030" ref-type="bibr"> Prince <italic> et al. </italic> , 2016 </xref> ). Interestingly, the mapping population generated by all of the HYS individuals and the hub parent IA3203 had higher yielded than other crosses did across several locations in the US differing in soil and climatic factors ( <xref rid="CIT0014" ref-type="bibr"> Gibson, 2015 </xref> ). Thus, these three HYS genotypes should be used as the breeding material to narrow the <span class="trait">yield</span> <span class="gene">gap</span> in soybean ( <xref rid="CIT0015" ref-type="bibr"> Grassini <italic> et al. </italic> , 2014 <italic> a </italic> </xref> ; <xref rid="CIT0016" ref-type="bibr"> Grassini <italic> et al. </italic> , 2015 </xref> ) by improving water uptake and use efficiency in both rain-fed and irrigated crops. The NAM population developed with these lines would provide valuable genetic material to map <span class="trait">yield</span> and yield-related components in water-limited environments. In addition to the elite lines (HYS1 and 2), the exotic PI 427136 (HYS3) line will bring novel alleles to the elite germplasm to enhance productivity in rain-fed soybean-growing environments.
					</p>
				
					<p>&nbsp;&nbsp;
						Modulating leaf and root adaptive mechanisms is essential to protect <span class="trait">yield</span> during <span class="trait">drought</span> stress. We propose that the selection of the key seedling root anatomical trait, MX, would facilitate the selection of genotypes suited for both optimum and water-deficit environments. Plasticity in MX number and xylem diameter facilitate improved water-use efficiency and <span class="trait">yield</span> protection under water stress. The high-yielding genotypes we have identified to be suited to both terminal <span class="trait">drought</span> stress and receding soil moisture conditions will help the research community to narrow the <span class="trait">yield</span> <span class="gene">gap</span> in soybean in water-limited conditions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Implications of terminal oxidase function in regulation of salicylic acid on soybean seedling photosynthetic performance under water stress.</h2>
			<h3>PMID:28024235  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Tang Y(1), Sun X(2), Wen T(3), Liu M(4), Yang M(4), Chen X(4).
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The aim of this study is to investigate whether exogenous application of salicylic acid (SA) could modulate the photosynthetic capacity of soybean seedlings in water stress tolerance, and to clarify the potential functions of terminal oxidase (plastid terminal oxidase (PTOX) and alternative oxidase (AOX)) in SA' s regulation on photosynthesis. The effects of SA and water stress on gas exchange, pigment contents, chlorophyll fluorescence, enzymes (guaiacol peroxidase (POD; EC 1.11.1.7), superoxide dismutase (SOD; EC 1.15.1.1), catalase (CAT; EC 1.11.1.6), ascorbate peroxidase (APX; EC 1.11.1.11) and NADP-malate dehydrogenase (NADP-MDH; EC1.1.1.82)) activity and transcript levels of PTOX, AOX1, AOX2a, AOX2b were examined in a hydroponic cultivation system. Results indicate that water stress significantly decreased the photosynthetic rate (Pn), stomatal conductance (Gs), transpiration rate (E), pigment contents (Chla + b, Chla/b, Car), maximum quantum <span class="trait">yield</span> of PSⅡphotochemistry (Fv/Fm), efficiency of excitation capture of open PSⅡcenter (Fv'/Fm'), quantum efficiency of PSⅡphotochemistry (ΦPSⅡ), photochemical quenching (qP), and increased malondialdehyde (MDA) content and the activity of all the enzymes. SA pretreatment led to significant decreases in Ci and MDA content, and increases in Pn, Gs, E, pigment contents, Fv/Fm, Fv'/Fm', ΦPSⅡ, qP, and the activity of all the enzymes. SA treatment and water stress alone significantly up-regulated the expression of PTOX, <span class="gene">AOX1</span> and AOX2b. SA pretreatment further increased the transcript levels of PTOX and AOX2b of soybean seedling under water stress. These results indicate that SA application alleviates the water stress-induced decrease in photosynthesis may mainly through maintaining a lower reactive oxygen species (ROS) level, a greater PSⅡefficiency, and an enhanced alternative respiration and chlororespiration. PTOX and AOX may play important roles in SA-mediated <span class="trait">resistance</span> to water stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptomic analysis reveals the flooding tolerant mechanism in flooding tolerant line and abscisic acid treated soybean.</h2>
			<h3>PMID:28012053  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yin X(1)(2), Hiraga S(2), Hajika M(2), Nishimura M(3), Komatsu S(4)(5).
				<br/> 
				年份：2017 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is highly sensitive to flooding stress and exhibits markedly reduced plant growth and grain <span class="trait">yield</span> under flooding conditions. To explore the mechanisms underlying initial flooding tolerance in soybean, RNA sequencing-based transcriptomic analysis was performed using a flooding-tolerant line and ABA-treated soybean. A total of 31 genes included 12 genes that exhibited similar temporal patterns were commonly changed in these plant groups in response to flooding and they were mainly involved in RNA regulation and protein metabolism. The mRNA expression of matrix metalloproteinase, glucose-6-phosphate isomerase, ATPase family <span class="gene">AAA</span> domain-containing protein 1, and cytochrome <span class="gene">P450</span> 77A1 was up-regulated in wild-type soybean under flooding conditions; however, no changes were detected in the flooding-tolerant line or ABA-treated soybean. The mRNA expression of cytochrome <span class="gene">P450</span> 77A1 was specifically up-regulated in root tips by flooding stress, but returned to the level found in control plants following treatment with the <span class="gene">P450</span> inhibitor uniconazole. The survival ratio and root fresh weight of plants were markedly improved by 3-h uniconazole treatment under flooding stress. Taken together, these results suggest that cytochrome <span class="gene">P450</span> 77A1 is suppressed by uniconazole treatment and that this inhibition may enhance soybean tolerance to flooding stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protein Selectivity Controlled by Polymer Charge Density and Protein <span class="trait">Yield:</span> Carboxylated Polysaccharides versus Sulfated Polysaccharides.</h2>
			<h3>PMID:27933875  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Li X(1), Hua Y(1), Chen Y(1), Kong X(1), Zhang C(1).
				<br/> 
				年份：2016 Nov 30
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effect of polymer charge density on protein selectivity in the presence of carboxylated polysaccharides (CPS) and sulfated polysaccharides (SPS) was investigated for Kunitz trypsin inhibitor/Bowman-Birk protease inhibitor (KTI/BBI, KBM). To determine the conditions for coacervation or precipitation as a function of polymer charge densities, turbidimetric titrations and Tricine-SDS-PAGE were used. Polymer charge density as well as chain flexibility greatly influenced the strength of interactions and protein recovery. Although charge compensation must occur for CPS-KBM complexes, SPS-KBM systems did not require conservation of charge neutrality. Despite their similar isoelectric points, <span class="gene">KTI</span> bound preferentially to CPS and SPS due to its higher affinity compared to BBI. Complexation of KBM with the polysaccharide with the lowest charge density, arabic gum, expectedly cannot realize the purification of BBI under conditions where binding to more highly charged polysaccharides occurs. This work will be beneficial to selective purification of target proteins through control of protein-polysaccharide complexation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genotyping-by-Sequencing-Based Investigation of the Genetic Architecture Responsible for a &#x0223c;Sevenfold Increase in Soybean Seed Stearic Acid</h2>
			<h3>PMID:27866151</h3>
				

				
					<h3>PMCID:5217118</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Heim </surname>  <given-names> Crystal B. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gillman </surname>  <given-names> Jason D. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Division of Plant Sciences, University of Missouri, Columbia, Missouri 65211 </aff>  <aff id="aff2">  <label> &#x02020; </label> USDA-ARS, Columbia, Missouri, 65211 </aff>
				<br/> 
				年份：<day> 15 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean oil is highly unsaturated but oxidatively unstable, rendering it nonideal for food applications. Until recently, the majority of soybean oil underwent partial chemical hydrogenation, which produces trans fats as an unavoidable consequence. Dietary intake of trans fats and most saturated fats are conclusively linked to negative impacts on cholesterol levels and cardiovascular health. Two major soybean oil breeding targets are: (1) to reduce or eliminate the need for chemical hydrogenation, and (2) to replace the functional properties of partially hydrogenated soybean oil. One potential solution is the elevation of seed stearic acid, a saturated fat which has no negative impacts on cardiovascular health, from 3 to 4% in typical cultivars to &gt; 20% of the seed oil. We performed QTL analysis of a population developed by crossing two mutant lines, one with a missense mutation affecting a stearoyl-acyl-carrier protein desaturase gene resulting in ∼11% seed stearic acid crossed to another mutant, A6, which has 24-28% seed stearic acid. Genotyping-by-sequencing (GBS)-based QTL mapping identified 21 minor and major effect QTL for six seed oil related traits and plant height. The inheritance of a large genomic deletion affecting chromosome 14 is the basis for largest effect QTL, resulting in ∼18% seed stearic acid. This deletion contains <span class="gene">SACPD-C</span> and another gene(s); loss of both genes boosts seed stearic acid levels to ≥ 18%. Unfortunately, this genomic deletion has been shown in previous studies to be inextricably correlated with reduced seed <span class="trait">yield.</span> Our results will help inform and guide ongoing breeding efforts to improve soybean oil oxidative stability.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To date, there is only one known naturally occurring source of elevated stearic acid, FAM94-41, which contains a missense mutation in <italic> <span class="gene">SACPD-C</span> </italic> and &#x0223c;13% seed stearic acid ( <xref rid="bib61" ref-type="bibr"> Zhang <italic> et al. </italic> 2008 </xref> ). All other lines with elevated stearic acid are due to mutagenesis ( <xref rid="bib4" ref-type="bibr"> Bolon <italic> et al. </italic> 2011 </xref> ; <xref rid="bib18" ref-type="bibr"> Gillman <italic> et al. </italic> 2014 </xref> ; <xref rid="bib37" ref-type="bibr"> Rahman <italic> et al. </italic> 1995 </xref> , <xref rid="bib38" ref-type="bibr"> 1997 </xref> ). The majority of lines contain 8&#x02013;12.5% seed stearic acid, but one notable exception is mutant line A6, which has been reported to have arisen by sodium azide-induced mutagenesis ( <xref rid="bib19" ref-type="bibr"> Hammond and Fehr 1983 </xref> ), but which contains several large genomic deletions more consistent with radiation-induced mutagenesis. A6 seeds exhibit a dramatic increase in seed stearic acid levels (&#x0223c;24&#x02013;28%, as compared to the wild-type 4%). A6 bears several large genomic deletions, including a large portion of chromosome 14 (&#x0223c;6,221,000 bp) corresponding to &#x0223c;1/8 of the chromosome ( <xref rid="bib18" ref-type="bibr"> Gillman <italic> et al. </italic> 2014 </xref> ). Regrettably, A6 is also extremely agronomically deficient, exhibiting poor germination, low seed <span class="trait">yield,</span> reduced seed quality, extreme early maturity (MG 0), and short plant stature when alleles for elevated stearic acid are present ( <xref rid="bib30" ref-type="bibr"> Lundeen <italic> et al. </italic> 1987 </xref> ; <xref rid="bib19" ref-type="bibr"> Hammond and Fehr 1983 </xref> ). Some of the defects are likely due to one (or more) of the significant genomic deletions ( <xref rid="bib18" ref-type="bibr"> Gillman <italic> et al. </italic> 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Early studies have definitively demonstrated that A6 alleles for elevated stearic acid are also associated with decreased seed <span class="trait">yield</span> and unacceptable agronomic performance ( <xref rid="bib30" ref-type="bibr"> Lundeen <italic> et al. </italic> 1987 </xref> ), presumably due to the large genomic deletion responsible for elevated stearic acid. In our recent work, we demonstrated that loss of <span class="gene">SACPD-C</span> activity is inextricably correlated with nodule morphological abnormalities, early nodule senescence, and an impaired ability to fix nitrogen ( <xref rid="bib18" ref-type="bibr"> Gillman <italic> et al. </italic> 2014 </xref> ; <xref rid="bib23" ref-type="bibr"> Krishnan <italic> et al. </italic> 2016 </xref> ). Although we did not measure seed <span class="trait">yield</span> in our present study, we noted a decrease in seed oil content in lines that inherited the A6 chromosome 14 deletion ( <xref ref-type="table" rid="t5"> Table 5 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We evaluated the genetic architecture for extremely elevated stearic acid (&#x0003e; 20%) through GBS/QTL analysis of RILs produced by reciprocal crosses between two mutant lines. Numerous minor effect QTL were identified for soybean fatty acids, total oil, and for plant height. In contrast to <italic> a priori </italic> expectation, a major effect QTL for stearic acid corresponded to a large genomic deletion from line A6, which explained &#x0223c;57% of the phenotypic variance for seed stearic acid. However, the loss of <span class="gene">SACPD-C</span> activity <italic> per se </italic> boosts seed stearic acid from &#x0223c;3% stearic acid to &#x0223c;12.5%. These results suggest that the large genomic deletion in A6 contains both <italic> <span class="gene">SACPD-C</span> </italic> and an unidentified gene; the loss of both in line A6 increases seed stearic acid content to &#x0223c;18% of seed oil. A combination of the major QTL and two minor effect QTL was able to boost seed stearic acid to above 20% of seed oil and meet the market target, but this elevation is expected to occur at a significant cost to seed <span class="trait">yield.</span> These genetic results will help guide and inform breeding efforts to increase soybean oil oxidative stability.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="gene">TIP</span> Gene Family Analysis and Characterization of <span class="gene">GmTIP1;5</span> and <span class="gene">GmTIP2;5</span> Water Transport Activity</h2>
			<h3>PMID:27818669</h3>
				

				
					<h3>PMCID:5073556</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103322/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Na </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/384539/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> Rupesh K. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103657/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant B. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/104588/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prince </surname>  <given-names> Silvas J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/142509/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249960/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mutava </surname>  <given-names> Raymond </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pike </surname>  <given-names> Sharon M. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/375516/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gassmann </surname>  <given-names> Walter </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/26586/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>
				<br/> 
				年份：<day> 04 </day>  <month> 10 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean, one of the most important crops worldwide, is severely affected by abiotic stress. <span class="trait">Drought</span> and flooding are the major abiotic stresses impacting soybean <span class="trait">yield.</span> In this regard, understanding water uptake by plants, its utilization and transport has great importance. In plants, water transport is mainly governed by channel forming aquaporin proteins (AQPs). Tonoplast intrinsic proteins (TIPs) belong to the plant-specific AQP subfamily and are known to have a role in abiotic stress tolerance. In this study, 23 soybean <span class="gene">TIP</span> genes were identified based on the latest soybean genome annotation. TIPs were characterized based on conserved structural features and phylogenetic distribution. Expression analysis of soybean <span class="gene">TIP</span> genes in various tissues and under abiotic stress conditions demonstrated tissue/stress-response specific differential expression. The natural variations for <span class="gene">TIP</span> genes were analyzed using whole genome re-sequencing data available for a set of 106 diverse soybean genotypes including wild types, landraces and elite lines. Results revealed 81 single-nucleotide polymorphisms (SNPs) and several large insertions/deletions in the coding region of TIPs. Among these, non-synonymous SNPs are most likely to have a greater impact on protein function and are candidates for molecular studies as well as for the development of functional markers to assist breeding. The solute transport function of two TIPs was further validated by expression in Xenopus laevis oocytes. <span class="gene">GmTIP1;5</span> was shown to facilitate the rapid movement of water across the oocyte membrane, while <span class="gene">GmTIP2;5</span> facilitated the movement of water and boric acid. The present study provides an initial insight into the possible roles of soybean <span class="gene">TIP</span> genes under abiotic stress conditions. Our results will facilitate elucidation of their precise functions during abiotic stress responses and plant development, and will provide potential breeding targets for modifying water movement in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Tonoplast intrinsic proteins genes also have been reported to be involved in the elevation of abiotic stress tolerance in several plant species. Notably, the <span class="gene">TIP</span> gene <italic> TsTIP1;2 </italic> from <italic> Thellungiella salsuginea </italic> provided increased tolerance against <span class="trait">drought,</span> salt and oxidative stresses when ectopically expressed in <italic> Arabidopsis </italic> ( <xref rid="B57" ref-type="bibr"> Wang et al., 2014 </xref> ). Similarly, increased salinity tolerance was achieved with the heterologous expression of tomato <italic> SITIP2;2 </italic> in <italic> Arabidopsis </italic> ( <xref rid="B59" ref-type="bibr"> Xin et al., 2014 </xref> ). In soybean, <xref rid="B61" ref-type="bibr"> Zhang et al. (2016) </xref> observed that the expression pattern of <italic> <span class="gene">GmTIP2;3</span> </italic> was affected by PEG and ABA, and overexpressing <italic> <span class="gene">GmTIP2;3</span> </italic> in yeast cells improved osmotic stress tolerance. However, another TIP2 gene, <italic> GsTIP2;1 </italic> cloned from <italic> Glycine soja </italic> , resulted in reduced tolerance to salt and dehydration stress when overexpressed in <italic> Arabidopsis </italic> ( <xref rid="B58" ref-type="bibr"> Wang et al., 2011 </xref> ). Such contrasting results indicate diverse regulation of TIPs within the subfamily that may be due to tissue-specific expression patterns, since <italic> <span class="gene">GmTIP2;3</span> </italic> was found to be highly expressed in roots whereas <italic> GsTIP2;1 </italic> showed comparatively higher expression in leaves. Higher water movement by <italic> <span class="gene">GmTIP2;3</span> </italic> in roots can be correlated with efficient water uptake, leading to enhanced osmotic stress tolerance. In contrast, water movement regulated by <italic> GsTIP2;1 </italic> seems to increase water loss through transpiration. Similar observations have been reported in rice, where unbalanced expression of most aquaporins in leaves compared to root is thought to result in rapid depletion of leaf water and subsequent inhibition of photosynthesis ( <xref rid="B32" ref-type="bibr"> Nada and Abogadallah, 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The RNA-seq data generated by <xref rid="B22" ref-type="bibr"> Libault et al. (2010) </xref> for nine different tissues including flower, leaves, nodules, pod, root, root hair, seed, shoot apical meristem, and stem were used to analyze expression patterns of <span class="gene">GmTIPs.</span> Expression profiling of <span class="gene">GmTIPs</span> in leaf tissues of PI 567690 <span class="trait">(drought</span> tolerant) and Pana <span class="trait">(drought</span> susceptible) grown under <span class="trait">drought</span> conditions, and PI 408105A (flooding tolerant) and S99-2281 (PI 654356, flooding susceptible) grown under flooding conditions, was extracted from earlier reported data ( <xref rid="B31" ref-type="bibr"> Mutava et al., 2015 </xref> ; <xref rid="B39" ref-type="bibr"> Prince et al., 2015 </xref> ; <xref rid="B49" ref-type="bibr"> Syed et al., 2015 </xref> ). Briefly, at the V5 stage (five unfolded trifoliate leaves), <span class="trait">drought</span> stress was imposed by withdrawing water for 21 days and flooding stress was imposed by overwatering for 15 days. Similarly, RNA-seq data for Williams 82 plants subjected to very mild stress (VMS), mild stress (MS), and severe stress (SS) conditions, as well as recovery from severe stress after re-watering (SR), was used to study expression of <span class="gene">GmTIPs</span> ( <xref rid="B48" ref-type="bibr"> Song et al., 2016 </xref> ). Hierarchical clustering of expression data was performed using dCHIP software ( <xref rid="B19" ref-type="bibr"> Li and Wong, 2001 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The tissue specific expression pattern of <span class="gene">GmTIPs</span> was investigated in the Williams 82 genotype under varying water-deficit conditions ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ). <italic> <span class="gene">GmTIP3;2,</span> 3;3 </italic> , and <italic> 3;4 </italic> were up-regulated in shoots under serious <span class="trait">drought</span> condition, even after water-recovery, and down-regulated in all other tissues and conditions. Interestingly, the whole <span class="gene">GmTIP2</span> subfamily was down-regulated under very mild <span class="trait">drought</span> stress in leaf tissue and up-regulated after severe <span class="trait">drought</span> stress and water-recovery, except one ( <italic> <span class="gene">GmTIP2;2</span> </italic> ). These results indicated that a gene subfamily may share similar regulatory elements, and also have similar physiological functions. Most of the root-specific expressed genes were induced under varying water-deficit conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						<bold> Expression profiles of soybean <span class="gene">TIP</span> genes under different abiotic stress conditions or in different germplasms. (A) </bold> Heatmap representation of expression patterns of soybean <span class="gene">TIP</span> genes across the root, leaf, and shoot tissue in Williams 82 under varying water-deficit stress conditions (VMS: very mild stress; MS: mild stress; SS: severe stress; SR: water recovery after severe stress). <bold> (B) </bold> Expression profiles of the soybean <span class="gene">TIP</span> genes in leaves of Pana (fast wilting, <span class="trait">drought</span> vs. control), PI 567690 (slow wilting, <span class="trait">drought</span> vs. control), PI 408105A (flooding tolerant, flooding vs. control), S99-2281(flooding sensitive, flooding vs. control). The expression data values were median-centered and normalized for each gene before transforming to the color scale (log2-transformed ratios). The color <span class="gene">bar</span> at the bottom shows the range of expression values from increased expression (red) to decreased expression (green), 0 means no gene expression pattern changed (Black).
					</p>
				
					<p>&nbsp;&nbsp;
						To further investigate the expression profiles of <span class="gene">TIP</span> genes under <span class="trait">drought</span> and flooding conditions, the expression patterns of <span class="gene">TIP</span> gene families were extracted from two RNA-seq datasets ( <xref rid="B39" ref-type="bibr"> Prince et al., 2015 </xref> and unpublished data) ( <bold> Figure <xref ref-type="fig" rid="F6"> 6B </xref> </bold> ). PI 567690 is a slow wilting soybean line, and Pana is a fast wilting line under <span class="trait">drought</span> conditions ( <xref rid="B35" ref-type="bibr"> Pathan et al., 2014 </xref> ; <xref rid="B31" ref-type="bibr"> Mutava et al., 2015 </xref> ). <italic> <span class="gene">GmTIP2;1,</span> 2;2, 2;3, 2;4, 2;5, 3;2 </italic> , and <italic> 5;1 </italic> mRNA levels were induced in leaves of Pana, but decreased in leaves of PI 567690 after <span class="trait">drought</span> stress. PI 408105A is a flooding tolerant line, and S99-2281 is a flooding sensitive line ( <xref rid="B31" ref-type="bibr"> Mutava et al., 2015 </xref> ). <italic> <span class="gene">GmTIP1;2,</span> 1;3, 1;4, 1;5, 1;6, 1;9, 2;6 </italic> , and <italic> 4;1 </italic> mRNA levels were induced in leaves of S99-2281, but decreased or were not induced in leaves of PI 408105A after flooding stress. Conversely, <italic> <span class="gene">GmTIP1;8</span> </italic> and <italic> 2;7 </italic> were induced in PI 408105A and decreased in S99-2281. These results indicated that <span class="gene">GmTIP</span> genes could play a role under <span class="trait">drought</span> or flooding stress.
					</p>
				
					<p>&nbsp;&nbsp;
						No non-synonymous SNP was found in <italic> <span class="gene">GmTIP1;5</span> </italic> and <italic> <span class="gene">GmTIP2;5</span> </italic> in 106 soybean germplasm; however, they were shown to be differentially expressed in response to <span class="trait">drought</span> ( <italic> <span class="gene">GmTIP1;5</span> </italic> and <italic> <span class="gene">GmTIP2;5</span> </italic> ) and flooding ( <italic> <span class="gene">GmTIP1;5</span> </italic> ) in different varieties of soybeans with contrasting phenotypes and were mainly expressed in stem where they could be highly important in water transport during abiotic stress. Further in-depth analysis of <italic> <span class="gene">GmTIP1;5</span> </italic> and <italic> <span class="gene">GmTIP2;5</span> </italic> using <italic> X. laevis </italic> oocytes was performed to establish a link between differential expression patterns and trait development. <span class="gene">GmTIP2;5</span> and <span class="gene">GmTIP1;5-expressing</span> oocytes showed a rapid rate of swelling for the first 5 min after being subjected to a hypoosmotic solution ( <bold> Figures <xref ref-type="fig" rid="F8"> 8A&#x02013;C </xref> </bold> ). After 5 min, the rate of swelling in <italic> <span class="gene">GmTIP2;5</span> </italic> expressing oocytes declined as they began to burst. In contrast, <italic> <span class="gene">GmTIP1;5</span> </italic> expressing oocytes continued to increase in diameter at a rate greater than the controls for 20 min. <span class="gene">GmTIP2;5-expressing</span> oocytes swelled more rapidly than <span class="gene">GmTIP1;5-expressing</span> oocytes during the first 5 min. Mock-injected oocytes displayed a slower rate of swelling, probably due to diffusion of water across the plasma membrane. It was concluded from these results that <span class="gene">GmTIP2;5</span> and <span class="gene">GmTIP1;5</span> facilitate the transport of water and are aquaporins.
					</p>
				
					<p>&nbsp;&nbsp;
						Here, 23 full-length aquaporin-coding sequences belonging to the <span class="gene">TIP</span> subfamily were identified in the soybean genome. The number of TIPs in the soybean genome is much higher than those identified in species like <italic> Arabidopsis </italic> , maize, rice, and sorghum ( <xref rid="B40" ref-type="bibr"> Regon et al., 2014 </xref> ). Several studies have shown differential expression of plant aquaporins in response to environmental stresses in several different species ( <xref rid="B1" ref-type="bibr"> Alexandersson et al., 2005 </xref> ; <xref rid="B63" ref-type="bibr"> Zhu et al., 2005 </xref> ; <xref rid="B23" ref-type="bibr"> Ligaba et al., 2011 </xref> ; <xref rid="B4" ref-type="bibr"> Cohen et al., 2013 </xref> ; <xref rid="B21" ref-type="bibr"> Li et al., 2014 </xref> ). Therefore, it is important to investigate the interaction between the expression of <span class="gene">GmTIPs</span> and abiotic stress. This study explored the expression patterns of <span class="gene">GmTIP</span> genes in soybean PI 567690 and Pana lines. The soybean genotype PI 567690 exhibits significantly lower wilting and less <span class="trait">yield</span> loss under <span class="trait">drought</span> condition than the elite cultivar Pana ( <xref rid="B35" ref-type="bibr"> Pathan et al., 2014 </xref> ). The difference in gene expression between PI 567690 and Pana had been investigated by RNA-seq ( <xref rid="B39" ref-type="bibr"> Prince et al., 2015 </xref> ), and our preliminary data indicated that the PI 567690 genotype has a limited transpiration response when exposed to a high vapor pressure deficit (unpublished data). In the present analysis, we found that the expression level of seven <span class="gene">TIP</span> genes was reduced in slow-wilting soybean lines but increased in the fast-wilting soybean lines after <span class="trait">drought</span> stress ( <bold> Figure <xref ref-type="fig" rid="F6"> 6B </xref> </bold> ). At the same time, the expression level of eight different <span class="gene">TIP</span> genes was down-regulated or unchanged in the waterlogging tolerant varieties, but upregulated in the waterlogging susceptible varieties. Only one gene was found to be up-regulated in slow-wilting lines under <span class="trait">drought</span> conditions, and only three genes were found to be up-regulated in the water-logging tolerant lines. These results are consistent with the hypothesis that due to lessened water transport in soybean leaves, the genotype PI 567690 exhibited more <span class="trait">resistance</span> to <span class="trait">drought</span> stress. We therefore conclude that <span class="gene">GmTIPs</span> may play an important role in soybean both in <span class="trait">drought</span> and flooding tolerance. However, the gene regulation patterns in root tissue of water-logging resistant lines should be further explored.
					</p>
				
					<p>&nbsp;&nbsp;
						The expression patterns of <span class="gene">GmTIPs</span> under a given condition varied among different tissues and was complex ( <bold> Figures <xref ref-type="fig" rid="F6"> 6A,B </xref> </bold> ). <italic> <span class="gene">GmTIP1;5</span> </italic> and <italic> <span class="gene">GmTIP2;5</span> </italic> showed a higher expression in stem, but lower expression in other tissues. Interestingly, <italic> <span class="gene">GmTIP2;5</span> </italic> transcript was up-regulated under varying water-deficit stress in root, but down-regulated after water-recovery. This is the <span class="gene">TIP</span> that showed a very rapid water uptake in <italic> Xenopus </italic> oocytes. <italic> <span class="gene">GmTIP1;5</span> </italic> , which showed a slower, but steady uptake in <italic> Xenopus </italic> oocytes, was mainly induced under the VMS condition in root, leaf and shoot, not in other water-deficit conditions. Furthermore, <italic> <span class="gene">GmTIP1;5</span> </italic> is the only one that showed upregulation in slow wilting lines under <span class="trait">drought</span> condition among all <italic> <span class="gene">TIP</span> </italic> genes. In addition, only one synonymous SNP was located in the coding sequences of <italic> <span class="gene">GmTIP1;5</span> </italic> and two synonymous SNPs were located in the coding sequence of <italic> <span class="gene">GmTIP2;5</span> </italic> . Therefore, observed variations in gene expression seem to have prominent roles in functional variation. Further investigation of the water transport function of these two genes in soybean would help to establish a link between differential expression patterns and trait development.
					</p>
				
					<p>&nbsp;&nbsp;
						To gain further insight into the possible physiological functions of the members of this large family and aid the development of genetic engineering in soybean, the <italic> <span class="gene">TIP</span> </italic> that directs tissue-specific expression pattern and functions in water transport would be highly desirable. Overall, a more comprehensive mechanism will emerge as multiple aquaporin transport functions and integration of the different stress signals are determined at the whole plant level. Further characterization of the <span class="gene">GmTIP</span> genes involved in abiotic stress <span class="trait">resistance</span> genotypes may aid in developing tolerant germplasm and cultivars.
					</p>
				
					<p>&nbsp;&nbsp;
						All <italic> G. max </italic> <span class="gene">TIP</span> aquaporins showed the canonical double NPA motif and a group ar/R SF that are conserved across different species. These results indicate that <span class="gene">GmTIPs</span> may play similar roles in regulating water absorption. As such, it is important to validate representative candidate genes for water transport by using other methods such as heterologous expression in <italic> Xenopus </italic> oocytes. Basic <span class="gene">GmTIP</span> gene family information and phylogenetic sequence analysis in the present study provide a list of candidate genes that may play important roles in soybean <span class="trait">drought</span> and flooding tolerance. The function of two of these <span class="gene">GmTIP</span> genes has been demonstrated to be involved in water transport. However, AQPs are not only water transporters but also solute transporters ( <xref rid="B30" ref-type="bibr"> Maurel et al., 2008 </xref> ). Most AQPs can transport glycerol, urea, boric acid or arsenic as we showed with boron and <span class="gene">GmTIP2;5.</span> Yet even though a protein has been predicted to be an aquaporin, it may not transport water. Thus <span class="gene">GmTIP</span> candidate genes must be functionally characterized.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, the soybean <span class="gene">TIP</span> gene family was identified based on the new soybean genome annotation. Twenty-three <span class="gene">TIP</span> members were assigned to five subfamilies based on sequence similarity and phylogenetic relationship. The modulation of expression profiles of these 23 <span class="gene">GmTIP</span> genes was examined with deep transcriptome sequencing under <span class="trait">drought</span> or flooding conditions and during plant development. The potential roles of these genes in transport of substrates were also discussed. The identified natural variations in this gene family from 106 soybean germplasms will benefit future gene functional analysis and utilization. We also demonstrated that <span class="gene">GmTIP1;5</span> and <span class="gene">GmTIP2;5</span> can function as water transporters in oocytes. All results presented here represent an important resource for designing experiments for functional validation of candidate genes in plant development and abiotic stress responses, and for developing soybean germplasm with improved abiotic stress tolerance.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Increasing seed size and quality by manipulating BIG SEEDS1 in legume species.</h2>
			<h3>PMID:27791139</h3>
				

				
			<p>
				作者：Ge L(1), Yu J(2), Wang H(1), Luth D(3)(4), Bai G(2)(5), Wang K(3)(4), Chen R(6).
				<br/> 
				年份：2016 Nov 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant organs, such as seeds, are primary sources of food for both humans and animals. Seed size is one of the major agronomic traits that have been selected in crop plants during their domestication. Legume seeds are a major source of dietary proteins and oils. Here, we report a conserved role for the BIG SEEDS1 (BS1) gene in the control of seed size and weight in the model legume Medicago truncatula and the grain legume soybean (Glycine max). <span class="gene">BS1</span> encodes a plant-specific transcription regulator and plays a key role in the control of the size of plant organs, including seeds, seed pods, and leaves, through a regulatory module that targets primary cell proliferation. Importantly, down-regulation of <span class="gene">BS1</span> orthologs in soybean by an artificial microRNA significantly increased soybean seed size, weight, and amino acid content. Our results provide a strategy for the increase in <span class="trait">yield</span> and seed quality in legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Polymalic acid fermentation by Aureobasidium pullulans for malic acid production from soybean hull and soy molasses: Fermentation kinetics and economic analysis.</h2>
			<h3>PMID:27792926  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cheng C(1), Zhou Y(2), Lin M(3), Wei P(4), Yang ST(5).
				<br/> 
				年份：2017 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Polymalic acid (PMA) production by Aureobasidium pullulans ZX-10 from soybean hull hydrolysate supplemented with corn steep liquor (CSL) gave a malic acid <span class="trait">yield</span> of ∼0.4g/g at a productivity of ∼0.5g/L·h. ZX-10 can also ferment soy molasses, converting all carbohydrates including the raffinose family oligosaccharides to PMA, giving a high titer (71.9g/L) and <span class="trait">yield</span> (0.69g/g) at a productivity of 0.29g/L·h in fed-batch fermentation under nitrogen limitation. A higher productivity of 0.64g/L·h was obtained in repeated batch fermentation with cell recycle and CSL supplementation. Cost analysis for a 5000 MT plant shows that malic acid can be produced at $1.10/kg from soy molasses, $1.37/kg from corn, and $1.74/kg from soybean hull. At the market price of $1.75/kg, malic acid production from soy molasses via <span class="gene">PMA</span> fermentation offers an economically competitive process for industrial production of bio-based malic acid.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine mapping of the Rsv1-h gene in the soybean cultivar Suweon 97 that confers <span class="trait">resistance</span> to two Chinese strains of the soybean mosaic virus.</h2>
			<h3>PMID:27544525  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ma FF(1), Wu XY(1), Chen YX(1), Liu YN(1), Shao ZQ(1), Wu P(1), Wu M(1), Liu CC(1), Wu WP(1), Yang JY(2), Li DX(3), Chen JQ(4), Wang B(5).
				<br/> 
				年份：2016 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: The Rsv1 - h gene in cultivar Suweon 97, which confers <span class="trait">resistance</span> to SMVs, was mapped to a 97.5-kb location (29,815,195-29,912,667 bp on chromosome 13) in the Rsv1 locus, thereby providing additional insights into the molecular nature underlying variations in <span class="trait">resistance</span> alleles in this particular locus. Soybean mosaic virus (SMV) is a well-known devastating pathogen of soybean (Glycine max (L.) Merrill.) causing significant <span class="trait">yield</span> losses and seed quality deterioration. A single dominant allele, Rsv1-h, which confers <span class="trait">resistance</span> to multiple SMV strains, was previously reported in the cultivar Suweon 97, but its exact location is unknown. In the present study, Suweon 97 was crossed with a SMV-sensitive cultivar, Williams 82. Inoculating 267 F 2 individuals with two Chinese SMV strains (SC6-N and SC7-N) demonstrated that one single dominant gene confers SMV <span class="trait">resistance.</span> Another 1,150 F 2 individuals were then screened for two simple sequence repeat (SSR) markers (BARCSOYSSR_13_1103 and BARCSOYSSR_13_1187) that flank the Rsv1 locus. Seventy-four recombinants were identified and 20 additional polymorphic SSR markers within the Rsv1 region were then employed in genotyping these recombinants. F 2:3 and F 3:4 recombinant lines were also inoculated with SC6-N and SC7-N to determine their phenotypes. The final data revealed that in Suweon 97, the Rsv1-h gene that confers <span class="trait">resistance</span> to SC6-N and SC7-N was flanked by BARCSOYSSR_13_1114 and BARCSOYSSR_13_1115, two markers that delimit a 97.5-kb region in the reference Williams 82 genome. In such region, eight genes were present, of which two, <span class="gene">Glyma13g184800</span> and <span class="gene">Glyma13g184900,</span> encode the characteristic CC-NBS-LRR type of <span class="trait">resistance</span> gene and were considered potential candidates for Rsv1-h.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmSALT3, Which Confers Improved Soybean Salt Tolerance in the Field, Increases Leaf Cl <sup> - </sup> Exclusion Prior to Na <sup> + </sup> Exclusion But Does Not Improve Early Vigor under Salinity</h2>
			<h3>PMID:27746805</h3>
				

				
					<h3>PMCID:5043451</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Ying </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/373443/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Lili </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/362771/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qu </surname>  <given-names> Yue </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/362222/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Jingjing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xiexiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hong </surname>  <given-names> Huilong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Zhangxiong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Ruzhen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gilliham </surname>  <given-names> Matthew </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/26895/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guan </surname>  <given-names> Rongxia </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/357972/overview"/>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 9 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinity reduces soybean growth and <span class="trait">yield.</span> The recently identified <span class="gene">GmSALT3</span> (Glycine max salt Tolerance-associated gene on chromosome 3) has the potential to improve soybean yields in salinized conditions. Here we evaluate the impact of <span class="gene">GmSALT3</span> on soybean performance under saline or non-saline conditions. Three sets of near isogenic lines (NILs), with genetic similarity of 95.6-99.3% between each pair of NIL-T and NIL-S, were generated from a cross between two varieties 85-140 (salt-sensitive, S) and Tiefeng 8 (salt-tolerant, T) by using marker-assisted selection. Each NIL-T; 782-T, 820-T and 860-T, contained a common ~1000 kb fragment on chromosome 3 where <span class="gene">GmSALT3</span> was located. We show that <span class="gene">GmSALT3</span> does not contribute to an improvement in seedling emergence rate or early vigor under salt stress. However, when 12-day-old seedlings were exposed to NaCl stress, the NIL-T lines accumulated significantly less leaf Na+ compared with their corresponding NIL-S, while no significant difference of K+ concentration was observed between NIL-T and NIL-S; the magnitude of Na+ accumulation within each NIL-T set was influenced by the different genetic backgrounds. In addition, NIL-T lines accumulated less Cl- in the leaf and more in the root prior to any difference in Na+; in the field they accumulated less pod wall Cl- than the corresponding NIL-S lines. Under non-saline field conditions, no significant differences were observed for <span class="trait">yield</span> related traits within each pair of NIL-T and NIL-S lines, indicating there was no <span class="trait">yield</span> penalty for having the <span class="gene">GmSALT3</span> gene. In contrast, under saline field conditions the NIL-T lines had significantly greater plant seed weight and 100-seed weight than the corresponding NIL-S lines, meaning <span class="gene">GmSALT3</span> conferred a <span class="trait">yield</span> advantage to soybean plants in salinized fields. Our results indicated that <span class="gene">GmSALT3</span> mediated regulation of both Na+ and Cl- accumulation in soybean, and contributes to improved soybean <span class="trait">yield</span> through maintaining a higher seed weight under saline stress.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Crop plants differ greatly in their salinity tolerance as reflected in their different growth responses at different growth stages ( <xref rid="B17" ref-type="bibr"> Foolad and Lin, 1997 </xref> ; <xref rid="B16" ref-type="bibr"> Foolad, 1999 </xref> ; <xref rid="B51" ref-type="bibr"> Takeda and Matsuoka, 2008 </xref> ). Studies on the mechanism of salt tolerance in soybean have focused mainly on seedling ion homeostasis, especially on the relative accumulation of Na <sup> + </sup> , Cl <sup> - </sup> and K <sup> + </sup> ( <xref rid="B2" ref-type="bibr"> Abel and MacKenzie, 1964 </xref> ; <xref rid="B31" ref-type="bibr"> L&#x000e4;uchli and Wieneke, 1979 </xref> ; <xref rid="B43" ref-type="bibr"> Pantalone et al., 1997 </xref> ; <xref rid="B3" ref-type="bibr"> An et al., 2002 </xref> ). <italic> Glycine max </italic> seedlings under NaCl stress have been reported to have sensitivity to both Cl <sup> - </sup> and Na <sup> + </sup> , while <italic> Glycine soja </italic> , a wild relative has been demonstrated to have strong Cl <sup> - </sup> tolerance ( <xref rid="B58" ref-type="bibr"> Umezawa et al., 2000 </xref> ; <xref rid="B36" ref-type="bibr"> Luo et al., 2005 </xref> ; <xref rid="B8" ref-type="bibr"> Chen and Yu, 2007 </xref> ). It was reported about half century ago that the salt tolerance of soybean was controlled by a single dominant allele q <italic> NaCl3 (Ncl) </italic> ( <xref rid="B1" ref-type="bibr"> Abel, 1969 </xref> ). A major QTL or dominant locus on soybean chromosome 3 was identified and validated by several research groups in both cultivated and wild soybean ( <xref rid="B32" ref-type="bibr"> Lee et al., 2004 </xref> ; <xref rid="B7" ref-type="bibr"> Chen et al., 2008 </xref> ; <xref rid="B24" ref-type="bibr"> Hamwieh and Xu, 2008 </xref> ; <xref rid="B23" ref-type="bibr"> Hamwieh et al., 2011 </xref> ; <xref rid="B21" ref-type="bibr"> Ha et al., 2013 </xref> ; <xref rid="B19" ref-type="bibr"> Guan et al., 2014a </xref> ). A salt candidate gene proposed to underpin this locus was identified from the wild soybean <italic> G. soja </italic> , accession W05, ( <italic> Glysoja01g005509) </italic> by using a whole-genome sequencing approach ( <xref rid="B45" ref-type="bibr"> Qi et al., 2014 </xref> ). This paper named the homologous gene in soybean ( <italic> <span class="gene">Glyma03g32900.1</span> </italic> ), <italic> <span class="gene">GmCHX1</span> </italic> , after its putative function as a cation/H <sup> + </sup> exchanger, and proposed that it improved soybean salt tolerance after functional tests in tobacco BY2 cells and soybean hairy root cultures ( <xref rid="B45" ref-type="bibr"> Qi et al., 2014 </xref> ). Concurrent with this study, through map-based cloning from a salt tolerant Chinese soybean variety Tiefeng 8, we identified the same gene <italic> <span class="gene">Glyma03g32900.1</span> </italic> , and named it <italic> <span class="gene">GmSALT3</span> </italic> as it is likely to encode the candidate salt tolerance-associated gene on chromosome 3 ( <xref rid="B20" ref-type="bibr"> Guan et al., 2014b </xref> ). Recently, <xref rid="B12" ref-type="bibr"> Do et al. (2016) </xref> identified the equivalent allele from the salt tolerant Brazilian cultivar FT-Abyara, and named it <italic> Ncl </italic> , which is an abbreviation of the QTL (q <italic> NaCl3 </italic> ) identified by <xref rid="B1" ref-type="bibr"> Abel (1969) </xref> . The tolerant <italic> <span class="gene">GmSALT3</span> </italic> allele was found in <italic> G. max </italic> and <italic> G. soja </italic> germplasm that originated all over China but was most frequently associated with regions with saline soil conditions; however, the sensitive <italic> <span class="gene">GmSALT3</span> </italic> alleles ( <italic> <span class="gene">Gmsalt3</span> </italic> ) were much more prevalent in non-saline regions than saline regions ( <xref rid="B20" ref-type="bibr"> Guan et al., 2014b </xref> ). It was proposed that the expression of the functional salt tolerance gene resulted in an energy burden on plants when salinity was not present ( <xref rid="B45" ref-type="bibr"> Qi et al., 2014 </xref> ), and this may explain its limited distribution in germplasm derived from non-saline areas ( <xref rid="B20" ref-type="bibr"> Guan et al., 2014b </xref> ). <xref rid="B12" ref-type="bibr"> Do et al. (2016) </xref> recently showed in the field using near-isogenic lines that there appeared to be no <span class="trait">yield</span> penalty in non-saline conditions for harboring <italic> <span class="gene">GmSALT3</span> </italic> , but a <span class="trait">yield</span> improvement under saline conditions. Such a property would make this gene ( <italic> CHX1/GmSALT3/Ncl </italic> ) an attractive prospect to breeders and farmers ( <xref rid="B20" ref-type="bibr"> Guan et al., 2014b </xref> ), so further field testing in other soil types and different genetic background is required to confirm its potential ( <xref rid="B12" ref-type="bibr"> Do et al., 2016 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The salt tolerance of different soybean varieties has predo-minantly been evaluated prior to the identification of alleles associated with soybean salt tolerance ( <xref rid="B31" ref-type="bibr"> L&#x000e4;uchli and Wieneke, 1979 </xref> ; <xref rid="B14" ref-type="bibr"> El-Samad and Shaddad, 1997 </xref> ; <xref rid="B58" ref-type="bibr"> Umezawa et al., 2000 </xref> ; <xref rid="B3" ref-type="bibr"> An et al., 2002 </xref> ; <xref rid="B15" ref-type="bibr"> Essa, 2002 </xref> ). Isolation of <italic> CHX1/GmSALT3/Ncl </italic> has allowed the examination of its effects on Na <sup> + </sup> , K <sup> + </sup> and Cl <sup> - </sup> accumulation during salt stress and its effect on the salt tolerance of soybean seedlings has been conducted by at least three different research groups ( <xref rid="B20" ref-type="bibr"> Guan et al., 2014b </xref> ; <xref rid="B45" ref-type="bibr"> Qi et al., 2014 </xref> ; <xref rid="B12" ref-type="bibr"> Do et al., 2016 </xref> ). Both transformation of the salt tolerant allele into the soybean variety Kariyutaka and its introgression into the salt-sensitive cultivar Jackson, significantly decreased the leaf Na <sup> + </sup> , K <sup> + </sup> and Cl <sup> - </sup> under 100 mM NaCl stress, and increased the soybean <span class="trait">yield</span> by 3.6&#x02013;5.5 fold when irrigated the 5-week-old seedling with 1/4 concentration seawater ( <xref rid="B12" ref-type="bibr"> Do et al., 2016 </xref> ). However, the timecourse for the effect of <italic> CHX1/GmSALT3/Ncl </italic> on the accumulation of ions and its effects in different genetic backgrounds at different developmental stages is yet to be evaluated. Here, in order to identify the behavior of <italic> <span class="gene">GmSALT3</span> </italic> in differing genetic backgrounds and in different environments, we developed three sets of near isogenic lines (NILs) derived from progenies of 85&#x02013;140 &#x000d7; Tiefeng 8 using marker-assisted selection for the target allele <italic> <span class="gene">GmSALT3</span> </italic> or <italic> <span class="gene">Gmsalt3</span> </italic> . These NILs were used to study whether <italic> <span class="gene">GmSALT3</span> </italic> had equivalent salinity tolerance in differing genetic backgrounds, to assess the effect of <italic> <span class="gene">GmSALT3</span> </italic> on salt tolerance at the emergence stage and the exclusion of Na <sup> + </sup> and Cl <sup> - </sup> in both seedling and mature seeds, to determine whether <italic> <span class="gene">GmSALT3</span> </italic> positively impacted soybean <span class="trait">yield</span> under saline field conditions and to further examine whether <italic> <span class="gene">GmSALT3</span> </italic> conferred a <span class="trait">yield</span> penalty to soybean <span class="trait">yield</span> under non-saline field conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						When the three sets of NILs were grown in non-saline field conditions (Shunyi, Beijing), NIL-T lines containing <italic> <span class="gene">GmSALT3</span> </italic> had a similar <span class="trait">yield</span> to the corresponding NIL-S lines containing <italic> <span class="gene">Gmsalt3</span> </italic> for the <span class="trait">yield</span> parameters of pod number, seed number, seed weight and 100-seed weight ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ). The <span class="trait">yield</span> related components of both NIL-T lines and NIL-S lines were substantially affected by salinity stress when NILs were grown on saline soil (Tanghai, Hebei province) with the respective NIL-S for each set of lines being noticeable smaller at harvest maturity (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> S3 </xref> ). In terms of the plant height, 820-T and 860-T were significantly taller than that of 820-S and 860-S when grown on saline soil whereas 782-T and 782-S were similar in size at Tanghai in 2014; while in 2015 saline field trial plant height was equal between NIL pairs ( <bold> Figure <xref ref-type="fig" rid="F6"> 6C </xref> </bold> ). Under saline conditions no significant difference in pod number and seed number were observed between each pair of NIL-T and NIL-S. However, the NIL-T lines containing <italic> <span class="gene">GmSALT3</span> </italic> had significantly greater seed weight and 100-seed weight than related NIL-S lines regardless genetic backgrounds under saline conditions ( <bold> Figures <xref ref-type="fig" rid="F6"> 6B,C </xref> </bold> ). The seed weight of NIL-S lines was 30&#x02013;57% lower than corresponding NIL-T lines in 2014 saline field trial, and 37&#x02013;58% lower in 2015 saline field trail. The mean above ground dry mass of the NIL-S lines was lower than the corresponding NIL-T lines, but a significant difference was only observed between 782-T and 782-S (Supplementary Figure <xref ref-type="supplementary-material" rid="SM1"> S4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies have mostly evaluated the salt tolerance of soybean at the seedling stage. Such studies have repeatedly identified the same major QTL independently ( <xref rid="B32" ref-type="bibr"> Lee et al., 2004 </xref> , <xref rid="B33" ref-type="bibr"> 2009 </xref> ; <xref rid="B7" ref-type="bibr"> Chen et al., 2008 </xref> ; <xref rid="B24" ref-type="bibr"> Hamwieh and Xu, 2008 </xref> ; <xref rid="B23" ref-type="bibr"> Hamwieh et al., 2011 </xref> ; <xref rid="B21" ref-type="bibr"> Ha et al., 2013 </xref> ; <xref rid="B19" ref-type="bibr"> Guan et al., 2014a </xref> ), with only a few studies focusing on QTL mapping at the germination stage ( <xref rid="B62" ref-type="bibr"> Zhang et al., 2014 </xref> ). Recently, we and other groups have isolated a dominant salt tolerance gene <italic> CHX1 </italic> / <italic> <span class="gene">GmSALT3/Ncl</span> </italic> located on chromosome 3 ( <xref rid="B20" ref-type="bibr"> Guan et al., 2014b </xref> ; <xref rid="B45" ref-type="bibr"> Qi et al., 2014 </xref> ; <xref rid="B12" ref-type="bibr"> Do et al., 2016 </xref> ). Here, we successfully created three sets of NILs containing <italic> <span class="gene">GmSALT3</span> </italic> and <italic> <span class="gene">Gmsalt3</span> </italic> , respectively. The three tolerant NILs had differing genetic backgrounds whilst containing a common ~1000 kb region including the <italic> <span class="gene">GmSALT3</span> </italic> gene in NIL-T lines or <italic> <span class="gene">Gmsalt3</span> </italic> in NIL-S lines. The NILs differed at the major salt tolerant locus are suitable for assessing the impact of tolerant allele on ions accumulation and <span class="trait">yield</span> related traits.
					</p>
				
					<p>&nbsp;&nbsp;
						The characterization of the different alleles of <italic> <span class="gene">GmSALT3</span> </italic> in our previous report suggested that the tolerant allele had been under significant selection pressure; it was frequently lost in non-saline environments ( <xref rid="B20" ref-type="bibr"> Guan et al., 2014b </xref> ). Thus, this raises an important question of whether <italic> <span class="gene">GmSALT3</span> </italic> incurs a penalty to <span class="trait">yield</span> under non-saline conditions. Since most salinity affected soils are not uniformly saline or apply a constant level of salinity during the entire crop growth cycle, genes with no penalty are more appropriate for breeding salt tolerant crops ( <xref rid="B40" ref-type="bibr"> Munns et al., 2012 </xref> ). In our case, we found that under non-saline field conditions NIL-T lines had similar <span class="trait">yield</span> related traits compared to their related NIL-S lines ( <bold> Figure <xref ref-type="fig" rid="F6"> 6A </xref> </bold> ), indicating no <span class="trait">yield</span> penalty associated with the presence of <italic> <span class="gene">GmSALT3</span> </italic> allele. We therefore find no reason why <italic> <span class="gene">GmSALT3</span> </italic> may have been selected against on the basis of <span class="trait">yield</span> under modern farming practices. This result corroborates the recent findings of <xref rid="B12" ref-type="bibr"> Do et al. (2016) </xref> who also found, using different soybean genetic backgrounds, that the functional <italic> <span class="gene">GmSALT3</span> </italic> allele does not harbor a <span class="trait">yield</span> penalty under non-saline conditions, whilst conferring improved yields under saline conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						The effect of <span class="gene">GmSALT3</span> on soybean growth and <span class="trait">yield</span> was evaluated by comparing three sets of NIL lines over 2 years in Tanghai, Hebei province. Pod number and seed number per plant decreased for both NIL-T and NIL-S lines, compare with control plants, while the difference between each pair of NIL-T and NIL-S was not significant. The lower <span class="trait">yield</span> reduction of a super-nodulating en-b0-1 compared with its normal-nodulating parent Enrei under salinity stress was primarily due to the larger seeds number of en-b0-1 ( <xref rid="B60" ref-type="bibr"> Yasuta and Kokubun, 2014 </xref> ). Analysis of chloride toxicity of soybeans grown in Flatwoods soils fertilized with KCl identified an average of 25% less 100-seed weight in susceptible cultivars than that for tolerant ones ( <xref rid="B44" ref-type="bibr"> Parker et al., 1983 </xref> ). Significant differences in 100-seed weight were observed between each pair of NIL-T and NIL-S, suggesting that NIL-S is compromised in its ability to produce larger seeds and this contributes to the 30&#x02013;58% loss of seed <span class="trait">yield</span> for NIL-S lines ( <bold> Figure <xref ref-type="fig" rid="F6"> 6 </xref> </bold> ). <xref rid="B12" ref-type="bibr"> Do et al. (2016) </xref> also showed that the <italic> <span class="gene">Glyma03g32900</span> </italic> gene cold increase soybean <span class="trait">yield</span> by 3.6&#x02013;5.5 fold when treated with diluted seawater in Japan ( <xref rid="B12" ref-type="bibr"> Do et al., 2016 </xref> ), indicating the wide potential for using this gene to improve the salt tolerance of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, three sets of NILs differing at <italic> <span class="gene">GmSALT3</span> </italic> locus were developed through marker-assisted selection, and used to evaluate the possible effect of <italic> <span class="gene">GmSALT3</span> </italic> on ion accumulation and <span class="trait">yield</span> production. Our results clearly indicate that <italic> <span class="gene">GmSALT3</span> </italic> alters both Na <sup> + </sup> and Cl <sup> - </sup> accumulation in shoots and mature pod walls of NIL-T lines. The salt tolerance gene <italic> <span class="gene">GmSALT3</span> </italic> was found to have no penalty on soybean <span class="trait">yield</span> under non-saline condition and contributes to improving soybean yields through increasing seed weight in different genetic backgrounds under salinity stress in the field ( <bold> Figures <xref ref-type="fig" rid="F6"> 6B,C </xref> </bold> ). The salinity tolerance of NIL-T lines was related with the maintenance of seed size under salt stress, with this ability associated, at least partially, with the ability to regulate Na <sup> + </sup> and Cl <sup> - </sup> in both vegetative and reproductive tissues. Interestingly we found that <span class="gene">GmSALT3</span> first limits Cl <sup> - </sup> accumulation in the leaf and then Na <sup> + </sup> , through a yet to be identified mechanism. This study provides useful molecular markers for introducing the <italic> <span class="gene">GmSALT3</span> </italic> gene into breeder&#x02019;s varieties; the NIL-T lines used in this experiment have the potential to be used as gene sources to accelerate breeding for improvement of salt tolerance in commercially grown soybean varieties.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Comparative analysis of soybean genotype <span class="trait">resistance</span> to Heterodera glycines and Meloidogyne species via <span class="trait">resistance</span> gene analogs.</h2>
			<h3>PMID:27706689  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vieira PM(1), Arêdes FA(2), Ferreira A(3), Ferreira MF(4).
				<br/> 
				年份：2016 Sep 2
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nematodes are important pests of soybean throughout the world and cause high <span class="trait">yield</span> losses. As a control strategy, the identification of <span class="trait">resistance</span> genes is an important aim of breeding studies. Plants possess <span class="trait">resistance</span> genes (R), which are responsible for the recognition of pathogens and activation of the defense system. R genes and <span class="trait">resistance</span> gene analogs (RGAs) possess conserved domains, from which nucleotide-binding site is the most common. Using degenerate primers originating from these domains, it is possible to identify and isolate sequences of R and RGA genes. In this study, soybean genotypes resistant to the nematodes Heterodera glycines, Meloidogyne incognita, M. javanica, and M. enterolobii were compared by the use of RGAs and simple sequence repeat (SSR) markers. Forty-six soybean genotypes were studied, including plant introductions (PIs), commercial crops, and source of <span class="trait">resistance</span> genotypes. Thirteen combinations of RGA primers and different SSRs linked to QTLs were used to confirm <span class="trait">resistance</span> to soybean cyst nematodes (SCN). Fragments associated with <span class="trait">resistance</span> to the studied nematodes were amplified in the source of <span class="trait">resistance</span> and PI genotypes. RGA markers were efficient at distinguishing groups of genotypes that were resistant and susceptible to Meloidogyne <span class="gene">spp</span> and SCN. Combinations of specific primers were identified through their ability to amplify nucleotide sequences from possible <span class="trait">resistance</span> candidate genes. SSR markers contributed to the analysis of SCN race specificity, showing that the QTLs identified by these markers are distinct from those identified by RGA markers.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Vernonia Diacylglycerol Acyltransferase Can Increase Renewable Oil Production.</h2>
			<h3>PMID:27578203  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hatanaka T(1), Serson W(2), Li R(3), Armstrong P(4), Yu K(2), Pfeiffer T(2), Li XL(2), Hildebrand D(2).
				<br/> 
				年份：2016 Sep 28
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Increasing the production of plant oils such as soybean oil as a renewable resource for food and fuel is valuable. Successful breeding for higher oil levels in soybean, however, usually results in reduced protein, a second valuable seed component. This study shows that by manipulating a highly active acyl-CoA:diacylglycerol acyltransferase (DGAT) the hydrocarbon flux to oil in oilseeds can be increased without reducing the protein component. Compared to other plant DGATs, a <span class="gene">DGAT</span> from Vernonia galamensis (VgDGAT1A) produces much higher oil synthesis and accumulation activity in yeast, insect cells, and soybean. Soybean lines expressing VgDGAT1A show a 4% increase in oil content without reductions in seed protein contents or <span class="trait">yield</span> per unit land area. Incorporation of this trait into 50% of soybeans worldwide could result in an increase of 850 million kg oil/year without new land use or inputs and be worth ∼U.S.$1 billion/year at 2012 production and market prices.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Clustering of soybean genotypes via Ward-MLM and ANNs associated with mixed models.</h2>
			<h3>PMID:27525912  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Teodoro PE(1), Torres FE(2), Corrêa AM(2).
				<br/> 
				年份：2016 Aug 5
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The objectives of this study were to use mixed models to confirm the presence of genetic variability in 16 soybean genotypes, to compare clusters generated by artificial neural networks (ANNs) with those created by the Ward modified location model (MLM) technique, and to indicate parental combinations that hold promise for obtaining superior segregating populations of soybean. A field trial was conducted between November 2014 and February 2015 at Universidade Estadual de Mato Grosso do Sul, Aquidauana, MS. The experimental design consisted of four replications of randomized blocks, each containing 16 treatments. We assessed the following agronomic traits: plant height, first pod height, number of branches per plant, number of pods per plant, number of grains per pod, hundred-grain weight, and grain <span class="trait">yield.</span> Mixed models were used to estimate variance components and genetic parameters, and obtain genotypic values for each trait. After verifying the presence of genetic variability for all traits, genotypic values were submitted to both a Ward-MLM procedure and ANNs to estimate genetic divergence among genotypes. The number of groups formed by both methods was the same, but there were differences in group constitutions. <span class="gene">ANN</span> analysis improved soybean genotypes clustering patterns compared to Ward-MLM procedure. Based on these methods, divergent crosses may be made between genotype 97R73 with genotypes AS3797 and SYN9070, whereas convergent crosses may be made between genotypes AS3797 and SYN9070.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional characterization of <italic> <span class="gene">GmBZL2</span> </italic> ( <italic> AtBZR1 </italic> like gene) reveals the conserved BR signaling regulation in <italic> Glycine max </italic></h2>
			<h3>PMID:27498784</h3>
				

				
					<h3>PMCID:4976319</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Yu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Yan-Jie </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Bao-Jun </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Xian-Xian </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Dun </given-names>  </name>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zu </surname>  <given-names> Song-Hao </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xue </surname>  <given-names> Hong-Wei </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Wen-Hui </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a3"> 3 </xref>  <xref ref-type="aff" rid="a6"> 6 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> State Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences </institution> , Beijing,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> University of Chinese Academy of Sciences </institution> , Beijing,  <country> China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> School of Life Sciences and Biotechnology, Shanghai Jiao Tong University </institution> , Shanghai,  <country> China </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> National Key Laboratory of Plant Molecular Genetics, Institute of Plant Physiology &#x00026; Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences </institution> , Shanghai,  <country> China </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> State Key Laboratory of Systematic and Evolutionary Botany, Institute of Botany, Chinese Academy of Sciences </institution> , Beijing,  <country> China </country>  </aff>  <aff id="a6">  <label> 6 </label>  <institution> Department of Plant and Microbial Biology, University of California </institution> , Berkeley, CA,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 07 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Brassinosteroids (BRs) play key roles in plant growth and development, and regulate various agricultural traits. Enhanced BR signaling leads to increased seed number and <span class="trait">yield</span> in Arabidopsis bzr1-1D (AtBZR1(P234L), gain-of-function mutant of the important transcription factor in BR signaling/effects). BR signal transduction pathway is well elucidated in Arabidopsis but less known in other species. Soybean is an important dicot crop producing edible oil and protein. Phylogenetic analysis reveals AtBZR1-like genes are highly conserved in angiosperm and there are 4 orthologues in soybean <span class="gene">(GmBZL1-4).</span> We here report the functional characterization of <span class="gene">GmBZL2</span> (relatively highly expresses in flowers). The P234 site in AtBZR1 is conserved in <span class="gene">GmBZL2</span> (P216) and mutation of <span class="gene">GmBZL2(P216L)</span> leads to <span class="gene">GmBZL2</span> accumulation. <span class="gene">GmBZL2(P216L)</span> <span class="gene">(GmBZL2*)</span> in Arabidopsis results in enhanced BR signaling; including increased seed number per silique. <span class="gene">GmBZL2*</span> partially rescued the defects of bri1-5, further demonstrating the conserved function of <span class="gene">GmBZL2</span> with AtBZR1. BR treatment promotes the accumulation, nuclear localization and dephosphorylation/phosphorylation ratio of <span class="gene">GmBZL2,</span> revealing that <span class="gene">GmBZL2</span> activity is regulated conservatively by BR signaling. Our studies not only indicate the conserved regulatory mechanism of <span class="gene">GmBZL2</span> and BR signaling pathway in soybean, but also suggest the potential application of <span class="gene">GmBZL2</span> in soybean seed <span class="trait">yield.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this study, using BLASTP tools to search soybean proteome sequences with the full-lengh AtBZR1 protein sequence we identified four orthologues of <italic> AtBZR1 </italic> in soybean, which named <italic> <span class="gene">GmBZL1-4</span> </italic> ( <italic> Glycine max AtBZR1 like gene 1-4 </italic> ) and focused on <italic> <span class="gene">GmBZL2</span> </italic> . We overexpressed <italic> <span class="gene">GmBZL2</span> </italic> by point mutation conserved with <italic> AtBZR1 </italic> in Arabidopsis wild type Col-0 and BR mutant to investigate <italic> <span class="gene">GmBZL2</span> </italic> functions in plant growth and development, as well as seed number regulation. The results demonstrate conservation of BR signaling regulation in dicot species, which gives potential application to increase seed <span class="trait">yield</span> of dicot crops.
					</p>
				
					<p>&nbsp;&nbsp;
						Compared to other three homologous genes on Phytozome ( <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" ext-link-type="uri" xlink:href="https://phytozome.jgi.doe.gov/pz/portal.html"> https://phytozome.jgi.doe.gov/pz/portal.html </ext-link> ), we found that <italic> <span class="gene">GmBZL2</span> </italic> ( <italic> <span class="gene">Glyma14g08320.1</span> </italic> ) had a relatively high expression level in the flowers ( <xref ref-type="supplementary-material" rid="S1"> Supplementary Fig. S2 </xref> ). We speculated <italic> <span class="gene">GmBZL2</span> </italic> might have a function in the reproductive development and seed <span class="trait">yield</span> control. We cloned full length of <italic> <span class="gene">GmBZL2</span> </italic> from genomic DNA of Zhongdou 32 cultivar, as well as full CDS of <italic> <span class="gene">GmBZL2</span> </italic> from flower cDNA of Zhongdou 32. We found the gene structure is as same as the predicted structure. In the full genomic sequence (1931&#x02009;bp), <italic> <span class="gene">GmBZL2</span> </italic> contains two exons (1-240, 1236-1931) and one intron (241-1235) ( <xref ref-type="fig" rid="f2"> Fig. 2a </xref> ). <span class="gene">GmBZL2</span> shows 64.99% amino acid sequence similarity compared with AtBZR1. It also contains a conserved PEST domain (region rich in proline, glutamate, serine, and threonine) and the important proline residue (P216, conserved with P234 in AtBZR1) ( <xref ref-type="fig" rid="f2"> Fig. 2b,c </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The average seed number per silique is around 50 in Arabidopsis Col-0 ecotype. Seed number per silique determined by ovule number, male <span class="trait">fertility,</span> fertilization and zygote development. As the precursor of seed, ovule number determines the maximal possibility of seed number per silique. Our publication demonstrated that BR positively regulated ovule number by influencing ovule early development related gene expression by AtBZR1 <xref ref-type="bibr" rid="b30"> 30 </xref> . In this work, we also illustrated that <span class="gene">GmBZL2*</span> increased BR signaling and seed number per silique in Arabidopsis. We could hypothesize that <span class="gene">GmBZL2*</span> can increase seed number per pod in soybean, as similar as it functioned in Arabidopsis, since the enhanced <span class="gene">GmBZL2</span> activity promoted BR signaling. Also similar to Arabidopsis, the BR signaling cannot be too high since it will induce the feedback inhibition of BR biosynthesis and lead to BR deficiency and reduced plant growth and seed <span class="trait">yield.</span> How to control BR signaling in suitable level is a challenge.
					</p>
				
					<p>&nbsp;&nbsp;
						Although ovule grows up from parietal placenta (development from carpel margin meristem) in Arabidopsis and from central placenta in soybean; the possible BR regulation might be similar. Soybean has hundreds of pods and the average seed number per pod is around 3 (valued from different cultivars), suggesting the increase of seed number per pod would contribute enormously in soybean seed <span class="trait">yield.</span> How AtBZR1 and <span class="gene">GmBZL2</span> work in different placenta pattern and play similar role in ovule identity and initiation to regulate seed set is worth investigating in future.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A Genetic Relationship between Phosphorus Efficiency and Photosynthetic Traits in Soybean As Revealed by QTL Analysis Using a High-Density Genetic Map</h2>
			<h3>PMID:27446154</h3>
				

				
					<h3>PMCID:4923142</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Hongyan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Yuming </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hengyou </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/334878/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chu </surname>  <given-names> Shanshan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/334879/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xingguo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yin </surname>  <given-names> Dongmei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Deyue </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Dan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/289272/overview"/>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant productivity relies on photosynthesis, and the photosynthetic process relies on phosphorus (P). The genetic basis of photosynthesis and P efficiency (PE) affecting <span class="trait">yield</span> has been separately characterized in various crop plants. However, the genetic relationship between PE and photosynthesis remains to be elucidated. In this study, we used a combined analysis of phenotypic correlation, linkage mapping, and expression analysis to dissect the relationship between PE and photosynthesis. We found significant phenotypic correlations between PE and photosynthetic related traits, particularly under low P stress. A total of 172 QTLs for both traits were detected and classified into 29 genomic regions. 12 (41.4%) of 29 regions were detected to be associated with both PE and photosynthetic related traits. Three major QTLs, q14-2, q15-2, and q19-2, were found to be associated with both traits and explained 6.6-58.9% of phenotypic variation. A photosynthetic-specific QTL cluster, q12-1, was detected under both normal and low P conditions, suggesting that genes responsible for this region were less effected by low P stress, and could be used in high photosynthetic efficiency breeding programs. In addition, several candidate genes with significantly differential expression upon low P stress, such as a purple acid phosphatase gene <span class="gene">(Glyma.19G193900)</span> within q19-2 region, were considered as promising candidates involved in regulating both soybean PE and photosynthetic capacity. Our results reveal a significant genetic relationship between PE and photosynthetic traits, and uncover several major genomic regions specific or common to these traits. The markers linked closely to these major QTLs may be used for selection of soybean varieties with improved P efficiency and photosynthetic capacity.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is an important crop providing the primary source of plant protein and vegetable oil for human and livestock consumption. Soybean is also a high P demand species but sensitive to low P stress. Low P stress has been the largest constraint affecting soybean production compared to other nutrient deficiencies, toxicities, or diseases (Gowin, <xref rid="B19" ref-type="bibr"> 1997 </xref> ). In recent years, in order to understand the genetic basis of soybean P efficiency, great efforts have been made in evaluating P-efficient soybean genotypes, and applying linkage/association analyses to map genes underlying or markers linked to soybean P efficiency. Significant genetic variation in the soybean germplasms suggests that the selection of better P-efficient varieties can be achieved by breeding processes (Li et al., <xref rid="B26" ref-type="bibr"> 2005 </xref> ; Zhang et al., <xref rid="B48" ref-type="bibr"> 2009 </xref> , <xref rid="B52" ref-type="bibr"> 2014 </xref> ; Liang et al., <xref rid="B27" ref-type="bibr"> 2010 </xref> ). In our recent study, a key QTL gene, <italic> <span class="gene">GmACP1</span> </italic> encoding an acid phosphatase, underlying P efficiency was identified (Zhang et al., <xref rid="B52" ref-type="bibr"> 2014 </xref> ). Overexpression of <italic> <span class="gene">GmACP1</span> </italic> in soybean plants significantly increased P efficiency by 11&#x02013;20% compared with controls. However, the <span class="trait">yield</span> of these transgenic plants was not significantly increased, which was mainly attributed to the photosynthetic capacity is not prominent, while photosynthesis is the basis of crop growth and <span class="trait">yield</span> formation. Thus, it is very important to evaluate soybean photosynthetic capacity when in cultivating P-efficient soybean varieties.
					</p>
				
					<p>&nbsp;&nbsp;
						An important photosynthetic-specific QTL, <italic> q12-1 </italic> , were detected on chromosome 12 under both NP and LP conditions, suggesting that this QTL's expression is less affected by low P stress. Thus, <italic> q12-1 </italic> could be a promising candidate locus for further high photosynthetic efficiency study. Development of near isogenic lines based on this region could be the best approach to fine map this QTL without the interference of other P efficiency genes. In addition, the role of Ribulose-bisphosphate in photosynthesis have been previously identified (Yin et al., <xref rid="B47" ref-type="bibr"> 2010b </xref> ), thus ( <italic> <span class="gene">Glyma.12G061600</span> </italic> ) could be one of the candidate genes underlying this loci. Correlation of Ribulose-bisphosphate gene expression levels with photosynthetic traits indicates that RCA genes could play an important role in regulating soybean photosynthetic capacity and seed <span class="trait">yield</span> (Yin et al., <xref rid="B47" ref-type="bibr"> 2010b </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast to <italic> q12-1 </italic> , the LP specific-QTLs ( <italic> q18-1 </italic> ) on chromosome 18 were detected for photosynthetic related traits only under LP conditions, suggesting that this QTL's expression may be inducible by low P stress. When QTLs for PC and Pup was added to the analysis as covariates, the LOD and additive effect of <italic> q18-1 </italic> were all decreased (data not shown), suggesting that these photosynthetic traits were P dependent. On the other hand, it may be a crucial response mechanism of tolerance to low P stress by slowing photosynthesis. Under low P stress, reduction of chlorophyll content thus slowed the photosynthesis rate and transpiration rate, which may be beneficial for plants to escape from stress at the expense of reducing <span class="trait">yield</span> potential. In addition, in this region, it is noteworthy that the <italic> q18-1 </italic> underlying gene, <italic> <span class="gene">Glyma.18G036400</span> </italic> , which encoding a RCA gene was corresponded to the photosynthetic key gene ( <italic> <span class="gene">GmRCA</span> </italic> &#x003b2;), with the largest average LOD score (8.4) and explain 7.8&#x02013;39.3% of the phenotypic variance. Correlation of gene expression levels with photosynthetic related traits indicates that <italic> <span class="gene">GmRCA</span> </italic> &#x003b2; gene could play an important role in regulating soybean photosynthetic capacity and seed <span class="trait">yield</span> (Yin et al., <xref rid="B47" ref-type="bibr"> 2010b </xref> ). Further study showed that regulating expression levels of Rubisco activase ( <italic> <span class="gene">GmRCA</span> </italic> &#x003b2;) gene to optimize Rubisco activation may provide an approach to enhance plant productivity (Chao et al., <xref rid="B6" ref-type="bibr"> 2014 </xref> ). Thus, co-localization of <italic> <span class="gene">GmRCA</span> </italic> &#x003b2; with <italic> q18-1 </italic> and the narrowed QTL regions provided a strong evidence showing the high accuracy of the mapping strategy in the present study. Deep dissection of these major QTLs and promising candidate genes for both traits could provide greater understanding of the genetic basis of PE and photosynthetic traits. Although the promising genes were provided within these loci, more studies such as overexpression or knock-out of these genes are needed to uncover the molecular genetic mechanism underlying PE and photosynthesis.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Introgression of null allele of Kunitz trypsin inhibitor through marker-assisted backcross breeding in soybean ( <italic> Glycine max </italic> L. Merr.)</h2>
			<h3>PMID:27407019</h3>
				

				
					<h3>PMCID:4943013</h3>
					<h4>期刊:BMC Genetics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Maranna </surname>  <given-names> Shivakumar </given-names>  </name>  <address>  <email> shivaiari9683@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Verma </surname>  <given-names> Khushbu </given-names>  </name>  <address>  <email> khushbu.9biotech@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Talukdar </surname>  <given-names> Akshay </given-names>  </name>  <address>  <phone> +91 11 25841481 </phone>  <email> atalukdar@iari.res.in </email>  <email> akshay.talukdar1@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lal </surname>  <given-names> Sanjay Kumar </given-names>  </name>  <address>  <email> sklal68@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kumar </surname>  <given-names> Anil </given-names>  </name>  <address>  <email> akanu90@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mukherjee </surname>  <given-names> Keya </given-names>  </name>  <address>  <email> jhum4u2006@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <aff id="Aff1">  <label/> ICAR- Indian Institute of Soybean Research, Indore, MP India  </aff>  <aff id="Aff2">  <label/> Division of Genetics, ICAR-Indian Agricultural Research Institute, New Delhi, 110012 India  </aff>
				<br/> 
				年份：<day> 6 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Presence of Kunitz trypsin inhibitor (KTI) in soybean seeds necessitates pre-heat treatment of the soy-flour for its inactivation before using it in food and feed products. The heat treatment not only enhances processing costs of the soy-based foods and feeds but also affects seed-protein quality and solubility. Genetic elimination of <span class="gene">KTI</span> is an important and effective approach. Therefore, molecular marker-assisted backcross breeding (MABB) approach was adopted for genetic elimination of <span class="gene">KTI</span> from two popular soybean genotypes, DS9712 and DS9814. PI542044, an exotic germplasm line was used as donor of the kti allele which inhibits functional <span class="gene">KTI</span> peptide production. RESULTS: Foreground selection for the kti allele was performed with three closely linked SSR markers while background selection was done with 93 polymorphic SSR markers. Plants in the BC1F1 generation were found to recover 70.4-87.63 % and 60.26-73.78 % of the recurrent parent genome (RPG) of DS9712 and DS9814, respectively. Similarly, selected plants in the BC2F1 generation had 93.01-98.92 % and 83.3-91.67 % recovery of their respective RPGs. Recombinant selection was performed so as to identify plants with minimal linkage drag. Biochemical analysis of the seeds of the selected plants (ktikti) confirmed absence of <span class="gene">KTI</span> peptides in the seeds. Phenotypically, the selected plants were comparable to the respective recurrent parent in <span class="trait">yield</span> and other traits. CONCLUSIONS: MABB approach helped in speedy development of 6 <span class="gene">KTI</span> free breeding lines of soybean. Such lines will be suitable for the farmers and the soybean industries to use in production of soy-based foods and feeds without pre-heat treatment of the soy-flour. It would contribute towards wider acceptability of soy-based foods and feeds.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Foreground selection for the <italic> kti </italic> allele was performed with three closely linked SSR markers while background selection was done with 93 polymorphic SSR markers. Plants in the BC <sub> 1 </sub> F <sub> 1 </sub> generation were found to recover 70.4&#x02013;87.63&#x000a0;% and 60.26&#x02013;73.78&#x000a0;% of the recurrent parent genome (RPG) of DS9712 and DS9814, respectively. Similarly, selected plants in the BC <sub> 2 </sub> F <sub> 1 </sub> generation had 93.01&#x02013;98.92&#x000a0;% and 83.3&#x02013;91.67&#x000a0;% recovery of their respective RPGs. Recombinant selection was performed so as to identify plants with minimal linkage drag. Biochemical analysis of the seeds of the selected plants ( <italic> ktikti </italic> ) confirmed absence of <span class="gene">KTI</span> peptides in the seeds. Phenotypically, the selected plants were comparable to the respective recurrent parent in <span class="trait">yield</span> and other traits.
					</p>
				
					<p>&nbsp;&nbsp;
						Two popular soybean varieties of North Plain Zone of India viz., DS9712 and DS9814 were chosen as recurrent parents. Both the varieties are high yielding with high degree of <span class="trait">resistance</span> to yellow mosaic virus disease (YMD), but contains higher level of <span class="gene">KTI</span> (83.37 and 123.96&#x000a0;mg&#x000a0;g <sup> &#x02212;1 </sup> seed meal, respectively), and takes longer duration to mature (125&#x02013;130 days). Therefore, elimination of <span class="gene">KTI</span> coupled with reduction of maturity duration would make DS9712 and DS9814 varieties more popular among the end users.
					</p>
				
					<p>&nbsp;&nbsp;
						The BC <sub> 2 </sub> F <sub> 2 </sub> families of DS9712 and DS9814 containing the null allele of <span class="gene">KTI</span> were raised along with the recurrent parents in the fields of ICAR-Indian Agricultural Research Institute, New Delhi (latitude: 28&#x000b0; 40&#x02019;N; longitude: 77&#x000b0; 13&#x02019; E; altitude: 228&#x000a0;m above mean sea level) during July-October 2013 for assessing the <span class="trait">yield</span> and other attributes. Temperature during crop growing period ranged from 25&#x02013;40&#x000a0;&#x000b0;C with occasional ups and down. Humidity level ranged from 60&#x02013;80&#x000a0;% with occasional changes caused by monsoon rain. Photoperiod ranged from 10&#x02013;11&#x000a0;h during sowing to 7&#x02013;8&#x000a0;h during harvesting period. Each introgressed line was planted in a single row of 3&#x000a0;m length with 45&#x000a0;cm row-to-row and 10&#x000a0;cm plant-to-plant distance. All the recommended agronomic practices were practiced to raise a healthy crop. The observations on days to flowering, plant height (cm), number of pods per plant and seed <span class="trait">yield</span> per plant were recorded on each family separately. These parameters were used as additional parameter in identifying introgressed lines resembling the recurrent parents.
					</p>
				
					<p>&nbsp;&nbsp;
						The BC <sub> 2 </sub> F <sub> 2 </sub> families introgressed with <italic> kti </italic> allele were evaluated during 2012 (July-October) in the experimental field of IARI, New Delhi-12 (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). The standard package of practices was followed to raise a good crop. Observations were recorded from each individual of the families on morpho- phenotypic traits. Nearly all the families flowered one week earlier (37&#x000a0;days) than the recurrent parents in both the crosses&#x000a0;(Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S5). The <span class="gene">KTI</span> free BC <sub> 2 </sub> F <sub> 2 </sub> plants which were confirmed by native PAGE were found to <span class="trait">yield</span> higher than the recurrent parents (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ). <fig id="Fig3"> <label> Fig. 3 </label> <caption> <p> Evaluation of introgressed BC <sub> 2 </sub> F <sub> 2 </sub> families under field condition. <bold> a </bold> : Recurrent parent (DS9712), ( <bold> b </bold> ) and ( <bold> c </bold> ): <span class="gene">KTI</span> free BC <sub> 2 </sub> F <sub> 2 </sub> families
					</p>
				
					<p>&nbsp;&nbsp;
						The resultant plants of MABB should not only have the desired trait but also to match the recurrent parent in performance in <span class="trait">yield</span> and other phenotypic traits. In this experiment, the homozygous plants selected in progeny rows were compared for their phenotypic performances including per plant <span class="trait">yield.</span> A total of 38 and 15 BC <sub> 2 </sub> F <sub> 2 </sub> families respectively from DS9712 and DS9814 populations were grown under field conditions to evaluate for their agronomic performance. The results indicated that four families along with six <span class="gene">KTI</span> free plants recorded higher <span class="trait">yield</span> than recurrent parent. Moreover, they were about one week early in maturity as compared to the respective recurrent parents. This would make the genotypes suitable for growing in rice-wheat cropping sequence easily.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The differential expression of soybean [Glycine max (L.) Merrill] WRKY genes in response to water deficit.</h2>
			<h3>PMID:27343875  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dias LP(1), de Oliveira-Busatto LA(2), Bodanese-Zanettini MH(3).
				<br/> 
				年份：2016 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is today, and perhaps even more in the future, the main challenge for grain crops, resulting in a drastic <span class="trait">yield</span> reduction. Thus, it is of great interest to obtain soybean genotypes tolerant to water deficit. The <span class="trait">drought</span> tolerance trait is difficult to obtain through classical breeding due to its polygenic basis. In this context, genetic engineering is presented as a way to achieve this attribute. The ability to modulate the expression of many genes placed the transcription factors as promising biotechnological targets to develop stress tolerant cultivars. The WRKY proteins form a large family of transcription factors that are involved in important physiological and biochemical processes in plants, including the response to water deficit. In this study, the expression pattern determined by qPCR showed that, <span class="gene">GmWRKY6,</span> <span class="gene">GmWRKY46,</span> <span class="gene">GmWRKY56,</span> <span class="gene">GmWRKY106</span> and <span class="gene">GmWRKY149</span> genes are differentially expressed between a <span class="trait">drought</span> tolerant and a susceptible soybean genotype in water stress conditions. The in silico promoter and coexpression analysis indicate that these genes act in a stress physiological background.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Black soldier fly as dietary protein source for broiler quails: apparent digestibility, excreta microbial load, feed choice, performance, carcass and meat traits.</h2>
			<h3>PMID:27339654  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cullere M(1), Tasoniero G(1), Giaccone V(1), Miotti-Scapin R(1), Claeys E(2), De Smet S(2), Dalle Zotte A(1).
				<br/> 
				年份：2016 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In order to expand with validated scientific data the limited knowledge regarding the potential application of insects as innovative feed ingredients for poultry, the present study tested a partial substitution of soya bean meal and soya bean oil with defatted black soldier fly (Hermetia illucens) larvae meal (H) in the diet for growing broiler quails (Coturnix coturnix japonica) on growth performance, mortality, nutrients apparent digestibility, microbiological composition of excreta, feed choice, carcass and meat traits. With this purpose, a total of 450 10-day-old birds were allocated to 15 cages (30 birds/cage) and received three dietary treatments: a Control diet (C) and two diets (H1 and H2) corresponding to 10% and 15% H inclusion levels, respectively (H substituted 28.4% soya bean oil and 16.1% soya bean meal for H1, and 100% soya bean oil and 24.8% soya bean meal for H2, respectively). At 28 days of age, quails were slaughtered, carcasses were weighed, breast muscles were then excised from 50 quails/treatment, weighed, and ultimate pH (pHu) and L*, a*, b* colour values were measured. Breast muscles were then cooked to assess cooking loss and meat toughness. For the digestibility trial, a total of 15 28-day-old quails were assigned to the three feeding groups. The excreta samples were subjected to chemical and microbiological analysis. The same 15 quails were then simultaneously provided with C and H2 diets for a 10-day feed choice trial. Productive performance, mortality and carcass traits were in line with commercial standards and similar in all experimental groups. With the exception of ether extract digestibility, which was lower in <span class="gene">H1</span> group compared with C and H2 (P=0.0001), apparent digestibility of dry matter, CP, starch and energy did not differ among treatments. Microbial composition of excreta was also comparable among the three groups. Feed choice trial showed that quails did not express a preference toward C or H2 diets. Breast meat weight and <span class="trait">yield</span> did not differ among C, <span class="gene">H1</span> and H2 quails. Differently, the inclusion of H meal reduced meat pHu compared with C. In conclusion, this study demonstrated that H. illucens larvae meal can partially replace conventional soya bean meal and soya bean oil in the diet for growing broiler quails, thus confirming to be a promising insect protein source for the feed industry. Further research to assess the impact of H meal on intestinal morphology as well as on meat quality and sensory profile would be of utmost importance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Co-infection of a hypovirulent isolate of Sclerotinia sclerotiorum with a new botybirnavirus and a strain of a mitovirus.</h2>
			<h3>PMID:27267756  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ran H(1), Liu L(1), Li B(1), Cheng J(1), Fu Y(1), Jiang D(1), Xie J(2).
				<br/> 
				年份：2016 Jun 6
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Sclerotinia sclerotiorum, a notorious plant fungal pathogen, causes <span class="trait">yield</span> loss of many crops and vegetables, and is a natural host of a diverse viruses with positive-sense RNA (+ssRNA), negative-sense RNA (-ssRNA), double-stranded RNA (dsRNA), or DNA genomes. Mixed-infection with multiple related or unrelated mycoviruses is a common phenomenon in S. sclerotiorum. However, a single strain co-infected with dsRNA and + ssRNA viruses has not been reported in S. sclerotiorum. RESULTS: We report two unrelated viruses, Sclerotinia sclerotiorum botybirnavirus 2 (SsBRV2) with a bipartite dsRNA genome and Sclerotinia sclerotiorum mitovirus 4 (SsMV4/AH16) with a + ssRNA genome, which were originally detected in a single hypovirulent strain AH16 of S. sclerotiorum. SsMV4/AH16 has a typical genome of mitovirus and is a strain of mitovirus SsMV4. The genome of SsBRV2 consists of two separated dsRNA segments. The large dsRNA segment is 6159 bp in length and only has a single open reading frame (ORF) encoding a putative 1868-aa <span class="gene">polyprotein</span> with a conserved RNA dependent RNA polymerase (RdRp) domain. The small dsRNA segment is 5872 bp in length and encodes a putative 1778-aa protein. Phylogenetic analysis using RdRp conserved domain sequences revealed that SsBRV2 is phylogenetically related to the previously reported three bipartite viruses SsBRV1, Botrytis porri RNA virus 1 (BpRV1), and soybean leaf-associated botybirnavirus 1 (SlaBRV1). Electron microscopy demonstrated that SsBRV2 forms rigid spherical virions with a diameter of approximately 40 nm in infected mycelia. The virion of SsBRV2 was successfully introduced into a virus-free strain, which provides conclusive evidence that SsBRV2 confers hypovirulence on phytopathogenic fungus S. sclerotiorum. CONCLUSIONS: A bisegmented dsRNA virus (SsBRV2/AH16) and a nonsegmented + ssRNA virus (SsMV4/AH16) were characterized in a hypovirulent strain AH16 of S. sclerotiorum. SsMV4/AH16 is a strain of a reported mitovirus, whereas SsBRV2 is a new botybirnavirus. SsBRV2 is the causal agent of hypovirulence on S. sclerotiorum. Our findings supplied a first evidence that a single S. sclerotiorum strain is co-infected by dsRNA and + ssRNA mycoviruses.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and functional characterization of the sulfate transporter gene <italic> <span class="gene">GmSULTR1;2b</span> </italic> in soybean</h2>
			<h3>PMID:27206527</h3>
				

				
					<h3>PMCID:4874011</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ding </surname>  <given-names> Yiqiong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xiaoqiong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zuo </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Wang </surname>  <given-names> Hui </given-names>  </name>  <address>  <phone> +86-25-84399527 </phone>  <fax> +86-25-84396410 </fax>  <email> wanghui0@njau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Yu </surname>  <given-names> Deyue </given-names>  </name>  <address>  <phone> +86-25-84396410 </phone>  <fax> +86-25-84396410 </fax>  <email> dyyu@njau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> National Key Laboratory of Crop Genetics and Germplasm Enhancement, National Center for Soybean Improvement, Nanjing Agricultural University, Weigang No. 1, Nanjing, 210095 China  </aff>
				<br/> 
				年份：<day> 5 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean is a major source of oil and protein in the human diet and in animal feed. However, as soybean is deficient in sulfur-containing amino acids, its nutritional value is limited. Increasing sulfate assimilation and utilization efficiency is a valuable approach to augment the concentration of sulfur-containing amino acids in soybean seeds, and sulfate transporters play important roles in both sulfate uptake and translocation within plants. RESULTS: In this study, we isolated and characterized a soybean sulfate transporter gene: <span class="gene">GmSULTR1;2b.</span> The gene was found to be specifically expressed in root tissues and induced by low-sulfur stress. In addition, <span class="gene">GmSULTR1;2b</span> expression in yeast could complement deficiency in the sulfate transporter genes SUL1 and SUL2. Under +S conditions, <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco plants accumulated higher levels of organic matter and exhibited enhanced biomass and seed weight compared to control plants. Under -S conditions, acclimation of <span class="gene">GmSULTR1;2b-overexpressing</span> plants was much better than control plants. <span class="gene">GmSULTR1;2b-overexpressing</span> tobacco seedlings showed better tolerance to <span class="trait">drought</span> stress than the control plants, but no significant difference was observed under salt stress. Transcriptome analysis revealed 515 genes with at least a 2-fold change in expression in the leaves of tobacco plants overexpressing <span class="gene">GmSULTR1;2b.</span> Of these, 227 gene annotations were classified into 12 functional categories associated with 123 relevant pathways, including biosynthesis and metabolism-related proteins, stress-related proteins, and transporters. CONCLUSIONS: The findings reported here indicate that the increased biomass and seed <span class="trait">yield</span> observed in transgenic tobacco plants could have resulted from greater nutrient uptake and transport capability as well as enhanced development and accumulation of organic matter. Taken together, our results indicate that <span class="gene">GmSULTR1;2b</span> plays an important role in sulfur uptake and could alter the sulfur status of plants. Our study suggests that overexpressing <span class="gene">GmSULTR1;2b</span> may enhance plant <span class="trait">yield</span> under +S conditions, reduce plant production loss under -S conditions, and improve plant tolerance to sulfur deficiency stress.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this study, we isolated and characterized a soybean sulfate transporter gene: <italic> <span class="gene">GmSULTR1;2b</span> </italic> . The gene was found to be specifically expressed in root tissues and induced by low-sulfur stress. In addition, <italic> <span class="gene">GmSULTR1;2b</span> </italic> expression in yeast could complement deficiency in the sulfate transporter genes <italic> SUL1 </italic> and <italic> SUL2 </italic> . Under +S conditions, <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants accumulated higher levels of organic matter and exhibited enhanced biomass and seed weight compared to control plants. Under -S conditions, acclimation of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants was much better than control plants. <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings showed better tolerance to <span class="trait">drought</span> stress than the control plants, but no significant difference was observed under salt stress. Transcriptome analysis revealed 515 genes with at least a 2-fold change in expression in the leaves of tobacco plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> . Of these, 227 gene annotations were classified into 12 functional categories associated with 123 relevant pathways, including biosynthesis and metabolism-related proteins, stress-related proteins, and transporters.
					</p>
				
					<p>&nbsp;&nbsp;
						The findings reported here indicate that the increased biomass and seed <span class="trait">yield</span> observed in transgenic tobacco plants could have resulted from greater nutrient uptake and transport capability as well as enhanced development and accumulation of organic matter. Taken together, our results indicate that <italic> <span class="gene">GmSULTR1;2b</span> </italic> plays an important role in sulfur uptake and could alter the sulfur status of plants. Our study suggests that overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> may enhance plant <span class="trait">yield</span> under +S conditions, reduce plant production loss under -S conditions, and improve plant tolerance to sulfur deficiency stress.
					</p>
				
					<p>&nbsp;&nbsp;
						To gain further insight into the function of <italic> <span class="gene">GmSULTR1;2b</span> </italic> , a construct containing the <italic> <span class="gene">GmSULTR1;2b</span> </italic> ORF driven by 35S promoter was transferred into tobacco plants; after PCR analysis (Additional file&#x000a0; <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S4)&#x000a0;and Southern blotting of all transformed plants, three independent lines, namely, S22, S26 and S29, with one copy of the <italic> <span class="gene">GmSULTR1;2b</span> </italic> gene, were selected for further study (data not shown). We found that the T <sub> 2 </sub> <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants grew better than the control plants (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). Moreover, the seed weights of lines of S22, S26 and S29 were significantly improved, with the average increasing by 11&#x000a0;% (Fig.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6a </xref> ). These findings indicate that <italic> <span class="gene">GmSULTR1;2b</span> </italic> overexpression could enhance seed <span class="trait">yield.</span> <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Phenotype of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants. ( <bold> a </bold> ) The <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing S22 line plant grown under normal conditions in soil is shown on the left and the control plant (CK) on the right. ( <bold> b </bold> ) The S26 line plant is shown on the left and the CK plant on the right. ( <bold> c </bold> ) The S29 line plant is shown on the left and the CK plant on the right
					</p>
				
					<p>&nbsp;&nbsp;
						Biomass, <span class="trait">yield</span> and content of sulfur-containing compounds in <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants grown under +S conditions. ( <bold> a </bold> ) Thousand seed weight of seeds from <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants and control plants grown in soil. ( <bold> b </bold> ) Dry weights of 2-month-old whole <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants and whole control plants grown under +S (1.5&#x000a0;mM MgSO <sub> 4 </sub> ) conditions. ( <bold> c </bold> ) Soluble protein content in the functional leaves of 2-month-old <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants and control plants grown under +S conditions. ( <bold> d </bold> ) Total thiol content in the functional leaves of 2-month-old <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants and control plants grown under +S conditions. ( <bold> e </bold> ) Cys content in the functional leaves of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants and control plants grown under +S conditions. ( <bold> f </bold> ) Total sulfur content in the functional leaves of 2-month-old <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants and control plants grown under +S conditions. ( <bold> g </bold> ) Sulfate ion content in whole roots of 2-month-old <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants and control plants grown under +S conditions. Three biological replicates were performed. Error bars represent the standard deviation. Asterisks (*) indicate significant differences at <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05 (Student&#x02019;s <italic> t </italic> -test), and double asterisks (**) indicate significant differences at <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.01 (Student&#x02019;s <italic> t </italic> -test). CK, tobacco plant transformed with the pMDC83 empty vector as a control; S22, S26 and S29, tobacco plants transformed with <italic> <span class="gene">GmSULTR1;2b</span> </italic> , FW fresh weight
					</p>
				
					<p>&nbsp;&nbsp;
						Under -S conditions, the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants showed <span class="trait">chlorosis</span> but not as severe as the controls, whereas the control plants displayed severe <span class="trait">chlorosis</span> symptoms after 2&#x000a0;weeks of growth (Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> ). Under sulfur deficiency stress, the tobacco plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> accumulated more biomass compared to the control plants, on average more than 18&#x000a0;% (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8a </xref> ). The chlorophyll content of the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants was significantly higher than that of the control tobacco plants (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8b </xref> ), though there were no significantly differences between the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing and control plants under +S conditions (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Figure S5 and Fig.&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> ). Moreover, under the -S condition, the soluble protein and total thiol contents of the leaves were also an average of 37 and 107&#x000a0;% greater in the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants compared to the control plants (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8c and d </xref> ), as was the SO <sub> 4 </sub> <sup> 2&#x02212; </sup> content of the roots (an average of more than 119&#x000a0;%) (Fig.&#x000a0; <xref rid="Fig8" ref-type="fig"> 8e </xref> ). The SO <sub> 4 </sub> <sup> 2&#x02212; </sup> content of the roots in the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants was not significantly changed under the -S condition compared to under the +S condition, whereas a substantial reduction in the control plants was observed (Figs.&#x000a0; <xref rid="Fig6" ref-type="fig"> 6g </xref> and <xref rid="Fig8" ref-type="fig"> 8e </xref> ). In addition, the differences under the -S condition were more obvious than under the +S condition, with the biomass and soluble protein and total thiol contents of the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants decreasing less than those of the control plants under the &#x02013;S condition compared to that under the +S condition. These findings indicated that under -S conditions, acclimation by the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants was much better than that of the control plants. All these results indicate that overexpression of <italic> <span class="gene">GmSULTR1;2b</span> </italic> enhanced plant <span class="trait">yield</span> under +S conditions, reduced plant production loss under -S conditions, and improved plant tolerance to sulfur deficiency stress. <fig id="Fig7"> <label> Fig. 7 </label> <caption> <p> Phenotype of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants under +S and -S conditions. ( <bold> a </bold> ) Plants of the line S22 grown under +S conditions (1.5&#x000a0;mM MgSO <sub> 4 </sub> ) for 2&#x000a0;weeks. ( <bold> b </bold> ) Plants of the line S26 grown under +S conditions for 2&#x000a0;weeks. ( <bold> c </bold> ) Plants of the line S29 grown under +S conditions for 2&#x000a0;weeks. ( <bold> d </bold> ) Control plants grown under +S conditions for 2&#x000a0;weeks. ( <bold> e </bold> ) Plants of the line S22 grown under -S conditions (0&#x000a0;mM MgSO <sub> 4 </sub> ) for 2&#x000a0;weeks. ( <bold> f </bold> ) Plants of the line S26 grown under -S conditions for 2&#x000a0;weeks. ( <bold> g </bold> ) Plants of the line S29 grown under -S conditions for 2&#x000a0;weeks. ( <bold> h </bold> ) Control plants grown under -S conditions for 2&#x000a0;weeks
					</p>
				
					<p>&nbsp;&nbsp;
						Many studies have shown that sulfur-containing compounds are associated with stress. To address whether an increased total thiol content could enhance stress tolerance, we also subjected <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants and control plants to <span class="trait">drought</span> stress (200&#x000a0;mM mannitol) and salt stress (200&#x000a0;mM NaCl) (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9a </xref> ). Under <span class="trait">drought</span> stress, it showed that the fresh weight of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings were average 132&#x000a0;% more than that of control seedlings (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9b </xref> ), the root length of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings were average 15&#x000a0;% longer than that of control seedlings (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9c </xref> ), and the number of lateral root of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings were average 67&#x000a0;% more than that of control seedlings (Fig.&#x000a0; <xref rid="Fig9" ref-type="fig"> 9d </xref> ). However, under salt stress, there were no significant difference in fresh weight, root length and the number of lateral root between <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings and control seedlings (Fig.&#x000a0; <xref rid="Fig10" ref-type="fig"> 10 </xref> ). These results indicated that the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings showed better tolerance to <span class="trait">drought</span> stress than the control plants, though no significant difference under salt stress was observed. <fig id="Fig9"> <label> Fig. 9 </label> <caption> <p> <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco plants grown under 200&#x000a0;mM mannitol treatment. ( <bold> a </bold> ) Phenotypes of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings grown under 200&#x000a0;mM mannitol treatment for 4&#x000a0;weeks. ( <bold> b </bold> ) Fresh weight of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings grown under 200&#x000a0;mM mannitol treatment. ( <bold> c </bold> ) Root length of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings grown under 200&#x000a0;mM mannitol treatment. ( <bold> d </bold> ) Number of lateral roots of <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing tobacco seedlings grown under 200&#x000a0;mM mannitol treatment. Ten biological replicates were performed. Error bars represent the standard deviation. Asterisks (*) indicate significant differences at <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.05 (Student&#x02019;s <italic> t </italic> -test), and double asterisks (**) indicate significant differences at <italic> P </italic> &#x02009;&#x0003c;&#x02009;0.01 (Student&#x02019;s <italic> t </italic> -test). CK, tobacco plant transformed with the pMDC83 empty vector as a control; S22, S26 and S29, tobacco plants transformed with <span class="gene">GmSULTR1;2b.</span> Scale bar&#x02009;=&#x02009;1&#x000a0;cm
					</p>
				
					<p>&nbsp;&nbsp;
						Altogether, overexpression of <italic> <span class="gene">GmSULTR1;2b</span> </italic> result in the genes categorized as biosynthesis- and metabolism-related proteins, transporters, photosynthesis-related proteins, and developmental process-related proteins were regulated. These genes may be the reason that overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> could enhance tobacco plant <span class="trait">yield</span> under +S conditions, reduce tobacco plant production loss under -S conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Soil sulfur deficiency is a widespread limiting factor in crop production [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ], and sulfur addition to the soil or foliar application is required to maintain or increase <span class="trait">yield.</span> Overall, an appropriate sulfur supply may help to maintain efficient photosynthesis and metabolism, enhance defense reactions and protein quality, and increase biomass production during periods of sulfur deficiency [ <xref ref-type="bibr" rid="CR39"> 39 </xref> &#x02013; <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. Genetic manipulation of sulfate transporters can improve sulfur utilization efficiency [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ], and one advantage of improving the sulfate transporter system of plants would be to reduce the need to add sulfur fertilizer because these plants would be able to take up sulfur more efficiently and/or would possess the ability to reduce the loss of available sulfur from the soil. To date, soybean has remained one of the difficult crops to transform; indeed, compared to other crops, such as rice and corn, the transformation frequency of soybean is rather low [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. Thus, it is very difficult for public research groups to transform genes into soybean, and researchers typically transform soybean genes into a model plant, such as tobacco or <italic> Arabidopsis </italic> for heterologous expression. Accordingly, we analyzed the function of <italic> <span class="gene">GmSULTR1;2b</span> </italic> through heterologous expression in tobacco plants and found that <italic> <span class="gene">GmSULTR1;2b-</span> </italic> overexpressing tobacco plants exhibited better growth and higher accumulation of biomass than the control plants. In addition, <span class="trait">chlorosis</span> in the control plants was more serious than in the plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> under conditions of sulfur deficiency, and the chlorophyll content increased compared to control plants under -S conditions. These results indicate that the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants were able to take up sulfur more efficiently. Thus, <italic> <span class="gene">GmSULTR1;2b</span> </italic> overexpression can improve the sulfur utilization efficiency and alter the sulfur status of transgenic plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Due to the limitations of gene chip analysis, not every gene changes have been detected. Moreover, the tobacco genome sequencing results have not been published. Our gene annotation information was not complete. But we still have some useful findings. In our study, many genes related to transporters, such as <italic> NRAMP </italic> and <italic> OPT </italic> , were found to be regulated in tobacco plants overexpressing <italic> <span class="gene">GmSULTR1;2b.</span> AtNRAMP3 </italic> and <italic> AtNRAMP4 </italic> influence metal accumulation (transporting manganese, iron, and zinc), are involved different processes, including <span class="trait">resistance</span> to bacterial pathogens, support early development, and are required for optimal photosynthesis and growth [ <xref ref-type="bibr" rid="CR47"> 47 </xref> &#x02013; <xref ref-type="bibr" rid="CR49"> 49 </xref> ]. <italic> OPT3 </italic> can rescue copper-deficient and manganese-deficient yeast mutants and plays a critical role in the maintenance of whole-plant iron homeostasis and iron nutrition in developing <italic> Arabidopsis </italic> seeds [ <xref ref-type="bibr" rid="CR50"> 50 </xref> , <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. These findings suggest that <italic> <span class="gene">GmSULTR1;2b</span> </italic> may regulate other types of transporters to accumulate more nutrition, resulting in enhanced biomass and seed weight in tobacco plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						SAHH catalyzes the reversible hydrolysis of S-adenosyl-L-homocysteine to L-homocysteine and adenosine and is important for cell growth and regulating gene expression [ <xref ref-type="bibr" rid="CR52"> 52 </xref> , <xref ref-type="bibr" rid="CR53"> 53 </xref> ]. Complete inactivation of SAHH activity in plants is embryonic lethal [ <xref ref-type="bibr" rid="CR54"> 54 </xref> ], and partial inactivation results in developmental malformations [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ]. However, there are potential beneficial outcomes of the controlled down-regulation of <italic> SAHH </italic> , including the suppression of viral replication and enhancement of <span class="trait">drought</span> stress tolerance [ <xref ref-type="bibr" rid="CR55"> 55 </xref> , <xref ref-type="bibr" rid="CR56"> 56 </xref> ]. In addition, down-regulation of <italic> SAHH1 </italic> expression in yeast leads to both SAH accumulation and impacts on cellular lipid homeostasis [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ]. In our study, the level of <italic> SAHH </italic> transcript was reduced in tobacco plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> compared to control tobacco plants. This result suggests that down-regulation of the <italic> SAHH </italic> gene may be a reason why the <italic> <span class="gene">GmSULTR1;2b</span> </italic> -overexpressing plants accumulated more sulfur-containing compounds (i.e., cysteine, thiol) than the control plants. In addition, the plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> exhibited enhanced <span class="trait">drought</span> stress tolerance. It has been reported that the application of sulfur fertilizer improves <span class="trait">drought</span> tolerance in plants [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ], and it is possible that plants may down-regulate SAHH in response to <italic> <span class="gene">GmSULTR1;2b</span> </italic> overexpression not only to accumulate more sulfur-containing compounds but also to enhance growth and tolerance to biotic and abiotic stresses.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we confirmed <span class="gene">GmSULTR1;2b</span> as a functional sulfate transporter that plays an important role in sulfate uptake and could alter the sulfur status of plants. We found that overexpression <italic> <span class="gene">GmSULTR1;2b</span> </italic> in tobacco plants could enhance <span class="trait">yield</span> under +S conditions, reduce production loss under -S conditions, and improve tolerance to sulfur deficiency stress. The mechanism by which increased biomass and seed <span class="trait">yield</span> in <span class="gene">GmSULTR1;2b-overexpressing</span> transgenic tobacco plants occurs could be due to greater nutrient uptake and transport capabilities as well as enhanced development and accumulation of organic matter, as indicated by microarray analysis.
					</p>
				
					<p>&nbsp;&nbsp;
						Tobacco ( <italic> Nicotianata bacum </italic> cv. SamSun) plants were grown in a growth room at 25&#x000a0;&#x000b0;C under a 16&#x000a0;h light/8&#x000a0;h dark photoperiod. <span class="trait">Drought</span> and salt stress treatments were performed as follows. Seeds of the T <sub> 2 </sub> generation of tobacco plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> and control plants were germinated on solid <span class="gene">MS</span> medium containing 50&#x000a0;mg&#x000a0;L <sup> &#x02212;1 </sup> hygromycin. After 10&#x000a0;days, the seedlings were transplanted onto solid <span class="gene">MS</span> medium containing 200&#x000a0;mM mannitol for osmotic treatment <span class="trait">(drought</span> stress) or 200&#x000a0;mM NaCl for salt treatment (salt stress) and maintained for 4&#x000a0;weeks. Ten biological replicates were used, and each one was measured for three repetitions. The sulfur deficiency treatment was performed as follows. T <sub> 2&#x02212; </sub> generation tobacco plants overexpressing <italic> <span class="gene">GmSULTR1;2b</span> </italic> and control plants were grown under normal conditions for 2&#x000a0;weeks and then cultured hydroponically in MGRL medium with 1.5&#x000a0;mM MgSO <sub> 4 </sub> (+S) or 0&#x000a0;mM MgSO <sub> 4 </sub> (&#x02212;S) for 2&#x000a0;weeks. The tobacco plants were then transferred for 1&#x000a0;month to vermiculite irrigating MGRL medium containing 1.5&#x000a0;mM MgSO <sub> 4 </sub> or 0&#x000a0;mM MgSO <sub> 4 </sub> . Three biological replicates were performed, and each one was measured for three repetitions.
					</p>
				
					<p>&nbsp;&nbsp;
						An expression vector was constructed using Gateway Technology with Clonase&#x02122; II Kit (Invitrogen, USA). The open reading frame (ORF) of <italic> <span class="gene">GmSULTR1;2b</span> </italic> was obtained by PCR using the specific primers attB1-clone-1;2b F and attB1-clone-1;2b R (Additional file <xref rid="MOESM5" ref-type="media"> 5 </xref> : Table S7). The amplified ORF containing the attB fragment was inserted into the expression vector pMDC83, which harbors a double CaMV 35S promoter and hygromycin <span class="trait">resistance</span> gene, resulting in 35S: <italic> <span class="gene">GmSULTR1;2b</span> </italic> . pMDC83 was digested with the restriction endonucleases <italic> Asc </italic> I and <italic> Spe </italic> I and then ligated using T4 DNA ligase, which resulted in an empty vector. The plasmid containing <italic> <span class="gene">GmSULTR1;2b</span> </italic> or empty vector (as a control) was introduced into <italic> Agrobacterium tumefaciens </italic> strain EHA105 using the freeze-thaw method, and <italic> N. tabacum </italic> was transformed by the leaf disk method. Transgenic tobacco plants containing <italic> <span class="gene">GmSULTR1;2b</span> </italic> or the empty vector were selected on 50&#x000a0;mg&#x000a0;L <sup> &#x02212;1 </sup> hygromycin and confirmed by PCR, RT-PCR and Southern blotting.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Comparative analysis of protocols for DNA extraction from soybean caterpillars.</h2>
			<h3>PMID:27173218  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Palma J(1), Valmorbida I(2), da Costa IF(3), Guedes JV(4).
				<br/> 
				年份：2016 Apr 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genomic DNA extraction is crucial for molecular research, including diagnostic and genome characterization of different organisms. The aim of this study was to comparatively analyze protocols of DNA extraction based on cell lysis by sarcosyl, cetyltrimethylammonium bromide, and sodium dodecyl sulfate, and to determine the most efficient method applicable to soybean caterpillars. DNA was extracted from specimens of Chrysodeixis includens and Spodoptera eridania using the aforementioned three methods. DNA quantification was performed using spectrophotometry and high molecular weight DNA ladders. The purity of the extracted DNA was determined by calculating the A260/A280 ratio. Cost and time for each DNA extraction method were estimated and analyzed statistically. The amount of DNA extracted by these three methods was sufficient for PCR amplification. The sarcosyl method yielded DNA of higher purity, because it generated a clearer pellet without viscosity, and yielded high quality amplification products of the <span class="gene">COI</span> gene I. The sarcosyl method showed lower cost per extraction and did not differ from the other methods with respect to preparation times. Cell lysis by sarcosyl represents the best method for DNA extraction in terms of <span class="trait">yield,</span> quality, and cost effectiveness.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Comparative Analysis of CBS Domain-Containing Proteins in Soybean ( <italic> Glycine max </italic> ) and the Primary Function of <italic> <span class="gene">GmCBS21</span> </italic> in Enhanced Tolerance to Low Nitrogen Stress</h2>
			<h3>PMID:27128900</h3>
				

				
					<h3>PMCID:4881446</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> Qingnan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00620"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shang </surname>  <given-names> Weijuan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00620"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Chanjuan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00620"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Haifeng </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00620"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Limiao </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00620"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Songli </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00620"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Shuilian </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaojuan </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xinan </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-17-00620"> 1 </xref>  <xref ref-type="aff" rid="af2-ijms-17-00620"> 2 </xref>  <xref rid="c1-ijms-17-00620" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitrogen is an important macronutrient required for plant growth, and is a limiting factor for crop productivity. Improving the nitrogen use efficiency (NUE) is therefore crucial. At present, the NUE mechanism is unclear and information on the genes associated with NUE in soybeans is lacking. cystathionine beta synthase (CBS) domain-containing proteins (CDCPs) may be implicated in abiotic stress tolerance in plants. We identified and classified a CBS domain-containing protein superfamily in soybean. A candidate gene for NUE, <span class="gene">GmCBS21,</span> was identified. <span class="gene">GmCBS21</span> gene characteristics, the temporal expression pattern of the <span class="gene">GmCBS21</span> gene, and the phenotype of <span class="gene">GmCBS21</span> overexpression in transgenic Arabidopsis thaliana under low nitrogen stress were analyzed. The phenotypes suggested that the transgenic Arabidopsis thaliana seedlings performed better under the nitrogen-deficient condition. <span class="gene">GmCBS21-overexpressing</span> transgenic plants exhibit higher low nitrogen stress tolerance than WT plants, and this suggests its role in low nitrogen stress tolerance in plants. We conclude that <span class="gene">GmCBS21</span> may serve as an excellent candidate for breeding crops with enhanced NUE and better <span class="trait">yield.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Under the low nitrogen condition, all transgenic plants produced high levels of mRNA for AlaAT, GDH, and AS. AlaAT catalyzes the reversible transfer of an amino group from glutamate to pyruvate to form 2-oxoglutarate and alanine. The regulation of <italic> AlaAT </italic> in several plant species has been studied in response to low oxygen stress light and nitrogen application, and it is involved in both carbon and nitrogen metabolism in plants [ <xref rid="B36-ijms-17-00620" ref-type="bibr"> 36 </xref> ]. Some studies discovered that overexpression of barley <italic> AlaAT </italic> in <italic> Brassica napus </italic> resulted in increased <span class="trait">yield</span> and biomass under N-limiting conditions compared with control plants [ <xref rid="B37-ijms-17-00620" ref-type="bibr"> 37 </xref> ]. Overexpression of AlaAT in transgenic rice plants with a tissue-specific promoter (OsAnt1) significantly increased biomass and grain <span class="trait">yield</span> [ <xref rid="B38-ijms-17-00620" ref-type="bibr"> 38 </xref> ]. Amino acids are catabolized by both glutamate GDH and transaminases. Ammonium is reassimilated by <span class="gene">GS</span> and, through the action of AS, is stored in asparagine (Asn). It has been reported that <italic> Arabidopsis </italic> plants deficient in GDH have shown decreased growth on reduced levels of C [ <xref rid="B39-ijms-17-00620" ref-type="bibr"> 39 </xref> ]. As previously reported, overexpression of AS with a CaMV35S promoter could enhance NUE in <italic> Arabidopsis </italic> plants; the transgenic plants had higher tolerance to low nitrogen when grown on plates and in the glasshouse [ <xref rid="B30-ijms-17-00620" ref-type="bibr"> 30 </xref> ]. Based on the above analyses, we speculate that overexpression of <italic> <span class="gene">GmCBS21</span> </italic> in transgenic plants activated the expression of <italic> AlaAT </italic> , <italic> GDH </italic> , and <italic> AS </italic> under low nitrogen conditions, thus affecting the low nitrogen tolerance of plants.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:High-Density Genetic Mapping Identifies New Major Loci for Tolerance to Low-Phosphorus Stress in Soybean</h2>
			<h3>PMID:27065041</h3>
				

				
					<h3>PMCID:4811872</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Dan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/289272/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Hongyan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jinshe </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Hengyou </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/334878/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Zhenbin </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/299381/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chu </surname>  <given-names> Shanshan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/334879/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lv </surname>  <given-names> Haiyan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Deyue </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 3 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphorus (P) is essential for all living cells and organisms, and low-P stress represents a major constraint on plant growth and <span class="trait">yield</span> worldwide. Soybean is an important economical resource of protein and oil for human and animals, and soybean is also a high-P demand species that is sensitive to low-P stress, which is considered a major constraint on soybean production. However, P efficiency is an important complex quantitative trait involving multiple genes, and the mechanisms underlying soybean P efficiency are largely unknown. Here, we reported the construction of a high-density genetic map using a specific-locus amplified fragment sequencing (SLAF-seq) strategy in soybean. This map, spanning 3020.59 cM in length, contained 6159 markers on 20 chromosomes, with an average distance of 0.49 cM between adjacent markers. Based on this map, 20 loci, including eight novel loci, associated with P efficiency-related traits were identified across multiple years and treatments. The confidence intervals of almost all QTLs were refined significantly, and the accuracy of this map was evidenced by coincident detections of the previously identified P efficiency-related genes <span class="gene">GmACP1</span> and <span class="gene">GmPT1.</span> Notably, a highly significant novel QTL located on chromosome 4, q4-2, was identified across traits, years and treatments. Several candidate genes, such as a pectin methylesterase-encoding gene <span class="gene">(Glyma.04G214000)</span> and a protein kinase gene <span class="gene">(Glyma.13G161900),</span> with significantly differential expression upon low-P stress were considered as promising candidates involved in regulating soybean P efficiency. Markers that tightly associated with P efficiency could be used for marker-assisted selection in a soybean P efficient breeding program. Further, dissection of these QTLs will facilitate gene cloning underlying P efficiency in soybean, and increase our understanding of efficient use of P in enhancing crop <span class="trait">yield.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Of the 20 identified loci, eight QTLs ( <italic> q1, q3, q4-1, q4-2, q7, q13-1, q13-2 </italic> , and <italic> q16 </italic> ) were not found in previous reports, representing novel loci for P efficiency. It is noted that <italic> q4-2 </italic> was associated with multiple P efficiency-related traits across various treatments. As shown in Table <xref ref-type="table" rid="T3"> 3 </xref> , Figure <xref ref-type="fig" rid="F2"> 2 </xref> , the LOD score and phenotypic variation of this locus reached a maximum of 8.57 and 28.10%, respectively. Therefore, the loci were identified as an importantly novel major QTL associated with P efficiency. The release of the soybean reference genome has facilitated the gene prediction for QTL associated with various traits. In this study, a pectin methylesterase gene ( <italic> <span class="gene">Glyma.04G214000</span> </italic> ) in this 0.36 cM region was regarded as a candidate, as it was proposed to be required for plant tolerance to various abiotic stress (An et al., <xref rid="B2" ref-type="bibr"> 2008 </xref> ), such as response to <span class="trait">drought</span> and oxidative stress (Legay et al., <xref rid="B25" ref-type="bibr"> 2011 </xref> ), salt stress (Walia et al., <xref rid="B41" ref-type="bibr"> 2005 </xref> ), heat stress (Kagan-Zur et al., <xref rid="B17" ref-type="bibr"> 1995 </xref> ), and cold stress etc. (Lee and Lee, <xref rid="B24" ref-type="bibr"> 2003 </xref> ). Moreover, previous studies have shown that the pectin methylesterase genes can have organ or stress-specific expression patterns (Pelloux et al., <xref rid="B34" ref-type="bibr"> 2007 </xref> ). Similarly, our transcriptome analysis results showed that the expression of <italic> <span class="gene">Glyma.04G214000</span> </italic> increased 1000-fold in soybean leaves under low-P conditions relative to the control (Zhang et al., <xref rid="B53" ref-type="bibr"> 2015 </xref> , in review). Thus, <italic> <span class="gene">Glyma.04G214000</span> </italic> was putatively thought to be related to P efficiency and will be further experimentally verified.
					</p>
				
					<p>&nbsp;&nbsp;
						The novel QTL, <italic> q13-2 </italic> , were detected only under &#x02212;P or &#x02212;/+P conditions on chromosome 13, suggesting that this QTL's expression may be induced expression by low-P stress. A comprehensive analysis reveals that the target QTL was located in a region of approximately 1.5 Mb between Marker1719580 and Marker1684017, and 146 annotated genes were predicted. Six annotated genes ( <italic> <span class="gene">Glyma.13G161800,</span> 161900, 162800, 167600 </italic> , and <italic> 172100 </italic> ) of 46 annotated genes within this QTL were considered candidates because of their relevance for P efficiency or plant stress. For example, one gene <italic> <span class="gene">Glyma.13G161900</span> </italic> , encodes a protein kinase with 86% of its amino acid sequence being identical to a rice protein kinase ( <italic> PSTOL1 </italic> ). Overexpression of <italic> PSTOL1 </italic> in rice significantly enhanced grain <span class="trait">yield</span> in phosphorus-deficient soil (Gamuyao et al., <xref rid="B10" ref-type="bibr"> 2012 </xref> ). In addition, we also found another <italic> PSTOL1 </italic> -like gene, <italic> <span class="gene">Glyma.13G280500</span> </italic> (74% identical to <italic> PSTOL1 </italic> ) within the region of the new QTL ( <italic> q13-1 </italic> ), which is physically close to <italic> q13-2 </italic> on chromosome 13. Although, the promising genes were provided within these loci, more studies such as characterizing the expression pattern of these genes are needed to uncover the genetic mechanism underlying P efficiency.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Transgenic soybean overexpressing <italic> Gm <styled-content style="fixed-case"> SAMT </styled-content> 1 </italic> exhibits <span class="trait">resistance</span> to multiple&#x02010; <styled-content style="fixed-case"> HG </styled-content> types of soybean cyst nematode <italic> Heterodera glycines </italic></h2>
			<h3>PMID:27064027</h3>
				

				
					<h3>PMCID:5095773</h3>
					<h4>期刊:Plant Biotechnology Journal</h4>
				
			<p>
				作者：<contrib id="pbi12566-cr-0001" contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Jingyu </given-names>  </name>  <xref ref-type="aff" rid="pbi12566-aff-0001">
				<br/> 
				年份：<day> 07 </day>  <month> 4 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max (L.) Merr.) salicylic acid methyl transferase <span class="gene">(GmSAMT1)</span> catalyses the conversion of salicylic acid to methyl salicylate. Prior results showed that when <span class="gene">GmSAMT1</span> was overexpressed in transgenic soybean hairy roots, <span class="trait">resistance</span> is conferred against soybean cyst nematode (SCN), Heterodera glycines Ichinohe. In this study, we produced transgenic soybean overexpressing <span class="gene">GmSAMT1</span> and characterized their response to various SCN races. Transgenic plants conferred a significant reduction in the development of SCN HG type 1.2.5.7 (race 2), HG type 0 (race 3) and HG type 2.5.7 (race 5). Among transgenic lines, <span class="gene">GmSAMT1</span> expression in roots was positively associated with SCN <span class="trait">resistance.</span> In some transgenic lines, there was a significant decrease in salicylic acid titer relative to control plants. No significant seed <span class="trait">yield</span> differences were observed between transgenics and control soybean plants grown in one greenhouse with 22 °C day/night temperature, whereas transgenic soybean had higher <span class="trait">yield</span> than controls grown a warmer greenhouse (27 °C day/23 °C night) temperature. In a 1-year field experiment in Knoxville, TN, there was no significant difference in seed <span class="trait">yield</span> between the transgenic and nontransgenic soybean under conditions with negligible SCN infection. We hypothesize that <span class="gene">GmSAMT1</span> expression affects salicylic acid biosynthesis, which, in turn, attenuates SCN development, without negative consequences to soybean <span class="trait">yield</span> or other morphological traits. Thus, we conclude that <span class="gene">GmSAMT1</span> overexpression confers broad <span class="trait">resistance</span> to multiple SCN races, which would be potentially applicable to commercial production.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In the United States, the soybean cyst nematode (SCN) ( <italic> Heterodera glycines </italic> Ichinohe) is the most damaging pest to soybean ( <italic> Glycine max </italic> [L.] Merr.) (Koenning and Wrather, <xref rid="pbi12566-bib-0021" ref-type="ref"> 2010 </xref> ). SCN&#x02010;resistant cultivars are currently the most effective way to manage SCN. However, among a total of 118 SCN&#x02010;resistant accessions that have been identified in the USA, few have any <span class="trait">resistance</span> to four or more SCN races (Arelli <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0002" ref-type="ref"> 2000 </xref> ). Soybean breeders have identified quantitative trait loci (QTL), which are distributed over the genome. Most SCN&#x02010;resistance QTL can be traced to &#x02018;Peking&#x02019; PI 437654 and PI 88788 (Concibido <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0006" ref-type="ref"> 2004 </xref> ), which provide some <span class="trait">resistance</span> to SCN races 2, 3 and 5 (Guo <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0010" ref-type="ref"> 2005 </xref> ). Genes cloned from two QTL from PI 88788 and cultivar &#x02018;Forrest&#x02019; (Cook <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0007" ref-type="ref"> 2012 </xref> ; Liu <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0024" ref-type="ref"> 2012 </xref> ) have provided new&#x000a0;insights into understanding of SCN <span class="trait">resistance</span> mechanisms. The most well&#x02010;known <span class="trait">resistance</span> QTL are located on chromosomes 18 ( <italic> rhg1 </italic> ) and 8 ( <italic> <span class="gene">Rhg4</span> </italic> ), which are considered to be the major sources of <span class="trait">resistance</span> in soybean cultivars (Concibido <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0006" ref-type="ref"> 2004 </xref> ). The <italic> rhg1 </italic> gene is vital for broad SCN <span class="trait">resistance</span> (Concibido <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0006" ref-type="ref"> 2004 </xref> ; Guo <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0010" ref-type="ref"> 2005 </xref> ), and the <italic> <span class="gene">Rhg4</span> </italic> gene is critical for <span class="trait">resistance</span> to race 3 (Concibido <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0006" ref-type="ref"> 2004 </xref> ; Webb <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0036" ref-type="ref"> 1995 </xref> ). A number of minor <span class="trait">resistance</span> genes have also been reported, which vary according to germplasm and nematode race (Winter <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0037" ref-type="ref"> 2006 </xref> ). The SCN <span class="trait">resistance</span> conferred by <italic> rhg1 </italic> in PI 88788 is mediated by a 31&#x02010;kb genomic region, where three genes were found to contribute to SCN <span class="trait">resistance</span> when expressed simultaneously (Cook <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0007" ref-type="ref"> 2012 </xref> ). High copy numbers of this genomic region resulted in increased basal expression levels of these three genes and SCN <span class="trait">resistance</span> (Cook <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0007" ref-type="ref"> 2012 </xref> ). <italic> <span class="gene">Rhg4</span> </italic> encodes a serine hydroxymethyltransferase, which plays a critical role in cellular one&#x02010;carbon metabolism (Liu <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0024" ref-type="ref"> 2012 </xref> ). However, the mechanism of interactions between soybean and variable SCN HG types remains elusive.
					</p>
				
					<p>&nbsp;&nbsp;
						We previously identified a soybean salicylic acid methyl transferase gene ( <italic> <span class="gene">GmSAMT1</span> </italic> ) as a candidate SCN defence&#x02010;related gene (Mazarei <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0029" ref-type="ref"> 2011 </xref> ) and studied SCN race 2 (HG type 1.2.5.7) development in soybean hairy roots overexpressing <italic> <span class="gene">GmSAMT1</span> </italic> (Lin <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0023" ref-type="ref"> 2013 </xref> ). <italic> <span class="gene">GmSAMT1</span> </italic> encodes an <italic> S </italic> &#x02010;adenosyl&#x02010; <sc> l </sc> &#x02010;methionine&#x02010;dependent salicylic acid carboxyl methyltransferase, which catalyses the methylation of SA to form methyl salicylate (MeSA) (Lin <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0023" ref-type="ref"> 2013 </xref> ). The role of MeSA in plant defence was indicated as a defence signal, which was systemically transported and converted to SA to activate systemic acquired <span class="trait">resistance</span> (SAR) (Park <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0030" ref-type="ref"> 2007 </xref> ). The SAMT genes from other plant species, including <italic> Arabidopsis </italic> (Chen <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0004" ref-type="ref"> 2003 </xref> ), rice (Koo <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0022" ref-type="ref"> 2007 </xref> ; Zhao <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0042" ref-type="ref"> 2010 </xref> ) and tobacco (Park <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0030" ref-type="ref"> 2007 </xref> ) are also involved in plant defence. For instance, SAR in response to tobacco mosaic&#x000a0;virus&#x000a0;(TMV) was found to be blocked when <italic> NtSAMT1 </italic> was silenced in primary infected tobacco leaves (Park <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0030" ref-type="ref"> 2007 </xref> ). The transgenic tomato with overexpression of <italic> SAMT </italic> exhibited delayed disease symptoms following infection with the bacterial pathogen <italic> Xanthomonas campestris </italic> pv. <italic> vesicatoria </italic> ( <italic> Xcv </italic> ) (Tieman <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0033" ref-type="ref"> 2010 </xref> ). Transgenic <italic> Arabidopsis </italic> overexpressing a rice salicylic acid/benzoic acid carboxyl methyltransferase gene <italic> OsBSMT1 </italic> was more susceptible to disease caused by the bacterial pathogen <italic> Pseudomonas syringae </italic> or the fungal pathogen <italic> Golovinomyces orontii </italic> than the wild&#x02010;type plants (Koo <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0022" ref-type="ref"> 2007 </xref> ). In our prior study, transgenic hairy root with overexpression of <italic> <span class="gene">GmSAMT1</span> </italic> in SCN&#x02010;susceptible soybean significantly reduced the development of SCN race 2, indicating that <italic> <span class="gene">GmSAMT1</span> </italic> played a role in <span class="trait">resistance</span> against SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						The objectives of this study were to characterize multiple generations of <italic> <span class="gene">GmSAMT1</span> </italic> transgenic soybean plants in the greenhouse and field for <span class="trait">resistance</span> to SCN, determine whether there were correlations between <italic> <span class="gene">GmSAMT1</span> </italic> expression with SCN <span class="trait">resistance,</span> key metabolite profiles and seed <span class="trait">yield.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Ten independent transgenic T1 soybean lines were generated, and nine lines were confirmed to contain a single T&#x02010;DNA insertion using herbicide&#x02010;resistance segregation analysis (Table S1). Subsequently, homozygous T3 transgenic soybean lines were obtained from self&#x02010;fertilization and used for further analysis. PCR analysis showed that <italic> <span class="gene">GmSAMT1</span> </italic> and <italic> Bar </italic> (bialaphos <span class="trait">resistance</span> gene) genes were stably inherited in these soybean lines (Figure S1). The copy number was further confirmed by quantitative PCR (qPCR). The T2 homozygous lines (L5&#x02010;5, L20&#x02010;3, L56&#x02010;3 and L65&#x02010;3) contained one copy number of the transgene, which is consistent with the herbicide&#x02010;resistance segregation analysis (Table S1). However, based on the qPCR results, the transgenic soybean lines L16&#x02010;2 and L16&#x02010;3, which were derived from the same event (L16), contained 2 copies of the transgene (Table S2).
					</p>
				
					<p>&nbsp;&nbsp;
						There was no significant difference on the number of eggs per cyst (ranging from 46 to 60 eggs per cyst) between the control and transgenic lines. Therefore, for comparison of the SCN <span class="trait">resistance</span> between the control and transgenic lines, the number of cysts from each line was counted for calculation of female index. Six transgenic homozygous T3 transgenic lines with varied transgene expression conferred a range of <span class="trait">resistance</span> against three HG types of SCN with decreases in the SCN HG type 1.2.5.7 (race 2) female index, which ranged from 46% to 66% relative to the control (Figure&#x000a0; <xref rid="pbi12566-fig-0002" ref-type="fig"> 2 </xref> a). Transgenic lines also exhibited statistically significant decrease in female index when they were inoculated with HG type 0 (race 3) and HG type 2.5.7 (race 5) SCN. For HG type 0 (race 3), the decrease in the SCN female index ranged from 46% to 79% (Figure&#x000a0; <xref rid="pbi12566-fig-0002" ref-type="fig"> 2 </xref> b). The decrease in the SCN female index ranging from 43% to 60% was also observed for HG type 2.5.7 (race 5) (Figure&#x000a0; <xref rid="pbi12566-fig-0002" ref-type="fig"> 2 </xref> c). There was a significant negative correlation between total <italic> <span class="gene">GmSAMT1</span> </italic> expression and SCN HG type 0 (race 3) female index ( <italic> r </italic> &#x000a0;=&#x000a0;&#x02212;0.51, <italic> P </italic> &#x000a0;=&#x000a0;0.018), as well as between transgene expression and SCN HG type 0 (race 3) female index ( <italic> r </italic> &#x000a0;=&#x000a0;&#x02212;0.56, <italic> P </italic> &#x000a0;=&#x000a0;0.009) (Figure&#x000a0; <xref rid="pbi12566-fig-0003" ref-type="fig"> 3 </xref> a,b). No significant correlation was observed between other SCN HG types and the expression of <italic> <span class="gene">GmSAMT1</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Five T4 homozygous transgenic soybean lines overexpressing <italic> <span class="gene">GmSAMT1</span> </italic> (L16&#x02010;2, L16&#x02010;3, L20&#x02010;3, L56&#x02010;3 and L65&#x02010;3) showed no statistically significant differences in plant phenotype (Figure S6), agronomic traits, such as maturity (ranging from 116&#x000a0;days to 124&#x000a0;days), seed <span class="trait">yield</span> (ranging from 3840.1 &#x000a0;kg/ha to 4282.6&#x000a0;kg/ha), seed size (ranging from 15.16&#x000a0;g to 16.36&#x000a0;g 100&#x02010;seed weight), plant height (ranging from 108.4&#x000a0;cm to 114.3&#x000a0;cm) and <span class="trait">lodging</span> score (ranging from 1.3 to 2.7) compared with the nontransgenic parent line (&#x02018;Williams 82&#x02019;) and null segregant control lines (Table&#x000a0; <xref rid="pbi12566-tbl-0001" ref-type="table-wrap"> 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In our previous study, biochemical assays characterized <span class="gene">GmSAMT1</span> to be a SA methyltransferase (Lin <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0023" ref-type="ref"> 2013 </xref> ). Overexpression of <italic> <span class="gene">GmSAMT1</span> </italic> in soybean hairy roots in two susceptible backgrounds exhibited increased <span class="trait">resistance</span> up to 46.6% against SCN HG type 1.2.5.7 (race 2) (Lin <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0023" ref-type="ref"> 2013 </xref> ). In the current study, stable <italic> <span class="gene">GmSAMT1</span> </italic> of transgenic soybean was produced, and homozygous T3 soybean lines with a range of transgene expression were challenged with multiple&#x02010;HG types of SCN. The transgenic soybean line (L20&#x02010;3), the line with the highest transgene expression also had the highest SCN <span class="trait">resistance</span> against HG type 0 (race 3) (Figure&#x000a0; <xref rid="pbi12566-fig-0002" ref-type="fig"> 2 </xref> b). Lower expressing lines, such as L16&#x02010;3 and L65&#x02010;3, had relatively higher SCN <span class="trait">resistance</span> against HG type 1.2.5.7 (race 2) and HG type 2.5.7 (race 5) (Figure&#x000a0; <xref rid="pbi12566-fig-0002" ref-type="fig"> 2 </xref> a,c). These results suggest <italic> <span class="gene">GmSAMT1</span> </italic> expression might play a variable role in <span class="trait">resistance</span> against different SCN HG types, which was an unexpected finding of this research. The role played by <italic> SAMT </italic> plant genes in disease <span class="trait">resistance</span> against bacterial and fungal pathogens including <italic> Pseudomonas syringae, Golovinomyces orontii </italic> and <italic> Hyaloperonospora arabidopsis </italic> has been previously reported (Koo <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0022" ref-type="ref"> 2007 </xref> ; Liu <italic> et&#x000a0;al., </italic> <xref rid="pbi12566-bib-3000" ref-type="ref"> 2010 </xref> ). These studies used transgenic <italic> Arabidopsis </italic> and investigated the relationship between gene expression and disease <span class="trait">resistance</span> in the above&#x02010;ground tissues.
					</p>
				
					<p>&nbsp;&nbsp;
						Considering SA as a major component of SAR, we examined the expression of the SAR marker genes <italic> PR1 </italic> and <italic> PR3 </italic> in the transgenic soybean. No significant difference in mRNA abundance of <italic> <span class="gene">GmPR1</span> </italic> and <italic> <span class="gene">GmPR3</span> </italic> was detected between the transgenic and the nontransgenic control lines (Figure S3), which is consistent with our prior hairy root study (Lin <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0023" ref-type="ref"> 2013 </xref> ). In other studies using transgenic plants, varied PR gene expression was found, such as the increased expression of <italic> PR1 </italic> and <italic> PR3 </italic> in transgenic soybean overexpressing <italic> <span class="gene">GmEREBP1</span> </italic> , a transcription factor activating defence genes (Mazarei <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0028" ref-type="ref"> 2007 </xref> ), 10% down&#x02010;regulated <italic> PR1 </italic> and 5&#x02010;fold increased <italic> PR3 </italic> expression in transgenic hairy roots overexpressing <italic> <span class="gene">GmSNC1</span> </italic> , an SAR&#x02010;related gene (Maldonado <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0025" ref-type="ref"> 2014 </xref> ). Our results may indicate that overexpression of <italic> <span class="gene">GmSAMT1</span> </italic> in soybean could contribute to the broad&#x02010;spectrum <span class="trait">resistance</span> observed for at least for the three SCN HG types tested here, but not through altering the steady&#x02010;state levels of PR gene expression.
					</p>
				
					<p>&nbsp;&nbsp;
						Plant tissues infected by nematodes are highly metabolically active (Hofmann <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0015" ref-type="ref"> 2010 </xref> ). Through analysis of several important metabolites in the SA biosynthesis pathway, intriguing alteration of some compounds in our transgenic soybean was observed, such as decreased SA and BA and increased PA in leaves (Figure&#x000a0; <xref rid="pbi12566-fig-0005" ref-type="fig"> 5 </xref> ). The most parsimonious explanation of our results is that <italic> <span class="gene">GmSAMT1</span> </italic> overexpression caused SA to decrease, subsequently impacting pools of its direct putative precursor, BA, presumably through a feedback regulatory loop. PA, as one important component in SA biosynthesis, was suggested to be regulated by chorismate mutase (Bekal <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0003" ref-type="ref"> 2003 </xref> ; Huang <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0016" ref-type="ref"> 2005 </xref> ; Jones <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0017" ref-type="ref"> 2003 </xref> ; Vanholme <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0034" ref-type="ref"> 2009 </xref> ). Plant&#x02010;parasitic cyst and root&#x02010;knot nematodes have been shown to synthesize and secrete chorismate mutases into plant cells and tissues, presumably to manipulate SA and basal defence responses (reviewed in Hewezi, <xref rid="pbi12566-bib-0013" ref-type="ref"> 2015 </xref> ). Overexpression of <italic> SAMT </italic> may drive the biosynthesis of PA for increased resistant response to SCN infection. While we did perform metabolite analysis on the transgenic soybean roots, metabolite levels were below detection. Basal SA levels vary among plant species and tissues within the same plant species (Cho <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0005" ref-type="ref"> 2013 </xref> ). No MeSA was detected in leaf and root tissues of transgenic and control plants. MeSA might be synthesized and emitted from plants in the volatile form. Overall, the change of metabolites in transgenic soybean leaf may lead to a nutritional deficiency of SCN, hinder the nematode development and increase the transgenic soybean <span class="trait">resistance</span> against SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, transgenic <italic> <span class="gene">GmSAMT1</span> </italic> soybean appears to have prospects as a useful crop&#x02010;based tool to manage SCN in the field. Moreover, as <italic> <span class="gene">GmSAMT1</span> </italic> is a soybean gene, it is a good candidate for using a cisgenic approach for deployment, which might decrease the regulatory burden for commercialization. Furthermore, tuning the expression of <italic> <span class="gene">GmSAMT1</span> </italic> might confer <span class="trait">resistance</span> to pathogens other than SCN. A broad&#x02010;based pathogen&#x02010;resistant cisgenic soybean plant would have value to a broad geographic range of soybean growers.
					</p>
				
					<p>&nbsp;&nbsp;
						The insertion of <italic> <span class="gene">GmSAMT1</span> </italic> and <italic> Bar </italic> (bialaphos <span class="trait">resistance</span> gene) genes was examined by PCR using genomic DNA as template. Genomic DNA was extracted from leaves of homozygous T3 transgenic soybeans by CTAB method (Stewart and Via, <xref rid="pbi12566-bib-0032" ref-type="ref"> 1993 </xref> ). Genomic DNA was diluted to 20&#x000a0;ng/&#x003bc;L for PCR. The PCR conditions were as follows: 94&#x000a0;&#x000b0;C for 5&#x000a0;min followed by 35 cycles at 94&#x000a0;&#x000b0;C for 30&#x000a0;s, 56&#x000a0;&#x000b0;C for 30&#x000a0;s and 72&#x000a0;&#x000b0;C for 30&#x000a0;s and a final extension at 72&#x000a0;&#x000b0;C for 10&#x000a0;min. To only amplify the transgene <italic> <span class="gene">GmSAMT1</span> </italic> , a specific forward primer was designed to locate on the ending region of <italic> <span class="gene">GmSAMT1</span> </italic> and a reverse primer was located at the beginning region of NOS terminator as follows: Transgene&#x02010;SAMT&#x02010;F (forward): 5&#x02032;&#x02010;CTACCAGCAAATCTTGGCTGAA&#x02010;3&#x02032; and Transgene&#x02010;SAMT&#x02010;R (reverse): 5&#x02032;&#x02010;GCAAGACCGGCAACAGGA&#x02010;3&#x02032;. The same pair of primers was also used for following genomic qPCR (for determining the copy number) and qRT&#x02010;PCR (for determining the expression level) of the transgene <italic> <span class="gene">GmSAMT1</span> </italic> . The primers for amplifying <italic> Bar </italic> gene are as follows: Bar&#x02010;F (forward): 5&#x02032;&#x02010;CGAGTCGACCGTGTACGTC&#x02010;3&#x02032; and Bar&#x02010;R (reverse): 5&#x02032;&#x02010;GCAACTGTCGGTCCAATAGAC&#x02010;3&#x02032;. To determine the copy number of the transgene <italic> <span class="gene">GmSAMT1</span> </italic> , real&#x02010;time qPCR with internal reference gene&#x02010;based method was performed as described (Mason <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0026" ref-type="ref"> 2002 </xref> ; Yuan <italic> et&#x000a0;al </italic> ., <xref rid="pbi12566-bib-0040" ref-type="ref"> 2007 </xref> ). Four 1&#x000a0;:&#x000a0;5 serial dilutions were made from 20&#x000a0;ng/&#x003bc;L genomic DNA to generate standard curves. Ct values and relative abundance were calculated using software supplied with the Applied Biosystems 7900 HT Fast Real&#x02010;Time PCR system. DNA accumulation was measured using SYBR Green (Applied Biosystems, Foster City, CA) as the reference dye. The soybean ubiquitin 3 gene <span class="gene">(GmUBI,</span> GenBank accession D28123) was used as an internal reference gene. The sequences of gene&#x02010;specific primers were as follows: <span class="gene">GmUBI&#x02010;F</span> (forward): 5&#x02032;&#x02010;GTGTAATGTTGGATGTGTTCCC&#x02010;3&#x02032; and <span class="gene">GmUBI&#x02010;R</span> (reverse): 5&#x02032;&#x02010;ACACAATTGAGTTCAACACAAACCG&#x02010;3&#x02032;. The following qPCR conditions were used: 50&#x000a0;&#x000b0;C for 2&#x000a0;min; 95&#x000a0;&#x000b0;C for 10&#x000a0;min; followed by 40 cycles of 95 &#x000a0;C for 15&#x000a0;s; 60&#x000a0;&#x000b0;C for 1&#x000a0;min; 72&#x000a0;&#x000b0;C for 30&#x000a0;s. All the assays were conducted in duplicate. PCR efficiencies for transgene and reference gene were equal among the samples. The copy number for each transgenic soybean lines was calculated from three samples with 95% confidence intervals.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Crop responses to elevated CO2 and interactions with H2O, N, and temperature.</h2>
			<h3>PMID:27043481  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kimball BA(1).
				<br/> 
				年份：2016 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						About twenty-seven years ago, free-air CO2 enrichment (FACE) technology was developed that enabled the air above open-field plots to be enriched with CO2 for entire growing seasons. Since then, FACE experiments have been conducted on cotton, wheat, ryegrass, clover, potato, grape, rice, barley, sugar beet, soybean, cassava, rape, mustard, coffee (C3 crops), and sorghum and maize (C4 crops). Elevated CO2 (550ppm from an ambient concentration of about 353ppm in 1990) decreased evapotranspiration about 10% on average and increased canopy temperatures about 0.7°C. Biomass and <span class="trait">yield</span> were increased by FACE in all <span class="gene">C3</span> species, but not in <span class="gene">C4</span> species except when water was limiting. Yields of <span class="gene">C3</span> grain crops were increased on average about 19%.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nitrogen redistribution and its relationship with the expression of <span class="gene">GmATG8c</span> during seed filling in soybean.</h2>
			<h3>PMID:26871505  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Islam MM(1), Ishibashi Y(2), Nakagawa AC(3), Tomita Y(3), Iwaya-Inoue M(2), Arima S(4), Zheng SH(5).
				<br/> 
				年份：2016 Mar 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It is well known that some nitrogen in the vegetative organs is redistributed to the seeds during seed filling in soybean (Glycine max [L.] Merrill). This redistribution is considered to affect the seed <span class="trait">yield</span> of soybean. However, it is still not clear when the nitrogen moves from the vegetative part to the seeds, and the relationship between nitrogen redistribution and leaf senescence has not been clarified. The soybean variety Fukuyutaka was grown in the experimental field of Saga University, Japan from 22 July to 31 October, 2014. After the first flower stage (R1), the plant samples were collected weekly and were separated into leaf, petiole, stem, podshell and seed. The nitrogen concentrations in each plant part were determined. Fresh leaf samples were provided for the determination of soluble protein and autophagy gene <span class="gene">GmATG8c</span> expression. The nitrogen that accumulated in the vegetative parts reached its highest level at 60days after sowing (DAS), then began to decrease at 73DAS (R6). This decrease is considered to be the consequence of nitrogen redistribution from the vegetative parts to the seeds. The movement of nitrogen from the vegetative parts to the seeds was estimated to occur at around 73DAS (R6). At this stage, leaf SPAD values, leaf nitrogen, and soluble protein concentrations began to decrease simultaneously, suggesting the onset of leaf senescence. Furthermore, the expression of the autophagy gene <span class="gene">GmATG8c</span> in the leaves increased dramatically from 73 to 85DAS, which is the duration of nitrogen redistribution. The results suggest that the nitrogen redistribution from the vegetative parts to the seeds could be one of the initiating factors of leaf senescence, and the autophagy gene <span class="gene">GmATG8c</span> was associated with this process.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Costs of Photorespiration to Food Production Now and in the Future.</h2>
			<h3>PMID:26865340  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Walker BJ(1)(2), VanLoocke A(3), Bernacchi CJ(1)(2)(4), Ort DR(1)(2)(4).
				<br/> 
				年份：2016 Apr 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photorespiration is essential for <span class="gene">C3</span> plants but operates at the massive expense of fixed carbon dioxide and energy. Photorespiration is initiated when the initial enzyme of photosynthesis, ribulose-1,5-bisphosphate carboxylase/oxygenase (Rubisco), reacts with oxygen instead of carbon dioxide and produces a toxic compound that is then recycled by photorespiration. Photorespiration can be modeled at the canopy and regional scales to determine its cost under current and future atmospheres. A regional-scale model reveals that photorespiration currently decreases US soybean and wheat yields by 36% and 20%, respectively, and a 5% decrease in the losses due to photorespiration would be worth approximately $500 million annually in the United States. Furthermore, photorespiration will continue to impact <span class="trait">yield</span> under future climates despite increases in carbon dioxide, with models suggesting a 12-55% improvement in gross photosynthesis in the absence of photorespiration, even under climate change scenarios predicting the largest increases in atmospheric carbon dioxide concentration. Although photorespiration is tied to other important metabolic functions, the benefit of improving its efficiency appears to outweigh any potential secondary disadvantages.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The <italic> <styled-content style="fixed-case"> A </styled-content> rabidopsis </italic> transcriptional regulator <styled-content style="fixed-case"> DPB </styled-content> 3&#x02010;1 enhances heat stress tolerance without growth retardation in rice</h2>
			<h3>PMID:26841113</h3>
				

				
					<h3>PMCID:5067654</h3>
					<h4>期刊:Plant Biotechnology Journal</h4>
				
			<p>
				作者：<contrib id="pbi12535-cr-0001" contrib-type="author">  <name>  <surname> Sato </surname>  <given-names> Hikaru </given-names>  </name>  <xref ref-type="aff" rid="pbi12535-aff-0001">
				<br/> 
				年份：<day> 28 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The enhancement of heat stress tolerance in crops is an important challenge for food security to facilitate adaptation to global warming. In Arabidopsis thaliana, the transcriptional regulator DNA polymerase II subunit B3-1 (DPB3-1)/nuclear factor Y subunit <span class="gene">C10</span> (NF-YC10) has been reported as a positive regulator of Dehydration-responsive element binding protein 2A (DREB2A), and the overexpression of DPB3-1 enhances heat stress tolerance without growth retardation. Here, we show that DPB3-1 interacts with <span class="gene">DREB2A</span> homologues in rice and soya bean. Transactivation analyses with Arabidopsis and rice mesophyll protoplasts indicate that DPB3-1 and its rice homologue OsDPB3-2 function as positive regulators of <span class="gene">DREB2A</span> homologues. Overexpression of DPB3-1 did not affect plant growth or <span class="trait">yield</span> in rice under nonstress conditions. Moreover, DPB3-1-overexpressing rice showed enhanced heat stress tolerance. Microarray analysis revealed that many heat stress-inducible genes were up-regulated in DPB3-1-overexpressing rice under heat stress conditions. However, the overexpression of DPB3-1 using a constitutive promoter had almost no effect on the expression of these genes under nonstress conditions. This may be because DPB3-1 is a coactivator and thus lacks inherent transcriptional activity. We conclude that DPB3-1, a coactivator that functions specifically under abiotic stress conditions, could be utilized to increase heat stress tolerance in crops without negative effects on vegetative and reproductive growth.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We previously reported that DPB3&#x02010;1 interacts with the transcription factor <span class="gene">DREB2A</span> and that the overexpression of <italic> DPB3&#x02010;1 </italic> improves heat stress tolerance in <italic> Arabidopsis </italic> by increasing the expression of many stress&#x02010;inducible genes under heat stress conditions. <span class="gene">DREB2A</span> is an important transcription factor that regulates both heat and <span class="trait">drought</span> stress responses in <italic> Arabidopsis </italic> , and <span class="gene">DREB2A</span> activates gene expression by binding to dehydration&#x02010;responsive element (DRE) on the target promoters (Sakuma <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0037" ref-type="ref"> 2006a </xref> , <xref rid="pbi12535-bib-0038" ref-type="ref"> b </xref> ). A phenotypic analysis suggested that DPB3&#x02010;1 would be a useful factor for generating heat stress&#x02010;tolerant crops because the overexpression of <italic> DPB3&#x02010;1 </italic> did not have negative effects on vegetative growth in <italic> Arabidopsis </italic> (Sato <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0039" ref-type="ref"> 2014 </xref> ). In this study, interactions between DPB3&#x02010;1 and <span class="gene">DREB2A</span> homologues in rice and soya bean were confirmed. The effects of <italic> DPB3&#x02010;1 </italic> overexpression on vegetative and reproductive growth were assessed under nonstress conditions in rice; heat stress tolerance in <italic> DPB3&#x02010;1 </italic> overexpressing rice was also assessed.
					</p>
				
					<p>&nbsp;&nbsp;
						Recently, we reported that DPB3&#x02010;1 interacts with the N&#x02010;terminal region of <span class="gene">DREB2A</span> and functions as a positive regulator of <span class="gene">DREB2A</span> specifically under heat stress conditions (Sato <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0039" ref-type="ref"> 2014 </xref> ). DPB3 family proteins are widely conserved among land plants, including rice and soya bean (Sato <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0039" ref-type="ref"> 2014 </xref> ), indicating that DPB3&#x02010;1 might positively regulate <span class="gene">DREB2A</span> homologues in various crop plants. Initially, we analysed the reproductive growth phenotypes and <span class="trait">yield</span> of <italic> DPB3&#x02010;1 </italic> &#x02010;overexpressing <italic> Arabidopsis </italic> to confirm the effects of <italic> DPB3&#x02010;1 </italic> overexpression on <span class="trait">yield</span> (Figure&#x000a0;S1). The results indicated that the overexpression of <italic> DPB3&#x02010;1 </italic> did not have negative effects on reproductive growth and seed formation (Table&#x000a0;S1). We also examined the expression levels of a rice <italic> DPB3&#x02010;1 </italic> homologue, <italic> OsDPB3&#x02010;2 </italic> ( <italic> LOC_Os03g63530 </italic> ), under abiotic stress conditions in seedlings. Expression of the <italic> OsDPB3&#x02010;2 </italic> gene was induced by heat stress in shoots (Figure&#x000a0; <xref rid="pbi12535-fig-0001" ref-type="fig"> 1 </xref> a). The gene was highly expressed in shoot basal regions, including meristematic cells (Figure&#x000a0; <xref rid="pbi12535-fig-0002" ref-type="fig"> 2 </xref> ); this expression pattern has also been observed for <italic> DPB3&#x02010;1 </italic> in <italic> Arabidopsis </italic> (Sato <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0039" ref-type="ref"> 2014 </xref> ). These findings suggest that OsDPB3&#x02010;2 has conserved functions similar to DPB3&#x02010;1 in rice. The expression patterns of <italic> OsDPB3&#x02010;2 </italic> were different from those of <italic> DPB3&#x02010;1 </italic> ; <italic> OsDPB3&#x02010;2 </italic> was expressed in both roots and shoots under nonstress conditions and was not repressed by dehydration stress (Figure&#x000a0; <xref rid="pbi12535-fig-0001" ref-type="fig"> 1 </xref> a). However, <italic> DPB3&#x02010;1 </italic> was primarily expressed in aerial regions and was repressed by dehydration (Figure&#x000a0; <xref rid="pbi12535-fig-0001" ref-type="fig"> 1 </xref> b). These results imply that OsDPB3&#x02010;2 has specific functions in these tissues under such conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Although our results clearly demonstrate that the overexpression of <italic> DPB3&#x02010;1 </italic> enhances heat stress tolerance in rice without growth retardation, additional experiments are important to confirm its usability for agricultural applications. First, the heat stress response and tolerance of reproductive stages should be analysed in future studies. Many previous works have reported that plants are more sensitive to abiotic stresses, including high temperature, during the reproductive stage (Kim <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0021" ref-type="ref"> 2001 </xref> ; Omae <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0032" ref-type="ref"> 2012 </xref> ; Tunc&#x02010;Ozdemir <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0045" ref-type="ref"> 2013 </xref> ; Zinn <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0057" ref-type="ref"> 2010 </xref> ). The expression profiles of <italic> <span class="gene">HSF</span> </italic> , <italic> HSP </italic> and <italic> DREB </italic> family genes were conserved between the vegetative and reproductive stages in rice, and the ABRE motif has been suggested to be important for gene induction in response to heat stress in reproductive organs (Chauhan <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0003" ref-type="ref"> 2011 </xref> ; Jagadish <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0017" ref-type="ref"> 2010 </xref> ; Zhang <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0054" ref-type="ref"> 2012 </xref> ). These data imply that the overexpression of <italic> DPB3&#x02010;1 </italic> may also confer enhanced heat stress tolerance during reproductive stages. Furthermore, microarray analysis revealed that the expression levels of <span class="trait">drought</span> stress&#x02010;responsive genes were also enhanced in the <italic> DPB3&#x02010;1 </italic> &#x02010;overexpressing rice under heat stress conditions (Figure&#x000a0; <xref rid="pbi12535-fig-0008" ref-type="fig"> 8 </xref> and Figure&#x000a0;S8a); however, there were no significant differences between the vector control and <italic> DPB3&#x02010;1 </italic> &#x02010;overexpressing rice in response to <span class="trait">drought</span> stress alone (Figure&#x000a0;S9), which suggests that DPB3&#x02010;1 could contribute to the induction of <span class="trait">drought</span> stress&#x02010;responsive genes only under heat stress conditions. Previous studies have revealed that the combination of heat and <span class="trait">drought</span> stress had more severe effects on plant growth and development (Mittler, <xref rid="pbi12535-bib-0029" ref-type="ref"> 2006 </xref> ; Rizhsky <italic> et&#x000a0;al </italic> ., <xref rid="pbi12535-bib-0034" ref-type="ref"> 2004 </xref> ; Smith and Katz, <xref rid="pbi12535-bib-0042" ref-type="ref"> 2013 </xref> ). Further studies are required to elucidate the mechanisms by which DPB3&#x02010;1 enhances the expression levels of <span class="trait">drought</span> stress&#x02010;inducible genes under heat stress conditions and whether DPB3&#x02010;1 might be a useful factor for increasing combined heat and <span class="trait">drought</span> stress tolerance in crops.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A recessive allele for delayed flowering at the soybean maturity locus <italic> E9 </italic> is a leaky allele of <italic> FT2a </italic> , a <italic> FLOWERING LOCUS T </italic> ortholog</h2>
			<h3>PMID:26786479</h3>
				

				
					<h3>PMCID:4719747</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Chen </given-names>  </name>  <address>  <email> zhaochen_1112@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Takeshima </surname>  <given-names> Ryoma </given-names>  </name>  <address>  <email> take-ryo@res.agr.hokudai.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Jianghui </given-names>  </name>  <address>  <email> zhu622@res.agr.hokudai.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Meilan </given-names>  </name>  <address>  <email> xumeilan_1984@yahoo.co.jp </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sato </surname>  <given-names> Masako </given-names>  </name>  <address>  <email> satomasa@res.agr.hokudai.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Watanabe </surname>  <given-names> Satoshi </given-names>  </name>  <address>  <email> nabemame@cc.saga-u.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kanazawa </surname>  <given-names> Akira </given-names>  </name>  <address>  <email> kanazawa@res.agr.hokudai.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Liu </surname>  <given-names> Baohui </given-names>  </name>  <address>  <email> liubh@neigaehrb.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Kong </surname>  <given-names> Fanjiang </given-names>  </name>  <address>  <email> kongfj@iga.ac.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamada </surname>  <given-names> Tetsuya </given-names>  </name>  <address>  <email> tetsuyay@res.agr.hokudai.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Abe </surname>  <given-names> Jun </given-names>  </name>  <address>  <email> jabe@res.agr.hokudai.ac.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Research Faculty of Agriculture, Hokkaido University, Sapporo, Hokkaido 060-8589 Japan  </aff>  <aff id="Aff2">  <label/> Faculty of Agriculture, Saga University, Saga, 840-0027 Japan  </aff>  <aff id="Aff3">  <label/> The Key Laboratory of Soybean Molecular Design Breeding, Northeast Institute of Geography and Agroecology, Chinese Academy of Sciences, Harbin, 150081 China  </aff>
				<br/> 
				年份：<day> 6 </day>  <month> 1 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Understanding the molecular mechanisms of flowering and maturity is important for improving the adaptability and <span class="trait">yield</span> of seed crops in different environments. In soybean, a facultative short-day plant, genetic variation at four maturity genes, <span class="gene">E1</span> to E4, plays an important role in adaptation to environments with different photoperiods. However, the molecular basis of natural variation in time to flowering and maturity is poorly understood. Using a cross between early-maturing soybean cultivars, we performed a genetic and molecular study of flowering genes. The progeny of this cross segregated for two maturity loci, <span class="gene">E1</span> and E9. The latter locus was subjected to detailed molecular analysis to identify the responsible gene. RESULTS: Fine mapping, sequencing, and expression analysis revealed that E9 is FT2a, an ortholog of Arabidopsis FLOWERING LOCUS T. Regardless of daylength conditions, the e9 allele was transcribed at a very low level in comparison with the E9 allele and delayed flowering. Despite identical coding sequences, a number of single nucleotide polymorphisms and insertions/deletions were detected in the promoter, untranslated regions, and introns between the two cultivars. Furthermore, the e9 allele had a Ty1/copia-like retrotransposon, SORE-1, inserted in the first intron. Comparison of the expression levels of different alleles among near-isogenic lines and photoperiod-insensitive cultivars indicated that the SORE-1 insertion attenuated FT2a expression by its allele-specific transcriptional repression. SORE-1 was highly methylated, and did not appear to disrupt FT2a RNA processing. CONCLUSIONS: The soybean maturity gene E9 is FT2a, and its recessive allele delays flowering because of lower transcript abundance that is caused by allele-specific transcriptional repression due to the insertion of SORE-1. The FT2a transcript abundance is thus directly associated with the variation in flowering time in soybean. The e9 allele may maintain vegetative growth in early-flowering genetic backgrounds, and also be useful as a long-juvenile allele, which causes late flowering under short-daylength conditions, in low-latitude regions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Understanding the molecular mechanisms of flowering and maturity is important for improving the adaptability and <span class="trait">yield</span> of seed crops in different environments. In soybean, a facultative short-day plant, genetic variation at four maturity genes, <italic> <span class="gene">E1</span> </italic> to <italic> E4 </italic> , plays an important role in adaptation to environments with different photoperiods. However, the molecular basis of natural variation in time to flowering and maturity is poorly understood. Using a cross between early-maturing soybean cultivars, we performed a genetic and molecular study of flowering genes. The progeny of this cross segregated for two maturity loci, <italic> <span class="gene">E1</span> </italic> and <italic> E9 </italic> . The latter locus was subjected to detailed molecular analysis to identify the responsible gene.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Genomic-assisted haplotype analysis and the development of high-throughput SNP markers for salinity tolerance in soybean</h2>
			<h3>PMID:26781337</h3>
				

				
					<h3>PMCID:4726057</h3>
					<h4>期刊:Scientific Reports</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Do </surname>  <given-names> Tuyen </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Jeong-Dong </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chaudhary </surname>  <given-names> Juhi </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J. Grover </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> National Center for Soybean Biotechnology and Division of Plant Sciences, University of Missouri </institution> , Columbia, 65211, MO,  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> School of Applied Biosciences, Kyungpook National University </institution> , Daegu, 41566,  <country> Republic of Korea </country>  </aff>
				<br/> 
				年份：<day> 07 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soil salinity is a limiting factor of crop <span class="trait">yield.</span> The soybean is sensitive to soil salinity, and a dominant gene, <span class="gene">Glyma03g32900</span> is primarily responsible for salt-tolerance. The identification of high throughput and robust markers as well as the deployment of salt-tolerant cultivars are effective approaches to minimize <span class="trait">yield</span> loss under saline conditions. We utilized high quality (15x) whole-genome resequencing (WGRS) on 106 diverse soybean lines and identified three major structural variants and allelic variation in the promoter and genic regions of the <span class="gene">GmCHX1</span> gene. The discovery of single nucleotide polymorphisms (SNPs) associated with structural variants facilitated the design of six KASPar assays. Additionally, haplotype analysis and pedigree tracking of 93 U.S. ancestral lines were performed using publically available WGRS datasets. Identified SNP markers were validated, and a strong correlation was observed between the genotype and salt treatment phenotype (leaf scorch, chlorophyll content and Na(+) accumulation) using a panel of 104 soybean lines and, an interspecific bi-parental population (F8) from PI483463 x Hutcheson. These markers precisely identified salt-tolerant/sensitive genotypes (&gt;91%), and different structural-variants (&gt;98%). These SNP assays, supported by accurate phenotyping, haplotype analyses and pedigree tracking information, will accelerate marker-assisted selection programs to enhance the development of salt-tolerant soybean cultivars.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						An important goal of whole genome re-sequencing data analysis of crop species is to explore genetic variation in diverse germplasm resources, such as wild species, landraces and improved cultivars and to identify molecular markers that accelerate breeding cycles. Soybean germplasm, both <italic> G. max </italic> and <italic> G. soja </italic> species, provide a wide range of salt tolerances. For many years, a great effort has been made to understand the mechanism of salt reaction <xref ref-type="bibr" rid="b39"> 39 </xref> and to precisely identify gene(s) underlying salt tolerance QTL in soybeans <xref ref-type="bibr" rid="b24"> 24 </xref> <xref ref-type="bibr" rid="b25"> 25 </xref> . Previous studies have shown that a QTL on Chr. 3 is the major genomic region determining salinity tolerance in soybean. This locus carries the dominant functional sodium/hydrogen exchanger family gene <italic> <span class="gene">Glyma03g32900</span> </italic> ( <italic> <span class="gene">GmCHX1</span> </italic> ) and accounts for more than 64% of the phenotypic variation <xref ref-type="bibr" rid="b25"> 25 </xref> . GWAS is a statistically powerful approach and provides a higher resolution to identify the position of genetic factors underlying the trait of interest <xref ref-type="bibr" rid="b45"> 45 </xref> <xref ref-type="bibr" rid="b50"> 50 </xref> <xref ref-type="bibr" rid="b51"> 51 </xref> <xref ref-type="bibr" rid="b52"> 52 </xref> . A large number of GWAS has been successfully conducted in soybean using SoySNP50K and WGRS data for nematode <span class="trait">resistance</span> <xref ref-type="bibr" rid="b50"> 50 </xref> , carbon-isotope <xref ref-type="bibr" rid="b51"> 51 </xref> , oil and protein content <xref ref-type="bibr" rid="b45"> 45 </xref> <xref ref-type="bibr" rid="b52"> 52 </xref> , and domestication traits <xref ref-type="bibr" rid="b45"> 45 </xref> . In this study, GWAS pin-pointed a single major and significant locus on Chr. 3 that harbors the previously characterized <italic> <span class="gene">GmCHX1</span> </italic> gene. While the SoySNP50K and WGRS data were able to identify the same major loci on Chr. 3, the number of SNPs was relatively higher in WGRS dataset, which was not surprising. Importantly, high quality WGRS data benefited the discovery of novel structural variants and the large number of SNPs that were translated into functional markers. The results obtained from GWAS thus, allowed us to further investigate the haplotype and SNP variation using WGRS datasets. The genetic basis of salt tolerance in soybean is relatively less complex compared with the response to other abiotic stresses (e.g., <span class="trait">drought,</span> flooding <xref ref-type="bibr" rid="b1"> 1 </xref> <xref ref-type="bibr" rid="b53"> 53 </xref> ) due to the presence of a single dominant locus as detected in the present study.
					</p>
				
					<p>&nbsp;&nbsp;
						A subset of 104 soybean lines were evaluated for salt tolerance in two independent tests in the salinity phenotyping facility at University of Missouri, Columbia, MO, following a well-established procedure as previously described <xref ref-type="bibr" rid="b38"> 38 </xref> . Briefly, at the <span class="gene">V2</span> growth stage, soybean seedlings in cone-trainers were exposed to salt water (120&#x02009;mM) added to a depth of one-third (7&#x02009;cm) of the cone to allow the uptake of salt water. Electrical conductivity (EC) was monitored daily. Individual soybean plants were visually rated for salt sensitivity or tolerance when the sensitive parent, cultivar Hutcheson (HN001), showed severe leaf scorch (approximately 2 weeks after treatment with salt water). Leaf scorch was scored using a 1&#x02013;5 scale, where 1&#x02009;=&#x02009;no apparent <span class="trait">chlorosis;</span> 2&#x02009;=&#x02009;slight (25% of the leaves showed <span class="trait">chlorosis);</span> 3&#x02009;=&#x02009;moderate (50% of the leaves showed <span class="trait">chlorosis</span> and some necrosis); 4&#x02009;=&#x02009;severe <span class="trait">chlorosis</span> (75% of the leaves showed <span class="trait">chlorosis</span> and severe necrosis); and 5&#x02009;=&#x02009;dead (leaves showed severe necrosis and were withered). The average leaf scorch score for each genotype was calculated using the equation (1):
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:A Novel Soybean Intrinsic Protein Gene, <italic> <span class="gene">GmTIP2;3</span> </italic> , Involved in Responding to Osmotic Stress</h2>
			<h3>PMID:26779248</h3>
				

				
					<h3>PMCID:4705450</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Dayong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/254519/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tong </surname>  <given-names> Jinfeng </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Xiaolan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhaolong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wei </surname>  <given-names> Peipei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Yihong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Brestic </surname>  <given-names> Marian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/289749/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Hongxiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/254841/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shao </surname>  <given-names> Hongbo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/59382/overview"/>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Water is essential for plant growth and development. Water deficiency leads to loss of <span class="trait">yield</span> and decreased crop quality. To understand water transport mechanisms in plants, we cloned and characterized a novel tonoplast intrinsic protein (TIP) gene from soybean with the highest similarity to TIP2-type from other plants, and thus designated <span class="gene">GmTIP2;3.</span> The protein sequence contains two conserved NPA motifs and six transmembrane domains. The expression analysis indicated that this gene was constitutively expressed in all detected tissues, with higher levels in the root, stem and pod, and the accumulation of <span class="gene">GmTIP2;3</span> transcript showed a significant response to osmotic stresses, including 20% PEG6000 (polyethylene glycol) and 100 μM ABA (abscisic acid) treatments. The promoter-GUS (glucuronidase) activity analysis suggested that <span class="gene">GmTIP2;3</span> was also expressed in the root, stem, and leaf, and preferentially expressed in the stele of root and stem, and the core promoter region was 1000 bp in length, located upstream of the ATG start codon. The GUS tissue and induced expression observations were consistent with the findings in soybean. In addition, subcellular localization showed that <span class="gene">GmTIP2;3</span> was a plasma membrane-localized protein. Yeast heterologous expression revealed that <span class="gene">GmTIP2;3</span> could improve tolerance to osmotic stress in yeast cells. Integrating these results, <span class="gene">GmTIP2;3</span> might play an important role in response to osmotic stress in plants.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						AQPS play important roles in various physiological processes in plants, such as growth, development, and response to biotic and abiotic stresses. Srivastava et al. ( <xref rid="B29" ref-type="bibr"> 2015 </xref> ) also reviewed the versatile functions of aquaporins as molecular conduits in the plant response to abiotic stresses. For example, Guenther and Roberts ( <xref rid="B12" ref-type="bibr"> 2000 </xref> ) isolated two major intrinsic membrane proteins from <italic> Lotus japonicus </italic> , named <italic> LIMP1 </italic> and <italic> LIMP2 </italic> . Functional analysis using the <italic> Xenopus oocytes </italic> system indicated that LIMP1 appeared to be a member of the <span class="gene">TIP</span> subfamily and LIMP2 was a nodulin 26 ortholog protein. Rodrigues et al. ( <xref rid="B24" ref-type="bibr"> 2013 </xref> ) investigated a gene encoding a root-specific tonoplast intrinsic aquaporin (TIP) from <italic> Eucalyptus grandis </italic> named <italic> EgTIP2 </italic> , whose expression was induced by PEG and mannitol treatments but was downregulated by abscisic acid, suggesting that <italic> EgTIP2 </italic> might be involved in the eucalyptus response to <span class="trait">drought.</span> Wang et al. ( <xref rid="B33" ref-type="bibr"> 2014 </xref> ) cloned and characterized a tonoplast AQP gene ( <italic> TsTIP1;2) </italic> from the halophyte <italic> Thellungiella salsuginea </italic> and reported that it mediated the transduction of both H <sub> 2 </sub> O and H <sub> 2 </sub> O <sub> 2 </sub> across the membranes and might contribute to the survival of <italic> T. salsuginea </italic> under multiple stresses. Ligaba et al. ( <xref rid="B17" ref-type="bibr"> 2011 </xref> ) studied the expression patterns of 7 <italic> MIP </italic> genes from barley under different abiotic stresses using quantitative real-time PCR (RT-PCR), indicating that abiotic stress modulates the expression of major intrinsic proteins in barley. Zelazny et al. ( <xref rid="B37" ref-type="bibr"> 2007 </xref> ) by using FRET imaging analysis showed that plasma membrane aquaporins could interact to regulate their subcellular localization in living maize cells. Tomato <italic> SiTIP2;2 </italic> expressing in transgenic <italic> Arabidopsis </italic> could enhance the plant's tolerance to salt stress and interact with its homologous proteins SiTIP1;1 and SiTIP2;1 (Xin et al., <xref rid="B35" ref-type="bibr"> 2014 </xref> ). Gao et al. ( <xref rid="B11" ref-type="bibr"> 2010 </xref> ) overexpressed <italic> TaNIP </italic> , a putative aquaporin gene from wheat, and found that it could enhance salt tolerance in transgenic <italic> Arabidopsis </italic> . Wang et al. ( <xref rid="B34" ref-type="bibr"> 2011 </xref> ) cloned the novel <italic> Glycine soja </italic> tonoplast intrinsic protein gene <italic> GsTIP2;1 </italic> , and the overexpression of <italic> GsTIP2;1 </italic> in <italic> Arabidopsis </italic> repressed/reduced tolerance to salt and dehydration stress, suggesting that <italic> GsTIP2;1 </italic> might mediate stress sensitivity by enhancing water loss in plants.
					</p>
				
					<p>&nbsp;&nbsp;
						The resulting pYES2- <italic> <span class="gene">GmTIP2;3</span> </italic> :GFP plasmid was introduced into <italic> S. cerevisiae </italic> INVSc1 strain cells using the lithium acetate method, with the empty vector pYES2-GFP as a control. <italic> S. cerevisiae </italic> INVSc1 strain cells transformed with the empty vector PYES2-GFP alone and with pYES2- <italic> <span class="gene">GmTIP2;3</span> </italic> :GFP were induced with galactose and spotted on the SC-Ura medium in 0, 10, 100, 1000, and 10,000-fold-dilutions, and the <span class="trait">drought</span> tolerance of yeast cells expressing <italic> <span class="gene">GmTIP2;3</span> </italic> was tested by 30% PEG6000 treatment for 40 h. The GFP in yeast was observed using a fluorescence microscope (Olympus BX61). All experiments were repeated three times.
					</p>
				
					<p>&nbsp;&nbsp;
						To test whether <italic> <span class="gene">GmTIP2;3</span> </italic> responds to <span class="trait">drought</span> stress, soybean seedling roots were treated with PEG and ABA. Then, the expression of <italic> <span class="gene">GmTIP2;3</span> </italic> was analyzed by quantitative real-time RT-PCR. The results indicated that the expression of <italic> <span class="gene">GmTIP2;3</span> </italic> decreased within 2 h after PEG-6000 (20%) treatment, and then the mRNA level continuously increased from 4 to 12 h and reached a maximum at 12 h (Figure <xref ref-type="fig" rid="F3"> 3B-1 </xref> ) However, ABA treatment (100 &#x003bc;M) initially significantly suppressed <italic> <span class="gene">GmTIP2;3</span> </italic> expression after 10 min treatment, reached its minimum at 30 min ( <italic> p </italic> &#x0003c; 0.05), then increased from 30 to 45 min and continuously decreased from 45 to 120 min, followed by a stable expression level (Figure <xref ref-type="fig" rid="F3"> 3B-2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To analyze the elements contained in the promoter region and the promoter activities of <span class="gene">GmTIP2;3,</span> a more than 2 kb (2081 bp)-long promoter region located upstream of the ATG start codon was amplified and inserted into the pGUSP vector by the T/A cloning method. The resulting construct was transformed into <italic> Lotus japonicus </italic> , and the transgenic plant was successfully obtained (Figures <xref ref-type="fig" rid="F4"> 4A,B </xref> ). The non-transgenic plant was used as a negative control (Figure <xref ref-type="fig" rid="F4"> 4C </xref> ). GUS staining revealed that <span class="gene">GmTIP2;3</span> was mainly expressed in the root (stele), stem (stele), and leaf (Figures <xref ref-type="fig" rid="F4"> 4D&#x02013;F </xref> ). Moreover, transgenic soybean hair roots using this construct also showed higher expression at the stele of the root (Figure <xref ref-type="fig" rid="F4"> 4G </xref> ), which was consistent with its function as a water transporter. The <span class="gene">GmTIP2;3</span> expression patterns in transgenic plants or hairy roots were identical to the patterns in different organs in the soybean plant. In addition, promoter sequence analysis using the PlantCARE online software indicated that it contained many light responsive elements, such as Box4, G-Box and I-Box, GATA-motif, MBS, and GARE-motif (Table <xref ref-type="table" rid="T2"> 2 </xref> ). To further dissect the core region of the <span class="gene">GmTIP2;3</span> promoter and explore the impact of external factors on its expression, a series of 8 truncated vectors were constructed, 2081, 1524, 1035, 935, 835, 735, 663, and 581 bp in length, named P1&#x02013;P8, and transformed into <italic> Agrobacterium rhizogenes </italic> strain K599 to generate soybean hairy roots. The GUS staining and quantity assays demonstrated that only <span class="gene">P1</span> and P3 exhibited GUS activities, and the activity of P3 was stronger than for <span class="gene">P1</span> (Figure <xref ref-type="fig" rid="F5"> 5A </xref> ). P5&#x02013;P8 had no GUS signal, indicating that the core promoter region of <span class="gene">GmTIP2;3</span> was 935 bp long from the ATG site. Interestingly, no GUS signal or GUS activity was detected for P2, implying that the inhibitor sequence occurred between <span class="gene">P1</span> and P3, which also explained why the activity of P3 was stronger than for P1. Meanwhile, the expression of <span class="gene">GmTIP2;3</span> was down-regulated under dark, <span class="trait">drought</span> (PEG and ABA), and salinity treatments for 2 h but showed no response to wounding treatment in transgenic lotus plants (Figure <xref ref-type="fig" rid="F5"> 5B </xref> ). These results were consistent with the results of the expression patterns after treatments with ABA and PEG in soybean roots.
					</p>
				
					<p>&nbsp;&nbsp;
						Spatial and temporal expression analysis showed that <italic> <span class="gene">GmTIP2;3</span> </italic> was constitutively expressed in all tested organs, with higher expression in the root and stem, indicating that it can absorb water from the soil through the root and then transport water through the stem to other organs, such as the leaf, flower, and pod. Tungngoen et al. ( <xref rid="B32" ref-type="bibr"> 2009 </xref> ) cloned and characterized two aquaporins, <italic> HbPIP2;1 </italic> and <italic> HbTIP1;1 </italic> , and induced expression analysis found that <italic> HbTIP1;1 </italic> was down-regulated in liber tissues but up-regulated in laticifers in response to bark Ethrel treatment. Regon et al. ( <xref rid="B22" ref-type="bibr"> 2014 </xref> ) also analyzed the expression patterns of 100 <span class="gene">TIP</span> aquaporin genes from dicots and monocots and indicated that the expression of <italic> <span class="gene">TIP</span> </italic> genes varies during different developmental stages and under stressed conditions. da Silva et al. ( <xref rid="B6" ref-type="bibr"> 2013 </xref> ) identified and analyzed the expression patterns of sugarcane aquaporin genes under water deficit, thereby finding the aquaporin transcription in sugarcane to be potentially genotype specific. These findings demonstrated that <italic> <span class="gene">TIP</span> </italic> expression was organ specific or genotype specific and performed different regulator roles in different tissues. Recently, Lee et al. ( <xref rid="B16" ref-type="bibr"> 2015 </xref> ) showed that the expressions of barley <italic> HvTIP1;2 </italic> and <italic> HvTIP3;1 </italic> were regulated by gibberellic acid (GA) and ABA and that these two hormones were involved in the fusion of protein storage vacuoles in aleurone cells, indicating that <span class="gene">TIP</span> plays another role in vacuole formation and transportation. When subjected to <span class="trait">drought</span> stress (ABA and PEG), the expression of <italic> <span class="gene">GmTIP2;3</span> </italic> showed a dynamic trend at different time points, with an increase after PEG and ABA treatments for 48 h and 45 min, respectively, indicating that the expression of <italic> <span class="gene">GmTIP2;3</span> </italic> exhibited a response to osmotic stress.
					</p>
				
					<p>&nbsp;&nbsp;
						In fact, <span class="gene">GmTIP2;3</span> should be a plasma membrane intrinsic protein (PIP). It was predicted to be localized at the plasma membrane by the online software <ext-link ext-link-type="uri" xlink:href="http://www.predictprotein.org/"> http://www.predictprotein.org/ </ext-link> , and this subcellular localization was proven using <italic> Arabidopsis </italic> protoplasts, yeast cells, and onion epidermal cells (data not shown) harboring GFP. However, BLAST result at NCBI showed <span class="gene">GmTIP2;3</span> to be a tonoplast intrinsic protein (TIP). Analysis of the promoter activity of <italic> <span class="gene">GmTIP2;3</span> </italic> indicated that the activity of P3 (~1000 bp in length) was stronger than the activity of <span class="gene">P1</span> (~2000 bp), implying that the inhibitor region occurred between these two regions, and P4 (~550 bp) exhibited no GUS activity. To further determine the core or minimum region for the <italic> <span class="gene">GmTIP2;3</span> </italic> promoter, five truncated constructs at 100 bp intervals between P3&#x02013;P4 were prepared, but no GUS signal was detected. Therefore, we concluded that the core promoter region for <italic> <span class="gene">GmTIP2;3</span> </italic> was located +1000 bp upstream of the ATG start codon containing the 5&#x02032;UTR region of the <italic> <span class="gene">GmTIP2;3</span> </italic> gene. The promoter&#x02013;GUS system was used to detect the GUS activity changes of transgenic <italic> Lotus </italic> leaf under different treatments, including ABA, Nacl, dark, wounding, and PEG for 2 h. The results showed that the expression of <italic> <span class="gene">GmTIP2;3</span> </italic> decreased under all treatments except wounding. The plant CARE software revealed that the promoter region contains many light-responsive elements, so the down-regulated expression under dark conditions was reasonable. Moreover, GUS activity under <span class="trait">drought</span> treatment for 2 h was consistent with the expression patterns after ABA and PEG treatments for 2 h. Lee et al. ( <xref rid="B16" ref-type="bibr"> 2015 </xref> ) detected the promoter activity of <italic> HvTIP3;1 </italic> in response to ABA and revealed that the ABA responsiveness of the <italic> HvTIP3;1 </italic> promoter is likely to occur via a unique regulatory system distinct from the one involving the ABA-response promoter complexes. Therefore, the mechanism of the ABA responsiveness of <italic> <span class="gene">GmTIP2;3</span> </italic> should be further examined. Here, we can hypothesize that the plants first reduce the water hole number or close water channels to reduce the loss under stress by decreasing the transcription level of <italic> <span class="gene">GmTIP2;3</span> </italic> , and then when the plants have adapted to the stress environment, the expression of <italic> <span class="gene">GmTIP2;3</span> </italic> recovers to its original level and continues to increase its transcript abundance to respond to stressed conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						The plant response to <span class="trait">drought</span> is dependent on the SPAC (Soil-Plant-Air-Continuum). Root absorption and soil play important roles in plant adaption to <span class="trait">drought</span> stress (Shao et al., <xref rid="B27" ref-type="bibr"> 2009 </xref> ). Higher expression of aquaporin proteins in plants can allow them to effectively absorb water from the soil using the roots and then transport water by the stem to other organs, such as the leaf, flower, and seed, especially under osmotic stress (Devi et al., <xref rid="B8" ref-type="bibr"> 2015 </xref> ; Ding et al., <xref rid="B9" ref-type="bibr"> 2015 </xref> ; Miniussi et al., <xref rid="B18" ref-type="bibr"> 2015 </xref> ; Olaetxea et al., <xref rid="B19" ref-type="bibr"> 2015 </xref> ). Here, the higher expression of <italic> <span class="gene">GmTIP2;3</span> </italic> in the steles of the root and stem might promote and speed up water transportation from the roots to other organs under osmotic stress, improving plant tolerance to osmotic stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Azad et al. ( <xref rid="B2" ref-type="bibr"> 2009 </xref> ) analyzed water channels by yeast heterologous expression of tulip petal plasma membrane aquaporins from <italic> Pichia pastoris </italic> and monitored their water channel activity (WCA) by <italic> in vivo </italic> spheroplast-bursting and hypo-osmotic shock assays, suggesting that <italic> P. pastoris </italic> can be employed as a heterologous expression system to assay the WCA and to monitor the AQP-mediating channel gating mechanism of aquaporins. The yeast heterologous expression assay in this study showed that <italic> <span class="gene">GmTIP2;3</span> </italic> could effectively improve the tolerance of yeast to <span class="trait">drought</span> stress. Previously, we performed this assay using salinity and <span class="trait">drought</span> treatments simultaneously, but the results indicated that yeast cells expressing <italic> <span class="gene">GmTIP2;3</span> </italic> did not show improved survival rates under salinity stress, implying that <italic> <span class="gene">GmTIP2;</span> </italic> 3 had the ability to transport water but not ions.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:The interaction of salicylic acid and Ca(2+) alleviates aluminum toxicity in soybean (Glycine max L.).</h2>
			<h3>PMID:26691059  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lan T(1), You J(1), Kong L(1), Yu M(1), Liu M(1), Yang Z(2).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Both calcium ion (Ca(2+)) and salicylic acid (SA) influence various stress responses in plants. In acidic soils, aluminum (Al) toxicity adversely affects crop <span class="trait">yield.</span> In this study, we determined the influences of Ca(2+) and SA on root elongation, Al accumulation, and citrate secretion in soybean plant. We also investigated the activity of antioxidative enzymes in Al-exposed soybean roots. Root elongation was severally inhibited when the roots were exposed to 30 μM Al. The Al-induced inhibition of root elongation was ameliorated by Ca(2+) and SA but aggravated by Ca(2+) channel inhibitor (VP), <span class="gene">CaM</span> antagonists (TFP), Ca(2+) chelator (EGTA), and SA biosynthesis inhibitor (PAC). Furthermore, 1.0 mM CaCl2 and 10 μM SA reduced the accumulation of Al in roots, but their inhibitors stimulated the accumulation of Al in roots. Citrate secretion from these roots increased with the addition of either 1.0 mM CaCl2 or 10 μM SA but did not increase significantly when treated with higher Ca(2+) concentration. Enzymatic analysis showed that Ca(2+) and SA stimulated the activities of superoxidase (SOD), peroxidase (POD), and ascorbate peroxidase (APX) in Al-treated roots. In addition, SA restored the inhibition of Ca(2+) inhibitors on root elongation and Al content. Thus, both Ca(2+) and SA contribute to Al tolerance in soybean. Furthermore, Ca(2+) supplements rapidly increased Al-induced accumulation of free-SA or conjugated SA (SAG), while Ca(2+) inhibitors delayed the accumulation of SA for more than 8 h. Within 4 h of treatment, SA increased cytosolic Ca(2+) concentration in Al-treated roots, and upregulated the expression of four genes that possibly encode calmodulin-like (CML) proteins. These findings indicate that SA is involved in Ca(2+)-mediated signal transduction pathways in Al tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genomic-assisted phylogenetic analysis and marker development for next generation soybean cyst nematode <span class="trait">resistance</span> breeding.</h2>
			<h3>PMID:26566850  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kadam S(1), Vuong TD(2), Qiu D(3), Meinhardt CG(4), Song L(5), Deshmukh R(6), Patil G(7), Wan J(8), Valliyodan B(9), Scaboo AM(10), Shannon JG(11), Nguyen HT(12).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode (SCN, Heterodera glycines Ichinohe) is a serious soybean pest. The use of resistant cultivars is an effective approach for preventing <span class="trait">yield</span> loss. In this study, 19,652 publicly available soybean accessions that were previously genotyped with the SoySNP50K iSelect BeadChip were used to evaluate the phylogenetic diversity of SCN <span class="trait">resistance</span> genes <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> in an attempt to identify novel sources of <span class="trait">resistance.</span> The sequence information of soybean lines was utilized to develop KASPar (KBioscience Competitive Allele-Specific PCR) assays from single nucleotide polymorphisms (SNPs) of Rhg1, Rhg4, and other novel quantitative trait loci (QTL). These markers were used to genotype a diverse set of 95 soybean germplasm lines and three recombinant inbred line (RIL) populations. SNP markers from the <span class="gene">Rhg1</span> gene were able to differentiate copy number variation (CNV), such as resistant-high copy (PI 88788-type), low copy (Peking-type), and susceptible-single copy (Williams 82) numbers. Similarly, markers for the <span class="gene">Rhg4</span> gene were able to detect Peking-type <span class="trait">(resistance)</span> genotypes. The phylogenetic information of SCN <span class="trait">resistance</span> loci from a large set of soybean accessions and the gene/QTL specific markers that were developed in this study will accelerate SCN <span class="trait">resistance</span> breeding programs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide association study, genomic prediction and marker-assisted selection for seed weight in soybean ( <italic> Glycine </italic> <italic> max </italic> )</h2>
			<h3>PMID:26518570</h3>
				

				
					<h3>PMCID:4703630</h3>
					<h4>期刊:TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jiaoping </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Jiang </surname>  <given-names> Guo-Liang </given-names>  </name>  <address>  <email> gjiang@vsu.edu </email>  <email> gljiang99@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <aff id="Aff1">  <label/> Plant Science Department, South Dakota State University, Brookings, SD 57006 USA  </aff>  <aff id="Aff2">  <label/> Soybean Genomics and Improvement Laboratory, US Department of Agriculture, Agricultural Research Service (USDA-ARS), 10300 Baltimore Ave, Beltsville, MD 20705 USA  </aff>  <aff id="Aff3">  <label/> Agricultural Research Station, Virginia State University, Carter G. Woodson Ave, P.O. Box 9061, Petersburg, VA 23806 USA  </aff>
				<br/> 
				年份：<day> 29 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: Twenty-two loci for soybean SW and candidate genes conditioning seed development were identified; and prediction accuracies of <span class="gene">GS</span> and MAS were estimated through cross-validation and validation with unrelated populations. Soybean (Glycine max) is a major crop for plant protein and oil production, and seed weight (SW) is important for <span class="trait">yield</span> and quality in food/vegetable uses of soybean. However, our knowledge of genes controlling SW remains limited. To better understand the molecular mechanism underlying the trait and explore marker-based breeding approaches, we conducted a genome-wide association study in a population of 309 soybean germplasm accessions using 31,045 single nucleotide polymorphisms (SNPs), and estimated the prediction accuracy of genomic selection (GS) and marker-assisted selection (MAS) for SW. Twenty-two loci of minor effect associated with SW were identified, including hotspots on <span class="gene">Gm04</span> and <span class="gene">Gm19.</span> The mixed model containing these loci explained 83.4% of phenotypic variation. Candidate genes with Arabidopsis orthologs conditioning SW were also proposed. The prediction accuracies of <span class="gene">GS</span> and MAS by cross-validation were 0.75-0.87 and 0.62-0.75, respectively, depending on the number of SNPs used and the size of training population. <span class="gene">GS</span> also outperformed MAS when the validation was performed using unrelated panels across a wide range of maturities, with an average prediction accuracy of 0.74 versus 0.53. This study convincingly demonstrated that soybean SW is controlled by numerous minor-effect loci. It greatly enhances our understanding of the genetic basis of SW in soybean and facilitates the identification of genes controlling the trait. It also suggests that <span class="gene">GS</span> holds promise for accelerating soybean breeding progress. The results are helpful for genetic improvement and genomic prediction of <span class="trait">yield</span> in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> ) is a major crop for plant protein and oil production, and seed weight (SW) is important for <span class="trait">yield</span> and quality in food/vegetable uses of soybean. However, our knowledge of genes controlling SW remains limited. To better understand the molecular mechanism underlying the trait and explore marker-based breeding approaches, we conducted a genome-wide association study in a population of 309 soybean germplasm accessions using 31,045 single nucleotide polymorphisms (SNPs), and estimated the prediction accuracy of genomic selection (GS) and marker-assisted selection (MAS) for SW. Twenty-two loci of minor effect associated with SW were identified, including hotspots on <span class="gene">Gm04</span> and <span class="gene">Gm19.</span> The mixed model containing these loci explained 83.4&#x000a0;% of phenotypic variation. Candidate genes with <italic> Arabidopsis </italic> orthologs conditioning SW were also proposed. The prediction accuracies of <span class="gene">GS</span> and MAS by cross-validation were 0.75&#x02013;0.87 and 0.62&#x02013;0.75, respectively, depending on the number of SNPs used and the size of training population. <span class="gene">GS</span> also outperformed MAS when the validation was performed using unrelated panels across a wide range of maturities, with an average prediction accuracy of 0.74 versus 0.53. This study convincingly demonstrated that soybean SW is controlled by numerous minor-effect loci. It greatly enhances our understanding of the genetic basis of SW in soybean and facilitates the identification of genes controlling the trait. It also suggests that <span class="gene">GS</span> holds promise for accelerating soybean breeding progress. The results are helpful for genetic improvement and genomic prediction of <span class="trait">yield</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						With the decreased genotyping cost and improved statistical methods, genome-wide association study (GWAS) and genomic selection (GS) present promising prospects for genetic improvement of complex traits in crop species. GWAS with a population of unrelated lines and high-density single nucleotide polymorphism (SNP) markers is capable of identifying causal genes for a broad range of complex traits in different crops (Huang et al. <xref ref-type="bibr" rid="CR21"> 2010 </xref> ; Li et al. <xref ref-type="bibr" rid="CR26"> 2013 </xref> ; Morris et al. <xref ref-type="bibr" rid="CR32"> 2013 </xref> ). In soybean, it has been used to disclose the genetic architecture of agronomic traits (Zhang et al. <xref ref-type="bibr" rid="CR60"> 2015 </xref> ), seed composition (Hwang et al. <xref ref-type="bibr" rid="CR22"> 2014 </xref> ) and disease <span class="trait">resistance</span> (Wen et al. <xref ref-type="bibr" rid="CR55"> 2014 </xref> ). <span class="gene">GS</span> refers to marker-based selection by capturing the total genetic variance with genome-wide markers without identifying a subset of trait-associated markers (Meuwissen et al. <xref ref-type="bibr" rid="CR31"> 2001 </xref> ). In general, <span class="gene">GS</span> is expected to be more effective than MAS that based on a few loci for quantitatively inherited traits (Bernardo and Yu <xref ref-type="bibr" rid="CR6"> 2007 </xref> ). A large number of <span class="gene">GS</span> studies have been reported in crop species such as maize (Albrecht et al. <xref ref-type="bibr" rid="CR1"> 2011 </xref> ; Bernardo <xref ref-type="bibr" rid="CR5"> 1996 </xref> ; Piepho <xref ref-type="bibr" rid="CR37"> 2009 </xref> ; Technow et al. <xref ref-type="bibr" rid="CR52"> 2013 </xref> ) and wheat (Heffner et al. <xref ref-type="bibr" rid="CR18"> 2011 </xref> ; Poland et al. <xref ref-type="bibr" rid="CR38"> 2012 </xref> ; Rutkoski et al. <xref ref-type="bibr" rid="CR44"> 2011 </xref> ) for various agronomic traits and disease <span class="trait">resistance.</span> However, its application to soybean is rarely addressed (Bao et al. <xref ref-type="bibr" rid="CR3"> 2014 </xref> ; Jarquin et al. <xref ref-type="bibr" rid="CR23"> 2014 </xref> ), and validation with unrelated population has not been reported.
					</p>
				
					<p>&nbsp;&nbsp;
						Therefore, the objectives of this study were to (1) better understand the genetic architecture underlying SW in soybean, and (2) explore the potential of marker-based prediction as a new approach in soybean breeding. Twenty-two loci associated with SW and putative candidate genes with <italic> Arabidopsis </italic> orthologs involved in seed mass determination were identified via GWAS. <span class="gene">GS</span> exhibited higher prediction accuracies than MAS in all the tests of both cross-validation and validation with unrelated panels obtained from GRIN ( <ext-link ext-link-type="uri" xlink:href="http://www.ars-grin.gov/"> http://www.ars-grin.gov/ </ext-link> ). This study enhances our understanding of the genetic architecture of SW and expedites the identification of genes conditioning SW in soybean. It demonstrates that <span class="gene">GS</span> can increase breeding efficiency and is also useful for genomic prediction of <span class="trait">yield</span> in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean, more than 200 QTLs for SW have been reported across 20 chromosomes (SoyBase, <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org/"> http://www.soybase.org/ </ext-link> ). Additionally, many QTLs associated with seed size, a trait highly related to SW, have also been identified. These QTLs could be used to confirm the loci identified via GWAS, especially the hot ones. In this study, 22 loci associated with SW were identified. Each individual locus could explain a small proportion (&#x0003c;4&#x000a0;%) of phenotypic variance. The result demonstrated that soybean seed weight is a typical quantitative trait, genetically conditioned by many minor-effect loci. Of the identified loci, 15 have been previously reported at least once (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). Among these loci, the SW11 locus at 37.1&#x000a0;Mb position on <span class="gene">Gm10</span> exhibited the strongest association. In the similar region, four SW QTLs have been reported previously. Another locus SW2 at the 14.2&#x000a0;Mb position on <span class="gene">Gm04</span> has been reported to be associated with SW and seed size related traits, e.g. seed length, height and width, two and three times, respectively (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). The SW21 on <span class="gene">Gm19</span> was mapped within an overlapped region of six SW QTLs and six QTLs for seed size related traits reported previously, indicating that SW21 might be in one of the hottest regions related to SW in soybean (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). The high repeatability of these loci across various environments and genetic backgrounds implies a great potential of marker-based breeding for SW in soybean. As SW plays an important role in soybean yielding, these loci are also useful for soybean <span class="trait">yield</span> improvement.
					</p>
				
					<p>&nbsp;&nbsp;
						Compared with a previous <span class="gene">GS</span> study in soybean, the prediction accuracy of <span class="gene">GS</span> in the present study was relatively high. Jarquin et al. ( <xref ref-type="bibr" rid="CR23"> 2014 </xref> ) conducted <span class="gene">GS</span> for grain <span class="trait">yield,</span> plant height and maturity date in a population of 301 elite soybean breeding lines with 52,349 SNPs. The highest estimate of broad-sense heritability was 0.94 for maturity date, which is similar to the SW (0.97) in this study. In that study, however, the highest heritability-adjusted prediction accuracy was 0.70 for maturity date with the optimal prediction model, which is lower than the highest prediction accuracy of 0.83 in the present study. Because the two studies had similar population size, number of SNPs and trait heritability, the difference in genetic architecture of traits and populations under study might be the major factors leading to the difference in prediction accuracies between the two studies.
					</p>
				
					<p>&nbsp;&nbsp;
						In most of the <span class="gene">GS</span> or MAS studies, the training and validation populations are related, and cross-validation is used to estimate the prediction accuracy (Bao et al. <xref ref-type="bibr" rid="CR3"> 2014 </xref> ; Hoeck et al. <xref ref-type="bibr" rid="CR20"> 2003 </xref> ; Jarquin et al. <xref ref-type="bibr" rid="CR23"> 2014 </xref> ). However, their selection efficiency for unrelated lines is usually undetermined. In the present study, besides cross-validation, validation with unrelated populations obtained from GRIN ( <ext-link ext-link-type="uri" xlink:href="http://www.ars-grin.gov/"> http://www.ars-grin.gov/ </ext-link> ) was also carried out. We found that the prediction accuracy of <span class="gene">GS</span> varied among the panels with different groups of maturity (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> c). This phenomenon might be caused by the different degrees of similarity in genetic compositions between the training panel and the validation sets. We noted that the prediction accuracies (0.74&#x02013;0.83) for the early and medium maturity validation panels (MG 000-IV) were higher than that (0.63) of the late maturity panel (MG V-VIII). The high prediction accuracy of <span class="gene">GS</span> across a wide range of maturities may be benefitted from the broad genetic diversity of the association population and the dense SNPs used in the study. The latter helps to maximize the LD between markers and causal genetic variants and also increases the potential of capturing the genetic relatedness between training and test sets (Gowda et al. <xref ref-type="bibr" rid="CR15"> 2014 </xref> ; Habier et al. <xref ref-type="bibr" rid="CR16"> 2007 </xref> ). According to the information from USDA ( <ext-link ext-link-type="uri" xlink:href="http://www.usda.gov/"> http://www.usda.gov/ </ext-link> ), soybean cultivars with MG 000-IV accounted for over three-fourths of the U.S. soybean production in 2013. Therefore, <span class="gene">GS</span> with the marker effects estimated in this study hold a great potential in predicting soybean SW. Given SW is an important component of grain <span class="trait">yield,</span> the results are also useful for the genomic prediction and genetic improvement of soybean <span class="trait">yield.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, we identified 22 loci associated with SW via GWAS and thus convincingly demonstrated that soybean SW is conditioned by numerous loci of minor effect. We also refined candidate chromosomal regions for the known QTLs, including the hotspots on <span class="gene">Gm04</span> and <span class="gene">Gm19.</span> Candidate genes with <italic> Arabidopsis </italic> homologs involved in seed development, auxin and cytokinin signalings controlling SW were proposed. Cross-validation showed that MAS could be a competitive alternative of <span class="gene">GS</span> when training and validation population were related or had similar genetic components. However, validation with unrelated populations across a broad range of maturities suggested that <span class="gene">GS</span> is superior to MAS. This study enhances our knowledge of the genetic basis of SW in soybean, and facilitates exploring the molecular mechanisms underlying the trait. It also benefits the genomic prediction of <span class="trait">yield</span> in soybean and suggests that <span class="gene">GS</span> would have a great potential in increasing genetic gains of soybean breeding.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Arabidopsis KLU homologue <span class="gene">GmCYP78A72</span> regulates seed size in soybean.</h2>
			<h3>PMID:26482479  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao B(1)(2), Dai A(2), Wei H(2), Yang S(3), Wang B(1), Jiang N(4), Feng X(5)(6).
				<br/> 
				年份：2016 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is one of the most important crops in the world, and its <span class="trait">yield</span> is largely determined by grain weight and grain size. However, the genes that regulate soybean seed size have not been identified. CYP78A, which is highly conserved within terrestrial plants, regulates organ development. In Arabidopsis, AtCYP78A5/KLU has been shown to determine seed size. In the present study, soybean CYP78A72 <span class="gene">(GmCYP78A72),</span> one of the orthologs of KLU, was over-expressed in both Arabidopsis and soybean to examine its function in plant development. <span class="gene">GmCYP78A72</span> heterologous expression in Arabidopsis resulted in enlarged sepals, petals, seeds and carpel. Over-expression of <span class="gene">GmCYP78A72</span> in soybean resulted in increased pea size, which is an extremely desirable trait for enhancing productivity. Moreover, knock-down of <span class="gene">GmCYP78A72</span> does not reduce grain size. However, silencing of <span class="gene">GmCYP78A57,</span> <span class="gene">GmCYP78A70</span> and <span class="gene">GmCYP78A72</span> genes in triplet reduces the seed size significantly indicating functional redundancy of these three <span class="gene">GmCYP78A</span> genes. In conclusion, we investigated the role of CYP78A in soybean seed regulation, and our strategy can be effectively used to engineer large seed traits in soybean varieties as well as other crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Artificial Intelligence versus Statistical Modeling and Optimization of Cholesterol Oxidase Production by using <italic> Streptomyces </italic> Sp.</h2>
			<h3>PMID:26368924</h3>
				

				
					<h3>PMCID:4569268</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Pathak </surname>  <given-names> Lakshmi </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Singh </surname>  <given-names> Vineeta </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Niwas </surname>  <given-names> Ram </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Osama </surname>  <given-names> Khwaja </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Saif </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Haque </surname>  <given-names> Shafiul </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tripathi </surname>  <given-names> C. K. M. </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mishra </surname>  <given-names> B. N. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Cholesterol oxidase (COD) is a bi-functional FAD-containing oxidoreductase which catalyzes the oxidation of cholesterol into 4-cholesten-3-one. The wider biological functions and clinical applications of COD have urged the screening, isolation and characterization of newer microbes from diverse habitats as a source of COD and optimization and over-production of COD for various uses. The practicability of statistical/ artificial intelligence techniques, such as response surface methodology (RSM), artificial neural network (ANN) and genetic algorithm (GA) have been tested to optimize the medium composition for the production of COD from novel strain Streptomyces sp. NCIM 5500. All experiments were performed according to the five factor central composite design (CCD) and the generated data was analysed using RSM and ANN. GA was employed to optimize the models generated by RSM and ANN. Based upon the predicted COD concentration, the model developed with <span class="gene">ANN</span> was found to be superior to the model developed with RSM. The RSM-GA approach predicted maximum of 6.283 U/mL COD production, whereas the ANN-GA approach predicted a maximum of 9.93 U/mL COD concentration. The optimum concentrations of the medium variables predicted through ANN-GA approach were: 1.431 g/50 mL soybean, 1.389 g/50 mL maltose, 0.029 g/50 mL MgSO4, 0.45 g/50 mL NaCl and 2.235 ml/50 mL glycerol. The experimental COD concentration was concurrent with the GA predicted <span class="trait">yield</span> and led to 9.75 U/mL COD production, which was nearly two times higher than the <span class="trait">yield</span> (4.2 U/mL) obtained with the un-optimized medium. This is the very first time we are reporting the statistical versus artificial intelligence based modeling and optimization of COD production by Streptomyces sp. NCIM 5500.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The COD production (in terms of enzyme activity) in an un-optimized medium was 4.2 U/ml which was significantly increased to 6.04 U/mL by employing RSM coupled with GA. Whereas <span class="gene">ANN</span> coupled with GA resulted in further enhancement in COD concentration (9.75 U/ml,) which was nearly 2.32 folds higher than the <span class="trait">yield</span> obtained with un-optimized production medium. A combinatorial method using RSM coupled with GA has been successfully used to solve the problems associated with process optimization [ <xref rid="pone.0137268.ref032" ref-type="bibr"> 32 </xref> , <xref rid="pone.0137268.ref033" ref-type="bibr"> 33 </xref> ]. Chauhan et al. (2009) reported 2.48 folds increase in COD productivity from <italic> S </italic> . <italic> lavendulae </italic> by using statistical approaches [ <xref rid="pone.0137268.ref034" ref-type="bibr"> 34 </xref> ]. Five medium components <italic> viz </italic> . soybean, glycerol, maltose, MgSO <sub> 4 </sub> and NaCl found important and were studied for the optimization of COD production. The results of the effect of individual medium component on COD activity correlated to the role of those components for COD production. Glycerol and maltose showed positive effect on COD production. Earlier study also reported that glycerol supports COD production in <italic> S </italic> . <italic> lavendulae </italic> [ <xref rid="pone.0137268.ref034" ref-type="bibr"> 34 </xref> ]. Soybean meal is a complex nitrogen source and contains amino acids, carbohydrates and also includes fatty acids [ <xref rid="pone.0137268.ref004" ref-type="bibr"> 4 </xref> , <xref rid="pone.0137268.ref034" ref-type="bibr"> 34 </xref> ], which enhance the enzyme (COD) production [ <xref rid="pone.0137268.ref034" ref-type="bibr"> 34 </xref> ]. Here, in this study, during the experiments with <italic> Streptomyces </italic> sp. NCIM 5500, MgSO <sub> 4 </sub> was found to be more effective than NaCl for COD activity, which is in contrast to the previous report of Amiri et al. (2008), where they reported NaCl favors COD production than MgSO <sub> 4 </sub> [ <xref rid="pone.0137268.ref033" ref-type="bibr"> 33 </xref> ]. However, other reports support the use of both the salts in the production medium [ <xref rid="pone.0137268.ref034" ref-type="bibr"> 34 </xref> ]. It was evident from linear and quadratic effect that higher concentration of MgSO <sub> 4 </sub> and lower concentration of NaCl is responsible for greater enzyme production. On the contrary to NaCl supplementation in the production medium for COD production, plethora of reports suggests the use of MgSO <sub> 4 </sub> for stabilization or even enhancement of COD activity [ <xref rid="pone.0137268.ref011" ref-type="bibr"> 11 </xref> , <xref rid="pone.0137268.ref034" ref-type="bibr"> 34 </xref> , <xref rid="pone.0137268.ref035" ref-type="bibr"> 35 </xref> ]. Also, El- Shoraet al. (2011) reported that COD production activates by Mg <sup> 2+ </sup> ions in case of <italic> Staphylococcus epidermidis </italic> [ <xref rid="pone.0137268.ref035" ref-type="bibr"> 35 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, RSM and <span class="gene">ANN</span> were used along with CCD to derive a model for interaction effects of medium components (i.e., soybean meal, glycerol, maltose, NaCl and MgSO <sub> 4 </sub> ) on COD production. Further GA was employed to optimize the RSM/ANN models. The media composition obtained by optimizing both of the models resulted in higher COD concentration than the <span class="trait">yield</span> recovered through un-optimized media. This hybrid methodology, i.e., coupling of <span class="gene">ANN</span> with GA was found to improve COD production significantly (nearly 2 folds) and proved better than RSM, as the model developed through <span class="gene">ANN</span> was found to give nearly 60% higher COD concentration than the <span class="trait">yield</span> predicted by RSM generated model. The combinatorial approach (coupling of <span class="gene">ANN</span> with GA) presented in this study is sufficiently general and thus can also be successfully employed for the optimization of various parameters used in other bioprocesses. Overall, the higher COD concentration achieved in this study through <span class="gene">ANN</span> coupled with GA approach will paves the way for future studies for the production of COD at commercial scale using <italic> Streptomyces sp </italic> . NICM 5500 as well as implication of other/ combination of artificial intelligence techniques for higher and sustainable production of COD.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:GmmiR156b overexpression delays flowering time in soybean.</h2>
			<h3>PMID:26341865  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cao D(1), Li Y(2), Wang J(1), Nan H(1), Wang Y(3), Lu S(1)(4), Jiang Q(3), Li X(1)(4), Shi D(1)(4), Fang C(1)(4), Yuan X(1), Zhao X(1)(4), Li X(5), Liu B(6), Kong F(7).
				<br/> 
				年份：2015 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merr.] is an important crop used for human consumption, animal feed and biodiesel fuel. Wering time and maturity significantly affect soybean grain <span class="trait">yield.</span> In Arabidopsis thaliana, miR156 has been proposed to regulate the transition from the juvenile to the adult phase of shoot development, which is accompanied by changes in vegetative morphology and an increase in reproductive potential. However, the molecular mechanisms underlying miR156 function in soybean flowering remain unknown. Here, we report that the overexpression of <span class="gene">GmmiR156b</span> delays flowering time in soybean. <span class="gene">GmmiR156b</span> may target SPL orthologs and negatively regulate <span class="gene">GmSPLs,</span> thereby delaying flowering in soybean under LD and natural conditions. <span class="gene">GmmiR156b</span> down-regulates several known flowering time regulators in soybean, such as <span class="gene">GmAP1</span> (a, b, c), <span class="gene">GmLFY2,</span> <span class="gene">GmLFY2,</span> <span class="gene">GmFULs,</span> <span class="gene">GmSOC1s,</span> <span class="gene">GmFT5a,</span> and <span class="gene">GmmiR172.</span> These data show that a similar miR156-SPL regulatory module was conserved in the soybean flowering pathway. However, <span class="gene">GmFULs,</span> <span class="gene">GmSOC1a</span> and <span class="gene">GmSOC1b</span> were significantly suppressed under LD conditions but not under SD conditions, which is different in Arabidopsis that these genes were down-regulated irrespective of photoperiod. In addition, <span class="gene">GmmiR156b</span> was up-regulated by E1, E2 <span class="gene">(GmGI),</span> E3 and E4, which control flowering time and maturity in soybean, and suppressed <span class="gene">E1</span> (E1-Like) and E2 (E2-Like) genes under LD conditions. These data indicated that the miR156-SPL regulatory module was also with some degree of divergent in soybean flowering pathway.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic characteristics of soybean <span class="trait">resistance</span> to HG type 0 and HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping</h2>
			<h3>PMID:26268218</h3>
				

				
					<h3>PMCID:4542112</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Han </surname>  <given-names> Yingpeng </given-names>  </name>  <address>  <email> hyp234286@aliyun.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhao </surname>  <given-names> Xue </given-names>  </name>  <address>  <email> xuezhao@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Cao </surname>  <given-names> Guanglu </given-names>  </name>  <address>  <email> 307920981@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yan </given-names>  </name>  <address>  <email> wy11544@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <address>  <email> liyinghui@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dongyuan </given-names>  </name>  <address>  <email> liudy@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Teng </surname>  <given-names> Weili </given-names>  </name>  <address>  <email> twlneau@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Zhiwu </given-names>  </name>  <address>  <email> zhiwu.zhang@wsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Dongmei </given-names>  </name>  <address>  <email> 278515240@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zheng </surname>  <given-names> Hongkun </given-names>  </name>  <address>  <email> zhenghk@biomarker.com.cn </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <address>  <email> wenbinli@neau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Key Laboratory of Soybean Biology in Chinese Ministry of Education (key Laboratory of Soybean Biology and Breeding/Genetics of Chinese Agriculture Ministry), Northeast Agricultural University, Harbin, 150030 China  </aff>  <aff id="Aff2">  <label/> Institute of Crop Science, National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI) Chinese Academy of Agricultural Sciences, 100081 Beijing, China  </aff>  <aff id="Aff3">  <label/> Bioinformatics Division, Biomarker Technologies Corporation, 101300 Beijing, China  </aff>
				<br/> 
				年份：<day> 27 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean cyst nematode (SCN, Heterodera glycines Ichinohe) is one of the most fatal pests of soybean (Glycine max (L.) Merr.) worldwide and causes huge loss of soybean <span class="trait">yield</span> each year. Multiple sources of <span class="trait">resistance</span> are urgently needed for effective management of SCN via the development of resistant cultivars. The aim of the present study was to investigate the genetic architecture of <span class="trait">resistance</span> to SCN HG Type 0 (race 3) and HG Type 1.2.3.5.7 (race 4) in landraces and released elite soybean cultivars mostly from China. RESULTS: A total of 440 diverse soybean landraces and elite cultivars were screened for <span class="trait">resistance</span> to SCN HG Type 0 and HG Type 1.2.3.5.7. Exactly 131 new sources of SCN <span class="trait">resistance</span> were identified. Lines were genotyped by SNP markers detected by the Specific Locus Amplified Fragment Sequencing (SLAF-seq) approach. A total of 36,976 SNPs were identified with minor allele frequencies (MAF) &gt; 4% that were present in 97% of all the genotypes. Genome-wide association mapping showed that a total of 19 association signals were significantly related to the <span class="trait">resistance</span> for the two HG Types. Of the 19 association signals, eight signals overlapped with reported QTL including <span class="gene">Rhg1</span> and <span class="gene">Rhg4</span> genes. Another eight were located in the linked regions encompassing known QTL. Three QTL were found that were not previously reported. The average value of female index (FI) of soybean accessions with resistant alleles was significantly lower than those with susceptible alleles for each peak SNP. Disease <span class="trait">resistance</span> proteins with leucine rich regions, cytochrome P450s, protein kinases, zinc finger domain proteins, RING domain proteins, MYB and WRKY transcription activation families were identified. Such proteins may participate in the resistant reaction to SCN and were frequently found in the tightly linked genomic regions of the peak SNPs. CONCLUSIONS: GWAS extended understanding of the genetic architecture of SCN <span class="trait">resistance</span> in multiple genetic backgrounds. Nineteen association signals were obtained for the <span class="trait">resistance</span> to the two Hg Types of SCN. The multiple beneficial alleles from resistant germplasm sources will be useful for the breeding of cultivars with improved <span class="trait">resistance</span> to SCN. Analysis of genes near association signals may facilitate the recognition of the causal gene(s) underlying SCN resistances.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A total of 440 diverse soybean landraces and elite cultivars were screened for <span class="trait">resistance</span> to SCN HG Type 0 and HG Type 1.2.3.5.7. Exactly 131 new sources of SCN <span class="trait">resistance</span> were identified. Lines were genotyped by SNP markers detected by the Specific Locus Amplified Fragment Sequencing (SLAF-seq) approach. A total of 36,976 SNPs were identified with minor allele frequencies (MAF)&#x02009;&#x0003e;&#x02009;4&#x000a0;% that were present in 97&#x000a0;% of all the genotypes. Genome-wide association mapping showed that a total of 19 association signals were significantly related to the <span class="trait">resistance</span> for the two HG Types. Of the 19 association signals, eight signals overlapped with reported QTL including <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> genes. Another eight were located in the linked regions encompassing known QTL. Three QTL were found that were not previously reported. The average value of female index (FI) of soybean accessions with resistant alleles was significantly lower than those with susceptible alleles for each peak SNP. Disease <span class="trait">resistance</span> proteins with leucine rich regions, cytochrome P450s, protein kinases, zinc finger domain proteins, RING domain proteins, MYB and WRKY transcription activation families were identified. Such proteins may participate in the resistant reaction to SCN and were frequently found in the tightly linked genomic regions of the peak SNPs.
					</p>
				
					<p>&nbsp;&nbsp;
						The knowledge of genetic architecture of SCN <span class="trait">resistance</span> is very important for breeding <span class="trait">resistance</span> varieties [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ]. So far, genetic architecture analyses mainly depended on traditional quantitative trait locus (QTL) linkage mapping based on bi-parental populations. Once the desired QTL were mapped, molecular markers that were tightly linked to the QTL could be applied in marker-assisted breeding to improve and shorten the process for developing cultivars with <span class="trait">resistance</span> to SCN. Many QTL conferring <span class="trait">resistance</span> to SCN in soybean have been mapped on almost all chromosomes (except of <span class="gene">Gm02;</span> LG D1b) based on SSR markers [ <xref ref-type="bibr" rid="CR12"> 12 </xref> &#x02013; <xref ref-type="bibr" rid="CR19"> 19 </xref> ]. Of them, two major QTL, <italic> <span class="gene">Rhg1</span> </italic> on <span class="gene">Gm18</span> and <italic> <span class="gene">Rhg4</span> </italic> on <span class="gene">Gm08</span> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ], were identified in many different <span class="trait">resistance</span> sources. Genes and their alleles underlying both loci were identified from PI88788 and cultivar Forrest, respectively [ <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Fine mapping, sequencing and gene analysis of <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> were reported [ <xref ref-type="bibr" rid="CR19"> 19 </xref> &#x02013; <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. The receptor like kinase at <italic> Rhg1-a </italic> caused pleiotropic <span class="trait">resistance</span> to sudden death syndrome and soybean cyst nematode as a transgene by altering signaling responses [ <xref ref-type="bibr" rid="CR23"> 23 </xref> , <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. The study on <italic> Rhg1-b </italic> allele&#x02019;s locus by Cook <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ] considered the SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg1</span> </italic> in PI 88788 was conferred by three genes, and the copy number variation of these genes caused the phenotypic differences between susceptible and resistant varieties. The SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg4</span> </italic> in Forrest was controlled by a serine hydroxymethyl transferase gene, that acted in cellular one-carbon metabolism [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. The mutation of this gene could disturb the concentration of folate and lead to a nutritional deficiency detrimental to the life cycle of SCN [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. Although few SCN <span class="trait">resistance</span> genes have been cloned based on the identified QTL, the large confidence genomic intervals of most QTL impaired the precise identification of causative genes.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic basis of natural variation for soybean <span class="trait">resistance</span> to SCN HG Type 0 and HG Type 1.2.3.5.7 were explored by using an association panel of 440 diverse germplasm genotyped with 36,976 SNP markers. A total of 19 association signals were identified with <italic> P </italic> &#x02009;&#x0003c;&#x02009;1.35&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;6 </sup> for the two traits in a genome wide scan under compressed mixed linear model that controlled the population structure and familial relatedness (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> and Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). Among the 19 association signals, eighteen were declared significant under the threshold of <italic> P </italic> &#x02009;&#x0003c;&#x02009;2.70&#x02009;&#x000d7;&#x02009;10 <sup> &#x02212;7 </sup> . The 19 peak SNPs explained 6.2&#x02013;15.4&#x000a0;% of the total phenotypic variation. Twelve signals were significantly associated with the <span class="trait">resistance</span> to Hg Type 0 and another seven signals were found to be related to the <span class="trait">resistance</span> to Hg Type 1,2,3,5,7. The known SCN resistant loci, <italic> <span class="gene">Rhg1</span> </italic> and <italic> <span class="gene">Rhg4</span> </italic> , were identified by the present study for soybean <span class="trait">resistance</span> to Hg Type 0. The <italic> <span class="gene">Rhg4</span> </italic> locus was also found to contribute <span class="trait">resistance</span> to Hg Type 1.2.3.5.7. The rs33704130 located on <span class="gene">Gm16</span> was significantly associated with the <span class="trait">resistance</span> for both Hg Types. It could be an important locus that offered <span class="trait">resistance</span> to multiple Hg Types. Eight signals overlapped with reported QTL, and another eight were located in the linked region of known QTL (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). Three QTL were newly found in the present study. <fig id="Fig4"> <label> Fig. 4 </label> <caption> <p> Genome wide association signals for soybean <span class="trait">resistance</span> to HG Type 0 and HG Type 1.2.3.5.7. <bold> a </bold> , <bold> c </bold> Genome wide Manhattan plots of associations from compressed mixed linear model. <italic> x </italic> axis showed the SNPs along each chromosome; <italic> y </italic> axis was the &#x02212;log 10 ( <italic> P </italic> -value) for the association for soybean <span class="trait">resistance</span> to Hg Type 0 and Hg Type 1.2.3.5.7, respectively. <bold> b </bold> , <bold> d </bold> Quantile-Quantile plots for mixed linear models of the <span class="trait">resistance</span> to Hg Type 0 and Hg Type 1.2.3.5.7
					</p>
				
					<p>&nbsp;&nbsp;
						The candidate genes related to the identified loci against SCN Hg Type 0 and Hg Type 1.2.3.5.7 was further evaluated. Gene models that located in 200 Kbp genomic region upstream and downstream of each peak SNP in the reference soybean genome (Wm82.a2.v1, see &#x02018;www. phetozome.net&#x02019;) were considered to be <span class="trait">resistance</span> gene candidates since the mean LD decay distance of soybean genome was around 200 Kbp according to the present study and the reports of Hwang <italic> et al. </italic> (~360 Kbp) and Lam <italic> et al. </italic> (~150Kbp) [ <xref ref-type="bibr" rid="CR37"> 37 </xref> , <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. A total of 395 soybean gene models were found in the flanking regions of each peak SNP. Of these gene models, 64 genes had no functional annotation. Fourteen genes belonged to the domains of unknown function families. Among the other 317 gene models, several genes encoded serine/threonine protein kinases known to be involved in plant disease response. Genes encoding leucine-rich repeat-containing proteins, which might function in plant disease <span class="trait">resistance</span> pathways in response to a variety of external stimuli from pathogens, were also identified. Five other domain types were common including cytochrome P450s, protein kinases, zinc fingers, RING domains, MYB and WRKY families (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). Of these gene families, there were five clusters of genes with disease <span class="trait">resistance</span> protein and/or LRR protein domains that were found in both side of rs33704130 on <span class="gene">Gm16</span> that was associated with the <span class="trait">resistance</span> to both Hg Type 0 and Hg Type 1.2.3.5.7. The Hg Type 0 <span class="trait">resistance</span> loci (rs46050545 on <span class="gene">Gm04,</span> rs33843704 on <span class="gene">Gm05,</span> rs33488383 on <span class="gene">Gm11</span> and rs43617172 on <span class="gene">Gm14),</span> as well as Hg Type 1.2.3.5.7 <span class="trait">resistance</span> locus (rs172388849 on <span class="gene">Gm17)</span> were tightly linked with the genes of RING/U-box superfamily protein or RING-H2 finger C1A gene. The distance of each RING gene to their corresponding peak SNP ranged from 8.9 Kbp to 161.7 Kbp with an average distance of 59.1 Kbp. It could be inferred that the RING/U-box superfamily protein might be involved in the disease <span class="trait">resistance</span> responsive to Hg Type 0 and Hg Type 1.2.3.5.7. <fig id="Fig5"> <label> Fig. 5 </label> <caption> <p> Main gene families in the flanking genomic regions of peak SNP
					</p>
				
					<p>&nbsp;&nbsp;
						Among the 12 QTN (nucleotides) associated with QTL underlying <span class="trait">resistance</span> to SCN Hg Type 0, one QTN (rs8050006) was identified on chromosome 8 (Gm 08, LG A2), where a known and major QTL region associated with <italic> <span class="gene">Rhg4</span> </italic> locus had been commonly reported before [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR47"> 47 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> &#x02013; <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. Concibido <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ] reported that a QTL near the <italic> <span class="gene">Rhg4</span> </italic> locus could explain 15&#x000a0;% of the total phenotypic variation in PI 209332 to SCN Hg Type 0. The <italic> <span class="gene">Rhg4</span> </italic> locus was confirmed as one of the major QTL controlling soybean <span class="trait">resistance</span> against SCN Hg Type 0 [ <xref ref-type="bibr" rid="CR50"> 50 </xref> , <xref ref-type="bibr" rid="CR53"> 53 </xref> ]. Moreover, Kazi <italic> et al. </italic> and Ferreira <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR49"> 49 </xref> , <xref ref-type="bibr" rid="CR54"> 54 </xref> ] confirmed the effect of <italic> <span class="gene">Rhg4</span> </italic> on <span class="trait">resistance</span> to race 3 and 14, while <italic> <span class="gene">Rhg4</span> </italic> was less associated with <span class="trait">resistance</span> to races 1, 2, 4 and 5. However, this locus was also detected to be related to the <span class="trait">resistance</span> to Hg Type 1.2.3.5.7 (race 4) in our study, indicating that it could underlie SCN <span class="trait">resistance</span> more than one races simultaneously and might facilitate to breed new soybean varieties resistant to SCN race 3, 4 and 14.
					</p>
				
					<p>&nbsp;&nbsp;
						In soybean, Gm 18 (LG G) was very important for SCN <span class="trait">resistance.</span> The <italic> rhg1 </italic> loci, located on Gm 18, had been conformed to have an impact on the depression of SCN development in several <span class="trait">resistance</span> lines and provided SCN <span class="trait">resistance</span> in many commercial varieties from several <span class="trait">resistance</span> sources [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR55"> 55 </xref> &#x02013; <xref ref-type="bibr" rid="CR57"> 57 </xref> ]. For example, Concibido <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR12"> 12 </xref> ] reported the recessive gene <italic> rhg 1 </italic> that located on Gm 18 was identified to resistant to SCN race 3 and 14 as a major SCN <span class="trait">resistance</span> source. Similarly, Wu <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ] also detected a major QTL near <italic> rhg1 </italic> with a relative large R <sup> 2 </sup> , resistant to SCN race 1, 2, 3, 5 and 14, respectively. Here, the SNP rs1643660, associated with the <span class="trait">resistance</span> to SCN Hg Type 0 (race 3), was detected to fall within the <italic> rhg1 </italic> locus and was located in the 31 Kbp copy number variation (CNV) region of <italic> <span class="gene">Rhg1</span> </italic> locus. The SNP marker rs1643660 was 5.1 Kpb, 4.6 Kbp and 5.1&#x000a0;K&#x000a0;bp distances from the three key genes ( <italic> Glyma.18&#x000a0;g022400, Glyma.18&#x000a0;g022500 </italic> ) in CNV region [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. However, the distance between SSR marker BARCSOYSSR_18_0090 and <italic> rhg 1 </italic> -b was 67 Kbp in a fine mapping study of <italic> rhg 1 </italic> locus that was reported by Kim <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ]. The distance from marker to causal genes in our study was much shorter than that reported by Kim <italic> et al., </italic> indicating that better mapping result could be available from GWAS using SNP markers. The locus rs1643660 could be more effective for marker-assisted selection of lines resistant to SCN.
					</p>
				
					<p>&nbsp;&nbsp;
						In the present study, rs2085816 on Gm 20 (LG I), rs21804864 on <span class="gene">Gm02</span> (LG D1b) and rs30581306 on <span class="gene">Gm14</span> (LG B2) were the novel loci reported for SCN <span class="trait">resistance.</span> They might have the potential to be used in MAS for enhanced SCN Hg Type 0 and Hg Type 1.2.3.5.7 <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Here, we mapped 19 QTL representing 17 genomic regions associated with soybean <span class="trait">resistance</span> to Hg Type 0 and/or Hg Type 1.2.3.5.7. More than 300 candidate genes were found in the linked region of these QTL, and as many as 25&#x000a0;% might have SCN response. Disease resistance-like proteins were found to be induced by different Hg Types of SCN in other studies too such as by Mazarei <italic> et al. </italic> and Matsye <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR65"> 65 </xref> , <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. That suggested that disease <span class="trait">resistance</span> like proteins was a class of genes with broad-spectrum <span class="trait">resistance</span> to SCN. In addition, expression patterns of genes that encoded cytochrome P450, zinc finger protein, MYB domain protein, and WRKY DNA-binding protein that were mapped here were changed in SCN induced soybean roots [ <xref ref-type="bibr" rid="CR65"> 65 </xref> &#x02013; <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. SAUR-like genes has been suggested to encode short-lived nuclear proteins involved in auxin signaling by interacting with calmodulin [ <xref ref-type="bibr" rid="CR69"> 69 </xref> , <xref ref-type="bibr" rid="CR70"> 70 </xref> ]. A report about auxin-responsive factor (ARF) in <italic> Arabidopsis thaliana </italic> described that double mutants of ARF gene showed a strong auxin phenotype that resulted in the absence of lateral root formation than single mutant expression. In addition, <italic> <span class="gene">Glyma.02g161600</span> </italic> that encoded the RING-H2 finger domain was the nearest gene to rs21804864 which is a novel locus associated with <span class="trait">resistance</span> to SCN. RING-H2 finger as an domain in E3 ubiquitin ligase was considered to participate in the degradation of proteins from the endoplasmic reticulum in Human [ <xref ref-type="bibr" rid="CR71"> 71 </xref> ]. E3 ubiquitin ligases was known to play an important role in mediating plant defence signaling [ <xref ref-type="bibr" rid="CR72"> 72 </xref> ]. Therefore, <italic> <span class="gene">Glyma.02g161600</span> </italic> with RING-H2 finger domain might be a new gene source of SCN <span class="trait">resistance.</span> However, many such genes alter the degree of susceptibility (60&#x02009;&#x0003c;&#x02009;FI&#x02009;&#x0003c;&#x02009;300) more than the degree of <span class="trait">resistance</span> (FI&#x02009;&#x0003c;&#x02009;10).
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Gamma radiation and magnetic field mediated delay in effect of accelerated ageing of soybean.</h2>
			<h3>PMID:26243899</h3>
				

				
			<p>
				作者：Kumar M(1), Singh B(2), Ahuja S(2), Dahuja A(3), Anand A(4).
				<br/> 
				年份：2015 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean seeds were exposed to gamma radiation (0.5, 1, 3 and 5 kGy), static magnetic field (50, 100 and 200 mT) and a combination of gamma radiation and magnetic energy (0.5 kGy + 200 mT and 5 kGy + 50 mT) and stored at room temperature for six months. These seeds were later subjected to accelerated ageing treatment at 42 °C temperature and 95-100 % relative humidity and were compared for various physical and biochemical characteristics between the untreated and the energized treatments. Energy treatment protected the quality of stored seeds in terms of its protein and oil content . Accelerated aging conditions, however, affected the oil and protein quantity and quality of seed negatively. Antioxidant enzymes exhibited a decline in their activity during aging while the <span class="gene">LOX</span> activity, which reflects the rate of lipid peroxidation, in general, increased during the aging. Gamma irradiated (3 and 5 kGy) and magnetic field treated seeds (100 and 200 mT) maintained a higher catalase and ascorbate peroxidase activity which may help in efficient scavenging of deleterious free radical produced during the aging. Aging caused peroxidative changes to lipids, which could be contributed to the loss of oil quality. Among the electromagnetic energy treatments, a dose of 1-5 kGy of gamma and 100 mT, 200 mT magnetic field effectively slowed the rate of biochemical degradation and loss of cellular integrity in seeds stored under conditions of accelerated aging and thus, protected the deterioration of seed quality. Energy combination treatments did not <span class="trait">yield</span> any additional protection advantage.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A high efficient protocol for soybean root transformation by Agrobacterium rhizogenes and most stable reference genes for RT-qPCR analysis.</h2>
			<h3>PMID:26232349  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kuma KM(1)(2), Lopes-Caitar VS(3)(2), Romero CC(2), Silva SM(1)(2), Kuwahara MK(2), Carvalho MC(4), Abdelnoor RV(2), Dias WP(2), Marcelino-Guimarães FC(5).
				<br/> 
				年份：2015 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: A 55% transformation efficiency was obtained by our optimized protocol; and we showed that <span class="gene">GmELF1</span> - β and <span class="gene">GmELF1</span> - α are the most stable reference genes for expression analyses under this specific condition. Gene functional analyses are essential to the validation of results obtained from in silico and/or gene-prospecting studies. Genetic transformation methods that <span class="trait">yield</span> tissues of transient expression quickly have been of considerable interest to researchers. Agrobacterium rhizogenes-mediated transformation methods, which are employed to generate plants with transformed roots, have proven useful for the study of stress caused by root phytopathogens via gene overexpression and/or silencing. While some protocols have been adapted to soybean plants, transformation efficiencies remain limited; thus, few viable plants are available for performing bioassays. Furthermore, mRNA analyses that employ reverse transcription quantitative polymerase chain reactions (RT-qPCR) require the use of reference genes with stable expression levels across different organs, development steps and treatments. In the present study, an A. rhizogenes-mediated soybean root transformation approach was optimized. The method delivers significantly higher transformation efficiency levels and rates of transformed plant recovery, thus enhancing studies of soybean abiotic conditions or interactions between phytopathogens, such as nematodes. A 55% transformation efficiency was obtained following the addition of an acclimation step that involves hydroponics and different selection processes. The present study also validated the reference genes <span class="gene">GmELF1-β</span> and <span class="gene">GmELF1-α</span> as the most stable to be used in RT-qPCR analysis in composite plants, mainly under nematode infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean ( <italic> Glycine max </italic> ) SWEET gene family: insights through comparative genomics, transcriptome profiling and whole genome re-sequence analysis</h2>
			<h3>PMID:26162601</h3>
				

				
					<h3>PMCID:4499210</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant </given-names>  </name>  <address>  <email> patilg@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <address>  <email> valliyodanb@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> Rupesh </given-names>  </name>  <address>  <email> rupesh0deshmukh@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prince </surname>  <given-names> Silvas </given-names>  </name>  <address>  <email> princesj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nicander </surname>  <given-names> Bjorn </given-names>  </name>  <address>  <email> Bjorn.Nicander@slu.se </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Mingzhe </given-names>  </name>  <address>  <email> soyshen@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sonah </surname>  <given-names> Humira </given-names>  </name>  <address>  <email> biohuma@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <address>  <email> songli@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Li </given-names>  </name>  <address>  <email> linl@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chaudhary </surname>  <given-names> Juhi </given-names>  </name>  <address>  <email> chaudharyj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Yang </given-names>  </name>  <address>  <email> ylmk2@mail.missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <address>  <email> joshitr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <address>  <email> xudong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> National Center for Soybean Biotechnology and Division of Plant Sciences, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Department of Plant Biology and Forest Genetics and Linnean Center for Plant Biology, Swedish University of Agricultural Sciences, Uppsala, Sweden  </aff>  <aff id="Aff3">  <label/> Department of Computer Science, Informatics Institute, and Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff4">  <label/> Current address: Agronomy College of Shenyang Agricultural University, Shenyang, China  </aff>
				<br/> 
				年份：<day> 26 </day>  <month> 6 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: SWEET (MtN3_saliva) domain proteins, a recently identified group of efflux transporters, play an indispensable role in sugar efflux, phloem loading, plant-pathogen interaction and reproductive tissue development. The SWEET gene family is predominantly studied in Arabidopsis and members of the family are being investigated in rice. To date, no transcriptome or genomics analysis of soybean SWEET genes has been reported. RESULTS: In the present investigation, we explored the evolutionary aspect of the SWEET gene family in diverse plant species including primitive single cell algae to angiosperms with a major emphasis on Glycine max. Evolutionary features showed expansion and duplication of the SWEET gene family in land plants. Homology searches with BLAST tools and Hidden Markov Model-directed sequence alignments identified 52 SWEET genes that were mapped to 15 chromosomes in the soybean genome as tandem duplication events. Soybean SWEET <span class="gene">(GmSWEET)</span> genes showed a wide range of expression profiles in different tissues and developmental stages. Analysis of public transcriptome data and expression profiling using quantitative real time PCR (qRT-PCR) showed that a majority of the <span class="gene">GmSWEET</span> genes were confined to reproductive tissue development. Several natural genetic variants (non-synonymous SNPs, premature stop codons and haplotype) were identified in the <span class="gene">GmSWEET</span> genes using whole genome re-sequencing data analysis of 106 soybean genotypes. A significant association was observed between SNP-haplogroup and seed sucrose content in three gene clusters on chromosome 6. CONCLUSION: Present investigation utilized comparative genomics, transcriptome profiling and whole genome re-sequencing approaches and provided a systematic description of soybean SWEET genes and identified putative candidates with probable roles in the reproductive tissue development. Gene expression profiling at different developmental stages and genomic variation data will aid as an important resource for the soybean research community and can be extremely valuable for understanding sink unloading and enhancing carbohydrate delivery to developing seeds for improving <span class="trait">yield.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The SWEET genes play a diverse functional role during plant development which is evident from their expression patterns in other plant species [ <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR40"> 40 </xref> , <xref ref-type="bibr" rid="CR43"> 43 </xref> , <xref ref-type="bibr" rid="CR50"> 50 </xref> , <xref ref-type="bibr" rid="CR58"> 58 </xref> ] and soybean (this report). In rice and Arabidopsis, the expression of the SWEET genes were relatively higher in flower, pollen, embryo sac and seeds suggesting their roles in reproductive tissue development [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> , <xref ref-type="bibr" rid="CR20"> 20 </xref> ]. In rice two members of the SWEET gene family were highly expressed in panicles and anthers and were associated with <span class="trait">fertility</span> and seed size [ <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR58"> 58 </xref> , <xref ref-type="bibr" rid="CR59"> 59 </xref> ]. In Arabidopsis, <italic> AtSWEET8 </italic> was expressed in the embryo sac suggesting that it might regulate female gametocyte development [ <xref ref-type="bibr" rid="CR72"> 72 </xref> ]. Developing seeds are the strongest sink tissues in many plants and they need a higher carbon source for development which implies that nutrient transporters including the SWEET genes might be key component for their development. In Arabidopsis, <italic> AtSWEET11 </italic> and &#x02212; <italic> 12 </italic> showed a higher expression in leaves and had important roles in leaf sucrose export [ <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. The comparison of <italic> AtSWEET11 </italic> and &#x02212; <italic> 12 </italic> expression pattern with soybean orthologs <italic> <span class="gene">GmSWEET6,</span> </italic> and &#x02212; <italic> 15 </italic> showed a relatively higher expression in leaves, suggesting that these genes also might have similar role in leaf sucrose export.
					</p>
				
					<p>&nbsp;&nbsp;
						Yuan and Wang [ <xref ref-type="bibr" rid="CR20"> 20 </xref> ] and Chen [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ] have reviewed the functional role of SWEET genes in different tissues, pathogen infestation, and environmental responses. Interestingly, <italic> <span class="gene">GmSWEET13</span> </italic> , <italic> 14 </italic> and <italic> 15 </italic> fall under the fungal disease <span class="trait">resistance</span> QTL on chromosome 6 in soybean [ <xref ref-type="bibr" rid="CR73"> 73 </xref> ]. It has been proven that fungal and bacterial symbionts induce SWEET gene expression for nutritional gain during pathogen infestation [ <xref ref-type="bibr" rid="CR15"> 15 </xref> , <xref ref-type="bibr" rid="CR25"> 25 </xref> , <xref ref-type="bibr" rid="CR40"> 40 </xref> , <xref ref-type="bibr" rid="CR74"> 74 </xref> , <xref ref-type="bibr" rid="CR75"> 75 </xref> ]. The statement that most of the reported SWEET genes are associated with reproductive development tissue is corroborated in this study using soybean transcriptome datasets. The transciptome and qRT-PCR data showed that multiple SWEET genes are expressed at higher levels in tissues involved in reproductive development. Relatively higher expression of <italic> <span class="gene">GmSWEET5</span> </italic> , &#x02212; <italic> 10 </italic> , &#x02212; <italic> 23 </italic> and &#x02212; <italic> 48 </italic> in the seed tissue, suggest that collectively these genes might assist the movement of sucrose in the developing soybean seeds. Unloading of nutrient in the developing seeds occurs from the seed coat [ <xref ref-type="bibr" rid="CR32"> 32 </xref> , <xref ref-type="bibr" rid="CR76"> 76 </xref> ]. In the developing legume seeds ( <italic> P. vulgaris </italic> and <italic> P. sativum </italic> ), a suite of sucrose transporters are expressed at a higher levels in seed coat tissue to facilitate the movement of sucrose [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. Sugar availability, starch content, and cytokinin levels are involved in the regulation of abscission of soybean flowers, the delay of which hampers seed development and leads to <span class="trait">yield</span> loss in soybean [ <xref ref-type="bibr" rid="CR77"> 77 </xref> &#x02013; <xref ref-type="bibr" rid="CR79"> 79 </xref> ]. Soybean flower abortion is primarily caused by deficiency in or competition for photo-assimilates and nutrients among growing organs.
					</p>
				
					<p>&nbsp;&nbsp;
						Beside the expression level, the genetic variation (natural or induced) also enforces the functionality of SWEET genes and causes a variation in phenotype [ <xref ref-type="bibr" rid="CR43"> 43 </xref> , <xref ref-type="bibr" rid="CR64"> 64 </xref> ]. Mutation in the SWEET gene or abolishing the activation of the SWEET promoter leads to <span class="trait">resistance</span> to bacterial pathogens in rice [ <xref ref-type="bibr" rid="CR5"> 5 </xref> , <xref ref-type="bibr" rid="CR59"> 59 </xref> , <xref ref-type="bibr" rid="CR80"> 80 </xref> ]. Identification of several non-synonymous SNPs and large effect SNPs in <italic> <span class="gene">GmSWEETs</span> </italic> are expected to affect the integrity of encoded proteins. Additionally, exploring SNP-haplotype diversity using whole-genome sequencing data mining provides a powerful resource for investigating diversity in a particular gene family [ <xref ref-type="bibr" rid="CR81"> 81 </xref> &#x02013; <xref ref-type="bibr" rid="CR83"> 83 </xref> ]. The data presented here, using a cluster of genes on chromosome 6 ( <italic> <span class="gene">GmSWEET13,</span> &#x02212;14 </italic> and &#x02212; <italic> 15 </italic> ), showed the association between the SNP-haplogroups and sucrose content in seeds. The allelic variation data presented in this study provides a valuable resource for association studies between the SNPs and important agronomic traits, although intensive studies with each candidate gene are required to examine this inference. Overall, the SWEET gene family signifies its role as a key component in reproductive tissue development, nutrient unloading and pathogen <span class="trait">resistance.</span> Manipulating SWEET expression in specific tissues (phloem sap, pedicel, and developing seeds) could enhance sugar delivery to developing seeds to increase <span class="trait">yield.</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Association mapping and genomic prediction for <span class="trait">resistance</span> to sudden death syndrome in early maturing soybean germplasm</h2>
			<h3>PMID:25999779</h3>
				

				
					<h3>PMCID:4434860</h3>
					<h4>期刊:Molecular Breeding</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bao </surname>  <given-names> Yong </given-names>  </name>  <address>  <email> baoy@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kurle </surname>  <given-names> James E. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Anderson </surname>  <given-names> Grace </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Young </surname>  <given-names> Nevin D. </given-names>  </name>  <address>  <email> neviny@umn.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <aff id="Aff1">  <label/> Department of Agronomy and Plant Genetics, University of Minnesota, 411 Borlaug Hall, St. Paul, MN 55108-6026 USA  </aff>  <aff id="Aff2">  <label/> Department of Plant Pathology, University of Minnesota, 495 Borlaug Hall, St. Paul, MN 55108 USA  </aff>
				<br/> 
				年份：<day> 8 </day>  <month> 5 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sudden death syndrome (SDS), caused by Fusarium virguliforme, has spread to northern soybean growing regions in the US causing significant <span class="trait">yield</span> losses. The objectives of this study were to identify loci underlying variation in plant responses to SDS through association mapping (AM) and to assess prediction accuracy of genomic selection (GS) in a panel of early maturing soybean germplasm. A set of 282 soybean breeding lines was selected from the University of Minnesota soybean breeding program and then genotyped using a genome-wide panel of 1536 single-nucleotide polymorphism markers. Four <span class="trait">resistance</span> traits, root lesion severity (RLS), foliar symptom severity (FSS), root retention (RR), and dry matter reduction (DMR), were evaluated using soil inoculation in the greenhouse. AM identified significant peaks in genomic regions of known SDS <span class="trait">resistance</span> quantitative trait loci cqSDS001, cqRfs4, and SDS11-2. Additionally, two novel loci, one on chromosome 3 and another on chromosome 18, were tentatively identified. A ninefold cross-validation scheme was used to assess the prediction accuracy of <span class="gene">GS</span> for SDS <span class="trait">resistance.</span> The prediction accuracy of single-trait <span class="gene">GS</span> (ST-GS) was 0.64 for RLS, but less than 0.30 for RR, DMR, and FSS. Compared to ST-GS, none of multi-trait <span class="gene">GS</span> (MT-GS) models significantly improved the prediction accuracy due to weak correlations between the four traits. This study suggests both AM and <span class="gene">GS</span> hold promise for implementation in genetic improvement of SDS <span class="trait">resistance</span> in existing soybean breeding programs.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Sudden death syndrome (SDS), caused by <italic> Fusarium virguliforme </italic> , has spread to northern soybean growing regions in the US causing significant <span class="trait">yield</span> losses. The objectives of this study were to identify loci underlying variation in plant responses to SDS through association mapping (AM) and to assess prediction accuracy of genomic selection (GS) in a panel of early maturing soybean germplasm. A set of 282 soybean breeding lines was selected from the University of Minnesota soybean breeding program and then genotyped using a genome-wide panel of 1536 single-nucleotide polymorphism markers. Four <span class="trait">resistance</span> traits, root lesion severity (RLS), foliar symptom severity (FSS), root retention (RR), and dry matter reduction (DMR), were evaluated using soil inoculation in the greenhouse. AM identified significant peaks in genomic regions of known SDS <span class="trait">resistance</span> quantitative trait loci c <italic> qSDS001 </italic> , c <italic> qRfs4 </italic> , and SDS11-2. Additionally, two novel loci, one on chromosome 3 and another on chromosome 18, were tentatively identified. A ninefold cross-validation scheme was used to assess the prediction accuracy of <span class="gene">GS</span> for SDS <span class="trait">resistance.</span> The prediction accuracy of single-trait <span class="gene">GS</span> (ST-GS) was 0.64 for RLS, but less than 0.30 for RR, DMR, and FSS. Compared to ST-GS, none of multi-trait <span class="gene">GS</span> (MT-GS) models significantly improved the prediction accuracy due to weak correlations between the four traits. This study suggests both AM and <span class="gene">GS</span> hold promise for implementation in genetic improvement of SDS <span class="trait">resistance</span> in existing soybean breeding programs.
					</p>
				
					<p>&nbsp;&nbsp;
						The genetic architecture of (partial) <span class="trait">resistance</span> to SDS is complex. A total of 58 QTL have been reported as providing <span class="trait">resistance</span> to SDS in bi-parental mapping populations ( <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> www.soybase.org </ext-link> , verified June 11, 2014), and only a few of them have been consistent across mapping populations from different genetic backgrounds (Kazi et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ). However, the <span class="trait">resistance</span> loci c <italic> qRfs4 </italic> on linkage group <span class="gene">C2</span> (chromosome 6), <italic> cqSDS001 </italic> on linkage group D2 (chromosome 17), <italic> cqRfs1 </italic> , <italic> cqRfs2 </italic> , <italic> cqRfs3 </italic> on linkage group G (chromosome 18), and <italic> cqRfs6 </italic> on linkage group N (chromosome 3) were repeatedly mapped in multiple populations (de Farias Neto et al. <xref ref-type="bibr" rid="CR12"> 2007 </xref> ; Hnetkovsky et al. <xref ref-type="bibr" rid="CR17"> 1996 </xref> ; Iqbal et al. <xref ref-type="bibr" rid="CR20"> 2001 </xref> ; Kassem et al. <xref ref-type="bibr" rid="CR24"> 2006 </xref> ; Kazi et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ; Lightfoot et al. <xref ref-type="bibr" rid="CR28"> 2001 </xref> ; Njiti et al. <xref ref-type="bibr" rid="CR41"> 1998 </xref> , <xref ref-type="bibr" rid="CR43"> 2002 </xref> ; Prabhu et al. <xref ref-type="bibr" rid="CR44"> 1999 </xref> ). Potentially, the genetic markers identified from previous QTL mapping studies can assist in the selection of SDS-resistant cultivars in a timely and resource-efficient manner (Prabhu et al. <xref ref-type="bibr" rid="CR44"> 1999 </xref> ). Luckew et al. ( <xref ref-type="bibr" rid="CR32"> 2013 </xref> ) recently evaluated ten confirmed SDS QTL in F <sub> 2 </sub> -derived lines from six populations and suggested the possibility of stacking QTL to achieve durable SDS <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						A new marker-based approach known as genomic selection (GS) could potentially be an alternative strategy to stack numerous genes to achieve durable SDS <span class="trait">resistance.</span> Rather than utilizing only molecular markers in tight association with targeted QTLs, <span class="gene">GS</span> has been developed with the aim of directly predicting genetic value for quantitative traits by taking advantage of all available genome-wide marker information (Bernardo and Yu <xref ref-type="bibr" rid="CR5"> 2007 </xref> ; Meuwissen and Goddard <xref ref-type="bibr" rid="CR37"> 2010 </xref> ). In the <span class="gene">GS</span> scheme, QTL mapping is replaced by genomic prediction model training which involves fitting both phenotypic and genotypic data from a training population in either linear or nonlinear models. Marker effects estimated from the models are subsequently summed up to estimate genomic breeding values of individuals in a validation or breeding population with only genotypic data. Previous results in crop species, including soybean, have indicated that <span class="gene">GS</span> holds the potential to improve disease <span class="trait">resistance</span> with complex genetic architecture in breeding programs (Bao et al. <xref ref-type="bibr" rid="CR3"> 2014 </xref> ; Lorenz et al. <xref ref-type="bibr" rid="CR29"> 2012 </xref> ; Rutkoski et al. <xref ref-type="bibr" rid="CR48"> 2012 </xref> , <xref ref-type="bibr" rid="CR49"> 2014 </xref> ). Here, we seek to investigate the potential use of <span class="gene">GS</span> to select SDS <span class="trait">resistance</span> in a typical public soybean breeding program with a focus on early maturing germplasm.
					</p>
				
					<p>&nbsp;&nbsp;
						Since there are four phenotypic traits associated with SDS <span class="trait">resistance</span> in the data set, both single-trait genomic selection (ST-GS) model and multi-trait genomic selection (MT-GS) were evaluated, and their accuracies for predicting SDS <span class="trait">resistance</span> were compared. For ST-GS, a mixed linear model was constructed to estimate marker effects of phenotypic traits: <inline-formula id="IEq1"> <alternatives> <tex-math id="M1"> \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$y = Xb + Z\alpha + e$$\end{document} </tex-math> <mml:math id="M2"> <mml:mrow> <mml:mi> y </mml:mi> <mml:mo> = </mml:mo> <mml:mi> X </mml:mi> <mml:mi> b </mml:mi> <mml:mo> + </mml:mo> <mml:mi> Z </mml:mi> <mml:mi mathvariant="italic"> &#x003b1; </mml:mi> <mml:mo> + </mml:mo> <mml:mi> e </mml:mi> </mml:mrow> </mml:math> <inline-graphic xlink:href="11032_2015_324_Article_IEq1.gif"/> </alternatives> </inline-formula> , where <italic> y </italic> is the vector ( <italic> n </italic> &#x000a0;&#x000d7;&#x000a0;1) of phenotypic observations of <italic> n </italic> individuals, <italic> X </italic> is the design matrix ( <italic> n </italic> &#x000a0;&#x000d7;&#x000a0; <italic> r </italic> ) for fixed planting effects, <italic> b </italic> is the vector ( <italic> r </italic> &#x000a0;&#x000d7;&#x000a0;1) of planting effects, <italic> Z </italic> is the design matrix ( <italic> n </italic> &#x000a0;&#x000d7;&#x000a0; <italic> m </italic> ) for additive effects of SNP markers, <italic> &#x003b1; </italic> is the vector ( <italic> m </italic> &#x000a0;&#x000d7;&#x000a0;1) of additive effects of SNP markers, and <italic> e </italic> is the vector ( <italic> n </italic> &#x000a0;&#x000d7;&#x000a0;1) of residuals. The variances of <italic> &#x003b1; </italic> and <italic> e </italic> are <inline-formula id="IEq2"> <alternatives> <tex-math id="M3"> \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\text{Var}}(\alpha ) = I_{m} \sigma_{\alpha }^{2}$$\end{document} </tex-math> <mml:math id="M4"> <mml:mrow> <mml:mtext> Var </mml:mtext> <mml:mrow> <mml:mo stretchy="false"> ( </mml:mo> <mml:mi mathvariant="italic"> &#x003b1; </mml:mi> <mml:mo stretchy="false"> ) </mml:mo> </mml:mrow> <mml:mo> = </mml:mo> <mml:msub> <mml:mi> I </mml:mi> <mml:mi> m </mml:mi> </mml:msub> <mml:msubsup> <mml:mi mathvariant="italic"> &#x003c3; </mml:mi> <mml:mrow> <mml:mi mathvariant="italic"> &#x003b1; </mml:mi> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> </mml:mrow> </mml:math> <inline-graphic xlink:href="11032_2015_324_Article_IEq2.gif"/> </alternatives> </inline-formula> and <inline-formula id="IEq3"> <alternatives> <tex-math id="M5"> \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\text{Var}}(e) = I_{n} \sigma_{e}^{2}$$\end{document} </tex-math> <mml:math id="M6"> <mml:mrow> <mml:mtext> Var </mml:mtext> <mml:mrow> <mml:mo stretchy="false"> ( </mml:mo> <mml:mi> e </mml:mi> <mml:mo stretchy="false"> ) </mml:mo> </mml:mrow> <mml:mo> = </mml:mo> <mml:msub> <mml:mi> I </mml:mi> <mml:mi> n </mml:mi> </mml:msub> <mml:msubsup> <mml:mi mathvariant="italic"> &#x003c3; </mml:mi> <mml:mrow> <mml:mi> e </mml:mi> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> </mml:mrow> </mml:math> <inline-graphic xlink:href="11032_2015_324_Article_IEq3.gif"/> </alternatives> </inline-formula> , respectively, where <italic> I </italic> <sub> <italic> m </italic> </sub> is the <italic> m </italic> &#x000a0;&#x000d7;&#x000a0; <italic> m </italic> identity matrix, <inline-formula id="IEq4"> <alternatives> <tex-math id="M7"> \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\sigma_{\alpha }^{2}$$\end{document} </tex-math> <mml:math id="M8"> <mml:msubsup> <mml:mi mathvariant="italic"> &#x003c3; </mml:mi> <mml:mrow> <mml:mi mathvariant="italic"> &#x003b1; </mml:mi> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> </mml:math> <inline-graphic xlink:href="11032_2015_324_Article_IEq4.gif"/> </alternatives> </inline-formula> is the additive genetic variance for each maker, <inline-formula id="IEq5"> <alternatives> <tex-math id="M9"> \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\sigma_{e}^{2}$$\end{document} </tex-math> <mml:math id="M10"> <mml:msubsup> <mml:mi mathvariant="italic"> &#x003c3; </mml:mi> <mml:mrow> <mml:mi> e </mml:mi> </mml:mrow> <mml:mn> 2 </mml:mn> </mml:msubsup> </mml:math> <inline-graphic xlink:href="11032_2015_324_Article_IEq5.gif"/> </alternatives> </inline-formula> is the residual variance, and <italic> I </italic> <sub> <italic> n </italic> </sub> is the <italic> n </italic> &#x000a0;&#x000d7;&#x000a0; <italic> n </italic> identity matrix. A computationally efficient method, ridge-regression best linear unbiased prediction (RR-BLUP) was employed to solve the mixed model. Previous <span class="gene">GS</span> studies suggested slight differences between different genomic prediction algorithms including G-BLUP (which is equivalent to RR-BLUP), Bayesian approaches, and machine learning algorithms (Asoro et al. <xref ref-type="bibr" rid="CR65"> 2011 </xref> ; Bao et al. <xref ref-type="bibr" rid="CR3"> 2014 </xref> ; Lorenzana and Bernardo <xref ref-type="bibr" rid="CR30"> 2009 </xref> ; Lorenz et al. <xref ref-type="bibr" rid="CR29"> 2012 </xref> ; Rutkoski et al. <xref ref-type="bibr" rid="CR48"> 2012 </xref> ). The marker effects were simultaneously estimated by solving the mixed model through the restricted maximum likelihood (REML) method implemented in R package &#x02018;rrBLUP&#x02019; (R Development Core Team <xref ref-type="bibr" rid="CR64"> 2005 </xref> ). Variance of additive effects and variance of residual effects were estimated.
					</p>
				
					<p>&nbsp;&nbsp;
						Association mapping (AM) was performed for RLS, FSS, RR, and DMR. The QQ plot indicated the model we implemented for AM was sufficient to control the false positive (Supplemental Fig. S2). We identified two and eight significant (qFDR&#x000a0;&#x0003c;&#x000a0;0.05) SNP markers for DMR and RR, respectively, but none for the other two traits (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ; Supplemental Fig. S3). Among the eight distinct significant markers, three were in the same genomic intervals as the known SDS <span class="trait">resistance</span> quantitative trait loci (QTL) <italic> cqSDS001 </italic> on linkage group D2 (chromosome 17) (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ; Supplemental Fig. S3). Another marker at position 80.28&#x000a0;cM on linkage group <span class="gene">C2</span> (chromosome 6) was in the genomic region of <italic> cqRfs4 </italic> (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ; Supplemental Fig. S3). Both <italic> cqSDS001 </italic> and <italic> cqRfs4 </italic> have been previously identified and confirmed in multiple bi-parental populations (de Farias Neto et al. <xref ref-type="bibr" rid="CR12"> 2007 </xref> ; Hnetkovsky et al. <xref ref-type="bibr" rid="CR17"> 1996 </xref> ; Iqbal et al. <xref ref-type="bibr" rid="CR20"> 2001 </xref> ; Kassem et al. <xref ref-type="bibr" rid="CR63"> 2012 </xref> ; Kazi et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ; Njiti et al. <xref ref-type="bibr" rid="CR43"> 2002 </xref> ). Additionally, two significant SNP markers in the present study confirmed a previously identified QTL, <italic> SDS11 </italic> - <italic> 2 </italic> , on linkage group D2 (chromosome 17) (Kazi et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ). The rediscovery of the previously identified QTL strengthened the confidence in the overall quality of AM analysis. Notably, a SNP marker near the telomere on chromosome 3 (linkage group N) was tentatively identified as associated with RR variation, and another SNP marker near the telomere on chromosome 18 (linkage group G) was associated with the variation in both RR and DMR (Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> ; Supplemental Fig. S3; Supplemental Fig. S4). Since no previous QTL had been discovered near these two genomic locations, the two newly identified loci were provisionally named <italic> SDS14 </italic> - <italic> 1 </italic> (on chromosome 3) and <italic> SDS14 </italic> - <italic> 2 </italic> (on chromosome 18). For each of the five loci (namely, <italic> cqSDS001 </italic> , <italic> cqRfs4, SDS11 </italic> - <italic> 2, SDS14 </italic> - <italic> 1, </italic> and <italic> SDS14 </italic> - <italic> 2 </italic> ), RR and DMR peaks were coincident with each other (Supplemental Fig. S4). <table-wrap id="Tab2"> <label> Table&#x000a0;2 </label> <caption> <p> The significant SNPs (false discovery rate&#x000a0;&#x0003c;&#x000a0;0.05) detected from association mapping (AM) for SDS <span class="trait">resistance</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Besides identifying causal loci associated with SDS <span class="trait">resistance</span> through AM, the phenotypic and genotypic data sets were used to evaluate the utility of <span class="gene">GS</span> in predicting SDS <span class="trait">resistance</span> phenotypes. Overall, the prediction accuracy of the ST models was 0.64, 0.20, 0.18, and 0.16 for RLS, FSS, RR, and DMR, respectively (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). However, among these ST models, only the prediction accuracy for RLS was significantly different from zero. To determine whether multi-trait (MT) models improved prediction accuracy, ST and MT models were also compared. Compared to ST models, none of MT-GS models significantly improved the prediction accuracy for any trait (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). The RLS_FSS_DMR model did increase the prediction accuracy for DMR from 0.16 to 0.25 while maintaining a similar accuracy for FSS, but reduced the accuracy for RLS to 0.26 (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). The FT model performed equivalently to ST models for all four traits (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). <fig id="Fig2"> <label> Fig.&#x000a0;2 </label> <caption> <p> Prediction accuracy with multi-trait genomic selection (GS) models compared with single-trait <span class="gene">GS</span> models for four SDS <span class="trait">resistance</span> traits. <italic> RLS </italic> root lesion severity, <italic> RR </italic> root retention, <italic> FSS </italic> foliar symptom severity, <italic> DMR </italic> dry matter reduction, <italic> ST </italic> single-trait model, RLS_FSS model for RLS and FSS; RLS_RR model for RLS and RR; RLS_DMR model for RLS and DMR; FSS_RR model for FSS and RR; FSS_DMR model for FSS and DMR; RR_DMR model for RR and DMR; RLS_FSS_DMR model for RLS, FSS, and DMR; RLS_FSS_RR model for RLS, FSS, and RR; RR_FSS_DMR model for RR, FSS, and DMR; and FT model for all four traits. <italic> Red dot </italic> represents median of accuracies for each model. <italic> Notch marks </italic> the 95&#x000a0;% confidence interval for the medians. (Color figure online)
					</p>
				
					<p>&nbsp;&nbsp;
						Accurate assessment of phenotypic variation is essential for understanding disease biology, effective <span class="trait">resistance</span> breeding, and dissection of genetic architecture. The heritability of greenhouse evaluation of SDS <span class="trait">resistance</span> ranged from 33 to 66&#x000a0;% in previous studies (Njiti et al. <xref ref-type="bibr" rid="CR42"> 2001 </xref> ). In the greenhouse experiment, the effect of soybean genotypes was significant ( <italic> p </italic> &#x000a0;&#x0003c;&#x000a0;0.05) in most plantings indicating an overall reliability of phenotypic data (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). However, we still observed substantial replication&#x000a0;&#x000d7;&#x000a0;genotype variation in four trait&#x000a0;&#x000d7;&#x000a0;planting experiments (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). The high level of phenotypic variation between replications has also been observed in previous studies (Kazi et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ; Luckew et al. <xref ref-type="bibr" rid="CR32"> 2013 </xref> ) and could be attributed to the complex genetic basis of SDS <span class="trait">resistance,</span> interactive effects of genotype with environment, and/or imperfect screening methods. Another limitation in the current study was the low throughput capacity of the phenotyping system; soybean lines had to be evaluated in six plantings, which might have reduced the ability to detect all causative QTL and/or led to biased estimations. In other words, the genetic effects might have been confounded by the effect of consecutive experimental plantings conducted over time, limiting the ability to induce SDS symptoms consistently, and as a result, reducing the explanatory power of AM. For example, changing light intensity and ambient temperature variation associated with seasonal changes in <span class="gene">sun</span> angle and ambient temperature presumably added to variability in SDS phenotype. To minimize the influences of these sources of variance of among plantings, we conducted the greenhouse experiments with supplemental lighting and air conditioning, and accounted for the effect of plantings as a fixed effect in the AM model.
					</p>
				
					<p>&nbsp;&nbsp;
						To pyramid these <span class="trait">resistance</span> QTL into commercial soybean cultivars, the significant SNP markers identified in present study can be developed as a breeder-friendly SNP array for conducting MAS in SDS <span class="trait">resistance</span> breeding programs. However, stacking multiple QTL and introgressing them to an adapted elite parent would require considerable resources and time. As an alternative to stacking major SDS <span class="trait">resistance</span> genes, <span class="gene">GS</span> may provide breeders an opportunity to integrate a broader set of causative loci underlying SDS <span class="trait">resistance</span> with the goal of more durable resistant soybean cultivars. Despite successful rediscovery of known QTL for RR and DMR, we failed to identify any significant signals (qFDR&#x000a0;&#x0003c;&#x000a0;0.05) for RLS and FSS with AM. This might indicate that the genetic variation of RLS and FSS captured in the population is associated with numerous causative genes each with a small effect. In this case, genome-wide selection as implemented through <span class="gene">GS</span> is expected to be more effective than MAS because <span class="gene">GS</span> would enable breeders to select candidate lines with higher levels of cumulative <span class="trait">resistance</span> to SDS conferred by numerous small effect loci. Estimates of prediction accuracy for RLS were as high as 0.64 (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ), which is comparable to that for SCN <span class="trait">resistance</span> in soybean (Bao et al. <xref ref-type="bibr" rid="CR3"> 2014 </xref> ), Fusarium head blight <span class="trait">resistance</span> in barley and wheat (Lorenz et al. <xref ref-type="bibr" rid="CR29"> 2012 </xref> ; Rutkoski et al. <xref ref-type="bibr" rid="CR48"> 2012 </xref> ), and northern leaf blight in corn (Technow et al. <xref ref-type="bibr" rid="CR54"> 2013 </xref> ). Given the high prediction accuracy, <span class="gene">GS</span> holds great potential for implementation in genetic evaluation of breeding candidates in an actual soybean improvement program targeting at SDS <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						SDS <span class="trait">resistance</span> breeding is further complicated by the existence of two apparently distinct <span class="trait">resistance</span> mechanisms involved in expression of root versus foliar responses to SDS (Kazi et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ; Triwitayakorn et al. <xref ref-type="bibr" rid="CR55"> 2005 </xref> ). Some known QTL confer specific <span class="trait">resistance</span> to root rot or foliar scorch, while others confer <span class="trait">resistance</span> to both (de Farias Neto et al. <xref ref-type="bibr" rid="CR12"> 2007 </xref> ; Hnetkovsky et al. <xref ref-type="bibr" rid="CR17"> 1996 </xref> ; Iqbal et al. <xref ref-type="bibr" rid="CR20"> 2001 </xref> ; Kassem et al. <xref ref-type="bibr" rid="CR24"> 2006 </xref> ; Kazi et al. <xref ref-type="bibr" rid="CR25"> 2008 </xref> ; Njiti et al. <xref ref-type="bibr" rid="CR41"> 1998 </xref> , <xref ref-type="bibr" rid="CR43"> 2002 </xref> ). To develop soybean cultivars with both root and foliar <span class="trait">resistance</span> to SDS, multi-trait <span class="gene">GS</span> (MT-GS) has the potential to be an effective selection strategy for implementing an SDS <span class="trait">resistance</span> improvement program. An MT-GS model is developed by simultaneously fitting phenotypic data from the evaluations of root and foliar symptoms as dependent variables in the model. Subsequently, the MT-GS model using one marker panel leads to simultaneous prediction of both root and foliar symptoms.
					</p>
				
					<p>&nbsp;&nbsp;
						Our results suggested that the prediction accuracy of <span class="gene">GS</span> model based on single traits (ST-GS) for FSS, RR, and DMR was comparatively low (&#x0003c;0.3) (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). In a simulation study, Jia and Jannink ( <xref ref-type="bibr" rid="CR21"> 2012 </xref> ) indicated that the prediction accuracy for low-heritability traits could be improved by <span class="gene">GS</span> models based on multiple related traits (MT-GS) models. The underlying mechanism of improved accuracy for low-heritability traits in MT-GS is presumably genetic relationship between the highly related traits (Jia and Jannink <xref ref-type="bibr" rid="CR21"> 2012 </xref> ). In the case of SDS <span class="trait">resistance,</span> we hypothesized that MT-GS might be capable of taking advantage of the genetic relationship between low-heritability traits: FSS, RR and DMR, and high-heritability trait: RLS. However, the FT model based on all four SDS <span class="trait">resistance</span> traits performed equivalently to the ST models in the study (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ) and none of the MT-GS models significantly improved the prediction accuracy. An increase in the prediction accuracy for DMR with the RLS_FSS_DMR model was observed, while the RLS_FSS_DMR model failed to maintain similar prediction accuracy for RLS and FSS as that in ST-GS models (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The present study suggests AM could be used as an alternative method for mapping QTL underlying SDS <span class="trait">resistance,</span> and <span class="gene">GS</span> holds potential for implementation in genetic evaluation of root lesion severity associated with SDS. We conclude that SDS <span class="trait">resistance</span> is a complex disease trait, leading to numerous challenges in evaluating and breeding for SDS-resistant soybean cultivars. Firstly, improving phenotypic screening methods to ensure high-quality and high-throughput evaluation of SDS <span class="trait">resistance</span> should remain as an important component of the current SDS breeding program. Secondly, high-density genome-wide markers or sequence-based genotyping methods could be employed to dissect the genetic architecture of SDS <span class="trait">resistance</span> more precisely. Lastly, the realized response and cost-effectiveness of <span class="gene">GS</span> deserves further investigation in both greenhouse and field prior to implementing <span class="gene">GS</span> for developing durable SDS <span class="trait">resistance</span> in soybeans.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Dual functions of <span class="gene">GmTOE4a</span> in the regulation of photoperiod-mediated flowering and plant morphology in soybean.</h2>
			<h3>PMID:25991035  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao X(1), Cao D, Huang Z, Wang J, Lu S, Xu Y, Liu B, Kong F, Yuan X.
				<br/> 
				年份：2015 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Flowering time, maturity, and plant morphology have considerable effects on the adaptation and grain <span class="trait">yield</span> of soybean (Glycine max). The identification of novel genes and an understanding of their molecular basis are critical to improve soybean productivity. In this study, we cloned a flowering time related APETALA2-like gene <span class="gene">GmTOE4a</span> and generated <span class="gene">GmTOE4a-overexpressing</span> lines in the cultivar Williams 82. The transgenic lines exhibited late flowering both under long day and short day conditions, and repressed the flowering-related genes, including <span class="gene">GmFT2a,</span> <span class="gene">GmFT5a,</span> <span class="gene">GmAP1,</span> and <span class="gene">GmLFY,</span> whereas the flowering repressors <span class="gene">GmFT4</span> and miR156 were upregulated. Interestingly, <span class="gene">GmTOE4a</span> was also mediated by photoperiod via maturity genes E3 and E4, which encode photoreceptors in soybean. Further, miR172-mediated <span class="gene">GmTOE4a,</span> which regulates flowering in soybean, is different in Arabidopsis in that it is dependent on the CONSTANS-like gene <span class="gene">GmCOL1a.</span> In addition to its effect on flowering time, <span class="gene">GmTOE4a</span> regulated plant morphology, increased stem thickness, and reduced plant height, internode length and leaf size, which are important agronomic traits that enhance the capacity to resist <span class="trait">lodging</span> and increase soybean <span class="trait">yield.</span> This is useful information to understand the molecular mechanism of flowering time and plant morphology in soybean and will greatly influence soybean <span class="trait">yield</span> improvement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Efficient production of free fatty acids from soybean meal carbohydrates.</h2>
			<h3>PMID:25943383  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang D(1)(2), Thakker C(3), Liu P(1), Bennett GN(3), San KY(4)(5).
				<br/> 
				年份：2015 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Conversion of biomass feedstock to chemicals and fuels has attracted increasing attention recently. Soybean meal, containing significant quantities of carbohydrates, is an inexpensive renewable feedstock. Glucose, galactose, and fructose can be obtained by enzymatic hydrolysis of soluble carbohydrates of soybean meal. Free fatty acids (FFAs) are valuable molecules that can be used as precursors for the production of fuels and other value-added chemicals. In this study, free fatty acids were produced by mutant Escherichia coli strains with plasmid pXZ18Z (carrying acyl-ACP thioesterase (TE) and (3R)-hydroxyacyl-ACP dehydratase) using individual sugars, sugar mixtures, and enzymatic hydrolyzed soybean meal extract. For individual sugar fermentations, strain ML211 (MG1655 fadD(-) fabR(-) )/pXZ18Z showed the best performance, which produced 4.22, 3.79, 3.49 g/L free fatty acids on glucose, fructose, and galactose, respectively. While the strain ML211/pXZ18Z performed the best with individual sugars, however, for sugar mixture fermentation, the triple mutant strain XZK211 (MG1655 fadD(-) fabR(-) ptsG(-) )/pXZ18Z with an additional deletion of <span class="gene">ptsG</span> encoding the glucose-specific transporter, functioned the best due to relieved catabolite repression. This strain produced approximately 3.18 g/L of fatty acids with a <span class="trait">yield</span> of 0.22 g fatty acids/g total sugar. Maximum free fatty acids production of 2.78 g/L with a high <span class="trait">yield</span> of 0.21 g/g was achieved using soybean meal extract hydrolysate. The results suggested that soybean meal carbohydrates after enzymatic treatment could serve as an inexpensive feedstock for the efficient production of free fatty acids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Application of (1)h NMR profiling to assess seed metabolomic diversity. A case study on a soybean <span class="gene">era</span> population.</h2>
			<h3>PMID:25940152  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Harrigan GG(1), Skogerson K(1), MacIsaac S(2), Bickel A(1), Perez T(1), Li X(1).
				<br/> 
				年份：2015 May 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						(1)H NMR spectroscopy offers advantages in metabolite quantitation and platform robustness when applied in food metabolomics studies. This paper provides a (1)H NMR-based assessment of seed metabolomic diversity in conventional and glyphosate-resistant genetically modified (GM) soybean from a genetic lineage representing ∼35 years of breeding and differing <span class="trait">yield</span> potential. (1)H NMR profiling of harvested seed allowed quantitation of 27 metabolites, including free amino acids, sugars, and organic acids, as well as choline, O-acetylcholine, dimethylamine, trigonelline, and p-cresol. Data were analyzed by canonical discriminant analysis (CDA) and principal variance component analysis (PVCA). Results demonstrated that (1)H NMR spectroscopy was effective in highlighting variation in metabolite levels in the genetically diverse sample set presented. The results also confirmed that metabolite variability is influenced by selective breeding and environment, but not genetic modification. Therefore, metabolite variability is an integral part of crop improvement that has occurred for decades and is associated with a history of safe use.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide association study for flowering time, maturity dates and plant height in early maturing soybean ( <italic> Glycine max </italic> ) germplasm</h2>
			<h3>PMID:25887991</h3>
				

				
					<h3>PMCID:4449526</h3>
					<h4>期刊:BMC Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jiaoping </given-names>  </name>  <address>  <email> jiaopingzhang@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <address>  <email> Qijian.Song@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B </given-names>  </name>  <address>  <email> Perry.Cregan@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nelson </surname>  <given-names> Randall L </given-names>  </name>  <address>  <email> Randall.Nelson@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xianzhi </given-names>  </name>  <address>  <email> xianzhiwang1@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Jixiang </given-names>  </name>  <address>  <email> jixiang.wu@sdstate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Jiang </surname>  <given-names> Guo-Liang </given-names>  </name>  <address>  <email> gjiang@vsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <aff id="Aff1">  <label/> Plant Science Department, South Dakota State University, Brookings, SD 57006 USA  </aff>  <aff id="Aff2">  <label/> Soybean Genomics and Improvement Laboratory, US Department of Agriculture, Agricultural Research Service (USDA-ARS), 10300 Baltimore Ave, Beltsville, MD 20705 USA  </aff>  <aff id="Aff3">  <label/> USDA-ARS, Soybean/Maize Germplasm, Pathology, and Genetics Research Unit and Department of Crop Sciences, University of Illinois, Urbana-Champaign, 1101 West Peabody Drive, Urbana, IL 61801 USA  </aff>  <aff id="Aff4">  <label/> Agricultural Research Station, Virginia State University, P.O. Box 9061, Petersburg, VA 23806 USA  </aff>
				<br/> 
				年份：<day> 6 </day>  <month> 3 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean (Glycine max) is a photoperiod-sensitive and self-pollinated species. Days to flowering (DTF) and maturity (DTM), duration of flowering-to-maturity (DFTM) and plant height (PH) are crucial for soybean adaptability and <span class="trait">yield.</span> To dissect the genetic architecture of these agronomically important traits, a population consisting of 309 early maturity soybean germplasm accessions was genotyped with the Illumina Infinium SoySNP50K BeadChip and phenotyped in multiple environments. A genome-wide association study (GWAS) was conducted using a mixed linear model that involves both relative kinship and population structure. RESULTS: The linkage disequilibrium (LD) decayed slowly in soybean, and a substantial difference in LD pattern was observed between euchromatic and heterochromatic regions. A total of 27, 6, 18 and 27 loci for DTF, DTM, DFTM and PH were detected via GWAS, respectively. The <span class="gene">Dt1</span> gene was identified in the locus strongly associated with both DTM and PH. Ten candidate genes homologous to Arabidopsis flowering genes were identified near the peak single nucleotide polymorphisms (SNPs) associated with DTF. Four of them encode MADS-domain containing proteins. Additionally, a pectin lyase-like gene was also identified in a major-effect locus for PH where LD decayed rapidly. CONCLUSIONS: This study identified multiple new loci and refined chromosomal regions of known loci associated with DTF, DTM, DFTM and/or PH in soybean. It demonstrates that GWAS is powerful in dissecting complex traits and identifying candidate genes although LD decayed slowly in soybean. The loci and trait-associated SNPs identified in this study can be used for soybean genetic improvement, especially the major-effect loci associated with PH could be used to improve soybean <span class="trait">yield</span> potential. The candidate genes may serve as promising targets for studies of molecular mechanisms underlying the related traits in soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Flowering, maturity and plant height in plants are complex traits controlled by internal and external factors. They considerably impact the adaptability, biomass and economic <span class="trait">yield</span> in agricultural crops. The genetic signaling pathways of flowering have been well characterized in <italic> Arabidopsis </italic> , a model organism in plant biology. The floral integrator genes <italic> FLOWERING LOCUS T </italic> ( <italic> FT </italic> ) and <italic> SUPPRESSOR OF OVEREXPRESSION OF CONSTANS 1 </italic> ( <italic> SOC1 </italic> ) play a central role in flowering regulation. They promote the expression of a group of floral meristem identity genes such as <italic> APETALA 1 </italic> ( <italic> <span class="gene">AP1</span> </italic> ), <italic> LEAFY </italic> ( <italic> <span class="gene">LFY</span> </italic> ) and <italic> CAULIFLOWER </italic> ( <italic> <span class="gene">CAL</span> </italic> ) to initiate the floral transition of the plant [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. In the floral pathway, upstream of <italic> FT </italic> and <italic> SOC1 </italic> are <italic> FLOWERING LOCUS C </italic> ( <italic> FLC </italic> ) and <italic> CONSTANS </italic> ( <italic> CO </italic> ), which are two key regulators of floral integrators but in different manners. In <italic> Arabidopsis </italic> , a high level of FLC represses expression of both <italic> FT </italic> and <italic> SOC1 </italic> , and it can be released by vernalization treatment or autonomous development process [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ]. <italic> CO </italic> is circadian-regulated and involved in the photoperiod pathway through promoting expression of <italic> FT </italic> and <italic> SOC1 </italic> [ <xref ref-type="bibr" rid="CR2"> 2 </xref> , <xref ref-type="bibr" rid="CR3"> 3 </xref> ]. Additionally, more genes involved in the inductive photoperiod pathway, the vernalization pathway, the autonomous pathway and the gibberellins pathway have been characterized in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> , <xref ref-type="bibr" rid="CR5"> 5 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The SNP at the 45.0&#x000a0;Mb position on <span class="gene">Gm19,</span> a region similar to previously reported QTLs Pod mat 13&#x02013;6 and Plant height 4&#x02013;2 and 13&#x02013;8 (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> ), was strongly associated with both DTM and PH. The <italic> <span class="gene">Dt1</span> </italic> gene was found in this region. <italic> <span class="gene">Dt1</span> </italic> is homologous to <italic> Arabidopsis terminal flower 1 </italic> , and plays a predominant role in determining stem growth habit in soybean [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. Based upon the stem growth habit, soybean cultivars can be classified into two major categories, determinate and indeterminate. For the determinate soybean cultivars ( <italic> dt1/dt1 </italic> ), SAM switches from vegetative growth to reproductive growth soon after photoperiod-induced floral transition, and stem growth stops [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. In contrast, the transition of SAM to floral meristem is suppressed in indeterminate cultivars ( <italic> Dt1/Dt1 </italic> ) and vegetative growth of SAM continues until a cessation caused by the demand of developing seeds [ <xref ref-type="bibr" rid="CR49"> 49 </xref> ]. Therefore, stem growth habit has broad effects on plant height and maturity of soybean. This is highly consistent with the result of the present study that the locus harboring <italic> <span class="gene">Dt1</span> </italic> was strongly associated with both DTM and PH, indicated that <italic> <span class="gene">Dt1</span> </italic> is very likely the causal gene for DTM5 and PH25. Because plant height is one of the major factors determining <span class="trait">yield</span> potential in soybean, PH25 with large effect on plant height may also affect soybean <span class="trait">yield</span> substantially. However, application of the preferred allele of PH25 locus needs to be careful, as it may also affect maturity dates.
					</p>
				
					<p>&nbsp;&nbsp;
						The locus PH24, explained 10% of phenotypic variation, was mapped to a small region on <span class="gene">Gm19</span> where LD decayed rapidly. Three QTLs associated with PH were previously reported in the similar region (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> ). The candidate gene <italic> <span class="gene">Glyma19g37180</span> </italic> was identified near the peak SNP (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). It was proposed to encode a pectinesterase (SoyBase, <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> www.soybase.org </ext-link> ). Pectin is a structurally complex polysaccharide contained in primary cell walls of plant and has functions in plant growth, morphology and plant defense [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. Pectinesterase catalyzes the de-esterification of pectin into pectate and methanol, and plays important roles in some physiological processes such as stem elongation that requires rearrangement of cell wall architecture. Transient stem elongation was observed in potato plants overexpressing a <italic> Petunia inflate </italic> pectinesterase [ <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. In <italic> Arabidopsis, </italic> a reduction in cross-linking of cell wall pectic polysaccharide resulted in dwarf phenotype [ <xref ref-type="bibr" rid="CR17"> 17 </xref> ]. More recent research showed that pectinesterase regulates cell growth and hypocotyl elongation in <italic> Arabidopsis </italic> by affecting the degree of pectin methyl-esterification [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. Therefore, <italic> <span class="gene">Glyma19g37180</span> </italic> encoding a putative pectinesterase was the most likely causal gene for PH24, a major-effect locus associated with plant height in soybean. Notably, unlike PH25, PH24 had no association with maturity, and selection of the desired allele of this locus might improve the <span class="trait">yield</span> potential of soybean without affecting the maturity dates.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, 27, 6, 18 and 27 loci associated with DTF, DTM, DFTM and PH were identified via GWAS, respectively. Thirty-five candidate genes were proposed, including a function-known gene ( <italic> <span class="gene">Dt1</span> </italic> ) and 16 genes orthologous to <italic> Arabidopsis </italic> genes functioning in similar traits. It evidently demonstrates the high efficiency of GWAS in dissecting complex traits in soybean. A medium number of SNPs generated from the SoySNP50K analysis is capable of capturing genome-wide allelic variation, and candidate genes are regionally accessible for crops like soybean with a low LD decay rate. The genetic variants and trait-associated SNPs identified in this study will be useful for soybean cultivar improvement, especially for major-effect loci associated with PH that may have great potential for soybean <span class="trait">yield</span> improvement. Additionally, biological validation of the candidate genes will be also of great interest.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Intersubgeneric hybridization between Glycine max and G. tomentella: production of F₁, amphidiploid, BC₁, BC₂, BC₃, and fertile soybean plants.</h2>
			<h3>PMID:25835560  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Singh RJ(1), Nelson RL.
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: This paper describes methods for unlocking genetic treasure from wild perennial Glycine species of Australia for soybean improvement. The genetic resources of the ca. 26 species of the genus Glycine subgenus Glycine have not been exploited to broaden the genetic base of soybean (Glycine max; 2n = 40). The objectives of this study were to develop methods for producing F1, amphidiploid, BC1, BC2, BC3, and fertile soybean plants from crosses of soybean and the genus Glycine subgenus Glycine species, in order to utilize the subgenus Glycine germplasm in soybean breeding. Soybean cultivars were hybridized with six accessions of 78-chromosome G. tomentella as well as one accession each of 40-chromosome G. tomentella, G. argyrea and G. latifolia. They were chosen because they exhibit <span class="trait">resistance</span> to soybean <span class="trait">rust.</span> We were successful in producing fertile soybean from soybean cv. 'Dwight' and 78-chromosome G. tomentella accession PI 441001, while other hybrids were discontinued either at F1 or amphidiploid stage. The F1 seeds aborted prior to reaching maturity, so developing seeds from 19 to 21 day old pods were cultured aseptically in various media formulations. Seed maturation and multiple embryo generation media were developed. F1 plants with shoots and roots (2n = 59) were transplanted to pots in greenhouse. Amphidiploid (2n = 118) plants were backcrossed to 'Dwight'. <span class="gene">BC1</span> (2n = 79) plants were propagated through in vitro and 43 mature BC2F1 seeds were harvested. Fifteen surviving BC2F1 plants were morphologically distinct, sterile, and had chromosome numbers ranging 2n = 56-59. Chromosome numbers of the BC3F1 plants ranged 2n = 40-49. Derived fertile soybeans were first planted in the field in 2008 and are being evaluated for <span class="trait">yield,</span> <span class="trait">resistance</span> to pathogens and pests and tolerance to salt through material transfer agreement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Loci and candidate gene identification for <span class="trait">resistance</span> to Sclerotinia sclerotiorum in soybean (Glycine max L. Merr.) via association and linkage maps.</h2>
			<h3>PMID:25736370  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao X(1), Han Y, Li Y, Liu D, Sun M, Zhao Y, Lv C, Li D, Yang Z, Huang L, Teng W, Qiu L, Zheng H, Li W.
				<br/> 
				年份：2015 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean white mold (SWM), caused by Sclerotinia sclerotiorum ((Lib.) W. Phillips), is currently considered to be the second most important cause of soybean <span class="trait">yield</span> loss due to disease. Research is needed to identify SWM-resistant germplasm and gain a better understanding of the genetic and molecular basis of SWM <span class="trait">resistance</span> in soybean. Stem pigmentation after treatment with oxaloacetic acid is an effective indicator of <span class="trait">resistance</span> to SWM. A total of 128 recombinant inbred lines (RILs) derived from a cross of 'Maple Arrow' (partial resistant to SWM) and 'Hefeng 25' (susceptible) and 330 diverse soybean cultivars were screened for the soluble pigment concentration of their stems, which were treated with oxalic acid. Four quantitative trait loci (QTLs) underlying soluble pigment concentration were detected by linkage mapping of the RILs. Three hundred and thirty soybean cultivars were sequenced using the whole-genome encompassing approach and 25 179 single-nucleotide polymorphisms (SNPs) were detected for the fine mapping of SWM <span class="trait">resistance</span> genes by genome-wide association studies. Three out of five SNP markers representing a linkage disequilibrium (LD) block and a single locus on chromosome 13 <span class="gene">(Gm13)</span> were significantly associated with the soluble pigment content of stems. Three more SNPs that represented three minor QTLs for the soluble pigment content of stems were identified on another three chromosomes by association mapping. A major locus with the largest effect on <span class="gene">Gm13</span> was found both by linkage and association mapping. Four potential candidate genes involved in disease response or the anthocyanin biosynthesis pathway were identified at the locus near the significant SNPs (&lt;60 kbp). The beneficial allele and candidate genes should be useful in soybean breeding for improving <span class="trait">resistance</span> to SWM.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean DREB1/CBF-type transcription factors function in heat and <span class="trait">drought</span> as well as cold stress-responsive gene expression.</h2>
			<h3>PMID:25495120  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kidokoro S(1), Watanabe K, Ohori T, Moriwaki T, Maruyama K, Mizoi J, Myint Phyu Sin Htwe N, Fujita Y, Sekita S, Shinozaki K, Yamaguchi-Shinozaki K.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max) is a globally important crop, and its growth and <span class="trait">yield</span> are severely reduced by abiotic stresses, such as <span class="trait">drought,</span> heat, and cold. The cis-acting element DRE (dehydration-responsive element)/CRT plays an important role in activating gene expression in response to these stresses. The Arabidopsis DREB1/CBF genes that encode DRE-binding proteins function as transcriptional activators in the cold stress responsive gene expression. In this study, we identified 14 DREB1-type transcription factors <span class="gene">(GmDREB1s)</span> from a soybean genome database. The expression of most <span class="gene">GmDREB1</span> genes in soybean was strongly induced by a variety of abiotic stresses, such as cold, <span class="trait">drought,</span> high salt, and heat. The <span class="gene">GmDREB1</span> proteins activated transcription via DREs (dehydration-responsive element) in Arabidopsis and soybean protoplasts. Transcriptome analyses using transgenic Arabidopsis plants overexpressing <span class="gene">GmDREB1s</span> indicated that many of the downstream genes are cold-inducible and overlap with those of Arabidopsis DREB1A. We then comprehensively analyzed the downstream genes of <span class="gene">GmDREB1B;1,</span> which is closely related to DREB1A, using a transient expression system in soybean protoplasts. The expression of numerous genes induced by various abiotic stresses were increased by overexpressing <span class="gene">GmDREB1B;1</span> in soybean, and DREs were the most conserved element in the promoters of these genes. The downstream genes of <span class="gene">GmDREB1B;1</span> included numerous soybean-specific stress-inducible genes that encode an ABA receptor family protein, <span class="gene">GmPYL21,</span> and translation-related genes, such as ribosomal proteins. We confirmed that <span class="gene">GmDREB1B;1</span> directly activates <span class="gene">GmPYL21</span> expression and enhances ABRE-mediated gene expression in an ABA-independent manner. These results suggest that <span class="gene">GmDREB1</span> proteins activate the expression of numerous soybean-specific stress-responsive genes under diverse abiotic stress conditions.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of an osmotin-like protein from Solanum nigrum confers <span class="trait">drought</span> tolerance in transgenic soybean.</h2>
			<h3>PMID:25492565  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Weber RL, Wiebke-Strohm B, Bredemeier C, Margis-Pinheiro M, de Brito GG, Rechenmacher C, Bertagnolli PF, de Sá ME, Campos Mde A, de Amorim RM, Beneventi MA, Margis R, Grossi-de-Sa MF, Bodanese-Zanettini MH.
				<br/> 
				年份：2014 Dec 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: <span class="trait">Drought</span> is by far the most important environmental factor contributing to <span class="trait">yield</span> losses in crops, including soybeans [Glycine max (L.) Merr.]. To address this problem, a gene that encodes an osmotin-like protein isolated from Solanum nigrum var. americanum (SnOLP) driven by the UBQ3 promoter from Arabidopsis thaliana was transferred into the soybean genome by particle bombardment. RESULTS: Two independently transformed soybean lines expressing SnOLP were produced. Segregation analyses indicated single-locus insertions for both lines. qPCR analysis suggested a single insertion of SnOLP in the genomes of both transgenic lines, but one copy of the <span class="gene">hpt</span> gene was inserted in the first line and two in the second line. Transgenic plants exhibited no remarkable phenotypic alterations in the seven analyzed generations. When subjected to water deficit, transgenic plants performed better than the control ones. Leaf physiological measurements revealed that transgenic soybean plants maintained higher leaf water potential at predawn, higher net CO2 assimilation rate, higher stomatal conductance and higher transpiration rate than non-transgenic plants. Grain production and 100-grain weight were affected by water supply. Decrease in grain productivity and 100-grain weight were observed for both transgenic and non-transgenic plants under water deficit; however, it was more pronounced for non-transgenic plants. Moreover, transgenic lines showed significantly higher 100-grain weight than non-transgenic plants under water shortage. CONCLUSIONS: This is the first report showing that expression of SnOLP in transgenic soybeans improved physiological responses and <span class="trait">yield</span> components of plants when subjected to water deficit, highlighting the potential of this gene for biotechnological applications.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular basis of a shattering <span class="trait">resistance</span> boosting global dissemination of soybean.</h2>
			<h3>PMID:25468966  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Funatsuki H(1), Suzuki M(2), Hirose A(3), Inaba H(2), Yamada T(4), Hajika M(4), Komatsu K(3), Katayama T(5), Sayama T(6), Ishimoto M(6), Fujino K(7).
				<br/> 
				年份：2014 Dec 16
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Pod dehiscence (shattering) is essential for the propagation of wild plant species bearing seeds in pods but is a major cause of <span class="trait">yield</span> loss in legume and crucifer crops. Although natural genetic variation in pod dehiscence has been, and will be, useful for plant breeding, little is known about the molecular genetic basis of shattering <span class="trait">resistance</span> in crops. Therefore, we performed map-based cloning to unveil a major quantitative trait locus (QTL) controlling pod dehiscence in soybean. Fine mapping and complementation testing revealed that the QTL encodes a dirigent-like protein, designated as Pdh1. The gene for the shattering-resistant genotype, pdh1, was defective, having a premature stop codon. The functional gene, Pdh1, was highly expressed in the lignin-rich inner sclerenchyma of pod walls, especially at the stage of initiation in lignin deposition. Comparisons of near-isogenic lines indicated that <span class="gene">Pdh1</span> promotes pod dehiscence by increasing the torsion of dried pod walls, which serves as a driving force for pod dehiscence under low humidity. A survey of soybean germplasm revealed that pdh1 was frequently detected in landraces from semiarid regions and has been extensively used for breeding in North America, the world's leading soybean producer. These findings point to a new mechanism for pod dehiscence involving the dirigent protein family and suggest that pdh1 has played a crucial role in the global expansion of soybean cultivation. Furthermore, the orthologs of pdh1, or genes with the same role, will possibly be useful for crop improvement.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of a cystatin transgene can confer <span class="trait">resistance</span> to root lesion nematodes in Lilium longiflorum cv. 'Nellie White'.</h2>
			<h3>PMID:25398618  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vieira P(1), Wantoch S, Lilley CJ, Chitwood DJ, Atkinson HJ, Kamo K.
				<br/> 
				年份：2015 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Lilium longiflorum cv. 'Nellie White' assumes a great economic importance as cut flowers, being one of the most valuable species (annual pot plants value above $20,000,000) in terms of wholesales in the US. The root lesion nematode Pratylenchus penetrans (RLN) constitutes one of the main pests for lily producers due to the significant root damage it causes. Our efforts have focused on the generation of soybean hairy roots (as a transient test model) and stable transgenic lilies overexpressing a modified rice cystatin (Oc-IΔD86) transgene and challenged with root lesion nematodes. Lily transformation was achieved by gene gun co-bombardment using both a pBluescript-based vector containing the cystatin gene and pDM307 that contains a <span class="gene">bar</span> gene for phosphinothricin selection. Both soybean hairy roots and lilies overexpressing the OcIΔD86 transgene exhibited enhanced <span class="trait">resistance</span> to RLN infection by means of nematode reduction up to 75 ± 5% on the total number of nematodes. In addition, lily plants overexpressing OcIΔD86 displayed an increase of plant mass and better growth performance in comparison to wild-type plants, thereby demonstrating an alternative strategy for increasing the <span class="trait">yield</span> and reducing nematode damage to this important floral crop.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evolution and association analysis of <span class="gene">GmCYP78A10</span> gene with seed size/weight and pod number in soybean.</h2>
			<h3>PMID:25324172  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Wang X(1), Li Y, Zhang H, Sun G, Zhang W, Qiu L.
				<br/> 
				年份：2015 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Seed-size/weight traits, controlled by multiple genes in soybean, play an important role in determining seed <span class="trait">yield.</span> However, the molecular mechanisms controlling the seed size and weight in soybean remain unclear. In Arabidopsis, P450/CYP78A gene family has been proved extremely relevant to seed size (such as AtCYP78A5, AtCYP78A6 and AtCYP78A9). We found that a soybean <span class="gene">GmCYP78A10</span> gene underwent artificial selection during soybean breeding. The <span class="gene">GmCYP78A10a</span> allele mainly distributed in wild soybean (Glycine soja), but has been eliminated in the cultivars during early stage of soybean breeding, while the <span class="gene">GmCYP78A10b</span> allele has been accumulated and become the predominant allele in cultivated soybean (G. max). ANOVA analysis showed that the mean seed weight, seed width and seed thickness of soybean varieties with <span class="gene">GmCYP78A10b</span> allele was significantly heavier/bigger than those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01). The allele could explain 7.2 % variation in seed weight. The pod number of the soybeans with <span class="gene">GmCYP78A10b</span> allele significantly decreased compared to those with <span class="gene">GmCYP78A10a</span> allele (P &lt; 0.01, R(2) = 5.8 %), while other agronomic traits including seed weight/plant were not significantly affected by these two alleles. We speculated that during the early stage of soybean breeding, breeders selected big seed carrying <span class="gene">GmCYP78A10b</span> allele, but lowered pod number simultaneously. Overall, the selection did not cause the significantly change in soybean seed <span class="trait">yield.</span> Our results suggests that the soybean <span class="gene">GmCYP78A10</span> gene may have a similar function to those genes belonging to P450/CYP78A subfamily in Arabidopsis and provides new information for the genetic control of seed size in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Association Studies Identifies Seven Major Regions Responsible for Iron Deficiency <span class="trait">Chlorosis</span> in Soybean ( <italic> Glycine max </italic> )</h2>
			<h3>PMID:25225893</h3>
				

				
					<h3>PMCID:4166409</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Mamidi </surname>  <given-names> Sujan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Rian K. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Goos </surname>  <given-names> Jay R. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McClean </surname>  <given-names> Phillip E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Iron deficiency <span class="trait">chlorosis</span> (IDC) is a <span class="trait">yield</span> limiting problem in soybean (Glycine max (L.) Merr) production regions with calcareous soils. Genome-wide association study (GWAS) was performed using a high density SNP map to discover significant markers, QTL and candidate genes associated with IDC trait variation. A stepwise regression model included eight markers after considering LD between markers, and identified seven major effect QTL on seven chromosomes. Twelve candidate genes known to be associated with iron metabolism mapped near these QTL supporting the polygenic nature of IDC. A non-synonymous substitution with the highest significance in a major QTL region suggests soybean orthologs of FRE1 on <span class="gene">Gm03</span> is a major gene responsible for trait variation. NAS3, a gene that encodes the enzyme nicotianamine synthase which synthesizes the iron chelator nicotianamine also maps to the same QTL region. Disease resistant genes also map to the major QTL, supporting the hypothesis that pathogens compete with the plant for Fe and increase iron deficiency. The markers and the allelic combinations identified here can be further used for marker assisted selection.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Other than the genes involved in iron metabolism, several genes involved in disease <span class="trait">resistance</span> are also identified in the QTL regions. For example, QTL on <span class="gene">Gm03</span> has two RING/U-box superfamily proteins that have a role in plant defense. Similarly on <span class="gene">Gm05,</span> two CC-NBS-LRR, one NBS and one LRR genes are present. On <span class="gene">Gm07</span> within the QTL region, three NBS genes are present. On <span class="gene">Gm16,</span> seven LRR proteins, and on <span class="gene">Gm19,</span> six NBS proteins, two LRR and one RING/U-box superfamily proteins are present.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Is there potential to adapt soybean (Glycine max Merr.) to future [CO₂]? An analysis of the <span class="trait">yield</span> response of 18 genotypes in free-air CO₂ enrichment.</h2>
			<h3>PMID:25211487  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bishop KA(1), Betzelberger AM(1), Long SP(1)(2), Ainsworth EA(1)(3).
				<br/> 
				年份：2015 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rising atmospheric [CO2] is a uniform, global change that increases <span class="gene">C3</span> photosynthesis and could offset some of the negative effects of global climate change on crop yields. Genetic variation in <span class="trait">yield</span> responsiveness to rising [CO2] would provide an opportunity to breed more responsive crop genotypes. A multi-year study of 18 soybean (Glycine max Merr.) genotypes was carried out to identify variation in responsiveness to season-long elevated [CO2] (550 ppm) under fully open-air replicated field conditions. On average across 18 genotypes, elevated [CO2] stimulated total above-ground biomass by 22%, but seed <span class="trait">yield</span> by only 9%, in part because most genotypes showed a reduction in partitioning of energy to seeds. Over four years of study, there was consistency from year to year in the genotypes that were most and least responsive to elevated [CO2], suggesting heritability of CO2 response. Further analysis of six genotypes did not reveal a photosynthetic basis for the variation in <span class="trait">yield</span> response. Although partitioning to seed was decreased, cultivars with the highest partitioning coefficient in current [CO2 ] also had the highest partitioning coefficient in elevated [CO2]. The results show the existence of genetic variation in soybean response to elevated [CO2], which is needed to breed soybean to the future atmospheric environment.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Constitutive overexpression of soybean plasma membrane intrinsic protein <span class="gene">GmPIP1;6</span> confers salt tolerance</h2>
			<h3>PMID:24998596</h3>
				

				
					<h3>PMCID:4105146</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Zhou </surname>  <given-names> Lian </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> zhoulian@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Wang </surname>  <given-names> Chuang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> w1983c@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Liu </surname>  <given-names> Ruifang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> liuruifang@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Han </surname>  <given-names> Qiang </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> han-qiang@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Vandeleur </surname>  <given-names> Rebecca K </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> rebecca.coleman@adelaide.edu.au </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Du </surname>  <given-names> Juan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> djuan@zju.edu.cn </email>  </contrib>  <contrib contrib-type="author" id="A7">  <name>  <surname> Tyerman </surname>  <given-names> Steven </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> steve.tyerman@adelaide.edu.au </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A8">  <name>  <surname> Shou </surname>  <given-names> Huixia </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> huixia@zju.edu.cn </email>  </contrib>
				<br/> 
				年份：<day> 30 </day>  <month> 6 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Under saline conditions, plant growth is depressed via osmotic stress and salt can accumulate in leaves leading to further depression of growth due to reduced photosynthesis and gas exchange. Aquaporins are proposed to have a major role in growth of plants via their impact on root water uptake and leaf gas exchange. In this study, soybean plasma membrane intrinsic protein 1;6 <span class="gene">(GmPIP1;6)</span> was constitutively overexpressed to evaluate the function of <span class="gene">GmPIP1;6</span> in growth regulation and salt tolerance in soybean. RESULTS: <span class="gene">GmPIP1;6</span> is highly expressed in roots as well as reproductive tissues and the protein targeted to the plasma membrane in onion epidermis. Treatment with 100 mM NaCl resulted in reduced expression initially, then after 3 days the expression was increased in root and leaves. The effects of constitutive overexpression of <span class="gene">GmPIP1;6</span> in soybean was examined under normal and salt stress conditions. Overexpression in 2 independent lines resulted in enhanced leaf gas exchange, but not growth under normal conditions compared to wild type (WT). With 100 mM NaCl, net assimilation was much higher in the <span class="gene">GmPIP1;6-Oe</span> and growth was enhanced relative to WT. <span class="gene">GmPIP1;6-Oe</span> plants did not have higher root hydraulic conductance (Lo) under normal conditions, but were able to maintain Lo under saline conditions compared to WT which decreased Lo. <span class="gene">GmPIP1;6-Oe</span> lines grown in the field had increased <span class="trait">yield</span> resulting mainly from increased seed size. CONCLUSIONS: The general impact of overexpression of <span class="gene">GmPIP1;6</span> suggests that it may be a multifunctional aquaporin involved in root water transport, photosynthesis and seed loading. <span class="gene">GmPIP1;6</span> is a valuable gene for genetic engineering to improve soybean <span class="trait">yield</span> and salt tolerance.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> <span class="gene">GmPIP1;6</span> </italic> is highly expressed in roots as well as reproductive tissues and the protein targeted to the plasma membrane in onion epidermis. Treatment with 100 mM NaCl resulted in reduced expression initially, then after 3 days the expression was increased in root and leaves. The effects of constitutive overexpression of <italic> <span class="gene">GmPIP1;6</span> </italic> in soybean was examined under normal and salt stress conditions. Overexpression in 2 independent lines resulted in enhanced leaf gas exchange, but not growth under normal conditions compared to wild type (WT). With 100 mM NaCl, net assimilation was much higher in the <italic> <span class="gene">GmPIP1;6</span> </italic> -Oe and growth was enhanced relative to WT. <italic> <span class="gene">GmPIP1;6</span> </italic> -Oe plants did not have higher root hydraulic conductance ( <italic> L </italic> <sub> o </sub> ) under normal conditions, but were able to maintain <italic> L </italic> <sub> o </sub> under saline conditions compared to WT which decreased <italic> L </italic> <sub> o </sub> . <italic> <span class="gene">GmPIP1;6</span> </italic> -Oe lines grown in the field had increased <span class="trait">yield</span> resulting mainly from increased seed size.
					</p>
				
					<p>&nbsp;&nbsp;
						The general impact of overexpression of <italic> <span class="gene">GmPIP1;6</span> </italic> suggests that it may be a multifunctional aquaporin involved in root water transport, photosynthesis and seed loading. <italic> <span class="gene">GmPIP1;6</span> </italic> is a valuable gene for genetic engineering to improve soybean <span class="trait">yield</span> and salt tolerance.
					</p>
				
					<p>&nbsp;&nbsp;
						Soybean is a major source of protein and oil for humans and animals, yet relatively mild salt stress significantly reduces soybean growth, nodulation, seed quality and <span class="trait">yield</span> [ <xref ref-type="bibr" rid="B41"> 41 </xref> ]. Recently it was found that the expression of <italic> <span class="gene">GmPIP1;6</span> </italic> in roots correlated with rapid and longer term changes in root <italic> L </italic> <sub> o </sub> in response to shoot treatments and appeared to be more concentrated in stellar tissue [ <xref ref-type="bibr" rid="B42"> 42 </xref> ]. These results indicated that <span class="gene">GmPIP1;6</span> may be important in the control of root water transport particularly in response to shoot signals. In this study, <italic> <span class="gene">GmPIP1;6</span> </italic> was cloned and functionally characterized. Overexpression of <italic> <span class="gene">GmPIP1;6</span> </italic> significantly increased salt tolerance of soybean by improving root <italic> L </italic> <sub> o </sub> and Na <sup> + </sup> exclusion.
					</p>
				
					<p>&nbsp;&nbsp;
						Four independent <italic> <span class="gene">GmPIP1;6</span> </italic> -Oe lines, where two of these lines were the same as the ones used in the physiology experiments, were grown in field conditions for an entire growing season in two continuous years. Each year, the transgenic plants were compared with WT and null transgenic plants, which were segregated from heterozygous transgenic plants. Interestingly, <italic> <span class="gene">GmPIP1;6</span> </italic> -Oe plants have a significantly higher seed weight per plant and per 100 seeds than WT (Table&#x000a0; <xref ref-type="table" rid="T3"> 3 </xref> ). Other <span class="trait">yield</span> parameters were similar between WT and <italic> <span class="gene">GmPIP1;6</span> </italic> -Oe plants (Table&#x000a0; <xref ref-type="table" rid="T3"> 3 </xref> , Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S6A). A detailed analysis showed that the increased seed weight of <italic> <span class="gene">GmPIP1;6</span> </italic> -Oe plants was caused by large seed size (Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S6B, Table&#x000a0; <xref ref-type="table" rid="T3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Water stress caused by <span class="trait">drought,</span> salt or cold has a complex effect on the expression of <italic> AQP </italic> genes [ <xref ref-type="bibr" rid="B36"> 36 </xref> ]. In summary, the expression of <italic> AQP </italic> genes could be divided into two stages. In the early stress response, the plant usually suppresses the expression of <italic> <span class="gene">PIP</span> </italic> genes, which is hypothesised to avoid water flow from the root to the soil when the soil water potential decreases [ <xref ref-type="bibr" rid="B45"> 45 </xref> , <xref ref-type="bibr" rid="B46"> 46 </xref> ]. After a few days of acclimation, the expression of <italic> <span class="gene">PIP</span> </italic> genes recovers or even increases and is correlated with increased hydraulic conductance [ <xref ref-type="bibr" rid="B47"> 47 </xref> - <xref ref-type="bibr" rid="B49"> 49 </xref> ]. The expression of <italic> <span class="gene">GmPIP1;6</span> </italic> in both roots and leaves showed this two stage response under salt stress (Figure&#x000a0; <xref ref-type="fig" rid="F2"> 2 </xref> B, Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S2), indicating <italic> <span class="gene">GmPIP1;6</span> </italic> may be involved in the salt stress acclimation of soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						Overexpression of several <italic> PIP1 </italic> genes increased the hydraulic conductance and salt tolerance of the transgenic plants, such as <italic> NtAQP1 </italic> , <italic> OsPIP1;1 </italic> , <italic> TaAQP8 </italic> and <italic> MusaPIP1;2 </italic> [ <xref ref-type="bibr" rid="B17"> 17 </xref> , <xref ref-type="bibr" rid="B37"> 37 </xref> , <xref ref-type="bibr" rid="B40"> 40 </xref> , <xref ref-type="bibr" rid="B50"> 50 </xref> ]. Here we show that <italic> <span class="gene">GmPIP1;6</span> </italic> conferred salt tolerance, but also under normal conditions the overexpression resulted in higher growth and greater <span class="trait">yield</span> under field conditions compared to WT plants (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> A, Table&#x000a0; <xref ref-type="table" rid="T3"> 3 </xref> ). However, the mechanism of how these <italic> PIP1 </italic> genes can improve plant growth and salt tolerance is largely unknown, though a high K <sup> + </sup> /Na <sup> + </sup> ratio was mentioned with overexpression of <italic> TaAQP8 </italic> [ <xref ref-type="bibr" rid="B37"> 37 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to function as a water channel, AtPIP1;2 and NtAQP1 may function to facilitate CO <sub> 2 </sub> transport and enhance photosynthesis by increasing the mesophyll conductance to CO <sub> 2 </sub> diffusion [ <xref ref-type="bibr" rid="B55"> 55 </xref> - <xref ref-type="bibr" rid="B59"> 59 </xref> ]. Overexpression of <italic> NtAQP1 </italic> in tobacco and tomato increased the A <sub> N </sub> , which resulted in increased WUE. The overexpression of <italic> NtAQP1 </italic> produced higher dry biomass and <span class="trait">yield</span> under normal irrigation and salt stressed conditions [ <xref ref-type="bibr" rid="B17"> 17 </xref> ]. <italic> <span class="gene">GmPIP1;6-Oe</span> </italic> plants also exhibited higher A <sub> N </sub> , g <sub> s </sub> and IWUE than WT under both normal and saline conditions (Figure&#x000a0; <xref ref-type="fig" rid="F6"> 6 </xref> , Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). However, the growth of <italic> <span class="gene">GmPIP1;6-Oe</span> </italic> plants was only enhanced under saline conditions compared to WT plants (Figure&#x000a0; <xref ref-type="fig" rid="F4"> 4 </xref> A, Table&#x000a0; <xref ref-type="table" rid="T1"> 1 </xref> ). Whether <span class="gene">GmPIP1;6</span> has a similar function as NtAQP1 to facilitate CO <sub> 2 </sub> diffusion across leaf cell membranes requires further research.
					</p>
				
					<p>&nbsp;&nbsp;
						Importantly, <italic> <span class="gene">GmPIP1;6-Oe</span> </italic> plants showed higher <span class="trait">yield</span> in the field than WT because the seed weight and size of <italic> <span class="gene">GmPIP1;6-Oe</span> </italic> were increased (Table&#x000a0; <xref ref-type="table" rid="T3"> 3 </xref> , Additional file <xref ref-type="supplementary-material" rid="S1"> 1 </xref> : Figure S6). This may be reflecting the higher net assimilation, but also may indicate sink limitation of seed loading that could be enhanced by greater water permeability in the seed loading process [ <xref ref-type="bibr" rid="B60"> 60 </xref> ]. In addition to being highly expressed in roots and stems, the transcripts of <italic> <span class="gene">GmPIP1;6-Oe</span> </italic> were abundant in flower and pod, which supports a role of <italic> <span class="gene">GmPIP1;6-Oe</span> </italic> in seed loading of assimilates via enhanced water permeability.
					</p>
				
					<p>&nbsp;&nbsp;
						In this study, the function of <italic> <span class="gene">GmPIP1;6</span> </italic> was analyzed by constitutive expressing in the soybean plants. The expression of <italic> <span class="gene">GmPIP1;6</span> </italic> was influenced by salt stress. Overexpression of <italic> <span class="gene">GmPIP1;6</span> </italic> improved salt tolerance of transgenic plants by increasing water transport, photosynthesis and Na <sup> + </sup> exclusion. Moreover, the <span class="trait">yield</span> of <italic> <span class="gene">GmPIP1;6</span> </italic> overexpression plants was improved in the field indicating the potential of <italic> <span class="gene">GmPIP1;6</span> </italic> in genetic engineering of soybean.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Deletions of the <italic> <span class="gene">SACPD-C</span> </italic> locus elevate seed stearic acid levels but also result in fatty acid and morphological alterations in nitrogen fixing nodules</h2>
			<h3>PMID:24886084</h3>
				

				
					<h3>PMCID:4058718</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" equal-contrib="yes" id="A1">  <name>  <surname> Gillman </surname>  <given-names> Jason D </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> Jason.Gillman@ars.usda.gov </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Stacey </surname>  <given-names> Minviluz G </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> StaceyM@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Cui </surname>  <given-names> Yaya </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> CuiY@missouri.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Berg </surname>  <given-names> Howard R </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> RHBerg@danforthcenter.org </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Stacey </surname>  <given-names> Gary </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> StaceyG@missouri.edu </email>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 5 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean (Glycine max) seeds are the primary source of edible oil in the United States. Despite its widespread utility, soybean oil is oxidatively unstable. Until recently, the majority of soybean oil underwent chemical hydrogenation, a process which also generates trans fats. An alternative to chemical hydrogenation is genetic modification of seed oil through identification and introgression of mutant alleles. One target for improvement is the elevation of a saturated fat with no negative cardiovascular impacts, stearic acid, which typically constitutes a minute portion of seed oil (~3%). RESULTS: We examined radiation induced soybean mutants with moderately increased stearic acid (10-15% of seed oil, ~3-5 X the levels in wild-type soybean seeds) via comparative whole genome hybridization and genetic analysis. The deletion of one <span class="gene">SACPD</span> isoform encoding gene (SACPD-C) was perfectly correlated with moderate elevation of seed stearic acid content. However, <span class="gene">SACPD-C</span> deletion lines were also found to have altered nodule fatty acid composition and grossly altered morphology. Despite these defects, overall nodule accumulation and nitrogen fixation were unaffected, at least under laboratory conditions. CONCLUSIONS: Although no <span class="trait">yield</span> penalty has been reported for moderate elevated seed stearic acid content in soybean seeds, our results demonstrate that genetic alteration of seed traits can have unforeseen pleiotropic consequences. We have identified a role for fatty acid biosynthesis, and <span class="gene">SACPD</span> activity in particular, in the establishment and maintenance of symbiotic nitrogen fixation.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Although no <span class="trait">yield</span> penalty has been reported for moderate elevated seed stearic acid content in soybean seeds, our results demonstrate that genetic alteration of seed traits can have unforeseen pleiotropic consequences. We have identified a role for fatty acid biosynthesis, and <span class="gene">SACPD</span> activity in particular, in the establishment and maintenance of symbiotic nitrogen fixation.
					</p>
				
					<p>&nbsp;&nbsp;
						Mutant soybean lines with elevated seed stearic acid content were first reported in the 1980&#x02019;s. One sodium azide induced mutant line, A6, has a remarkable ~28% of the total seed oil present in the form of stearic acid (~8 to 10 fold higher than conventional soybeans) [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B14"> 14 </xref> ]. The increase in stearic acid content of seeds in A6 [ <xref ref-type="bibr" rid="B13"> 13 </xref> , <xref ref-type="bibr" rid="B14"> 14 </xref> ] was reported to be due solely to deletion of <italic> <span class="gene">SACPD-C</span> </italic> [ <xref ref-type="bibr" rid="B12"> 12 </xref> ]. Unfortunately, a significant negative correlation was found between elevated stearic acid content and seed <span class="trait">yield</span> using the A6 mutant line. Additional mutant sources with slightly less stearic acid content (~11 to 15%) do not have the same negative association with seed <span class="trait">yield</span> [ <xref ref-type="bibr" rid="B15"> 15 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Inheritance of the elevated stearic trait from A6 has been reported to be associated with reduced seed <span class="trait">yield</span> [ <xref ref-type="bibr" rid="B15"> 15 </xref> ] and greater than normal sensitivity to temperature stresses [ <xref ref-type="bibr" rid="B32"> 32 </xref> ]. Although unproven, it is very likely that the seed <span class="trait">yield</span> reduction and temperature sensitivity of A6 may be due to the extremely large deletion affecting chromosome 14 (Figure&#x000a0; <xref ref-type="fig" rid="F3"> 3 </xref> ). Studies with mutations allelic to A6 (Walter Fehr, personal communication 2012) but which only accumulate 10-15% stearic acid, failed to reveal a correlation with reduced protein content or reduced seed <span class="trait">yield</span> [ <xref ref-type="bibr" rid="B15"> 15 </xref> ]. These deleterious effects are not apparently due to the loss of <italic> <span class="gene">SACPD-C</span> </italic> , though the deletion may contain other linked genes whose loss results in these effects.
					</p>
				
					<p>&nbsp;&nbsp;
						Root nodule senescence is often initiated by stress conditions such as extremes of temperature, <span class="trait">drought,</span> pathogen or heavy metals [ <xref ref-type="bibr" rid="B38"> 38 </xref> ]. This is consistent with the known function of polyunsaturated fatty acids in enhancing the ability of plants to tolerate environmental stresses [ <xref ref-type="bibr" rid="B47"> 47 </xref> , <xref ref-type="bibr" rid="B51"> 51 </xref> ]. Although several early nodule senescence mutants have been identified [ <italic> e.g </italic> . [ <xref ref-type="bibr" rid="B52"> 52 </xref> - <xref ref-type="bibr" rid="B54"> 54 </xref> ]], and the value of nodulation is well documented in agriculture, relatively little is known about developmental and stress-induced nodule senescence. It is probable that <span class="gene">SACPD-C</span> contributes to significantly increased nodule sustainability under field conditions. Nevertheless, the full role of <span class="gene">SACPD</span> function in the interaction of soybean with symbiotic bacteria remains to be elucidated.
					</p>
				
					<p>&nbsp;&nbsp;
						Previous studies had reported that deletion of one specific soybean <italic> Stearoyl Acyl Carrier Protein Desaturase </italic> gene ( <italic> <span class="gene">SACPD-C</span> </italic> ) as solely causative for highly increased (~28%) seed stearic acid in mutant line A6. We investigated a series of five independent mutation events with moderate increases (10-15%) in seed stearic acid content, which arose from multiple genetic backgrounds and/or mutagenic agents, using comparative genome hybridization and targeted sequencing of <italic> <span class="gene">SACPD</span> </italic> genes. In contrast to expectation, all lines with moderately elevated seed stearic acid bear deletions or loss-of-function mutations affecting <italic> <span class="gene">SACPD-C</span> </italic> . A6 was found to have multiple, extremely large genomic deletions; the deletion of <italic> SAPCD-C </italic> in A6 consists of ~1/8 of chromosome 14 and contains at least 56 genes. Defective nodule development is unlikely to explain the <span class="trait">yield</span> drag seen in the A6 deletion line since all the mutants examined were altered in <italic> SAPD-C </italic> function but not all <italic> <span class="gene">SACPD-C</span> </italic> mutants show a reduction in <span class="trait">yield.</span> Therefore, it is more likely that the extraordinarily large deletion may explain A6&#x02019;s extremely poor agronomic characteristics, which has hindered commercialization of high seed stearic acid lines. Another, independent locus must be deleted or mutated in line A6, which acts synergistically with the <italic> <span class="gene">SACPD-C</span> </italic> deletion to elevate seed stearic acid from ~12.5% to ~28%.
					</p>
				
					<p>&nbsp;&nbsp;
						KK24, MM106 and M25 [ <xref ref-type="bibr" rid="B16"> 16 </xref> , <xref ref-type="bibr" rid="B17"> 17 </xref> ] were generously provided by Dr. Toyoaki Anai and the Legumebase seed repository. These lines were developed by X-ray mutagenesis performed on seeds of soybean cultivar &#x02018;Bay&#x02019; [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] and identified in forward genetic screens for alteration in fatty acid composition. Seeds from lines A6 and &#x02018;FA 8077&#x02019; [ <xref ref-type="bibr" rid="B14"> 14 </xref> ] were kindly provided by Dr. Walter Fehr of Iowa State University. FN8 was derived from cultivar &#x02018;Williams 82&#x02019; seed [ <xref ref-type="bibr" rid="B23"> 23 </xref> ] dosed in 2007 with 30&#x000a0;Gy fast neutrons at the McClellan Nuclear Radiation Center by Dr. Kristin Bilyeu (USDA-ARS) and a portion of the total population was kindly donated to Dr. Gary Stacey. Dosed seeds were advanced three generations and were screened for a chlorotic phenotype. One of the lines selected through this screen was line FN8. Subsequent analyses showed that the chlorotic phenotype is not associated with the deletion in chromosome 14 encoding <italic> <span class="gene">SACPD-C</span> </italic> since the two traits ( <italic> i.e. </italic> <span class="trait">chlorosis</span> and high stearate) segregated independently in subsequent generations. Line 194D is derived from an Ethyl Methane Sulphonate (EMS) induced &#x02018;Williams 82&#x02019; mutant population used for TILLiNG [ <xref ref-type="bibr" rid="B55"> 55 </xref> ]; this population was kindly donated by Kristin Bilyeu (USDA-ARS). Mutations affecting <italic> <span class="gene">SACPD-C</span> </italic> were not detected during reverse genetic analysis but later identified through a forward genetic analysis of the mutant population for fatty acid alterations.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Evalution of DNA extraction methods in order to monitor genetically modified materials in soy foodstuffs and feeds commercialised in Turkey by multiplex real-time PCR.</h2>
			<h3>PMID:24816675  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Turkec A(1), Kazan H, Baykut A, Lucas SJ.
				<br/> 
				年份：2015 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Soybean is one of the most important biotech crops, widely used as an ingredient in both foodstuffs and feed. DNA extraction methods have been evaluated to detect the presence of genetically modified (GM) materials in soya-containing food and feed products commercialised in Turkey. RESULTS: All extraction methods performed well for the majority of soya foods and feed products analysed. However, the most successful method varied between different products; the Foodproof, Genespin and the cetyltrimethylammonium bromide (CTAB) methods each produced the highest DNA <span class="trait">yield</span> and purity for different soya foodstuffs and feeds. Of the samples tested, 20% were positive for the presence of at least two GM elements (35S/NOS) while 11% contained an additional GM element (35S/NOS/FMV). Of the tested products, animal feeds showed a larger prevalence of GM material (50%) than the soya-containing foodstuffs (13%). CONCLUSION: The best performing extraction methods proved to be the Foodproof, Genespin and CTAB methods for soya-containing food and feed products. The results obtained herein clearly demonstrate the presence of GM soybean in the Turkish market, and that the Foodproof <span class="gene">GMO</span> Screening Kit provides reliable screening of soy-containing food and feed products.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Induced over-expression of AtDREB2A CA improves <span class="trait">drought</span> tolerance in sugarcane.</h2>
			<h3>PMID:24656336  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reis RR(1), da Cunha BA(2), Martins PK(2), Martins MT(2), Alekcevetch JC(3), Chalfun A Jr(3), Andrade AC(4), Ribeiro AP(1), Qin F(5), Mizoi J(6), Yamaguchi-Shinozaki K(6), Nakashima K(5), Carvalho Jde F(7), de Sousa CA(2), Nepomuceno AL(7), Kobayashi AK(2), Molinari HB(8).
				<br/> 
				年份：2014 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is one of the most challenging agricultural issues limiting sustainable sugarcane production and, in some cases, <span class="trait">yield</span> losses caused by <span class="trait">drought</span> are nearly 50%. DREB proteins play vital regulatory roles in abiotic stress responses in plants. The transcription factor <span class="gene">DREB2A</span> interacts with a cis-acting DRE sequence to activate the expression of downstream genes that are involved in <span class="trait">drought-,</span> salt- and heat-stress response in Arabidopsis thaliana. In the present study, we evaluated the effects of stress-inducible over-expression of AtDREB2A CA on gene expression, leaf water potential (ΨL), relative water content (RWC), sucrose content and gas exchanges of sugarcane plants submitted to a four-days water deficit treatment in a rhizotron-grown root system. The plants were also phenotyped by scanning the roots and measuring morphological parameters of the shoot. The stress-inducible expression of AtDREB2A CA in transgenic sugarcane led to the up-regulation of genes involved in plant response to <span class="trait">drought</span> stress. The transgenic plants maintained higher RWC and ΨL over 4 days after withholding water and had higher photosynthetic rates until the 3rd day of water-deficit. Induced expression of AtDREB2A CA in sugarcane increased sucrose levels and improved bud sprouting of the transgenic plants. Our results indicate that induced expression of AtDREB2A CA in sugarcane enhanced its <span class="trait">drought</span> tolerance without biomass penalty.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and Characterization of the Brassinosteroid Receptor Gene ( <italic> <span class="gene">GmBRI1</span> </italic> ) from <italic> Glycine max </italic></h2>
			<h3>PMID:24599079</h3>
				

				
					<h3>PMCID:3975373</h3>
					<h4>期刊:International Journal of Molecular Sciences</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Miao </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-15-03871"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Shi </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-15-03871"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Cunxiang </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-15-03871"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Tianfu </given-names>  </name>  <xref ref-type="aff" rid="af2-ijms-15-03871"> 2 </xref>  <xref rid="c1-ijms-15-03871" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Qingyu </given-names>  </name>  <xref ref-type="aff" rid="af1-ijms-15-03871"> 1 </xref>  <xref rid="c1-ijms-15-03871" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 17 </day>  <month> 2 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Brassinosteroids (BRs) constitute a group of steroidal phytohormones that contribute to a wide range of plant growth and development functions. The genetic modulation of BR receptor genes, which play major roles in the BR signaling pathway, can create semi-dwarf plants that have great advantages in crop production. In this study, a brassinosteroid insensitive gene homologous with AtBRI1 and other BRIs was isolated from Glycine max and designated as <span class="gene">GmBRI1.</span> A bioinformatic analysis revealed that <span class="gene">GmBRI1</span> shares a conserved kinase domain and 25 tandem leucine-rich repeats (LRRs) that are characteristic of a BR receptor for BR reception and reaction and bear a striking similarity in protein tertiary structure to AtBRI1. <span class="gene">GmBRI1</span> transcripts were more abundant in soybean hypocotyls and could be upregulated in response to exogenous BR treatment. The transformation of <span class="gene">GmBRI1</span> into the Arabidopsis dwarf mutant bri1-5 restored the phenotype, especially regarding pod size and plant height. Additionally, this complementation is a consequence of a restored BR signaling pathway demonstrated in the light/dark analysis, root inhibition assay and BR-response gene expression. Therefore, <span class="gene">GmBRI1</span> functions as a BR receptor to alter BR-mediated signaling and is valuable for improving plant architecture and enhancing the <span class="trait">yield</span> of soybean.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Since brassinosteroids (BRs) were first isolated and purified from <italic> Brassica napus </italic> pollen in 1979 [ <xref rid="b39-ijms-15-03871" ref-type="bibr"> 39 </xref> ] and widely recognized as a novel phytohormone in the 1990s, components of BR signaling and biosynthesis have been identified in a broad range of species, but rarely in soybean, an economically important legume crop. In the 1990s, only two soybean genes, <italic> BRU1 </italic> [ <xref rid="b40-ijms-15-03871" ref-type="bibr"> 40 </xref> , <xref rid="b41-ijms-15-03871" ref-type="bibr"> 41 </xref> ] and <italic> SMT </italic> [ <xref rid="b42-ijms-15-03871" ref-type="bibr"> 42 </xref> ], were identified. This study is the first time that the <italic> BRI1 </italic> gene was isolated and characterized in soybean. Because the <italic> BRI </italic> gene plays a major role in BR signaling, which participates in plant development, and because the relationship between <italic> BRI </italic> and plant <span class="trait">yield</span> needs to be understood to further benefit production, the isolation and characterization of <italic> <span class="gene">GmBRI1</span> </italic> is only the beginning.
					</p>
				
					<p>&nbsp;&nbsp;
						The complementation of the <italic> bri1-5 </italic> mutant highlights the functionality of the <italic> <span class="gene">GmBRI1</span> </italic> gene that we isolated from soybean. <italic> <span class="gene">GmBRI1</span> </italic> expression in <italic> bri1-5 </italic> causes changes in the pod size and plant height, indicating that these two important traits could be controlled through <italic> <span class="gene">GmBRI1</span> </italic> gene modulation. There may exist a balance in which the plant height is reduced to a level to satisfy the production demands while maintaining or even increasing the <span class="trait">yield.</span> The evidence in rice that modifying the expression of the <italic> OsBRI1 </italic> gene can be used to generate semi-dwarf plants without defects in reproductive development supports this hypothesis [ <xref rid="b5-ijms-15-03871" ref-type="bibr"> 5 </xref> ]. Consequently, our data provide a foundation for future studies into the regulation of soybean production by BR signaling. The successful deciphering of the <italic> <span class="gene">GmBRI1</span> </italic> genes will contribute toward generating new soybean germplasm.
					</p>
				
					<p>&nbsp;&nbsp;
						The deduced <span class="gene">GmBRI1</span> has 69% identity with <italic> Arabidopsis </italic> AtBRI1. A sequence analysis indicated that the <span class="gene">GmBRI1</span> protein contains a kinase domain and 25 tandem LRRs, which are conserved among plant BRIs and are important for BR binding and reaction. The <span class="gene">GmBRI1</span> protein is strikingly similar to AtBRI1 in the prediction of its tertiary structure. A phylogenetic analysis suggested that <span class="gene">GmBRI1</span> is closest in homology to BRIs from dicotyledons, especially leguminous plants. Additionally, the transformation of <italic> <span class="gene">GmBRI1</span> </italic> into the <italic> Arabidopsis bri1-5 </italic> mutant, restored the phenotype, especially regarding pod size and plant height. Additionally, this complementation is a consequence of a restored BR signaling pathway. These results support <italic> <span class="gene">GmBRI1</span> </italic> as a novel <italic> BRI1 </italic> gene. Furthermore, the high level of <italic> <span class="gene">GmBRI1</span> </italic> transcript in soybean rapidly growing tissues, such as hypocotyls, and its response to exogenous BR treatment suggest the vital role of <italic> <span class="gene">GmBRI1</span> </italic> in soybean development. Further investigations are needed to confirm the relationship between <italic> <span class="gene">GmBRI1</span> </italic> and plant architecture, which could lead to the potential development of soybean <span class="trait">yield.</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Milk production is unaffected by replacing barley or sodium hydroxide wheat with maize <span class="gene">cob</span> silage in rations for dairy cows.</h2>
			<h3>PMID:24594308  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hymøller L(1), Hellwing AL(1), Lund P(1), Weisbjerg MR(1).
				<br/> 
				年份：2014 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Starch is an important energy-providing nutrient for dairy cows that is most commonly provided from cereal grains. However, ruminal fermentation of large amounts of easily degradable starch leads to excessive production and accumulation of volatile fatty acids (VFA). VFA not only play a vital role in the energy metabolism of dairy cows but are also the main cause of ruminal acidosis and depressed feed intake. The aim of the present study was to compare maize <span class="gene">cob</span> silage (MCS) as an energy supplement in rations for dairy cows with highly rumen-digestible rolled barley and with sodium hydroxide wheat (SHW), which has a higher proportion of by-pass starch than barley. Two studies were carried out: (1) a production study on 45 Danish Holstein cows and (2) an intensive study to determine digestibilities, rumen fermentation patterns and methane emission using three rumen-cannulated Danish Holstein cows. Both studies were organised as a 3×3 Latin square with three experimental periods and three different mixed rations. The rations consisted of grass-clover silage and maize silage (~60% of dry matter (DM)), rapeseed cake, soybean meal, sugar beet pulp and one of three different cereals as a major energy supplement: MCS, SHW or rolled barley (~25% of DM). When MCS replaced barley or SHW as an energy supplement in the mixed rations, it resulted in a lower dry matter intake; however, the apparent total tract digestibilities of DM, organic matter, NDF, starch and protein were not different between treatments. The energy-corrected milk <span class="trait">yield</span> was unaffected by treatment. The fat content of the milk on the MCS ration was not different from the SHW ration, whereas it was higher on the barley ration. The protein content of the milk decreased when MCS was used in the ration compared with barley and SHW. From ruminal VFA patterns and pH measures, it appeared that MCS possessed roughage qualities with respect to rumen environment, while at the same time being sufficiently energy rich to replace barley and SHW as a major energy supplement for milk production. The environmental impact, expressed as methane emissions, was not different when comparing MCS, SHW and barley.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Effects of phosphorus fertilization on leaf area index, biomass accumulation and allocation, and phosphorus use efficiency of intercropped maize].</h2>
			<h3>PMID:24483073  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chen YX(1), Li HH(2), Zhou T(2), Chen XP(3), Huang W(2), Liu J(2), Zhang CC(3), Xu KW(2).
				<br/> 
				年份：2013 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A 2-year field experiment was conducted in 2011 and 2012 to investigate the effects of phosphorus (P) fertilization on the leaf area index (LAI), dry matter accumulation (DMA), and P use efficiency (PUE) of maize in wheat/maize/soybean intercropping system. Five P fertilization rates were installed, i.e., 0, 45, 90, 135, and 180 kg P2O5 x hm(-2) for wheat, marked as WP0, WP1, WP2, WP3, and WP4, respectively, and 0, 37.5, 75, 112.5, and 150 kg P2O5 x hm(-2) for maize, marked as MP0, MP1, MP2, MP3, and MP4, respectively. During the coexisted growth periods of wheat and maize, P fertilization increased the LAI, leaf area duration (LAD), and stem and leaf DMA of maize significantly. After the jointing stage of maize, the maize LAI, LAD, DMA, and crop growth rate (CGR) all decreased after an initial increase with the increasing P rate, with the maximum growth in treatment <span class="gene">MP2</span> or MP3. During the reproductive stage of maize, the maize dry mass translocation from vegetative to reproductive organ increased with increasing P fertilization rate, and the grain <span class="trait">yield</span> of both maize and whole cropping system increased firstly and decreased then, with the maximum grain <span class="trait">yield</span> of maize and whole cropping system being 6588 and 11955 kg x hm(-2) in treatment P3, respectively. The P apparent recovery efficiency of maize was the highest (26.3%) in treatment MP2, being 82.6%, 38.4%, and 152.9% higher than that in MP1 (14.4%), MP3 (19.0%), and MP4 (10.4%), respectively. In sum, for the wheat/maize/soybean intercropping system, applying appropriate amount of P fertilizer could promote maize growth, alleviate the impact of wheat on maize, and consequently, increase the P apparent recovery efficiency of maize. In this study, the appropriate P fertilization rate was 75-112.5 kg P2O5 x hm(-2).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Acid Phosphatase-Encoding Gene <italic> <span class="gene">GmACP1</span> </italic> Contributes to Soybean Tolerance to Low-Phosphorus Stress</h2>
			<h3>PMID:24391523</h3>
				

				
					<h3>PMCID:3879153</h3>
					<h4>期刊:PLoS Genetics</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Dan </given-names>  </name>  <xref ref-type="aff" rid="aff1">
				<br/> 
				年份：<day> 11 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphorus (P) is essential for all living cells and organisms, and low-P stress is a major factor constraining plant growth and <span class="trait">yield</span> worldwide. In plants, P efficiency is a complex quantitative trait involving multiple genes, and the mechanisms underlying P efficiency are largely unknown. Combining linkage analysis, genome-wide and candidate-gene association analyses, and plant transformation, we identified a soybean gene related to P efficiency, determined its favorable haplotypes and developed valuable functional markers. First, six major genomic regions associated with P efficiency were detected by performing genome-wide associations (GWAs) in various environments. A highly significant region located on chromosome 8, qPE8, was identified by both GWAs and linkage mapping and explained 41% of the phenotypic variation. Then, a regional mapping study was performed with 40 surrounding markers in 192 diverse soybean accessions. A strongly associated haplotype (P = 10(-7)) consisting of the markers Sat_233 and BARC-039899-07603 was identified, and qPE8 was located in a region of approximately 250 kb, which contained a candidate gene <span class="gene">GmACP1</span> that encoded an acid phosphatase. <span class="gene">GmACP1</span> overexpression in soybean hairy roots increased P efficiency by 11-20% relative to the control. A candidate-gene association analysis indicated that six natural <span class="gene">GmACP1</span> polymorphisms explained 33% of the phenotypic variation. The favorable alleles and haplotypes of <span class="gene">GmACP1</span> associated with increased transcript expression correlated with higher enzyme activity. The discovery of the optimal haplotype of <span class="gene">GmACP1</span> will now enable the accurate selection of soybeans with higher P efficiencies and improve our understanding of the molecular mechanisms underlying P efficiency in plants.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this study, haplotype analysis of <italic> <span class="gene">GmACP1</span> </italic> gene revealed ten haplotypes (H1 to H10) ( <xref ref-type="fig" rid="pgen-1004061-g006"> Figure 6A </xref> ). The RAPA, RPC and RSW of these ten haplotypes were significantly associated with each other, indicating that the P efficiency gene could contribute to seed <span class="trait">yield</span> and has therefore been retained during soybean domestication and breeding. Notably, the <italic> in vitro </italic> expressed protein encoded by the favorable haplotype H9 had 1.6-fold higher enzymatic activity than the protein encoded by the unfavorable haplotype H4 ( <xref ref-type="fig" rid="pgen-1004061-g006"> Figure 6B </xref> ), indicating that haplotype H9 might be significant for engineering plants with higher P efficiencies. Further studies on the origins of this favorable haplotype would be interesting.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Symbiotic effectiveness of inoculation with <italic> Bradyrhizobium </italic> isolates on soybean [ <italic> Glycine max </italic> (L.) Merrill] genotypes with different maturities</h2>
			<h3>PMID:26034703</h3>
				

				
					<h3>PMCID:4447725</h3>
					<h4>期刊:SpringerPlus</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Argaw </surname>  <given-names> Anteneh </given-names>  </name>  <address>  <email> antarga@yahoo.com </email>  </address>  <xref ref-type="aff" rid="Aff126"/>  </contrib>  <aff id="Aff126"> College of Agriculture and Environmental Sciences, School of Natural Resources Management and Environmental Sciences, Haramaya University, P.O.Box 337, Dire Dawa, Ethiopia  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 12 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The influence of soybean genotypes with different maturity groups on the symbiotic effectiveness of Bradyrhizobium <span class="gene">spp</span> under high native soil N is not well known. Therefore, the objective of this work was to evaluate the influence of maturity time of soybean genotypes on the symbiotic effectiveness of Bradyrhizobium spp. at higher soil N. Three isolates of Bradyrhizobium spp. (UK-isolate, TAL-379 isolate and local-isolate) and six soybean genotypes, three late maturing (Wogayen, TGx-1336424 and Belsa) and three medium maturing (GIZA, Afgat and Gishame) were used for greenhouse experiment. Only GIZA and TGx-1336424 were selected for field experiment. The result of the experiments showed that significantly (P &lt; 0.05) differences in all investigated traits, except total plant tissue N, was observed in TGx-1336424 with UK-Isolate and Local-Isolate.TAL-379 inoculation performed better in all investigated traits of GIZA genotype than other inoculation treatments. N-fertilization in the greenhouse experiment significantly (P &lt; 0.05) improved the shoot biomass of Wogayen and Belsa-95, but did not observe in GIZA and Gishame. The regression analysis obtained between nodule number and nodule dry weight with that of grain <span class="trait">yield</span> indicated generally higher R(2) value for late maturing than that of the medium maturing genotypes. This indicates high importance of nodulation for improving the GY of late maturing genotypes. Hence, this study proves the need for inoculation to improve the production and productivity of soybean sustainably in Ethiopia, with particularly pronounced effect on late maturing genotypes of soybean.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Early Transcriptional Response of Soybean Contrasting Accessions to Root Dehydration</h2>
			<h3>PMID:24349513</h3>
				

				
					<h3>PMCID:3861472</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ferreira Neto </surname>  <given-names> Jos&#x000e9; Ribamar Costa </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pandolfi </surname>  <given-names> Valesca </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guimaraes </surname>  <given-names> Francismar Corr&#x000ea;a Marcelino </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Benko-Iseppon </surname>  <given-names> Ana Maria </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Romero </surname>  <given-names> Cynara </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Silva </surname>  <given-names> Roberta Lane de Oliveira </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Fabiana Aparecida </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdelnoor </surname>  <given-names> Ricardo Vilela </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kido </surname>  <given-names> Ederson Akio </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is a significant constraint to <span class="trait">yield</span> increase in soybean. The early perception of water deprivation is critical for recruitment of genes that promote plant tolerance. DeepSuperSAGE libraries, including one control and a bulk of six stress times imposed (from 25 to 150 min of root dehydration) for drought-tolerant and sensitive soybean accessions, allowed to identify new molecular targets for <span class="trait">drought</span> tolerance. The survey uncovered 120,770 unique transcripts expressed by the contrasting accessions. Of these, 57,610 aligned with known cDNA sequences, allowing the annotation of 32,373 unitags. A total of 1,127 unitags were up-regulated only in the tolerant accession, whereas 1,557 were up-regulated in both as compared to their controls. An expression profile concerning the most representative Gene Ontology (GO) categories for the tolerant accession revealed the expression "protein binding" as the most represented for "Molecular Function", whereas <span class="gene">CDPK</span> and CBL were the most up-regulated protein families in this category. Furthermore, particular genes expressed different isoforms according to the accession, showing the potential to operate in the distinction of physiological behaviors. Besides, heat maps comprising GO categories related to abiotic stress response and the unitags regulation observed in the expression contrasts covering tolerant and sensitive accessions, revealed the unitags potential for plant breeding. Candidate genes related to "hormone response" (LOX, ERF1b, XET), "water response" (PUB, BMY), "salt stress response" (WRKY, MYB) and "oxidative stress response" (PER) figured among the most promising molecular targets. Additionally, nine transcripts (HMGR, XET, WRKY20, RAP2-4, EREBP, NAC3, PER, GPX5 and BMY) validated by RT-qPCR (four different time points) confirmed their differential expression and pointed that already after 25 minutes a transcriptional reorganization started in response to the new condition, with important differences between both accessions.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A recent survey (May, 2013) at PubMed database (NCBI) showed 81 reports related to &#x0201c;soybean and transcriptome&#x0201d;, most of them using microarray approaches, as in the case of Le et al. [ <xref ref-type="bibr" rid="B13"> 13 </xref> ] that used the 66 K Affymetrix Soybean Array GeneChip for genome-wide expression profiling of leaf tissues (soybean cv. Williams 82) subjected to <span class="trait">drought</span> stress (soil moisture content of 5% and leaf relative water content = 32&#x000b1;2%) from two stages (V6 and R2). Concerning the reports using high-throughput sequencing methods, Libaut et al. [ <xref ref-type="bibr" rid="B14"> 14 </xref> ] studied the transcriptome of root hair cells under <italic> Bradyrhizobium japonicum </italic> infection. After that, Libault et al. [ <xref ref-type="bibr" rid="B15"> 15 </xref> ] tried to generate a transcriptome atlas using various soybean tissues; Le et al. [ <xref ref-type="bibr" rid="B16"> 16 </xref> ] focused on the NAC transcription factor family in soybean during development and dehydration stress; Li et al. [ <xref ref-type="bibr" rid="B17"> 17 </xref> ], otherwise, looked for stress associated microRNAs in <italic> G. max </italic> by deep sequencing, while Hao et al. [ <xref ref-type="bibr" rid="B18"> 18 </xref> ] searched for soybean genes associated with nitrogen-use efficiency, and Kido et al. [ <xref ref-type="bibr" rid="B19"> 19 </xref> ] looked for plant antimicrobial peptides in soybean transcriptome after <italic> P. pachirizy </italic> induction. Moreover, <span class="gene">Fan</span> et al. [ <xref ref-type="bibr" rid="B20"> 20 </xref> ] analyzed the late expression (48 h after stress) to different conditions including <span class="trait">drought</span> (2% PEG 8000), in leaves and roots of seedlings (two-leaf stage) of the soybean inbred line HJ-1. Using RNA-Seq method, specifically, Severin et al. [ <xref ref-type="bibr" rid="B21"> 21 </xref> ] searched for a high-resolution gene expression in a collection of fourteen different tissues; Hunt et al. [ <xref ref-type="bibr" rid="B22"> 22 </xref> ] tried to characterize the transcriptional profiles of a wild-type and glabrous soybean lines while Reid et al. [ <xref ref-type="bibr" rid="B23"> 23 </xref> ] looked for transcript abundance changes that occur during AON (autoregulation of nodulation), and Peiffer et al. [ <xref ref-type="bibr" rid="B24"> 24 </xref> ] attempted to identify candidate genes underlying an iron efficiency quantitative trait locus.
					</p>
				
					<p>&nbsp;&nbsp;
						The Cluster 2 ( <xref ref-type="fig" rid="pone-0083466-g007"> Figure 7A </xref> ; <xref ref-type="supplementary-material" rid="pone.0083466.s006"> Table S6 </xref> ) contain unitags induced in approaches I and III (absent or n.s. in the approach II). One of the most expressed modulated unitags in this group was TD22175 (FC <sub> I </sub> = 28.8, FC <sub> III </sub> = 3.4), a possible MYB transcription factor. Members of the MYB family are abundant in all eukaryotes, being the most frequent transcription factor family (TF) in plants [ <xref ref-type="bibr" rid="B73"> 73 </xref> ]. In the present data, 425 unitags annotated as MYB TFs (data not show). MYB TFs are key factors in the regulation pathways that control development, metabolism and response to biotic and abiotic stress [ <xref ref-type="bibr" rid="B74"> 74 </xref> ]. Concerning their role in the <span class="trait">drought</span> tolerance, Seo et al. [ <xref ref-type="bibr" rid="B75"> 75 </xref> ] reported that a R2R3-type MYB TF (MYB96) regulated <span class="trait">drought</span> stress response by integrating ABA and auxin signals. The putative <span class="gene">MYB124</span> observed in the DeepSuperSAGE data (MYB124_TD22175) could be, along with MYB88, generating regular stomatal patterning, as in Arabidopsis [ <xref ref-type="bibr" rid="B76"> 76 </xref> ], optimizing gas exchange and guard cell ion transport.
					</p>
				
					<p>&nbsp;&nbsp;
						The main steps in plant tolerance to adverse environmental conditions process are stress condition perception, signal transduction, activation and regulation of stress responsive genes. The two previous steps requiring greater efforts at the transcriptional level, with a large portion of the plant genomics capacity driven by TFs. Soybean has 5,671 putative TFs, distributed in 63 families, which equates to 12.2% of the 46,430 predicted soybean protein-coding loci [ <xref ref-type="bibr" rid="B25"> 25 </xref> ], while Arabidopsis and rice genomes code more than 2,100 and 2,300 TFs respectively [ <xref ref-type="bibr" rid="B92"> 92 </xref> ]. Among TFs, WRKY is one of the largest families of plant transcriptional regulators modulating plant processes [ <xref ref-type="bibr" rid="B93"> 93 </xref> ], also in plant stress responses. In rice, for instance, OsWRKY11 overexpression (under the control of HSP101 promoter) led to enhance <span class="trait">drought</span> tolerance [ <xref ref-type="bibr" rid="B94"> 94 </xref> ]. Recently, Luo et al. [ <xref ref-type="bibr" rid="B95"> 95 </xref> ] observed that the expression of wild soybean <span class="gene">WRKY20</span> in Arabidopsis enhances <span class="trait">drought</span> tolerance and regulates ABA signalling. The differential behavior of the analyzed accessions for the expression (RT-qPCR) based on the TD57178 unitag was evident. The tolerant accession kept its induction over all the stress times tested (25, 50, 75, 100 min) modulating the expression (MRGEL) 2.3 to 3.6 times in relation to T0 ( <xref ref-type="fig" rid="pone-0083466-g009"> Figure 9 </xref> ); the sensitive accession, in turn, showed overexpression only in the early time points (25 and 50, MRGEL: 3.1 and 3.4, respectively, <xref ref-type="fig" rid="pone-0083466-g009"> Figure 9 </xref> ). According to Chen et al. [ <xref ref-type="bibr" rid="B96"> 96 </xref> ] the strict control and fine-tuning of WRKY proteins during plant stress responses contribute to the installation of complex signaling networks, highlighting the importance of WRKY proteins in plant abiotic stress response.
					</p>
				
					<p>&nbsp;&nbsp;
						Under stress conditions, plants do not induce only gene transcriptions that operate in cellular protection, namely, enzyme coding genes and other functional proteins, but they also produce the regulatory transcripts that act in the transduction of signals from their perception organs. In this context are the transcription factors (TF) coding genes. Among the plant-specific transcription factors, NAC (NAM, ATAF, CUC) proteins constitute one of the largest families, present in a wide range of land plants [ <xref ref-type="bibr" rid="B97"> 97 </xref> ]. The NAC was the most represented in the expression cluster &#x0201c;Response to hormonal stimulus&#x0201d; ( <xref ref-type="fig" rid="pone-0083466-g006"> Figure 6A </xref> ). Specifically, the TD33178 unitag (NAC3 isoform) showed large modulation for both accessions ( <xref ref-type="fig" rid="pone-0083466-g006"> Figure 6A </xref> ; <xref ref-type="supplementary-material" rid="pone.0083466.s004"> Table S4 </xref> ); this being induced at all analyzed time intervals [tolerant accession (MRGEL: 3.0 to 3.9; sensitive accession: MRGEL: 2.0 to 3.1; <xref ref-type="fig" rid="pone-0083466-g009"> Figure 9 </xref> ). This TF participation in the tolerance process to abiotic stresses has been demonstrated. Liu et al. [ <xref ref-type="bibr" rid="B98"> 98 </xref> ] obtained tobacco transgenic lines transformed with <italic> AhNAC3 </italic> (from peanut), and those showed hyper-resistance to dehydration and <span class="trait">drought</span> stresses and accumulated more proline and less superoxide anion (O <sub> 2 </sub> <sup> &#x02212; </sup> ) than wild type under dehydration and <span class="trait">drought</span> conditions. They also observed that four functional genes, superoxide dismutase, pyrroline-5-carboxylate synthetase, late embryogenic abundant proteins, and early response to <span class="trait">drought</span> 10, were induced in the transgenic lines, been suggested that <span class="gene">NAC3</span> improves water stress tolerance by increasing superoxide scavenging and promoting the accumulation of various protective molecules.
					</p>
				
					<p>&nbsp;&nbsp;
						The AP2/ERF (EREBP) is a plant-specific TF large family that shares a well-conserved DNA-binding domain, comprising AP2, RAV, EREBP subfamilies, with the EREBP subfamily <underline> subdivided </underline> into DREB (Dehydration-responsive element-binding) or A subgroup and the ERF (Ethylene response factor) or B subgroup [ <xref ref-type="bibr" rid="B101"> 101 </xref> ]. The up-regulation of those unitags observed in both accessions, in relation to the expression in the appropriate unstressed controls, suggests a conservative action even in contrasting accessions. Concerning the TD1001 unitag, the level of expression was similar for both accessions, considering each time evaluated, showing differential expression since 25 min after stress ( <xref ref-type="fig" rid="pone-0083466-g009"> Figure 9 </xref> ). Additionally, the unitag expression level in both accessions was smaller than those observed in TD200 ( <xref ref-type="fig" rid="pone-0083466-g009"> Figure 9 </xref> ). BLASTn analysis based on the RefSeq_RNA database (NCBI) revealed that <span class="gene">Glyma13g01930.1</span> represents, specifically, an FT-type RAP2-4 (data not shown). RAP2-4 is a TF AP2/DREB-type, which belongs to EREBP subfamily. This TF was down-regulated by light but up-regulated by salt and <span class="trait">drought</span> stresses, in Arabidopsis [ <xref ref-type="bibr" rid="B102"> 102 </xref> ]. Recently, Rae et al. [ <xref ref-type="bibr" rid="B103"> 103 </xref> ] investigated the expression and function of RAP2-4B and RAP2-4 (both DREB TFs) using microarray-based transcriptional profiling of double knockout and overexpression lines. Expression analysis of stressed and control plants revealed both genes highly expressed in stems and roots and differentially induced in response to cold, dehydration and osmotic stress. The same authors also concluded that RAP2-4 is a probable significant aquaporin co-expression network regulator during the early phase of dehydration response. During that study, six aquaporin genes &#x02013; from which three ( <italic> AtPIP2; 1 </italic> , <italic> AtPIP2;2 </italic> and <italic> AtPIP2;3 </italic> ) from the <span class="gene">PIP</span> group and three ( <italic> AtTIP1;1 </italic> , <italic> AtTIP2;2 </italic> and <italic> At </italic> <span class="gene">TIP</span> 2;3) from the <span class="gene">TIP</span> group &#x02013; were down-regulated in the double knockout line and consequently up-regulated in the appropriate overexpression line [103]. In relation to the TD200 unitag <span class="gene">(Glyma16g27950.1;</span> annotated as a TF AP2/ERF, EREBP subfamily), the tolerant accession response revealing a faster response (25 min) than the sensitive accession (50 min; <xref ref-type="fig" rid="pone-0083466-g009"> Figure 9 </xref> ). Also, in general, the average gene expression level presented by the tolerant accession was higher than the observed in the sensitive ( <xref ref-type="fig" rid="pone-0083466-g009"> Figure 9 </xref> ) also considering each analyzed time point. BLAST2Seq analysis performed to gather similarity between <span class="gene">Glyma16g27950.1and</span> <span class="gene">Glyma13g01930.1</span> transcripts, since they belong to the same EREBP subfamily, did not show significance (data not shown). As mentioned before, this TF subfamily comprises <underline> DREBs </underline> and ERFs [ <xref ref-type="bibr" rid="B101"> 101 </xref> ]. In soybean, the overexpression of a DREB homologous gene <span class="gene">(GmDREB2)</span> activated expression of downstream genes in transgenic Arabidopsis, resulting in enhanced tolerance to <span class="trait">drought</span> and high-salt stresses, without plant growth retardation [ <xref ref-type="bibr" rid="B104"> 104 </xref> ]. Besides, its overexpression in tobacco resulted in higher proline content rates compared to wild type plants under <span class="trait">drought</span> condition [ <xref ref-type="bibr" rid="B104"> 104 </xref> ]. ERFs also respond to <span class="trait">drought</span> tolerance. In soybean, <span class="gene">GmERF3,</span> a member of this subfamily, showed its expression induced by biotic stress [soybean mosaic virus, SMV] and abiotic stresses, such as high salinity, <span class="trait">drought</span> and hormones (ABA, SA, JA and ET) [ <xref ref-type="bibr" rid="B105"> 105 </xref> ]. Aditionally, osmoregulation is among the known ERF-associated mechanisms. The overexpression of <span class="gene">GmERF3</span> in transgenic tobacco led to higher levels of free proline and soluble carbohydrates compared to wild-type plants under <span class="trait">drought</span> conditions [ <xref ref-type="bibr" rid="B105"> 105 </xref> ].
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Yield and Economic Performance of Organic and Conventional Cotton-Based Farming Systems &#x02013; Results from a Field Trial in India</h2>
			<h3>PMID:24324659</h3>
				

				
					<h3>PMCID:3852008</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Forster </surname>  <given-names> Dionys </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Andres </surname>  <given-names> Christian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Verma </surname>  <given-names> Rajeev </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zundel </surname>  <given-names> Christine </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Messmer </surname>  <given-names> Monika M. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> M&#x000e4;der </surname>  <given-names> Paul </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 10 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The debate on the relative benefits of conventional and organic farming systems has in recent time gained significant interest. So far, global agricultural development has focused on increased productivity rather than on a holistic natural resource management for food security. Thus, developing more sustainable farming practices on a large scale is of utmost importance. However, information concerning the performance of farming systems under organic and conventional management in tropical and subtropical regions is scarce. This study presents agronomic and economic data from the conversion phase (2007-2010) of a farming systems comparison trial on a Vertisol soil in Madhya Pradesh, central India. A cotton-soybean-wheat crop rotation under biodynamic, organic and conventional (with and without Bt cotton) management was investigated. We observed a significant <span class="trait">yield</span> <span class="gene">gap</span> between organic and conventional farming systems in the 1(st) crop cycle (cycle 1: 2007-2008) for cotton (-29%) and wheat (-27%), whereas in the 2(nd) crop cycle (cycle 2: 2009-2010) cotton and wheat yields were similar in all farming systems due to lower yields in the conventional systems. In contrast, organic soybean (a nitrogen fixing leguminous plant) yields were marginally lower than conventional yields (-1% in cycle 1, -11% in cycle 2). Averaged across all crops, conventional farming systems achieved significantly higher gross margins in cycle 1 (+29%), whereas in cycle 2 gross margins in organic farming systems were significantly higher (+25%) due to lower variable production costs but similar yields. Soybean gross margin was significantly higher in the organic system (+11%) across the four harvest years compared to the conventional systems. Our results suggest that organic soybean production is a viable option for smallholder farmers under the prevailing semi-arid conditions in India. Future research needs to elucidate the long-term productivity and profitability, particularly of cotton and wheat, and the ecological impact of the different farming systems.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Results from various farming systems comparison trials between organic and conventional management have shown, that even though yields may be slightly lower, organic farming systems exhibit several ecological and economic advantages, particularly long-term improvement of soil <span class="trait">fertility</span> <xref ref-type="bibr" rid="pone.0081039-Reganold2"> [20] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0081039-Birkhofer1"> [25] </xref> . However, most of the data has been obtained from trials in the temperate zones <xref ref-type="bibr" rid="pone.0081039-Reganold2"> [20] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0081039-Gattinger1"> [26] </xref> . The little data available under tropical and subtropical conditions <xref ref-type="bibr" rid="pone.0081039-Eyhorn1"> [9] </xref> , <xref ref-type="bibr" rid="pone.0081039-Bachmann1"> [27] </xref> &#x02013; <xref ref-type="bibr" rid="pone.0081039-Rasul1"> [29] </xref> calls for more long-term farming systems comparison trials to provide a better basis for decision making in these regions <xref ref-type="bibr" rid="pone.0081039-Seufert1"> [17] </xref> . To address this issue, the Research Institute of Organic Agriculture (FiBL) has set up three farming systems comparison trials in Kenya, India and Bolivia, thereby encompassing different cropping systems and ethnologies. The main objective of these trials is to collect solid agronomic and socio-economic data on major organic and conventional agricultural production systems in the selected project regions. These trials will contribute to close the existing knowledge <span class="gene">gap</span> regarding the estimation of profitability of organic agriculture in developing countries ( <ext-link ext-link-type="uri" xlink:href="http://www.systems-comparison.fibl.org/"> http://www.systems-comparison.fibl.org/ </ext-link> ). This paper presents results from cotton-based farming systems in India.
					</p>
				
					<p>&nbsp;&nbsp;
						Non-GM soybean and wheat varieties were cultivated in both CON and CONBtC systems. In 2007, average soybean yields across all farming systems were 45% lower compared to the other three years ( <xref ref-type="fig" rid="pone-0081039-g003"> Figure 3 </xref> ), as the whole trial had to be re-sown due to severe water logging. Soybean yields were, averaged across the four years, 7% lower in organic compared to conventional farming systems. The <italic> System </italic> &#x000d7; <italic> Cycle </italic> interaction had a significant effect ( <italic> P </italic> &#x0003c;0.05) on soybean yields ( <xref ref-type="table" rid="pone-0081039-t003"> Table 3 </xref> ). No significant difference in <span class="trait">yield</span> could be identified between farming systems in cycle 1. However, in cycle 2 CON and CONBtC showed significantly higher yields than BIOORG ( <xref ref-type="table" rid="pone-0081039-t003"> Table 3 </xref> ). This is likely due to higher pest incidences and thus lower yields in organic systems in 2009. BIODYN produced similar soybean yields as both conventional systems throughout the experimental period ( <italic> P </italic> &#x0003e;0.05). The 1% and 11% lower yields in organic farming systems in cycle 1 and 2, respectively, are considerably lower than the 18% lower yields reported for organic soybean in the Karnataka region <xref ref-type="bibr" rid="pone.0081039-Ramesh1"> [76] </xref> . These results indicate similar productivity of conventional and organic soybean production systems under subtropical (semi-arid) conditions and suggest that in similar settings no further <span class="trait">yield</span> gains can be achieved through the provision of synthetic inputs compared to organic management practices. The smaller difference in <span class="trait">yield</span> between conventional and organic soybean - when compared to cotton and wheat (see below) - could be explained by considering the plant type. Soybean is the only legume crop in the crop rotation, possessing the ability to fix atmospheric N, thereby avoiding potential nitrogen shortage for optimal plant growth. These results confirm the findings of Seufert <italic> et al </italic> . <xref ref-type="bibr" rid="pone.0081039-Seufert1"> [17] </xref> whose meta-analysis showed a lower <span class="trait">yield</span> <span class="gene">gap</span> between conventional and organic legume crops when compared to non-legume crops, and indicate that cotton and wheat yields in organic farming systems in our trial may be restricted by nitrogen limitation in the soil.
					</p>
				
					<p>&nbsp;&nbsp;
						Wheat yields were, averaged across the four years, 15% lower in organic compared to conventional farming systems, which is similar to the 20% <span class="trait">yield</span> <span class="gene">gap</span> reported for Uttarakhand <xref ref-type="bibr" rid="pone.0081039-Ramesh1"> [76] </xref> . The <italic> System </italic> &#x000d7; <italic> Cycle </italic> interaction had a significant effect ( <italic> P </italic> &#x0003c;0.001) on wheat grain <span class="trait">yield.</span> Similar to cotton, there was a significant <span class="trait">yield</span> <span class="gene">gap</span> between conventional and organic farming systems in cycle 1 (+37% <span class="trait">yield</span> increase in conventional farming systems), but not in cycle 2 due to both slightly lower yields in the conventional systems and slightly higher yields in the organic systems compared to cycle 1 ( <xref ref-type="table" rid="pone-0081039-t003"> Table 3 </xref> , <xref ref-type="fig" rid="pone-0081039-g003"> Figure 3 </xref> ). For both soybean and wheat no <span class="trait">yield</span> differences were detected between CON and CONBtC farming systems, except for significantly higher wheat yields in CON in cycle 1 ( <xref ref-type="table" rid="pone-0081039-t003"> Table 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In general, the first four years of the farming systems comparison trial in India revealed that there was a significant <span class="trait">yield</span> <span class="gene">gap</span> in cycle 1 (2007&#x02013;2008) for cotton (&#x02212;29%) and wheat (&#x02212;27%), in organic compared to conventional farming systems, whereas in cycle 2 yields of the three crops were similar in all farming systems due to low yields in the conventional systems ( <xref ref-type="table" rid="pone-0081039-t003"> Table 3 </xref> ). Because there was no clear trend of <span class="trait">yield</span> development for cotton and wheat during the four year period in any of the systems, observed results rather reflect growth conditions in respective years than long-term <span class="trait">yield</span> trends of cotton and wheat. However, the marginal <span class="trait">yield</span> <span class="gene">gap</span> between the BIOORG system and the conventional systems, and the par soybean yields of BIODYN and the conventional systems show that leguminous crops are a promising option for conversion to organic systems under the given conditions. The <span class="trait">yield</span> development across the whole crop rotation needs to be verified during future crop cycles.
					</p>
				
					<p>&nbsp;&nbsp;
						The following examples show that comparative findings on <span class="trait">yield</span> and economics between organic and conventional cotton are highly contextual. Eyhorn <italic> et al </italic> . <xref ref-type="bibr" rid="pone.0081039-Eyhorn1"> [9] </xref> surveyed more than 50 conventional (without Bt cotton) and 30 organic cotton farmers in the Nimar Valley, Madhya Pradesh, India, during the period of 2003&#x02013;2004. Their findings support our results of cycle 2 (years three and four, 2009&#x02013;2010): yields of cotton and other cash crops were on par with conventional farmers, but with economic benefits for the organic farmers due to lower production costs. These findings also underline the practical relevance of our results for cotton production in the in the smallholder context in Madhya Pradesh. Likewise, Venugopalan <italic> et al </italic> . <xref ref-type="bibr" rid="pone.0081039-Venugopalan1"> [81] </xref> reported similar or slightly higher cotton yields in an organic compared to a non-organic system under low input and semi-arid conditions in the Yavatmal district, Maharashtra, India (observation phase 2001&#x02013;2005). In a long-term trial under rainfed conditions in Nagpur (Maharashtra, India), Menon <xref ref-type="bibr" rid="pone.0081039-Menon1"> [82] </xref> reported a <span class="trait">yield</span> <span class="gene">gap</span> of 25% of organic cotton compared to the modern method of cultivation (&#x0200a;=&#x0200a; conventional without Bt cotton) within the first six years after conversion (1994&#x02013;2000). Thereafter (2002&#x02013;2004), the organic farming systems outyielded the conventional systems by up to 227 kg seed cotton ha <sup> &#x02212;1 </sup> <xref ref-type="bibr" rid="pone.0081039-Blaise1"> [28] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In contrast to Kathage and Qaim <xref ref-type="bibr" rid="pone.0081039-Kathage1"> [36] </xref> , our study also includes other cash crops such as soybean and wheat as part of cotton-based farming systems. These are essential components for enabling long-term cotton cropping and for securing livelihoods of smallholders, as they enable the distribution of risks. According to our findings, the investigated organic farming systems also showed a significant <span class="trait">yield</span> <span class="gene">gap</span> compared to the conventional farming systems in wheat in cycle 1, as well as for total productivity per crop rotation (including additional wheat crop after Bt cotton (wheat 2)) in cycle 2. Furthermore, a smaller yet significant <span class="trait">yield</span> <span class="gene">gap</span> was observed for soybean in cycle 2 for the BIOORG system (but not for the BIODYN system) ( <xref ref-type="table" rid="pone-0081039-t003"> Table 3 </xref> ). Nevertheless, as in our study organic farming systems were less capital-intensive than conventional ones for all crops, they may be of particular interest to smallholder farmers who often do not have the financial means to purchase inputs and would thus need to seek loans. If this can be verified in on-farm trials, organic farmers might be less exposed to financial risks associated with fluctuating market prices of synthetic fertilizers and crop protection products <xref ref-type="bibr" rid="pone.0081039-Bachmann1"> [27] </xref> , <xref ref-type="bibr" rid="pone.0081039-Panneerselvam2"> [79] </xref> . Additional on-farm investigations have been started in order to classify regional farms into several farm typologies with corresponding levels of available production factors. This should allow for the assessment of the perspectives of each farm type regarding conversion to organic farming systems. If organic farming is to be adopted more widely, more inter- and transdisciplinary research giving focus to the problems and benefits of organic management practices needs to be undertaken <xref ref-type="bibr" rid="pone.0081039-Forster1"> [84] </xref> . Furthermore, large efforts have to be made to gather and disseminate knowledge on production techniques. Intensifying research on organic farming systems to a similar extent as was the case for research on GM crops <xref ref-type="bibr" rid="pone.0081039-Vanloqueren1"> [85] </xref> may help to provide additional relevant information to policy makers, advisors and farmers about comparative advantages and limitations of different cotton production systems.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Variation in Rubisco activase ( <italic> RCA&#x003b2; </italic> ) gene promoters and expression in soybean [ <italic> Glycine max </italic> (L.) Merr.]</h2>
			<h3>PMID:24170743</h3>
				

				
					<h3>PMCID:3883283</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				
			<p>
				作者：<contrib contrib-type="author" corresp="no">  <name>  <surname> Chao </surname>  <given-names> Maoni </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Understanding the genetic basis of Rubisco activase (RCA) gene regulation and altering its expression levels to optimize Rubisco activation may provide an approach to enhance plant productivity. However, the genetic mechanisms and the effect of RCA expression on phenotype are still unknown in soybean. This work analysed the expression of RCA genes and demonstrated that two RCA isoforms presented different expression patterns. Compared with <span class="gene">GmRCAα,</span> <span class="gene">GmRCAβ</span> was expressed at higher mRNA and protein levels. In addition, <span class="gene">GmRCAα</span> and <span class="gene">GmRCAβ</span> were positively correlated with chlorophyll fluorescence parameters and seed <span class="trait">yield,</span> suggesting that changes in expression of RCA has a potential applicability in breeding for enhanced soybean productivity. To identify the genetic factors that cause expression level variation of <span class="gene">GmRCAβ,</span> expression quantitative trait loci (eQTL) mapping was combined with allele mining in a natural population including 219 landraces. The eQTL mapping showed that a combination of both cis- and trans-acting eQTLs might control <span class="gene">GmRCAβ</span> expression. As promoters can affect both cis- and trans-acting eQTLs by altering cis-acting regulatory elements or transcription factor binding sites, this work subsequently focused on the promoter region of <span class="gene">GmRCAβ.</span> Single-nucleotide polymorphisms in the <span class="gene">GmRCAβ</span> promoter were identified and shown to correlate with expression level diversity. These SNPs were classified into two groups, A and B. Further transient expression showed that GUS expression driven by the group A promoter was stronger than that by the group B promoter, suggesting that promoter sequence types could influence gene expression levels. These results would improve understanding how variation within promoters affects gene expression and, ultimately, phenotypic diversity in natural populations.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Correlation coefficients among <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> and <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression and <span class="trait">yield</span> components in a natural soybean population
					</p>
				
					<p>&nbsp;&nbsp;
						Empirical chlorophyll transient parameters, such as F <sub> v </sub> /F <sub> m </sub> , &#x003a6; <sub> PSII </sub> , qP, and NPQ, have been widely accepted as reflecting the structure and function of the photosynthetic apparatus ( <xref ref-type="bibr" rid="CIT0036"> Papageorgiou, 2004 </xref> ). To further investigate the relationship between the effect of RCA gene expression on soybean photosynthesis and productivity, gene expression, chlorophyll &#x0fb02;uorescence parameters, and <span class="trait">yield</span> components was measured in a natural population. As shown in <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert346/-/DC1"> Supplementary Table S1 </ext-link> , the expression levels of the two <italic> RCA </italic> genes displayed high diversity in soybean germplasm, ranging from 0.47 to 66.41 for <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> expression and from 2.38 to 196.49 for <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression. Variation analysis indicated that these wide ranges might mainly be caused by genotypic differences. Further analysis of gene expression in the natural population indicated that <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> accumulated more mRNA in all landraces than <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert346/-/DC1"> Supplementary Fig. S3A </ext-link> ), and the levels of <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> and <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression were significantly different ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert346/-/DC1"> Supplementary Fig. S3B </ext-link> ). Correlation analysis suggested that the expression levels of both genes were positively correlated with &#x003a6; <sub> PSII </sub> , qP, and seed <span class="trait">yield,</span> but the correlations between <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression level and &#x003a6; <sub> PSII </sub> , qP, and seed <span class="trait">yield</span> were slightly higher than for <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> . In addition, <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression level displayed a positive correlation with seed weight ( <xref ref-type="table" rid="T2"> Table 2 </xref> ). Overall, these results suggested that soybean landraces exhibited considerable natural variation in RCA gene expression level and that both isoforms could play an important role in modulating soybean photosynthetic capacity and seed <span class="trait">yield.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Since RCA genes were positively correlated with chlorophyll fluorescence parameters and seed <span class="trait">yield,</span> changes in <italic> RCA </italic> expression at the transcript level may have a potential applicability in breeding for enhanced soybean productivity. The present study analysed the genetic regulation of <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression and evaluated the associations with 1142 SNPs by genome-wide association analysis. A total of 13 SNPs were identified as having significant marker-trait associations at <italic> P &#x02264; </italic> 0.01 (&#x02013;log <italic> P </italic> &#x02265; 2.00) for the expression of the <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> ( <xref ref-type="table" rid="T3"> Table 3 </xref> ). Here, a <italic> cis </italic> -eQTL was defined as being located within 5Mb of the target gene; otherwise, it was termed as a <italic> trans </italic> -eQTL ( <xref ref-type="bibr" rid="CIT0034"> Morley <italic> et al. </italic> , 2004 </xref> ; <xref ref-type="bibr" rid="CIT0015"> Ghazalpour <italic> et al. </italic> , 2006 </xref> ; <xref ref-type="bibr" rid="CIT0013"> Gatti <italic> et al. </italic> , 2009 </xref> , <xref ref-type="bibr" rid="CIT0014"> 2011 </xref> ; <xref ref-type="bibr" rid="CIT0051"> Swanson-Wagner <italic> et al. </italic> , 2009 </xref> ; <xref ref-type="bibr" rid="CIT0016"> Grundberg <italic> et al. </italic> , 2010 </xref> ). Among the 13 SNPs, BARC-040479-07752 was located in the same chromosome as <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> (chromosome 18) and was 1.62Mb away from <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> , suggesting that this SNP might be a <italic> cis </italic> -eQTL, while BARC-021337-04040 was also located in chromosome 18 and was 7.87Mb away from <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> , suggesting that this SNP might act in <italic> trans </italic> , but <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> was located between the two SNPs. The other 11 SNPs were located in other chromosomes, indicating that they may influence the expression levels of <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> via <italic> trans </italic> -regulation. These results indicated that <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression might be controlled by a combination of both <italic> cis </italic> -acting and <italic> trans </italic> -acting eQTL. In addition, the <italic> cis </italic> -acting eQTL that had the stronger association (&#x02013;log <italic> P </italic> = 5.64) with expression may mainly control the expression of <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Although different patterns of RCA gene expression were observed in soybean, the effect of this on <span class="trait">yield</span> components remains unknown. A direct link between growth and gene expression has been confirmed in rice, in which different levels of expression that are positively correlated with these traits of this agronomically important grain are dependent on sequence variation in the promoter of the GS5 gene ( <xref ref-type="bibr" rid="CIT0029"> Li <italic> et al. </italic> , 2011 </xref> ). Studies have shown that endogenous levels of <italic> RCA </italic> expression are of importance to plant photosynthesis under both optimal ( <xref ref-type="bibr" rid="CIT0032"> Mart&#x000ed;nez-Barajas <italic> et al. </italic> , 1997 </xref> ) and supraoptimal temperatures ( <xref ref-type="bibr" rid="CIT0043"> Ristic <italic> et al. </italic> , 2009 </xref> ). Thus, the present study investigated the effect of expression on soybean <span class="trait">yield</span> by evaluating the expression levels of <italic> RCA </italic> in a natural population. The expression levels of both genes were positively correlated with chlorophyll fluorescence parameters and seed <span class="trait">yield,</span> suggesting that changes in <italic> RCA </italic> expression has a potential applicability in breeding for enhanced soybean productivity. Moreover, the expression levels displayed high diversity in the germplasm, and the phenotypic values in natural population ranged from 0.47 to 66.41 for <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> expression and from 2.38 to 196.49 for <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression. However, the regression analysis indicated that a given change in <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> was associated with a greater change in photosynthesis or <span class="trait">yield</span> ( <ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert346/-/DC1"> Supplementary Table S2 </ext-link> ). Therefore, further studies need to compare <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> and <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> in transgenic soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						<ext-link ext-link-type="uri" xlink:href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/ert346/-/DC1"> Supplementary Table S2 </ext-link> . Regression coefficients among <italic> <span class="gene">GmRCA&#x003b1;</span> </italic> and <italic> <span class="gene">GmRCA&#x003b2;</span> </italic> expression and <span class="trait">yield</span> components in a natural soybean population.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Mapping the low palmitate fap1 mutation and validation of its effects in soybean oil and agronomic traits in three soybean populations.</h2>
			<h3>PMID:24132738  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cardinal AJ(1), Whetten R, Wang S, Auclair J, Hyten D, Cregan P, Bachlava E, Gillman J, Ramirez M, Dewey R, Upchurch G, Miranda L, Burton JW.
				<br/> 
				年份：2014 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						KEY MESSAGE: fap 1 mutation is caused by a G174A change in <span class="gene">GmKASIIIA</span> that disrupts a donor splice site recognition and creates a GATCTG motif that enhanced its expression. Soybean oil with reduced palmitic acid content is desirable to reduce the health risks associated with consumption of this fatty acid. The objectives of this study were: to identify the genomic location of the reduced palmitate fap1 mutation, determine its molecular basis, estimate the amount of phenotypic variation in fatty acid composition explained by this locus, determine if there are epistatic interactions between the fap1 and fap nc loci and, determine if the fap1 mutation has pleiotropic effects on seed <span class="trait">yield,</span> oil and protein content in three soybean populations. This study detected two major QTL for 16:0 content located in chromosome 5 <span class="gene">(GmFATB1a,</span> fap nc) and chromosome 9 near BARCSOYSSR_09_1707 that explained, with their interaction, 66-94 % of the variation in 16:0 content in the three populations. Sequencing results of a putative candidate gene, <span class="gene">GmKASIIIA,</span> revealed a single unique polymorphism in the germplasm line C1726, which was predicted to disrupt the donor splice site recognition between exon one and intron one and produce a truncated KASIIIA protein. This G to A change also created the GATCTG motif that enhanced gene expression of the mutated <span class="gene">GmKASIIIA</span> gene. Lines homozygous for the <span class="gene">GmKASIIIA</span> mutation (fap1) had a significant reduction in 16:0, 18:0, and oil content; and an increase in unsaturated fatty acids content. There were significant epistatic interactions between <span class="gene">GmKASIIIA</span> (fap1) and fap nc for 16:0 and oil contents, and seed <span class="trait">yield</span> in two populations. In conclusion, the fap1 phenotype is caused by a single unique SNP in the <span class="gene">GmKASIIIA</span> gene.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of <span class="gene">GmCaM4</span> in soybean enhances <span class="trait">resistance</span> to pathogens and tolerance to salt stress.</h2>
			<h3>PMID:24118726  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rao SS(1), El-Habbak MH, Havens WM, Singh A, Zheng D, Vaughn L, Haudenshield JS, Hartman GL, Korban SS, Ghabrial SA.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Plant diseases inflict heavy losses on soybean <span class="trait">yield,</span> necessitating an understanding of the molecular mechanisms underlying biotic/abiotic stress responses. Ca(2) (+) is an important universal messenger, and protein sensors, prominently calmodulins (CaMs), recognize cellular changes in Ca(2) (+) in response to diverse signals. Because the development of stable transgenic soybeans is laborious and time consuming, we used the Bean pod mottle virus (BPMV)-based vector for rapid and efficient protein expression and gene silencing. The present study focuses on the functional roles of the gene encoding the soybean <span class="gene">CaM</span> isoform <span class="gene">GmCaM4.</span> Overexpression of <span class="gene">GmCaM4</span> in soybean resulted in enhanced <span class="trait">resistance</span> to three plant pathogens and increased tolerance to high salt conditions. To gain an understanding of the underlying mechanisms, we examined the potential defence pathways involved. Our studies revealed activation/increased expression levels of pathogenesis-related (PR) genes in <span class="gene">GmCaM4-overexpressing</span> plants and the accumulation of jasmonic acid (JA). Silencing of <span class="gene">GmCaM4,</span> however, markedly repressed the expression of PR genes. We confirmed the in vivo interaction between <span class="gene">GmCaM4</span> and the <span class="gene">CaM</span> binding transcription factor Myb2, which regulates the expression of salt-responsive genes, using the yeast two-hybrid (Y2H) system and bimolecular fluorescence complementation assays. <span class="gene">GmCaM4</span> and Glycine max <span class="gene">CaM</span> binding receptor-like kinase <span class="gene">(GmCBRLK)</span> did not interact in the Y2H assays, but the interaction between <span class="gene">GmCaM2</span> and <span class="gene">GmCBRLK</span> was confirmed. Thus, a <span class="gene">GmCaM2-GmCBRLK-mediated</span> salt tolerance mechanism, similar to that reported in Glycine soja, may also be functional in soybean. Confocal microscopy showed subcellular localization of the green fluorescent protein (GFP)-GmCaM4 fusion protein in the nucleus and cytoplasm.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Role of soybean mosaic virus-encoded proteins in seed and aphid transmission in soybean.</h2>
			<h3>PMID:23927427  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jossey S(1), Hobbs HA, Domier LL.
				<br/> 
				年份：2013 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean mosaic virus (SMV) is seed and aphid transmitted and can cause significant reductions in <span class="trait">yield</span> and seed quality in soybean (Glycine max). The roles in seed and aphid transmission of selected SMV-encoded proteins were investigated by constructing mutants in and chimeric recombinants between SMV 413 (efficiently aphid and seed transmitted) and an isolate of SMV G2 (not aphid or seed transmitted). As previously reported, the DAG amino acid sequence motif near the amino terminus of the coat protein (CP) was the major determinant in differences in aphid transmissibility of the two SMV isolates, and helper component proteinase (HC-Pro) played a secondary role. Seed transmission of SMV was influenced by P1, HC-Pro, and CP. Replacement of the <span class="gene">P1</span> coding region of SMV 413 with that of SMV G2 significantly enhanced seed transmissibility of SMV 413. Substitution in SMV 413 of the two amino acids that varied in the CPs of the two isolates with those from SMV G2, G to D in the DAG motif and Q to P near the carboxyl terminus, significantly reduced seed transmission. The Q-to-P substitution in SMV 413 also abolished virus-induced seed-coat mottling in plant introduction 68671. This is the first report associating P1, CP, and the DAG motif with seed transmission of a potyvirus and suggests that HC-Pro interactions with CP are important for multiple functions in the virus infection cycle.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Algae biodiesel life cycle assessment using current commercial data.</h2>
			<h3>PMID:23900083  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Passell H(1), Dhaliwal H, Reno M, Wu B, Ben Amotz A, Ivry E, Gay M, Czartoski T, Laurin L, Ayer N.
				<br/> 
				年份：2013 Nov 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Autotrophic microalgae represent a potential feedstock for transportation fuels, but life cycle assessment (LCA) studies based on laboratory-scale or theoretical data have shown mixed results. We attempt to bridge the <span class="gene">gap</span> between laboratory-scale and larger scale biodiesel production by using cultivation and harvesting data from a commercial algae producer with ∼1000 m(2) production area (the base case), and compare that with a hypothetical scaled up facility of 101,000 m(2) (the future case). Extraction and separation data are from Solution Recovery Services, Inc. Conversion and combustion data are from the Greenhouse Gases, Regulated Emissions, and Energy Use in Transportation Model (GREET). The LCA boundaries are defined as "pond-to-wheels". Environmental impacts are quantified as NER (energy in/energy out), global warming potential, photochemical oxidation potential, water depletion, particulate matter, and total NOx and SOx. The functional unit is 1 MJ of energy produced in a passenger car. Results for the base case and the future case show an NER of 33.4 and 1.37, respectively and GWP of 2.9 and 0.18 kg CO2-equivalent, respectively. In comparison, petroleum diesel and soy diesel show an NER of 0.18 and 0.80, respectively and GWP of 0.12 and 0.025, respectively. A critical feature in this work is the low algal productivity (3 g/m(2)/day) reported by the commercial producer, relative to the much higher productivities (20-30 g/m(2)/day) reported by other sources. Notable results include a sensitivity analysis showing that algae with an oil <span class="trait">yield</span> of 0.75 kg oil/kg dry biomass in the future case can bring the NER down to 0.64, more comparable with petroleum diesel and soy biodiesel. An important assumption in this work is that all processes are fully co-located and that no transport of intermediate or final products from processing stage to stage is required.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional properties and expression quantitative trait loci for phosphate transporter <span class="gene">GmPT1</span> in soybean.</h2>
			<h3>PMID:23889314  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Song H(1), Yin Z, Chao M, Ning L, Zhang D, Yu D.
				<br/> 
				年份：2014 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phosphate (Pi) remobilization within a plant is critical for plant survival under Pi-limiting conditions. In this paper, a soybean Pi transporter gene, <span class="gene">GmPT1,</span> was characterized. A marked induction of <span class="gene">GmPT1</span> transcript was observed in young leaves, mature leaves and lateral roots during long-term Pi starvation. Transgenic tobacco plants containing the <span class="gene">GmPT1</span> gene were obtained using an Agrobacterium-mediated transformation system. Compared with wild-type plants, transgenic plants showed significant increases in phosphorus-use efficiency (PUE), photosystem II (PSII) function, total dry weight and seed weight under Pi-deficient conditions. <span class="gene">GmPT1</span> expression levels and PUE were determined in a soybean recombinant inbred line population during a pot experiment that was conducted to measure chlorophyll fluorescence parameters, photosynthetic rate (PN ) and seed <span class="trait">yield.</span> Correlation analysis revealed that <span class="gene">GmPT1</span> expression levels had significantly positive correlations with seed <span class="trait">yield,</span> PUE, PN and the quantum <span class="trait">yield</span> of PSII primary photochemistry (ΦPSII ). Expression quantitative trait loci (eQTL) mapping for <span class="gene">GmPT1</span> revealed two eQTLs, one of which coincided with both the physical location of <span class="gene">GmPT1</span> and a QTL associated with seed <span class="trait">yield.</span> These results suggest that <span class="gene">GmPT1</span> plays a role in Pi remobilization, and it may be possible to improve soybean seed yields under Pi-limiting conditions by modulating <span class="gene">GmPT1</span> expression levels.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic and biochemical analyses of the cotyledon and root of flooding-stressed soybean plants.</h2>
			<h3>PMID:23799004  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Komatsu S(1), Makino T, Yasue H.
				<br/> 
				年份：2013 Jun 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Flooding significantly reduces the growth and grain <span class="trait">yield</span> of soybean plants. Proteomic and biochemical techniques were used to determine whether the function of cotyledon and root is altered in soybean under flooding stress. RESULTS: Two-day-old soybean plants were flooded for 2 days, after which the proteins from root and cotyledon were extracted for proteomic analysis. In response to flooding stress, the abundance of 73 and 28 proteins was significantly altered in the root and cotyledon, respectively. The accumulation of only one protein, 70 kDa heat shock protein (HSP70) <span class="gene">(Glyma17g08020.1),</span> increased in both organs following flooding. The ratio of protein abundance of <span class="gene">HSP70</span> and biophoton emission in the cotyledon was higher than those detected in the root under flooding stress. Computed tomography and elemental analyses revealed that flooding stress decreases the number of calcium oxalate crystal the cotyledon, indicating calcium ion was elevated in the cotyledon under flooding stress. CONCLUSION: These results suggest that calcium might play one role through <span class="gene">HSP70</span> in the cotyledon under flooding stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Replication protein A subunit 3 and the iron efficiency response in soybean.</h2>
			<h3>PMID:23742135  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Atwood SE(1), O'Rourke JA, Peiffer GA, Yin T, Majumder M, Zhang C, Cianzio SR, Hill JH, Cook D, Whitham SA, Shoemaker RC, Graham MA.
				<br/> 
				年份：2014 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In soybean [Glycine max (L.) Merr.], iron deficiency results in interveinal <span class="trait">chlorosis</span> and decreased photosynthetic capacity, leading to stunting and <span class="trait">yield</span> loss. In this study, gene expression analyses investigated the role of soybean replication protein A (RPA) subunits during iron stress. Nine RPA homologs were significantly differentially expressed in response to iron stress in the near isogenic lines (NILs) Clark (iron efficient) and Isoclark (iron inefficient). RPA homologs exhibited opposing expression patterns in the two NILs, with RPA expression significantly repressed during iron deficiency in Clark but induced in Isoclark. We used virus induced gene silencing (VIGS) to repress <span class="gene">GmRPA3</span> expression in the iron inefficient line Isoclark and mirror expression in Clark. <span class="gene">GmRPA3-silenced</span> plants had improved IDC symptoms and chlorophyll content under iron deficient conditions and also displayed stunted growth regardless of iron availability. RNA-Seq comparing gene expression between <span class="gene">GmRPA3-silenced</span> and empty vector plants revealed massive transcriptional reprogramming with differential expression of genes associated with defense, immunity, aging, death, protein modification, protein synthesis, photosynthesis and iron uptake and transport genes. Our findings suggest the iron efficient genotype Clark is able to induce energy controlling pathways, possibly regulated by SnRK1/TOR, to promote nutrient recycling and stress responses in iron deficient conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						Published 2013. This article is a U.S. Government work and is in the public domain in the USA.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Overexpression of a <span class="gene">GmGBP1</span> ortholog of soybean enhances the responses to flowering, stem elongation and heat tolerance in transgenic tobaccos.</h2>
			<h3>PMID:23636865  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhao L(1), Wang Z, Lu Q, Wang P, Li Y, Lv Q, Song X, Li D, Gu Y, Liu L, Li W.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a typical short-day crop, and its photoperiodic and gibberellin (GA) responses for the control of flowering are critical to seed <span class="trait">yield.</span> The <span class="gene">GmGBP1</span> mRNA abundance in leaves was dramatically increased in short-days (SDs) compared to that in long-days in which it was consistently low at all time points from 0 to 6 days (days after transfer to SDs). <span class="gene">GmGBP1</span> was highly expressed in leaves and exhibited a circadian rhythm in SDs. Ectopic overexpression of <span class="gene">GmGBP1</span> in tobaccos caused photoperiod-insensitive early flowering by increasing NtCO mRNA levels. <span class="gene">GmGBP1</span> mRNA abundance was also increased by GAs. Transgenic <span class="gene">GmGBP1</span> overexpressing (-ox) tobacco plants exhibited increased GA signaling-related phenotypes including flowering and plant height promotion. Furthermore, the hypocotyl elongation, early-flowering and longer internode phenotypes were largely accelerated by GA3 application in the <span class="gene">GmGBP1-ox</span> tobacco seedlings. Being consistent, overexpression of <span class="gene">GmGBP1</span> resulted in significantly enhanced GA signaling (evidenced suppressed expression of NtGA20ox) both with and without GA treatments. <span class="gene">GmGBP1</span> was a positive regulator of both photoperiod and GA-mediated flowering responses. In addition, <span class="gene">GmGBP1-ox</span> tobaccos were hypersensitive to ABA, salt and osmotic stresses during seed germination. Heat-inducible <span class="gene">GmGBP1</span> also enhanced thermotolerance in transgenic <span class="gene">GmGBP1-ox</span> tobaccos during seed germination and growth. <span class="gene">GmGBP1</span> protein was localized in the nucleus. Analyses of a series of 5'-deletions of the <span class="gene">GmGBP1</span> promoter suggested that several cis-acting elements, including P-BOX, TCA-motif and three HSE elements necessary to induce gene expression by GA, salicic acid and heat stress, were specifically localized in the <span class="gene">GmGBP1</span> promoter region.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heterologous production of pentane in the oleaginous yeast Yarrowia lipolytica.</h2>
			<h3>PMID:23602802  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Blazeck J(1), Liu L, Knight R, Alper HS.
				<br/> 
				年份：2013 Jun 10
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The complete biosynthetic replacement of petroleum transportation fuels requires a metabolic pathway capable of producing short chain n-alkanes. Here, we report and characterize a proof-of-concept pathway that enables microbial production of the <span class="gene">C5</span> n-alkane, pentane. This pathway utilizes a soybean lipoxygenase enzyme to cleave linoleic acid to pentane and a tridecadienoic acid byproduct. Initial expression of the soybean lipoxygenase enzyme within a Yarrowia lipolytica host yielded 1.56 mg/L pentane. Efforts to improve pentane <span class="trait">yield</span> by increasing substrate availability and strongly overexpressing the lipoxygenase enzyme successfully increased pentane production three-fold to 4.98 mg/L. This work represents the first-ever microbial production of pentane and demonstrates that short chain n-alkane synthesis is conceivable in model cellular hosts. In this regard, we demonstrate the potential pliability of Y. lipolytica toward the biosynthetic production of value-added molecules from its generous fatty acid reserves.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Using the candidate gene approach for detecting genes underlying seed oil concentration and <span class="trait">yield</span> in soybean.</h2>
			<h3>PMID:23568222  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Increasing the oil concentration in soybean seeds has been given more attention in recent years because of demand for both edible oil and biodiesel production. Oil concentration in soybean is a complex quantitative trait regulated by many genes as well as environmental conditions. To identify genes governing seed oil concentration in soybean, 16 putative candidate genes of three important gene families (GPAT: acyl-CoA:sn-glycerol-3-phosphate acyltransferase, DGAT: acyl-CoA:diacylglycerol acyltransferase, and PDAT: phospholipid:diacylglycerol acyltransferase) involved in triacylglycerol (TAG) biosynthesis pathways were selected and their sequences retrieved from the soybean database ( http://www.phytozome.net/soybean ). Three sequence mutations were discovered in either coding or noncoding regions of three <span class="gene">DGAT</span> soybean isoforms when comparing the parents of a 203 recombinant inbreed line (RIL) population; OAC Wallace and OAC Glencoe. The RIL population was used to study the effects of these mutations on seed oil concentration and other important agronomic and seed composition traits, including seed <span class="trait">yield</span> and protein concentration across three field locations in Ontario, Canada, in 2009 and 2010. An insertion/deletion (indel) mutation in the <span class="gene">GmDGAT2B</span> gene in OAC Wallace was significantly associated with reduced seed oil concentration across three environments and reduced seed <span class="trait">yield</span> at Woodstock in 2010. A mutation in the 3' untranslated (3'UTR) region of <span class="gene">GmDGAT2C</span> was associated with seed <span class="trait">yield</span> at Woodstock in 2009. A mutation in the intronic region of <span class="gene">GmDGAR1B</span> was associated with seed <span class="trait">yield</span> and protein concentration at Ottawa in 2010. The genes identified in this study had minor effects on either seed <span class="trait">yield</span> or oil concentration, which was in agreement with the quantitative nature of the traits. However, the novel gene-specific markers designed in the present study can be used in soybean breeding for marker-assisted selection aimed at increasing seed <span class="trait">yield</span> and oil concentration with no significant impact on seed protein concentration.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic control of soybean seed oil: II. QTL and genes that increase oil concentration without decreasing protein or with increased seed <span class="trait">yield.</span></h2>
			<h3>PMID:23536049  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eskandari M(1), Cober ER, Rajcan I.
				<br/> 
				年份：2013 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [Glycine max (L.) Merrill] seed oil is the primary global source of edible oil and a major renewable and sustainable feedstock for biodiesel production. Therefore, increasing the relative oil concentration in soybean is desirable; however, that goal is complex due to the quantitative nature of the oil concentration trait and possible effects on major agronomic traits such as seed <span class="trait">yield</span> or protein concentration. The objectives of the present study were to study the relationship between seed oil concentration and important agronomic and seed quality traits, including seed <span class="trait">yield,</span> 100-seed weight, protein concentration, plant height, and days to maturity, and to identify oil quantitative trait loci (QTL) that are co-localized with the traits evaluated. A population of 203 F4:6 recombinant inbred lines, derived from a cross between moderately high oil soybean genotypes OAC Wallace and OAC Glencoe, was developed and grown across multiple environments in Ontario, Canada, in 2009 and 2010. Among the 11 QTL associated with seed oil concentration in the population, which were detected using either single-factor ANOVA or multiple QTL mapping methods, the number of QTL that were co-localized with other important traits QTL were six for protein concentration, four for seed <span class="trait">yield,</span> two for 100-seed weight, one for days to maturity, and one for plant height. The oil-beneficial allele of the QTL tagged by marker Sat_020 was positively associated with seed protein concentration. The oil favorable alleles of markers Satt001 and <span class="gene">GmDGAT2B</span> were positively correlated with seed <span class="trait">yield.</span> In addition, significant two-way epistatic interactions, where one of the interacting markers was solely associated with seed oil concentration, were identified for the selected traits in this study. The number of significant epistatic interactions was seven for <span class="trait">yield,</span> four for days to maturity, two for 100-seed weight, one for protein concentration, and one for plant height. The identified molecular markers associated with oil-related QTL in this study, which also have positive effects on other important traits such as seed <span class="trait">yield</span> and protein concentration, could be used in the soybean marker breeding programs aimed at developing either higher seed <span class="trait">yield</span> and oil concentration or higher seed protein and oil concentration per hectare. Alternatively, selecting complementary parents with greater breeding values due to positive epistatic interactions could lead to the development of higher oil soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Aggressiveness of Fusarium species and impact of root infection on growth and <span class="trait">yield</span> of soybeans.</h2>
			<h3>PMID:23514263  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Arias MM(1), Leandro LF, Munkvold GP.
				<br/> 
				年份：2013 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fusarium spp. are commonly isolated from soybean roots but the pathogenic activity of most species is poorly documented. Aggressiveness and <span class="trait">yield</span> impact of nine species of Fusarium were determined on soybean in greenhouse (50 isolates) and field microplot (19 isolates) experiments. Root rot severity and shoot and root dry weights were compared at growth stages <span class="gene">V3</span> or R1. Root systems were scanned and digital image analysis was conducted; <span class="trait">yield</span> was measured in microplots. Disease severity and root morphology impacts varied among and within species. Fusarium graminearum was highly aggressive (root rot severity &gt;90%), followed by F. proliferatum and F. virguliforme. Significant variation in damping-off (20 to 75%) and root rot severity (&lt;20 to &gt;60%) was observed among F. oxysporum isolates. In artificially-infested microplots, root rot severity was low (&lt;25%) and mean <span class="trait">yield</span> was not significantly reduced. However, there were significant linear relationships between <span class="trait">yield</span> and root symptoms for some isolates. Root morphological characteristics were more consistent indicators of <span class="trait">yield</span> loss than root rot severity. This study provides the first characterization of aggressiveness and <span class="trait">yield</span> impact of Fusarium root rot species on soybean at different plant stages and introduces root image analysis to assess the impact of root pathogens on soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Global warming can negate the expected CO2 stimulation in photosynthesis and productivity for soybean grown in the Midwestern United States.</h2>
			<h3>PMID:23512883  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ruiz-Vera UM(1), Siebers M, Gray SB, Drag DW, Rosenthal DM, Kimball BA, Ort DR, Bernacchi CJ.
				<br/> 
				年份：2013 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Extensive evidence shows that increasing carbon dioxide concentration ([CO2]) stimulates, and increasing temperature decreases, both net photosynthetic carbon assimilation (A) and biomass production for <span class="gene">C3</span> plants. However the [CO2]-induced stimulation in A is projected to increase further with warmer temperature. While the influence of increasing temperature and [CO2], independent of each other, on A and biomass production have been widely investigated, the interaction between these two major global changes has not been tested on field-grown crops. Here, the interactive effect of both elevated [CO2] (approximately 585 μmol mol(-1)) and temperature (+3.5°C) on soybean (Glycine max) A, biomass, and <span class="trait">yield</span> were tested over two growing seasons in the Temperature by Free-Air CO2 Enrichment experiment at the Soybean Free Air CO2 Enrichment facility. Measurements of A, stomatal conductance, and intercellular [CO2] were collected along with meteorological, water potential, and growth data. Elevated temperatures caused lower A, which was largely attributed to declines in stomatal conductance and intercellular [CO2] and led in turn to lower yields. Increasing both [CO2] and temperature stimulated A relative to elevated [CO2] alone on only two sampling days during 2009 and on no days in 2011. In 2011, the warmer of the two years, there were no observed increases in <span class="trait">yield</span> in the elevated temperature plots regardless of whether [CO2] was elevated. All treatments lowered the harvest index for soybean, although the effect of elevated [CO2] in 2011 was not statistically significant. These results provide a better understanding of the physiological responses of soybean to future climate change conditions and suggest that the potential is limited for elevated [CO2] to mitigate the influence of rising temperatures on photosynthesis, growth, and yields of <span class="gene">C3</span> crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The evolution and pathogenic mechanisms of the rice sheath blight pathogen.</h2>
			<h3>PMID:23361014  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zheng A(1), Lin R, Zhang D, Qin P, Xu L, Ai P, Ding L, Wang Y, Chen Y, Liu Y, Sun Z, Feng H, Liang X, Fu R, Tang C, Li Q, Zhang J, Xie Z, Deng Q, Li S, Wang S, Zhu J, Wang L, Liu H, Li P.
				<br/> 
				年份：2013
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rhizoctonia solani is a major fungal pathogen of rice (Oryza sativa L.) that causes great <span class="trait">yield</span> losses in all rice-growing regions of the world. Here we report the draft genome sequence of the rice sheath blight disease pathogen, R. solani <span class="gene">AG1</span> IA, assembled using next-generation Illumina Genome Analyser sequencing technologies. The genome encodes a large and diverse set of secreted proteins, enzymes of primary and secondary metabolism, carbohydrate-active enzymes, and transporters, which probably reflect an exclusive necrotrophic lifestyle. We find few repetitive elements, a closer relationship to Agaricomycotina among Basidiomycetes, and expand protein domains and families. Among the 25 candidate pathogen effectors identified according to their functionality and evolution, we validate 3 that trigger crop defence responses; hence we reveal the exclusive expression patterns of the pathogenic determinants during host infection.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A soybean cyst nematode <span class="trait">resistance</span> gene points to a new mechanism of plant <span class="trait">resistance</span> to pathogens.</h2>
			<h3>PMID:23235880  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu S(1), Kandoth PK, Warren SD, Yeckel G, Heinz R, Alden J, Yang C, Jamai A, El-Mellouki T, Juvale PS, Hill J, Baum TJ, Cianzio S, Whitham SA, Korkin D, Mitchum MG, Meksem K.
				<br/> 
				年份：2012 Dec 13
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean (Glycine max (L.) Merr.) is an important crop that provides a sustainable source of protein and oil worldwide. Soybean cyst nematode (Heterodera glycines Ichinohe) is a microscopic roundworm that feeds on the roots of soybean and is a major constraint to soybean production. This nematode causes more than US$1 billion in <span class="trait">yield</span> losses annually in the United States alone, making it the most economically important pathogen on soybean. Although planting of resistant cultivars forms the core management strategy for this pathogen, nothing is known about the nature of <span class="trait">resistance.</span> Moreover, the increase in virulent populations of this parasite on most known <span class="trait">resistance</span> sources necessitates the development of novel approaches for control. Here we report the map-based cloning of a gene at the <span class="gene">Rhg4</span> (for <span class="trait">resistance</span> to Heterodera glycines 4) locus, a major quantitative trait locus contributing to <span class="trait">resistance</span> to this pathogen. Mutation analysis, gene silencing and transgenic complementation confirm that the gene confers <span class="trait">resistance.</span> The gene encodes a serine hydroxymethyltransferase, an enzyme that is ubiquitous in nature and structurally conserved across kingdoms. The enzyme is responsible for interconversion of serine and glycine and is essential for cellular one-carbon metabolism. Alleles of <span class="gene">Rhg4</span> conferring <span class="trait">resistance</span> or susceptibility differ by two genetic polymorphisms that alter a key regulatory property of the enzyme. Our discovery reveals an unprecedented plant <span class="trait">resistance</span> mechanism against a pathogen. The mechanistic knowledge of the <span class="trait">resistance</span> gene can be readily exploited to improve nematode <span class="trait">resistance</span> of soybean, an increasingly important global crop.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Protocol for augmented shoot organogenesis in selected variety of soybean [Glycine max L. (Merr.)].</h2>
			<h3>PMID:23214267  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Devi MK(1), Sakthivelu G, Giridhar P, Ravishankar GA.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Development of a reproducible, versatile and efficient in vitro plant regeneration system is highly warranted for Indian soybean varieties for their mass multiplication in view of their commercial significance. Accordingly a protocol for direct shoot organogenesis in soybean variety JS 335 has been developed. Using cotyledonary node explants significant organogenic responses, mean shoot number and shoot length were observed when these were incubated on <span class="gene">MS</span> medium supplemented with 0.89 microM Benzyladenine (BA) and 5 microg/L triacontanol (TRIA) where in 9.3 +/- 0.5 shoots were obtained. TRIA at 5 microg/L able to produce 6.8 +/- 0.5 shoot buds in presence of 0.98 microM IBA and 0.89 microM BA. Highest mean shoot buds (14.0 +/- 0.5 and 9.0 +/- 0.5) and mean shoot length (4.6 +/- 0.3 and 10.0 +/- 0.7) were obtained when cotyledonary node and shoot tip explants were cultured on <span class="gene">MS</span> medium containing 0.14 microM gibberellic acid (GA3), 0.89 microM BA and 5 microg/L TRIA. Moreover, TRIA supported highest mean root number (6.3 +/- 0.5) and root length (21.5 +/- 0.57 cm). Field survival of in vitro derived plants of TRIA treatment was 70% and the overall growth and seed <span class="trait">yield</span> was also significantly better than control plants. This protocol may be used for improving the in vitro regeneration of soybean variety JS 335 for transformation studies.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Effects of plant polysaccharide compound agents on the photosynthetic characteristics and dry matter of soybean].</h2>
			<h3>PMID:23173460  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bai WB(1), Song JQ, Guo JY, Liu XH, Li JH.
				<br/> 
				年份：2012 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A field experiment was conducted to study the effects of foliar spraying three compound agents [plant polysaccharides (P1), plant polysaccharides and 5-aminolevulinic acid (P2), and plant polysaccharides and 5-aminolevulinic acid and dimethylpiperidinium chloride (P3)] at the initial flowering stage of soybean on its leaf chlorophyll content, photosynthesis and transpiration, dry matter accumulation and allocation, and grain <span class="trait">yield.</span> Within 35 days after spraying the three compound agents, the leaf chlorophyll content had obvious increase, and its decreasing trend with plant growth had somewhat delay. Compared with the control, spraying <span class="gene">P1</span> and P3 increased the leaf photosynthetic rate and water use efficiency by more than 13.2% and 10.3%, respectively. With the spraying of the three compound agents, the dry matter accumulation in aerial part increased, and the allocation of dry matter from leaf to pod was also enhanced, with the contribution of post-anthesis assimilates to grain <span class="trait">yield</span> increased by more than 17.1%. The 100-grain mass and the pods and seeds per plant increased significantly after spraying <span class="gene">P1</span> and P3, but had no significant increase after spraying P2. The grain <span class="trait">yield</span> of soybean treated with the three compound agents increased by more than 5.9%, compared with the control. This study showed that the three plant polysaccharide compound agents could increase the leaf chlorophyll content, delay the leaf-senescence, improve the leaf photosynthetic capacity and water status, effectively control the dry matter accumulation and post-anthesis assimilates allocation, and increase the grain <span class="trait">yield</span> of soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:In vitro analysis of the monolignol coupling mechanism using dehydrogenative polymerization in the presence of peroxidases and controlled feeding ratios of coniferyl and sinapyl alcohol.</h2>
			<h3>PMID:22884779  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Moon SJ(1), Kwon M, Choi D, Won K, Kim YH, Choi IG, Choi JW.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In this study, dehydrogenative polymers (DHP) were synthesized in vitro through dehydrogenative polymerization using different ratios of coniferyl alcohol (CA) and sinapyl alcohol (SA) (10:0, 8:2, 6:4, 2:8, 0:10), in order to investigate the monolignol coupling mechanism in the presence of horseradish peroxidase (HRP), Coprinus cinereus peroxidase (CiP) or soybean peroxidase (SBP) with H(2)O(2), respectively. The turnover capacities of HRP, CiP and <span class="gene">SBP</span> were also measured for coniferyl alcohol (CA) and sinapyl alcohol (SA), and CiP and <span class="gene">SBP</span> were found to have the highest turnover capacity for CA and SA, respectively. The yields of HRP-catalyzed DHP (DHP-H) and CiP-catalyzed DHP (DHP-C) were estimated between ca. 7% and 72% based on the original weights of CA/SA in these synthetic conditions. However, a much lower <span class="trait">yield</span> of SBP-catalyzed DHP (DHP-S) was produced compared to that of DHP-H and DHP-C. In general, the DHP yields gradually increased as the ratio of CA/SA increased. The average molecular weight of DHP-H also increased with increasing CA/SA ratios, while those of DHP-C and DHP-S were not influenced by the ratios of monolignols. The frequency of β-O-4 linkages in the DHPs decreased with increasing CA/SA ratios, indicating that the formation of β-O-4 linkages during DHP synthesis was influenced by peroxidase type.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A simple, economical and reproducible protein extraction protocol for proteomics studies of soybean roots</h2>
			<h3>PMID:22802720</h3>
				

				
					<h3>PMCID:3392887</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Elisete Pains </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-348">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="af2-gmb-35-1-sup-348">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Torres </surname>  <given-names> Adalgisa Ribeiro </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-348">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> da Silva Batista </surname>  <given-names> Jesiane Stef&#x000e2;nia </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-348">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huergo </surname>  <given-names> Luciano </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-35-1-sup-348">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hungria </surname>  <given-names> Mariangela </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-35-1-sup-348">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="c1-gmb-35-1-sup-348"/>  </contrib>
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sample preparation is a critical step in two-dimensional gel electrophoresis (2-DE) of plant tissues. Here we describe a phenol/SDS procedure that, although greatly simplified, produced well-resolved and reproducible 2-DE profiles of protein extracts from soybean [Glycine max (L.) Merril] roots. Extractions were made in three replicates using both the original and simplified procedure. To evaluate the quality of the extracted proteins, ten spots were randomly selected and identified by mass spectrometry (MS). The 2-DE gels were equally well resolved, with no streaks or smears, and no significant differences were observed in protein <span class="trait">yield,</span> reproducibility, resolution or number of spots. Mass spectra of the ten selected spots were compared with database entries and allowed high-quality identification of proteins. The simplified protocol described here presents considerable savings of time and reagents without compromising the quality of 2-DE protein profiles and compatibility with <span class="gene">MS</span> analysis, and may facilitate the progress of proteomics studies of legume-rhizobia interactions.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Involvement of the N-terminal B-box domain of Arabidopsis BBX32 protein in interaction with soybean BBX62 protein.</h2>
			<h3>PMID:22801428  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Qi Q(1), Gibson A, Fu X, Zheng M, Kuehn R, Wang Y, Wang Y, Navarro S, Morrell JA, Jiang D, Simmons G, Bell E, Ivleva NB, McClerren AL, Loida P, Ruff TG, Petracek ME, Preuss SB.
				<br/> 
				年份：2012 Sep 7
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Previous studies have demonstrated that Arabidopsis thaliana BBX32 (AtBBX32) represses light signaling in A. thaliana and that expression of AtBBX32 in soybean increases grain <span class="trait">yield</span> in multiple locations and multiyear field trials. The BBX32 protein is a member of the B-box zinc finger family from A. thaliana and contains a single conserved Zn(2+)-binding B-box domain at the N terminus. Although the B-box domain is predicted to be involved in protein-protein interactions, the mechanism of interaction is poorly understood. Here, we provide in vitro and in vivo evidence demonstrating the physical and functional interactions of AtBBX32 with another B-box protein, soybean BBX62 <span class="gene">(GmBBX62).</span> Deletion analysis and characterization of the purified B-box domain indicate that the N-terminal B-box region of AtBBX32 interacts with <span class="gene">GmBBX62.</span> Computational modeling and site-directed mutagenesis of the AtBBX32 B-box region identified specific residues as critical for mediating the interaction between AtBBX32 and <span class="gene">GmBBX62.</span> This study defines the plant B-box as a protein interaction domain and offers novel insight into its role in mediating specific protein-protein interactions between different plant B-box proteins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Multi-environment mapping and meta-analysis of 100-seed weight in soybean.</h2>
			<h3>PMID:22740134  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sun YN(1), Pan JB, Shi XL, Du XY, Wu Q, Qi ZM, Jiang HW, Xin DW, Liu CY, Hu GH, Chen QS.
				<br/> 
				年份：2012 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						100-Seed weight (100-SW) of soybean is an important but complicated quantitative trait to <span class="trait">yield.</span> This study was focus on the quantitative trait loci (QTLs) of soybean 100-SW from 2006 to 2010, using recombination inbred lines population that was derived from a cross between Charleston and Dongnong 594. A total of 23 QTLs for 100-SW were detected in the linkage group C2, D1a, F, G and O. Nine QTLs were identified by composite interval mapping including one QTL with the minimum confidence interval (CI) of 1.3 cM, while 14 QTLs by multiple interval mapping. Furthermore, 94 reported QTLs of 100-SW were integrated with our QTL mapping results using BioMercator. As a result, 15 consensus QTLs and their corresponding markers were identified. The minimum <span class="gene">CI</span> was reduced to 1.52 cM by the combination of meta-analysis. These findings may merit fine-mapping of these QTL in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative estimation of sampling uncertainties for mycotoxins in cereal shipments.</h2>
			<h3>PMID:22651820  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bourgeois FS(1), Lyman GJ.
				<br/> 
				年份：2012
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Many countries receive shipments of bulk cereals from primary producers. There is a volume of work that is on-going that seeks to arrive at appropriate standards for the quality of the shipments and the means to assess the shipments as they are out-loaded. Of concern are mycotoxin and heavy metal levels, pesticide and herbicide residue levels, and contamination by genetically modified organisms <span class="gene">(GMOs).</span> As the ability to quantify these contaminants improves through improved analytical techniques, the sampling methodologies applied to the shipments must also keep pace to ensure that the uncertainties attached to the sampling procedures do not overwhelm the analytical uncertainties. There is a need to understand and quantify sampling uncertainties under varying conditions of contamination. The analysis required is statistical and is challenging as the nature of the distribution of contaminants within a shipment is not well understood; very limited data exist. Limited work has been undertaken to quantify the variability of the contaminant concentrations in the flow of grain coming from a ship and the impact that this has on the variance of sampling. Relatively recent work by Paoletti et al. in 2006 [Paoletti C, Heissenberger A, Mazzara M, Larcher S, Grazioli E, Corbisier P, Hess N, Berben G, Lübeck PS, De Loose M, et al. 2006. Kernel lot distribution assessment (KeLDA): a study on the distribution of <span class="gene">GMO</span> in large soybean shipments. Eur Food Res Tech. 224:129-139] provides some insight into the variation in <span class="gene">GMO</span> concentrations in soybeans on cargo out-turn. Paoletti et al. analysed the data using correlogram analysis with the objective of quantifying the sampling uncertainty (variance) that attaches to the final cargo analysis, but this is only one possible means of quantifying sampling uncertainty. It is possible that in many cases the levels of contamination passing the sampler on out-loading are essentially random, negating the value of variographic quantitation of the sampling variance. <span class="gene">GMOs</span> and mycotoxins appear to have a highly heterogeneous distribution in a cargo depending on how the ship was loaded (the grain may have come from more than one terminal and set of storage silos) and mycotoxin growth may have occurred in transit. This paper examines a statistical model based on random contamination that can be used to calculate the sampling uncertainty arising from primary sampling of a cargo; it deals with what is thought to be a worst-case scenario. The determination of the sampling variance is treated both analytically and by Monte Carlo simulation. The latter approach provides the entire sampling distribution and not just the sampling variance. The sampling procedure is based on rules provided by the Canadian Grain Commission (CGC) and the levels of contamination considered are those relating to allowable levels of ochratoxin A (OTA) in wheat. The results of the calculations indicate that at a loading rate of 1000 tonnes h(-1), primary sample increment masses of 10.6 kg, a 2000-tonne lot and a primary composite sample mass of 1900 kg, the relative standard deviation (RSD) is about 1.05 (105%) and the distribution of the mycotoxin (MT) level in the primary composite samples is highly skewed. This result applies to a mean MT level of 2 ng g(-1). The rate of false-negative results under these conditions is estimated to be 16.2%. The corresponding contamination is based on initial average concentrations of MT of 4000 ng g(-1) within average spherical volumes of 0.3 m diameter, which are then diluted by a factor of 2 each time they pass through a handling stage; four stages of handling are assumed. The Monte Carlo calculations allow for variation in the initial volume of the MT-bearing grain, the average concentration and the dilution factor. The Monte Carlo studies seek to show the effect of variation in the sampling frequency while maintaining a primary composite sample mass of 1900 kg. The overall results are presented in terms of operational characteristic curves that relate only to the sampling uncertainties in the primary sampling of the grain. It is concluded that cross-stream sampling is intrinsically unsuited to sampling for mycotoxins and that better sampling methods and equipment are needed to control sampling uncertainties. At the same time, it is shown that some combination of cross-cutting sampling conditions may, for a given shipment mass and MT content, <span class="trait">yield</span> acceptable sampling performance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heterologous Expression of <italic> ATG8c </italic> from Soybean Confers Tolerance to Nitrogen Deficiency and Increases <span class="trait">Yield</span> in Arabidopsis</h2>
			<h3>PMID:22629371</h3>
				

				
					<h3>PMCID:3358335</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Xia </surname>  <given-names> Tongmei </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Xiao </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Wenting </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gong </surname>  <given-names> Qingqiu </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Ning Ning </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 4 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Nitrogen is an essential element for plant growth and <span class="trait">yield.</span> Improving Nitrogen Use Efficiency (NUE) of crops could potentially reduce the application of chemical fertilizer and alleviate environmental damage. To identify new NUE genes is therefore an important task in molecular breeding. Macroautophagy (autophagy) is an intracellular process in which damaged or obsolete cytoplasmic components are encapsulated in double membraned vesicles termed autophagosomes, then delivered to the vacuole for degradation and nutrient recycling. One of the core components of autophagosome formation, ATG8, has been shown to directly mediate autophagosome expansion, and the transcript of which is highly inducible upon starvation. Therefore, we postulated that certain homologs of Saccharomyces cerevisiae ATG8 (ScATG8) from crop species could have potential for NUE crop breeding. A soybean (Glycine max, cv. Zhonghuang-13) ATG8, <span class="gene">GmATG8c,</span> was selected from the 11 family members based on transcript analysis upon nitrogen deprivation. <span class="gene">GmATG8c</span> could partially complement the yeast atg8 mutant. Constitutive expression of <span class="gene">GmATG8c</span> in soybean callus cells not only enhanced nitrogen starvation tolerance of the cells but accelerated the growth of the calli. Transgenic Arabidopsis over-expressing <span class="gene">GmATG8c</span> performed better under extended nitrogen and carbon starvation conditions. Meanwhile, under optimum growth conditions, the transgenic plants grew faster, bolted earlier, produced larger primary and axillary inflorescences, eventually produced more seeds than the wild-type. In average, the <span class="trait">yield</span> was improved by 12.9%. We conclude that <span class="gene">GmATG8c</span> may serve as an excellent candidate for breeding crops with enhanced NUE and better <span class="trait">yield.</span>
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In higher plants, homologous autophagic systems have been identified. <italic> In vitro </italic> reconstitution of the two conjugation systems (ATG5-ATG12 and ATG8-PE) has been successfully done, indicating that the mechanism of autophagy is conserved from yeasts to plants <xref ref-type="bibr" rid="pone.0037217-Fujioka1"> [28] </xref> . Analyses on the loss-of-function <italic> atg </italic> mutants revealed the involvement of autophagy in many physiological processes, including nutrient deficiency adaptation, disease <span class="trait">resistance,</span> innate immune response, and stress <span class="trait">resistance</span> <xref ref-type="bibr" rid="pone.0037217-Liu1"> [15] </xref> , <xref ref-type="bibr" rid="pone.0037217-Doelling1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0037217-Yoshimoto2"> [30] </xref> , <xref ref-type="bibr" rid="pone.0037217-Yoshimoto3"> [31] </xref> , <xref ref-type="bibr" rid="pone.0037217-Thompson1"> [32] </xref> , <xref ref-type="bibr" rid="pone.0037217-Phillips1"> [33] </xref> , <xref ref-type="bibr" rid="pone.0037217-Lenz1"> [34] </xref> , <xref ref-type="bibr" rid="pone.0037217-Lai1"> [35] </xref> , <xref ref-type="bibr" rid="pone.0037217-Liu2"> [36] </xref> . An intriguing fact is that plants generally have many <italic> ATG8 </italic> s; there are nine in Arabidopsis <xref ref-type="bibr" rid="pone.0037217-Doelling1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0037217-Yoshimoto2"> [30] </xref> , presumably five in maize <xref ref-type="bibr" rid="pone.0037217-Chung1"> [37] </xref> , and five in rice <xref ref-type="bibr" rid="pone.0037217-Chung1"> [37] </xref> . The distinct expression patterns of individual <italic> ATG8 </italic> s in Arabidopsis suggest that, apart from possible redundancy, at least some <italic> ATG8 </italic> s may possess unique functions <xref ref-type="bibr" rid="pone.0037217-Phillips1"> [33] </xref> , <xref ref-type="bibr" rid="pone.0037217-Hanaoka1"> [38] </xref> . Accordantly, Honig <italic> et al </italic> had found two plant specific AtATG8-interacting proteins (ATI1 and ATI2) that participate in the formation of a new type of compartment which was induced while exposure to carbon starvation <xref ref-type="bibr" rid="pone.0037217-Honig1"> [39] </xref> . They also showed that the constitutive expression of <italic> AtATG8f </italic> led to an increment of plant size, and that the transgenic plants became more tolerant to nitrogen and carbon starvation <xref ref-type="bibr" rid="pone.0037217-Slavikova1"> [40] </xref> . Given the specific function of <italic> ATG8 </italic> and the expression characteristics of the plant <italic> ATG8 </italic> gene families, we postulated that it is possible for us to identify at least one specific <italic> ATG8 </italic> for the development of NUE crops. Furthermore, it has been shown that, under normal growth conditions, loss-of-function autophagy mutants such as <italic> atg7 </italic> , <italic> <span class="gene">atg9</span> </italic> , and <italic> atg10 </italic> produced less flowers, siliques, and seeds <xref ref-type="bibr" rid="pone.0037217-Doelling1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0037217-Phillips1"> [33] </xref> , <xref ref-type="bibr" rid="pone.0037217-Hanaoka1"> [38] </xref> . Whether constitutive expression of an <italic> ATG </italic> could improve seed <span class="trait">yield</span> is therefore worthy of exploring.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to screen for potential soybean NUE genes, our lab has previously set up a fast and effective soybean calli transformation system (patent pending), through which large amounts of uniformly transformed soybean cells could be obtained and analyzed on plates. Here, we firstly identified a soybean <italic> ATG8 </italic> gene, namely <italic> <span class="gene">GmATG8c</span> </italic> , which is strongly induced by nitrogen starvation. We then confirmed its function as an ATG8 by yeast complementation. Constitutive expression of <italic> <span class="gene">GmATG8c</span> </italic> in soybean calli not only led to better tolerance towards nitrogen starvation, but accelerated calli growth under optimum growth conditions. Heterologous expression of <italic> <span class="gene">GmATG8c</span> </italic> in <italic> Arabidopsis </italic> led to better performance of the transgenic lines under both starvation conditions and normal growth condition. The transgenic lines had a better vegetative growth, entered the reproductive stage slightly earlier, and produced more flowers, siliques, and seeds. Our results implied that <italic> <span class="gene">GmATG8c</span> </italic> functions in soybean autophagy process, and that it may be an important candidate for breeding crops with enhanced NUE and better <span class="trait">yield.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						A, Schematic diagrams of <span class="trait">branching</span> patterns of wild-type plants (WT) and <italic> <span class="gene">GmATG8c</span> </italic> over-expressing lines (L-2 to L-4). Bar &#x0200a;=&#x0200a;5 cm. B, The number of siliques on the primary inflorescence of the wild-type and <italic> 35S:GmATG8c </italic> transgenic lines. C, The number of siliques on lateral branches generated from the primary inflorescence. D, The number of siliques on axillary inflorescence of the wild-type and <italic> 35S:GmATG8c </italic> transgenic lines. E, Total number of siliques of the wild-type and <italic> 35S:GmATG8c </italic> transgenic lines. Data shown are the means &#x000b1;SE of one representative biological replicate (n&#x0200a;=&#x0200a;10) out of two. <sup> &#x0002a; </sup> , p&#x0003c;0.05 (t-test); significant difference from the wild-type (WT).
					</p>
				
					<p>&nbsp;&nbsp;
						To further evaluate the potential of <italic> <span class="gene">GmATG8c</span> </italic> in <span class="trait">yield</span> increment, four transgenic lines (L1-4) were grown to stage 9.70 <xref ref-type="bibr" rid="pone.0037217-Boyes1"> [49] </xref> for seed harvest. All seeds from individual plants were harvested and weighed. All four transgenic lines evaluated had higher yields than the wild-type ( <xref ref-type="table" rid="pone-0037217-t002"> Table 2 </xref> ). One of the lines was subjected to further analysis on the reason for the increment of <span class="trait">yield.</span> The thousand grain weight was not different from that of the wild-type ( <xref ref-type="table" rid="pone-0037217-t003"> Table 3 </xref> ). The total number of seeds per silique, however, differed slightly (by two seeds) but significantly from that of the wild-type ( <xref ref-type="table" rid="pone-0037217-t003"> Table 3 </xref> ). The total number of siliques was 12.7&#x00025; more than the wild-type, which may have a major contribution to the increment of <span class="trait">yield</span> in <italic> <span class="gene">GmATG8c</span> </italic> over-expressing plants ( <xref ref-type="table" rid="pone-0037217-t003"> Table 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To further explore the <span class="trait">yield</span> increment potential of <italic> <span class="gene">GmATG8c</span> </italic> , we also transformed tomato (cv. Micro-Tom), a model plant for fruit bearing, with <italic> 35S:GmATG8c </italic> . Similar to our observations in transgenic Arabidopsis, <italic> 35S:GmATG8c </italic> transgenic tomato also grew faster and entered reproductive phase earlier than the wild-type, and produced more fruits ( <xref ref-type="supplementary-material" rid="pone.0037217.s005"> Fig. S5 </xref> , <xref ref-type="supplementary-material" rid="pone.0037217.s009"> Table S4 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Based on our observations, we concluded that <italic> <span class="gene">GmATG8c</span> </italic> not only has NUE potential but also has potential for <span class="trait">yield</span> increment, and may be an excellent target gene for breeding better crops.
					</p>
				
					<p>&nbsp;&nbsp;
						Though autophagy has been established as a ubiquitous intracellular degradation pathway of eukaryotic cells, much of the detailed process, and even components of autophagy, remains elusive in plants, especially in crop species. We identified eleven soybean <italic> ATG8 </italic> genes through sequence analysis. It is well known that a number of ATG8 isoforms exist in plants and mammals, and the existence of an <italic> ATG8 </italic> gene family may reflect the existence of specific functions of individual <italic> ATG8 </italic> s, as have been suggested by other studies <xref ref-type="bibr" rid="pone.0037217-Doelling1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0037217-Yoshimoto2"> [30] </xref> , <xref ref-type="bibr" rid="pone.0037217-Hanaoka1"> [38] </xref> , <xref ref-type="bibr" rid="pone.0037217-Behrends1"> [52] </xref> . When making a phylogenetic tree for ATG8 homologs in crops and lower plants, we noticed two interesting facts. Firstly, all six ATG8s of the moss <italic> Physcomitrella patens </italic> were clustered together to form a branch relatively close to the AtATG8a and <span class="gene">GmATG8c</span> ( <xref ref-type="fig" rid="pone-0037217-g001"> Fig. 1A </xref> ). Secondly, all other ATG8s formed two more sub-groups represented by AtATG8f and the longer ATG8s, separately ( <xref ref-type="fig" rid="pone-0037217-g001"> Fig. 1A </xref> ). The <span class="trait">branching</span> pattern of the phylogenetic tree suggests that a major functional specialization event of plant ATG8s may have happened when tracheophytes firstly emerged.
					</p>
				
					<p>&nbsp;&nbsp;
						In order to improve the NUE and <span class="trait">yield</span> of crops, many genes have been analyzed mainly through transgenic (over-expression) studies. Traditional approach has focused on the essential genes in nitrogen uptake and assimilation, including <italic> NRT </italic> s ( <italic> nitrate transporters </italic> ), <italic> NR </italic> ( <italic> nitrate reductase </italic> ), <italic> <span class="gene">NiR</span> </italic> ( <italic> nitrite reductase </italic> ), <italic> <span class="gene">GS</span> </italic> ( <italic> glutamine synthesis </italic> ), and <italic> GOGAT </italic> ( <italic> glutamate synthesis </italic> ) <xref ref-type="bibr" rid="pone.0037217-Pathak1"> [58] </xref> . Despite the great efforts, the corresponding transgenic plants have shown little improvement in NUE and <span class="trait">yield</span> <xref ref-type="bibr" rid="pone.0037217-Pathak1"> [58] </xref> , except in one case: the over-expression of <italic> NADH-GOGAT </italic> ( <italic> NADH-dependent glutamate synthesis </italic> ) of rice enhanced grain filling and increased grain weight <xref ref-type="bibr" rid="pone.0037217-Yamaya1"> [59] </xref> . Therefore, new pathways should be explored for transgenic purposes. Since autophagy is one of the main pathways for protein degradation, and very likely contributes to the nitrogen pool for seed production, autophagy-related genes may have NUE potential and could probably improve the <span class="trait">yield.</span> Previous reports and our study have seen the reduced fitness and <span class="trait">fertility</span> of several <italic> atg </italic> mutants, including <italic> <span class="gene">atg5</span> </italic> (data not shown), <italic> atg7 </italic> , <italic> atg9, atg10 </italic> <xref ref-type="bibr" rid="pone.0037217-Doelling1"> [29] </xref> , <xref ref-type="bibr" rid="pone.0037217-Phillips1"> [33] </xref> , <xref ref-type="bibr" rid="pone.0037217-Hanaoka1"> [38] </xref> , and our study showed that the over-expression of a specific ATG8 could make soybean cells and Arabidopsis plants grow better under both nitrogen-rich and nitrogen-poor conditions. Clearly, autophagy is a pathway with NUE potentials. It is noteworthy that our strategy accord with one of the strategies proposed in <xref ref-type="bibr" rid="pone.0037217-Ainsworth1"> [60] </xref> , which is to modify C- and N- allocation and to elevate the strength of the sink <xref ref-type="bibr" rid="pone.0037217-Ainsworth1"> [60] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						In summary, our study has linked autophagy to plant nitrogen homeostasis/distribution as well as <span class="trait">yield</span> formation, and has identified a new node with great potential to be targeted in molecular/transgenic breeding to enhance NUE and improve <span class="trait">yield</span> in crop species. Further study will be carried out to elucidate the detailed molecular mechanisms through which <italic> <span class="gene">GmATG8c</span> </italic> promotes NUE and increases <span class="trait">yield.</span>
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Mapping of quantitative trait loci for canopy-wilting trait in soybean (Glycine max L. Merr).</h2>
			<h3>PMID:22566068  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abdel-Haleem H(1), Carter TE Jr, Purcell LC, King CA, Ries LL, Chen P, Schapaugh W Jr, Sinclair TR, Boerma HR.
				<br/> 
				年份：2012 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress adversely affects [Glycine max (L.) Merr] soybean at most developmental stages, which collectively results in <span class="trait">yield</span> reduction. Little information is available on relative contribution and chromosomal locations of quantitative trait loci (QTL) conditioning <span class="trait">drought</span> tolerance in soybean. A Japanese germplasm accession, PI 416937, was found to possess <span class="trait">drought</span> <span class="trait">resistance.</span> Under moisture-deficit conditions, PI 416937 wilted more slowly in the field than elite cultivars and has been used as a parent in breeding programs to improve soybean productivity. A recombinant inbred line (RIL) population was derived from a cross between PI 416937 and Benning, and the population was phenotyped for canopy wilting under rain-fed field conditions in five distinct environments to identify the QTL associated with the canopy-wilting trait. In a combined analysis over environments, seven QTL that explained 75 % of the variation in canopy-wilting trait were identified on different chromosomes, implying the complexity of this trait. Five QTL inherited their positive alleles from PI 416937. Surprisingly, the other two QTL inherited their positive alleles from Benning. These putative QTL were co-localized with other QTL previously identified as related to plant abiotic stresses in soybean, suggesting that canopy-wilting QTL may be associated with additional morpho-physiological traits in soybean. A locus on chromosome 12 <span class="gene">(Gm12)</span> from PI 416937 was detected in the combined analysis as well as in each individual environment, and explained 27 % of the variation in canopy-wilting. QTL identified in PI 416937 could provide an efficient means to augment field-oriented development of drought-tolerant soybean cultivars.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of the <italic> Arabidopsis thaliana BBX32 </italic> Gene in Soybean Increases Grain <span class="trait">Yield</span></h2>
			<h3>PMID:22363475</h3>
				

				
					<h3>PMCID:3281879</h3>
					<h4>期刊:PLoS ONE</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Preuss </surname>  <given-names> Sasha B. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meister </surname>  <given-names> Robert </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Qingzhang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Urwin </surname>  <given-names> Carl P. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tripodi </surname>  <given-names> Federico A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Screen </surname>  <given-names> Steven E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Anil </surname>  <given-names> Veena S. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn2">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Shuquan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Morrell </surname>  <given-names> James A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Grace </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ratcliffe </surname>  <given-names> Oliver J. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reuber </surname>  <given-names> T. Lynne </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khanna </surname>  <given-names> Rajnish </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Goldman </surname>  <given-names> Barry S. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bell </surname>  <given-names> Erin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ziegler </surname>  <given-names> Todd E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> McClerren </surname>  <given-names> Amanda L. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ruff </surname>  <given-names> Thomas G. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Petracek </surname>  <given-names> Marie E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 12 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Crop <span class="trait">yield</span> is a highly complex quantitative trait. Historically, successful breeding for improved grain <span class="trait">yield</span> has led to crop plants with improved source capacity, altered plant architecture, and increased <span class="trait">resistance</span> to abiotic and biotic stresses. To date, transgenic approaches towards improving crop grain <span class="trait">yield</span> have primarily focused on protecting plants from herbicide, insects, or disease. In contrast, we have focused on identifying genes that, when expressed in soybean, improve the intrinsic ability of the plant to <span class="trait">yield</span> more. Through the large scale screening of candidate genes in transgenic soybean, we identified an Arabidopsis thaliana B-box domain gene (AtBBX32) that significantly increases soybean grain <span class="trait">yield</span> year after year in multiple transgenic events in multi-location field trials. In order to understand the underlying physiological changes that are associated with increased <span class="trait">yield</span> in transgenic soybean, we examined phenotypic differences in two AtBBX32-expressing lines and found increases in plant height and node, flower, pod, and seed number. We propose that these phenotypic changes are likely the result of changes in the timing of reproductive development in transgenic soybean that lead to the increased duration of the pod and seed development period. Consistent with the role of BBX32 in A. thaliana in regulating light signaling, we show that the constitutive expression of AtBBX32 in soybean alters the abundance of a subset of gene transcripts in the early morning hours. In particular, AtBBX32 alters transcript levels of the soybean clock genes <span class="gene">GmTOC1</span> and LHY-CCA1-like2 <span class="gene">(GmLCL2).</span> We propose that through the expression of AtBBX32 and modulation of the abundance of circadian clock genes during the transition from dark to light, the timing of critical phases of reproductive development are altered. These findings demonstrate a specific role for AtBBX32 in modulating soybean development, and demonstrate the validity of expressing single genes in crops to deliver increased agricultural productivity.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Data from <italic> A. thaliana </italic> indicated that overexpression of <italic> AtBBX32 </italic> caused increased hypocotyl growth <xref ref-type="bibr" rid="pone.0030717-Holtan1"> [9] </xref> , suggesting that, when expressed in a crop plant, the gene might lead to higher overall rates of growth. These results led us to test the efficacy of <italic> AtBBX32 </italic> in improving soybean <span class="trait">yield.</span> We generated eight independently transformed <italic> AtBBX32 </italic> expressing soybean lines and assayed <span class="trait">yield</span> in multi-location field trials conducted over three seasons; two seasons in the United States and one season in Argentina. Six of the eight transgenic events showed consistent <span class="trait">yield</span> gains (an increase in kilograms of seed per hectare) in a meta-analysis across the three seasons ( <xref ref-type="table" rid="pone-0030717-t001"> Table 1 </xref> ). Four of the eight transgenic events yielded more than 5 percent over controls. Transcript analysis from <span class="gene">V3</span> leaf tissue revealed that seven of the eight lines express <italic> AtBBX32 </italic> at similar levels, while line 4 does not express the transgene at detectable levels ( <xref ref-type="supplementary-material" rid="pone.0030717.s004"> Table S1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Since constitutive expression of the <italic> A. thaliana </italic> B-box gene <italic> BBX32 </italic> leads to changes in soybean gene expression that are restricted to a specific time of day, we suspected that soybean may encode homologs of <italic> AtBBX32 </italic> with similar function. In order to better understand the existing pathway in soybean that <italic> AtBBX32 </italic> functions in and identify other candidate genes that lead to increased <span class="trait">yield</span> when overexpressed in soybean, we identified the B-box family in soybean, including the soybean homologs of <italic> AtBBX32 </italic> . We generated a phylogenetic analysis of the soybean and <italic> A. thaliana </italic> B-box gene families and further analyzed the expression patterns of the soybean B-box gene family in the microarray described above. The <italic> A. thaliana </italic> genome encodes 32 predicted B-box domain proteins <xref ref-type="bibr" rid="pone.0030717-Khanna1"> [17] </xref> , while the paleopolyploid soybean genome encodes 61 predicted B-box containing proteins ( <xref ref-type="supplementary-material" rid="pone.0030717.s001"> Figure S1 </xref> , <xref ref-type="supplementary-material" rid="pone.0030717.s006"> Table S3 </xref> ). Microarray analysis from R5 stage field grown wildtype soybean plants indicated that a significant proportion of the soy B-box genes are temporally regulated. Phylogenetic analysis of the soybean B-box family was used to identify putative soybean homologs of <italic> AtBBX32 </italic> . Of the <italic> AtBBX32 </italic> -like soybean genes that encode a single B-box domain, thirteen grouped with the seven Arabidopsis single B-box genes ( <xref ref-type="supplementary-material" rid="pone.0030717.s002"> Figure S2 </xref> ). Of these, <italic> Glycine max BBX52 <span class="gene">(GmBBX52)</span> </italic> and <italic> Glycine max BBX53 <span class="gene">(GmBBX53)</span> </italic> have the closest phylogenetic relationship to the <italic> A. thaliana BBX32 </italic> gene and show evidence for a microsyntenic relationship between the Arabidopsis and soybean genomic regions <xref ref-type="bibr" rid="pone.0030717-Tang1"> [18] </xref> . The two soybean genes are similar in both nucleotide (92 percent identical) and amino acid sequence (88 percent identical) and presumably arose from the duplication of the soybean genome. Like <italic> AtBBX32 </italic> , both soybean genes are predicted to encode a single N-terminal B-box domain. Additionally, both soybean genes showed temporal patterns of transcript abundance with peaks in expression at similar times to each other in the microarray experiment ( <xref ref-type="supplementary-material" rid="pone.0030717.s001"> Figure S1 </xref> ), though each are detected by different probesets ( <xref ref-type="supplementary-material" rid="pone.0030717.s006"> Table S3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To examine whether <italic> <span class="gene">GmBBX52</span> </italic> and <italic> <span class="gene">GmBBX53</span> </italic> are functional homologs of <italic> AtBBX32 </italic> , we tested whether overexpression of each gene can produce phenotypes in soybean similar to those caused by expression of <italic> AtBBX32 </italic> . To do this, we generated transgenic lines that constitutively express either <italic> <span class="gene">GmBBX52</span> or <span class="gene">GmBBX53</span> </italic> as well as lines containing a miRNA construct designed to decrease transcript levels of <italic> <span class="gene">GmBBX52</span> </italic> . We tested eight independently generated transgenic lines of both the <italic> <span class="gene">GmBBX52</span> </italic> constitutive expression construct and the <italic> <span class="gene">GmBBX52</span> </italic> - miRNA construct and 4 independent events from the <italic> <span class="gene">GmBBX53</span> </italic> expression construct in a single year of field trials designed to assess <span class="trait">yield</span> ( <xref ref-type="table" rid="pone-0030717-t004"> Table 4 </xref> ). Lines overexpressing <italic> <span class="gene">GmBBX52</span> </italic> yielded, on average, 6.1 percent more kilograms per hectare than did wildtype control plants, while the top performing line yielded 9.0 percent more per hectare. Transgenic lines overexpressing <italic> <span class="gene">GmBBX53</span> </italic> yielded 4.1 percent higher than the wildtype control, while the top event improved <span class="trait">yield</span> by 6.7 percent over the wildtype control. In contrast, miRNA mediated suppression of the <italic> <span class="gene">GmBBX52</span> </italic> transcript led to a significant decrease in <span class="trait">yield</span> ( <xref ref-type="table" rid="pone-0030717-t004"> Table 4 </xref> ). The <italic> <span class="gene">GmBBX52</span> </italic> miRNA lines yielded, on average, 5.5 percent less than control lines while the lowest yielding line across the eight miRNA lines produced 11.8 percent less than controls. Confirming the miRNA impact, seven of eight events showed significant decreases in <italic> <span class="gene">GmBBX52</span> </italic> transcript levels ( <xref ref-type="supplementary-material" rid="pone.0030717.s004"> Table S1 </xref> ). These data indicate that overexpression of the soybean homologs of <italic> AtBBX32 </italic> , <italic> <span class="gene">GmBBX52</span> </italic> and <italic> <span class="gene">GmBBX53</span> </italic> , enhances <span class="trait">yield</span> similarly to what was observed in soybean plants expressing the <italic> A. thaliana </italic> gene, <italic> BBX32 </italic> . In addition, we found that overexpression of either <italic> <span class="gene">GmBBX52</span> </italic> or <italic> <span class="gene">GmBBX53</span> </italic> delayed maturity by 2&#x02013;4.5 days compared to control, while the day of flowering compared to control was not consistently affected ( <xref ref-type="table" rid="pone-0030717-t004"> Table 4 </xref> ). Neither day of maturity or day of flowering was consistently altered in soybean expressing the <italic> <span class="gene">GmBBX52</span> </italic> miRNA construct. These data support the hypothesis that <italic> <span class="gene">GmBBX52</span> </italic> and <italic> <span class="gene">GmBBX53</span> </italic> share a comparable mechanism of action to <italic> AtBBX32 </italic> . Furthermore, the field data from our RNAi lines suggest that wild-type expression levels of the <italic> <span class="gene">GmBBX52</span> </italic> gene may be required to maintain <span class="trait">yield</span> potential in soybean. Our findings suggest that the soybean and <italic> A. thaliana </italic> genes likely perform similar <italic> in-vivo </italic> functions and that the <italic> AtBBX32 </italic> gene product in soybean is likely to function within the same biochemical framework as the endogenous soybean homologs of <italic> AtBBX32, <span class="gene">GmBBX52</span> and <span class="gene">GmBBX53</span> </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Analysis of individual lines from one season of field testing of soybean plants overexpressing the soybean homologs of <italic> AtBBX32 </italic> , <italic> <span class="gene">GmBBX52</span> </italic> and <italic> <span class="gene">GmBBX53</span> </italic> , demonstrate that the expression of the soybean genes leads to significant <span class="trait">yield</span> improvements relative to control. miRNA mediated knockdown of <italic> <span class="gene">GmBBX52</span> </italic> leads to decreased <span class="trait">yield</span> relative to control. The difference in the days to flower (DOF) between the transgenic lines and control was calculated to determine delta DOF. The difference in days to final maturity (MAT) was examined in transgenic lines and compared to control to determine delta MAT (units&#x0200a;=&#x0200a;days). p-values were based on the difference between the transgenic lines and wildtype control.
					</p>
				
					<p>&nbsp;&nbsp;
						To further test whether the soybean homologs of <italic> AtBBX32 </italic> , <italic> <span class="gene">GmBBX52</span> </italic> and <italic> <span class="gene">GmBBX53</span> </italic> , are functionally analogous to <italic> AtBBX32 </italic> , we measured the transcript abundance of <italic> <span class="gene">GmTOC1</span> </italic> and <italic> <span class="gene">GmLCL2</span> </italic> in two lines overexpressing either <italic> <span class="gene">GmBBX52</span> </italic> (line 7) or <italic> <span class="gene">GmBBX53</span> </italic> (line 2) from previous <span class="trait">yield</span> tests over a 24 hour time course ( <xref ref-type="supplementary-material" rid="pone.0030717.s003"> Figure S3 </xref> ). In both lines we observed similar results to those in <italic> AtBBX32 </italic> expressing soybean lines; <italic> <span class="gene">GmLCL2Y</span> </italic> expression is reduced near ZT 0, while <italic> <span class="gene">GmTOC1</span> </italic> expression is increased near ZT 0 in transgenic lines compared to control. In addition to the <span class="trait">yield</span> data, these data further support our hypothesis that the two soybean homologs have a similar role to <italic> AtBBX32 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Yield</span> was collected from <italic> <span class="gene">GmBBX52</span> </italic> and <italic> <span class="gene">GmBBX53</span> </italic> overexpression and miRNA constructs from one season. Events overexpressing <italic> <span class="gene">GmBBX53</span> </italic> (pMON98939) were evaluated in 2009 in two row, 4.6 meter plots (0.9 meter alley, 0.75 meter row space) planted in a Group Unbalanced Block Design (GUBD). Each of four pMON98939 events were randomly assigned to plots within a dedicated pMON98939 block. Every block (1 per replication) was randomized within each replication for each environment. This trial was planted at 22 environments in the United States with 3 replications per environment. Standard regional agronomic soybean production practices were used for trial management. Environments or replications within an environment with a coefficient of variation &#x02265;15 percent or damaged by severe environmental were excluded from analysis, resulting in 21 environments included in analysis of <span class="trait">yield.</span> Day of flowering and day of maturity data were collected on each event from 6 and 8 environments, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Events overexpressing <italic> <span class="gene">GmBBX52</span> </italic> (pMON108080) or miRNA of <italic> <span class="gene">GmBBX52/53</span> </italic> (pMON93914) were evaluated in 2009 in two row, 4.6 meter plots (0.9 meter alley, 0.75 meter row space) planted in a GUBD design. Each of 8 events of pMON108080 and pMON93914 were randomly assigned to plots within dedicated pMON108080 and pMON93914 blocks. pMON108080 and pMON93914 blocks were randomized within each environment. In 2009 events of pMON108080 and pMON93914 were planted at 10 environments in the United States with a single replication at each environment. Standard regional agronomic soybean production practices were used for trial management. Environments or replications within an environment with a coefficient of variation &#x02265;15 percent or damaged by severe environmental were excluded from analysis, resulting in 9 and 10 environments included in analysis of <span class="trait">yield</span> for pMON108080 and pMON93914, respectively. Day of flowering data were collected from 7 environments for both trials and day of maturity data were collected from 6 and 7 environments for BA831 and BA832, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						A field study was conducted at Jerseyville, IL following a split plot design with three plot replications per entry. Each plot consisted of two rows 0.75 m apart and 3.6 m long. Plots were over-seeded and thinned at the <span class="gene">V2</span> stage to a final population of 56 plants m <sup> &#x02212;2 </sup> . Leaf senescence was rated on a 1&#x02013;9 scale based on whole plot appearance. 9&#x0200a;=&#x0200a;dark green, no yellow leaves on the top canopy; 5&#x0200a;=&#x0200a;40 percent yellow leaves, 10 percent fallen leaves; 1&#x0200a;=&#x0200a;more than 95 percent fallen leaves. Assessment was determined based on visual inspection. Maturity was rated on 0&#x02014;100 percent scale based on the appearance of brown pods across the whole plot based on visual observation. Pod numbers were counted when 80 percent of the pods turned brown across the field. Plants were harvested from a 0.30 m section and the number of pods at each node on the main stem and branches was determined. Plant height was measured at R5 stage by using a barcoded ruler and a barcode scanner connected to a laptop computer. The barcoded ruler was placed in the middle of the row in a plot and the barcode corresponding to the average height of the row was scanned. <span class="trait">Yield</span> was determined after combine harvest of the plots used to collect physiology measurements. Data were analyzed with a split-plot design. Thinning density was the whole plot factor and event was the split factor. Individual replicated site analyses used a mixed model <disp-formula> <graphic xlink:href="pone.0030717.e007"/> </disp-formula> where Y <sub> ijr </sub> &#x0200a;=&#x0200a;unique individual observation, U&#x0200a;=&#x0200a;overall mean, T <sub> i </sub> &#x0200a;=&#x0200a;density effect, E <sub> j </sub> &#x0200a;=&#x0200a;gene effect, TE <sub> ij </sub> &#x0200a;=&#x0200a;density by gene interaction effect, B <sub> r </sub> &#x0200a;=&#x0200a;random replication effect, BT <sub> ir </sub> &#x0200a;=&#x0200a;random replication by density interaction, and e <sub> ijr </sub> &#x0200a;=&#x0200a;residual error. T, E and TE are fixed effects and others are random effects. Statistical analyses were carried out using PROC MIXED procedure of SAS/STAT 9.2.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Enhanced production of medicinal polysaccharide by submerged fermentation of Lingzhi or Reishi medicinal mushroom Ganoderma lucidium (W.Curt.:Fr.) P. Karst. Using statistical and evolutionary optimization methods.</h2>
			<h3>PMID:22324411  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Baskar G(1), Sathya SR.
				<br/> 
				年份：2011
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Statistical and evolutionary optimization of media composition was employed for the production of medicinal exopolysaccharide (EPS) by Lingzhi or Reishi medicinal mushroom Ganoderma lucidium MTCC 1039 using soya bean meal flour as low-cost substrate. Soya bean meal flour, ammonium chloride, glucose, and pH were identified as the most important variables for EPS <span class="trait">yield</span> using the two-level Plackett-Burman design and further optimized using the central composite design (CCD) and the artificial neural network (ANN)-linked genetic algorithm (GA). The high value of coefficient of determination of <span class="gene">ANN</span> (R² = 0.982) indicates that the <span class="gene">ANN</span> model was more accurate than the second-order polynomial model of CCD (R² = 0.91) for representing the effect of media composition on EPS <span class="trait">yield.</span> The predicted optimum media composition using ANN-linked GA was soybean meal flour 2.98%, glucose 3.26%, ammonium chloride 0.25%, and initial pH 7.5 for the maximum predicted EPS <span class="trait">yield</span> of 1005.55 mg/L. The experimental EPS <span class="trait">yield</span> obtained using the predicted optimum media composition was 1012.36 mg/L, which validates the high degree of accuracy of evolutionary optimization for enhanced production of EPS by submerged fermentation of G. lucidium.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> In Silico </italic> Identification and Comparative Genomics of Candidate Genes Involved in Biosynthesis and Accumulation of Seed Oil in Plants</h2>
			<h3>PMID:22312320</h3>
				

				
					<h3>PMCID:3270531</h3>
					<h4>期刊:Comparative and Functional Genomics</h4>
				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Sharma </surname>  <given-names> Arti </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 14 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genes involved in fatty acids biosynthesis, modification and oil body formation are expected to be conserved in structure and function in different plant species. However, significant differences in the composition of fatty acids and total oil contents in seeds have been observed in different plant species. Comparative genomics was performed on 261 genes involved in fatty acids biosynthesis, TAG synthesis, and oil bodies formation in Arabidopsis, Brassica rapa, castor bean and soybean. In silico expression analysis revealed that stearoyl desaturase, FatB, FAD2, oleosin and <span class="gene">DGAT</span> are highly abundant in seeds, thereby considered as ideal candidates for mining of favorable alleles in natural population. Gene structure analysis for major genes, ACCase, FatA, FatB, FAD2, <span class="gene">FAD3</span> and DGAT, which are known to play crucial role in oil synthesis revealed that there are uncommon variations (SNPs and INDELs) which lead to varying content and composition of fatty acids in seed oil. The predicted variations can provide good targets for seed oil QTL identification, understanding the molecular mechanism of seed oil accumulation, and genetic modification to enhance seed oil <span class="trait">yield</span> in plants.
					</p>
				
				<h2>正文</h2>
				
			

			<hr/> 
		

			
			<h2>文献标题:Biosurfactant-producing bacterium, Pseudomonas aeruginosa MA01 isolated from spoiled apples: physicochemical and structural characteristics of isolated biosurfactant.</h2>
			<h3>PMID:22036074  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abbasi H(1), Hamedi MM, Lotfabad TB, Zahiri HS, Sharafi H, Masoomi F, Moosavi-Movahedi AA, Ortiz A, Amanlou M, Noghabi KA.
				<br/> 
				年份：2012 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						An extensive investigation was conducted to isolate indigenous bacterial strains with outstanding performance for biosurfactant production from different types of spoiled fruits, food-related products and food processing industries. An isolate was selected from 800 by the highest biosurfactant <span class="trait">yield</span> in soybean oil medium and it was identified by 16S rRNA and the two most relevant hypervariable regions of this gene; <span class="gene">V3</span> and V6 as Pseudomonas aeruginosa MA01. The isolate was able to produce 12 g/l of a glycolipid-type biosurfactant and generally less efficient to emulsify vegetable oils compared to hydrocarbons and could emulsify corn and coconut oils more than 50%. However, emulsification index (E(24)) of different hydrocarbons including hexane, toluene, xylene, brake oil, kerosene and hexadecane was between 55.8% and 100%. The surface tension of pure water decreased gradually with increasing biosurfactant concentration to 32.5 mNm(-1) with critical micelle concentration (CMC) value of 10.1mg/l. Among all carbon substrates examined, vegetable oils were the most effective on biosurfactant production. Two glycolipid fractions were purified from the biosurfactant crude extracts, and FTIR and ES-MS were used to determine the structure of these compounds. The analysis indicated the presence of three major monorhamnolipid species: R(1)C(10)C(10), R(1)C(10)C(12:1), and R(1)C(10)C(12); as well as another three major dirhamnolipid species: R(2)C(10)C(10), R(2)C(10)C(12:1), and R(2)C(10)C(12). The strain sweep experiment for measuring the linear viscoelastic of biosurfactant showed that typical behavior characteristics of a weak viscoelastic gel, with storage modulus greater than loss modulus at all frequencies examined, both showing some frequency dependence.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcript profiling reveals expression differences in wild-type and glabrous soybean lines</h2>
			<h3>PMID:22029708</h3>
				

				
					<h3>PMCID:3217893</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes" id="A1">  <name>  <surname> Hunt </surname>  <given-names> Matt </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> hunt.547@osu.edu </email>  </contrib>  <contrib contrib-type="author" equal-contrib="yes" id="A2">  <name>  <surname> Kaur </surname>  <given-names> Navneet </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> navneet@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Stromvik </surname>  <given-names> Martina </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> martina.stromvik@mcgill.ca </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A4">  <name>  <surname> Vodkin </surname>  <given-names> Lila </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> l-vodkin@illinois.edu </email>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Trichome hairs affect diverse agronomic characters such as seed weight and <span class="trait">yield,</span> prevent insect damage and reduce loss of water but their molecular control has not been extensively studied in soybean. Several detailed models for trichome development have been proposed for Arabidopsis thaliana, but their applicability to important crops such as cotton and soybean is not fully known. RESULTS: Two high throughput transcript sequencing methods, Digital Gene Expression (DGE) Tag Profiling and RNA-Seq, were used to compare the transcriptional profiles in wild-type (cv. Clark standard, CS) and a mutant (cv. Clark glabrous, i.e., trichomeless or hairless, CG) soybean isoline that carries the dominant <span class="gene">P1</span> allele. DGE data and RNA-Seq data were mapped to the cDNAs (Glyma models) predicted from the reference soybean genome, Williams 82. Extending the model length by 250 bp at both ends resulted in significantly more matches of authentic DGE tags indicating that many of the predicted gene models are prematurely truncated at the 5' and 3' UTRs. The genome-wide comparative study of the transcript profiles of the wild-type versus mutant line revealed a number of differentially expressed genes. One highly-expressed gene, <span class="gene">Glyma04g35130,</span> in wild-type soybean was of interest as it has high homology to the cotton gene GhRDL1 gene that has been identified as being involved in cotton fiber initiation and is a member of the BURP protein family. Sequence comparison of <span class="gene">Glyma04g35130</span> among Williams 82 with our sequences derived from CS and CG isolines revealed various SNPs and indels including addition of one nucleotide C in the CG and insertion of ~60 bp in the third exon of CS that causes a frameshift mutation and premature truncation of peptides in both lines as compared to Williams 82. CONCLUSION: Although not a candidate for the <span class="gene">P1</span> locus, a BURP family member <span class="gene">(Glyma04g35130)</span> from soybean has been shown to be abundantly expressed in the CS line and very weakly expressed in the glabrous CG line. RNA-Seq and DGE data are compared and provide experimental data on the expression of predicted soybean gene models as well as an overview of the genes expressed in young shoot tips of two closely related isolines.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Unlike Arabidopsis and cotton, the developmental genetics of soybean trichomes has not been studied extensively. However, there are several soybean trichome developmental mutants available, including <italic> <span class="gene">P1</span> </italic> (glabrous), <italic> pc </italic> (curly pubescence), <italic> Pd </italic> (dense pubescence), <italic> Ps </italic> (sparse pubescence), and <italic> p2 </italic> (puberulent) that are each controlled by a different single Mendelian locus [ <xref ref-type="bibr" rid="B27"> 27 </xref> ]. These mutants have been used to relate the importance of trichome to insect <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="B4"> 4 </xref> , <xref ref-type="bibr" rid="B28"> 28 </xref> , <xref ref-type="bibr" rid="B29"> 29 </xref> ], evapotranspiration [ <xref ref-type="bibr" rid="B2"> 2 </xref> , <xref ref-type="bibr" rid="B30"> 30 </xref> , <xref ref-type="bibr" rid="B31"> 31 </xref> ] and other <span class="trait">yield</span> related characteristics. However, until now, none of these glabrous classical mutations has been studied at the molecular level. We studied the dominant <italic> <span class="gene">P1</span> </italic> glabrous soybean mutant using two high throughput transcript sequencing technologies to reveal major expression differences between the two genotypes. RNA and DNA blots further characterized a highly differentially expressed BURP family member <italic> <span class="gene">Glyma04g35130</span> </italic> that varied between the two genotypes and may be associated with trichome development in soybean although it is not a candidate for the <italic> <span class="gene">P1</span> </italic> locus.
					</p>
				
					<p>&nbsp;&nbsp;
						Among the tags overexpressed in the CS line, one particular tag corresponds to a gene located on <italic> Glyma04 </italic> chromosome, specifically <italic> <span class="gene">Glyma04g35130</span> </italic> , and showed &#x0003e;2000-fold expression difference between CS/CG = 2,545/1.06 tags per million aligned tags (Table <xref ref-type="table" rid="T1"> 1 </xref> ). The <italic> <span class="gene">Glyma04g35130</span> </italic> gene is a member of the <italic> BURP </italic> gene family. It has high homology to the cotton gene- <italic> <span class="trait">RESISTANCE</span> TO <span class="trait">DROUGHT</span> RD22-like 1 </italic> ( <italic> GhRDL1 </italic> ), involved in cotton fiber initiation and member of the BURP protein domain family [ <xref ref-type="bibr" rid="B15"> 15 </xref> , <xref ref-type="bibr" rid="B16"> 16 </xref> ]. Soybean has a total of 23 BURP domain containing genes and BURP gene family members from other species are known to have diverse functions [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. Some of the proposed functions of BURP family members include: regulation of fruit ripening in tomato [ <xref ref-type="bibr" rid="B36"> 36 </xref> , <xref ref-type="bibr" rid="B37"> 37 </xref> ], response to <span class="trait">drought</span> stress induced by abscisic acid in Arabidopsis [ <xref ref-type="bibr" rid="B38"> 38 </xref> ], tapetum development in rice [ <xref ref-type="bibr" rid="B39"> 39 </xref> ], and seed coat development in soybean [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]. In Clark, the DGE0000012 tag found to correspond to <italic> <span class="gene">Glyma04g35130</span> </italic> is the 12 <sup> th </sup> most abundant tag in the DGE data set. For perspective, the 4 <sup> th </sup> most abundant tag with a normalized count of 4,903 tags matches a chlorophyll a/b binding factor as do several of the most abundant tags (Additional file <xref ref-type="supplementary-material" rid="S2"> 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> <span class="gene">Glyma04g35130</span> BURP </italic> gene showed high homology to the cotton gene <italic> <span class="trait">RESISTANCE</span> TO <span class="trait">DROUGHT</span> RD22-like 1 </italic> ( <italic> GhRDL1 </italic> ) that is involved in cotton fiber initiation and is also a member of the BURP protein family. The <italic> <span class="gene">Glyma04g35130</span> BURP </italic> gene and <italic> <span class="gene">SCB1</span> </italic> , seed coat burp domain protein 1 ( <italic> <span class="gene">Glyma07g28940</span> </italic> ) fall into one BURP protein family- BURPV, when 41 BURP proteins from different species were classified into 5 subfamilies [ <xref ref-type="bibr" rid="B26"> 26 </xref> ]. <italic> <span class="gene">SCB1</span> </italic> may play a role in the differentiation of the seed coat parenchyma cells and is localized on the cell wall of soybean [ <xref ref-type="bibr" rid="B40"> 40 </xref> ]. But it should be noted that despite high sequence homology among the <italic> BURP </italic> domain containing genes, the function of each BURP protein seems to greatly vary among plants. The <italic> <span class="gene">Glyma04g35130</span> </italic> BURP gene does not seem to have a direct role in trichome formation but the possibility is open that it may be indirectly involved in some soybean genotypes.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Plant growth-promotion (PGP) activities and molecular characterization of rhizobacterial strains isolated from soybean (Glycine max L. Merril) plants against charcoal rot pathogen, Macrophomina phaseolina.</h2>
			<h3>PMID:21833548  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Choudhary DK(1).
				<br/> 
				年份：2011 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Charcoal rot disease, caused by the fungus Macrophomina phaseolina, leads to significant <span class="trait">yield</span> losses of soybean crops. One strategy to control charcoal rot is the use of antagonistic, root-colonizing bacteria. Rhizobacteria A(5)F and FPT(7)21 and Pseudomonas sp. strain GRP(3) were characterized for their plant growth-promotion activities against the pathogen. Rhizobacterium FPT(7)21 exhibited higher antagonistic activity against the pathogen on dual plate assay compared to strain A(5)F and GRP(3). FPT(7)21 and GRP(3) gave decreased disease intensity in terms of average number of pathogen-infested plants. Lipoxygenase (LOX), phenylalanine ammonia-lyase (PAL), and peroxidase (POD) activities were estimated in extracts of plants grown from seeds that were treated with rhizobacteria, and inoculated with spore suspension of M. phaseolina. The activity of these enzymes after challenge with the test pathogen increased. Strains FPT(7)21 and GRP(3) exhibited maximum increases in LOX, <span class="gene">PAL</span> and POD activity (U mg(-1) fresh leaf wt) compared to strain A(5)F.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The alleles at the <span class="gene">E1</span> locus impact the expression pattern of two soybean FT-like genes shown to induce flowering in Arabidopsis.</h2>
			<h3>PMID:21681526  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Thakare D(1), Kumudini S, Dinkins RD.
				<br/> 
				年份：2011 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A small gene family of phosphatidyl ethanolamine-binding proteins (PEBP) has been shown to function as key regulators in flowering; in Arabidopsis thaliana the FT protein promotes flowering whilst the closely related TFL1 protein represses flowering. Control of flowering time in soybean [Glycine max (L.) Merrill] is important for geographic adaptation and maximizing <span class="trait">yield.</span> Soybean breeders have identified a series of loci, the E-genes, that control photoperiod-mediated flowering time, yet how these loci control flowering is poorly understood. The objectives of this study were to evaluate the expression of <span class="gene">GmFT-like</span> genes in the <span class="gene">E1</span> near-isogenic line (NIL) background. Of the 20 closely related PEBP proteins in the soybean genome, ten are similar to the Arabidopsis FT protein. Expression analysis of these ten <span class="gene">GmFT-like</span> genes confirmed that only two are detectable in the conditions tested. Further analysis of these two genes in the <span class="gene">E1</span> NILs grown under short-day (SD) and long-day (LD) conditions showed a diurnal expression and tissue specificity expression commensurate with soybean flowering time under SD and LD conditions, suggesting that these were good candidates for flowering induction in soybean. Arabidopsis ft mutant lines flowered early when transformed with the two soybean genes, suggesting that the soybean genes can complement the Arabidopsis FT function. Flowering time in <span class="gene">E1</span> NILs is consistent with the differential expression of the two <span class="gene">GmFT-like</span> genes under SD and LD conditions, suggesting that the <span class="gene">E1</span> locus, at least in part, impacts time to flowering through the regulation of soybean FT expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GmFtsH9 expression correlates with in vivo photosystem II function: chlorophyll a fluorescence transient analysis and eQTL mapping in soybean.</h2>
			<h3>PMID:21638036  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yin Z(1), Meng F, Song H, Wang X, Chao M, Zhang G, Xu X, Deng D, Yu D.
				<br/> 
				年份：2011 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Filamentation temperature-sensitive H (FtsH) is an ATP-dependent zinc metalloprotease involved in diverse biological functions. There are 12 FtsH proteins in Arabidopsis, among which AtFtsH2 plays an important role in regulating the turnover of photosystem II (PSII) reaction center D1 protein and the development of the photosynthetic apparatus. Here, we have identified 11 FtsH genes in the soybean genome by a bioinformatics approach. These soybean FtsH genes corresponded to seven Arabidopsis FtsH genes, suggesting that the main characteristics of soybean FtsH genes were formed before the evolutionary split of soybean and Arabidopsis. Phylogenetic analyses allowed us to clone a soybean AtFtsH2-like gene designated as <span class="gene">GmFtsH9.</span> The predicted protein of <span class="gene">GmFtsH9</span> consists of 690 amino acids and contains three typical FtsH proteins conserved domains. The expression level of <span class="gene">GmFtsH9</span> was determined in a soybean recombinant inbred line population under a pot experiment conducted for measuring chlorophyll a fluorescence transient parameters, photosynthetic CO(2) fixation rate (P (N)), and seed <span class="trait">yield.</span> Expression quantitative trait loci (eQTL) mapping revealed two trans-acting eQTLs for <span class="gene">GmFtsH9.</span> The significant correlation of gene expression level with chlorophyll a fluorescence transient parameters and the presence of overlapping eQTL (QTL) between gene expression level and chlorophyll a fluorescence transient parameters indicated that <span class="gene">GmFtsH9</span> could be involved in regulating PSII function. These results further lead to the understanding of the mechanism underlying FtsH gene expression, and contribute to the development of marker-assisted selection breeding programs for modulating soybean FtsH gene expression.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic analysis and QTL detection of reproductive period and post-flowering photoperiod responses in soybean.</h2>
			<h3>PMID:21556700  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cheng L(1), Wang Y, Zhang C, Wu C, Xu J, Zhu H, Leng J, Bai Y, Guan R, Hou W, Zhang L, Han T.
				<br/> 
				年份：2011 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Reproductive period (RP) is an important trait of soybean [Glycine max (L.) Merr.] It is closely related to <span class="trait">yield,</span> quality and tolerances to environmental stresses. To investigate the inheritance and photoperiod response of RP in soybean, the F(1), F(2), and F(2:3) populations derived from nine crosses were developed. The inheritance of RP was analyzed through the joint segregation analysis. It was shown that the RP was controlled by one major gene plus polygenes. 181 recombinant inbred lines (RILs) generated from the cross of Xuyong Hongdou × Baohexuan 3 were further used for QTL mapping of RP under normal conditions across 3 environments, using 127 SSR markers. Four QTLs, designated qRP-c-1, qRP-g-1, qRP-m-1 and qRP-m-2, were mapped on C1, G and M linkage groups, respectively. The QTL qRP-c-1 on the linkage group <span class="gene">C1</span> showed stable effect across environments and explained 25.6, 27.5 and 21.4% of the phenotypic variance in Nanjing 2002, Beijing 2003 and Beijing 2004, respectively. Under photoperiod-controlled conditions, qRP-c-1, and two different QTLs designated qRP-l-1 and qRP-o-1, respectively, were mapped on the linkage groups L and O. qRP-o-1 was detected under SD condition and can explained 10.70% of the phenotypic variance. qRP-c-1 and qRP-l-1 were detected under LD condition and for photoperiod sensitivity. The two major-effect QTLs can explain 19.03 and 19.00% of the phenotypic variance, respectively, under LD condition and 16.25 and 14.12%, respectively, for photoperiod sensitivity. Comparative mapping suggested that the two major-effect QTLs, qRP-c-1 and qRP-l-1, might associate with E8 or <span class="gene">GmCRY1a</span> and the maturity gene E3 or <span class="gene">GmPhyA3,</span> respectively. These results could facilitate our understanding of the inheritance of RP and provide information on marker-assisted breeding for high <span class="trait">yield</span> and wide adaptation in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Production of lovastatin by wild strains of Aspergillus terreus.</h2>
			<h3>PMID:21425670  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Patil RH(1), Krishnan P, Maheshwari VL.
				<br/> 
				年份：2011 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A wild fungal strain of Aspergillus terreus, labeled as PM3, was isolated by using the Candida albicans bioassay and confirmed by 18S r DNA analyses. Lovastatin was produced by submerged and solid state fermentations. Of the 30 isolated fungal strains, 11 showed lovastatin production with Aspergillus terreus <span class="gene">PM3</span> being the best with a <span class="trait">yield</span> of 240 mg/L at the 10th day of submerged fermentation. Carboxymethylcellulose had a stimulatory effect on lovastatin production. It restricted uncontrolled filamentous growth, induced pellet formation and, thereby, improved lovastatin <span class="trait">yield.</span> In solid state fermentation (SSF), of the agro wastes from five crops (bran of wheat and rice, husks of red gram and soybean, and green gram straw), wheat bran showed maximum lovastatin production (12.5 mg/g of dry substrate) at pH 7.1 and a temperature of 30 +/- 2 degrees C. Development of a lovastatin production process based on wheat bran as a substrate in SSF is economically attractive as it is a cheap and readily available raw material in agriculture-based countries.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:pHairyRed: a novel binary vector containing the DsRed2 reporter gene for visual selection of transgenic hairy roots.</h2>
			<h3>PMID:21324970  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lin MH(1), Gresshoff PM, Indrasumunar A, Ferguson BJ.
				<br/> 
				年份：2011 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We developed a new plant transformation vector, pHairyRed, for enabling high throughput, non-destructive selection of Agrobacterium rhizogenes-mediated 'hairy-root' transformation. pHairyRed allows easy in planta visualization of transgenic tissue with minimal disturbance to the plant. The DsRed2 reporter gene, encoding a red fluorescent protein, was cloned to <span class="trait">yield</span> pHairyRed (harbouring a multiple cloning site), which was used with the highly efficient K599 A. rhizogenes strain to infect soybean (Glycine max L. Merrill) plants. DsRed2 fluorescence was easily detected in planta for the duration of a 5-week study with negligible levels of background autofluorescence. This enabled visual selection of transformed roots and subsequent excission of non-transformed roots. pHairyRed-transformed roots nodulated normally when inoculated with Bradyrhizobium japonicum. Within the nodule, DsRed2 fluorescence was plant-specific, being absent in the bacteroid-dominated nodule infected zone. To test the reliability of pHairyRed as a high-fidelity binary vector reporter system, the gene encoding the soybean Nod factor receptor, <span class="gene">GmNFR1α,</span> was cloned into the vector for use in a complementation study with a non-nodulating nfr1α mutant of soybean. Complementation was achieved and, without exception, DsRed2 fluorescence was detected in all hairy roots that successfully formed nodules (100%, n = 34). We anticipate broad application of this reporter system for the further analysis of root-related events in soybean and related legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nodulation factor receptor kinase 1α controls nodule organ number in soybean (Glycine max L. Merr).</h2>
			<h3>PMID:21175888  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Indrasumunar A(1), Searle I(1), Lin MH(1), Kereszt A(1), Men A(1), Carroll BJ(1), Gresshoff PM(1).
				<br/> 
				年份：2011 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two allelic non-nodulating mutants, nod49 and rj1, were characterized using map-based cloning and candidate gene approaches, and genetic complementation. From our results we propose two highly related lipo-oligochitin LysM-type receptor kinase genes <span class="gene">(GmNFR1α</span> and <span class="gene">GmNFR1β)</span> as putative Nod factor receptor components in soybean. Both mutants contained frameshift mutations in <span class="gene">GmNFR1α</span> that would <span class="trait">yield</span> protein truncations. Both mutants contained a seemingly functional <span class="gene">GmNFR1β</span> homeologue, characterized by a 374-bp deletion in intron 6 and 20-100 times lower transcript levels than <span class="gene">GmNFR1α,</span> yet both mutants were unable to form nodules. Mutations in <span class="gene">GmNFR1β</span> within other genotypes had no defects in nodulation, showing that <span class="gene">GmNFR1β</span> was redundant. Transgenic overexpression of <span class="gene">GmNFR1α,</span> but not of <span class="gene">GmNFR1β,</span> increased nodule number per plant, plant nitrogen content and the ability to form nodules with restrictive, ultra-low Bradyrhizobium japonicum titres in transgenic roots of both nod49 and rj1. <span class="gene">GmNFR1α</span> overexpressing roots also formed nodules in nodulation-restrictive acid soil (pH 4.7). Our results show that: (i) NFR1α expression controls nodule number in soybean, and (ii) acid soil tolerance for nodulation and suppression of nodulation deficiency at low titre can be achieved by overexpression of <span class="gene">GmNFR1α.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of QTL for increased fibrous roots in soybean.</h2>
			<h3>PMID:21165732  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abdel-Haleem H(1), Lee GJ, Boerma RH.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress adversely affects soybean at various developmental stages, which collectively results in <span class="trait">yield</span> reduction. Unpredictable rainfall has been reported to contribute about 36% to variation of <span class="trait">yield</span> difference between the rain-fed and irrigated fields. Among the <span class="trait">drought</span> <span class="trait">resistance</span> mechanisms, <span class="trait">drought</span> avoidance in genotypes with fibrous roots was recognized to be associated with <span class="trait">drought</span> <span class="trait">resistance</span> in soybean. Plant introduction PI416937 was shown to possess fibrous roots and has been used as a parent in breeding programs to improve soybean productivity. Little information is available on relative contribution and chromosomal location of quantitative trait loci (QTL) conditioning fibrous roots in soybean. To identify the genomic locations and genetic bases of this trait, a recombinant inbred line population was derived from a cross between PI416937 and 'Benning'. To detect associated QTLs, phenotypic data were collected and analyzed for 2 years under rain-fed field conditions. The selective genotyping approach was used to reduce the costs and work associated with conducting the QTL analysis. A total of five QTLs were identified on chromosomes <span class="gene">Gm01</span> (Satt383), <span class="gene">Gm03</span> (Satt339), <span class="gene">Gm04</span> (Sct_191), <span class="gene">Gm08</span> (Satt429), and <span class="gene">Gm20</span> (Sat_299), and together explained 51% of the variation in root score. Detected QTLs were co-localized with QTLs related to root morphology, suggesting that fibrous roots QTL may be associated with other morpho-physiological traits and seed <span class="trait">yield</span> in soybean. Genetic dissection of the fibrous roots trait at the individual marker loci will allow for marker-assisted selection to develop soybean genotypes with enhanced levels of fibrous roots.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine genetic mapping of the genomic region controlling leaflet shape and number of seeds per pod in the soybean.</h2>
			<h3>PMID:21104397  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Jeong N(1), Moon JK, Kim HS, Kim CG, Jeong SC.
				<br/> 
				年份：2011 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Narrow leaflet cultivars tend to have more seeds per pod than broad leaflet cultivars in soybean [Glycine max (L.) Merr.], which suggests that the leaflet-shape trait locus is tightly linked to or cosegregates with the trait locus controlling the number of seeds per pod (NSPP). Here, we attempted to further elucidate the relationship between leaflet shape and NSPP. A BC(3)F(2) population from a cross between the 'Sowon' (narrow leaflets and high NSPP) and 'V94-5152' (broad leaflets and low NSPP) variants was used. The results of the molecular genetic analyses indicated that, although the NSPP characteristic, in particular, the occurrence of 4-seeded pods, is governed by additional modifying genes that are likely present in Sowon, the two traits cosegregate in the BC(3)F(2) population. The mapping results generated using public markers demonstrated that the narrow leaflet-determining gene in Sowon is an allele of the previously highly studied <span class="gene">ln</span> gene on chromosome 20. A high-resolution map delimited the genomic region controlling both the leaflet shape and NSPP traits to a sequence length of 66 kb, corresponding to 0.7 cM. Among the three genes annotated in this 66 kb region, <span class="gene">Glyma20g25000.1</span> appeared to be a good candidate for the Ln-encoding gene, owing to its 47.8% homology with the protein encoding for the JAGGED gene that regulates lateral organ development in Arabidopsis. Taken together, our results suggested that phenotypic variations for narrow leaflet and NSPP are predominantly from the pleiotropic effects of the <span class="gene">ln</span> gene. Thus, our results should provide a molecular framework for soybean breeding programs with the objective of improving soybean <span class="trait">yield.</span>
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Glyphosate affects micro-organisms in rhizospheres of glyphosate-resistant soybeans.</h2>
			<h3>PMID:20880215  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zobiole LH(1), Kremer RJ, Oliveira RS Jr, Constantin J.
				<br/> 
				年份：2011 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						AIMS: Glyphosate-resistant (GR) soybean production increases each year because of the efficacy of glyphosate for weed management. A new or 'second' generation of <span class="gene">GR</span> soybean (GR2) is now commercially available for farmers that is being promoted as higher yielding relative to the previous, 'first generation' (GR1) cultivars. Recent reports show that glyphosate affects the biology and ecology of rhizosphere micro-organisms in <span class="gene">GR</span> soybean that affect <span class="trait">yield.</span> The objective of this research was to evaluate the microbiological interactions in the rhizospheres of GR2 and GR1 soybean and the performance of the cultivars with different rates of glyphosate applied at different growth stages. METHODS AND RESULTS: A greenhouse study was conducted using GR1 and GR2 soybean cultivars grown in a silt loam soil. Glyphosate was applied at V2, V4 and V6 growth stages at three rates. Plants harvested at R1 growth stage had high root colonization by Fusarium spp.; reduced rhizosphere fluorescent pseudomonads, Mn-reducing bacteria, and indoleacetic acid-producing rhizobacteria; and reduced shoot and root biomass. CONCLUSIONS: Glyphosate applied to <span class="gene">GR</span> soybean, regardless of cultivar, negatively impacts the complex interactions of microbial groups, biochemical activity and root growth that can have subsequent detrimental effects on plant growth and productivity. SIGNIFICANCE AND IMPACT OF THE STUDY: The information presented here will be crucial in developing strategies to overcome the potential detrimental effects of glyphosate in <span class="gene">GR</span> cropping systems.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Natural variability in abundance of prevalent soybean proteins.</h2>
			<h3>PMID:20709130  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Natarajan SS(1).
				<br/> 
				年份：2010 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is an inexpensive source of protein for humans and animals. Genetic modifications <span class="gene">(GMO)</span> to soybean have become inevitable on two fronts, both quality and <span class="trait">yield</span> will need to improve to meet increasing global demand. To ensure the safety of the crop for consumers it is important to determine the natural variation in seed protein constituents as well as any unintended changes that may occur in the <span class="gene">GMO</span> as a result of genetic modification. Understanding the natural variation of seed proteins in wild and cultivated soybeans that have been used in conventional soybean breeding programs is critical for determining unintended protein expression in <span class="gene">GMO</span> soybeans. In recent years, proteomic technologies have been used as an effective analytical tool for examining modifications of protein profiles. We have standardized and applied these technologies to determine and quantify the spectrum of proteins present in soybean seed. We used two-dimensional polyacrylamide gel electrophoresis (2D-PAGE), matrix-assisted laser desorption/ionization time of flight mass spectrometry (MALDI-TOF-MS), and liquid chromatography mass spectrometry (LC-MS) for the separation, quantification, and identification of different classes of soybean seed proteins. We have observed significant variations in different classes of proteins, including storage, allergen and anti-nutritional protein profiles, between non-GMO cultivated and wild soybean varieties. This information is useful for scientists and regulatory agencies to determine whether the unintended expression of proteins found in transgenic soybean is within the range of natural variation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean peroxidase propeptides are functional signal peptides and increase the <span class="trait">yield</span> of a foreign protein.</h2>
			<h3>PMID:20535473  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Schnell JA(1), Han S, Miki BL, Johnson DA.
				<br/> 
				年份：2010 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Elements that contribute to the high, stable <span class="trait">yield</span> of soybean peroxidase (SBP) in soybean seed coats can be exploited in the development of this tissue as a protein production platform. <span class="gene">SBP</span> contains an N-terminal and a C-terminal propeptide that are predicted to direct vacuolar targeting; this may be one factor that contributes to its high <span class="trait">yield</span> and stability. We characterized the function of the <span class="gene">SBP</span> propeptides and investigated their ability to increase the <span class="trait">yield</span> of a foreign protein in a heterologous plant system. <span class="gene">SBP</span> propeptides are functional signal peptides capable of directing vacuolar transport in Arabidopsis. The use of these propeptides as well as an endoplasmic reticulum (ER)-retention signal to direct a foreign protein to the apoplast, ER, or vacuole can significantly increase <span class="trait">yield</span> and will therefore be useful for the development of the seed coat as a protein production platform. We also demonstrate that growth conditions may have a significant impact on the <span class="trait">yield</span> of a foreign protein and that this may be subcellular compartment-specific.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Mapping QTLs for seed <span class="trait">yield</span> and <span class="trait">drought</span> susceptibility index in soybean (Glycine max L.) across different environments.</h2>
			<h3>PMID:20129399  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Du W(1), Wang M, Fu S, Yu D.
				<br/> 
				年份：2009 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> stress has long been a major constraint in maintaining <span class="trait">yield</span> stability of soybean (Glycine max (L.) Merr.) in rainfed ecosystems. The identification of consistent quantitative trait loci (QTL) involving seed <span class="trait">yield</span> per plant (YP) and <span class="trait">drought</span> susceptibility index (DSI) in a population across different environments would therefore be important in molecular marker-assisted breeding of soybean cultivars suitable for rainfed regions. The YP of a recombinant line population of 184 F(2:7:11) lines from a cross of Kefeng1 and Nannong1138-2 was studied under water-stressed (WS) and well-watered (WW) conditions in field (F) and greenhouse (G) trials, and DSI for <span class="trait">yield</span> was calculated in two trials. Nineteen QTLs associated with YP-WS and YP-WW, and 10 QTLs associated with DSI, were identified. Comparison of these QTL locations with previous findings showed that the majority of these regions control one or more traits related to <span class="trait">yield</span> and other agronomic traits. One QTL on molecular linkage group (MLG) K for YP-F, and two QTLs on MLG <span class="gene">C2</span> for YP-G, remained constant across different water regimes. The regions on MLG <span class="gene">C2</span> for YP-WW-F and MLG H for YP-WS-F had a pleiotropic effect on DSI-F, and MLG A1 for YP-WS-G had a pleiotropic effect on DSI-G. The identification of consistent QTLs for YP and DSI across different environments will significantly improve the efficiency of selecting for <span class="trait">drought</span> tolerance in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						2009 Institute of Genetics and Developmental Biology and the Genetics Society of China. Published by Elsevier Ltd. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of flowering-time genes in soybean <span class="gene">E1</span> near-isogenic lines under short and long day conditions.</h2>
			<h3>PMID:20091337  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Thakare D(1), Kumudini S, Dinkins RD.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Control of soybean flowering time is important for geographic adaptation and maximizing <span class="trait">yield.</span> Plant breeders have identified a series of genes (E genes) that condition time to flowering; however, the molecular basis in the control of flowering by these E genes, in conjunction with canonical flowering-time genes, has not been studied. Time to flowering in near-isogenic lines (NILs) at the <span class="gene">E1</span> locus was tested using a reciprocal transfer experiment under short day (SD) and long day (LD) conditions. Beginning 8 days after planting, three plant samples were harvested every 3 h for a 48-h period. RNA was isolated from these plants, and RNA samples were pooled for each line and each time period for cDNA synthesis. RT-PCR analysis was performed using primers synthesized for a number of putative flowering-time genes based on homology of soybean EST and genomic sequences to Arabidopsis genes. The results of the reciprocal transfer experiment suggest that the pre-inductive photoperiod-sensitive phase of the <span class="gene">E1</span> NILs responsible for inducing flowering is perceived as early as 5-7-day post-planting. No gene expression differences were found between the <span class="gene">E1</span> and <span class="gene">e1</span> NILs, suggesting that the <span class="gene">E1</span> gene does not directly affect the flowering-time genes during the time period tested; however, differences were observed in gene expression between SD and LD treatments for the putative soybean TOC1, CO, and FT genes. The gene expression results in this study were similar to those of flowering-time genes found in other SD species, suggesting that the selected genes correspond to the soybean flowering-time orthologs.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression quantitative trait loci analysis of two genes encoding rubisco activase in soybean.</h2>
			<h3>PMID:20032079  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yin Z(1), Meng F, Song H, Wang X, Xu X, Yu D.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Rubisco activase (RCA) catalyzes the activation of Rubisco in vivo and plays a crucial role in photosynthesis. However, until now, little was known about the molecular genetics of RCA in soybean (Glycine max), one of the most important legume crops. Here, we cloned and characterized two genes encoding the longer alpha -isoform and the shorter beta -isoform of soybean RCA <span class="gene">(GmRCA</span> alpha and <span class="gene">GmRCA</span> beta, respectively). The two corresponding cDNAs are divergent in both the translated and 3 ' untranslated regions. Analysis of genomic DNA sequences suggested that the corresponding mRNAs are transcripts of two different genes and not the products of a single alternatively splicing pre-mRNA. Two additional possible alpha -form RCA-encoding genes, <span class="gene">GmRCA03</span> and <span class="gene">GmRCA14,</span> and one additional beta -form RCA-encoding gene, <span class="gene">GmRCA11,</span> were also isolated. To examine the function and modulation of RCA genes in soybean, we determined the expression levels of <span class="gene">GmRCA</span> alpha and <span class="gene">GmRCA</span> beta, Rubisco initial activity, photosynthetic rate, and seed <span class="trait">yield</span> in 184 soybean recombinant inbred lines. Correlation of gene expression levels with three other traits indicates that RCA genes could play an important role in regulating soybean photosynthetic capacity and seed <span class="trait">yield.</span> Expression quantitative trait loci mapping revealed four trans-expression quantitative trait loci for <span class="gene">GmRCA</span> alpha and <span class="gene">GmRCA</span> beta. These results could provide a new approach for the modulation of RCA genes to improve photosynthetic rate and plant growth in soybean and other plants.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulation of seed <span class="trait">yield</span> and agronomic characters by photoperiod sensitivity and growth habit genes in soybean.</h2>
			<h3>PMID:20012856  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Cober ER(1), Morrison MJ.
				<br/> 
				年份：2010 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean genotypes are adapted to narrow bands of latitude due to photoperiod sensitivity. There are several photoperiod-sensitive loci (E1, E2, E3, E4, E5, E6, E7, E8). Determinate and indeterminate growth habits are controlled by a single locus. The objective of our research was to examine the effects of photoperiod sensitivity and growth habit alleles on seed <span class="trait">yield</span> and other agronomic characters using isogenic lines. Twenty 'Harosoy' isolines with 11 photoperiod-sensitive genotypes many with both indeterminate and determinate growth habits were grown in the field at Ottawa, ON, from 2003 to 2007. Maturity ranged from 97 to 127 days, and seed <span class="trait">yield</span> increased linearly with maturity until about 112 days when it plateaued. Determinate lines were always shorter than indeterminate lines of equivalent maturity. Seed <span class="trait">yield</span> was associated with plant height, maturity, seed sugar concentration, seed weight and <span class="trait">lodging.</span> Effects of alleles at individual loci, and additive and epistatic effects across multiple loci were examined. At a single locus, photoperiod-insensitive alleles produced isolines that matured 8-11 days earlier, yielded less, and had shorter plants with reduced <span class="trait">lodging.</span> In multiple loci analyses, additive effects explained most of the variation in agronomic characters since additive models with E1, E3, E4, E7 and <span class="gene">Dt1</span> loci included compared well to additive plus epistatic models and genotype-based models. Variation in photoperiod sensitivity and growth habit alleles results in a range of maturity, with pleiotropic effects on seed <span class="trait">yield</span> and agronomic characteristics, and play an important role in providing adaptation across latitudes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Natural and synthetic 2'-hydroxy-chalcones and aurones: synthesis, characterization and evaluation of the antioxidant and soybean lipoxygenase inhibitory activity.</h2>
			<h3>PMID:19853459  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Detsi A(1), Majdalani M, Kontogiorgis CA, Hadjipavlou-Litina D, Kefalas P.
				<br/> 
				年份：2009 Dec 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A series of 2'-hydroxy-chalcones and their oxidative cyclization products, aurones, have been synthesized and tested for their antioxidant and lipoxygenase inhibitory activity. The natural product aureusidin (31) was synthesized in high <span class="trait">yield</span> by a new approach. An extensive structure-relationship study was performed and revealed that several chalcones and aurones possess an appealing pharmacological profile combining high antioxidant and lipid peroxidation activity with potent soybean <span class="gene">LOX</span> inhibition.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Detection of quantitative trait loci for <span class="trait">yield</span> and <span class="trait">drought</span> tolerance traits in soybean using a recombinant inbred line population.</h2>
			<h3>PMID:19723246  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Du W(1), Yu D, Fu S.
				<br/> 
				年份：2009 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						To investigate the genetic basis of <span class="trait">drought</span> tolerance in soybean (Glycine max L. Merr.) a recombinant inbred population with 184 F(2:7:11) lines developed from a cross between Kefeng1 <span class="trait">(drought</span> tolerant) and Nannong1138-2 <span class="trait">(drought</span> sensitive) were tested under water-stressed and well-watered conditions in field and greenhouse trials. Traits measured included leaf wilting coefficient, excised leaf water loss and relative water content as indicators of plant water status and seed <span class="trait">yield.</span> A total of 40 quantitative trait loci (QTLs) were identified: 17 for leaf water status traits under <span class="trait">drought</span> stress and 23 for seed <span class="trait">yield</span> under well-watered and drought-stressed conditions in both field and greenhouse trials. Two seed <span class="trait">yield</span> QTLs were detected under both well-watered and drought-stressed conditions in the field on molecular linkage group H and D1b, while two seed <span class="trait">yield</span> QTLs on molecular linkage group <span class="gene">C2</span> were found under greenhouse conditions. Several QTLs for traits associated with plant water status were identified in both field and greenhouse trials, including two leaf wilting coefficient QTLs on molecular linkage group A2 and one excised leaf water loss QTL on molecular linkage group H. Phenotypic correlations of traits suggested several QTLs had pleiotropic or location-linked associations. These results will help to elucidate the genetic basis of <span class="trait">drought</span> tolerance in soybean, and could be incorporated into a marker-assisted selection breeding program to develop high-yielding soybean cultivars with improved tolerance to <span class="trait">drought</span> stress.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Generation of <italic> Phaseolus vulgaris </italic> ESTs and investigation of their regulation upon <italic> Uromyces appendiculatus </italic> infection</h2>
			<h3>PMID:19397807</h3>
				

				
					<h3>PMCID:2684537</h3>
					<h4>期刊:BMC Plant Biology</h4>
				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> Thibivilliers </surname>  <given-names> Sandra </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> st5y7@mizzou.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> joshitr@missouri.edu </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Campbell </surname>  <given-names> Kimberly B </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> campbelk@ba.ars.usda.go </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Scheffler </surname>  <given-names> Brian </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> bscheffler@msa-stoneville.ars.usda.gov </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> xudong@missouri.edu </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Cooper </surname>  <given-names> Bret </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> cooperb@ba.ars.usda.gov </email>  </contrib>  <contrib id="A7" contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> nguyenhenry@missouri.edu </email>  </contrib>  <contrib id="A8" corresp="yes" contrib-type="author">  <name>  <surname> Stacey </surname>  <given-names> Gary </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> staceyg@missouri.edu </email>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 4 </month>  <year> 2009 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Phaseolus vulgaris (common bean) is the second most important legume crop in the world after soybean. Consequently, <span class="trait">yield</span> losses due to fungal infection, like Uromyces appendiculatus (bean <span class="trait">rust),</span> have strong consequences. Several resistant genes were identified that confer <span class="trait">resistance</span> to bean <span class="trait">rust</span> infection. However, the downstream genes and mechanisms involved in bean <span class="trait">resistance</span> to infection are poorly characterized. RESULTS: A subtractive bean cDNA library composed of 10,581 unisequences was constructed and enriched in sequences regulated by either bean <span class="trait">rust</span> race 41, a virulent strain, or race 49, an avirulent strain on cultivar Early Gallatin carrying the <span class="trait">resistance</span> gene Ur-4. The construction of this library allowed the identification of 6,202 new bean ESTs, significantly adding to the available sequences for this plant. Regulation of selected bean genes in response to bean <span class="trait">rust</span> infection was confirmed by qRT-PCR. Plant gene expression was similar for both race 41 and 49 during the first 48 hours of the infection process but varied significantly at the later time points (72-96 hours after inoculation) mainly due to the presence of the <span class="gene">Avr4</span> gene in the race 49 leading to a hypersensitive response in the bean plants. A biphasic pattern of gene expression was observed for several genes regulated in response to fungal infection. CONCLUSION: The enrichment of the public database with over 6,000 bean ESTs significantly adds to the genomic resources available for this important crop plant. The analysis of these genes in response to bean <span class="trait">rust</span> infection provides a foundation for further studies of the mechanism of fungal disease <span class="trait">resistance.</span> The expression pattern of 90 bean genes upon <span class="trait">rust</span> infection shares several features with other legumes infected by biotrophic fungi. This finding suggests that the P. vulgaris-U. appendiculatus pathosystem could serve as a model to explore legume-rust interaction.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A subtractive bean cDNA library composed of 10,581 unisequences was constructed and enriched in sequences regulated by either bean <span class="trait">rust</span> race 41, a virulent strain, or race 49, an avirulent strain on cultivar Early Gallatin carrying the <span class="trait">resistance</span> gene <italic> Ur-4 </italic> . The construction of this library allowed the identification of 6,202 new bean ESTs, significantly adding to the available sequences for this plant. Regulation of selected bean genes in response to bean <span class="trait">rust</span> infection was confirmed by qRT-PCR. Plant gene expression was similar for both race 41 and 49 during the first 48 hours of the infection process but varied significantly at the later time points (72&#x02013;96 hours after inoculation) mainly due to the presence of the <italic> <span class="gene">Avr4</span> </italic> gene in the race 49 leading to a hypersensitive response in the bean plants. A biphasic pattern of gene expression was observed for several genes regulated in response to fungal infection.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Two <span class="gene">MATE</span> proteins play a role in iron efficiency in soybean.</h2>
			<h3>PMID:19342121  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Rogers EE(1), Wu X, Stacey G, Nguyen HT.
				<br/> 
				年份：2009 Sep 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Iron is a necessary but often limiting nutrient for plant growth and development. Soybeans grown on the high-pH calcareous soils are especially prone to developing iron deficiency <span class="trait">chlorosis</span> and suffering the resultant <span class="trait">yield</span> losses. Once iron is transported into the root, it must be translocated from the root to the shoot where it is needed for photosynthesis and other processes. Previous work has indicated that iron is likely to move through the xylem as ferric citrate. In Arabidopsis thaliana, citrate is effluxed into the xylem by the ferric reductase defective3 (FRD3) protein. Here, we present the identification and characterization of two soybean genes, <span class="gene">GmFRD3a</span> and <span class="gene">GmFRD3b,</span> with similar sequence and function to AtFRD3. The expression of both <span class="gene">GmFRD3a</span> and <span class="gene">GmFRD3b</span> is induced by iron deficiency in the iron-efficient reference cultivar Williams 82. <span class="gene">GmFRD3b,</span> but not <span class="gene">GmFRD3a,</span> is expressed at higher levels in the iron-efficient cultivar Clark than in its iron-inefficient near isogenic line iso-Clark (iso), likely accounting for the higher xylem citrate levels in Clark. Increased xylem citrate levels lead to increased solubility of ferric iron in Clark xylem exudate as compared to iso-Clark exudate. These results support the hypothesis that high xylem citrate levels are needed for efficient root to shoot translocation of iron. Along with efficient ferric chelate reductase activity and root iron uptake activity, high expression levels of FRD3 genes are also proposed as a target for future iron efficiency breeding projects.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Allantoate amidohydrolase transcript expression is independent of <span class="trait">drought</span> tolerance in soybean.</h2>
			<h3>PMID:19129162  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Charlson DV(1), Korth KL, Purcell LC.
				<br/> 
				年份：2009
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Drought</span> is a limiting factor for N(2) fixation in soybean [Glycine max (L.) Merr.] thereby resulting in reduced biomass accumulation and <span class="trait">yield.</span> Drought-sensitive genotypes accumulate ureides, a product of N(2) fixation, during <span class="trait">drought</span> stress; however, drought-tolerant genotypes have lower shoot ureide concentrations, which appear to alleviate <span class="trait">drought</span> stress on N(2) fixation. A key enzyme involved in ureide breakdown in shoots is allantoate amidohydrolase (AAH). It is hypothesized that <span class="gene">AAH</span> gene expression in soybean determines shoot ureide concentrations during water-deficit stress and is responsible for the differential sensitivities of the N(2)-fixation response to <span class="trait">drought</span> among soybean genotypes. The objectives were to examine the relationship between <span class="gene">AAH</span> transcript levels and shoot ureide concentration and <span class="trait">drought</span> tolerance. Drought-tolerant (Jackson) and drought-sensitive (Williams) genotypes were subjected to three water-availability treatments: well-watered control, moderate water-deficit stress, and severe water-deficit stress. Shoot ureide concentrations were examined, in addition to gene expression of <span class="gene">AAH</span> and DREB2, a gene expressed during water-deficit stress. As expected, <span class="gene">DREB2</span> expression was detected only during severe water-deficit stress, and shoot ureide concentrations were greatest in the drought-sensitive genotype relative to the drought-tolerant genotype during water-deficit stress. However, expression of <span class="gene">AAH</span> transcripts was similar among water treatments and genotypes, indicating that <span class="gene">AAH</span> mRNA was not closely associated with <span class="trait">drought</span> tolerance. Ureide concentrations in shoots were weakly associated with <span class="gene">AAH</span> mRNA levels. These results indicate that <span class="gene">AAH</span> expression is probably not associated with the increased ureide catabolism observed in drought-tolerant genotypes, such as Jackson. Further study of <span class="gene">AAH</span> at the post-translational and enzymatic levels is warranted in order to dissect the potential role of this gene in <span class="trait">drought</span> tolerance.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparison of broiler performance when fed diets containing event DP-3O5423-1, nontransgenic near-isoline control, or commercial reference soybean meal, hulls, and oil.</h2>
			<h3>PMID:19038811  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：McNaughton J(1), Roberts M, Smith B, Rice D, Hinds M, Sanders C, Layton R, Lamb I, Delaney B.
				<br/> 
				年份：2008 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DP-3Ø5423-1 (305423) is a genetically modified soybean that was produced by biolistic insertion of the gm-fad2-1 gene fragment and gm-hra genes into the germline of soybean seeds. Expression of gm-fad2-1 results in greater concentrations of oleic acid (18:1) by suppressing expression of the endogenous <span class="gene">FAD2-1</span> gene, which encodes an n-6 fatty acid desaturase enzyme that catalyzes desaturation of 18:1 to linoleic acid (18:2). The <span class="gene">GM-HRA</span> protein expressed by the gm-hra gene is a modified version of the soybean acetolactate synthase enzyme that is used as a selectable marker during transformation. A 42-d feeding trial was conducted with broiler chickens to compare the nutritional performance of 305423 soybeans with nontransgenic soybeans. Diets were prepared using processed fractions (meal, hulls, and oil) from 305423 soybean plants. For comparison, additional diets were produced with soybean fractions obtained from a nontransgenic near-isoline (control) and nontransgenic commercial Pioneer brand varieties (93B86, 93B15, and 93M40). Diets were fed to Ross x Cobb broilers (n = 120/group, 50% male and 50% female) in 3 phases. Starter, grower, and finisher diets contained 26.5, 23, and 21.5% soybean meal, respectively. Soybean hulls and oil were added at 1.0 and 0.5%, respectively, across all diets in each phase. No statistically significant differences were observed in growth performance (BW, mortality, feed efficiency), organ <span class="trait">yield</span> (liver and kidney), or carcass <span class="trait">yield</span> (breast, thigh, leg, wing, and abdominal fat) variables between broilers consuming diets prepared with isolated fractions from 305423 or near-isoline control soybean. Additionally, all performance and carcass variables from control and 305423 soybean treatment groups fell within tolerance intervals constructed for each response variable using data from broilers fed diets prepared with reference soybean fractions. Based on the results from this study, it was concluded that 305423 soybeans were nutritionally equivalent to non-transgenic control soybeans with a comparable genetic background.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic structure of populations of Rhizoctonia solani anastomosis group-1 IA from soybean in Brazil.</h2>
			<h3>PMID:18943212  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ciampi MB(1), Meyer MC, Costa MJ, Zala M, McDonald BA, Ceresini PC.
				<br/> 
				年份：2008 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The Basidiomycete fungus Rhizoctonia solani anastomosis group (AG)-1 IA is a major pathogen of soybean in Brazil, where the average <span class="trait">yield</span> losses have reached 30 to 60% in some states in Northern Brazil. No information is currently available concerning levels of genetic diversity and population structure for this pathogen in Brazil. A total of 232 isolates of R. solani <span class="gene">AG1</span> IA were collected from five soybean fields in the most important soybean production areas in central-western, northern, and northeastern Brazil. These isolates were genotyped using 10 microsatellite loci. Most of the multilocus genotypes (MLGTs) were site-specific, with few MLGTs shared among populations. Significant population subdivision was evident. High levels of admixture were observed for populations from Mato Grosso and Tocantins. After removing admixed genotypes, three out of five field populations (Maranhao, Mato Grosso, and Tocantins), were in Hardy-Weinberg (HW) equilibrium, consistent with sexual recombination. HW and gametic disequilibrium were found for the remaining soybean-infecting populations. The findings of low genotypic diversity, departures from HW equilibrium, gametic disequilibrium, and high degree of population subdivision in these R. solani AG-1 IA populations from Brazil are consistent with predominantly asexual reproduction, short-distance dispersal of vegetative propagules (mycelium or sclerotia), and limited long-distance dispersal, possibly via contaminated seed. None of the soybean-infecting populations showed a reduction in population size (bottleneck effect). We detected asymmetric historical migration among the soybean-infecting populations, which could explain the observed levels of subdivision.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Diversity Among Isolates of Bean pod mottle virus.</h2>
			<h3>PMID:18942958</h3>
				

				
			<p>
				作者：Gu H, Clark AJ, de Sá PB, Pfeiffer TW, Tolin S, Ghabrial SA.
				<br/> 
				年份：2002 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						ABSTRACT Isolates of Bean pod mottle virus (BPMV), a member of the genus Comovirus, collected from soybean fields in Kentucky, Virginia, Arkansas, and Iowa were classified into two distinct subgroups, I and II, based on nucleic acid hybridization analysis using cloned cDNA probes to RNA-1 from BPMV strains K-G7 and K-Ha1. Slot blot hybridization analysis using cloned cDNA probes to RNA-2 from the same two strains (K-G7 and K-Ha1), however, revealed that some of the isolates, initially classified as belonging to subgroup I after analysis with RNA-1 probes, are in fact natural reassortants between the two strain subgroups. This was corroborated by nucleotide sequence analysis of full-length cDNA clones of both RNA-1 and RNA-2 from a putative reassortant strain (K-Ho1). These results indicate that BPMV strain diversity is more complex than initially anticipated, and that the use of cloned probes to both genomic RNAs during nucleic acid hybridization analysis is required to unravel the extent of such diversity. In a field plot experiment, BPMV isolates that belong to distinct strain subgroups induced symptoms that varied in severity and in the level of <span class="trait">yield</span> losses. In this regard, the reassortant strain K-Ho1 caused the most serious damage compared with four other BPMV isolates tested. Furthermore, the soybean alleles Rsv(1) and Rsv(4), known to confer <span class="trait">resistance</span> against Soybean mosaic virus, a member of the genus Potyvirus, did not provide any protection against BPMV. Additionally, we developed a reverse transcription-polymerase chain reaction procedure based on the sequence of a highly conserved region in the capsid <span class="gene">polyprotein</span> coding sequence that provides efficient and highly sensitive detection of all BPMV isolates tested, regardless of their strain classification.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RAPD and mitochondrial DNA analysis of the soybean stalk weevil, Sternechus subsignatus (Coleoptera: Curculionidae).</h2>
			<h3>PMID:18826663  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Sosa-Gomez DR(1), Coronel N, Binneck E, Zucchi MI, Rosado-Neto G.
				<br/> 
				年份：2008 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Sternechus subsignatus Boheman (Curculionidae: Sternechini) is one of the primary Curculionidae species that reduces soybean <span class="trait">yield</span> in Brazil. Initially, outbreaks were reported in southern Brazil in 1973; but, more recent, outbreaks were reported in Bahia (summer 1997-1998) and Maranhão (summer 2003-2004), two states in northeastern Brazil. A putative related species, S. pinguis (Fabricius), was first detected in Salta Province, Argentina. The objective of this study was to evaluate intraspecific molecular polymorphisms of geographically distinct Sternechus populations. Randomly amplified polymorphic DNA (RAPD) profiles and partial mitochondrial cytochrome B (CytB) gene sequences were used to determine whether individual soybean stalk weevils were one of two different species and to infer pest invasion pattern. Putative S. pinguis and S. subsignatus populations were collected in San Agustin (Cruz Alta, Tucumán Province, Argentina) and different sampling sites in the Brazilian states of Paraná, Bahia and Maranhão. Polymorphic bands were obtained by RAPD and analyzed by Dice coefficients. Populations from southern Brazil were more closely related genetically to an Argentinean group than the populations sampled in northeastern Brazil. The Londrina Co., Brazil population displayed the highest intra-population genetic similarity. Most of the soybean stalk weevils collected from San Agustin, Tucumán, Argentina were divergent from those collected in Brazil. Sequencing and parsimony analysis of <span class="gene">CytB</span> did not differentiate specimens collected in Argentina and Brazil. Thus, our data show that soybean stalk weevil outbreaks and population increases in northeastern Brazil involved local genotypes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The critical period of weed control in soybean (Glycine max (L.) Merr.) in north of Iran conditions.</h2>
			<h3>PMID:18817174  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Keramati S(1), Pirdashti H, Esmaili MA, Abbasian A, Habibi M.
				<br/> 
				年份：2008 Feb 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A field study was conducted in 2006 at Sari Agricultural and Natural Resources University, in order to determine the best time for weed control in soybean promising line, 033. Experiment was arranged in randomized complete block design with 4 replications and two series of treatments. In the first series, weeds were kept in place until crop reached <span class="gene">V2</span> (second trifoliolate), V4 (fourth trifoliolate), V6 (sixth trifoliolate), R1 (beginning bloom, first flower), R3 (beginning pod), R5 (beginning seed) and were then removed and the crop kept weed-free for the rest of the season. In the second series, crops were kept weed-free until the above growth stages after which weeds were allowed to grow in the plots for the rest of the season. Whole season weedy and weed-free plots were included in the experiment for <span class="trait">yield</span> comparison. The results showed that among studied traits, grain <span class="trait">yield,</span> pod numbers per plant and weed biomass were affected significantly by control and interference treatments. The highest number of pods per plant was obtained from plots which kept weed-free for whole season control. Results showed that weed control should be carried out between <span class="gene">V2</span> (26 day after planting) to R1 (63 day after planting) stages of soybean to provide maximum grain <span class="trait">yield.</span> Thus, it is possible to optimize the timing of weed control, which can serve to reduce the costs and side effects of intensive chemical weed control.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed <span class="trait">yield</span> of near-isogenic soybean lines with introgressed quantitative trait loci conditioning <span class="trait">resistance</span> to corn earworm (Lepidoptera: Noctuidae) and soybean looper (Lepidoptera: Noctuidae) from PI 229358.</h2>
			<h3>PMID:18767762  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Warrington CV(1), Zhu S, Parrott WA, All JN, Boerma HR.
				<br/> 
				年份：2008 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The development of superior soybean, Glycine max (L.) Merr., cultivars exhibiting <span class="trait">resistance</span> to insects has been hindered due to linkage drag, a common phenomenon when introgressing alleles from exotic germplasm. Simple-sequence repeat (SSR) markers were used previously to map soybean insect <span class="trait">resistance</span> (SIR) quantitative trait loci (QTLs) in a'Cobb' X PI 229358 population, and subsequently used to create near-isogenic lines (NILs) with <span class="gene">SIR</span> QTL i n a 'Benning' genetic background. <span class="gene">SIR</span> QTLs were mapped on linkage groups (LGs) M (SIRQTL-M), G (SIRQTL-G), and H (SIRQTL-H). The objectives of this study were to 1) evaluate linkage drag for seed <span class="trait">yield</span> by using Benning-derived NILs selected for SIRQTL-M, SIRQTL-H, and SIRQTL-G; 2) assess the amount of PI 229358 genome surrounding the <span class="gene">SIR</span> QTL in each Benning NIL; and 3) evaluate the individual effects these three QTLs on antibiosis and antixenosis to corn earworm, Helicoverpa zea (Boddie), and soybean looper, Pseudoplusia includens (Walker). <span class="trait">Yield</span> data collected in five environments indicated that a significant <span class="trait">yield</span> reduction is associated with SIRQTL-G compared with NILs without <span class="gene">SIR</span> QTL. Overall, there was no <span class="trait">yield</span> reduction associated with SIRQTL-M or SIRQTL-H. A significant antixenosis and antibiosis effect was detected for SIRQTL-M in insect feeding assays, with no effect detected in antixenosis or antibiosis assays for SIRQTL-G or SIRQTL-H without the presence of PI 229358 alleles at SIRQTL-M. These results support recent findings concerning these loci.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Analysis of clock gene homologs using unifoliolates as target organs in soybean (Glycine max).</h2>
			<h3>PMID:18707796  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu H(1), Wang H, Gao P, Xü J, Xü T, Wang J, Wang B, Lin C, Fu YF.
				<br/> 
				年份：2009 Feb 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is a typical short-day crop, and its photoperiodic response of flowering is critical to its <span class="trait">yield.</span> This phenomenon was well studied during the last century, but the molecular mechanism governing it is unknown. The clock-gene homologs, <span class="gene">GmLCL1</span> and <span class="gene">GmLCL2</span> (Glycine max LHY/CCA1 Like 1 and 2) and <span class="gene">GmTOC1</span> (Glycine max TOC1) were cloned from Glycine max L. KN18, and their expression patterns were analyzed using a system developed in this study. We employed 8h light/16h dark and 18h light/6h dark as short- and long-day conditions, respectively, because in these conditions soybean plants had significant indexes of flowering time. We also used unifoliolates, not the whole plant as Arabidopsis, as target organs for gene expression analysis. <span class="gene">GmLCL1</span> and <span class="gene">GmLCL2</span> had similar circadian expression patterns and both were morning genes, while <span class="gene">GmTOC1</span> was an evening gene and peaked in the evening. The expressions of <span class="gene">GmLCL1</span> and <span class="gene">GmLCL2</span> were obviously antiphase to that of <span class="gene">GmTOC1.Our</span> study provided a system that simplified the experiments without compromising the quality of data obtained and was suitable for analyzing the molecular mechanism of flowering in soybean. <span class="gene">GmLCL1,</span> <span class="gene">GmLCL2</span> and <span class="gene">GmTOC1</span> may be the components of central clock in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Optimization of medium composition for actinomycin X2 production by Streptomyces <span class="gene">spp</span> JAU4234 using response surface methodology.</h2>
			<h3>PMID:18347826  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Xiong ZQ(1), Tu XR, Tu GQ.
				<br/> 
				年份：2008 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effects of cultivation medium compositions including soybean meal, peptone, soybean oil and cornstarch for actinomycin X2 production by Streptomyces <span class="gene">spp</span> JAU4234 were accessed by using response surface methodology. The 2(4) full factorial designs and the paths of steepest ascent were effective in searching for the major factors of actinomycin X2 production. In this study, cornstarch and soybean oil showed negative effect on actinomycin X2 production based on the first-order regression coefficients derived from MINITAB software. Subsequently, a central composite design for optimization was further investigated. Preliminary studies showed that soybean meal and peptone were believed to be the major factors for actinomycin X2 production. Estimated optimum compositions for the production of actionmycin X2 were as follows (g/l): soybean meal 21.65 and peptone 9.41, and result in a maximum actionmycin X2 production of 617.4 mg/l. This value was closed to the 612 mg/l actionmycin X2 production from actual experimental observations. The <span class="trait">yield</span> of actionmycin X2 was increased by 36.9% by culturing the strain Streptomyces <span class="gene">spp</span> JAU4234 in the nutritionally optimized fermentation medium.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A comparative study of free and immobilized soybean and horseradish peroxidases for 4-chlorophenol removal: protective effects of immobilization.</h2>
			<h3>PMID:18270748  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Bódalo A(1), Bastida J, Máximo MF, Montiel MC, Gómez M, Murcia MD.
				<br/> 
				年份：2008 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Horseradish peroxidase (HRP) and soybean peroxidase (SBP) were covalently immobilized onto aldehyde glass through their amine groups. The activity <span class="trait">yield</span> and the protein content for the immobilized <span class="gene">SBP</span> were higher than for the immobilized HRP. When free and immobilized peroxidases were tested for their ability to remove 4-chlorophenol from aqueous solutions, the removal percentages were higher with immobilized HRP than with free HRP, whereas immobilized <span class="gene">SBP</span> needs more enzyme to reach the same conversion than free enzyme. In the present paper the two immobilized derivatives are compared. It was found that at an immobilized enzyme concentration in the reactor of 15 mg l(-1), <span class="gene">SBP</span> removed 5% more of 4-chlorophenol than HRP, and that a shorter treatment was necessary. Since immobilized <span class="gene">SBP</span> was less susceptible to inactivation than HRP and provided higher 4-chlorophenol elimination, this derivative was chosen for further inactivation studies. The protective effect of the immobilization against the enzyme inactivation by hydrogen peroxide was demonstrated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Advances in development of transgenic pulse crops.</h2>
			<h3>PMID:18055156  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Eapen S(1).
				<br/> 
				年份：2008 Mar-Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						It is three decades since the first transgenic pulse crop has been developed. Todate, genetic transformation has been reported in all the major pulse crops like Vigna species, Cicer arietinum, Cajanus cajan, Phaseolus spp, Lupinus spp, Vicia <span class="gene">spp</span> and Pisum sativum, but transgenic pulse crops have not yet been commercially released. Despite the crucial role played by pulse crops in tropical agriculture, transgenic pulse crops have not moved out from laboratories to large farm lands compared to their counterparts - 'cereals' and the closely related leguminous oil crop - 'soybean'. The reason for lack of commercialization of transgenic pulse crops can be attributed to the difficulty in developing transgenics with reproducibility, which in turn is due to lack of competent totipotent cells for transformation, long periods required for developing transgenics and lack of coordinated research efforts by the scientific community and long term funding. With optimization of various factors which influence genetic transformation of pulse crops, it will be possible to develop transgenic plants in this important group of crop species with more precision and reproducibility. A translation of knowledge from information available in genomics and functional genomics in model legumes like Medicago truncatula and Lotus japonicus relating to factors which contribute to enhancing crop <span class="trait">yield</span> and ameliorate the negative consequences of biotic and abiotic stress factors may provide novel insights for genetic manipulation to improve the productivity of pulse crops.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:2-Hydroxyisoflavanone dehydratase is a critical determinant of isoflavone productivity in hairy root cultures of Lotus japonicus.</h2>
			<h3>PMID:17921150  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Shimamura M(1), Akashi T, Sakurai N, Suzuki H, Saito K, Shibata D, Ayabe S, Aoki T.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Hairy root cultures of a model legume, Lotus japonicus, were established to characterize two heterologous cDNAs encoding enzymes involved in isoflavone biosynthesis, i.e. licorice 2-hydroxyisoflavanone synthase (IFS) and soybean 2-hydroxyisoflavanone dehydratase (HID) catalyzing sequential reactions to <span class="trait">yield</span> isoflavones. While the control and the <span class="gene">IFS</span> overexpressor did not accumulate detectable isoflavones, the HID overexpressors did accumulate daidzein and genistein, showing that HID is a critical determinant of isoflavone productivity. Production of coumestrol in all the genotypes and isoliquiritigenin/liquiritigenin in <span class="gene">IFS</span> + HID-overexpressing lines was also noted. These results provide insight into the regulatory mechanism that controls isoflavonoid biosynthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Generation and characterization of two novel low phytate mutations in soybean (Glycine max L. Merr.).</h2>
			<h3>PMID:17701395  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Yuan FJ(1), Zhao HJ, Ren XL, Zhu SL, Fu XJ, Shu QY.
				<br/> 
				年份：2007 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytic acid (PA, myo-inositol 1, 2, 3, 4, 5, 6 hexakisphosphate) is important to the nutritional quality of soybean meal. Organic phosphorus (P) in PA is indigestible in humans and non-ruminant animals, which affects nutrition and causes P pollution of ground water from animal wastes. Two novel soybean [(Glycine max L. (Merr.)] low phytic acid (lpa) mutations were isolated and characterized. <span class="gene">Gm-lpa-TW-1</span> had a phytic acid P (PA-P) reduction of 66.6% and a sixfold increase in inorganic P (Pi), and <span class="gene">Gm-lpa-ZC-2</span> had a PA-P reduction of 46.3% and a 1.4-fold increase in Pi, compared with their respective non-mutant progenitor lines. The reduction of PA-P and increase of Pi in <span class="gene">Gm-lpa-TW-1</span> were molar equivalent; the decrease of PA-P in <span class="gene">Gm-lpa-ZC-2,</span> however, was accompanied by the increase of both Pi and lower inositol phosphates. In both mutant lines, the total P content remained similar to their wild type parents. The two lpa mutations were both inherited in a single recessive gene model but were non-allelic. Sequence data and progeny analysis indicate that <span class="gene">Gm-lpa-TW-1</span> lpa mutation resulted from a 2 bp deletion in the soybean D: -myo-inositol 3-phosphate synthase (MIPS1 EC 5.5.1.4) gene 1 (MIPS1). The lpa mutation in <span class="gene">Gm-lpa-ZC-2</span> was mapped on LG B2, closely linked with microsatellite loci Satt416 and Satt168, at genetic distances of approximately 4.63 and approximately 9.25 cM, respectively. Thus this mutation probably represents a novel soybean lpa locus. The seed emergence rate of <span class="gene">Gm-lpa-ZC-2</span> was similar to its progenitor line and was not affected by seed source and its lpa mutation. However, <span class="gene">Gm-lpa-TW-1</span> had a significantly reduced field emergence when seeds were produced in a subtropic environment. Field tests of the mutants and their progenies further demonstrated that the lpa mutation in <span class="gene">Gm-lpa-ZC-2</span> does not negatively affect plant <span class="trait">yield</span> traits. These results will advance understanding of the genetic, biochemical and molecular control of PA synthesis in soybean. The novel lpa mutation in <span class="gene">Gm-lpa-ZC-2,</span> together with linked simple sequence repeat (SSR) markers, will be of value for breeding productive lpa soybeans, with meal high in digestible Pi eventually to improve animal nutrition and lessen environmental pollution.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Bacillus subtilis fusion protein system to produce soybean Bowman-Birk protease inhibitor.</h2>
			<h3>PMID:17574434  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Vogtentanz G(1), Collier KD, Bodo M, Chang JH, Day AG, Estell DA, Falcon BC, Ganshaw G, Jarnagin AS, Kellis JT Jr, Kolkman MA, Lai CS, Meneses R, Miller JV, de Nobel H, Power S, Weyler W, Wong DL, Schmidt BF.
				<br/> 
				年份：2007 Sep
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A fusion protein based expression system was developed in the Gram-positive bacterium Bacillus subtilis to produce the soybean Bowman-Birk protease inhibitor (sBBI). The N-terminus of the mature sBBI was fused to the C-terminus of the 1st cellulose binding domain linker (CBD linker) of the BCE103 cellulase (from an alkalophilic Bacillus sp.). The strong <span class="gene">aprE</span> promoter was used to drive the transcription of the fusion gene and the AprE signal sequence was fused to the mature BCE103 cellulase for efficient secretion of the fusion protein into the culture medium. It was necessary to use a B. subtilis strain deficient in nine protease genes in order to reduce the proteolytic degradation of the fusion protein during growth. The fusion protein was produced in shake flasks at concentrations &gt;1g/L. After growth, the sBBI was activated by treatment with 2-mercaptoethanol to allow the disulfide bonds to form correctly. An economical and scalable purification process was developed to purify sBBI based on acid precipitation of the fusion protein followed by acid/heat cleavage of the fusion protein at labile Asp-Pro bonds in the CBD linker. If necessary, non-native amino acids at the N- and C-termini were trimmed off using glutamyl endopeptidase I. After purification, an average of 72 mg of active sBBI were obtained from 1L of culture broth representing an overall <span class="trait">yield</span> of 21% based on the amount of sBBI activated before purification.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean disease <span class="trait">resistance</span> protein RHG1-LRR domain expressed, purified and refolded from Escherichia coli inclusion bodies: preparation for a functional analysis.</h2>
			<h3>PMID:17287130  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Afzal AJ(1), Lightfoot DA.
				<br/> 
				年份：2007 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Introduction and expression of foreign genes in bacteria often results accumulation of the foreign protein(s) in inclusion bodies (IBs). The subsequent processes of refolding are slow, difficult and often fail to <span class="trait">yield</span> significant amounts of folded protein. <span class="gene">RHG1</span> encoded by rhg1 was a soybean (Glycine max L. Merr.) transmembrane receptor-like kinase (EC 2.7.11.1) with an extracellular leucine-rich repeat domain. The LRR of <span class="gene">RHG1</span> was believed to be involved in elicitor recognition and interaction with other plant proteins. The aim, here, was to express the LRR domain in Escherichia coli (RHG1-LRR) and produce refolded protein. Urea titration experiments showed that the IBs formed in E. coli by the extracellular domain of the <span class="gene">RHG1</span> protein could be solubilized at different urea concentrations. The <span class="gene">RHG1</span> proteins were eluted with 1.0-7.0M urea in 0.5M increments. Purified <span class="gene">RHG1</span> protein obtained from the 1.5 and 7.0M elutions was analyzed for secondary structure through circular dichroism (CD) spectroscopy. Considerable secondary structure could be seen in the former, whereas the latter yielded CD curves characteristic of denatured proteins. Both elutions were subjected to refolding by slowly removing urea in the presence of arginine and reduced/oxidized glutathione. Detectable amounts of refolded protein could not be recovered from the 7.0M urea sample, whereas refolding from the 1.5M urea sample yielded 0.2mg/ml protein. The 7.0M treatment resulted in the formation of a homogenous denatured state with no apparent secondary structure. Refolding from this fully denatured state may confer kinetic and/or thermodynamic constraints on the refolding process, whereas the kinetic and/or thermodynamic barriers to attain the folded conformation appeared to be lesser, when refolding from a partially folded state.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:GMOs: building the future on the basis of past experience.</h2>
			<h3>PMID:17143405  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Reis LF(1), Van Sluys MA, Garratt RC, Pereira HM, Teixeira MM.
				<br/> 
				年份：2006 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Biosafety of genetically modified organisms <span class="gene">(GMOs)</span> and their derivatives is still a major topic in the agenda of government and societies worldwide. The aim of this review is to bring into light that data that supported the decision taken back in 1998 as an exercise to stimulate criticism from the scientific community for upcoming discussions and to avoid emotional and senseless arguments that could jeopardize future development in the field. It must be emphasized that Roundup Ready soybean is just one example of how biotechnology can bring in significant advances for society, not only through increased productivity, but also with beneficial environmental impact, thereby allowing more rational use of agricultural pesticides for improvement of the soil conditions. The adoption of agricultural practices with higher <span class="trait">yield</span> will also allow better distribution of income among small farmers. New species of genetically modified plants will soon be available and society should be capable of making decisions in an objective and well-informed manner, through collegiate bodies that are qualified in all aspects of biosafety and environmental impact.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Production of soybean isoflavone genistein in non-legume plants via genetically modified secondary metabolism pathway.</h2>
			<h3>PMID:17029902  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Liu R(1), Hu Y, Li J, Lin Z.
				<br/> 
				年份：2007 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Genetic modification of secondary metabolic pathways to produce desirable natural products is an attractive approach in plant biotechnology. In our study, we attempted to produce a typical soybean isoflavone genistein, a well-known health-promoting metabolite, in non-legume plants via genetic engineering. Both overexpression and antisense suppression strategies were used to manipulate the expression of several genes encoding key enzymes in the flavonoids/isoflavonoids pathway in transgenic tobacco, lettuce, and petunia. Introducing soybean isoflavone synthase (IFS) into these plants, which naturally do not produce isoflavonoids due to a lack of this leguminous enzyme, resulted in genistein biosynthesis in tobacco petals, petunia leaves and petals, and lettuce leaves. In tobacco, when flavanone 3-hydroxylase (F3H) expression was suppressed by its antisense gene while soybean <span class="gene">IFS</span> was overexpressed at the same time, genistein <span class="trait">yield</span> increased prominently. In addition, overexpression of phenylalanine ammonia-lyase (PAL) also led to an enhanced genistein production in tobacco petals and lettuce leaves in the presence of <span class="gene">IFS</span> than in the plants that overexpressed only IFS.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Critical points of DNA quantification by real-time PCR &#x02013; effects of DNA extraction method and sample matrix on quantification of genetically modified organisms</h2>
			<h3>PMID:16907967</h3>
				

				
					<h3>PMCID:1569826</h3>
					<h4>期刊:BMC Biotechnology</h4>
				
			<p>
				作者：<contrib id="A1" corresp="yes" contrib-type="author">  <name>  <surname> Cankar </surname>  <given-names> Katarina </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> katja.cankar@nib.si </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> &#x00160;tebih </surname>  <given-names> Dejan </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> dejan.stebih@nib.si </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Dreo </surname>  <given-names> Tanja </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> tanja.dreo@nib.si </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> &#x0017d;el </surname>  <given-names> Jana </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> jana.zel@nib.si </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Gruden </surname>  <given-names> Kristina </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> kristina.gruden@nib.si </email>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 8 </month>  <year> 2006 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						BACKGROUND: Real-time PCR is the technique of choice for nucleic acid quantification. In the field of detection of genetically modified organisms <span class="gene">(GMOs)</span> quantification of biotech products may be required to fulfil legislative requirements. However, successful quantification depends crucially on the quality of the sample DNA analyzed. Methods for <span class="gene">GMO</span> detection are generally validated on certified reference materials that are in the form of powdered grain material, while detection in routine laboratories must be performed on a wide variety of sample matrixes. Due to food processing, the DNA in sample matrixes can be present in low amounts and also degraded. In addition, molecules of plant origin or from other sources that affect PCR amplification of samples will influence the reliability of the quantification. Further, the wide variety of sample matrixes presents a challenge for detection laboratories. The extraction method must ensure high <span class="trait">yield</span> and quality of the DNA obtained and must be carefully selected, since even components of DNA extraction solutions can influence PCR reactions. <span class="gene">GMO</span> quantification is based on a standard curve, therefore similarity of PCR efficiency for the sample and standard reference material is a prerequisite for exact quantification. Little information on the performance of real-time PCR on samples of different matrixes is available. RESULTS: Five commonly used DNA extraction techniques were compared and their suitability for quantitative analysis was assessed. The effect of sample matrix on nucleic acid quantification was assessed by comparing 4 maize and 4 soybean matrixes. In addition 205 maize and soybean samples from routine analysis were analyzed for PCR efficiency to assess variability of PCR performance within each sample matrix. Together with the amount of DNA needed for reliable quantification, PCR efficiency is the crucial parameter determining the reliability of quantitative results, therefore it was chosen as the primary criterion by which to evaluate the quality and performance on different matrixes and extraction techniques. The effect of PCR efficiency on the resulting <span class="gene">GMO</span> content is demonstrated. CONCLUSION: The crucial influence of extraction technique and sample matrix properties on the results of <span class="gene">GMO</span> quantification is demonstrated. Appropriate extraction techniques for each matrix need to be determined to achieve accurate DNA quantification. Nevertheless, as it is shown that in the area of food and feed testing matrix with certain specificities is impossible to define strict quality controls need to be introduced to monitor PCR. The results of our study are also applicable to other fields of quantitative testing by real-time PCR.
					</p>
				
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Real-time PCR is the technique of choice for nucleic acid quantification. In the field of detection of genetically modified organisms <span class="gene">(GMOs)</span> quantification of biotech products may be required to fulfil legislative requirements. However, successful quantification depends crucially on the quality of the sample DNA analyzed. Methods for <span class="gene">GMO</span> detection are generally validated on certified reference materials that are in the form of powdered grain material, while detection in routine laboratories must be performed on a wide variety of sample matrixes. Due to food processing, the DNA in sample matrixes can be present in low amounts and also degraded. In addition, molecules of plant origin or from other sources that affect PCR amplification of samples will influence the reliability of the quantification. Further, the wide variety of sample matrixes presents a challenge for detection laboratories. The extraction method must ensure high <span class="trait">yield</span> and quality of the DNA obtained and must be carefully selected, since even components of DNA extraction solutions can influence PCR reactions. <span class="gene">GMO</span> quantification is based on a standard curve, therefore similarity of PCR efficiency for the sample and standard reference material is a prerequisite for exact quantification. Little information on the performance of real-time PCR on samples of different matrixes is available.
					</p>
				
					<p>&nbsp;&nbsp;
						The method of choice for gene quantification is real-time PCR. This technique proves to be more suitable for the diagnostic laboratory than conventional PCR, due to its quantitative performance, greater sensitivity and the use of closed-tube assays. Its use in quantitative analysis of genetically modified organisms has been reviewed [ <xref ref-type="bibr" rid="B2"> 2 </xref> - <xref ref-type="bibr" rid="B4"> 4 </xref> ]. The most frequent quantitative application of real-time PCR is in gene expression studies. In this case the interest is in the order of changes in expression, and the limit of detection has been reported as being a two fold difference [ <xref ref-type="bibr" rid="B5"> 5 </xref> ]. However, greater accuracy is required in <span class="gene">GMO</span> diagnostics, therefore it is necessary to be fully aware of the factors influencing quantification since inaccurate analyses can result in liability issues. The choice of DNA extraction procedure can crucially influence the quantifiability of <span class="gene">GMOs,</span> but the choice of extraction method is often a trade-off between costs, optimal <span class="trait">yield</span> of DNA and removal of substances that could influence the PCR reaction. A procedure that results in an optimal <span class="trait">yield</span> of DNA and in removal of substances that could influence the PCR reaction is essential. DNA extraction techniques have been compared [ <xref ref-type="bibr" rid="B6"> 6 </xref> - <xref ref-type="bibr" rid="B8"> 8 </xref> ]. Methods for <span class="gene">GMO</span> analysis are normally validated on only one sample type, typically powdered grain material, or on a very limited range of sample matrixes [ <xref ref-type="bibr" rid="B4"> 4 </xref> , <xref ref-type="bibr" rid="B9"> 9 </xref> ]. In reality, <span class="gene">GMO</span> laboratories are faced with a broad spectrum of different foodstuffs, from raw plant materials to highly processed food products.
					</p>
				
			

			<hr/> 
		

			
			<h2>文献标题:Chip-based polyketide biosynthesis and functionalization.</h2>
			<h3>PMID:16889386  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ku B(1), Cha J, Srinivasan A, Kwon SJ, Jeong JC, Sherman DH, Dordick JS.
				<br/> 
				年份：2006 Jul-Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						We demonstrate construction and novel compound synthesis from a synthetic metabolic pathway consisting of a type III polyketide synthase (PKS) known as 1,3,6,8-tetrahydroxynaphthalene synthase (THNS) from Streptomyces coelicolor and soybean peroxidase (SBP) in a microfluidic platform. THNS immobilized to Ni-NTA agarose beads is prepacked into a microfluidic channel, while <span class="gene">SBP</span> is covalently attached to the walls of a second microfluidic channel precoated with a reactive poly(maleic anhydride) derivative. The result is a tandem, two-step biochip that enables the synthesis of novel polyketide derivatives. The first microchannel, consisting of THNS, results in the conversion of malonyl-CoA to flaviolin in yields up to 40% with a residence time of 6 min. This conversion is similar to that obtained in several-milliliter batch reactions after 2 h. Linking this microchannel to the <span class="gene">SBP</span> microchannel results in biflaviolin synthesis. During the course of this work, we discovered that the substrate specificity of THNS could be manipulated by simply changing the reaction pH. As a result, the starter acyl-CoA specificity can be broadened to <span class="trait">yield</span> a series of truncated pyrone products. When combined with variations in the ratio of acyl-CoA and malonyl-CoA (extender substrate) feed rates, high yields of the pyrone products could be achieved, which is further structurally diversified from self- and cross-coupling in the <span class="gene">SBP</span> microchannel. The ability to rapidly evaluate the effects of reaction conditions and synthetic multienzyme pathways on a microfludic platform provides a new paradigm for performing metabolic pathway engineering, namely, the reconstruction of pathways for use in new compound discovery.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of palm kernel meal and corn distillers grains in corn silage-based diets for lactating dairy cows.</h2>
			<h3>PMID:16772589  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Carvalho LP(1), Cabrita AR, Dewhurst RJ, Vicente TE, Lopes ZM, Fonseca AJ.
				<br/> 
				年份：2006 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The effects of increasing levels of solvent-extracted palm kernel meal (SPKM) and corn distillers dried grains (CDG) in corn silage-based diets on feed intake and milk production were examined in 2 experiments. In Experiment 1, 20 Holstein cows averaging 100 d in milk (DIM) (SD = 61.5) at the start of the experiment were utilized in an 11-wk randomized complete block design with 4 treatments in 5 blocks to study effects of increasing levels of SPKM in the diet. During a 3-wk preliminary period, cows were fed a standard diet. At the end of the preliminary period, cows were blocked by 4% fat-corrected milk <span class="trait">yield,</span> parity number (primiparous and multiparous), and DIM, and were assigned randomly to 1 of 4 experimental diets. The total mixed ration (TMR) consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of SPKM were achieved by replacing protein sources and citrus pulp with SPKM and urea (0, 5, 10, and 15% SPKM and 0.06, 0.22, 0.38, and 0.55% urea for SPKM0, SPKM5, SPKM10, and SPKM15, respectively). In Experiment 2, 18 Holstein cows averaging 93 <span class="gene">DIM</span> (SD = 49.1) at the start of the experiment were utilized in an 11-wk randomized complete block design with 3 treatments in 6 blocks to study effects of increasing levels of CDG in the diet. The preliminary period lasted for 2 wk. Assignment of cows to treatments was the same as in Experiment 1. The TMR consisted of (dry matter basis) 40% corn silage, 5% coarsely chopped wheat straw, and 55% concentrate. The increasing dietary levels of CDG were achieved by replacing soybean meal and citrus pulp with CDG and urea (0, 7, and 14% CDG and 0, 0.22, and 0.49% urea for CDG0, CDG7, and CDG14, respectively). There were no significant treatment effects on dry matter intake, milk <span class="trait">yield,</span> or milk composition in Experiment 1. Inclusion of SPKM tended to increase protein and lactose contents of milk. The SPKM0 diet promoted body weight loss. There were no treatment effects in Experiment 2, except for milk protein content, which was decreased by CDG. Plasma Lys concentration tended to be affected by SPKM and CDG inclusions. Plasma concentrations of 3-methylhistidine and Leu seemed to be related to body protein degradation/synthesis. The feeding of SPKM up to 15% in the diet decreased feed costs without detrimental effects on productive responses and tended to increase milk protein content. The inclusion of CDG in diets based on corn silage and corn byproducts might decrease milk protein content due to an unbalanced supply of AA, particularly Lys.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Determining the optimal timing of foliar insecticide applications for control of soybean aphid (Hemiptera: Aphididae) on soybean.</h2>
			<h3>PMID:16539126  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Myers SW(1), Hogg DB, Wedberg JL.
				<br/> 
				年份：2005 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Field experiments were performed over 3 yr to examine the impact of insecticide application timing to control soybean aphid, Aphis glycines Matsumura (Hemiptera: Aphididae), populations and to prevent soybean <span class="trait">yield</span> losses. Experiments were conducted in early and late-planted soybean, Glycine max (L.) Merr. Insecticide applications were made based on soybean growth stages. In 2001, applications were made at V1, V3, R2, and R3 growth stages; in 2002 and 2003, applications were made at R2, R3, and R4 stages. Additional treatments consisted of an unsprayed control and a multiple spray treatment that received insecticide applications at 7-10-d intervals. Soybean aphid densities were recorded throughout the growing season, and yields were measured. Soybean aphid populations varied considerably across years and planting dates. In general, late-planted soybean exhibited higher aphid pressure than early planted soybean, and experiments in 2002 had lower aphid numbers than those in 2001 and 2003. The multiple spray treatment significantly increased <span class="trait">yield</span> over the control in four of the six experiments, the exceptions being 2002 late planted and 2003 early planted. This suggests that soybean aphid populations were not large enough to cause <span class="trait">yield</span> losses in these two experiments. The R3 spray treatment increased <span class="trait">yield</span> in three of the six experiments (2001 late planting, 2002 early planting, and 2003 late planting), the R2 spray treatment increased <span class="trait">yield</span> in two of six experiments (2001 and 2003 late plantings), and the <span class="gene">V1</span> application increased <span class="trait">yield</span> over the control in the 2001 late-planted experiment. Results suggest that when aphid populations are high insecticide applications made at R2 and R3 plant stages are most effective in preventing <span class="trait">yield</span> loss.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Global food insecurity. treatment of major food crops with elevated carbon dioxide or ozone under large-scale fully open-air conditions suggests recent models may have overestimated future yields.</h2>
			<h3>PMID:16433090  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Long SP(1), Ainsworth EA, Leakey AD, Morgan PB.
				<br/> 
				年份：2005 Nov 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Predictions of <span class="trait">yield</span> for the globe's major grain and legume arable crops suggest that, with a moderate temperature increase, production may increase in the temperate zone, but decline in the tropics. In total, global food supply may show little change. This security comes from inclusion of the direct effect of rising carbon dioxide (CO2) concentration, [CO2], which significantly stimulates <span class="trait">yield</span> by decreasing photorespiration in <span class="gene">C3</span> crops and transpiration in all crops. Evidence for a large response to [CO2] is largely based on studies made within chambers at small scales, which would be considered unacceptable for standard agronomic trials of new cultivars or agrochemicals. Yet, predictions of the globe's future food security are based on such inadequate information. Free-Air Concentration Enrichment (FACE) technology now allows investigation of the effects of rising [CO2] and ozone on field crops under fully open-air conditions at an agronomic scale. Experiments with rice, wheat, maize and soybean show smaller increases in <span class="trait">yield</span> than anticipated from studies in chambers. Experiments with increased ozone show large <span class="trait">yield</span> losses (20%), which are not accounted for in projections of global food security. These findings suggest that current projections of global food security are overoptimistic. The fertilization effect of CO2 is less than that used in many models, while rising ozone will cause large <span class="trait">yield</span> losses in the Northern Hemisphere. Unfortunately, FACE studies have been limited in geographical extent and interactive effects of CO2, ozone and temperature have yet to be studied. Without more extensive study of the effects of these changes at an agronomic scale in the open air, our ever-more sophisticated models will continue to have feet of clay.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Relationships of intercropped maize, stem borer damage to maize <span class="trait">yield</span> and land-use efficiency in the humid forest of Cameroon.</h2>
			<h3>PMID:16197562  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chabi-Olaye A(1), Nolte C, Schulthess F, Borgemeister C.
				<br/> 
				年份：2005 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Stem borers are the most important maize pests in the humid forest zone of Cameroon. Field trials were conducted in the long and short rainy seasons of 2002 and 2003 to assess the level of damage and <span class="trait">yield</span> reductions caused by stem borers in monocropped maize and in maize intercropped with non-host plants such as cassava, cowpea and soybean. The intercrops were planted in two spatial arrangements, i.e. alternating hills or alternating rows. All intercrops and the maize monocrop were grown with and without insecticide treatment for assessment of maize <span class="trait">yield</span> loss due to borer attacks. The land-use efficiency of each mixed cropping system was evaluated by comparing it with the monocrop. The temporal fluctuation of larval infestations followed the same pattern in all cropping systems, but at the early stage of plant growth, larval densities were 21.3-48.1% higher in the monocrops than in intercrops, and they tended to be higher in alternating rows than alternating hills arrangements. At harvest, however, pest densities did not significantly vary between treatments. Maize monocrops had 3.0-8.8 times more stems tunnelled and 1.3-3.1 times more <span class="gene">cob</span> damage than intercrops. Each percentage increase in stem tunnelling lowered maize grain <span class="trait">yield</span> by 1.10 and 1.84 g per plant, respectively, during the long and short rainy season in 2002, and by 5.39 and 1.41 g per plant, respectively, in 2003. Maize <span class="trait">yield</span> losses due to stem borer were 1.8-3.0 times higher in monocrops than in intercrops. Intercrops had generally a higher land-use efficiency than monocrops, as indicated by land-equivalent-ratios and area-time-equivalent-ratios of &gt;1.0. Land-use efficiency was similar in both spatial arrangements. At current price levels, the net production of mixed cropping systems was economically superior to controlling stem borers with insecticide in monocropped maize. The maize-cassava intercrop yielded the highest land equivalent ratios and the highest replacement value of the intercrop. At medium intensity cropping this system is thus recommended for land-constrained poor farmers who do not use external inputs such as fertilizer and insecticides.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Hydrophilic gels containing chlorophyllin-loaded liposomes: development and stability evaluation.</h2>
			<h3>PMID:16124401  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Dragićević-Curić N(1), Stupar M, Milić J, Zorić T, Krajisnik D, Vasiljević D.
				<br/> 
				年份：2005 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The aim of this study was to develop and characterize hydrophilic gels containing chlorophyllin(CHL)-loaded liposomes as well as to evaluate their stability. Two different CHL-loaded liposome dispersions using non-hydrogenated and hydrogenated soybean lecithin were prepared, characterized for their particle size, polydispersity index and trapping efficiency and incorporated in Carbopol 940 NF hydrogel. The gels obtained were analyzed for flow properties, pH values and <span class="gene">CHL</span> content. The control liposome-free gel was obtained by incorporating the <span class="gene">CHL</span> solution in the hydrogel. The stability of the gels was evaluated in terms of rheological properties, pH values and <span class="gene">CHL</span> content during 6 months' storage at 20 +/- 2 degrees C. Suitable gel formulations for topical use were obtained revealing shear-thinning plastic flow behaviour without significant thixotropy during the whole period of examination. High <span class="trait">yield</span> values of the samples during the whole period indicated a long-term stability of the gel formulations. The gel formulations expressed a mild acid value acceptable for topical preparations. After 6 months' storage the <span class="gene">CHL</span> content was highest in the gel containing non-hydrogenated lecithin liposomes, followed by the gel containing hydrogenated lecithin liposomes and liposome-free gel, indicating that the encapsulation of <span class="gene">CHL</span> in liposomes led to a greater stability of CHL.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Technical performance of some commercial glyphosate-resistant crops.</h2>
			<h3>PMID:15662723  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Pline-Srnic W(1).
				<br/> 
				年份：2005 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glyphosate-resistant (GR) crops have been sold commercially in the USA since 1996. The use of glyphosate alone or with conventional pre- and post-emergence herbicides with different modes of action gives growers many options for affordable, safe, easy, effective wide-spectrum weed control. Despite the overwhelming popularity of this technology, technical issues have surfaced from time to time as US growers adopt these crops for use on their farms. The types of concern raised by growers vary from year to year depending on the crop and the environment, but include perceptions of increased sensitivity to diseases, increased fruit abortion, reduced pollination efficiency, increased sensitivity to environmental stress, and differences in <span class="trait">yield</span> and agronomic characteristics between transgenic and sister conventional varieties. Although several glyphosate-resistant crops are commercially available, maize, soybean and cotton constitute the largest cultivated acreage and have likewise been associated with the highest number of technical concerns. Because glyphosate is rapidly translocated to and accumulates in metabolic sink tissues, reproductive tissues and roots are particularly vulnerable. Increased sensitivity to glyphosate in reproductive tissues has been documented in both glyphosate-resistant cotton and maize, and results in reduced pollen production and viability, or increased fruit abortion. Glyphosate treatments have the potential to affect relationships between the <span class="gene">GR</span> crop, plant pathogens, plant pests and symbiotic micro-organisms, although management practices can also have a large impact. Despite these potential technical concerns, this technology remains popular, and is a highly useful tool for weed control in modern crop production.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 2005 Society of Chemical Industry
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:High-pressure refolding of bikunin: efficacy and thermodynamics.</h2>
			<h3>PMID:15388859  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Seefeldt MB(1), Ouyang J, Froland WA, Carpenter JF, Randolph TW.
				<br/> 
				年份：2004 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Bikunin is a glycosylated protein that aggregates extensively during mammalian cell culture, resulting in loss of activity, loss of native secondary structure, and the formation of nonnative disulfide bonds. We investigated the use of high hydrostatic pressure (1000-3000 bar) for the refolding of bikunin aggregates. The refolding <span class="trait">yield</span> obtained with pressure-modulated refolding at 2000 <span class="gene">bar</span> was 70 (+/-5%) by reverse-phase chromatography (RP-HPLC), significantly higher than the value of 55 (+/-6%) (RP-HPLC) obtained with traditional guanidine HCl "dilution-refolding." In addition, we determined the thermodynamics of pressure-modulated refolding. The change in volume for the transition of aggregate to monomer DeltaV(refolding) was calculated to be -28 (+/-5) mL/mole. Refolding was accompanied by a loss of hydrophobic exposure, resulting in a positive contribution to the DeltaV(refolding). These findings suggest that the disruption of electro-static interactions or the differences in size of solvent-free cavities between the aggregate and the monomer are the prevailing contributions to the negative DeltaV(refolding).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Impact of glyphosate on the Bradyrhizobium japonicum symbiosis with glyphosate-resistant transgenic soybean: a minireview.</h2>
			<h3>PMID:15224916  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zablotowicz RM(1), Reddy KN.
				<br/> 
				年份：2004 May-Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Glyphosate-resistant (GR) soybean [Glycine max (L.) Merr.] expressing an insensitive 5-enolpyruvylshikimic acid-3-phosphate synthase (EPSPS) gene has revolutionized weed control in soybean production. The soybean nitrogen fixing symbiont, Bradyrhizobium japonicum, possesses a glyphosate-sensitive enzyme and upon exposure to glyphosate accumulates shikimic acid and hydroxybenzoic acids such as protocatechuic acid (PCA), accompanied with B. japonicum growth inhibition and death at high concentrations. In a series of greenhouse and field experiments, glyphosate inhibited nodulation and nodule leghemoglobin content of <span class="gene">GR</span> soybean. Glyphosate accumulated in nodules of field-grown <span class="gene">GR</span> soybean, but its effect on nitrogenase activity of <span class="gene">GR</span> soybean was inconsistent in field studies. In greenhouse studies, nitrogenase activity of <span class="gene">GR</span> soybean following glyphosate application was transiently inhibited especially in early growth stages, with the greatest inhibition occurring under moisture stress. Studies using bacteroid preparations showed that the level of glyphosate inhibition of bacteroid nitrogenase activity was related to in vitro glyphosate sensitivity of the B. japonicum strains. These studies indicate the potential for reduced nitrogen fixation in the <span class="gene">GR</span> soybean system; however, <span class="trait">yield</span> reductions due to this reduced N2 fixation in early stages of growth have not been demonstrated.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:QTL mapping of ten agronomic traits on the soybean ( Glycine max L. Merr.) genetic map and their association with EST markers.</h2>
			<h3>PMID:15067400  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Zhang WK(1), Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY.
				<br/> 
				年份：2004 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A set of 184 recombinant inbred lines (RILs) derived from soybean vars. Kefeng No.1 x Nannong 1138-2 was used to construct a genetic linkage map. The two parents exhibit contrasting characteristics for most of the traits that were mapped. Using restricted fragment length polymorphisms (RFLPs), simple sequence repeats (SSRs) and expressed sequence tags (ESTs), we mapped 452 markers onto 21 linkage groups and covered 3595.9 cM of the soybean genome. All of the linkage groups except linkage group F were consistent with those of the consensus map of Cregan et al. (1999). Linkage group F was divided into two linkage groups, F1 and F2. The map consisted of 189 RFLPs, 219 SSRs, 40 ESTs, three R gene loci and one phenotype marker. Ten agronomic traits-days to flowering, days to maturity, plant height, number of nodes on main stem, <span class="trait">lodging,</span> number of pods per node, protein content, oil content, 100-seed weight, and plot yield-were studied. Using winqtlcart, we detected 63 quantitative trait loci (QTLs) that had LOD&gt;3 for nine of the agronomic traits (only exception being seed oil content) and mapped these on 12 linkage groups. Most of the QTLs were clustered, especially on groups B1 and C2. Some QTLs were mapped to the same loci. This pleiotropism was common for most of the QTLs, and one QTL could influence at most five traits. Seven EST markers were found to be linked closely with or located at the same loci as the QTLs. EST marker <span class="gene">GmKF059a,</span> encoding a repressor protein and mapped on group C2, accounted for about 20% of the total variation of days to flowering, plant height, <span class="trait">lodging</span> and nodes on the main stem, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Introduction of DPR, an enterostatin fragment peptide, into soybean beta-conglycinin <span class="gene">alpha'</span> subunit by site-directed mutagenesis.</h2>
			<h3>PMID:14745196  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Takenaka Y(1), Doyama N, Maruyama N, Utsumi S, Yoshikawa M.
				<br/> 
				年份：2004 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						DPR, a fragment peptide of enterostatin (VPDPR) having hypocholesterolemic activity, was introduced into the three homologous sites, EPR, DYR, and DPI, in the soybean beta-conglycinin <span class="gene">alpha'</span> subunit by site-directed mutagenesis. The modified beta-conglycinin was expressed in Escherichia coli and recovered in the soluble fraction. After purification on ion-exchange HPLC, the modified beta-conglycinin was digested by trypsin to release integrated DPR. The <span class="trait">yield</span> of DPR from 1 mole of the modified beta-conglycinin was 1.2 mole.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Purification, properties and substrate specificity of a digestive trypsin from Periplaneta americana (Dictyoptera) adults.</h2>
			<h3>PMID:12650689  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Lopes AR(1), Terra WR.
				<br/> 
				年份：2003 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						A digestive trypsin from the American cockroach (Periplaneta americana, Dictyoptera) males was purified by a combination of anionic chromatographies in low and high pressure systems. The <span class="trait">yield</span> was 70% with a final specific activity of 2,000 units per mg protein (substrate: benzoyl-Arg-p-nitroanilide, BRpNA). Chemical modification with TLCK (k(obs)=3.3 M(-1) s(-1); stoichiometry 1:1) and PMSF (k(obs)=0.18 M(-1) s(-1); stoichiometry 1:1) confirmed that this peptidase is a trypsin. This enzyme has a molecular weight of 29 kDa (SDS-PAGE), a pI of 6.0 and a pH optimum of 8.9. Kinetic parameters using different colorimetric, fluorimetric and internally-quenched substrates indicated that P. americana trypsin prefers to hydrolyze synthetic substrates containing more than one amino acid residue and with an arginine residue at <span class="gene">P1</span> position and a hydrophobic residue at P2. This enzyme presented a Km of 120 microM for BRpNA and is competitively inhibited by benzamidine (Ki=0.25 microM). Soybean trypsin inhibitor is a tight-binding inhibitor presenting a K(D) of 0.4 nM. Differences in substrate specificity and in the reactivity of the trypsin active site groups can be related to adaptation of insects to different hosts. P. americana trypsin is an excellent model for comparison as a basal group on evolutionary studies of insect trypsins.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Common loci underlie field <span class="trait">resistance</span> to soybean sudden death syndrome in Forrest, Pyramid, Essex, and Douglas.</h2>
			<h3>PMID:12582700</h3>
				

				
			<p>
				作者：Njiti VN(1), Meksem K, Iqbal MJ, Johnson JE, Kassem MA, Zobrist KF, Kilo VY, Lightfoot DA.
				<br/> 
				年份：2002 Feb
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean [ Glycine max (L.) Merr.] sudden death syndrome (SDS) caused by Fusarium solani f. sp. glycines results in severe <span class="trait">yield</span> losses. Resistant cultivars offer the most-effective protection against <span class="trait">yield</span> losses but resistant cultivars such as 'Forrest' and 'Pyramid' vary in the nature of their response to SDS. Loci underlying SDS <span class="trait">resistance</span> in 'Essex'&emsp14;x&emsp14;Forrest are well defined. Our objectives were to identify and characterize loci and alleles that underlie field <span class="trait">resistance</span> to SDS in Pyramidx'Douglas'. SDS disease incidence and disease severity were determined in replicated field trials in six environments over 4 years. One hundred and twelve polymorphic DNA markers were compared with SDS disease response among 90 recombinant inbred lines from the cross PyramidxDouglas. Two quantitative trait loci (QTLs) for <span class="trait">resistance</span> to SDS derived their beneficial alleles from Pyramid, identified on linkage group G by BARC-Satt163 (261-bp allele, P=0.0005, R(2)=16.0%) and linkage group N by BARC-Satt080 (230-bp allele, P=0.0009, R(2)=15.6%). Beneficial alleles of both QTLs were previously identified in Forrest. A QTL for re- sistance to SDS on linkage group <span class="gene">C2</span> identified by BARC-Satt307 (292-bp allele, P=0.0008, R(2)=13.6%) derived the beneficial allele from Douglas. A beneficial allele of this QTL was previously identified in Essex. Recombinant inbred lines that carry the beneficial alleles for all three QTLs for <span class="trait">resistance</span> to SDS were significantly ( P&lt;/=0.05) more resistant than other recombinant inbred lines. Among these recombinant inbred lines <span class="trait">resistance</span> to SDS was environmentally stable. Therefore, gene pyramiding will be an effective method for developing cultivars with stable <span class="trait">resistance</span> to SDS.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Cleavage specificity of the subtilisin-like protease <span class="gene">C1</span> from soybean.</h2>
			<h3>PMID:12007608  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Boyd PM(1), Barnaby N, Tan-Wilson A, Wilson KA.
				<br/> 
				年份：2002 Apr 29
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The cleavage specificity of protease C1, isolated from soybean (Glycine max (L.) Merrill) seedling cotyledons, was examined using oligopeptide substrates in an HPLC based assay. A series of peptides based on the sequence Ac-KVEKEESEEGE-NH2 was used, mimicking a natural cleavage site of protease <span class="gene">C1</span> in the alpha subunit of the storage protein beta-conglycinin. A study of substrate peptides truncated from either the N- or C-terminus indicates that the minimal requirements for cleavage by protease <span class="gene">C2</span> are three residues N-terminal to the cleaved bond, and two residues C-terminal (i.e. P3-P2'). The maximal rate of cleavage is reached with substrates containing four to five residues N-terminal to the cleaved bond and four residues C-terminal (i.e. P4 or <span class="gene">P5</span> to P4'). The importance of Glu residues at the P1, P1', and P4 positions was examined using a series of substituted nonapeptides (P5-P4') with a base sequence of Ac-KVEKEESEE-NH2. At the <span class="gene">P1</span> position, the relative ranking, based on kcat/Km, was E&gt;Q&gt;K&gt;A&gt;D&gt;F&gt;S. Substitutions at the P1' position <span class="trait">yield</span> the ranking E congruent withQ&gt;A&gt;S&gt;D&gt;K&gt;F, while those at P4' had less effect on kcat/Km, yielding the ranking F congruent with S congruent with E congruent withD&gt;K&gt;A congruent withQ. These data show that protease <span class="gene">C1</span> prefers to cleave at Glu-Glu and Glu-Gln bonds, and that the nature of the P4' position is less important. The fact that there is specificity in the cleavage of the oligopeptides suggests that the more limited specific cleavage of the alpha and <span class="gene">alpha'</span> subunits of beta-conglycinin by protease <span class="gene">C1</span> is due to a combination of the sequence cleavage specificity of the protease and the accessibility of appropriate scissile peptide bonds on the surface of the substrate protein.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative trait loci in Two Soybean Recombinant Inbred Line Populations Segregating for <span class="trait">Yield</span> and Disease <span class="trait">Resistance.</span></h2>
			<h3>PMID:11756285</h3>
				

				
			<p>
				作者：Yuan J(1), Njiti VN, Meksem K, Iqbal MJ, Triwitayakorn K, Kassem MA, Davis GT, Schmidt ME, Lightfoot DA.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Molecular makers linked to quantitative trait loci (QTL) can assist soybean [Glycine max (L.) Merr.] breeders to combine traits of low heritability, such as <span class="trait">yield,</span> with disease <span class="trait">resistance.</span> The objective of this study was to identify markers linked to <span class="trait">yield</span> QTL in two recombinant inbred line (RIL) populations ['Essex' x 'Forrest' (ExF; n = 100) and 'Flyer' x 'Hartwig' (FxH; n = 94)] that also segregate for soybean cyst nematode (SCN) <span class="trait">resistance</span> genes (rhg1 and Rhg4). Each population was <span class="trait">yield</span> tested in four environments between 1996 and 1999. The resistant parents produced lower yields. Heritability of <span class="trait">yield</span> across four environments was 47% for ExF and 57% for FxH. <span class="trait">Yield</span> was normally distributed in both populations. High yielding, SCN resistant transgressive segregants were not observed. In the ExF RIL population, 134 microsatellite markers were compared against <span class="trait">yield</span> by ANOVA and MAPMAKER QTL. Regions associated with <span class="trait">yield</span> were identified by SATT294 on linkage group (LG.) <span class="gene">C1</span> (P = 0.006, R(2) = 10%), SATT440 on LG. I (P = 0.007, R(2) = 10%), and SATT337 on LG. K (P = 0.004, R(2) = 10%). Essex provided the beneficial allele at SATT337. Mean yields among FxH RILs were compared against 33 microsatellite markers from LG. K. In addition 136 markers from randomly selected LGs were compared with extreme phenotypes by bulk segregant analysis. Two regions on LG. K (20 cM apart) associated with <span class="trait">yield</span> were identified by SATT326 (P = 0.0004, R(2) = 15%) and SATT539 (P = 0.0008, R(2) = 14%). Flyer provided both beneficial alleles. Both populations revealed a <span class="trait">yield</span> QTL in the interval (5 cM) between SATT337 and SATT326. These populations may share a common allele for <span class="trait">yield</span> in this region, given that about 40% of Flyer genome derived from Essex.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biplot Analysis of Test Sites and Trait Relations of Soybean in Ontario.</h2>
			<h3>PMID:11756248</h3>
				

				
			<p>
				作者：Yan W(1), Rajcan I.
				<br/> 
				年份：2002 Jan
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Superior crop cultivars must be identified through multi-environment trials (MET) and on the basis of multiple traits. The objectives of this paper were to describe two types of biplots, the GGE biplot and the GT biplot, which graphically display genotype by environment data and genotype by trait data, respectively, and hence facilitate cultivar evaluation on the basis of <span class="gene">MET</span> data and multiple traits. Genotype main effect plus genotype by environment interaction effect (GGE) biplot analysis of the soybean [Glycine max (L.) Merr.] <span class="trait">yield</span> data for the 2800 crop heat unit area of Ontario for <span class="gene">MET</span> in the period 1994-1999 revealed yearly crossover genotype by site interactions. The eastern Ontario site Winchester showed a different genotype response pattern from the three southwestern Ontario sites in four of the six years. The interactions were not large enough to divide the area into different mega-environments as when analyzed over years, a single cultivar yielded the best in all four sites. The southwestern site, St. Pauls, was found to always group together with at least one of the other three sites; it did not provide unique information on genotype performance. Therefore, in future cultivar evaluations, Winchester should always be used but St. Pauls can be dismissed. Applying GT biplot to the 1994-1999 multiple trait data illustrated that GT biplots graphically displayed the interrelationships among seed <span class="trait">yield,</span> oil content, protein content, plant height, and days to maturity, among other traits, and facilitated visual cultivar comparisons and selection. It was found that selection for seed <span class="trait">yield</span> alone was not only the simplest, but also the most effective strategy in the early stages of soybean breeding.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Production of secretory IgA antibodies in plants.</h2>
			<h3>PMID:11566600  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Larrick JW(1), Yu L, Naftzger C, Jaiswal S, Wycoff K.
				<br/> 
				年份：2001 Oct 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Functional antibodies produced in tobacco plants were first reported over a decade ago (1989). The basic protocol used to generate these 'plantibodies' involved the independent cloning of H and L chain antibody genes in Agrobacterium tumefaciens vectors, the transformation of plant tissue in vitro with the recombinant bacterium, the reconstitution of whole plants expressing individual chains, and their sexual cross. In a 'Mendelian' fashion, a fully assembled and functional antibody was recovered from plant tissue in some double-transgenic plants. In mammalian cells, the antibody H and L chains are produced as precursor proteins that are translocated into the endoplasmic reticulum (ER), under the guidance of signal sequences. Within the ER, the signal peptides are proteolytically cleaved, and several stress proteins act as chaperonins to bind the unassembled antibody chains, and direct subsequent folding and tetramer formation. A similar process occurs in plant cells, and expression can be directed via signal sequences (even of foreign origin) into the aqueous environment of the apoplasm, or to be accumulated in other specific plant tissues, including tubers, fruit, or seed. Plants can facilely assemble secretory IgA, which is comprised of four chains, H and L chains, J chain and secretory component. Plant 'bioreactors' are expected to <span class="trait">yield</span> over 10 kg of therapeutic antibody/acre in tobacco, maize, soybean, and alfalfa [(Ann. NY Acad. Sci.)721(1994)235; (Biotechnol. Bioeng.)20(1999)135]. Compared with conventional steel tank bioreactors using mammalian cells, or microorganisms, the costs of <span class="gene">GMP</span> plantibodies are expected to perhaps one tenth. The differences in glycosylation patterns of plant and mammalian cell produced antibodies apparently have no effect on antigen-binding or specificity, but there is some concern about potential immunogenicity in humans. N-linked glycans of plants differ from human by having fucose-linked alpha 1,3 and the sugar xylose. No adverse effects or human anti-mouse antibodies (HAMA) have been observed in &gt;40 patients receiving topical oral application of a plant produced secretory IgA specific to Streptococcus mutans, for the control of caries [(Nat. Med.)4(1998)601]. The progressive improvement of expression vectors for plantibodies, and purification strategies, as well as the increase in transformable crop species, is expected to lead to almost limitless availability of inexpensive (even edible forms of) recombinant immunoglobulins free of human pathogens for human and animal therapy, and for novel industrial applications (e.g. catalytic antibodies).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Conjugated linoleic acid and other beneficial fatty acids in milk fat from cows fed soybean meal, fish meal, or both.</h2>
			<h3>PMID:11518309  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Abu-Ghazaleh AA(1), Schingoethe DJ, Hippen AR.
				<br/> 
				年份：2001 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Twelve multiparous Holstein cows at 48 +/- 8 <span class="gene">DIM</span> were used in a 4 x 4 Latin square with 21-d periods to determine the effect of replacing soybean meal with fish meal on feed intake, milk <span class="trait">yield,</span> and milk composition. Fish meal substituted for soybean meal on an isonitrogenous basis at 0, 25, 50, and 100% of supplemental protein. Total mixed diets were (DM basis) 25% corn silage, 25% alfalfa hay, and 50% concentrate mix. Intake of DM (27.9, 27.8, 26.1, and 25.8 kg/d for diets 1 to 4, respectively) was similar for all diets. Milk <span class="trait">yield</span> (37.5, 37.8, 37.2, and 37.7 kg/d) was not affected by diets. Milk protein percentages (3.23, 3.24, 3.31, and 3.35) increased with 100% fish meal supplementation and tended to be higher with 50% fish meal supplementation compared with 100% soybean meal diet. Milk fat percentages (3.18, 2.99, 3.04, and 2.87) and <span class="trait">yield</span> were lower with the 100% fish meal than with the 100% soybean meal diet. Concentration of n-3 fatty acids in milk fat (0.54, 0.56, 0.63, and 0.72 g/100 g fatty acids) increased as the proportion of fish meal in the diet increased. Concentrations of c9,t11 conjugated linoleic acid (CLA; 0.39, 0.44, 0.46, and 0.72 g/100 g fatty acids) and transvaccenic acid (TVA; 1.09, 1.19, 1.28, and 1.54 g/100 g of fatty acids) were higher with the 100% fish meal diet than with the 100% soybean meal diet. A total replacement of soybean meal with fish meal in the diet of lactating cows increased milk protein percentages and the beneficial fatty acids (CLA, TVA, and n-3 FA) in milk fat.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A novel phytase with sequence similarity to purple acid phosphatases is expressed in cotyledons of germinating soybean seedlings.</h2>
			<h3>PMID:11500558  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hegeman CE(1), Grabau EA.
				<br/> 
				年份：2001 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytic acid (myo-inositol hexakisphosphate) is the major storage form of phosphorus in plant seeds. During germination, stored reserves are used as a source of nutrients by the plant seedling. Phytic acid is degraded by the activity of phytases to <span class="trait">yield</span> inositol and free phosphate. Due to the lack of phytases in the non-ruminant digestive tract, monogastric animals cannot utilize dietary phytic acid and it is excreted into manure. High phytic acid content in manure results in elevated phosphorus levels in soil and water and accompanying environmental concerns. The use of phytases to degrade seed phytic acid has potential for reducing the negative environmental impact of livestock production. A phytase was purified to electrophoretic homogeneity from cotyledons of germinated soybeans (Glycine max L. Merr.). Peptide sequence data generated from the purified enzyme facilitated the cloning of the phytase sequence <span class="gene">(GmPhy)</span> employing a polymerase chain reaction strategy. The introduction of <span class="gene">GmPhy</span> into soybean tissue culture resulted in increased phytase activity in transformed cells, which confirmed the identity of the phytase gene. It is surprising that the soybean phytase was unrelated to previously characterized microbial or maize (Zea mays) phytases, which were classified as histidine acid phosphatases. The soybean phytase sequence exhibited a high degree of similarity to purple acid phosphatases, a class of metallophosphoesterases.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heat-shock responses in two leguminous plants: a comparative study.</h2>
			<h3>PMID:11479337  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Ortiz C(1), Cardemil L.
				<br/> 
				年份：2001 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Relative growth rates, basal and acclimated thermotolerance, membrane damage, fluorescence emission, and relative levels of free and conjugated ubiquitin and <span class="gene">HSP70</span> were compared after 2 h of treatment at different temperatures between Prosopis chilensis and Glycine max (soybean), cv. McCall, to evaluate if the thermotolerance of these two plants was related to levels of accumulation of heat shock proteins. Seedlings of P. chilensis germinated at 25 degrees C and at 35 degrees C and grown at temperatures above germination temperature showed higher relative growth than soybean seedlings treated under the same conditions. The lethal temperature of both species was 50 degrees C after germination at 25 degrees C. However, they were able to grow at 50 degrees C after germination at 35 degrees C. Membrane damage determinations in leaves showed that P. chilensis has an LT(50) 6 degrees C higher than that of soybean. There were no differences in the quantum <span class="trait">yield</span> of photosynthesis (F(v)/F(m)), between both plants when the temperatures were raised. P. chilensis showed higher relative levels of free ubiquitin, conjugated ubiquitin and <span class="gene">HSP70</span> than soybean seedlings when the temperatures were raised. Time-course studies of accumulation of these proteins performed at 40 degrees C showed that the relative accumulation rates of ubiquitin, conjugated ubiquitin and <span class="gene">HSP70</span> were higher in P. chilensis than in soybean. In both plants, free ubiquitin decreased during the first 5 min and increased after 30 min of heat shock, conjugated ubiquitin increased after 30 min and <span class="gene">HSP70</span> began to increase dramatically after 20 min of heat shock. From these data it is concluded that P. chilensis is more tolerant to acute heat stress than soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Total body electrical conductivity (TOBEC) measurement of compositional differences in hams, loins, and bellies from conjugated linoleic acid (CLA)-fed stress-genotype pigs.</h2>
			<h3>PMID:11424684  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Swan JE(1), Parrish FC Jr, Wiegand BR, Larsen ST, Baas TJ, Berg EP.
				<br/> 
				年份：2001 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						This study was designed to observe the effects of conjugated linoleic acid (CLA) supplementation on lean content of pork carcass primal cuts (hams, loins, and bellies) and to determine the ability of total body electrical conductivity (TOBEC) to predict lean content. A total of 64 crossbred growing-finishing barrows were placed on a control (soybean oil) or CLA (0.75%) diet at an average weight of 40 kg. Pigs were penned in pairs according to diet and stress genotype (negative, carrier, and positive) and slaughtered at 115 kg. Stress genotype was included because of known variations in lean content. Hams (IMPS 401A; n = 64), loins (IMPS 410; n = 24), and bellies (IMPS 408; n = 63) were fabricated from carcasses at 24 h postmortem and scanned for electromagnetic (EM) absorption by a MQ-25 EM scanner. Each wholesale cut was scanned in triplicate at 2.5 MHz to <span class="trait">yield</span> a peak mean average (PMA) value then separated into lean, fat, bone, and skin components. Bellies were skinned prior to scanning then subjected to a belly <span class="gene">bar</span> firmness test before dissection. Supplementation with CLA had no effect (P &gt; 0.05) on lean ham composition. Regression analysis was used for lean weight prediction using primal weight and <span class="gene">PMA</span> value as predictors. Lean content prediction of hams by TOBEC resulted in an R2 of 0.80. Loins from CLA-supplemented pigs exhibited increased lean weight (P &lt; 0.05) and <span class="gene">PMA</span> values (P &lt; 0.05) compared to controls. Lean prediction of loins by TOBEC resulted in an R2 of 0.66. Bellies from CLA-supplemented pigs had a higher percentage of moisture (P &lt; 0.03) and protein (P &lt; 0.01) and decreased percentage of lipid (P &lt; 0.01). The R2 values from the regression analysis predicting protein, moisture, protein + moisture, and fat-free soft tissue composition of the skinless bellies were 0.67, 0.68, 0.71, and 0.78, respectively.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Functional expression in insect cells, one-step purification and characterization of a recombinant phospholipase D from cowpea (Vigna unguiculata L. Walp).</h2>
			<h3>PMID:11161113  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：El Maarouf H(1), Carrière F, Rivière M, Abousalham A.
				<br/> 
				年份：2000 Nov
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phospholipase D (PLD) is an important enzyme involved in signal transduction, vesicle trafficking and membrane metabolism. In this study, large amounts of a recombinant plant PLD alpha were secreted into the culture medium of baculovirus-infected insect cells and purified to homogeneity in the form of a fully active enzyme. The transient production of recombinant PLD alpha yielded a protein (rPLD alpha a, 88 kDa) together with a shorter form (rPLD alpha b, 87 kDa), which accumulated in the medium. N-Terminal amino acid sequencing of the rPLD alpha a and rPLD alpha b showed that rPLD alpha b resulted from proteolytic cleavage at Gly8-Ile9. Immunoblotting showed that both rPLD alpha a and rPLD alpha b are recognized by a polyclonal antibody previously raised against native soybean PLD alpha. One-step calcium-dependent octyl-Sepharose chromatography was used to obtain the two highly purified forms of rPLD alpha, as attested by gel electrophoresis, N-terminal amino acid sequence and mass spectrometry. The N-terminal region of PLD alpha is homologous with the <span class="gene">C2</span> domains which are present in a number of enzymes known to be involved in signal transduction and/or phospholipid metabolism. The truncated rPLD alpha b lacks the first acidic amino acid in its N-terminus, which is probably involved in the calcium binding site. The rPLD alpha b was thus easily eluted from the octyl-Sepharose column by decreasing the calcium concentration of the buffer from 50 to 30 mM, whereas, the rPLD alpha a was eluted after chelating calcium ions with EDTA. The purified rPLD alpha <span class="trait">yield</span> reached a level of 10 mg per liter of serum-free culture medium. The availability of baculovirus-derived rPLD alpha constitutes a valuable source of enzyme for future crystallographic studies to determine its three-dimensional structure.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fumaric acid: an overlooked form of fixed carbon in Arabidopsis and other plant species.</h2>
			<h3>PMID:11089689  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Chia DW(1), Yoder TJ, Reiter WD, Gibson SI.
				<br/> 
				年份：2000 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photoassimilates are used by plants for production of energy, as carbon skeletons and in transport of fixed carbon between different plant organs. Many studies have been devoted to characterizing the factors that regulate photoassimilate concentrations in different plant species. Most studies examining photoassimilate concentrations in <span class="gene">C3</span> plants have focused on analyzing starch and soluble sugars. However, work presented here demonstrates that a number of <span class="gene">C3</span> plants, including the popular model organism Arabidopsis thaliana (L.) Heynh., and agriculturally important plants, such as soybean, Glycine max (L.) Merr., contain significant quantities of fumaric acid. In fact, fumaric acid can accumulate to levels of several milligrams per gram fresh weight in Arabidopsis leaves, often exceeding those of starch and soluble sugars. Fumaric acid is a component of the tricarboxylic acid cycle and, like starch and soluble sugars, can be metabolized to <span class="trait">yield</span> energy and carbon skeletons for production of other compounds. Fumaric acid concentrations increase with plant age and light intensity in Arabidopsis leaves. Moreover, Arabidopsis phloem exudates contain significant quantities of fumaric acid, raising the possibility that fumaric acid may function in carbon transport.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean peroxidase as an effective bromination catalyst*</h2>
			<h3>PMID:10713205</h3>
				

				
			<p>
				作者：Munir IZ(1), Dordick JS.
				<br/> 
				年份：2000 Mar 1
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Soybean peroxidase (SBP), an acidic peroxidase isolated from the seed coat, has been shown to be an effective catalyst for the oxidation of a variety of organic compounds. In the present study, we demonstrate that <span class="gene">SBP</span> can catalyze halogenation reactions. In the presence of H(2)O(2), <span class="gene">SBP</span> catalyzed the oxidation of bromide and iodide but not chloride. Veratryl alcohol (3,4-dimethoxybenzyl alcohol) served as a useful substrate for SBP-catalyzed halogenations yielding the 6-bromo derivative. Halogenation of veratryl alcohol was optimal at pHs below 2.5 with rates of 2.4 µm/min, achieving complete conversions of 150-µm veratryl alcohol in 24 h. The enzyme showed essentially no brominating activity at pHs above 5.5. SBP-catalyzed bromination of veratryl alcohol proceeded with a maximum reaction velocity, (V(max))(apparent), of 5.8 x 10(-1) µm/min, a K(m) of 78 µm and a catalytic efficiency (k(cat)/K(m) of 1.37 x 10(5) M/min at pH 4.0, assuming all of the enzyme's active sites participate in the reaction. <span class="gene">SBP</span> also catalyzed the bromination of several other organic substrates including pyrazole to produce a single product 1-bromopyrazole, indole to <span class="trait">yield</span> both 5-bromoindole and 5-hydroxyindole, and the decarboxylative bromination of 3,4 dimethoxy-trans-cinnamic acid to trans-2-bromo-1-(3,4 dimethoxyphenyl)ethylene. A catalytic mechanism for SBP-catalyzed bromination has been proposed based on experimental results in this and related studies.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Distribution of two isoforms of NADP-dependent isocitrate dehydrogenase in soybean (Glycine max).</h2>
			<h3>PMID:10394941  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Park KS(1), Kahn ML.
				<br/> 
				年份：1999 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two different cDNAs that encode NADP-specific isocitrate dehydrogenase (NADP-IDH) isozymes of soybean (Glycine max) were characterized. The nucleotide sequences of the coding regions of these cDNAs have 74% identity to each other and give predicted amino acid sequences that have 83% identity to each other. Using PCR techniques, their coding regions were subcloned into a protein overexpression vector, pQE32, to <span class="trait">yield</span> pIDH4 and pIDH1, respectively. Both IDH4 and <span class="gene">IDH1</span> enzymes were expressed in Escherichia coli as catalytically active His6 tagged proteins, purified to homogeneity by affinity chromatography on nickel chelate resin and rabbit polyclonal antibodies to each were generated. Surprisingly, antiserum to IDH4 did not react with <span class="gene">IDH1</span> protein and <span class="gene">IDH1</span> antiserum reacted only very weakly with IDH4 protein. IDH4 antibody reacts with a protein of expected molecular weight in cotyledon, young leaf, young root, mature root and nodules but the reaction with mature leaf tissue was low compared to other tissues. Western blot results show that <span class="gene">IDH1</span> was not expressed in young roots but a protein that reacts with the <span class="gene">IDH1</span> antibody was highly expressed in leaves, showing that there was tissue-specific accumulation of NADP-IDH isozymes in soybean.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular strategies to improve the nutritional quality of legume proteins.</h2>
			<h3>PMID:10335390  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：de Lumen BO(1), Galvez AF, Revilleza MJ, Krenz DC.
				<br/> 
				年份：1999
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Legume proteins are relatively deficient in the sulfur amino acids. Among the different strategies to increase the sulfur amino acid contents of legumes, we have chosen to increase the biosynthesis of endogenous, non-abundant sulfur-rich proteins we have identified in soybean seed. We identified and isolated an 8 kDa sulfur-rich protein from the albumin fraction of soybean seed. Subsequently, a cDNA designated <span class="gene">Gm2S-1,</span> has been isolated and characterized. The <span class="gene">Gm2S-1</span> cDNA encodes a pre-proprotein that contains a signal peptide and a precursor protein that undergoes post-translational processing to <span class="trait">yield</span> the mature 8 kDa protein containing 7.8% methionine and 7.8% cysteine. The <span class="gene">Gm2S-1</span> gene is an ideal candidate for overexpression to improve the nutritional quality of soybean and other legumes.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expression of an Aspergillus niger phytase gene (phyA) in Saccharomyces cerevisiae.</h2>
			<h3>PMID:10223979  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Han Y(1), Wilson DB, Lei XG.
				<br/> 
				年份：1999 May
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Phytase improves the bioavailability of phytate phosphorus in plant foods to humans and animals and reduces phosphorus pollution of animal waste. Our objectives were to express an Aspergillus niger phytase gene (phyA) in Saccharomyces cerevisiae and to determine the effects of glycosylation on the phytase's activity and thermostability. A 1.4-kb DNA fragment containing the coding region of the <span class="gene">phyA</span> gene was inserted into the expression vector pYES2 and was expressed in S. cerevisiae as an active, extracellular phytase. The <span class="trait">yield</span> of total extracellular phytase activity was affected by the signal peptide and the medium composition. The expressed phytase had two pH optima (2 to 2.5 and 5 to 5.5) and a temperature optimum between 55 and 60 degrees C, and it cross-reacted with a rabbit polyclonal antibody against the wild-type enzyme. Due to the heavy glycosylation, the expressed phytase had a molecular size of approximately 120 kDa and appeared to be more thermostable than the commercial enzyme. Deglycosylation of the phytase resulted in losses of 9% of its activity and 40% of its thermostability. The recombinant phytase was effective in hydrolyzing phytate phosphorus from corn or soybean meal in vitro. In conclusion, the <span class="gene">phyA</span> gene was expressed as an active, extracellular phytase in S. cerevisiae, and its thermostability was affected by glycosylation.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evidence for the formation of dinor isoprostanes <span class="gene">E1</span> from alpha-linolenic acid in plants.</h2>
			<h3>PMID:9830005  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Parchmann S(1), Mueller MJ.
				<br/> 
				年份：1998 Dec 4
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The free radical oxidation of arachidonic acid is known to generate complex metabolites, termed isoprostanes, that share structural features of prostaglandins and exert potent receptor-mediated biological activities. In the present study, we show that alpha-linolenic acid can undergo a similar oxidation process, resulting in a series of isomeric dinor isoprostanes E1. E-ring dinor isoprostane formation from linolenate was found to be catalyzed by soybean lipoxygenase. The main enzymatic products were 13- and 9-hydroperoxylinolenate but in addition, two dinor isoprostane <span class="gene">E1</span> regioisomers were formed with a <span class="trait">yield</span> of 0.31%. Identification and quantification of two dinor isoprostane <span class="gene">E1</span> regioisomers in plant cell cultures was achieved by a negative chemical ionization gas chromatography-mass spectrometry method using [18O]dinor isoprostanes <span class="gene">E1</span> as internal standards. Endogenous levels of these compounds were determined in four taxonomically distant plant species and found to be in the range of 4.5 to 60.9 ng/g of dry weight. Thus analogous pathways in animals and plants exist, each leading to a family of prostaglandin-like compounds derived from polyunsaturated fatty acids. It remains to be shown whether the dinor isoprostanes exert biological activities in plants as has been demonstrated for their C20 congeners in mammals.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Supplemental protein plus ruminally protected methionine and lysine for primiparous beef cattle consuming annual rye hay.</h2>
			<h3>PMID:9690631  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hess BW(1), Scholljegerdes EJ, Coleman SA, Williams JE.
				<br/> 
				年份：1998 Jul
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Primiparous beef cows (n = 35 Bos taurus, average initial BW of 498 kg) were allotted to treatments in a split-plot designed experiment to determine the effects of supplemental ruminally protected amino acids on cow and calf productivity and metabolic changes in the cows. Cows were fed chopped annual rye hay at 1.5% of BW. The following treatments were used: 1) .8 kg soybean hulls, 1.4 kg ground corn, .6 kg soybean meal (CON); 2) 1.4 kg ground corn, 1.4 kg soybean meal (PRO); 3) PRO plus ruminally protected methionine and lysine (supplied 5 and 10 g, respectively; PRO1); and 4) PRO with twice the level of ruminally protected amino acids in <span class="gene">PRO1</span> (PRO2). Cow weight gain was not different (P &gt; .26) among treatments and averaged 1.2 kg/d for the 45 + 6 d before parturition. After parturition, cow weight gain did not differ (P &gt; .47) between CON and PRO treatments, but it decreased quadratically (P &lt; .01) with increasing level of ruminally protected amino acids. Total milk <span class="trait">yield,</span> protein, and fat (4 h) were greater (P &lt; .05) for cows consuming PRO supplements than for CON, whereas CON cattle tended (P = .11) to lose less body condition. Total milk protein showed a quadratic increase (P &lt; .05) in response to level of ruminally protected amino acids that was the inverse of the quadratic response noted for cow weight gain. Serum urea N concentration was greater (P = .07) for cattle consuming additional protein. Metabolic hormones were not affected (P &gt; .18) by dietary treatment, but they responded (P &lt; .05) to changes in physiological state. Supplements with additional protein supported greater (P = .0001) milk urea N concentration and output. Milk urea N concentration increased (P &lt; .05) and milk IGF-I decreased (P &lt; .05) as the lactation period progressed. The measurement of CON and PRO diets revealed that supplements with additional soybean meal had greater (P &lt; .05) DM and N degradation; the extent of forage DM and NDF degradation was similar (P &gt; .05) among treatments. Production shifted away from body weight gain to increased milk protein production when daily supplementation of ruminally protected methionine and lysine increased from 5 and 10 to 10 and 20 g, respectively. This shift in production was not reflective of changes in the metabolic regulators measured.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The use of single cell gel electrophoresis and flow cytometry to identify antimutagens from commercial soybean by-products.</h2>
			<h3>PMID:9675285  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Plewa MJ(1), Wagner ED, Kirchoff L, Repetny K, Adams LC, Rayburn AL.
				<br/> 
				年份：1998 Jun 18
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Single cell gel electrophoresis (alkaline Comet assay) and flow cytometric methods were combined into an assay that enables the analysis of direct DNA damage and longer-term whole cell clastogenicity in mammalian cells. We employed these techniques to analyze the antimutagenic activity of by-products of commercial soybean processing. At a concentration of 1 mg/ml, the soybean molasses by-product was found to repress 66% of the mutagenic capacity of the direct-acting mutagen 2-acetoxyacetylaminofluorene (2AAAF) in Chinese hamster lung (CHL) cells. At a concentration of 50 microg/ml, fraction PCC (an ethanol extract of soybean molasses) repressed 70% of the genotoxic potency of 500 nM 2AAAF as measured by the Comet assay. Fraction PCC was also effective in protecting <span class="gene">CHL</span> cells from 2AAAF-induced clastogenic damage. Using a forward mutation assay in Chinese hamster ovary cells (line AS52), PCC protected the cells against 2AAAF-induced cytotoxicity and point mutation at a specific gene target. These data indicate that agronomic crops such as soybean may <span class="trait">yield</span> a wealth of commercially available antimutagenic agents that may be suitable as chemoprotective food supplements.
					</p>
				
					<p>&nbsp;&nbsp;
						Copyright 1998 Elsevier Science B. V. All rights reserved.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:[Feed intake and milk production in dairy cows fed whole fat soybeans].</h2>
			<h3>PMID:9735101  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Preissinger W(1), Schwarz FJ, Kirchgessner M.
				<br/> 
				年份：1997
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						In two experiments (E 1, E 2) two different basic diets were fed to 36 diary cows with addition of 1 or 2 kg of full fat soya beans (SB) or without SB for six weeks. The cows were fed individually. The basic diet consisted of 49% grass silage, 23% maize silage and 21% hay, supplemented with 7% manioc (% DM) in E 1 and of grass, supplemented with 2.0 kg maize silage, 3.0 kg hay and 1.5 kg manioc (kg DM/cow.d) in E 2. The basic diet used in E 1 and the grass in E 2 were offered ad libitum. Further a mixture of 1 kg SB, soya bean meal and manioc (Treatment 1), a mixture of 2 kg SB and manioc (Treatment 2) and a mixture of soya bean meal and manioc (Treatment 3, control), respectively, was supplemented to achieve diets with equivalent concentrations of energy and protein. Depending on the daily milk <span class="trait">yield</span> a concentrate was fed to high yielding cows (&gt; 21.5 kg milk in E 1 and &gt; 25.0 kg milk in E 2, respectively). The addition of 1 kg SB increased the forage intake in both experiments by 1.2-1.5 kg DM/cow.d significantly, while the addition of 2 kg SB increased the forage intake only in <span class="gene">E1</span> to the same extent. The mean total DMI was 19.2 kg in E 1 and 17.7 kg in E 2, respectively. The daily milk <span class="trait">yield</span> was not influenced by addition of SB. The average milk <span class="trait">yield</span> was 28.8 kg and 26.7 kg in E 1 and E 2, respectively. The milk fat content increased significantly in E 1 by 0.3-0.4% with no respect to the amount of the supplied SB (4.11% vs. 3.74%). In E 2 the milk fat content increased only slightly (3.84% vs. 3.74%). Milk protein content was not influenced in E 1, but decreased in E 2 after addition of 2 kg SB.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Total Glutamine Synthetase Activity during Soybean Nodule Development Is Controlled at the Level of Transcription and Holoprotein Turnover.</h2>
			<h3>PMID:12226474</h3>
				

				
			<p>
				作者：Temple SJ(1), Kunjibettu S, Roche D, Sengupta-Gopalan C.
				<br/> 
				年份：1996 Dec
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Gln synthetase (GS) catalyzes the ATP-dependent condensation of ammonia with glutamate to <span class="trait">yield</span> Gln. In higher plants <span class="gene">GS</span> is an octameric enzyme and the subunits are encoded by members of a small multigene family. In soybeans (Glycine max), following the onset of N2 fixation there is a dramatic increase in <span class="gene">GS</span> activity in the root nodules. <span class="gene">GS</span> activity staining of native polyacrylamide gels containing nodule and root extracts showed a common band of activity (GSrs). The nodules also contained a slower-migrating, broad band of enzyme activity (GSns). The GSns activity band is a complex of many isozymes made up of different proportions of two kinds of <span class="gene">GS</span> subunits: GSr and GSn. Root nodules formed following inoculation with an Nif- strain of Bradyrhizobium japonicum showed the presence of <span class="gene">GS</span> isoenzymes (GSns1) with low enzyme activity, which migrated more slowly than GSns. Gsns1 is most likely made up predominantly of GSn subunits. Our data suggest that, whereas the class I <span class="gene">GS</span> genes encoding the GSr subunits are regulated by the availability of NH3, the class II <span class="gene">GS</span> genes coding for the GSn subunits are developmentally regulated. Furthermore, we have demonstrated that the GSns1 isozymes in the Nif- nodules are relatively more labile. Our overall conclusion is that GSns activity in soybean nodules is regulated by N2 fixation both at the level of transcription and at the level of holoprotein stability.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybeans versus animal sources of rumen-undergradable protein and fat for early lactation dairy cows.</h2>
			<h3>PMID:8923252  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Grummer RR(1), Slark K, Bertics SJ, Luck ML, Barmore JA.
				<br/> 
				年份：1996 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fifteen multiparous Holstein cows averaging 39 <span class="gene">DIM</span> were used in a replicated 5 x 5 Latin square design to compare roasted soybeans and animal by-products as supplements for increasing the RUP content of lactation diets based on alfalfa. The control diet contained roasted soybeans and was formulated to meet requirements for RUP. Rumen-undegradable protein in the diet was increased by including additional roasted soybeans or animal by-product proteins. To achieve diets with similar amounts of fat, we added tallow, hydrolyzed tallow fatty acids, or partially hydrogenated tallow to the control diet and to diets containing animal by-product proteins. Control diets and diets with high RUP were estimated to be 5.8 and 7.6% RUP (DM basis). The RDP of all diets was estimated to be approximately 11.3% of DM. Milk <span class="trait">yield</span> was increased by additional dietary RUP. <span class="trait">Yield</span> of FCM was greater for cows receiving animal byproducts than for those fed additional roasted soybeans. Cows fed additional RUP yielded more milk protein, but milk protein percentage was decreased. Milk fat percentage was reduced by additional dietary RUP, but fat <span class="trait">yield</span> was not affected. Nutrient digestibility was greatest for diets containing animal byproducts. Cows fed tallow yielded more milk, FCM, and protein that did those fed hydrolyzed tallow fatty acids or partially hydrogenated tallow. Lactation performance can be enhanced by supplementing animal by-products or roasted soybeans to diets based on alfalfa and formulated to meet NRC recommendations for RUP.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of intraperitoneal L-leucine, L-isoleucine, L-valine, and L-arginine on milk fat depression in early lactation cows.</h2>
			<h3>PMID:8201043  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Hopkins BA(1), Rakes AH, Daniel TE, Zimmerman CA, Croom WJ Jr.
				<br/> 
				年份：1994 Apr
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Eight Holstein cows were assigned following calving to two groups, balanced for parity, using a continuous completely randomized block design. Cows were fed a diet with 13.5% CP and 22.4% ADF from 35 to 55 <span class="gene">DIM</span> and then 13.8% CP and 15% ADF from 56 to 92 DIM. Alfalfa grass hay was the forage source, and concentrate mixtures contained primarily corn and soybean meal. Cows were given daily intraperitoneal infusions of a solution of L-Leu (46.1 g, 84.2 mM), L-Ile (31.4 g, 57.3 mM), L-Val (38.3 g, 78.2 mM), and L-Arg (25.0 g, 34.4 mM) or physiological saline following the a.m. milking from 42 through 84 DIM. Infusion of AA significantly increased plasma concentrations of Leu, Ile, Val, and Arg. Effects of AA infusion occurred during the low fiber period. Cows receiving AA maintained daily milk fat <span class="trait">yield,</span> increased p.m. milk fat <span class="trait">yield,</span> decreased less in p.m. milk fat percentage, and increased daily and p.m. yields of C16 fatty acids in milk. During the posttreatment period, cows previously receiving AA declined in daily milk fat <span class="trait">yield,</span> milk fat percentage, and total daily <span class="gene">C4</span> to C16 milk fatty acid <span class="trait">yield.</span> Results suggest that the infused AA may have increased de novo synthesis of C16 milk fatty acids.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Developmentally regulated transcription of the four liver-specific genes for inter-alpha-inhibitor family in mouse.</h2>
			<h3>PMID:7504460  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Salier JP(1), Chan P, Raguenez G, Zwingman T, Erickson RP.
				<br/> 
				年份：1993 Nov 15
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						The inter-alpha-inhibitor family is composed of the plasma-protease inhibitors inter-alpha-inhibitor, pre-alpha-inhibitor and bikunin. Inter-alpha-inhibitor and pre-alpha-inhibitor are distinct assemblies of bikunin with distinct sets from three heavy (H) chains designated H1, H2 and H3. These H chains are encoded by a set of three evolutionarily related H genes, and bikunin by an alpha-1-microglobulin/bikunin precursor gene (AMBP). This precursor is cleaved to <span class="trait">yield</span> bikunin, a member of the Kunitz-type protease-inhibitor superfamily, and alpha-1-microglobulin, which belongs to the lipocalin superfamily. Northern-blot experiments with RNAs obtained from various tissues in fetal and in adult mice indicated that the transcription of the four AMBP and H genes is liver-restricted, although there is expression of <span class="gene">H3</span> in brain. An analysis of the H1, H2, <span class="gene">H3</span> and AMBP transcripts, as well as of transcripts for other control genes, in liver during development showed a progressive increase in the amounts of the H1, H2, <span class="gene">H3</span> and AMBP RNAs, which all peak transiently at day 5 after birth. This was shown by a nuclear run-on experiment to originate from a change in transcription rate. The transient and postnatal increase in transcription could be explained neither by the liver-restricted expression nor by a common origin of these four genes, nor by a perinatal requirement for many lipocalins or protease inhibitors. This suggests that all four genes are perinatally triggered at the level of similar elements in their transcriptional regulatory regions, a conclusion strengthened by the weak expression of the four genes that is seen in a mutant mouse strain (albino) that is deficient in some liver-specific transcription factors.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Further characterization of cytochrome <span class="gene">P450</span> involved in phytoalexin synthesis in soybean: cytochrome <span class="gene">P450</span> cinnamate 4-hydroxylase and 3,9-dihydroxypterocarpan 6a-hydroxylase.</h2>
			<h3>PMID:1731635  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kochs G(1), Werck-Reichhart D, Grisebach H.
				<br/> 
				年份：1992 Feb 14
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two cytochrome <span class="gene">P450</span> enzymes, cinnamate 4-hydroxylase (C4H) and 3,9-dihydroxypterocarpan 6a-hydroxylase (D6aH), were isolated from elicitor-challenged soybean (Glycine max) cell cultures (G. Kochs and H. Grisebach, 1989, Arch. Biochem. Biophys. 273, 543-553). An earlier purification protocol was improved by the use of new chromatographic media, leading to a higher <span class="trait">yield</span> of enzymatic activity. After separation of <span class="gene">C4H</span> from D6aH on hydroxyapatite, the <span class="gene">C4H</span> was identified using anti-C4H antibody from Jerusalem artichoke (Helianthus tuberosus) (B. Gabriac et al., 1991, Arch. Biochem. Biophys. 288, 302-309). The two proteins show molecular weights of about 58,000 for <span class="gene">C4H</span> and about 55,000 for D6aH on sodium dodecyl sulfate-polyacrylamide gel electrophoresis. Both enzyme activities are dependent on NADPH:cytochrome <span class="gene">P450</span> reductase and cross-react with their respective antibodies. Both cytochrome <span class="gene">P450</span> subspecies show substrate binding and CO-difference spectra typical for cytochrome <span class="gene">P450</span> and were found to be glycoproteins by their cross-reaction with biotinylated lectins in Western blot. The N-terminal sequence of <span class="gene">C4H</span> from soybean shows high similarity to the N-terminus of <span class="gene">C4H</span> from Jerusalem artichoke.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Defined, serum-free culture conditions for the <span class="gene">GM-micro-clonogenic</span> assay using agar-capillaries.</h2>
			<h3>PMID:2765676  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Echarti C(1), Maurer HR.
				<br/> 
				年份：1989 Aug
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Culture conditions were determined and optimised for the serum-free growth of granulocyte-macrophage (GM) colonies, derived from mouse bone marrow cells, in agar-containing glass capillaries. The standard medium is DMEM/F-12 (1:1) mixture containing per ml: 10 mg BSA, 35 micrograms transferrin, 20 micrograms soybean lipids and 5 micrograms insulin. In contrast to previous attempts by others, <span class="gene">GM-colony</span> <span class="trait">yield</span> in serum-free medium was found to be equal to that in serum-containing medium (around 25 colonies/capillary) and to correlate satisfactorily with the cell density at seeding.--The role of polyamine oxidases in cell-proliferation experiments could be studied by this microclonogenic assay without interference from any type of serum.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Product <span class="trait">yield</span> in oxygenation of linoleate by soybean lipoxygenase: the value of the molar extinction coefficient in the spectrophotometric assay.</h2>
			<h3>PMID:3116881  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Gibian MJ(1), Vandenberg P.
				<br/> 
				年份：1987 Jun
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Two iodimetric methods and a gravimetric method were used to determine the spectrophotometric molar absorptivity of the purified product of lipoxygenase-catalyzed dioxygenation of linoleate (13-LS-hydroperoxy-cis,trans-9,11-octadecadienoate). Earlier determinations had led to the use of values varying from 24,000 to 28,000 M-1 <span class="gene">cm-1</span> for epsilon at 235 nm. In the current work, the two iodimetric values (spectrophotometric and titrimetric) average 22,500, while gravimetric analysis of scrupulously purified material gives 22,900. Final 235-nm absorbancies for lipoxygenase runs over a wide range of linoleic acid concentrations up to 200 microM give a constant final percentage completion. If one assumes a 100% reaction, epsilon is 23,600. Each method has less than 1.5% standard error; the average of the three independent methods is 23,000 +/- 580 (2.5%), all being lower than the previous values. In the enzyme-catalyzed reaction of linoleate at less than 200 microM substrate, only 235-nm-absorbing material is formed. Above 200 microM linoleate, yields at 235 nm decrease and yields of materials absorbing at 280 nm increase (the latter is known to arise from lipoxygenase-catalyzed reaction of linoleyl hydroperoxide). Below 200 microM substrate, linoleate purified by HPLC produces only one HPLC-observable product, 13-linoleyl hydroperoxide. At higher substrate concentrations other HPLC peaks arise, again with higher wave-length absorptions. Spectrophotometric data using the epsilon determined here agree with those from the O2 electrode. It is concluded that at S less than 200 microM, saturating air, and sufficient enzyme, soybean lipoxygenase-1 produces a sole product and the reaction proceeds to completion.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Heme protein fluorescence versus pressure.</h2>
			<h3>PMID:3697474  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Marden MC, Hui Bon Hoa G, Stetzkowski-Marden F.
				<br/> 
				年份：1986 Mar
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Fluorescence spectra of several ferric heme proteins have been measured vs. pressure to 6,000 bars. Sperm whale myoglobin (SW Mb), Aplysia myoglobin, leghemoglobin (Lb), and cytochrome <span class="gene">P450</span> all show excitation and emission spectra characteristic of tryptophan in proteins with peak emission at 330-340 nm. At one bar, the fluorescence is weak due to energy transfer to the heme group, which makes the <span class="trait">yield</span> a sensitive probe of protein unfolding at high pressure. After an initial decrease of a few percent per kbar, the protein shows a large increase in fluorescence at high pressure. The increase is pH dependent and the results indicate that several high pressure states occur. For SW Mb at 15 degrees C an increase of a factor of 20 occurs with midpoint at 2,000 bars at pH 5 and is only partially reversible, while the increase at pH 7 occurs at 4,000 bars and is only half as large and is completely reversible. Aplysia Mb and Lb show a similar effect, but unfold at a higher pressure than SW Mb. <span class="gene">P450</span> also shows a transition to a state of higher fluorescence, but the transition in this case is irreversible as a stable form, P420, is formed. The fluorescence intensity measurements permit an estimation of the increase in the TRY-heme distance in the high pressure state.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthesis and increased production of protein.</h2>
			<h3>PMID:727014  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Platt SG, Bassham JA.
				<br/> 
				年份：1978
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Photosynthesis, the use of light energy in the conversion of CO2 and inorganic nutrients into plant material, is the ultimate source of the food protein necessary to man's existence. Given certain assumptions, the overall maximal theoretical photosynthetic efficiency of agricultural plants can be calculated. Actual measured maximal growth rates of plants are equivalent to efficiency levels well below that theoretical maximum. In air, <span class="gene">C4</span> plants can some closer to the theoretical value than <span class="gene">C3</span> plants, perhaps because <span class="gene">C4</span> plants avoid the occurrence of measurable photorespiration and oxygen inhibition of photosynthesis. Alfalfa, a <span class="gene">C3</span> legume, is an extremely productive protein source. Its protein <span class="trait">yield</span> per acre can surpass that of commonly grown <span class="gene">C4</span> crops (corn, sorghum) and <span class="gene">C3</span> seed crops (soybean, wheat, rice). Alfalfa leaf protein is of high nutritional quality and can apparently be used directly in the human diet, eliminating the protein loss involved in animal production. Plant protein productivity can be raised as part of an increase in overall crop <span class="trait">yield.</span> The growth of plants in atmospheres with elevated CO2 levels can result in increased yields. In <span class="gene">C3</span> plants this is due, at least in part, to the suppression of photorespiration and oxygen inhibition of photosynthesis. We have investigated the effect of CO2 concentration on alfalfa photosynthetic metabolism. Our results support the contention that alfalfa productivity can be increased by an environment of elevated CO2. A second approach toward increased plant protein productivity is through regulation of carbon flow during photosynthesis so as to increase portein production relative to that of other plant constituents. In particular, we have investigated whether ammonia (the form in which plants first incorporate nitrogen) can act to regulate leaf carbon metabolism. Our results indicate that NH4+, in part through stimulation of pyruvate kinase, brings about increased production of amino acids at the expense of sucrose production in alfalfa. The effect may be of considerable importance in the regulation of green leaf protein synthesis.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Studies on urinary kallikreins. I. Purification and characterization of human urinary kallikreins.</h2>
			<h3>PMID:1087637  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Matsuda Y, Miyazaki K, Moriya H, Fujimoto Y, Hojima Y, Moriwaki C.
				<br/> 
				年份：1976 Oct
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Human urinary kallikrein [EC 3.4.21.8] (HUK) was purified about 200-fold with an overall <span class="trait">yield</span> of 40 percent from crude powder by DEAE-cellulose chromatography, acetone fractionation, Sephadex G-100 gel filtration and DEAE-Sephadex A-50 chromatography. Its activity was 200 kallikrein units (KU) per A280. HUK from active fractions obtained by DEAE-Sephadex A-50 chromatography was separated into three active components showing isoelectric points of 3.9 (HUK-1), 4.0 (HUK-2), and 4.2 (HUK-3) by isoelectric focusing: each HUK component was homogeneous on disc electrophoresis. The approximate molecular weights of HUK-1, -2 and -3 were estimated to be 2.7 X 10(4), 2.7 X 10(4), and 2.9 X 10(4), respectively, by gel filtration on a Sephadex G-100 column. The optimum pH's of HUK-1, -2, and -3 in esterolytic action were found to be 8.0, 8.3, and 7.5, respectively, and they were fairly heat stable in comparison with other glandular kallikreins. The three components of HUK were weakly inhibited by Trasylol, but were not affected by soybean and ovomucoid trypsin inhibitors. They were strongly resistant to treatment with urea and weakly resistant to treatment with guanidine. The activation energies of HUK-1, -2, and -3 were found to by 1.17 X 10(4), 5.1 X 10(3), and 1.45 X 10(4) <span class="gene">cal</span> per mole, respectively. The Km values were estimated toward N-alpha-tosyl-L-arginine methyl ester (TAME), N-alpha-benozyl-L-arginine ethyl ester (BAEE), and N-alpha-benozyl-L-arginine methyl ester (BAME).
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Chemical-enzymatic insertion of an amino acid residue in the reactive site of soybean trypsin inhibitor (Kunitz).</h2>
			<h3>PMID:1252450  [Indexed for MEDLINE]</h3>
				

				
			<p>
				作者：Kowalski D, Laskowski M Jr.
				<br/> 
				年份：1976 Mar 23
				<br/>
			</p>
			<br/> 
			
			
			
				<h2>摘要</h2>
				
					<p>&nbsp;&nbsp;
						Modified (Arg63-Ile64 reactive-site peptide bond hydrolyzed) soybean trypsin inhibitor (Kunitz) with all reactive amino groups, except that of Ile64, protected was described in the preceding paper (Kowalski, D., and Laskowski, M., Jr. (1976), Biochemistry, preceding paper in this issue). Treatment of this inhibitor with tert-butyloxycarbonyl-Ala- and tert-butyloxycarbonyl-Ile-N-hydroxy-succinimide esters yields inactive endo-tert-butyloxycarbonyl-Ala63A-and endo-tert-butyloxycarbonyl-Ile63A-modified inhibitors. The tert-butyloxycarbonyl groups were removed by treatment of the proteins with trifluoroacetic acid. After renaturation and purification, the resultant endo-Ala63A- and endo-Ile63A-modified inhibitors co-electrophorese with modified inhibitor both on disc gels (pH 9.4) and sodium dodecyl sulfate gels (after reduction of disulfide bonds) and show end groups corresponding to the 63A residue. These derivatives fail to form stable complexes with trypsin, extending the previous observation (Kowalski, D., and Laskowski, M., Jr. (1972), Biochemistry 11, 3451) that acylation of the P1' residue in modified inhibitors leads to inactivation. However, the incubation of endo-Ala63A- and endo-Ile63A-modified inhibitors with trypsin at pH 6.5 leads to the synthesis of the Arg63-Ala63A and Arg63-Ile63A peptide bonds in 4% <span class="trait">yield.</span> This is very close to the <span class="trait">yield</span> anticipated from a semiquantitative theory for the value of the equilibrium constant for reactive-site peptide bond. An alternative chemical method of insertion is also described. Controlled treatment of modified inhibitor with the N-carboxyanhydride of Glu produced inactive endo-Glu63A-modified inhibitor. Incubation of this inactive derivative with trypsin at pH 6.5 leads to 16% synthesis of the Arg63-Glu63A peptide bond. The higher <span class="trait">yield</span> of single chain protein in this case is attributed to the influence of the negative charge of the Glu63A side chain. Thus, the insertion of an amino acid residue between the <span class="gene">P1</span> and P1' residues in soybean trypsin inhibitor (Kunitz) converts a trypsin inhibitor into a trypsin substrate.
					</p>
				
			

			
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Microarray analysis of iron deficiency <span class="trait">chlorosis</span> in near-isogenic soybean lines</h2>
			<h3>PMID:18154662</h3>
				
					<h3>PMCID:2253546</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib id="A1" contrib-type="author">  <name>  <surname> O'Rourke </surname>  <given-names> Jamie A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> utehawk@iastate.edu </email>  </contrib>  <contrib id="A2" contrib-type="author">  <name>  <surname> Charlson </surname>  <given-names> Dirk V </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> charlsond@gmail.com </email>  </contrib>  <contrib id="A3" contrib-type="author">  <name>  <surname> Gonzalez </surname>  <given-names> Delkin O </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> dogonzal@uiuc.edu </email>  </contrib>  <contrib id="A4" contrib-type="author">  <name>  <surname> Vodkin </surname>  <given-names> Lila O </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> l-vodkin@uiuc.edu </email>  </contrib>  <contrib id="A5" contrib-type="author">  <name>  <surname> Graham </surname>  <given-names> Michelle A </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> magraham@iastate.edu </email>  </contrib>  <contrib id="A6" contrib-type="author">  <name>  <surname> Cianzio </surname>  <given-names> Silvia R </given-names>  </name>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> scianzio@iastate.edu </email>  </contrib>  <contrib id="A7" contrib-type="author">  <name>  <surname> Grusak </surname>  <given-names> Michael A </given-names>  </name>  <xref ref-type="aff" rid="I6"> 6 </xref>  <email> mgrusak@bcm.tmc.edu </email>  </contrib>  <contrib id="A8" corresp="yes" contrib-type="author">  <name>  <surname> Shoemaker </surname>  <given-names> Randy C </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <xref ref-type="aff" rid="I5"> 5 </xref>  <email> rcsshoe@iastate.edu </email>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 12 </month>  <year> 2007 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Near isogenic soybean lines (NILs) were developed by the USDA in 1972 [ <xref ref-type="bibr" rid="B22"> 22 </xref> ] specifically for their response to Fe deficiency. Fe efficient PI 548533 (Clark) was crossed with Fe inefficient PI 54619 (T203). Progeny were selfed and resulting F2 plants were screened for Fe inefficiency. The Fe inefficient progeny were backcrossed to the Fe efficient PI 548533 for six generations [ <xref ref-type="bibr" rid="B22"> 22 </xref> ], resulting in an Fe inefficient plant with the Clark genetic background. The Fe inefficient isoline was released as PI 547430 (IsoClark) [ <xref ref-type="bibr" rid="B22"> 22 </xref> ]. Both the Fe efficient PI 548533 (Clark) and Fe inefficient PI 547430 (IsoClark) lines were grown in the Ames, Iowa USDA greenhouse under 16 hr photoperiods. Plants were germinated in sterile vermiculite with distilled deionized water. After one week they were transplanted into a DTPA nutrient buffered hydroponics system [ <xref ref-type="bibr" rid="B3"> 3 </xref> ] containing all minerals necessary for normal growth. Experimental 10L systems to induce Fe deficiency stress had 50 uM Fe(NO <sub> 3 </sub> ) <sub> 3 </sub> Fe levels while systems for the control experiment contained 100 uM Fe(NO <sub> 3 </sub> ) <sub> 3 </sub> . Additionally, each 10 L system contained 2 mM MgSO <sub> 4 </sub> *7H <sub> 2 </sub> O, 3 mM Mg(NO <sub> 3 </sub> ) <sub> 2 </sub> *6H <sub> 2 </sub> O, 2.5 mM KNO <sub> 3 </sub> , 1 mM CaCl <sub> 2 </sub> *2H <sub> 2 </sub> O, 4.0 mM Ca(NO <sub> 3 </sub> ) <sub> 2 </sub> *4H <sub> 2 </sub> O, 0.020 mM KH <sub> 2 </sub> PO <sub> 4 </sub> , 542.5 uM KOH, 217 uM DTPA, 1.52 uM MnCl <sub> 2 </sub> *4H <sub> 2 </sub> O, 4.6 uM ZnSO <sub> 4 </sub> *7H <sub> 2 </sub> O, 2 uM CuSO <sub> 4 </sub> *5H <sub> 2 </sub> O, 0.20 uM NaMoO <sub> 4 </sub> *2H <sub> 2 </sub> O, 1 uM CoSO <sub> 4 </sub> *7H <sub> 2 </sub> O, 1 uM NiSO <sub> 4 </sub> *6H <sub> 2 </sub> O, 10 uM H <sub> 3 </sub> BO <sub> 3 </sub> , and 20 mM HCO <sub> 3 </sub> . A pH of 7.8 was maintained by the aeration of a 3% CO <sub> 2 </sub> : air mixture. A supplemental nutrient solution containing 16 mM potassium phosphate, 0.287 mM boric acid and 355 mM ammonium nitrate was added daily to maintain proper plant nutrition. To ensure the <span class="trait">chlorosis</span> was due to Fe deficiency stress, A15, an Fe efficient plant, and T203, Fe inefficient plant, were included with each experimental replication. Plants were grown in the hydroponics system for two weeks, until they reached the <span class="gene">V3</span> stage [ <xref ref-type="bibr" rid="B63"> 63 </xref> ], at which point tissue was harvested for RNA extraction. This experiment was replicated three times, for three independent biological replicates, each with two technical replicates.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Plant Glutathione Biosynthesis: Diversity in Biochemical Regulation and Reaction Products</h2>
			<h3>PMID:22645536</h3>
				
					<h3>PMCID:3355797</h3>
					<h4>期刊:Frontiers in plant science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Galant </surname>  <given-names> Ashley </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Preuss </surname>  <given-names> Mary L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cameron </surname>  <given-names> Jeffrey C. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jez </surname>  <given-names> Joseph M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 8 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						These structure/function studies established the basis of glutathione versus homoglutathione specificity, but it remains unclear how other glutathione analogs are formed in different plants. Skipsey et al. ( <xref ref-type="bibr" rid="B59"> 2005 </xref> ) cloned and characterized five GS-like proteins from soybean, wheat, and corn noting that the soybean enzyme was an <span class="gene">hGS</span> and that all the other enzymes functioned as GS. Interestingly, their analysis demonstrated that different GS-like enzymes accepted various alternate nucleophiles, including d-serine, non-natural amino acids, and alpha-amino alcohols, to <span class="trait">yield</span> a range of glutathione variants, but did not produce either serine- or glutamate-substituted analogs. Although altered specificity in the <span class="gene">GS</span> reaction mechanism offers a direct route to analog synthesis, it is possible that other enzymes catalyze additional reactions that modify glutathione into the specialized peptides, like hydroxymethyl&#x02013;glutathione.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Conventional and Novel G&#x003b3; Protein Families Constitute the Heterotrimeric G-Protein Signaling Network in Soybean</h2>
			<h3>PMID:21853116</h3>
				
					<h3>PMCID:3154445</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Choudhury </surname>  <given-names> Swarup Roy </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bisht </surname>  <given-names> Naveen C. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn1">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Thompson </surname>  <given-names> Rheannon </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="author-notes" rid="fn2">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Todorov </surname>  <given-names> Oleg </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pandey </surname>  <given-names> Sona </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 13 </day>  <month> 7 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We also identified a homolog of <span class="gene">GmG&#x003b3;9</span> in the <italic> Arabidopsis </italic> genome at locus At5g20635. This gene has been recently described as a G&#x003b3; protein in <italic> Arabidopsis </italic> <xref ref-type="bibr" rid="pone.0023361-Chakravorty1"> [23] </xref> . Homologs of group III proteins are present in all plant species. The proteins also show some homology to a keratin associated protein present in mammals. Interestingly the homologs of group III proteins in rice which are named <italic> DEP1 </italic> and <italic> GS3 </italic> have been recently identified as major QTLs for grain size and <span class="trait">yield</span> determination <xref ref-type="bibr" rid="pone.0023361-Huang1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0023361-Mao1"> [25] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Rice has two proteins that show homology to group III proteins, <italic> DEP1 </italic> (dense and erect panicle 1) and <italic> GS3 </italic> (grain size 3), that have been isolated as major QTLs for seed size and <span class="trait">yield</span> <xref ref-type="bibr" rid="pone.0023361-Huang1"> [24] </xref> , <xref ref-type="bibr" rid="pone.0023361-Mao1"> [25] </xref> . Interestingly, the rice G&#x003b1; protein RGA1 is also involved in regulation of seed size <xref ref-type="bibr" rid="pone.0023361-UeguchiTanaka1"> [41] </xref> . The rice DEP1 and GS3 proteins have been described as novel proteins containing a TNFR motif, a transmembrane domain and proteins with homology to human keratin-associated protein. We also identified a TNFR motif in the soybean group III family proteins; however, using multiple transmembrane prediction programs including TMHMM ( <ext-link ext-link-type="uri" xlink:href="http://www.cbs.dtu.dk/services/TMHMM/"> http://www.cbs.dtu.dk/services/TMHMM/ </ext-link> ), HMMTOP ( <ext-link ext-link-type="uri" xlink:href="http://www.enzim.hu/hmmtop/"> http://www.enzim.hu/hmmtop/ </ext-link> ) and DAS ( <ext-link ext-link-type="uri" xlink:href="http://www.sbc.su.se/~miklos/DAS/"> http://www.sbc.su.se/~miklos/DAS/ </ext-link> ), we did not identify any transmembrane domains in the <span class="gene">GmG&#x003b3;8,</span> <span class="gene">GmG&#x003b3;9</span> or <span class="gene">GmG&#x003b3;10</span> proteins. Rice and <italic> Arabidopsis </italic> group III proteins are predicted to have a single transmembrane domain using DAS, but not with TMHMM or HMMTOP. Experimental verification of the presence of a transmembrane domain and any possible role it might play in localization and/or positioning of these at the plasma membrane will be needed to evaluate its importance. Interestingly, YFP-fused group III <span class="gene">GmG&#x003b3;</span> proteins, in addition to the peripheral YFP fluorescence, also showed small vesicle like structures which were very evident in <span class="gene">GmG&#x003b3;8.</span> These proteins might be localized to endosomes structures in addition to the plasma membrane. However, since these proteins are highly cysteine rich such structures could also be due to protein aggregate formation or self-interaction ( <xref ref-type="fig" rid="pone-0023361-g008"> Figure 8 </xref> ). Our data at this time cannot differentiate between these possibilities. Expression of proteins with native promoters in a protein null background will help decipher correct localization.
					</p>
				
					<p>&nbsp;&nbsp;
						Since the two rice homologs of group III genes DEP1 and GS3 are involved in grain size determination and <span class="trait">yield,</span> we focused on the expression pattern of soybean group III <italic> <span class="gene">GmG&#x003b3;</span> </italic> genes during seed development. Our data showed that of the three group III genes in soybean, <italic> <span class="gene">GmG&#x003b3;8</span> </italic> shows the most interesting expression pattern during seed development and germination ( <xref ref-type="fig" rid="pone-0023361-g004"> Figure 4 </xref> , <xref ref-type="fig" rid="pone-0023361-g005"> 5 </xref> ). The expression of this gene was highly up-regulated when seed is undergoing maturation (stages S7 onwards), whereas a sharp decrease was observed during seed germination. This gene could be a true functional homolog of the rice <italic> DEP1 </italic> or <italic> GS3 </italic> gene. Additionally the expression of this gene in seeds could be dependent on the endogenous ABA and/or GA concentration as the levels of both these hormones change significantly during seed maturation and germination. Interestingly, <italic> <span class="gene">GmG&#x003b3;5</span> </italic> also followed a similar expression profile where the expression was up-regulated during seed maturation and generally down-regulated during germination. These genes could be potential targets for manipulation to regulate soybean seed development. It was also obvious that this group of genes is highly variable functionally as <italic> <span class="gene">GmG&#x003b3;10</span> </italic> shows very little expression in seeds at all developmental stages and exhibits no change in expression profile during germination.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of genes associated with nitrogen-use efficiency by genome-wide transcriptional analysis of two soybean genotypes</h2>
			<h3>PMID:22029603</h3>
				
					<h3>PMCID:3210170</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Hao </surname>  <given-names> Qing N </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> haoqingnan@126.com </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A2">  <name>  <surname> Zhou </surname>  <given-names> Xin A </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> zhouocri@sina.com </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Sha </surname>  <given-names> Ai H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> aihuasha@163.com </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Wang </surname>  <given-names> Cheng </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> wangcheng1983@163.com </email>  </contrib>  <contrib contrib-type="author" id="A5">  <name>  <surname> Zhou </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> zhourong@oilcrops.cn </email>  </contrib>  <contrib contrib-type="author" id="A6">  <name>  <surname> Chen </surname>  <given-names> Shui L </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> chenshuilianjy@163.com </email>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Nitrogen assimilation is a fundamental biological process in plants. The assimilation of nitrogen has profound effects on plant productivity, biomass, and crop <span class="trait">yield,</span> and nitrogen deficiency can inhibit the formation of structural components. Some genes involved in nitrogen assimilation showed significant differential expressions in this study. For example, our DGE results indicated that seven genes encoding amino acid transporter proteins were differentially expressed: four genes were up-regulated <span class="gene">(Glyma06g09270,</span> <span class="gene">Glyma10g12290,</span> <span class="gene">Glyma11g11310,</span> <span class="gene">Glyma14g05890)</span> and three genes were down-regulated <span class="gene">(Glyma01g36590,</span> <span class="gene">Glyma13g44450,</span> <span class="gene">Glyma17g26590).</span> In addition, two genes encoding a glutamate synthase family protein <span class="gene">(Glyma02g35560)</span> and an asparagine synthetase <span class="gene">(Glyma14g37440)</span> were up-regulated; and one nitrate gene <span class="gene">(Glyma10g08730)</span> was down-regulated.
					</p>
				
					<p>&nbsp;&nbsp;
						Several lines of biochemical and genetic research indicate that reversible protein phosphorylation is involved in the regulation of plant stress responses to various environmental stimuli. Some protein kinases might be involved in the regulation of cell differentiation and N-metabolism in nitrogen-fixing filamentous cyanobacteria [ <xref ref-type="bibr" rid="B41"> 41 </xref> , <xref ref-type="bibr" rid="B42"> 42 </xref> ]. Wall-associated kinases are also involved in various processes in plants, including pathogen <span class="trait">resistance,</span> heavy-metal tolerance and organ development [ <xref ref-type="bibr" rid="B43"> 43 </xref> ]. Unfortunately, little is known about their function in tolerance to nutrient deficiency. Our DGE results indicated that two genes encoding wall associated kinases, <span class="gene">Glyma19g21700</span> and <span class="gene">Glyma19g21690,</span> were up-regulated under N-limited conditions. In addition, a gene encoding receptor-like kinase <span class="gene">(Glyma13g09810)</span> was differentially expressed between the two varieties under N-limited conditions. Recent studies revealed that higher plants also have genes encoding putative receptor kinases (receptor-like Kinases; RLKs). For instance, the completely sequenced Arabidopsis genome contains more than 500 genes encoding RLKs, suggesting that higher plants, like animals, use receptor kinase signaling widely to modulate expressions of genes in response to diverse stimuli. Some research indicated that receptor-like kinases (RLK) play important roles in plant growth and development as well as in hormone and stress responses [ <xref ref-type="bibr" rid="B44"> 44 </xref> ]. Therefore, we hypothesize that the <span class="gene">Glyma13g09810</span> gene might be important for adaptation to low-N conditions in soybean.
					</p>
				
					<p>&nbsp;&nbsp;
						In addition to the genes described above, several other transcript profiles were altered under low N conditions. For example, a gene encoding BURP domain protein <span class="gene">(Glyma04g35360)</span> was differentially expressed. Some reports suggest that genes from the BURP family may be crucial for responses and adaptations to stresses. All the members of this family were shown to be induced by at least one type of stress treatment, for example, <span class="trait">drought,</span> salt, cold, abscisic acid and nutrition, etc. [ <xref ref-type="bibr" rid="B45"> 45 </xref> ]. Therefore, the soybean BURP gene may be N responsive to N-stress. One gene encoding CBS domain-containing protein which was differentially expressed in two soybean varieties was also found. Previous research revealed that CBS domain-containing proteins play important roles in stress response/tolerance and development in plants [ <xref ref-type="bibr" rid="B46"> 46 </xref> ]. To determine whether this protein has the potential to improve tolerance of transgenic plants to low N-stress, its role in development and N stresses should be further investigated. In addition, some published results suggest that a phosphatase is involved in modulating phosphoinositide signals during the stress response [ <xref ref-type="bibr" rid="B47"> 47 </xref> ]. This results showed that one gene <span class="gene">(Glyma10g38990)</span> putatively encoding a phosphoinositide binding protein was up-regulated. We suggest that this gene may function as a component of a stress response pathway that protects the plant against the effects of N-deficiency.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Genomic Analysis of Soybean Flowering Genes</h2>
			<h3>PMID:22679494</h3>
				
					<h3>PMCID:3367986</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jung </surname>  <given-names> Chol-Hee </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wong </surname>  <given-names> Chui E. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Mohan B. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bhalla </surname>  <given-names> Prem L. </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 5 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Flowering is an important agronomic trait that determines crop <span class="trait">yield.</span> Soybean is a major oilseed legume crop used for human and animal feed. Legumes have unique vegetative and floral complexities. Our understanding of the molecular basis of flower initiation and development in legumes is limited. Here, we address this by using a computational approach to examine flowering regulatory genes in the soybean genome in comparison to the most studied model plant, Arabidopsis. For this comparison, a genome-wide analysis of orthologue groups was performed, followed by an <italic> in silico </italic> gene expression analysis of the identified soybean flowering genes. Phylogenetic analyses of the gene families highlighted the evolutionary relationships among these candidates. Our study identified key flowering genes in soybean and indicates that the vernalisation and the ambient-temperature pathways seem to be the most variant in soybean. A comparison of the orthologue groups containing flowering genes indicated that, on average, each Arabidopsis flowering gene has 2-3 orthologous copies in soybean. Our analysis highlighted that the <italic> CDF3 </italic> , <italic> <span class="gene">VRN1</span> </italic> , <italic> SVP </italic> , <italic> AP3 </italic> and <italic> PIF3 </italic> genes are paralogue-rich genes in soybean. Furthermore, the genome mapping of the soybean flowering genes showed that these genes are scattered randomly across the genome. A paralogue comparison indicated that the soybean genes comprising the largest orthologue group are clustered in a 1.4 Mb region on chromosome 16 of soybean. Furthermore, a comparison with the undomesticated soybean ( <italic> Glycine soja </italic> ) revealed that there are hundreds of SNPs that are associated with putative soybean flowering genes and that there are structural variants that may affect the genes of the light-signalling and ambient-temperature pathways in soybean. Our study provides a framework for the soybean flowering pathway and insights into the relationship and evolution of flowering genes between a short-day soybean and the long-day plant, Arabidopsis.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Metabolite Adjustments in <span class="trait">Drought</span> Tolerant and Sensitive Soybean Genotypes in Response to Water Stress</h2>
			<h3>PMID:22685583</h3>
				
					<h3>PMCID:3369847</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Silvente </surname>  <given-names> Sonia </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sobolev </surname>  <given-names> Anatoly P. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lara </surname>  <given-names> Miguel </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 8 </day>  <month> 5 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						(a&#x02013;d) trifoliate leaves and (e&#x02013;h) nodules. (a, e) dry weight (DW); (b) relative water content (RWC), (f) nitrogenase activity, (c, g) ureide levels and (d, h) proline levels in 17-day-old plants subjected to 10 days water sufficient and deficient conditions. TC: Tolerant genotype Control; TD: Tolerant subjected to <span class="trait">Drought,</span> SC: Sensitive genotype Control; SD: Sensitive subjected to <span class="trait">Drought.</span> Data represent the mean &#x000b1; standard deviation (SD) of five to 15 replicates. Different letters at the top of each <span class="gene">bar</span> indicate significant differences at <italic> P </italic> &#x0003c;0.01.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcript Profile of the Response of Two Soybean Genotypes to Potassium Deficiency</h2>
			<h3>PMID:22792192</h3>
				
					<h3>PMCID:3390323</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wang </surname>  <given-names> Cheng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Chen </surname>  <given-names> HaiFeng </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hao </surname>  <given-names> QingNan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sha </surname>  <given-names> AiHua </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shan </surname>  <given-names> ZhiHui </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> LiMiao </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhi </surname>  <given-names> HaiJian </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> XinAn </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> &#x0002a; </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 5 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						One other type of high-level differentially expressed gene related to low-K stress conditions was shown. After analysis of the DEGs, five genes encoding the ATP binding cassette (ABC) transporter family were found to be differentially expressed <span class="gene">(Glyma09g04980,</span> <span class="gene">Glyma05g32620,</span> <span class="gene">Glyma08g00280,</span> <span class="gene">Glyma08g17110,</span> and <span class="gene">Glyma08g10710).</span> Four defense response genes were also identified, those encoding abscisic stress response protein <span class="gene">(Glyma20g30720),</span> NBS-LRR type disease <span class="trait">resistance</span> proteins <span class="gene">(Glyma12g36510),</span> wound-induced basic proteins <span class="gene">(Glyma08g46240),</span> and salt-tolerance protein <span class="gene">(Glyma12g05570).</span> In addition, 16 genes related to oxidoreductase activity were identified, those encoding alcohol dehydrogenase <span class="gene">(Glyma08g39520),</span> NADPH-specific isocitrate dehydrogenase <span class="gene">(Glyma14g39160),</span> a short-chain dehydrogenase <span class="gene">(Glyma16g04630),</span> fatty acid desaturase <span class="gene">(Glyma01g29630),</span> and others. In addition, two gibberellin-responsive protein genes were found to be up-regulated under low-K stress conditions <span class="gene">(Glyma17g36220-4.11-fold,</span> <span class="gene">Glyma06g12700-2.34-fold).</span> One gene encoding a BURP domain protein <span class="gene">(Glyma14g20440)</span> and ten encoding dehydration-responsive binding proteins were differentially expressed. The expression of <span class="gene">Glyma09g08330</span> and <span class="gene">Glyma15g19910</span> was found to be up-regulated. The others were down-regulated.
					</p>
				
					<p>&nbsp;&nbsp;
						We also noticed a significant phenomenon involving several transcription factors. A single transcription factor can regulate the expression of many genes. Transcriptional control of the expression of stress-response genes is a crucial part of the plant response to a range of abiotic and biotic stresses <xref ref-type="bibr" rid="pone.0039856-Singh1"> [68] </xref> . This means that transcription factors can show some complexity and overlap in their responses to different stressors and are likely to lead to new ways of enhancing crop tolerance to environmental stress <xref ref-type="bibr" rid="pone.0039856-Chen2"> [69] </xref> . Some transcription factors are related to the development of potassium-related metabolic pathways <xref ref-type="bibr" rid="pone.0039856-Young1"> [70] </xref> . For example, the expression level of the <span class="gene">Glyma04g08060</span> (WRKY50) (2.97-fold) gene was up-regulated in the low-K-tolerant variety over the short term. <span class="gene">WRKY50</span> is a defense-related transcription factor belonging to the WRKY family of proteins <xref ref-type="bibr" rid="pone.0039856-Eulgem1"> [46] </xref> . Armengaud <italic> et al </italic> . pointed out that increased jasmonic acid (JA) levels in K <sup> &#x0002b; </sup> -starved plants might repress otherwise inducible defense responses <xref ref-type="bibr" rid="pone.0039856-Armengaud2"> [71] </xref> . Although <span class="gene">WRKY50</span> proteins have not been reported to regulate the expression of JA signaling components, the absence of these proteins relieves this repression and restores JA-derived signaling in plants <xref ref-type="bibr" rid="pone.0039856-Gao1"> [72] </xref> . In this way, <span class="gene">WRKY50</span> may have positive effects under low-K stress conditions. In our study, several AP2/EREBP (apetala 2-ethylene-responsive element binding proteins) transcription factors were found to be among our DEG. The AP2/EREBP family proteins are unique to plants and share a highly conserved region of about 60 to 70 amino acids <xref ref-type="bibr" rid="pone.0039856-Riechmann1"> [44] </xref> . Many AP2/EREBP transcription factors can activate genes in response to abiotic stress and are involved in the ABA response <xref ref-type="bibr" rid="pone.0039856-Kizis1"> [43] </xref> , <xref ref-type="bibr" rid="pone.0039856-Becker1"> [73] </xref> . The expression levels of these genes were up-regulated in the roots and shoots of the low-K-sensitive plant varieties. We assume that these genes can indirectly regulate the potassium channel involved in K-metabolic pathways in soybeans. The function of this gene will be studied by RNA-interference or by over-expression in transgenic plants in the future. In addition, we found many members of the MYB transcription factor family to be differentially expressed between two varieties (i.e., <span class="gene">Glyma11g22960,</span> <span class="gene">Glyma07g05960,</span> etc.). Some reports have suggested that these genes regulate a wide array of processes in plants, including the synthesis of secondary metabolites, different aspects of plant development, abiotic stress responses, and other processes <xref ref-type="bibr" rid="pone.0039856-Stracke1"> [41] </xref> . This indicates that the soybean MYB family genes may be responsive to K-stress. One gene encoding <span class="gene">GATA</span> transcription factors <span class="gene">(Glyma11g04060)</span> was found to be up-regulated (3.51-fold) in both soybean varieties. Many <span class="gene">GATA</span> factors can activate or inactivate genes in response to environmental deficiencies <xref ref-type="bibr" rid="pone.0039856-Crespo1"> [74] </xref> . For this reason, we assume that this gene is involved in low K- <span class="trait">resistance</span> in soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						We also found many protein kinases to be differentially expressed. Among these, plant Ca2 <sup> &#x0002b; </sup> sensors. These included calmodulin (CaM), calcineurin B-like proteins (CBLs), CBL-interacting protein kinases (CIPKs), and Ca2 <sup> &#x0002b; </sup> -dependent protein kinases (CDPKs). CDPKs are Ca <sup> 2&#x0002b; </sup> sensors unique to higher plants. They are structurally different from Ca <sup> 2&#x0002b; </sup> -regulated protein kinases in other species <xref ref-type="bibr" rid="pone.0039856-Wan1"> [75] </xref> . The CDPKs have been proven to be involved in regulating plant responses to environmental stresses in different species <xref ref-type="bibr" rid="pone.0039856-Saijo1"> [76] </xref> , <xref ref-type="bibr" rid="pone.0039856-Abbasi1"> [77] </xref> . CIPK23 has been shown to directly phosphorylate the K <sup> &#x0002b; </sup> transporter AtAKT1. CIPK23 is activated by the binding of two calcineurin B-like proteins, CBL1 and CBL9 <xref ref-type="bibr" rid="pone.0039856-Xu1"> [78] </xref> . The CBL1/9&#x02013;CIPK23 pathway efficiently improves the activity of AKT1 and enhances K <sup> &#x0002b; </sup> uptake under low-K stress conditions. The tag representing the CBL1/9 gene <span class="gene">(Glyma05g05580-2.10-fold),</span> CIPK23 gene <span class="gene">(Glyma14g04430-1.83-fold),</span> and CDPKs <span class="gene">(Glyma14g02680-16.23-fold,</span> <span class="gene">Glyma04g38150-2.31-fold,</span> and <span class="gene">Glyma14g00320-2.47-fold)</span> was found to be up-regulated in both varieties. We hypothesize that these genes might make an important contribution to adaptation to low-K stress conditions in soybeans. In addition, wall-associated kinases are also involved in various processes in plants, including pathogen <span class="trait">resistance,</span> heavy-metal tolerance, and organ development <xref ref-type="bibr" rid="pone.0039856-Kanneganti1"> [79] </xref> . However, their role in tolerance to potassium deficiency is only minimally understood. In our DGE results, three genes encoding the wall-associated kinases were found to be up-regulated under low-K stress conditions. Meanwhile, two genes encoding MAP-kinase <span class="gene">(Glyma17g06020,</span> <span class="gene">Glyma15g18860)</span> were up-regulated in the low-K-tolerant variety. Recent investigations have confirmed major roles for defined MAPK pathways in plant development, cell proliferation, and hormone physiology, and both biotic and abiotic stress signaling <xref ref-type="bibr" rid="pone.0039856-Nakagami1"> [80] </xref> . These genes might play an important role in plant growth and development as well as in hormone and stress responses, and might be important for adaptation to low-K stress condition in soybeans.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of QTL controlling post-flowering period in soybean</h2>
			<h3>PMID:23136504</h3>
				
					<h3>PMCID:3406778</h3>
					<h4>期刊:Breeding Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Komatsu </surname>  <given-names> Kunihiko </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-646"> 1 </xref>  <xref rid="c1-bs-61-646" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hwang </surname>  <given-names> Tae-Young </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-646"> 1 </xref>  <xref rid="fn2-bs-61-646" ref-type="author-notes"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Takahashi </surname>  <given-names> Masakazu </given-names>  </name>  <xref ref-type="aff" rid="af2-bs-61-646"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sayama </surname>  <given-names> Takashi </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-646"> 1 </xref>  <xref ref-type="aff" rid="af3-bs-61-646"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Funatsuki </surname>  <given-names> Hideyuki </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-646"> 1 </xref>  <xref rid="fn3-bs-61-646" ref-type="author-notes"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oki </surname>  <given-names> Nobuhiko </given-names>  </name>  <xref ref-type="aff" rid="af2-bs-61-646"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ishimoto </surname>  <given-names> Masao </given-names>  </name>  <xref ref-type="aff" rid="af1-bs-61-646"> 1 </xref>  <xref ref-type="aff" rid="af3-bs-61-646"> 3 </xref>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 10 </month>  <year> 2011 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean ( <italic> Glycine max </italic> (L.) Merr) is a short-day crop and its life cycle is largely determined by variety-specific daylength requirements for the initiation of floral development. The life span of each variety is a principal factor for <span class="trait">yield</span> potential in different regions, because the suitable season for soybean production is dependent on latitude, temperature, and other environmental factors ( <xref rid="b13-bs-61-646" ref-type="bibr"> Heatherly and Elmore 2004 </xref> ). Genetic factors controlling the developmental phases, such as flowering and maturation, have been comprehensively investigated in soybean, and several qualitative trait loci have been identified as a series <italic> E </italic> loci; <italic> <span class="gene">E1</span> </italic> and <italic> E2 </italic> ( <xref rid="b1-bs-61-646" ref-type="bibr"> Bernard 1971 </xref> ), <italic> E3 </italic> ( <xref rid="b3-bs-61-646" ref-type="bibr"> Buzzell 1971 </xref> ), <italic> E4 </italic> ( <xref rid="b4-bs-61-646" ref-type="bibr"> Buzzell and Voldeng 1980 </xref> ), <italic> E5 </italic> ( <xref rid="b22-bs-61-646" ref-type="bibr"> McBlain and Bernard 1987 </xref> ), <italic> E6 </italic> ( <xref rid="b2-bs-61-646" ref-type="bibr"> Bonato and Vello 1999 </xref> ), <italic> E7 </italic> ( <xref rid="b9-bs-61-646" ref-type="bibr"> Cober and Voldeng 2001 </xref> ) and <italic> E8 </italic> ( <xref rid="b10-bs-61-646" ref-type="bibr"> Cober <italic> et al. </italic> 2010 </xref> ). Most of these have been identified as factors affecting the flowering time, as hereafter defined as DSF; <underline> d </underline> uration from <underline> s </underline> owing to <underline> f </underline> lowering (R1 stage described by <xref rid="b12-bs-61-646" ref-type="bibr"> Fehr <italic> et al. </italic> [1977] </xref> ), or growing period from sowing to R8 stage (DSM; <underline> d </underline> uration from <underline> s </underline> owing to <underline> m </underline> aturation), and the effect of each locus has been confirmed in further genetic analyses, except for <italic> E6 </italic> and <italic> E7 </italic> . The effects of <italic> <span class="gene">E1</span> </italic> to <italic> E5 </italic> on DSF and DSM have been assessed in near-isogenic lines developed by recurrent backcrossing ( <xref rid="b7-bs-61-646" ref-type="bibr"> Cober <italic> et al. </italic> 1996 </xref> , <xref rid="b9-bs-61-646" ref-type="bibr"> 2001 </xref> , <xref rid="b23-bs-61-646" ref-type="bibr"> McBlain <italic> et al. </italic> 1987 </xref> , <xref rid="b24-bs-61-646" ref-type="bibr"> Messina <italic> et al. </italic> 2006 </xref> , <xref rid="b27-bs-61-646" ref-type="bibr"> Saindon <italic> et al. </italic> 1989 </xref> ), and these studies confirmed that the dominant alleles of these loci prolong DSM as well as DSF in response to photoperiod. The effect of <italic> E8 </italic> on DSM was also confirmed using near-isogenic lines ( <xref rid="b10-bs-61-646" ref-type="bibr"> Cober <italic> et al. </italic> 2010 </xref> ). In addition, it has been revealed that most of the <italic> E </italic> series genes affect the reproductive period (post-flowering period) as well as DSF and DSM. It was reported that the dominant alleles of <italic> E2 </italic> to <italic> E4 </italic> prolonged the reproductive period from stage R1 to R8 (DFM; <underline> d </underline> uration from <underline> f </underline> lowering to <underline> m </underline> aturation) with different degrees ( <xref rid="b23-bs-61-646" ref-type="bibr"> McBlain <italic> et al. </italic> 1987 </xref> , <xref rid="b27-bs-61-646" ref-type="bibr"> Saindon <italic> et al. </italic> 1989 </xref> ). <xref rid="b19-bs-61-646" ref-type="bibr"> Kumudini <italic> et al. </italic> (2007) </xref> investigated the duration from stage R1 to R5 of near-isogenic lines of <italic> <span class="gene">E1</span> </italic> to <italic> E4 </italic> and <italic> E7 </italic> under different day-length conditions, and estimated that the <italic> E </italic> series genes play a key role in photoperiod mediated control of the duration, though the respective effect of each gene was not clarified. Summarizing these studies, <italic> E </italic> loci seem to determine DSM through the regulation both of DFM and DSF.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Uncovering the Salt Response of Soybean by Unraveling Its Wild and Cultivated Functional Genomes Using Tag Sequencing</h2>
			<h3>PMID:23209559</h3>
				
					<h3>PMCID:3509101</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ali </surname>  <given-names> Zulfiqar </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Da Yong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Zhao Long </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ling </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yi </surname>  <given-names> Jin Xin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Xiao Lan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Yi Hong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xiao Qing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Asif Ali </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Trethowan </surname>  <given-names> Richard M. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Hong Xiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 10 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Around 136 different genes were annotated which expressed specifically in STGoGS (see <xref ref-type="supplementary-material" rid="pone.0048819.s013"> Table S8 </xref> ). Among them, 14 genes were stress related ( <xref ref-type="table" rid="pone-0048819-t003"> Table 3 </xref> , also see <xref ref-type="supplementary-material" rid="pone.0048819.s014"> Table S9 </xref> ). Significantly up-regulated genes were cation diffusion facilitator 9 (gnl|UG|Gma#S53090447), hydrolase (gnl|UG|Gma#S48316466), potassium channel tetramerization domain-containing protein (gnl|UG|Gma#S48316331), protein phosphatase 2C (gnl|UG|Gma#S48316268), abscisic acid 8&#x02032;-hydroxylase (gnl|UG|Gma#S48312398), multidrug <span class="trait">resistance</span> protein ABC transporter (gnl|UG|Gma#S48316693), and stress-related protein (gnl|UG|Gma#S48315849). Significantly down-regulated stress related genes included chitinase class I (gnl|UG|Gma#S5146241), tubulin A (gnl|UG|Gma#S48314982), glutathione S-transferase <span class="gene">GST</span> 19 (gnl|UG|Gma#S5146336), putative protease inhibitor (gnl|UG|Gma#S19677361), cytochrome <span class="gene">P450</span> mono-oxygenase <span class="gene">CYP81E11</span> (gnl|UG|Gma#S30676931), tubulin beta-1 chain (gnl|UG|Gma#S48316404), and hypothetical protein (gnl|UG|Gma#S52651928). About 329 genes with unknown functional annotation were found in STGoGS under salt stress. Among these 24 genes were up-regulated and 19 down-regulated in salt stress with a difference in their transcripts per million (TPM) &#x02265;100 from control. Of 151 annotated commonly expressed genes in both species, five belonged to calmodulin, five to glutathione S-transferase and six to the Cytochrome <span class="gene">P450</span> gene families. Among the remaining 135 miscellaneous genes, 37 were related to stresses. About 439 genes with unknown functional annotation were also identified in both species when grown under salt stress. Among them 55 genes appeared as interesting unknown genes present in both species.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Waterlogging Tolerance of Crops: Breeding, Mechanism of Tolerance, Molecular Approaches, and Future Prospects</h2>
			<h3>PMID:23484164</h3>
				
					<h3>PMCID:3591200</h3>
					<h4>期刊:BioMed Research International</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ahmed </surname>  <given-names> F. </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 14 </day>  <month> 11 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Flooding is a natural calamity that destroys production of rice throughout the world especially in low-laying areas, but all economically important varieties are not tolerant to flooding. Identification of the <span class="gene">SUB1</span> QTL enables the scientist to introgress this gene by MAB into the popular high-yielding varieties [ <xref rid="B87" ref-type="bibr"> 86 </xref> , <xref rid="B88" ref-type="bibr"> 87 </xref> ]. Mapping of the Sub1 allele reveals SNP (single-nucleotide polymorphisms) within <italic> Sub1A </italic> and <italic> Sub1C, which </italic> is useful for the breeders for molecular markers and in precision breeding [ <xref rid="B89" ref-type="bibr"> 88 </xref> , <xref rid="B90" ref-type="bibr"> 89 </xref> ]. By the use of MAB, <italic> Sub1A </italic> has been already inserted to modern varieties of different countries; BR11, Swarna, Samba Mahsuri IR64, CR1009 and Thadokkam 1 (TDK1) are some of the examples of those varieties [ <xref rid="B89" ref-type="bibr"> 88 </xref> ]. SSR markers, which were polymorphic between two parents, were generally used for the background conformation or recurrent parent genome conformation also well combined with the Sub1 region originated from FR13A on chromosome 9. Normally newly developed Sub1 lines shows more significant waterlogging tolerance compared to parents [ <xref rid="B89" ref-type="bibr"> 88 </xref> , <xref rid="B91" ref-type="bibr"> 90 </xref> ] which is find out by evaluation. These studies show the opportunities of the insertion of the Sub1 region from developed tolerant variety through MAB to produce tolerant varieties with a diverse genetic background. Furthermore, the effect of <italic> Sub1 </italic> on other agronomical characteristics of plant such as, grain quality, growth, maturation, and grain production was determined in IR64 Sub1, Swarna Sub1, and Samba Mahsuri Sub1. Further research of the above mentioned submergence tolerant varieties and their original parents revealed that insertion of <italic> Sub1 </italic> gene does not alter field performances including quality and <span class="trait">yield</span> of grain under normal growth condition [ <xref rid="B80" ref-type="bibr"> 83 </xref> , <xref rid="B90" ref-type="bibr"> 89 </xref> , <xref rid="B92" ref-type="bibr"> 91 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A meta-analysis of responses of canopy photosynthetic conversion efficiency to environmental factors reveals major causes of <span class="trait">yield</span> <span class="gene">gap</span></h2>
			<h3>PMID:23873996</h3>
				
					<h3>PMCID:3745731</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Slattery </surname>  <given-names> Rebecca A. </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						where <span class="trait">yield</span> potential (Y) is dependent upon the amount of incident solar radiation (S <sub> t </sub> ) received over the growing season, the proportion of S <sub> t </sub> that is photosynthetically active radiation (PAR; estimated to be 0.487), the radiation interception efficiency of the plant (&#x003b5; <sub> i </sub> ), the conversion efficiency of intercepted radiation into biomass (&#x003b5; <sub> c </sub> ), and the partition efficiency of biomass into harvestable product (&#x003b5; <sub> p </sub> ). The quantity of S <sub> t </sub> during a growing season for a specific crop is largely predetermined by latitude, altitude, and time of year in a particular location, and the percentage that is PAR varies by location and can fluctuate within a day due to solar elevation and cloudiness ( <xref rid="CIT0037" ref-type="bibr"> Monteith, 1965 </xref> , <xref rid="CIT0038" ref-type="bibr"> 1972 </xref> ). Since the middle of the 20th century, &#x003b5; <sub> i </sub> and &#x003b5; <sub> p </sub> have been vastly improved due to management and breeding practices introduced during the Green Revolution, thereby closing the <span class="gene">gap</span> between actual and theoretical values ( <xref rid="CIT0017" ref-type="bibr"> Evans, 1993 </xref> ; <xref rid="CIT0024" ref-type="bibr"> Hay, 1995 </xref> ; <xref rid="CIT0054" ref-type="bibr"> Sinclair, 1998 </xref> ). The greatest opportunity for <span class="trait">yield</span> potential improvement therefore lies in increasing &#x003b5; <sub> c </sub> ( <xref rid="CIT0062" ref-type="bibr"> Zhu <italic> et al. </italic> , 2010 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Elevated [CO <sub> 2 </sub> ] increased &#x003b5; <sub> c </sub> by 20% overall ( <xref ref-type="fig" rid="F1"> Fig. 1 </xref> ), which was similar to a reported 17% mean increase in <span class="trait">yield</span> due to elevated [CO <sub> 2 </sub> ] in FACE studies ( <xref rid="CIT0004" ref-type="bibr"> Ainsworth and Long, 2005 </xref> ). The mean stimulation of &#x003b5; <sub> c </sub> was 2-fold greater for [CO <sub> 2 </sub> ] enrichment &#x0003e;599 ppm compared to enrichments of 450&#x02013;599 ppm ( <xref ref-type="fig" rid="F1"> Fig. 1 </xref> ). &#x003b5; <sub> c </sub> stimulation was significantly affected by photosynthetic type with a 5-fold greater stimulation in C <sub> 3 </sub> compared to C <sub> 4 </sub> plants ( <xref ref-type="fig" rid="F1"> Fig. 1 </xref> ). This agreed with previous meta-analyses with a greater response in C <sub> 3 </sub> versus C <sub> 4 </sub> plants to elevated [CO <sub> 2 </sub> ] in variables related to &#x003b5; <sub> c </sub> such as A <sub> <span class="gene">sat</span> </sub> and biomass accumulation ( <xref rid="CIT0060" ref-type="bibr"> Wand <italic> et al. </italic> , 1999 </xref> ; <xref rid="CIT0004" ref-type="bibr"> Ainsworth and Long, 2005 </xref> ). However, the small sample size prevents strong conclusions in this area. Based on these findings, <span class="trait">yield</span> potential will increase with increasing [CO <sub> 2 </sub> ], which will beneficially widen the <span class="trait">yield</span> <span class="gene">gap</span> to allow for greater rates of increase in average yields. Engineering higher capacity forms of Rubisco, greater electron transport capacity, and stronger sink capacity will further enhance the responsiveness to elevated [CO <sub> 2 </sub> ] ( <xref rid="CIT0006" ref-type="bibr"> Ainsworth <italic> et al. </italic> , 2008 </xref> ) and therefore overall <span class="trait">yield</span> potential.
					</p>
				
					<p>&nbsp;&nbsp;
						Using a novel approach to a meta-analytical study, this study statistically quantified how and to what extent &#x003b5; <sub> c </sub> is sensitive to several climate/weather factors and management practices that affect the ability of crops to close the <span class="trait">yield</span> <span class="gene">gap</span> between average and potential yields. Elevated [CO <sub> 2 </sub> ] increases &#x003b5; <sub> c </sub> and therefore raises the ceiling on <span class="trait">yield</span> potential, but using biotechnology to engineer for greater responsiveness to elevated [CO <sub> 2 </sub> ] will result in more substantial increases. Although shade also increases &#x003b5; <sub> c </sub> , the fact that it does so by decreasing S <sub> t </sub> forecasts that it will not contribute as directly as elevated [CO <sub> 2 </sub> ] to increasing <span class="trait">yield</span> potentials unless excess light is used more efficiently through intercropping, modified canopy architecture, or reduced chlorophyll content. The greatest &#x003b5; <sub> c </sub> response was seen with large quantities of added nitrogen, which also means severe decreases in &#x003b5; <sub> c </sub> could occur when nitrogen is limiting. Therefore, engineering plants for greater response to nitrogen applications is crucial to prevent decreases in &#x003b5; <sub> c </sub> and average yields. This is especially important in nutrient-poor areas predicted to suffer from worsening <span class="trait">drought</span> conditions since substantial decreases in legume productivity due to <span class="trait">drought</span> will negatively impact soil nitrogen availability for subsequent crops. Elevated [O <sub> 3 </sub> ], water stress, temperature stress, and foliar damage depress measured yields and prevent <span class="gene">gap</span> closure, and these effects warrant advanced engineering for stress tolerance and agronomical practices such as intercropping to decrease pest and disease damage. Further experimentation to determine optimal practices is warranted, but growth condition analyses emphasize the importance of obtaining estimates of &#x003b5; <sub> c </sub> in field conditions for reliable results. Overall, these results indicate the significant extent to which environmental factors vary &#x003b5; <sub> c </sub> and support the concept that &#x003b5; <sub> c </sub> can be improved to increase <span class="trait">yield</span> potential and raise the ceiling to allow for greater rates of increase while closing the <span class="trait">yield</span> gap.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biotechnological potential of rhizobial metabolites to enhance the performance of <italic> Bradyrhizobium </italic> spp. and <italic> Azospirillum brasilense </italic> inoculants with soybean and maize</h2>
			<h3>PMID:23594921</h3>
				
					<h3>PMCID:3642020</h3>
					<h4>期刊:AMB Express</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" id="A1">  <name>  <surname> Marks </surname>  <given-names> Bettina Berqu&#x000f3; </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> bettinamarks@gmail.com </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Meg&#x000ed;as </surname>  <given-names> Manuel </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> megiasg@us.es </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Nogueira </surname>  <given-names> Marco Antonio </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> marco.nogueira@embrapa.br </email>  </contrib>  <contrib contrib-type="author" corresp="yes" id="A4">  <name>  <surname> Hungria </surname>  <given-names> Mariangela </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> mariangela.hungria@embrapa.br </email>  </contrib>
				<br/> 
				年份：<day> 6 </day>  <month> 4 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Lipo-chitooligosaccharides affect a number of physiological processes in the legume host plant, including root-hair curling and stimulation of division of cortical cells (Schultze and Kondorosi <xref ref-type="bibr" rid="B54"> 1996 </xref> ; Hungria and Stacey <xref ref-type="bibr" rid="B35"> 1997 </xref> ; Perret et al. <xref ref-type="bibr" rid="B49"> 2000 </xref> ; Oldroyd and Downie <xref ref-type="bibr" rid="B44"> 2008 </xref> ; Ferguson et al. <xref ref-type="bibr" rid="B26"> 2010 </xref> ). It has also been demonstrated that LCOs effects resemble those of cytokinins, and of the inhibitors of the transport of auxins (Relic et al. <xref ref-type="bibr" rid="B50"> 1993 </xref> ); interestingly, they also activate enzymes related to plant defense (Inui et al. <xref ref-type="bibr" rid="B36"> 1997 </xref> ). Finally, other benefits attributed to LCOs require further exploration, e.g., increases in seed germination (Miransari and Smith <xref ref-type="bibr" rid="B42"> 2009 </xref> ). We attribute the effects observed in our study to LCOs; they are consistent with the activity of these compounds at concentrations as low as 10 <sup> &#x02013;12 </sup> &#x000a0;M (Hungria and Stacey <xref ref-type="bibr" rid="B35"> 1997 </xref> ). Activity at such low concentrations may be responsible for the differences observed between greenhouse and field experiments. In addition, it is known that the LCOs of <italic> B. japonicum </italic> are very specific, with a methyl-fucose moiety under the control of <italic> <span class="gene">nodZ</span> </italic> (L&#x000f3;pez-Lara et al. <xref ref-type="bibr" rid="B40"> 1996 </xref> ), which may explain the responses exclusively to homologous metabolites. Finally, in our study the addition of a plant molecular signal (genistein) was not as successful as the addition of LCOs, although the concentration applied (5&#x000a0;&#x003bc;M) was much lower than doses previously reported for soybean and common bean (40&#x000a0;&#x003bc;M) (Hungria and Stacey <xref ref-type="bibr" rid="B35"> 1997 </xref> ). The effects of adding CM to the maize crop surpassed those observed with soybean. Considering three field experiments, increases in grain <span class="trait">yield</span> by the addition of CM-RT1 at the lower dose (10 <sup> &#x02013;9 </sup> &#x000a0;M) were of 11.4%. The increases may be attributable to EPSs and plant hormones, and in the latter case, it is known that the effects are strongly dependent on concentration, and can be inhibitory at higher concentrations (Arshad and Frankenberger <xref ref-type="bibr" rid="B4"> 1991 </xref> ), which may explain the better performance at the lower concentration of CM. In addition, benefits can also be attributed to the LCOs, which may activate cell division in non-leguminous plants, such as tobacco ( <italic> Nicotiana </italic> sp.) (Baier et al. <xref ref-type="bibr" rid="B7"> 1999 </xref> ), tomato ( <italic> Solanum lycopersicum </italic> ) (Staehelin et al. <xref ref-type="bibr" rid="B57"> 1994 </xref> ), carrot ( <italic> Dacus carota </italic> ) (De Jong et al. <xref ref-type="bibr" rid="B17"> 1993 </xref> ), and also maize (Khana et al. <xref ref-type="bibr" rid="B38"> 2008 </xref> ). Often, the effects mimic those of plant hormones, such as cytokinins and auxins (Dyachok et al. <xref ref-type="bibr" rid="B21"> 2000 </xref> ). Interestingly, under greenhouse conditions Souleimanov et al. ( <xref ref-type="bibr" rid="B56"> 2002 </xref> ) reported increases in soybean and maize biomass seven days after the addition of LCO (at 10 <sup> &#x02013;7 </sup> &#x000a0;M) of <italic> B. japonicum </italic> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Introduction of the <italic> rd29A:AtDREB2A </italic> CA gene into soybean ( <italic> Glycine max </italic> L. Merril) and its molecular characterization in leaves and roots during dehydration</h2>
			<h3>PMID:24385860</h3>
				
					<h3>PMCID:3873188</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Engels </surname>  <given-names> Cibelle </given-names>  </name>  <xref ref-type="aff" rid="af1-gmb-36-556">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="c1-gmb-36-556"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fuganti-Pagliarini </surname>  <given-names> Renata </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-36-556">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marin </surname>  <given-names> Silvana Regina Rockenbach </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-36-556">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcelino-Guimar&#x000e3;es </surname>  <given-names> Francismar Corr&#x000ea;a </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-36-556">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oliveira </surname>  <given-names> Maria Cristina Neves </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-36-556">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kanamori </surname>  <given-names> Norihito </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-36-556">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mizoi </surname>  <given-names> Junya </given-names>  </name>  <xref ref-type="aff" rid="af4-gmb-36-556">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakashima </surname>  <given-names> Kazuo </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-36-556">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamaguchi-Shinozaki </surname>  <given-names> Kazuko </given-names>  </name>  <xref ref-type="aff" rid="af3-gmb-36-556">  <sup> 3 </sup>  </xref>  <xref ref-type="aff" rid="af4-gmb-36-556">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre Lima </given-names>  </name>  <xref ref-type="aff" rid="af2-gmb-36-556">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 09 </day>  <month> 8 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Transcription factor (TF) <italic> At </italic> DREB1A, which is responsive to cold, was inserted into soybean and subjected to molecular, anatomical and physiological analysis by <xref ref-type="bibr" rid="b22-gmb-36-556"> Polizel <italic> et al. </italic> (2011) </xref> using experimental water deficit conditions. Transcription factor DREB2A, which belongs to the same TF family, was also characterized in <italic> A. thaliana </italic> and responded to <span class="trait">drought,</span> high salinity and heat. Overexpression of constitutively active (CA) <span class="gene">DREB2A</span> resulted in significant tolerance to <span class="trait">drought</span> and heat stress in transgenic <italic> Arabidopsis </italic> plants ( <xref ref-type="bibr" rid="b28-gmb-36-556"> Sakuma <italic> et al. </italic> , 2006a </xref> , <xref rid="b29-gmb-36-556" ref-type="bibr"> b </xref> ). <italic> AtDREB2A </italic> -homologous genes were studied in maize ( <xref ref-type="bibr" rid="b23-gmb-36-556"> Qin <italic> et al. </italic> , 2007 </xref> ), rice ( <xref ref-type="bibr" rid="b6-gmb-36-556"> Dubouzet <italic> et al. </italic> , 2003 </xref> ), sunflower ( <xref ref-type="bibr" rid="b1-gmb-36-556"> Almogueva <italic> et al. </italic> , 2009 </xref> ), wheat ( <xref ref-type="bibr" rid="b33-gmb-36-556"> Terashima and Takumi, 2009 </xref> ) and chrysanthemum ( <xref ref-type="bibr" rid="b15-gmb-36-556"> Liu <italic> et al. </italic> , 2008 </xref> ). Recently, <xref ref-type="bibr" rid="b19-gmb-36-556"> Mizoi <italic> et al. </italic> (2013) </xref> identified a soybean <span class="gene">DREB2</span> gene, <span class="gene">GmDREB2A;2,</span> and showed that its heterologous expression in <italic> Arabidopsis </italic> improved the plants tolerance to stress. These findings indicate that plants overexpressing <italic> AtDREB2A </italic> and <italic> DREB2A-like </italic> proteins have increased tolerance to abiotic stress, <span class="trait">drought</span> and heat, which often occur together under field conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						There is a pressing need for soybean cultivars that are more tolerant to <span class="trait">drought</span> and heat, particularly since data provided by the IPCC ( <xref ref-type="bibr" rid="b8-gmb-36-556"> Intergovernmental Panel on Climate Change, 2007 </xref> ) indicate that the Earth&#x02019;s average temperature will increase in forthcoming years and lead to a reduction in land area suitable for growing soybean and other crops. The aim of this study was to insert the TF <italic> <span class="gene">DREB2A</span> </italic> isolated from <italic> A. thaliana </italic> into soybean using biolistics and then examine this genes effect on <span class="trait">resistance</span> to <span class="trait">drought.</span> The number of transgene copies inserted into the soybean genome was quantified using quantitative polymerase chain reaction (qPCR) and Southern blotting. The transgene expression under <span class="trait">drought</span> conditions was analyzed by qPCR in leaves and roots. The results described here may be useful in developing soybean cultivars that are more tolerant to <span class="trait">drought</span> and heat.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Impact of Fertilizing Pattern on the Biodiversity of a Weed Community and Wheat Growth</h2>
			<h3>PMID:24416223</h3>
				
					<h3>PMCID:3885565</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Leilei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Chuanpeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wan </surname>  <given-names> Kaiyuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Ruhai </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Daozhong </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tao </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pan </surname>  <given-names> Junfeng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xie </surname>  <given-names> Juan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Fang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 21 </day>  <month> 11 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The PK treatment showed the highest Shannon-Wiener and Pielou Indices, followed by the control and NK treatments. In contrast, these three treatments had lower Simpson Indices than the treatments with N and P. Solar penetration on the ground surface was also one of the reasons for the variation in weed community biodiversity, which is similar to the findings of other studies <xref ref-type="bibr" rid="pone.0084370-Kleijn1"> [16] </xref> , <xref ref-type="bibr" rid="pone.0084370-Goldberg1"> [39] </xref> , <xref ref-type="bibr" rid="pone.0084370-Poggio1"> [28] </xref> . Biomass is often used as an indicator of the amount of resources captured by a crop. Higher biomass and grain <span class="trait">yield</span> of wheat in the NP, NPK and N1/2PK treatments indicated that wheat intercepted a larger proportion of light radiation ( <xref ref-type="fig" rid="pone-0084370-g007"> Fig. 7 </xref> ), thereby reducing the amount reaching the weeds. Wheat <span class="trait">yield</span> showed a significant negative correlation with light penetration (Pearson correlations &#x0003e;0.880, P&#x0003c;0.0001). The species richness, Shannon-Wiener and Pielou Indices and the Simpson Index of weed communities were proved to have significantly negative and positive linear function relationships with wheat <span class="trait">yield</span> <xref ref-type="bibr" rid="pone.0084370-Tang1"> [30] </xref> . Shading leads to thinning mortality in individuals of subordinate weed species, and consequently reduces evenness and biodiversity among weed species <xref ref-type="bibr" rid="pone.0084370-Goldberg1"> [39] </xref> , <xref ref-type="bibr" rid="pone.0084370-Tilman3"> [40] </xref> . Only those weed species which possess either physiological shade tolerance or are able to climb into more favorable light conditions could grow well <xref ref-type="bibr" rid="pone.0084370-Murphy1"> [18] </xref> . In contrast to other species, <italic> G. aparine </italic> had higher plant density, shoot biomass and nutrient accumulations in the NP, NPK and N1/2PK treatments with lower light penetration. Mahn <xref ref-type="bibr" rid="pone.0084370-Mahn1"> [15] </xref> also reported that the mean biomass values for <italic> G. aparine </italic> in higher N fertilization treatments within a spring barley field were almost two times more than the values under the no N treatment. Under high soil <span class="trait">fertility</span> conditions in this study, <italic> G. aparine </italic> could enhance its plant height through internode elongation, especially from the 4 <sup> th </sup> to the 12 <sup> th </sup> internodes for capturing more <span class="gene">sun</span> light. Its plants were 14% and 9% taller than wheat plants in the NP and NPK treatments, respectively ( <xref ref-type="fig" rid="pone-0084370-g005"> Fig. 5 </xref> ). Therefore, weed vegetation under higher soil <span class="trait">fertility</span> conditions was predominantly composed of a few species, such as <italic> G. aparine </italic> , resulting in a higher Simpson Index and lower Pielou and Shannon-Wiener Indices.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Aboveground Feeding by Soybean Aphid, <italic> Aphis glycines </italic> , Affects Soybean Cyst Nematode, <italic> Heterodera glycines </italic> , Reproduction Belowground</h2>
			<h3>PMID:24466080</h3>
				
					<h3>PMCID:3899248</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> McCarville </surname>  <given-names> Michael T. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Soh </surname>  <given-names> David H. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tylka </surname>  <given-names> Gregory L. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> O&#x02019;Neal </surname>  <given-names> Matthew E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 12 </day>  <month> 12 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Aphid-density treatments were applied to whole plots when plants reached the first trifoliate or <span class="gene">V1</span> stage <xref ref-type="bibr" rid="pone.0086415-Fehr1"> [27] </xref> , which occurred 10 d after planting. Mixed-aged apterous <italic> A. glycines </italic> were transferred from a greenhouse biotype-1 colony (i.e. avirulent to all known <italic> A. glycines </italic> <span class="trait">resistance</span> genes, Hill <italic> et al. </italic> <xref ref-type="bibr" rid="pone.0086415-Hill1"> [28] </xref> ) to each plant assigned to the five-aphid and ten-aphid treatments. Each whole plot bucket was then covered with a modified paint strainer (Trimaco, Morrisville, NC) to prevent the movement of aphids among whole plots. <italic> Aphis glycines </italic> populations were then allowed to increase for the remainder of the experiment.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Systems and Evolutionary Characterization of MicroRNAs and Their Underlying Regulatory Networks in Soybean Cotyledons</h2>
			<h3>PMID:24475082</h3>
				
					<h3>PMCID:3903507</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Goettel </surname>  <given-names> Wolfgang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Zongrang </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Weixiong </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Patrick X. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> An </surname>  <given-names> Yong-Qiang (Charles) </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 5 </day>  <month> 12 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Interestingly, the inverted repeat units containing the insertion were highly homologous to part of exon 5 (&#x0223c;240 bp) and intron 4 (&#x0223c;210 bp) of a gene encoding a neutral invertase <span class="gene">(Glyma12g02690).</span> Their terminal sequences contained a 38nt sequence of unknown origin and a 25nt sequence homologous to the same LTR family that was found in the 12 kb direct repeats ( <xref ref-type="fig" rid="pone-0086153-g006"> Figure 6 </xref> and <xref ref-type="supplementary-material" rid="pone.0086153.s001"> Figure S1 </xref> ). This patchwork of sequences was reminiscent of filler DNA that results from the repair of chromosome breaks through non-homologous end-joining. Thus, we propose that the soy-miR15/49 family most likely evolved recently from the neutral invertase gene <span class="gene">(Glyma12g02690)</span> through complex DNA rearrangements and amplifications. The DNA fragment was duplicated and joined in a tail-to-tail fashion to form an inverted repeat structure. This inverted repeat was amplified and spread to chromosomes 20, 14, 2 and 8 and 3 over the past 8.3 million years. Subsequently, the ancestral inverted repeat of IR1-7 lost the 360 bp indel sequence between 2 to 5 million years ago and was later amplified through a series of local DNA duplications until the IR cluster reached its current shape. We did not detect miRNA reads aligning to IR10-12. It is likely that the miR15/49 family arose after the split of the IR1-7 clade from the IR8-9 clade. Deletion of the 360 bp long indel significantly decreased the distance between the miR15/miR15* duplex and the loop of the hairpin structure, and might promote the birth of the soy-miR15/49 family. soy-miR15/49 represented a young and recently evolved miRNA family, and still maintained a strong sequence similarity to the neutral invertase gene. The length of the duplicated soy-miR15 gene sequences extends beyond their miRNA fold-back structure, which has been reported for many Arabidopsis miRNA genes as well <xref rid="pone.0086153-Fahlgren1" ref-type="bibr"> [19] </xref> . It has also been proposed that <italic> MIR </italic> genes can evolve from inverted duplicates of protein-coding genes or gene segments <xref rid="pone.0086153-Fahlgren1" ref-type="bibr"> [19] </xref> , <xref rid="pone.0086153-Rajagopalan1" ref-type="bibr"> [21] </xref> , <xref rid="pone.0086153-Fahlgren2" ref-type="bibr"> [46] </xref> , <xref rid="pone.0086153-Allen1" ref-type="bibr"> [49] </xref> &#x02013; <xref rid="pone.0086153-Chen2" ref-type="bibr"> [52] </xref> . Initially, these perfect inverted repeats would produce hairpin transcripts that are processed by DCL4 and DCL3 to <span class="trait">yield</span> 24nt siRNAs. Over time, drift mutations would disrupt the perfect hairpin structures that would cause the transition from DCL4 to <span class="gene">DCL1</span> cleavage resulting in the production of 21nt miRNAs. Further accumulation of mutations over time in regions surrounding the miRNA/miRNA* sequence could generate miRNA genes that have lost similarities with the parental donor sequence and therefore may appear unrelated to their locus of origin. Our in-depth analysis of the soy-miR15/49 family not only provided strong evidence for the hypothesis that miRNAs can evolve from inverted duplications of protein-coding gene fragments, but also illustrated the evolutionary pathway that gave birth to the miR15/49 family.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The genome sequence and effector complement of the flax <span class="trait">rust</span> pathogen <italic> Melampsora lini </italic></h2>
			<h3>PMID:24715894</h3>
				
					<h3>PMCID:3970004</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Nemri </surname>  <given-names> Adnane </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Saunders </surname>  <given-names> Diane G. O. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Anderson </surname>  <given-names> Claire </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Upadhyaya </surname>  <given-names> Narayana M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Win </surname>  <given-names> Joe </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lawrence </surname>  <given-names> Gregory J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jones </surname>  <given-names> David A. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kamoun </surname>  <given-names> Sophien </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ellis </surname>  <given-names> Jeffrey G. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dodds </surname>  <given-names> Peter N. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 2 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> Properties of the top 200 tribes of <italic> M. lini </italic> candidate effectors </bold> . Tribes were assembled with 2642 proteins coming from four <span class="trait">rust</span> species ( <italic> Mli </italic> , <italic> Mlp </italic> , <italic> Pgt </italic> , and <italic> Pst </italic> ), and all contain at least one member from <italic> Mli </italic> . Tribes were grouped according to whether they contain members from <italic> Mli </italic> only or members from the two <italic> Melampsora </italic> species only or members from all four <span class="trait">rust</span> species. On a circle, each <span class="gene">bar</span> represents a tribe, with the width of <span class="gene">bar</span> proportional to the number of members in that tribe. Previously known <italic> Mli </italic> avirulence proteins and fungal virulence effectors are indicated with a red asterisk. For each tribe, heatmaps indicate the scores for ( <italic> Sc </italic> ) overall tribe rank, ( <italic> Se </italic> ) prediction of secretion, ( <italic> H </italic> ) similarity to haustorial ESTs, ( <italic> C </italic> ) presence of small cysteine-rich proteins, ( <italic> E </italic> ) presence of an effector motif or nuclear localization signal, ( <italic> R </italic> ) presence of tandem repeats, and ( <italic> Ann </italic> ) fraction of proteins with a PFAM annotation. Note that the ( <italic> H </italic> ) field represents similarity across the whole tribe with <italic> Mli </italic> haustorial ESTs, as opposed to haustorial expression of tribe members from all four rusts. As <italic> Mli </italic> is the focus of this study, tribes with members expressed in the haustoria of <italic> Mli </italic> were highlighted. Some previously known haustorially secreted proteins either from <italic> Mli </italic> or other rusts may thus appear to have low score.
					</p>
				
					<p>&nbsp;&nbsp;
						We have identified a large number of candidate effectors in <italic> Mli </italic> that show variable degrees of evolutionary conservation, i.e., shared across the four <span class="trait">rust</span> species included in this comparison, shared between the two <italic> Melamspora </italic> species or specific to flax <span class="trait">rust.</span> Within the 200 selected tribes out of 940 described here, we found 37% of <italic> Mli </italic> candidate effectors to be specific to the <italic> Melampsora </italic> genus, which contrasts with the ~74% of all small secreted proteins present in <italic> Mlp </italic> but without <italic> Blast </italic> matches in <italic> Pgt </italic> (Duplessis et al., <xref rid="B18" ref-type="bibr"> 2011a </xref> ). The difference may result from our clustering and tribe-ranking approaches, which emphasized larger (more shared) tribes and does not solely rely on secretome predictions. Here, we have highlighted candidate effectors that may contribute to plant pathogenicity across <span class="trait">rust</span> fungi, largely recapitulating previous related studies (Duplessis et al., <xref rid="B18" ref-type="bibr"> 2011a </xref> ; Saunders et al., <xref rid="B64" ref-type="bibr"> 2012 </xref> ; Garnica et al., <xref rid="B27" ref-type="bibr"> 2013 </xref> ), as well as potential determinants of host specificity in <italic> Mli </italic> . Importantly, all known <italic> Mli </italic> Avr proteins were in tribes that are either specific to <italic> Mli </italic> alone or <italic> Mli </italic> and <italic> Mlp </italic> , whereas a majority of the conserved <span class="trait">rust</span> fungal tribes contain enzymes with expected apoplastic activity. In flax, all <span class="trait">resistance</span> genes identified so far are predicted to act in the cytoplasm (Ellis et al., <xref rid="B22" ref-type="bibr"> 2007 </xref> ). Taken together, this may indicate that <span class="trait">rust</span> fungal tribes specific to the genus level are enriched in intracellular effectors and thus may be a primary source of Avr proteins targeted by host intracellular immune receptors. It is unclear whether these genus-specific effectors determine host specificity of <span class="trait">rust</span> fungi species via their virulence action, and not just their potential avirulence properties. So far, most of the research on <span class="trait">rust</span> fungal effectors has focused on these translocated effectors (see review by Duplessis et al., <xref rid="B20" ref-type="bibr"> 2011c </xref> ). In contrast, little is known of the putative apoplastic effectors of <span class="trait">rust</span> fungi identified here and in previous studies (Duplessis et al., <xref rid="B18" ref-type="bibr"> 2011a </xref> ; Hacquard et al., <xref rid="B32" ref-type="bibr"> 2012 </xref> ; Saunders et al., <xref rid="B64" ref-type="bibr"> 2012 </xref> ; Garnica et al., <xref rid="B27" ref-type="bibr"> 2013 </xref> ). The apparent wide conservation across <span class="trait">rust</span> fungi of some of these apoplastic effectors that perform a more general virulence function on a wide variety of hosts (e.g., cellulases) makes them particularly interesting for future studies. In the plant-pathogenic Ascomycete fungus <italic> Cladosporium fulvum </italic> , several apoplastic effectors have been characterized, including some that are recognized extra-cellularly by immune receptors (reviewed in Wit et al., <xref rid="B78" ref-type="bibr"> 2009 </xref> ). For example, <italic> Ecp6 </italic> and <span class="gene">Avr4</span> function as chitin-binding proteins that inhibit host chitin-elicited immunity and a host-chitinase inhibitor, respectively (Van Den Burg et al., <xref rid="B74" ref-type="bibr"> 2006 </xref> ; De Jonge et al., <xref rid="B13" ref-type="bibr"> 2010 </xref> ). Further work is required to assess whether candidate apoplastic effectors from <span class="trait">rust</span> fungi have similar roles to the <italic> C. fulvum </italic> effectors and whether there are components of host immunity that may target them in the apoplast.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Impacts of nucleotide fixation during soybean domestication and improvement</h2>
			<h3>PMID:25849896</h3>
				
					<h3>PMCID:4358728</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" equal-contrib="yes">  <name>  <surname> Zhao </surname>  <given-names> Shancen </given-names>  </name>  <address>  <email> zhaoshancen@genomics.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zheng </surname>  <given-names> Fengya </given-names>  </name>  <address>  <email> zfky@mail.ustc.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Weiming </given-names>  </name>  <address>  <email> hewm@genomics.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Haiyang </given-names>  </name>  <address>  <email> wuhaiyang@genomics.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pan </surname>  <given-names> Shengkai </given-names>  </name>  <address>  <email> panshengkai@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Lam </surname>  <given-names> Hon-Ming </given-names>  </name>  <address>  <email> honming@cuhk.edu.hk </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Centre for Soybean Research, Partner State Key Laboratory of Agrobiotechnology, The Chinese University of Hong Kong, Shatin, New Territories, Hong Kong  </aff>  <aff id="Aff2">  <label/> BGI-Shenzhen, Main Building, Beishan Industrial Zone, Yantian District, Shenzhen, 518083 China  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 2 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The ability of <span class="trait">resistance</span> to pathogen in wild soybeans is much broader than that in cultivated soybeans [ <xref ref-type="bibr" rid="CR23"> 23 </xref> , <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. Interestingly, <italic> <span class="gene">Glyma20g08290</span> </italic> gene is an ortholog of the disease <span class="trait">resistance</span> gene <italic> RPM1 </italic> , which was detected and characterized using molecular genetic approach in Arabidopsis [ <xref ref-type="bibr" rid="CR40"> 40 </xref> ]. In soybeans, the <italic> RPM1 </italic> gene was recently reported being under purifying selection [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. It serves as an example that natural selection in the wild population also caused nucleotide fixations, although its strength was less than artificial selection.
					</p>
				
					<p>&nbsp;&nbsp;
						As an agriculturally important trait, grain filling makes a significant contribution to grain weight [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ]. The gene <italic> Grain Incomplete Filling 1 </italic> ( <italic> GIF1 </italic> ) was detected to be responsible and associated with this domestication syndrome [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. It was reported to encode a cell-wall invertase required for carbon partitioning during early grain filling in rice. A selective gene <italic> <span class="gene">Glyma03g35520</span> </italic> with nucleotide fixation in domestication is an ortholog of <italic> GIF1 </italic> and this gene was involved in the carbohydrate metabolism pathway by searching KEGG (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S4). Besides, this gene was covered by the QTLs responsible for <span class="trait">lodging</span> and pod number. It indicates that <italic> <span class="gene">Glyma03g35520</span> </italic> is a potential candidate gene, which could be used for further soybean breeding.
					</p>
				
					<p>&nbsp;&nbsp;
						Flower and pod numbers per plant are important agronomic traits for grain <span class="trait">yield</span> in soybean. To detect the genes involved in flower and pod numbers will help to understand the genetic basis of soybean <span class="trait">yield</span> [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ]. Two genes, <italic> <span class="gene">Glyma07g05470</span> </italic> and <italic> <span class="gene">Glyma07g05480</span> </italic> , with nucleotide fixation introduced in improvement, are orthologs of <italic> COMT2 </italic> gene encoding caffeic acid 3-O-methyltransferase (Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S5). It differentially expressed in hair cells of growing pod, the possible location of vanillin biosynthesis [ <xref ref-type="bibr" rid="CR47"> 47 </xref> ]. Another five selective genes with nucleotide fixation mediated by domestication and improvement encode a kind of protein responsible for the transportation of inositol. These genes were covered by QTLs responsible for seed-coat color, protein and pod number. Previous study showed that the total number of mature pods considerably higher due to the application of inositol, indicating the positive effect in pod number [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ]. It suggested that deficiency of lignin biosynthesis resulted in growth reduction and dwarfing [ <xref ref-type="bibr" rid="CR49"> 49 </xref> ]. The gene <italic> <span class="gene">Glyma13g21010</span> </italic> is linked to marker Sat103 that associate with seed weight. As an orthologs of <italic> NifU </italic> gene, it is required for full activation of nitrogenase catalytic components [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. <italic> NifU </italic> protein has been suggested to either mobilize the Fe necessary for nitrogenase Fe-S cluster formation or provide an intermediate Fe-S cluster assembly site [ <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. In addition, the gene was reported to be related to seed weight [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. As nitrogen fixation is imperative in soybean growth, <italic> <span class="gene">Glyma13g21010</span> </italic> gene could also be a putative candidate gene responsible for seed weight through activating biological nitrogen fixation.
					</p>
				
					<p>&nbsp;&nbsp;
						The flowering of soybean represents the transition from a vegetative state to a reproductive state, making a contribution to the <span class="trait">yield.</span> Meta-analysis of QTLs identified 14 selective genes with non-synonymous nucleotide fixation responsible for flower number in soybean. Carbon fixation in the process of photosynthesis is pivotal to soybean production. Seven selective genes with nucleotide fixation were involved in photosynthesis or photosystem. Besides, two selective genes <italic> <span class="gene">Glyma03g36970</span> </italic> and <italic> <span class="gene">Glyma19g39620</span> </italic> with nucleotide fixation were identified as orthologs of <italic> Luminidependens </italic> , which is involved in the timing of flowering in Arabidopsis [ <xref ref-type="bibr" rid="CR53"> 53 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 37 selective genes with nucleotide fixation were involved in plant-pathogen interactions (Additional file <xref rid="MOESM7" ref-type="media"> 7 </xref> : Figure S6). Of them, two selective genes <italic> <span class="gene">Glyma14g36511</span> </italic> and <italic> <span class="gene">Glyma08g12560</span> </italic> with nucleotide fixation are orthologs of <italic> RPS2 </italic> gene. The disease <span class="trait">resistance</span> gene <italic> RPS2 </italic> was isolated using positional cloning and further screen for susceptible mutant [ <xref ref-type="bibr" rid="CR61"> 61 </xref> , <xref ref-type="bibr" rid="CR62"> 62 </xref> ]. The <italic> RPS2 </italic> protein contains two characteristics of a large family of plant <italic> R </italic> genes: a nucleotide-binding site and a leucine-rich repeat region [ <xref ref-type="bibr" rid="CR63"> 63 </xref> ]. It is consistent with previous report that <italic> RPS2 </italic> locus exhibit selection signals by examining a worldwide sample of 27 Arabidopsis accessions, and the N-terminal part of the leucine-rich repeat region was a probable target of selection [ <xref ref-type="bibr" rid="CR64"> 64 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Land usage attributed to corn ethanol production in the United States: sensitivity to technological advances in corn grain <span class="trait">yield,</span> ethanol conversion, and co-product utilization</h2>
			<h3>PMID:24725504</h3>
				
					<h3>PMCID:4022103</h3>
					<h4>期刊:Biotechnology for Biofuels</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes" id="A1">  <name>  <surname> Mumm </surname>  <given-names> Rita H </given-names>  </name>  <xref ref-type="aff" rid="I1"> 1 </xref>  <email> ritamumm@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A2">  <name>  <surname> Goldsmith </surname>  <given-names> Peter D </given-names>  </name>  <xref ref-type="aff" rid="I2"> 2 </xref>  <email> pgoldsmi@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A3">  <name>  <surname> Rausch </surname>  <given-names> Kent D </given-names>  </name>  <xref ref-type="aff" rid="I3"> 3 </xref>  <email> krausch@illinois.edu </email>  </contrib>  <contrib contrib-type="author" id="A4">  <name>  <surname> Stein </surname>  <given-names> Hans H </given-names>  </name>  <xref ref-type="aff" rid="I4"> 4 </xref>  <email> hstein@illinois.edu </email>  </contrib>
				<br/> 
				年份：<day> 19 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						This effect is even more exaggerated in Scenario 6, which features fermentation technology to convert <span class="gene">C5</span> and C6 sugar to ethanol, converting not only residual starch but pericarp and endosperm fiber fractions as well. With complete fiber (and starch) conversion, the ethanol <span class="trait">yield</span> per bushel of processed corn increases from 2.759 to 3.078 gallons, compared with baseline ethanol processing efficiencies. Correspondingly, the quantity of DDGS produced as co-product decreases from 17.44 to 12.67 pounds per bushel of processed corn (Table&#x000a0; <xref ref-type="table" rid="T5"> 5 </xref> ). Similar to Scenario 5, Scenario 6 presents an interesting trade-off: an overall 11.6% increase in ethanol <span class="trait">yield</span> per bushel of corn for a 27.4% decrease in the quantity of DDGS produced per bushel of corn. The decrease in feed co-products results in less corn and soybean meal being substituted, which in turn raises the land area attributed to corn ethanol. In general, land area for corn ethanol is higher when the system extracts more ethanol per bushel of corn.
					</p>
				
					<p>&nbsp;&nbsp;
						&#x02022; Agronomic improvements are anticipated in three waves by 2026, each accounting for a 3 bu/A <span class="trait">yield</span> increase (Table&#x000a0; <xref ref-type="table" rid="T2"> 2 </xref> ). Seed companies are working with manufacturers of farm machinery to create &#x02018;smart systems&#x02019; to maximize <span class="trait">yield</span> through best possible agronomic conditions. For example, Monsanto Company plans the market introduction of <span class="gene">IFS</span> (Integrated Farming Systems) I, featuring variable rate planting, as early as 2014 [ <xref ref-type="bibr" rid="B33"> 33 </xref> , <xref ref-type="bibr" rid="B34"> 34 </xref> ]. A second wave of agronomic improvements is anticipated, with prescription placement for <span class="trait">fertility</span> and water added to <span class="gene">IFS</span> II [ <xref ref-type="bibr" rid="B33"> 33 </xref> ]. The development of agricultural biologicals that boost the efficiency of pest controls such as BioDirect&#x02122; Technology by Monsanto [ <xref ref-type="bibr" rid="B33"> 33 </xref> ] may factor into a third wave of agronomic improvements. Conservative estimation of <span class="trait">yield</span> impact forecasts effect step changes at maximal technology adoption in 2016, 2020, and 2024, with no phase-in period.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Historical gains in soybean ( <italic> Glycine max </italic> Merr.) seed <span class="trait">yield</span> are driven by linear increases in light interception, energy conversion, and partitioning efficiencies</h2>
			<h3>PMID:24790116</h3>
				
					<h3>PMCID:4071847</h3>
					<h4>期刊:Journal of Experimental Botany</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Koester </surname>  <given-names> Robert P. </given-names>  </name>  <xref ref-type="aff" rid="AF0001">
				<br/> 
				年份：未知
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In this equation, <italic> S </italic> <sub> t </sub> is total incident solar radiation during the growing season of which ~48.7% is photosynthetically active. Light interception efficiency (&#x003b5; <sub> i </sub> ) is determined by the speed and duration of canopy closure along with canopy size and architecture. Energy conversion efficiency (&#x003b5; <sub> c </sub> ), or radiation use efficiency, is determined by the amount of solar energy that is transformed into biomass through the balance of photosynthesis and respiration. Partitioning efficiency (&#x003b5; <sub> p </sub> ), or harvest index, is determined by the amount of biomass energy allocated to vegetative versus reproductive structures ( <xref rid="CIT0024" ref-type="bibr"> Zhu <italic> et al. </italic> , 2010 </xref> ). The Monteith equation tracks energy transfer from the <span class="gene">sun</span> to the seed and provides insight into the physiological mechanisms that ultimately govern <span class="trait">yield</span> potential. As a result, the Monteith equation has been used to assess which parameters are at their theoretical maxima and which could be improved further to advance <span class="trait">yield</span> ( <xref rid="CIT0013" ref-type="bibr"> Gifford <italic> et al. </italic> , 1984 </xref> ; <xref rid="CIT0017" ref-type="bibr"> Loomis and Amthor, 1999 </xref> ; <xref rid="CIT0030" ref-type="bibr"> Reynolds <italic> et al. </italic> , 2000 </xref> ; <xref rid="CIT0024" ref-type="bibr"> Reynolds <italic> et al. </italic> , 2010 </xref> ; <xref rid="CIT0024" ref-type="bibr"> Zhu <italic> et al. </italic> , 2010 </xref> ; <xref rid="CIT0001" ref-type="bibr"> Ainsworth <italic> et al. </italic> , 2012 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of <span class="trait">Drought</span> Tolerance of the Vietnamese Soybean Cultivars Provides Potential Resources for Soybean Production and Genetic Engineering</h2>
			<h3>PMID:24804248</h3>
				
					<h3>PMCID:3997955</h3>
					<h4>期刊:BioMed Research International</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0002-0210-1900 </contrib-id>  <name>  <surname> Thu </surname>  <given-names> Nguyen Binh Anh </given-names>  </name>  <xref ref-type="aff" rid="I1">
				<br/> 
				年份：<day> 3 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Taken together, we recorded DT51 and MTD720 as two cultivars having contrasting drought-tolerant features, of which DT51 was the highest drought-tolerant cultivar, whereas MTD720 was the lowest drought-tolerant cultivar. This finding was also supported by a differential expression analysis of a subset of <italic> <span class="gene">GmNAC</span> </italic> genes [ <xref rid="B34" ref-type="bibr"> 34 </xref> ], which are known as transcriptional factors involved in regulation of plant response to <span class="trait">drought</span> [ <xref rid="B35" ref-type="bibr"> 35 </xref> &#x02013; <xref rid="B37" ref-type="bibr"> 37 </xref> ]. The expression of drought-responsive <italic> <span class="gene">GmNAC</span> </italic> s in roots of DT51 and MTD720 was significantly different. The better drought-tolerant capacity of DT51 was shown to be related to the higher number of drought-inducible <italic> <span class="gene">GmNAC</span> </italic> genes, as well as the higher number of <italic> <span class="gene">GmNAC</span> </italic> genes with higher transcript accumulation in comparison with MTD720 [ <xref rid="B34" ref-type="bibr"> 34 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Grazing Soybean to Increase Voluntary Cow Traffic in a Pasture-based Automatic Milking System</h2>
			<h3>PMID:25049970</h3>
				
					<h3>PMCID:4093256</h3>
					<h4>期刊:Asian-Australasian Journal of Animal Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Clark </surname>  <given-names> C. E. F. </given-names>  </name>  <xref rid="c1-ajas-27-3-422-16" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Horadagoda </surname>  <given-names> A. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kerrisk </surname>  <given-names> K. L. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Scott </surname>  <given-names> V. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Islam </surname>  <given-names> M. R. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kaur </surname>  <given-names> R. </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Garcia </surname>  <given-names> S. C. </given-names>  </name>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 9 </month>  <year> 2013 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The study was conducted at the AMS research farm, Elizabeth MacArthur Agricultural Institute, Camden (34&#x000b0; 04&#x02032;S; 150&#x000b0; 69&#x02032;E) from 28 January 2012 to 11 February 2012. One hundred and eighty mixed age, primiparous and multiparous Holstein-Friesian/Illawarra cows were randomly assigned to two treatment groups (n = 90/group) matched for (means&#x000b1;standard deviation) <span class="gene">DIM</span> 174&#x000b1;123 days; parity 2.8&#x000b1;1.9; and milk <span class="trait">yield</span> 22.2&#x000b1;8.1 L in a 2 (treatment)&#x000d7;2 (periods) Latin square design.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparison of Soil Quality Index Using Three Methods</h2>
			<h3>PMID:25148036</h3>
				
					<h3>PMCID:4141864</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Mukherjee </surname>  <given-names> Atanu </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lal </surname>  <given-names> Rattan </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 7 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soil samples from the field (on-farm) were collected from Logan county, Ohio that included an organic (Martisco Variant silt loam: Mc, organic parent material, &#x0003e;26% organic matter, 40&#x000b0;25&#x02032;12.4&#x02033;N, 83&#x000b0;40&#x02032;55.9&#x02033;W), and a mineral soil (Crosby silt loam: CrB: <italic> Fine, mixed, active, mesic Aeric Epiaqualfs </italic> , sand: 23%, silt: 35%, clay: 42%, 40&#x000b0;24&#x02032;52.4&#x02033;N, 83&#x000b0;39&#x02032;21.9&#x02033;W), and from Franklin county, Ohio that included a Kokomo silty clay loam (Ko: <italic> Fine, mixed, superactive, mesic Typic Argiaquolls </italic> , sand: 9%, silt: 62%, clay: 29%, 40&#x000b0;00&#x02032;41.2&#x02033;N, 83&#x000b0;12&#x02032;23.1&#x02033;W) in May, 2013. These locations were farmer-owned field sites and before collecting the soils from specific sites permissions of the farmers were obtained from the land-owners or farmers. Initial permission was granted through The Ohio State University&#x02019;s extension managers and thereafter communication was established directly by the primary author. Additionally, no endangered or protected species were involved in the current study sites, and thus no such permission was required from any other regulatory agencies. The properties of the soils and other management details were presented elsewhere <xref rid="pone.0105981-Mukherjee1" ref-type="bibr"> [20] </xref> , <xref rid="pone.0105981-Mukherjee2" ref-type="bibr"> [21] </xref> and briefly presented in the discussion section. Soil samples were obtained from 0&#x02013;10, 10&#x02013;20, 20&#x02013;40, and 40&#x02013;60 cm depths under no-tillage (NT) and conventional tillage (CT) practices for Mc and CrB soil series and under NT for one year cover crop (CC) and no-cover crop (NCC) for Ko series. The organic soil (Mc) has been under practice of CT corn ( <italic> Zea mays </italic> ) for 10 years which was compared with the adjacent grassland soil (termed as NT). The mineral soil (CrB) was under NT and recent introduction (one year) of CT corn practices. Both soils received fertilizer (N:P:K as 5.5&#x02236;26&#x02236;30) with the rates of 336 (30, 142 and 164 kg ha <sup> &#x02212;1 </sup> of N, P, and K, respectively) and 280 (25, 118 and 137 kg ha <sup> &#x02212;1 </sup> of N, P, and K, respectively) kg ha <sup> &#x02212;1 </sup> for corn, for Mc and CrB soils, respectively. The other site (Ko) was under NT corn and soybean ( <italic> Glycin max L. </italic> ) annual rotation. A mixture of pea ( <italic> Pisum sativum L. </italic> ) and turnip ( <italic> Brassica rapa L. </italic> ) cover crops with the seeding rate of 140 kg ha <sup> &#x02212;1 </sup> was seeded prior to growing soybean in 2013. Roundup-ready soybean was seeded on 20 April, 2013 and harvested on 30 September, 2013. A row spacing of 38 cm was used during planting for corn and soybean. Soil core (inside diameter: 5.3 cm, height: 5.9 cm) samples were collected manually using soil sampler before planting (7 May, 2013) from four depths (0&#x02013;10, 10&#x02013;20, 20&#x02013;40, and 40&#x02013;60) for determination of bulk density (BD) by dividing dry mass of soil in the core by core volume <xref rid="pone.0105981-Grossman1" ref-type="bibr"> [22] </xref> and hydrologic properties. Undisturbed soil cores were used to determine the water retention at field capacity (0.033 MPa), while sieved samples (&#x0003c;2-mm size) were used to determine the permanent wilting point (1.5 MPa). The potential available water capacity (AWC) of the soil was calculated as the difference in volumetric water content at 0.033 and 1.5 MPa moisture potentials. The water content of a soil layer was calculated by multiplying thickness of the layer with the volumetric water capacity. A minimum of three field penetration <span class="trait">resistance</span> (PR) measurements for soil&#x02019;s mechanical strength were made for each depth using a CP40II cone penetrometer <xref rid="pone.0105981-Herrick1" ref-type="bibr"> [23] </xref> . Bulk samples were obtained to measure the aggregate size distributions and fraction of water stable aggregates (WSA) using the wet sieving method <xref rid="pone.0105981-Yoder1" ref-type="bibr"> [24] </xref> . Five sieves of diameter sizes 4.75, 2, 1, 0.5, and 0.25 mm openings were placed into a Yoder apparatus. Air-dried soils (5&#x02013;8 mm, 51 g) were slowly wetted by capillarity action by adjusting the water level in the container so that the base of the top sieve just touched the water. The sieve combination was oscillated mechanically in the water at 60 oscillations per minute for 30 min. Aggregates retained in the sieves were transferred to glass beakers and the weight of each of five fractions was measured after drying at 60&#x000b0;C overnight. The data were used to compute WSA, mean weight diameter (MWD), and geometric mean diameter <span class="gene">(GMD)</span> <xref rid="pone.0105981-Youker1" ref-type="bibr"> [25] </xref> . Soil water retention at matrix potentials of 0.033 and 1.5 MPa were measured using a pressure plate apparatus <xref rid="pone.0105981-Dane1" ref-type="bibr"> [26] </xref> . The pH and electrical conductivity (EC) of soil were determined in 1&#x02236;2 soil:water ratio slurry using a Thermo-scientific Orion Star Series pH/Conductivity Meter. Concentrations of soil organic C and total N were determined using an Elemental analyzer (Vario Max, Elementar Americas, Inc., Germany) by the dry combustion method (900&#x000b0;C) after grinding subsamples to 0.25 mm. Total C and N stocks were calculated by multiplying the respective elemental concentrations by BD and the thickness of the soil layer <xref rid="pone.0105981-Chhabra1" ref-type="bibr"> [27] </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Abbreviations: SD: standard deviation, EC: electrical conductivity, BD: bulk density, WSA: water stable aggregates, <span class="gene">GMD:</span> geometrical mean diameter, MWD: mean weight diameter, PR: penetration <span class="trait">resistance,</span> N: nitrogen, SOC: soil organic carbon, AWC: available water capacity.
					</p>
				
					<p>&nbsp;&nbsp;
						In this approach, each soil parameter was first assigned unitless score ranging from o to 1 by employing linear scoring functions <xref rid="pone.0105981-Andrews1" ref-type="bibr"> [8] </xref> . Non-linear scoring functions were avoided because of their lower capacity of predicting the end point variable or crop <span class="trait">yield</span> <xref rid="pone.0105981-Andrews1" ref-type="bibr"> [8] </xref> . Soil parameters were divided into groups based on three mathematical algorithm functions: (a) &#x02018;more is better&#x02019; (e.g., WSA, <span class="gene">GMD,</span> C-Stock, N-Stock, and AWC) (b) &#x02018;less is better&#x02019; (e.g., BD, PR), and (c) &#x02018;optimum&#x02019; (e.g., pH and EC). &#x02018;Optimum&#x02019; properties are those which have positive influence up to a certain level beyond which the influence could be considered detrimental <xref rid="pone.0105981-Fernandes1" ref-type="bibr"> [13] </xref> . For &#x02018;more is better&#x02019; parameters, each observation was divided by the highest observed value of the entire dataset so that the highest observed value would have a score of 1; for &#x02018;less is better&#x02019; parameters, the lowest observed value in the entire dataset was divided by each observation so that the lowest observed value received a score of 1; and &#x02018;optimum&#x02019; parameters were scored up to a threshold value as &#x02018;more is better&#x02019;, and thereafter above the threshold values were scored as &#x02018;less is better&#x02019; <xref rid="pone.0105981-Andrews1" ref-type="bibr"> [8] </xref> , <xref rid="pone.0105981-Liebig1" ref-type="bibr"> [29] </xref> . For example, pHs up to 5.5&#x02013;7.2 were scored as &#x02018;more is better&#x02019;, and pHs &#x0003e;7.2 were scored as &#x02018;less is better&#x02019;.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of candidate genes involved in early iron deficiency <span class="trait">chlorosis</span> signaling in soybean ( <italic> Glycine max </italic> ) roots and leaves</h2>
			<h3>PMID:25149281</h3>
				
					<h3>PMCID:4161901</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Moran Lauter </surname>  <given-names> Adrienne N </given-names>  </name>  <address>  <email> Adrienne.MoranLauter@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Peiffer </surname>  <given-names> Gregory A </given-names>  </name>  <address>  <email> greg.peiffer@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yin </surname>  <given-names> Tengfei </given-names>  </name>  <address>  <email> yintengfei@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Whitham </surname>  <given-names> Steven A </given-names>  </name>  <address>  <email> swhitham@iastate.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cook </surname>  <given-names> Dianne </given-names>  </name>  <address>  <email> dicook@iastate.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shoemaker </surname>  <given-names> Randy C </given-names>  </name>  <address>  <email> rcsshoe@iastate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Graham </surname>  <given-names> Michelle A </given-names>  </name>  <address>  <email> michelle.graham@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <aff id="Aff1">  <label/> USDA-Agricultural Research Service, Corn Insects and Crop Genetics Research Unit, 1565 Agronomy Hall, Ames, IA 50011 USA  </aff>  <aff id="Aff2">  <label/> Department of Statistics, Iowa State University, Ames, Iowa 50011 USA  </aff>  <aff id="Aff3">  <label/> Department of Plant Pathology and Microbiology, Iowa State University, Ames, Iowa 50011 USA  </aff>  <aff id="Aff4">  <label/> Department of Agronomy, Iowa State University, Ames, Iowa 50011 USA  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To develop an understanding of which genes were affected by iron stress, we began focusing on the top ten induced and repressed genes in each sample, paying particular attention to those genes with homology to Arabidopsis genes with known roles in nutrient stress responses, growth and signaling (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). In one hour leaves, iron deficiency led to decreased expression of 74 of the 81 DEGs. The seven genes induced by iron deficiency included homologs of previously described Arabidopsis genes <italic> AtPLP1 </italic> <span class="gene">(Glyma07g13790,</span> 6E <sup> -89 </sup> ), <italic> AtGASA1 </italic> <span class="gene">(Glyma14g40400,</span> 4E <sup> -32 </sup> ), <italic> AtSWEET12 </italic> <span class="gene">(Glyma05g38351,</span> 1E <sup> -53 </sup> ), two homologs of <italic> AtOXS3 </italic> <span class="gene">(Glyma11g33040</span> [2E <sup> -15 </sup> ] and <span class="gene">Glyma18g05160</span> [6E <sup> -16 </sup> ]) and a copper amine oxidase <span class="gene">(Glyma01g07860</span> [E&#x02009;=&#x02009;0]) (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). <italic> AtPLP1 </italic> is patatin-related phospholipase that is differentially expressed in response to phosphate stress [ <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. <italic> AtGASA1 </italic> (GA-Stimulated in Arabidopsis) is a gibberellic acid-regulated protein and expressed in rosettes [ <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. <italic> AtSWEET12 </italic> is a sucrose transporter involved in phloem loading that transfers sucrose from the leaves to nonphotosynthetic tissues elsewhere in the plant [ <xref ref-type="bibr" rid="CR23"> 23 </xref> ]. Overexpression of <italic> AtOXS3 </italic> in Arabidopsis resulted in greater tolerance to heavy metals and oxidative stress. Copper amine oxidase is upregulated in response to wounding in chickpea [ <xref ref-type="bibr" rid="CR24"> 24 </xref> ] and in response to nematode infection but not wounding, in Arabidopsis [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. The top ten genes repressed by iron deficiency included two homologs of <italic> AtNIA1 </italic> <span class="gene">(Glyma06g11430</span> [E&#x02009;=&#x02009;0] and <span class="gene">Glyma13g02510</span> [E&#x02009;=&#x02009;0]) and a homolog of <italic> AtDXR </italic> <span class="gene">(Glyma16g10880,</span> E&#x02009;=&#x02009;0). <italic> AtNIA </italic> is required for nitric oxide (NO) production [ <xref ref-type="bibr" rid="CR26"> 26 </xref> ]. NO acts an important signaling molecule for a variety of abiotic stresses including iron deficiency and <span class="trait">drought</span> [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. <italic> AtDXR </italic> is localized to the chloroplast and catalyzes the first committed step of isoprenoid biosynthesis leading to the production of chlorophyll, carotenoids, ABA, brassinosteroids, cytokinins and gibberellins [ <xref ref-type="bibr" rid="CR28"> 28 </xref> ]. <table-wrap id="Tab1"> <label> Table 1 </label> <caption> <p> <bold> Top ten significantly induced and repressed DEGs under iron stress at each time and tissue </bold>
					</p>
				
					<p>&nbsp;&nbsp;
						Six hours after plants were transferred from iron sufficient to iron deficient media, iron deficiency induced the expression of 246 genes in leaves but repressed the expression of 154. Induced genes included a homolog of <italic> AtRD22 </italic> <span class="gene">(Glyma06g08540</span> [4E <sup> -124 </sup> ]), <italic> <span class="gene">WNK5</span> </italic> <span class="gene">(Glyma01g32450</span> [E&#x02009;=&#x02009;0]) and two homologs of <italic> ASN1/AtDIN6 </italic> <span class="gene">(Glyma02g39320</span> [E&#x02009;=&#x02009;0] and <span class="gene">Glyma11g27480</span> [E&#x02009;=&#x02009;0]). Arabidopsis <italic> AtRD22 </italic> is responsive to abscisic acid, water and salt stress [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. The rice homolog of <span class="gene">WNK5</span> (with no lysine kinase), <italic> OsWNK1 </italic> , has a suspected role in abiotic stress tolerance and is involved in circadian rhythm [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. The wheat homolog of asparagine synthetase <italic> TaASN1 </italic> has been shown to be upregulated in roots in response to salt, <span class="trait">drought</span> and ABA stress [ <xref ref-type="bibr" rid="CR37"> 37 </xref> ]. The top ten genes repressed by iron deficiency included three homologs of <italic> AtEXO </italic> <span class="gene">(Glyma14g35340</span> [2E <sup> -146 </sup> ], <span class="gene">Glyma14g35330</span> [5E <sup> -158 </sup> ] and <span class="gene">Glyma06g10710</span> [9E <sup> -154 </sup> ]). The extracellular EXO protein is essential for cell expansion and promotes shoot and root growth [ <xref ref-type="bibr" rid="CR38"> 38 </xref> ]. <italic> AtEXO </italic> mutants have altered expression of sugar-responsive genes and increased ABA levels. It is interesting to note that eight EXO and EXO-like (EXL5) homologs were differentially expressed in response to iron stress at this time point.
					</p>
				
					<p>&nbsp;&nbsp;
						In one hour roots, we identified 35 differentially expressed transcription factors (Figure&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ). These included representative members from eight different families such as AP2-EREBP (16), <span class="gene">AS2</span> (1), &#x003b2;-helix-loop-helix (2), Homeodomain/HOMEOBOX (2), MYB/HD-like (6), NAC (2), PHD (2), PLATZ (2), TPR (1), and <span class="gene">TUB</span> (1). Of the 16 AP2-EREBP proteins expressed, five are homologs of ERF1 <span class="gene">(Glyma19g34696</span> [2E <sup> -54 </sup> ], <span class="gene">Glyma20g34570</span> [2E <sup> -71 </sup> ], <span class="gene">Glyma13g18410</span> [9E <sup> -52 </sup> ], <span class="gene">Glyma10g33060</span> [1E <sup> -73 </sup> ], and <span class="gene">Glyma11g03910,</span> [9E <sup> -52 </sup> ]). ERF1 has been shown to regulate abiotic stress responses in Arabidopsis [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ]. <span class="gene">Glyma13g34920</span> encodes an ERF4 homolog (3E <sup> -47 </sup> ), which functions as a repressor in ethylene, jasmonic acid, and abscisic acid pathways [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ]. <span class="gene">Glyma03g28630</span> encodes a homolog of the &#x003b2;-helix-loop-helix transcription factor <italic> &#x003b2;HLH038 </italic> (5E <sup> -52 </sup> ). This gene is upregulated under soybean iron stress conditions, and is thought to function as a binding partner for FIT [ <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. <span class="gene">Glyma16g01911</span> is most similar to <italic> AtNTL6 </italic> (6E <sup> -53 </sup> ), which is a plant-specific NAC that is phosphorylated by SnRK2.8 in drought-resistance [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. The PHD family member, <span class="gene">Glyma20g01070,</span> is upregulated 6-fold in response to iron deficiency and is most similar to <italic> AtXLG2 </italic> (extra-large G protein 2, 1E <sup> -96 </sup> ), a GTP-binding protein that has been implicated in <span class="trait">resistance</span> to <italic> Pseudomonas syringae </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						We found 74 unique transcription factor motifs significantly overrepresented across the four tissue/time points within the DEGs. Focusing on transcription factors known to be involved in abiotic and biotic stress response, ARF, BZR1, DREB1B, HY5, MYBPH3, TGA1, and TRAB1 binding sites were all significantly overrepresented (t&#x02009;&#x0003c;&#x02009;0.05) in promoters of genes differentially expressed in one hour leaves (Additional file <xref rid="MOESM8" ref-type="media"> 8 </xref> ). ARF (auxin response factor) has been implicated in both biotic and abiotic stress responses in several plant systems [ <xref ref-type="bibr" rid="CR68"> 68 </xref> , <xref ref-type="bibr" rid="CR69"> 69 </xref> ]. In our dataset, its binding site is present in a high percentage of DEG promoters (one hour leaf, 74%; six hour leaf, 70%; one hour root, 69.2%). BZR1 is a central regulator of brassinosteroid (BR) signaling, synthesis and growth responses [ <xref ref-type="bibr" rid="CR70"> 70 </xref> ]. Soybean <italic> <span class="gene">GmDREBa</span> </italic> and <italic> <span class="gene">GmDREBb</span> </italic> are induced by cold, <span class="trait">drought</span> and salt in the leaves of seedlings. While expression of <italic> <span class="gene">GmDREBc</span> </italic> is low in leaves, it has high levels of expression in roots following <span class="trait">drought,</span> salt and ABA treatments [ <xref ref-type="bibr" rid="CR71"> 71 </xref> ]. HY5 (LONG HYPOCOTYL 5) is a bZIP transcription factor that has been shown to positively regulate anthocyanin biosynthesis [ <xref ref-type="bibr" rid="CR72"> 72 </xref> ]. The MYBPH3 transcription factor functions in the regulation of flavonoid biosynthesis in petunia [ <xref ref-type="bibr" rid="CR73"> 73 </xref> ] and may also be involved in salt and cold-tolerance in pea [ <xref ref-type="bibr" rid="CR74"> 74 </xref> ]. TGA1 is controlled by nitric oxide and regulates systemic acquired <span class="trait">resistance</span> in plants through salicylic acid (SA)&#x02013;mediated signal transduction pathway [ <xref ref-type="bibr" rid="CR75"> 75 </xref> , <xref ref-type="bibr" rid="CR76"> 76 </xref> ]. TRAB1, responsible for ABA regulation, is phosphorylated in response to osmotic stress and by the SnRK2 kinase in response to ABA [ <xref ref-type="bibr" rid="CR77"> 77 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						In six hour leaves, ABF1, ABZ1, Alfin1, ARF, AtMYB15, AtMYB77, BZR1, C1, DREB1B, E2F, HY5, KNOX3, LIM1, MYBAS1, NAC6, OSBZ8, P, RAV1, TGA1 and TRAB1 binding sites were significantly (t &#x0003c; 0.05) overrepresented in the promoters of our DE genes and all have reported roles in stress responses (Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). ABF (ABA-responsive elements binding factor) is ABA and stress inducible, and in turn, activates ABREs (ABA-responsive elements) in response to abiotic stress [ <xref ref-type="bibr" rid="CR78"> 78 </xref> ]. ABZ1 (anaerobic basic leucine zipper) was isolated from a tomato cDNA library enriched for anaerobically induced genes [ <xref ref-type="bibr" rid="CR79"> 79 </xref> ]. The soybean genome contains six genes identified as Alfin1-type PHD finger protein and their expression responds differentially to <span class="trait">drought,</span> salt, cold and ABA treatments when expressed transgenically in Arabidopsis [ <xref ref-type="bibr" rid="CR80"> 80 </xref> ]. Like ARF, Alfin binding sites are highly represented in our DEG promoters (six hour leaves, 74.3%; one hour roots, 70.6%). Myb factors have been implicated in a variety of biotic and abiotic stress responses [ <xref ref-type="bibr" rid="CR81"> 81 </xref> ]. Transgenic <italic> AtMYB15 </italic> can confer improved tolerance to <span class="trait">drought</span> and salt stress in Arabidopsis [ <xref ref-type="bibr" rid="CR82"> 82 </xref> ]. <italic> AtMYB77 </italic> expression responds to wounding, pathogen infection, abiotic stress and hormone treatment [ <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. The E2F transcription factor regulates the cell cycle and DNA replication [ <xref ref-type="bibr" rid="CR83"> 83 </xref> , <xref ref-type="bibr" rid="CR84"> 84 </xref> ]. Atwood <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR11"> 11 </xref> ] and O&#x02019;Rourke <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR13"> 13 </xref> ] found that DNA replication was inhibited in iron efficient soybean lines. HAHB4, a HD-Zip transcription factor, regulates crosstalk between ethylene and <span class="trait">drought</span> signaling in sunflower [ <xref ref-type="bibr" rid="CR85"> 85 </xref> ]. <italic> AP </italic> ETALA <italic> 2 </italic> / <italic> e </italic> thylene- <italic> r </italic> esponsive element binding <italic> f </italic> actor (AP2/ERF) family includes four major subfamilies: the AP2, RAV, ERF and DREB subfamilies and many have been shown to play a role in abiotic stress [ <xref ref-type="bibr" rid="CR86"> 86 </xref> ]. Binding sites for two of those subfamilies (RAV and DREB) were overrepresented in the promoters of genes from the six hour leaf time point. Knotted1-like homeobox (KNOX) genes are involved in plant morphogenesis, and barley KNOX3 has been shown to be regulated by the ethylene signaling pathway [ <xref ref-type="bibr" rid="CR87"> 87 </xref> ]. Lignin plays an important role in mechanical support, water transport and pathogen <span class="trait">resistance.</span> <italic> NtLIM1 </italic> encodes a Pal-box binding protein involved in lignin biosynthesis [ <xref ref-type="bibr" rid="CR88"> 88 </xref> ]. Tobacco <italic> NtMYBAS1 </italic> is involved in phenylpropanoid biosynthesis [ <xref ref-type="bibr" rid="CR89"> 89 </xref> ], which has long been known to be stress-induced [ <xref ref-type="bibr" rid="CR90"> 90 </xref> ]. Soybean <italic> <span class="gene">GmNAC6</span> </italic> is induced by both endoplasmic reticulum-stress and osmotic-stress signaling to promote cell death [ <xref ref-type="bibr" rid="CR91"> 91 </xref> ]. In rice, the bZIP class Abscisic acid Responsive Element (ABRE)-binding factor, OSBZ8 has been shown to function in ABA signaling and in salt stress [ <xref ref-type="bibr" rid="CR92"> 92 </xref> ]. The P transcription factor in maize is involved in flavonoid biosynthesis, leading to the production of a red phlobaphene pigment [ <xref ref-type="bibr" rid="CR93"> 93 </xref> , <xref ref-type="bibr" rid="CR94"> 94 </xref> ]. <italic> AtRAV1 </italic> , a <span class="gene">RAV</span> ( <italic> R </italic> elated to <italic> A </italic> BI3/ <italic> V </italic> P1) transcription factor family gene has been shown to positively regulate leaf senescence, and is induced in response to ethylene and methyl jasmonate [ <xref ref-type="bibr" rid="CR95"> 95 </xref> ]. DREB1B, TGA1, and TRAB1 binding sites were all overrepresented in both one hour and six hour leaves.
					</p>
				
					<p>&nbsp;&nbsp;
						We also identified a putative family (Group 42) of 9 2OG-Fe(II)-dependent oxygenases differentially expressed in one hour roots. While seven of these were induced by iron deficiency, two were repressed. Reciprocal best BLASTP [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ] identified two orthologs of FERULOYL-COA 6&#x02019; HYDROXYLASE 1 (AtF6&#x02019;H1, <span class="gene">Glyma03g23770</span> and <span class="gene">Glyma07g12210),</span> both induced by iron deficiency. Schmid <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR114"> 114 </xref> ] found that F6&#x02019;H1 was required for coumarin synthesis and was also induced by iron deficiency. Recently, Rodr&#x000ed;guez-Celma <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR115"> 115 </xref> ] demonstrated that Arabidopsis excretes phenolic compounds, such as coumarin, in response to low iron. Fourcroy <italic> et al. </italic> [ <xref ref-type="bibr" rid="CR116"> 116 </xref> ] found that <italic> AtPDR9 </italic> was induced by iron deficiency in Arabidopsis and was required for the secretion of coumarin compounds aiding in iron acquisition. Therefore, we examined our DEGs to identify similar genes. We found differential expression of a number of these genes in one hour roots including orthologs of pleiotrophic drug <span class="trait">resistance</span> protein <italic> AtPDR9 </italic> <span class="gene">(Glyma17g03863</span> and <span class="gene">Glyma07g36166),</span> <italic> AtPDR11 </italic> <span class="gene">(Glyma19g37760),</span> <italic> AtPDR12 </italic> <span class="gene">(Glyma13g43860)</span> and caffeic acid O-methyltransferase <italic> AtOMT1 </italic> <span class="gene">(Glyma04g40591</span> and <span class="gene">Glyma06g14210).</span> Further, we found the GO terms associated with coumarin (GO:00098040 and phenylpropanoid (GO:0009698) biosynthesis and metabolism were significantly overrepresented in one hour roots. This suggests that secretion of coumarins is essential for iron uptake in soybean as well.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genotyping by sequencing for genomic prediction in a soybean breeding population</h2>
			<h3>PMID:25174348</h3>
				
					<h3>PMCID:4176594</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jarqu&#x000ed;n </surname>  <given-names> Diego </given-names>  </name>  <address>  <email> diego.jarquin@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kocak </surname>  <given-names> Kyle </given-names>  </name>  <address>  <email> kyle.kocak@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Posadas </surname>  <given-names> Luis </given-names>  </name>  <address>  <email> lposadas2@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hyma </surname>  <given-names> Katie </given-names>  </name>  <address>  <email> keh233@cornell.edu </email>  </address>  <xref ref-type="aff" rid="Aff9"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jedlicka </surname>  <given-names> Joseph </given-names>  </name>  <address>  <email> jjedlic@huskers.unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Graef </surname>  <given-names> George </given-names>  </name>  <address>  <email> ggraef1@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Lorenz </surname>  <given-names> Aaron </given-names>  </name>  <address>  <email> alorenz2@unl.edu </email>  </address>  <xref ref-type="aff" rid="Aff8"/>  </contrib>  <aff id="Aff8">  <label/> Department of Agronomy and Horticulture, University of Nebraska, 363 Keim Hall, Lincoln, NE 68583 USA  </aff>  <aff id="Aff9">  <label/> Institute of Genomic Diversity, Cornell University, Ithaca, NY USA  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 8 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A large number of studies on <span class="gene">GS</span> in multiple crops has been reported [ <xref ref-type="bibr" rid="CR2"> 2 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> &#x02013; <xref ref-type="bibr" rid="CR22"> 22 </xref> ]. A study on <span class="gene">GS</span> in soybean, however, has not. Moreover, there are only a few reports on the use of GBS for <span class="gene">GS</span> [ <xref ref-type="bibr" rid="CR14"> 14 </xref> &#x02013; <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. In light of the current research on <span class="gene">GS</span> and the dearth of reported research on <span class="gene">GS</span> and GBS in soybean breeding, the objectives of this study were (i) evaluate prospects for <span class="gene">GS</span> using GBS in a typical soybean breeding program and (ii) evaluate the effect of GBS marker selection and imputation on genomic prediction accuracy. To achieve these objectives, a set of soybean experimental lines sampled from the University of Nebraska-Lincoln Soybean Breeding Program were genotyped using GBS and evaluated for grain <span class="trait">yield</span> and other agronomic traits at multiple Nebraska locations. Reported findings are important to the application of GBS to selection for grain <span class="trait">yield</span> in future soybean breeding efforts.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:A Meta-Analysis of the Impacts of Genetically Modified Crops</h2>
			<h3>PMID:25365303</h3>
				
					<h3>PMCID:4218791</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kl&#x000fc;mper </surname>  <given-names> Wilhelm </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qaim </surname>  <given-names> Matin </given-names>  </name>  <xref ref-type="aff" rid="aff1"/>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 3 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						On average, GM technology has increased crop yields by 21% ( <xref ref-type="fig" rid="pone-0111629-g002"> Figure 2 </xref> ). These <span class="trait">yield</span> increases are not due to higher genetic <span class="trait">yield</span> potential, but to more effective pest control and thus lower crop damage <xref rid="pone.0111629-Qaim4" ref-type="bibr"> [27] </xref> . At the same time, GM crops have reduced pesticide quantity by 37% and pesticide cost by 39%. The effect on the cost of production is not significant. GM seeds are more expensive than non-GM seeds, but the additional seed costs are compensated through savings in chemical and mechanical pest control. Average profit gains for <span class="gene">GM-adopting</span> farmers are 69%.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Bayesian Inference of Baseline <span class="trait">Fertility</span> and Treatment Effects via a Crop Yield-Fertility Model</h2>
			<h3>PMID:25405353</h3>
				
					<h3>PMCID:4236125</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Hungyen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamagishi </surname>  <given-names> Junko </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kishino </surname>  <given-names> Hirohisa </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The relationship between crop <span class="trait">yield</span> and soil <span class="trait">fertility</span> under different fertilization regimes has been studied for decades. Aref and Wander <xref rid="pone.0112785-Aref1" ref-type="bibr"> [12] </xref> investigated the long-term trends of corn <span class="trait">yield</span> and soil organic matter in different crop sequences and soil <span class="trait">fertility</span> treatments. Merick and N&#x000e9;meth <xref rid="pone.0112785-Mercik1" ref-type="bibr"> [13] </xref> used results from 60-year field experiments to provide information on the relationship between fertilization and yields of rye and potato. Hallin et al. <xref rid="pone.0112785-Hallin1" ref-type="bibr"> [14] </xref> investigated the relationship between microbial communities and total crop <span class="trait">yield</span> and nitrogen content in the crop in a 50-year-old fertilization experiment. <span class="gene">Fan</span> et al. <xref rid="pone.0112785-Fan1" ref-type="bibr"> [15] </xref> studied the trends in grain yields and soil organic carbon (SOC) in a 26-year dryland fertilization trial. By carrying out a 27-year experiment with various fertilization treatments in a rotation cropping system with wheat and maize in a red soil, Zhang et al. <xref rid="pone.0112785-Zhang1" ref-type="bibr"> [16] </xref> investigated trends in SOC, soil nitrogen, and grain <span class="trait">yield.</span>
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Regulatory interplay between soybean root and soybean cyst nematode during a resistant and susceptible reaction</h2>
			<h3>PMID:25421055</h3>
				
					<h3>PMCID:4262236</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Hosseini </surname>  <given-names> Parsa </given-names>  </name>  <address>  <email> parsa.hosseini@nih.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Matthews </surname>  <given-names> Benjamin F </given-names>  </name>  <address>  <email> ben.matthews@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <aff id="Aff1">  <label/> School of Systems Biology, George Mason University, Manassas, VA USA  </aff>  <aff id="Aff2">  <label/> Computational Biology Branch, National Center for Biotechnology Information, National Institutes of Health, Bethesda, MD USA  </aff>  <aff id="Aff3">  <label/> Soybean Genomics and Improvement Laboratory, United States Department of Agriculture, Beltsville, MD USA  </aff>
				<br/> 
				年份：<day> 22 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The <italic> A. thaliana </italic> <span class="gene">MYB77</span> homolog, AtMYB77, exhibits a mild change in IPF score across both resistant and susceptible reactions. Across both reactions, AtMYB77 IPF scores were generally under&#x02013;represented at 6 dai but become slightly over&#x02013;represented at 8 dai. An earlier study revealed interaction between <span class="gene">MYB77</span> and auxin response factor 7 (ARF7) [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ], further accentuating the role AtMYB77 could play in host&#x02013;pathogen interplay [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ]. The OsCBT TFBS exhibited pronounced IPF scores across all four treatments. In both the resistant and susceptible reaction, OsCBT was highly over&#x02013;represented only at 6 dai. It was shown that OsCBT mutants conferred increased pathogen <span class="trait">resistance</span> upon inoculation with <italic> Magnaporthe grisea </italic> , revealing that OsCBT suppresses defense response [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of Corn and Soybean Meal Types on Rumen Fermentation, Nitrogen Metabolism and Productivity in Dairy Cows</h2>
			<h3>PMID:25656206</h3>
				
					<h3>PMCID:4341079</h3>
					<h4>期刊:Asian-Australasian Journal of Animal Sciences</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> J. S. </given-names>  </name>  <xref ref-type="aff" rid="af1-ajas-28-3-351"> 1 </xref>  <xref ref-type="aff" rid="af2-ajas-28-3-351"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> L. J. </given-names>  </name>  <xref ref-type="aff" rid="af1-ajas-28-3-351"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> H. Z. </given-names>  </name>  <xref ref-type="aff" rid="af1-ajas-28-3-351"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> B. </given-names>  </name>  <xref ref-type="aff" rid="af1-ajas-28-3-351"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Z. </given-names>  </name>  <xref ref-type="aff" rid="af1-ajas-28-3-351"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chacher </surname>  <given-names> B. </given-names>  </name>  <xref ref-type="aff" rid="af1-ajas-28-3-351"> 1 </xref>  <xref ref-type="aff" rid="af3-ajas-28-3-351"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> J. X. </given-names>  </name>  <xref ref-type="aff" rid="af1-ajas-28-3-351"> 1 </xref>  <xref rid="c1-ajas-28-3-351" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 10 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Twelve multiparous Holstein dairy cows (body weight [BW] = 594&#x000b1;31 kg; days in milk [DIM] = 130&#x000b1;14 d; milk <span class="trait">yield</span> = 31.3&#x000b1;1.8 kg) were used in a replicated 4&#x000d7;4 Latin square design with 21-d periods and a 2&#x000d7;2 factorial dietary treatment arrangement. Each experimental period consisted of 14 d of dietary adaptation and 7 d for sample and data collection. Cows were housed in individual tie stalls and had continuous access to water from water bowls. Treatment factors were i) corn type (DGC [Hangzhou Hangjiang Dairy Farm, China] vs SFC [Kaite Feed Corp., Hebei, China; the flake density was 360 g/L]) and ii) SBM type (SSBM [Hangzhou Hangjiang Dairy Farm, China] vs HSBM [Shanghai Bright Holstan Co. Ltd., Shanghai, China]), which differed in their ruminal degradation rate and extent. Ingredients and chemical compositions of diets are shown in <xref rid="t1-ajas-28-3-351" ref-type="table"> Table 1 </xref> . All diets had similar CP and energy contents and were formulated to meet the net energy for lactation requirements of cows weighing 600 kg, averaging 130 <span class="gene">DIM</span> and producing 31 kg of milk ( <xref rid="b24-ajas-28-3-351" ref-type="bibr"> Ministry of Agriculture of P. R. China, 2004 </xref> ). Diets were offered three times daily at 0630, 1330, and 2000 h and there was approximately 5% feed refusal. Cows were milked three times daily at 0700, 1400, and 2030 h.
					</p>
				
					<p>&nbsp;&nbsp;
						Ruminal degradation data for two corn types and two SBM types were determined <italic> in situ </italic> by using the nylon bag technique ( <xref rid="b28-ajas-28-3-351" ref-type="bibr"> &#x000d8;rskov et al., 1980 </xref> ). Approximately 5 g of each sample was placed in a nylon bag (10&#x000d7;20 cm; 50-&#x003bc;m pore size; Ankom Technology Corp., Macedon, NY, USA) with 6 replicates in each incubation time. Bags were tied to the end of a 40-cm nylon line and attached to a stainless-steel weight, and then placed in the rumen ventral sac of three ruminally cannulated Chinese Holstein cows (BW = 634&#x000b1;17 kg; <span class="gene">DIM</span> = 83&#x000b1;7 d; milk <span class="trait">yield</span> = 21.4&#x000b1;4.2 kg). Diet ingredients for cannulated cows included 45.0% concentrate, 17.0% corn silage, 28.0% alfalfa hay, and 10.0% Chinese wild rye on a DM basis. The bags with samples were removed from the rumen after 0, 2, 4, 8, 16, 24, and 36 h incubation. Bags were rinsed thoroughly, dried and weighed. The <italic> in situ </italic> digestion parameters were estimated using the nonlinear model described by <xref rid="b28-ajas-28-3-351" ref-type="bibr"> &#x000d8;rskov et al. (1980) </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic variants in root architecture-related genes in a <italic> Glycine soja </italic> accession, a potential resource to improve cultivated soybean</h2>
			<h3>PMID:25765991</h3>
				
					<h3>PMCID:4354765</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Prince </surname>  <given-names> Silvas J </given-names>  </name>  <address>  <email> princesj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <address>  <email> songli@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Dan </given-names>  </name>  <address>  <email> qiud@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Maldonado dos Santos </surname>  <given-names> Joao V </given-names>  </name>  <address>  <email> maldonadodossantosj@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chai </surname>  <given-names> Chenglin </given-names>  </name>  <address>  <email> chaic@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Joshi </surname>  <given-names> Trupti </given-names>  </name>  <address>  <email> joshitr@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant </given-names>  </name>  <address>  <email> patilg@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <address>  <email> valliyodanb@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D </given-names>  </name>  <address>  <email> vuongt@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Murphy </surname>  <given-names> Mackensie </given-names>  </name>  <address>  <email> murphymc@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Krampis </surname>  <given-names> Konstantinos </given-names>  </name>  <address>  <email> agbiotec@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tucker </surname>  <given-names> Dominic M </given-names>  </name>  <address>  <email> dotucke2@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Biyashev </surname>  <given-names> Ruslan </given-names>  </name>  <address>  <email> rbiyashe@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dorrance </surname>  <given-names> Anne E </given-names>  </name>  <address>  <email> dorrance.1@osu.edu </email>  </address>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Maroof </surname>  <given-names> MA Saghai </given-names>  </name>  <address>  <email> smaroof@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <address>  <email> xudong@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shannon </surname>  <given-names> J Grover </given-names>  </name>  <address>  <email> ShannonG@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Nguyen </surname>  <given-names> Henry T </given-names>  </name>  <address>  <email> nguyenhenry@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <aff id="Aff1">  <label/> National Center for Soybean Biotechnology and Division of Plant Sciences, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label/> Christopher S. Bond Life Sciences Center, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff3">  <label/> Department of Computer Science, University of Missouri, Columbia, MO 65211 USA  </aff>  <aff id="Aff4">  <label/> Department of Crop and Soil Environmental Sciences, Virginia Tech, Blacksburg, VA 24061 USA  </aff>  <aff id="Aff5">  <label/> Department of Plant Pathology, The Ohio State University, OARDC, Wooster, OH 44691 USA  </aff>
				<br/> 
				年份：<day> 9 </day>  <month> 2 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Among the 10 candidate genes (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> ) identified based on transcript abundance from the microarray analysis, <italic> kinesin </italic> was also found to be highly expressed in maize root tissues [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. Other genes, such as <italic> carboxylesterase 6 like </italic> and <italic> histone-like CCAAT transcription factor </italic> , were also found to be highly expressed (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ) in root pericycle cells, which regulate lateral root formation. These genes were also found to be upregulated in water-deficit conditions based on an Affymetrix gene chip study (Babu et al. unpublished) of different root regions. The <italic> carboxylesterase 6 like </italic> gene was highly upregulated under <span class="trait">drought</span> stress conditions in root region 2 (4- to 8-mm tip). It was also upregulated in the root region 1 (0&#x02013;4&#x000a0;mm) following <span class="trait">drought</span> stress, but at lower levels compared with region 2. The role of this gene in the maintenance of root growth during <span class="trait">drought</span> stress conditions has yet to be studied. Candidate genes identified in the <italic> G. soja </italic> line, such as <italic> Apoptosis inhibitor 5 related </italic> ( <italic> <span class="gene">Glyma07g32480</span> </italic> ), <italic> slow anion channel associated 1-like </italic> ( <italic> <span class="gene">Glyma15g42220</span> </italic> ), and <italic> Metacaspase </italic> ( <italic> <span class="gene">Glyma08g19050</span> </italic> ) (Table&#x000a0; <xref rid="Tab4" ref-type="table"> 4 </xref> ), showed high expression in root-related tissues (Figure&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ) and could be key candidates to improve root length and diameter in cultivated soybean, which might increase nutrient and water use efficiency. Emphasis should be placed on characterizing the function of an uncharacterized gene, <italic> <span class="gene">Glyma15g42220</span> </italic> , which is associated with soybean root system architecture, because it showed higher expression in RILs with the shortest root length and surface area compared with RILs with the longest root length. The sequence of this gene is similar to that of <italic> slow anion channel associated 1 </italic> in other crops, which is involved in osmoregulation, phosphorus uptake [ <xref ref-type="bibr" rid="CR51"> 51 </xref> ], and aluminum tolerance [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. It is also reported to be involved in regulating the exchange of water vapor and gas during <span class="trait">drought</span> stress [ <xref ref-type="bibr" rid="CR53"> 53 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The genes with high levels of transcript abundance expressed only in root tissues within the QTL region of chromosomes 6 (Table&#x000a0; <xref rid="Tab7" ref-type="table"> 7 </xref> ) had distinct expression patterns in the cultivated and <italic> G. soja </italic> parents. Similar trends of gene expression were reported for genes related to stress, defense response, and redox pathways observed in wild soybean [ <xref ref-type="bibr" rid="CR54"> 54 </xref> ] and wild tomato [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ] compared with their respective cultivated types. Among the 18 genes identified based on sequence information from a wild soybean, two <italic> peroxidase </italic> genes ( <italic> <span class="gene">Glyma06g45910</span> </italic> and <italic> <span class="gene">Glyma06g45920</span> </italic> ) in different clades of the heat map showed high expression in the <italic> G. soja </italic> line (Figure&#x000a0; <xref rid="Fig7" ref-type="fig"> 7 </xref> B). One of these <italic> peroxidase </italic> genes maps to a duplicated region [ <xref ref-type="bibr" rid="CR56"> 56 </xref> ] and showed higher expression in the RILs with small or larger root phenotypes. A similar trend of gene expression was observed in tomato introgression lines developed between cultivated and wild tomatoes [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ]. This peroxidase gene in PI407162 also had a non-synonymous SNP variation that changed an amino acid from serine to alanine. Similar effects on root size of amino acid changes were reported in the phosphorylation of the ETHYLENE <italic> INSENSITIVE2 </italic> gene in Arabidopsis [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ]. Zhu et al. [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] reported that the peroxidase activity was spatiotemporal in root tissues and was involved in both cell wall loosening and tightening under water-deficit conditions. Voothuluru and Sharp [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ] also reported the effect of <italic> peroxidase </italic> genes on apoplastic reactive oxygen species in the root apical region and its effect on cell wall modification in maize roots as a part of a <span class="trait">drought</span> adaptive mechanism. The single copy genes identified in the present study could also be used as informative markers for phylogenetic and comparative analyses across various taxonomic levels [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Based on the whole genome re-sequence, information from 17 accessions, which represent diverse <italic> G. soja </italic> accessions from Korea [ <xref ref-type="bibr" rid="CR8"> 8 </xref> ] and China [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ], also showed conserved non-synonymous SNPs in most of the root related genes identified in this study (Table&#x000a0; <xref rid="Tab10" ref-type="table"> 10 </xref> ). Among different plant species, soybeans are reported to have a higher average ratio of Nonsyn/Syn SNPs than Arabidopsis and rice, with larger effect SNPs in 4,648 genes that have greater effects on their functions [ <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. In addition, 21% of potential SNP loci were fixed during the domestication process from wild soybeans. Therefore, the genes identified in the present study might represent candidates with high biological significance for root system growth and development. These genes may play a key role in maintaining short root phenotypes of wild soybeans. A strong candidate for further study is the auxin responsive gene, <italic> NEDD8-activating complex, </italic> because it influences the total root length and involves a key hormone, auxin, which regulates the root growth and development. In Arabidopsis, this gene interacts with a <italic> CULLIN </italic> gene, which then alters the growth of lateral roots and root hairs [ <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. Similar results were observed in elite lines of rice in which SNPs altered the expression of <italic> 9-cis-epoxycarotenoid dioxygenase </italic> gene associated with the ABA content. A rice line with this missense mutation had produced more lateral roots [ <xref ref-type="bibr" rid="CR67"> 67 </xref> ]. For the uncharacterized gene ( <italic> <span class="gene">Glyma06g46170</span> </italic> ), a gain of a stop codon in the <italic> G. soja </italic> PI 407162 makes this a potential candidate gene to further study the functional effects associated with root development in soybeans. Therefore, the root-related genes with non-synonymous mutations identified in this study are a valuable genetic resource to study the evolution of root system development in soybeans. The candidate genes identified from the <italic> G. soja </italic> accession, PI 407162, are promising targets to improve root system architecture in cultivated soybeans. However, RILs with soja alleles show undesirable phenotypes, such as <span class="trait">lodging</span> and poor <span class="trait">yield,</span> which could be eliminated by a series of backcrosses with the cultivated soybean, V71-370.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide SNP Identification and Characterization in Two Soybean Cultivars with Contrasting <italic> Mungbean Yellow Mosaic India Virus </italic> Disease <span class="trait">Resistance</span> Traits</h2>
			<h3>PMID:25875830</h3>
				
					<h3>PMCID:4395324</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Yadav </surname>  <given-names> Chandra Bhan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Bhareti </surname>  <given-names> Priyanka </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Muthamilarasan </surname>  <given-names> Mehanathan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mukherjee </surname>  <given-names> Minakshi </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Yusuf </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rathi </surname>  <given-names> Pushpendra </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prasad </surname>  <given-names> Manoj </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 3 </day>  <month> 3 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Dot-blot analysis for screening disease <span class="trait">resistance</span> was performed following Sahu et al. [ <xref rid="pone.0123897.ref012" ref-type="bibr"> 12 </xref> ]. Total genomic DNA was denatured by adding an equal volume of 0.6 M sodium hydroxide and an equal amount of each denatured DNA (~ 1 &#x003bc;g) was spotted onto three Hybond N+ membranes (Amersham Bioscience) embedded in a BIO-DOT dot-blot apparatus (Bio-Rad). The samples were spotted in 96-well formats to prepare three identical arrays. The membranes were hybridized with &#x003b1; <sup> 32 </sup> P-dCTP labeled DNA A specific <span class="gene">AC1</span> gene sequence, and then neutralized with neutralization buffer (0.5 M Tris-Cl, pH 7.4, 1.5 M NaCl) for 3 min, washed with 2% standard saline citrate (SSC) and cross-linked using UV cross-linker (Stratagene). Radiolabeled probe was made by random primer labeling method by using Megaprime DNA labeling system (Amersham Biosciences) [ <xref rid="pone.0123897.ref012" ref-type="bibr"> 12 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						MYMIV infected soybean seedlings initially showed <span class="trait">chlorosis</span> of leaves, which is the characteristic symptom of the disease ( <xref rid="pone.0123897.g001" ref-type="fig"> Fig 1 </xref> ). The disease progressed with reduced leaf area and finally resulted in stunted growth. The resistant soybean cv. UPSM-534 and susceptible cv. JS-335 along with 10 Indian soybean cultivars were phenotyped as resistant and susceptible based on percentage infectivity at 30 days post inoculation (dpi). Of the 10 cultivars, 5 were found to be resistant (0%), whereas the remaining 5 were susceptible (91&#x02013;100%) ( <xref rid="pone.0123897.t001" ref-type="table"> Table 1 </xref> ). This result was verified by dot-blot of total genomic DNA with <span class="gene">AC1</span> gene sequence used as a probe ( <xref rid="pone.0123897.t001" ref-type="table"> Table 1 </xref> ). The dot-blot densities for resistant cultivars ranged from 1.93 to 5.93, whereas for susceptible cultivars, the values ranged from 94.94 to 263.48 ( <xref rid="pone.0123897.t001" ref-type="table"> Table 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A total of 30 primers targeting 76 SNPs were selected for validation based on their presence in genes which were associated with disease resistance-related QTLs ( <xref rid="pone.0123897.s010" ref-type="supplementary-material"> S6 Table </xref> ). The SNPs were amplified by flanking primers in 12 soybean cultivars ( <xref rid="pone.0123897.g008" ref-type="fig"> Fig 8 </xref> ). The amplicons were re-sequenced and found that the sequences showed perfect similarity with the reference genomes with variation at single nucleotide. A frequency of 5.06 SNPs/Mb was observed in the re-sequenced 12 Indian soybean cultivars. Of note, a non-synonymous SNP in chromosome 18 (18&#x02013;1861613) was observed to be associated with MYMIV-resistance at the 149 base-pair (a G/C transversion) in the <italic> LEUCINE-RICH REPEAT RECEPTOR-LIKE PROTEIN KINASE </italic> ( <italic> LRR-RP </italic> ) gene <span class="gene">(Glyma18g02850)</span> ( <xref rid="pone.0123897.g009" ref-type="fig"> Fig 9 </xref> ). The G to C transversion causes a codon change of CCC to GCC, resulting in a mis-sense non-synonymous substitution of proline to alanine. Noteworthy, this SNP is not present in the functional domain of <italic> LRR-RP </italic> gene. The physical location of the <italic> LRR-RP </italic> gene is shown in <xref rid="pone.0123897.g010" ref-type="fig"> Fig 10 </xref> . LRR-RPs are a class of nucleotide binding site&#x02014;leucine-rich repeat (NBS-LRR) playing crucial roles in <span class="trait">resistance</span> to a broad-spectrum of phytopathogens [ <xref rid="pone.0123897.ref028" ref-type="bibr"> 28 </xref> , <xref rid="pone.0123897.ref029" ref-type="bibr"> 29 </xref> ]. The SNP identified in the present study would serve as an important source for the improvement of disease <span class="trait">resistance</span> in soybean through either molecular breeding or transgene-based approaches. Further, development of CAPS/allele-specific marker for this SNP is in progress. Precisely, these types of synonymous and non-synonymous changes may affect the expression, structural and functional changes in gene product towards the alteration in various developmental phenomena, particularly the tolerance to various diseases.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome-Wide Identification of Salt-Responsive Members of the <italic> WRKY </italic> Gene Family in <italic> Gossypium aridum </italic></h2>
			<h3>PMID:25951083</h3>
				
					<h3>PMCID:4423833</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Fan </surname>  <given-names> Xinqi </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Qi </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Peng </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gong </surname>  <given-names> YuanYong </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shu </surname>  <given-names> Hongmei </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Yang </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ni </surname>  <given-names> Wanchao </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xianggui </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Xinlian </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 30 </day>  <month> 3 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Since the first report of <italic> SPF1 </italic> WRKY transcription factors from sweet potato in 1994 [ <xref rid="pone.0126148.ref007" ref-type="bibr"> 7 </xref> ], numerous WRKY transcription factors have been experimentally identified from many other plant species including <italic> Arabidopsis </italic> , rice, tobacco, potato, cotton, barley, wheat, chamomile, soybean, cacao, grape, tomato, cucumber and so on [ <xref rid="pone.0126148.ref008" ref-type="bibr"> 8 </xref> &#x02013; <xref rid="pone.0126148.ref020" ref-type="bibr"> 20 </xref> ]. Except for plant development, WRKY family genes also play important roles in plant biotic and abiotic stress responses including pathogen-induced defense programs and responses to <span class="trait">drought,</span> salt, stress et al.[ <xref rid="pone.0126148.ref020" ref-type="bibr"> 20 </xref> &#x02013; <xref rid="pone.0126148.ref022" ref-type="bibr"> 22 </xref> ]. One of the most important functions of WRKYs appears to be regulation of the salt stress response. For example, <italic> AtWRKY25 </italic> and <italic> AtWRKY33 </italic> play a role in the salt stress response and improve salt tolerance in <italic> Arabidopsis </italic> [ <xref rid="pone.0126148.ref023" ref-type="bibr"> 23 </xref> ]. <italic> OsWRKY11 </italic> and <italic> OsWRKY45 </italic> play a regulatory role in salt tolerance in transgenic rice [ <xref rid="pone.0126148.ref024" ref-type="bibr"> 24 </xref> , <xref rid="pone.0126148.ref025" ref-type="bibr"> 25 </xref> ]. <italic> <span class="gene">GmWRKY54</span> </italic> -overexpressing plants are more salt and <span class="trait">drought</span> tolerant than the wild type, and <italic> <span class="gene">GmWRKY13</span> </italic> overexpression results in increased sensitivity to salt and mannitol stress [ <xref rid="pone.0126148.ref026" ref-type="bibr"> 26 </xref> ]. Finally, overexpression of <italic> TaWRKY10 </italic> enhances <span class="trait">drought</span> and salt tolerance in transgenic tobacco plants [ <xref rid="pone.0126148.ref027" ref-type="bibr"> 27 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						A number of studies have shown that WRKY genes are induced by salt stress and that overexpression of some WRKY genes alters stress tolerance in plants. In <italic> Arabidopsis </italic> , 18 out of 35 WRKY genes from roots are induced by salt stress, such as <italic> AtWRKY8 </italic> , 25, 33, 46, 57 and so on [ <xref rid="pone.0126148.ref043" ref-type="bibr"> 43 </xref> ]. In rice, 27 WRKY transcription factor genes are induced in response to salt stress, 26 of which are upregulated [ <xref rid="pone.0126148.ref044" ref-type="bibr"> 44 </xref> ]. In soybean, 22 of 64 WRKY genes are differentially expressed under salt stress [ <xref rid="pone.0126148.ref026" ref-type="bibr"> 26 </xref> ]. In transgenic rice, overexpression of <italic> OsWRKY11 </italic> /01g43650 and <italic> OsWRKY45 </italic> /05g2577 results in enhanced salt, heat and <span class="trait">drought</span> tolerance [ <xref rid="pone.0126148.ref025" ref-type="bibr"> 25 </xref> ]. Overexpression of <italic> OsWRKY45 </italic> results in enhanced salt and <span class="trait">drought</span> tolerance, in addition to increased disease <span class="trait">resistance</span> [ <xref rid="pone.0126148.ref024" ref-type="bibr"> 24 </xref> ]. In <italic> Arabidopsis </italic> , overexpression of either <italic> AtWRKY25 </italic> or <italic> AtWRKY33 </italic> increases salt tolerance [ <xref rid="pone.0126148.ref023" ref-type="bibr"> 23 </xref> ], while overexpression of <italic> AtWRKY18 </italic> or <italic> AtWRKY60 </italic> increases plant sensitivity to salt [ <xref rid="pone.0126148.ref045" ref-type="bibr"> 45 </xref> ]. <italic> <span class="gene">GmWRKY54-</span> </italic> overexpressing plants are more salt and <span class="trait">drought</span> tolerant than the control, and <italic> <span class="gene">GmWRKY13</span> </italic> overexpression results in increased sensitivity to salt and mannitol stress [ <xref rid="pone.0126148.ref026" ref-type="bibr"> 26 </xref> ]. In the current study, integrated transcriptome analysis indicated that 28 WRKY genes in <italic> G </italic> . <italic> aridum </italic> were differentially regulated in response to salt stress conditions. To further explore the functions of <italic> GarWRKY </italic> s that may be involved in plant defense responses to salt stress, we identified orthologous pairs among <italic> GarWRKY </italic> s, <italic> AtWRKYs </italic> , <italic> OsWRKYs </italic> and <italic> <span class="gene">GmWRKYs</span> </italic> based on full-length protein sequence similarities. Of the 28 salt-induced <italic> GarWRKY </italic> genes, 20 orthologs in <italic> Arabidopsis </italic> , rice and soybean also exhibit significant induction under salt stress. Among these, <italic> GarWRKY22 </italic> , <italic> GarWRKY65 </italic> and <italic> GarWRKY5 </italic> are orthologous to <italic> AtWRKY33 </italic> , <italic> <span class="gene">GmWRKY54</span> </italic> and <italic> OsWRKY45 </italic> , respectively. Motif compositions analysis showed that similar motif compositions were generally shared by WRKY members from <italic> Arabidopsis </italic> , rice and soybean within the same subgroup. These findings suggest that these members are important regulators of the response to salt stress. We also observed there was a possible relation between motif composition and gene function. <italic> GarWRKY22 </italic> contained motif 10, which was downregulated under salt stress. However, its orthologous gene in <italic> Arabidopsis </italic> ( <italic> AtWRKY33 </italic> ) didn&#x02019;t possessed motif 10, which was upregulated under salt stress [ <xref rid="pone.0126148.ref023" ref-type="bibr"> 23 </xref> ]. Further study is required to clarify whether motif 10 is a key composition in regulating alteration of gene expression and functional divergence in different species.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine Mapping and Characterization of Candidate Genes that Control <span class="trait">Resistance</span> to <italic> Cercospora sojina </italic> K. Hara in Two Soybean Germplasm Accessions</h2>
			<h3>PMID:25993056</h3>
				
					<h3>PMCID:4437980</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Pham </surname>  <given-names> Anh-Tung </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff001">  <sup> &#x000a4;a </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Harris </surname>  <given-names> Donna K. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Buck </surname>  <given-names> James </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hoskins </surname>  <given-names> Aaron </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="currentaff002">  <sup> &#x000a4;b </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Serrano </surname>  <given-names> Jonathan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abdel-Haleem </surname>  <given-names> Hussein </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Boerma </surname>  <given-names> H. Roger </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Zenglu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 7 </day>  <month> 4 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						PI 594891 plants and 24 F <sub> 2: 3 </sub> plants with the genotype homozygous for PI 594891 alleles in the interval between GSM150 and GSM160 were resistant to FLS, and Blackhawk and 15 plants with the Blackhawk haplotype were observed as susceptible ( <xref rid="pone.0126753.g001" ref-type="fig"> Fig 1 </xref> ). In total, six recombination break points were found among these 66 F <sub> 2: 3 </sub> plants. The region containing the <span class="trait">resistance</span> gene(s) to FLS in PI 594891 was fine-mapped in a set of 15 plants that have the same recombination breakpoint (No. 2 in <xref rid="pone.0126753.g001" ref-type="fig"> Fig 1 </xref> ) but opposite phenotypes (highlighted in bold and underlined in <xref rid="pone.0126753.g001" ref-type="fig"> Fig 1 </xref> ). These 15 F <sub> 2: 3 </sub> plants contained a recombination breakpoint between GSM154 and GSM202. However, 13 F <sub> 2: 3 </sub> plants with this genotype showed a <span class="trait">resistance</span> reaction to <italic> C </italic> . <italic> sojina </italic> isolates 21 and 23, suggesting that the <span class="trait">resistance</span> gene(s) did not reside on the left side of GSM154. In contrast, two F <sub> 2: 3 </sub> plants with this genotype and susceptible reactions to both isolates, indicating that the <span class="trait">resistance</span> gene(s) should not be located in the region bordered to the left by GSM202. Altogether, the genotyping and phenotyping data of these 15 plants indicated that the <italic> Rcs </italic> (PI 594891) gene is located in a region of approximately 72.6 kb between the SNP markers GSM154 and GSM202. This region is predicted to harbor five genes, three of which have a predicted function obtained from the Phytozome.net database based on protein sequence and high similarity with known genes from other species. The first known gene is <span class="gene">Glyma13g25340,</span> and was annotated to be an STRUBBELIG-receptor like protein with leucine rich repeats. The second known gene is <span class="gene">Glyma13g25350,</span> which was predicted to be transducin or Pleiotropic regulatory gene. The third gene, <span class="gene">Glyma13g25380</span> was annotated to be a FALZ-Related Bromodomain-containing protein. Two remaining genes, <span class="gene">Glyma13g25320</span> and <span class="gene">Glyma13g25331,</span> encode proteins with unknown functions. The three genes with annotated functions are referred to as <span class="gene">Glyma13g25340</span> (LRR), <span class="gene">Glyma13g25350</span> (TPR), and <span class="gene">Glyma13g25380</span> (FBP).
					</p>
				
					<p>&nbsp;&nbsp;
						The haplotype window defined was based on the fine-mapping of the <span class="trait">resistance</span> genes to a 72.6 kb region with 11 SNP markers, with physical positions of the first and last SNPs being <span class="gene">Gm13_28559988</span> and <span class="gene">Gm13_28632634,</span> respectively. In this interval both PI 594774 and PI 594891 have unique haplotypes which are different from those of Davis, Blackhawk, and the 45 soybean cultivars and lines ( <xref rid="pone.0126753.s004" ref-type="supplementary-material"> S4 Table </xref> ). We have also evaluated the haplotype allele variation at the <italic> Rcs3 </italic> locus from Davis based on the mapped interval of the <italic> Rcs3 </italic> gene on chr16 [ <xref rid="pone.0126753.ref031" ref-type="bibr"> 31 </xref> ]. The physical positions of the first and last SNP in the Davis haplotype window were <span class="gene">Gm16_32681330</span> and <span class="gene">Gm16_33360539,</span> and there were a total of 30 SNPs in the defined window ( <xref rid="pone.0126753.s005" ref-type="supplementary-material"> S5 Table </xref> ). Similarly, Davis had a unique haplotype from PI 594774, PI 594891, and Blackhawk. However, there were four cultivars &#x02018;Young&#x02019;, &#x02018;Cook&#x02019;, &#x02018;Doles&#x02019;, and &#x02018;N6201&#x02019; with a 100% match to the Davis haplotype ( <xref rid="pone.0126753.s005" ref-type="supplementary-material"> S5 Table </xref> ), suggesting that these cultivars all have Davis in their pedigrees.
					</p>
				
					<p>&nbsp;&nbsp;
						The region on chr13 surrounding Satt114 marker is a <span class="trait">resistance</span> gene-rich region. The <span class="trait">resistance</span> gene <italic> Rsp8 </italic> conditioning the <span class="trait">resistance</span> to <italic> P </italic> . <italic> sojae </italic> isolate OH25 was positioned between the markers Satt425 and Satt114 on chr13 (Satt114 was found to be significantly associated with FLS <span class="trait">resistance</span> in PI 594891 and PI 594774) [ <xref rid="pone.0126753.ref045" ref-type="bibr"> 45 </xref> ]. Aside from <italic> Rps8 </italic> , other R-genes that have been mapped to chr13 near Satt114 include <italic> Rpa1 </italic> , <italic> Rsv1 </italic> , <italic> Rag2 </italic> , <italic> rag4 </italic> , and <italic> Rag5 </italic> . <italic> Rpa1 </italic> confers race-specific <span class="trait">resistance</span> to <italic> Pythium </italic> damping-off caused by <italic> Pythium aphanidermatum </italic> in &#x02018;Archer&#x02019; soybean, <italic> Rsv1 </italic> confers strain-specific <span class="trait">resistance</span> to soybean mosaic virus, and <italic> Rag2 </italic> , <italic> rag4 </italic> , and <italic> Rag5 </italic> confer biotype-specific <span class="trait">resistance</span> to soybean aphid ( <italic> Aphis glycines </italic> ) [ <xref rid="pone.0126753.ref046" ref-type="bibr"> 46 </xref> &#x02013; <xref rid="pone.0126753.ref049" ref-type="bibr"> 49 </xref> ]. A fine-mapping of the <italic> Rag2 </italic> gene from PI 200538 on chr13 showed that the <italic> Rag2 </italic> gene(s) lies within a 54 kb interval containing seven genes one of which is a nucleotide-binding site-leucine-rich repeat gene. However, similar to other fine-mapping studies of disease <span class="trait">resistance</span> genes in soybean, this study did not further characterize all of the candidate genes within the fine-mapped region [ <xref rid="pone.0126753.ref037" ref-type="bibr"> 37 </xref> ]. The fine-mapping study presented here is the first that fine-mapped as well as characterized the candidate genes within the fine-mapped interval containing the <span class="trait">resistance</span> genes. In five candidate genes annotated based on the Williams 82 DNA sequence from the 72.6 kb genomic region fine-mapped for the population FLS-594891, there were no prominent SNPs or mutations that may be linked to the FLS <span class="trait">resistance</span> in PI 594891. In addition, the expression level of the three aforementioned genes in PI 594891 was higher but not statistically different compared to that in Blackhawk. In this case, further study needs to be conducted to validate and pinpoint a true underlying gene that controls the <span class="trait">resistance</span> to FLS in PI 594891. As there were SNPs identified in <span class="gene">Glyma13g25320,</span> <span class="gene">Glyma13g25340</span> (LRR), and <span class="gene">Glyma13g25350</span> (transducin); and these three genes had higher expression levels in the two PIs compared to Blackhawk, a <span class="trait">resistance</span> mechanism via a combined action of multiple genes, as is the case at the <italic> <span class="gene">Rhg1</span> </italic> complex locus, is possible [ <xref rid="pone.0126753.ref050" ref-type="bibr"> 50 </xref> ]. On the other hand, if only one gene is responsible for the <span class="trait">resistance,</span> <span class="gene">Glyma13g25350</span> (transducin or Pleiotropic regulatory gene based on the prediction in the Phytozome.net) should be the first gene of interest. In PI 594774, this gene has a mutation that resulted in an amino acid change predicted to be deleterious for the protein function. PI 594774 and PI 594891 also had two common SNPs in the promoter region of this gene. Additionally, members of this class of the gene were reported to be highly expressed as a defense mechanism against attacking pathogens, such as in potato infected with <italic> P </italic> . <italic> infestans </italic> and in <italic> Arabidopsis </italic> challenged with <italic> C </italic> . <italic> higginsianum </italic> [ <xref rid="pone.0126753.ref051" ref-type="bibr"> 51 </xref> &#x02013; <xref rid="pone.0126753.ref055" ref-type="bibr"> 55 </xref> ]. A pleiotropic regulatory gene (PLRG) was reported to be one of the three most important components of a protein complex that is essential for plant innate immunity [ <xref rid="pone.0126753.ref055" ref-type="bibr"> 55 </xref> ]. In Arabidopsis, PLRG is a WD40 repeat protein shown to bind directly to an atypical R2R3 Myb transcription factor and together with a nuclear protein they make up the spliceosome-associated PRP Nineteen Complex (NTC) [ <xref rid="pone.0126753.ref055" ref-type="bibr"> 55 </xref> ]. A mutant <italic> plrg Arabidopsis </italic> line had an increase in growth of <italic> Pseudomonas syringae </italic> pv <italic> maculicola </italic> up to more than 50 times compared to wild-type Col-0 control, indicating the role of this protein in the defense mechanism against plant pathogens [ <xref rid="pone.0126753.ref055" ref-type="bibr"> 55 </xref> ]. The lack of deleterious SNPs in candidate genes in PI 594891 may hint at other possibilities such as the presence of a causal element residing outside of the sequenced regions of these five annotated genes or the effects of epigenetics. Nevertheless, a fine-mapped region of 72.6 kb would make it more feasible for future studies to enhance our knowledge about the mechanism of the <span class="trait">resistance</span> to <italic> C </italic> . <italic> sojina </italic> in soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:RNA-seq analysis reveals genetic response and tolerance mechanisms to ozone exposure in soybean</h2>
			<h3>PMID:26040850</h3>
				
					<h3>PMCID:4456062</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Whaley </surname>  <given-names> Adam </given-names>  </name>  <address>  <email> awhaley9@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sheridan </surname>  <given-names> Jaime </given-names>  </name>  <address>  <email> jbeine@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Safari </surname>  <given-names> Sajedeh </given-names>  </name>  <address>  <email> ssafari@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Burton </surname>  <given-names> Amy </given-names>  </name>  <address>  <email> amyburton425@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Burkey </surname>  <given-names> Kent </given-names>  </name>  <address>  <email> kent.burkey@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Schlueter </surname>  <given-names> Jessica </given-names>  </name>  <address>  <email> jschluet@uncc.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, 9201 University City Blvd, Charlotte, NC 28223 USA  </aff>  <aff id="Aff2">  <label/> United States Department of Agriculture, Agricultural Research Service, 3127 Ligon St, Raleigh, NC 27695 USA  </aff>
				<br/> 
				年份：<day> 18 </day>  <month> 5 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Using <italic> cuffdiff </italic> [ <xref ref-type="bibr" rid="CR31"> 31 </xref> ], differential analysis of the sequencing results of the Mandarin (Ottawa) at low and high ozone concentrations results yielded 135 to 275 differentially expressed genes per time point, or less than 1&#x000a0;% of the total number of annotated genes in the reference genome (Table&#x000a0; <xref rid="Tab1" ref-type="table"> 1 </xref> ). In total, 535 unique genes showed differential expression at one or more time points. Genes with annotated functions or processes matching phenylpropanoids, and lignin biosynthesis were found in all four time points, while genes corresponding to glutathione, lipoxygenase, and phenylalanine ammonia-lyases were found in one or more time points but not in all four time points. Of the entire set of genes differentially expressed in all four time points 267 genes had a consistent pattern of up or down regulation across all four time points. Of these genes, 230 showed higher expression in the high ozone condition, and 15 correspond with GO functions related to oxidative or general stress response in soybean: <span class="gene">Glyma.09G156700,</span> <span class="gene">Glyma.11G078400</span> and <span class="gene">Glyma.17G036200</span> are putative oxidative stress response genes that code for perioxidase genes. <span class="gene">Glyma.05G231900</span> is a gene involved in phenylpropanoid biosynthesis, but also match GO terms for systemic acquired <span class="trait">resistance,</span> a pathway in plants that &#x0201c;warns&#x0201d; surrounding plant tissues after a localized pathogen exposure, making its expression here particularly interesting. <span class="gene">Glyma.10G162400</span> is a transcription-regulating gene that matches GO terms implicating it in auxin, cadmium, and osmotic stress response. The remaining 10 genes, <span class="gene">Glyma.02G142500,</span> <span class="gene">Glyma.04G003200,</span> <span class="gene">Glyma.04G088500,</span> <span class="gene">Glyma.04G248500,</span> <span class="gene">Glyma.05G109600,</span> <span class="gene">Glyma.07G018000,</span> <span class="gene">Glyma.10G001800,</span> <span class="gene">Glyma.10G179400,</span> <span class="gene">Glyma.14G141000,</span> <span class="gene">Glyma.14G216200,</span> all match GO terms related to salt stress, and other terms such as jasmonic acid response, response to cold, cell wall biogenesis, and oxidation-reduction (Fig. <xref rid="Fig2" ref-type="fig"> 2 </xref> ; see Additional file <xref rid="MOESM1" ref-type="media"> 1 </xref> : Table S1). Because of the prevalence of these pathways in the results, it is likely that many of the genes in salt and cold stress response also play a yet uncharacterized role in oxidative stress response. <table-wrap id="Tab1"> <label> Table 1 </label> <caption> <p> Sample comparisons and differentially expressed genes
					</p>
				
					<p>&nbsp;&nbsp;
						Comparing the high ozone exposure between both Mandarin (Ottawa) and Fiskeby III at individual time points resulted in 258 to 536 differentially expressed genes, a similar number to the comparisons of each cultivar at low and high exposure described above. Lipoxygenase and lignin genes were found at all four time points, whereas phenylpropanoids and glutathione genes were found at time points 2&#x02013;4. Phenylalanine ammonia-lyases were found at the first, third, and fourth time points. The 258 to 536 differentially expressed genes per time point correspond to 564 unique genes with differential expression at a minimum of one time point. An AgriGO analysis of these 564 genes identified a significant abundance of GO terms related to oxidation reduction (p&#x02009;=&#x02009;5.1e&#x02013;6). When the time series was examined as a whole, only 64 genes were shown to have a consistent pattern of expression across all four time points. Of these 64 genes, 58 had higher expression in the ozone sensitive Mandarin (Ottawa), and only 6 had higher expression in Fiskeby. Of the 6 genes that had higher expression in Fiskeby, we were only able to identify GO terms for four. <span class="gene">Glyma.14G164900</span> and <span class="gene">Glyma.14G165000</span> matched identical GO terms for oxidation-reduction and response to light stimulus, and are likely tandemly duplicated genes. <span class="gene">Glyma.11G197300</span> is a cytochrome <span class="gene">P450</span> gene that matches a large number of biological process GO terms including defense response, response to water deprivation, and induced systemic <span class="trait">resistance.</span> <span class="gene">Glyma.08G287500</span> matches GO terms for cell wall organization, anthocyanin accumulation, and regulation of hormone levels. 58 genes had higher expression in Mandarin (Ottawa), and 24 of these matched GO terms related to oxidative or general stress response. A summary of these genes and a brief description of their matching biological process GO terms can be found in Table&#x000a0; <xref rid="Tab2" ref-type="table"> 2 </xref> . Of particular note in these results are the two transcription regulators, <span class="gene">Glyma.13G203700</span> and <span class="gene">Glyma.10G035500,</span> as well as <span class="gene">Glyma.15G182600</span> which matched GO terms for regulation of hydrogen peroxide and systemic acquired <span class="trait">resistance.</span> The previously described SAR gene, <span class="gene">Glyma.05G231900</span> was expressed in both cultivars and not deemed significantly different. <table-wrap id="Tab2"> <label> Table 2 </label> <caption> <p> Genes showing higher differential expression in Mandarin (Ottawa) at high ozone when compared to Fiskeby at high ozone, and relevant GO term matches
					</p>
				
					<p>&nbsp;&nbsp;
						Another pattern of expression examined were the genes that were expressed in both cultivars at the first time point at high ozone exposure but were turned off at a later time point in one cultivar while the other maintained expression under high ozone conditions. Both possible configurations of this pattern had a small number of genes associated with it: 65 genes in which expression was maintained in Fiskeby III, 39 genes in which expression was maintained in Mandarin (Ottawa) (Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> ). In the first scenario, where expression is maintained in Fiskeby III but reduced or eliminated in Mandarin (Ottawa), these genes were overwhelmingly related to defense or stress response. Of these genes, 10 matched GO terms related to systemic acquired <span class="trait">resistance,</span> 4 additional genes matched GO terms for oxidative stress response, 7 more were related to oxidation-reduction, and 6 others related to jasmonic acid or ethylene response. In addition to there being fewer genes with expression maintained in Mandarin (Ottawa), the genes that did match this pattern did not show the same prevalence as the previous pattern towards stress or defense responses. Only one gene in this set, <span class="gene">Glyma.01G153300</span> was related to oxidation-reduction, and no genes matched GO terms for oxidative stress response. One additional gene matched GO terms related to jasmonic acid response and several abiotic stress responses, another matched terms related to hydrogen peroxide response.
					</p>
				
					<p>&nbsp;&nbsp;
						When the read mapping was updated for the new genome version, using Cufflinks 2.2.1, the script for determining differentially expressed genes matching certain patterns no longer returned the same abundance of wax biosynthesis pathway genes that our previous analysis had. At this time, we hypothesize that this is likely due to the overall lower number of differential gene calls, which in turn causes there to be too few genes to identify the pathway through AgriGO or other GO term abundance tools. In the initial pattern analysis that revealed the cuticle wax pathway we observed 143 genes where we now only observe 43. With the exception of the Fiskeby low exposure to Fiskeby high ozone exposure time series comparison, all other comparisons saw a decrease in the number of differentially expressed genes of a similar magnitude. Additionally, the exclusion of cuticle wax genes from the pattern results could be also attributed to the new genome or annotations as well; only 66 genes in the new annotations match GO terms for wax biosynthesis. If we examine the individual gene annotations from the pattern script, there are several genes we can attribute to the wax biosynthesis pathway. These include two ketoacyl-CoA synthases <span class="gene">(Glyma.10G179400</span> and Glyma.U033000), which are involved in fatty acid elongation for cuticle wax formation [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. There is also a wax ester synthase gene <span class="gene">(Glyma.06G291700).</span> Using the 66 genes matching GO terms for wax biosynthesis, we were able to query the data for those genes specifically and were able to identify three genes that showed differential expression. Of these genes, <span class="gene">Glyma.11G185100,</span> <span class="gene">Glyma.12G183400</span> showed significant differential expression at time point 2, and had a fold change corresponding to higher expression in Fiskeby at that time point. However, by time point four the fold change had switched to correspond to higher expression in Mandarin (Ottawa), although it was not found to be significant (log2(fold change)&#x02009;=&#x02009;8.925). The other gene, <span class="gene">Glyma.13G317600,</span> showed differential expression at time points one, two, and four. At the first two time points, the fold change showed higher expression in Fiskeby, though by time point four <span class="gene">Glyma.13G317600</span> had higher expression in Mandarin (Ottawa). All three of these genes substantiate the previously observed pattern of expression, where initial expression favors Fiskeby or neither cultivar, and over the time points shifts to higher expression in Mandarin (Ottawa). Additionally, we also used a list of Arabidopsis cuticle wax biosynthesis genes from Le Provost et al., a study that linked cuticle wax and <span class="trait">drought</span> tolerance [ <xref ref-type="bibr" rid="CR40"> 40 </xref> ]. Soybean orthologs to the <italic> A. thaliana </italic> genes from the Le Provost study were identified using BLAST, and the cufflinks results for those genes, in addition to the 66 genes matches the GO term for wax biosynthesis, can be seen in Supplemental Table&#x000a0; <xref rid="Tab3" ref-type="table"> 3 </xref> (See Additional File <xref rid="MOESM1" ref-type="media"> 1 </xref> ). While only a limited number of these genes have statistically significant differential expression, there is a strong overall trend that matches that seen in the pattern 3 genes. Specifically, we see more genes with fold changes values corresponding to higher expression in Fiskeby in the first time point, which decreases to more genes showing higher expression in Mandarin (Ottawa) by the final time point.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of Row-Type, Row-Spacing, Seeding Rate, Soil-Type, and Cultivar Differences on Soybean Seed Nutrition under US Mississippi Delta Conditions</h2>
			<h3>PMID:26061182</h3>
				
					<h3>PMCID:4465643</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Bellaloui </surname>  <given-names> Nacer </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Bruns </surname>  <given-names> Herbert A. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Abbas </surname>  <given-names> Hamed K. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Mengistu </surname>  <given-names> Alemu </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Fisher </surname>  <given-names> Daniel K. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reddy </surname>  <given-names> Krishna N. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 14 </day>  <month> 5 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A field experiment was conducted at Stoneville, MS, in Sharkey clay soil in 2009 and 2010, and in Beulah fine sandy loam soil in 2008, 2009, and 2010. This research was a part of a larger research project that involved different research components. We focus here only on the seed nutrition quality component; however, the agronomic component, including <span class="trait">yield,</span> was reported [ <xref rid="pone.0129913.ref009" ref-type="bibr"> 9 </xref> ] where a detailed description of field management and growth conditions was provided. Briefly, the sites were prepared each year by disking and bedding in the fall of the previous year. Raised beds spaced 102 cm apart were formed with a bedding hipper. Prior to planting, raised beds were smoothed with a harrow as needed to plant soybean in a 102 cm single-row and 25-cm twin rows on 102-cm centers and to facilitate furrow irrigation. Cultivar Armor GP 533 (Armor Seed Company, Waldenburg, AR), a mid-MG V [ <xref rid="pone.0129913.ref020" ref-type="bibr"> 20 </xref> ] and Pioneer 94M80 (Pioneer Hi-Bred Int., Huntsville, AL), a late MG IV, were used. Each plot was four-rows wide, planted 11 m long, and end-trimmed to 9 m at the V4 growth stage. Single-row plantings were conducted using an Almaco cone plot planter and twin-row plots were planted using a four unit Monosem NG-3 twin-row vacuum planter set on 102 cm centers between planting units and 25 cm between rows within a unit. In the sandy loam soil, planting dates were 16 April in 2008, 18 April in 2009, and 14 April in 2010. On the Sharkey clay site, planting dates were 22 April in 2009 and 12 April in 2010. Weed control in both sites was achieved by applying a pre-plant application of trifluralin [2,6-Dinitro- <italic> N </italic> , <italic> N </italic> -dipropyl-4-(trifluoromethyl)aniline] at 0.7 kg ai ha <sup> &#x02013;1 </sup> followed by two postemergence applications of metolachlor [2-chloro-N-(2-ethyl-6-methylphenyl)-N-(2-methoxy- 1-methylethyl) acetamide] and glyphosate [2-[(phosphonomethyl)amino]acetic acid] at growth stage <span class="gene">V2</span> to <span class="gene">V3</span> (two to three trifoliate) and at V5 to V6 (five to six trifoliate). For fungal control, pyraclostrobin (carbamic acid, [[[1-(4-cholrophenyl)-H-pyrazol-3-yl]oxy]methyl]phenyl]methoxy-,methyl ester) was applied according to factory label directions at V5 to V6. Plots were furrow-irrigated from R1 (beginning flowering) through R6 (full seed-fill) with approximately 25 mm water applied at 10 day intervals. Seeds at maturity harvest (R8 stage) were collected and processed, and nutrients were determined as described below. All samples (seeds, leaves, and soil) were processed after sampling and harvesting in a similar way in each year and measured with the same instruments under similar conditions to minimize the variability due to instruments and measurement conditions.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Lipase-catalyzed biodiesel production and quality with <italic> Jatropha curcas oil </italic> : exploring its potential for Central America</h2>
			<h3>PMID:26213567</h3>
				
					<h3>PMCID:4513677</h3>
					<h4>期刊:Journal of Biological Engineering</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Bueso </surname>  <given-names> Francisco </given-names>  </name>  <address>  <email> fjbueso@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Moreno </surname>  <given-names> Luis </given-names>  </name>  <address>  <email> lem_260@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cede&#x000f1;o </surname>  <given-names> Mathew </given-names>  </name>  <address>  <email> mathewcedeno@hotmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Manzanarez </surname>  <given-names> Karla </given-names>  </name>  <address>  <email> kgmanzanarez@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1"> Department of Food Science and Technology, EAP Zamorano University, P.O. Box 93, Tegucigalpa, Honduras  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Compliance of alkali-catalyzed Jatropha biodiesel with ASTM D6751 has been well documented [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR14"> 14 </xref> ]. However, lipase-catalyzed Jatropha biodiesel compliance has been reported scarcely, and lack stability parameters [ <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. To fill this <span class="gene">gap</span> in the literature, the potential of lipase-catalyzed biodiesel as an alternative to conventional alkaline transesterification was evaluated with Jatropha oil in comparison to palm oil. For this purpose, the catalytic performance of a free, liquid TL lipase in terms of biodiesel <span class="trait">yield</span> and quality was evaluated vs. immobilized CA.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Application of genomics-assisted breeding for generation of climate resilient crops: progress and prospects</h2>
			<h3>PMID:26322050</h3>
				
					<h3>PMCID:4531421</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kole </surname>  <given-names> Chittaranjan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/240614/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Muthamilarasan </surname>  <given-names> Mehanathan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/164737/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Henry </surname>  <given-names> Robert </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103236/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Edwards </surname>  <given-names> David </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sharma </surname>  <given-names> Rishu </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/255210/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abberton </surname>  <given-names> Michael </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Batley </surname>  <given-names> Jacqueline </given-names>  </name>  <xref ref-type="aff" rid="aff7">  <sup> 7 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/255579/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bentley </surname>  <given-names> Alison </given-names>  </name>  <xref ref-type="aff" rid="aff8">  <sup> 8 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/176074/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Blakeney </surname>  <given-names> Michael </given-names>  </name>  <xref ref-type="aff" rid="aff9">  <sup> 9 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bryant </surname>  <given-names> John </given-names>  </name>  <xref ref-type="aff" rid="aff10">  <sup> 10 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/257311/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cai </surname>  <given-names> Hongwei </given-names>  </name>  <xref ref-type="aff" rid="aff11">  <sup> 11 </sup>  </xref>  <xref ref-type="aff" rid="aff12">  <sup> 12 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/255752/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cakir </surname>  <given-names> Mehmet </given-names>  </name>  <xref ref-type="aff" rid="aff13">  <sup> 13 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cseke </surname>  <given-names> Leland J. </given-names>  </name>  <xref ref-type="aff" rid="aff14">  <sup> 14 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cockram </surname>  <given-names> James </given-names>  </name>  <xref ref-type="aff" rid="aff8">  <sup> 8 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Oliveira </surname>  <given-names> Antonio Costa </given-names>  </name>  <xref ref-type="aff" rid="aff15">  <sup> 15 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> De Pace </surname>  <given-names> Ciro </given-names>  </name>  <xref ref-type="aff" rid="aff16">  <sup> 16 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/240938/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dempewolf </surname>  <given-names> Hannes </given-names>  </name>  <xref ref-type="aff" rid="aff17">  <sup> 17 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/255557/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ellison </surname>  <given-names> Shelby </given-names>  </name>  <xref ref-type="aff" rid="aff18">  <sup> 18 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249967/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gepts </surname>  <given-names> Paul </given-names>  </name>  <xref ref-type="aff" rid="aff19">  <sup> 19 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/123721/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Greenland </surname>  <given-names> Andy </given-names>  </name>  <xref ref-type="aff" rid="aff8">  <sup> 8 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hall </surname>  <given-names> Anthony </given-names>  </name>  <xref ref-type="aff" rid="aff20">  <sup> 20 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/45580/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hori </surname>  <given-names> Kiyosumi </given-names>  </name>  <xref ref-type="aff" rid="aff21">  <sup> 21 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/156735/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hughes </surname>  <given-names> Stephen </given-names>  </name>  <xref ref-type="aff" rid="aff22">  <sup> 22 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Humphreys </surname>  <given-names> Mike W. </given-names>  </name>  <xref ref-type="aff" rid="aff23">  <sup> 23 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/254600/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Iorizzo </surname>  <given-names> Massimo </given-names>  </name>  <xref ref-type="aff" rid="aff24">  <sup> 24 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/251111/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ismail </surname>  <given-names> Abdelbagi M. </given-names>  </name>  <xref ref-type="aff" rid="aff25">  <sup> 25 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/67160/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marshall </surname>  <given-names> Athole </given-names>  </name>  <xref ref-type="aff" rid="aff23">  <sup> 23 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/254632/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mayes </surname>  <given-names> Sean </given-names>  </name>  <xref ref-type="aff" rid="aff26">  <sup> 26 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249970/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff27">  <sup> 27 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ogbonnaya </surname>  <given-names> Francis C. </given-names>  </name>  <xref ref-type="aff" rid="aff28">  <sup> 28 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/191962/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ortiz </surname>  <given-names> Rodomiro </given-names>  </name>  <xref ref-type="aff" rid="aff29">  <sup> 29 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/202395/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Paterson </surname>  <given-names> Andrew H. </given-names>  </name>  <xref ref-type="aff" rid="aff30">  <sup> 30 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Simon </surname>  <given-names> Philipp W. </given-names>  </name>  <xref ref-type="aff" rid="aff31">  <sup> 31 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/257305/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tohme </surname>  <given-names> Joe </given-names>  </name>  <xref ref-type="aff" rid="aff32">  <sup> 32 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tuberosa </surname>  <given-names> Roberto </given-names>  </name>  <xref ref-type="aff" rid="aff33">  <sup> 33 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/52290/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="aff27">  <sup> 27 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249960/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Varshney </surname>  <given-names> Rajeev K. </given-names>  </name>  <xref ref-type="aff" rid="aff34">  <sup> 34 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/25772/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wullschleger </surname>  <given-names> Stan D. </given-names>  </name>  <xref ref-type="aff" rid="aff35">  <sup> 35 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/46603/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yano </surname>  <given-names> Masahiro </given-names>  </name>  <xref ref-type="aff" rid="aff36">  <sup> 36 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/125229/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prasad </surname>  <given-names> Manoj </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/145332/overview"/>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 7 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Genomics-based approaches and NGS have ushered in sequence-based breeding strategies that will expedite the dissection and cloning of the loci controlling abiotic stress tolerance, while providing unparalleled opportunities to tap into wild relatives of crops, hence expanding the reservoir of genetic diversity available to breeders (Tuberosa et al., <xref rid="B170" ref-type="bibr"> 2011 </xref> ; Edwards et al., <xref rid="B42" ref-type="bibr"> 2013 </xref> ) (Figure <xref ref-type="fig" rid="F1"> 1 </xref> ). In view of the genetic complexity of <span class="trait">yield,</span> particularly under <span class="trait">drought</span> and other abiotic stresses, <span class="gene">GS</span> will provide the most powerful approach to raise the <span class="trait">yield</span> potential to the levels required to meet the fast-increasing global demand in cereal grain. However, MAS will remain a valid option for major loci or QTL, while QTL cloning will become a more routine activity facilitated through increased utilization of high-throughput, accurate phenotyping (Araus and Cairns, <xref rid="B8" ref-type="bibr"> 2014 </xref> ), sequencing (Imelfort et al., <xref rid="B68" ref-type="bibr"> 2009 </xref> ; Edwards and Wang, <xref rid="B41" ref-type="bibr"> 2012 </xref> ), and identification of suitable candidate genes through &#x0201c;omics&#x0201d; profiling (Gupta et al., <xref rid="B58" ref-type="bibr"> 2013 </xref> ). Cloned QTL will provide novel opportunities for genetic engineering for abiotic stress tolerance and for a more targeted search for novel alleles in wild germplasm (Salvi et al., <xref rid="B144" ref-type="bibr"> 2007 </xref> ). Even with the application of advanced genomics technologies, mitigating the negative effects of climate change on crop productivity will remain a daunting undertaking. This requires a multidisciplinary and integrated approach, which will eventually allow plant breeders to more effectively select crops that are more resilient to climate change and ensure a sufficient level of food security for the decades to come.
					</p>
				
					<p>&nbsp;&nbsp;
						Waterlogging is a major problem for cereal production worldwide, as in sodic environments, soils are affected by seepage from irrigation canals, and excess wetting due to rainfall or floods, especially if it rains after irrigation. Genetic diversity in waterlogging tolerance was reported in various crops, including wheat, barley, maize, and oats (Kerr, <xref rid="B80" ref-type="bibr"> 1986 </xref> ), and diverse mechanisms have been associated with tolerance. They are associated with phenology and morphology, nutrition balances, metabolism, including anaerobic catabolism and anoxia tolerance, and post-anoxia damage and recovery (Setter and Waters, <xref rid="B153" ref-type="bibr"> 2003 </xref> ). Tolerance of flooding during germination and early seedling growth is essential for direct seeding of rice, both in rainfed and irrigated areas, where even waterlogging is sufficient to cause considerable reduction in crop stand because of their high sensitivity to hypoxia at this stage (Ismail et al., <xref rid="B69" ref-type="bibr"> 2009 </xref> ). Substantial genetic variation was recently observed in the ability to germinate and establish in flooded soil. Tolerant genotypes are capable of catabolizing starch reserves in seeds germinating under hypoxia into simple sugars, and use them as substrates to generate energy via anaerobic pathways for the growing embryos (Miro and Ismail, <xref rid="B116" ref-type="bibr"> 2013 </xref> ; Septiningsih et al., <xref rid="B151" ref-type="bibr"> 2013 </xref> ). Several QTL originating from a few rice landraces were identified, two of them with large effects; on chromosome 9 ( <italic> qAG-9-2 </italic> ) (Angaji et al., <xref rid="B6" ref-type="bibr"> 2010 </xref> ) and chromosome 7 ( <italic> qAG-7-1 </italic> ) (Septiningsih et al., <xref rid="B151" ref-type="bibr"> 2013 </xref> ). These QTL are being targeted for cloning and for use through MAB, which could eventually result in high yielding rice cultivars for direct seeding in flood-borne areas. Recently, tolerant rice varieties carrying <italic> <span class="gene">SUB1</span> </italic> locus became available. <italic> <span class="gene">SUB1</span> </italic> is a major QTL on chromosome 9 that has been cloned and the gene responsible for tolerance identified as <italic> SUB1A-1 </italic> . This gene encodes an ERF that suppresses ethylene-mediated responses under submergence, and subsequently limits excessive elongation and halts chlorophyll degradation. Both processes are essential to prevent carbohydrate starvation of the submerged plants. These varieties can survive 4&#x02013;18 days of complete submergence, with <span class="trait">yield</span> benefits of 1 to over 3.5 t ha <sup> &#x02212;1 </sup> (depending on flood duration and floodwater condition), compared to current farmers' varieties, and without any undesirable effects on the features of the original varieties (Singh et al., <xref rid="B158" ref-type="bibr"> 2009 </xref> ; Bailey-Serres et al., <xref rid="B12" ref-type="bibr"> 2010 </xref> ; Mackill et al., <xref rid="B108" ref-type="bibr"> 2012 </xref> ; Ismail et al., <xref rid="B71" ref-type="bibr"> 2013 </xref> ). Additional genes are being targeted for submergence tolerance, and once identified they could be combined with <italic> <span class="gene">SUB1</span> </italic> for higher tolerance during complete submergence. Further, the progress made in rice could potentially be exploited to improve flood tolerance of other crop species and provide more resilient varieties for current and future flood-affected areas.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genomic consequences of selection and genome-wide association mapping in soybean</h2>
			<h3>PMID:26334313</h3>
				
					<h3>PMCID:4559069</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wen </surname>  <given-names> Zixiang </given-names>  </name>  <address>  <email> wzxsoy@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Boyse </surname>  <given-names> John F. </given-names>  </name>  <address>  <email> boyse@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <address>  <email> qijian.song@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry B. </given-names>  </name>  <address>  <email> perry.cregan@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Wang </surname>  <given-names> Dechun </given-names>  </name>  <address>  <phone> (517) 355-0271 </phone>  <email> wangdech@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Plant, Soil and Microbial Sciences, Michigan State University, 1066 Bogue St., Rm. A384-E, East Lansing, MI 48824-1325 USA  </aff>  <aff id="Aff2">  <label/> Soybean Genomics and Improvement Laboratory, Agricultural Research Service, United States Department of Agriculture, Beltsville, MD 20705 USA  </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We assigned the gene closest to the 20-kb window with the maximum XP-CLR score in each LD block as the most likely candidate (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). These candidates can be classified into different gene categories, with a significant higher proportion (using Fisher&#x02019;s exact test relative to whole genome level) of genes associated with DNA (or RNA) binding and catalytic processes (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ). About 40&#x000a0;% of these candidates (or orthologous genes) have been anticipated to be involved in improvement of soybean or maize (Additional file <xref rid="MOESM3" ref-type="media"> 3 </xref> ) [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ]. Several candidates are interesting based on their homology, even though their exact functions in soybean are not known. For instance, <italic> Glyma.14&#x000a0;g201100 </italic> has protein homology to <italic> OsNAC9 </italic> , which affects grain <span class="trait">yield</span> and <span class="trait">drought</span> <span class="trait">resistance</span> in rice [ <xref ref-type="bibr" rid="CR30"> 30 </xref> ]; <italic> Glyma.02&#x000a0;g138100 </italic> , highly expressed in soybean seed, is orthologous to maize SAD that is involved in converting stearic acid to oleic acid [ <xref ref-type="bibr" rid="CR31"> 31 </xref> ]. A few genes among our improvement candidates have been functionally characterized in soybean. For example, <italic> Glyma.18&#x000a0;g022400 </italic> (an amino acid transmembrane transporter) is one of the three genes known to contribute to SCN <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR32"> 32 </xref> ]; <italic> Glyma.04&#x000a0;g101500 </italic> <span class="gene">(GmCRY1)</span> is a major regulator of photoperiodic response in soybean and correlates with latitudinal distribution of soybean [ <xref ref-type="bibr" rid="CR33"> 33 </xref> ]; <italic> Glyma.08&#x000a0;g109300 </italic> (a chalcone synthase gene) is one of the genes which controls the distribution of seed-coat color by inhibiting coloration over the entire seed coat [ <xref ref-type="bibr" rid="CR34"> 34 </xref> ]. The three genes showed strong selection signals that may be caused by positive selection for <span class="trait">resistance</span> to SCN, wider geographical regions of adaptation and yellow seed coat color, respectively, by which the favorable alleles have been captured and accumulated in this set of improved lines.
					</p>
				
					<p>&nbsp;&nbsp;
						We further conducted GWAS for six quantitative traits including grain <span class="trait">yield,</span> plant height, <span class="trait">lodging,</span> maturity date, and protein and oil content. For grain <span class="trait">yield,</span> a total of 15 loci were identified in the elite cultivars and explained 36.6&#x000a0;% of the phenotypic variation. Since <span class="trait">yield</span> is a very complex trait and no candidate gene has been identified as a functional gene underlying soybean <span class="trait">yield</span> or <span class="trait">yield</span> component traits, it is difficult to assess the exhaustiveness and accuracy for these QTLs. However, when compared with chromosomal regions previously reported to contain <span class="trait">yield</span> QTLs ( <ext-link ext-link-type="uri" xlink:href="http://www.soybase.org"> www.soybase.org </ext-link> ) or GWAS signals, a total of 10 loci found in this study fall within such intervals or adjacent to previous GWAS signals (Additional files <xref rid="MOESM8" ref-type="media"> 8 </xref> and <xref rid="MOESM9" ref-type="media"> 9 </xref> ). For example, two significant SNPs, <span class="gene">Gm01-55794390</span> and <span class="gene">Gm20-41706616,</span> detected in this work were adjacent to SNPs previously identified as being associated with number of pods per plant and seed weight, respectively [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. Moreover, <span class="gene">Gm20-41706616</span> ( <italic> P </italic> &#x02009;=&#x02009;1.06&#x02009;&#x000d7;&#x02009;10 <sup> -5 </sup> , R <sup> 2 </sup> &#x02009;=&#x02009;2.3&#x000a0;%) is adjacent to a cluster of genes that are homologous to <italic> Arabidopsis APETALA2 </italic> genes (homeotic regulator) known to influence seed weight and size [ <xref ref-type="bibr" rid="CR40"> 40 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						For <span class="trait">lodging,</span> height and maturity date, three highly correlated traits, two major regions were significantly associated with these traits, one on Chromosome 10 spanning from 44.3 to 44.9&#x000a0;Mb, and the other on Chromosome 19 spanning from 47.3 to 47.9&#x000a0;Mb (Additional file <xref rid="MOESM9" ref-type="media"> 9 </xref> ). The two regions coincide with the E2 and E3 maturity loci. Three SNPs <span class="gene">(Gm10-44722784,</span> <span class="gene">Gm10-44723907</span> and <span class="gene">Gm10-44724890)</span> on chromosome 10 were strongly associated with maturity date, height and <span class="trait">lodging</span> simultaneously. These SNPs are located within an intron of <italic> <span class="gene">GmGIa</span> </italic> , a gene that has been demonstrated to be involved in soybean maturity and flowering time [ <xref ref-type="bibr" rid="CR41"> 41 </xref> ]. Similarly, a total of 11 SNPs on chromosome 19 spanning 47.3 to 47.9&#x000a0;Mb on <span class="gene">Gm19</span> were strongly associated with maturity, as well as plant height (Additional files <xref rid="MOESM8" ref-type="media"> 8 </xref> and <xref rid="MOESM9" ref-type="media"> 9 </xref> ). These SNPs are adjacent to <italic> <span class="gene">GmPhyA3</span> </italic> gene, a phytochrome receptor corresponding to the <italic> E3 </italic> maturity locus [ <xref ref-type="bibr" rid="CR42"> 42 </xref> ]. The co-localization of significant regions for these traits in the current work is similar with that reported by Sonah et al. [ <xref ref-type="bibr" rid="CR39"> 39 </xref> ]. This could be the result of pleiotropy or closely linked genes within the same region [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Improved Production of Sublancin 168 Biosynthesized by <italic> Bacillus subtilis </italic> 168 Using Chemometric Methodology and Statistical Experimental Designs</h2>
			<h3>PMID:26339632</h3>
				
					<h3>PMCID:4538419</h3>
					<h4>期刊:BioMed Research International</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ji </surname>  <given-names> Shengyue </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Weili </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xin </surname>  <given-names> Haiyun </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Shan </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Binyun </given-names>  </name>  <xref ref-type="aff" rid="I1"/>  <xref ref-type="corresp" rid="cor1">
				<br/> 
				年份：<day> 5 </day>  <month> 1 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The availability of the regression model of the sublancin 168 production using the calculated optimal medium compositions and culture condition, namely, 22.99&#x02009;g/L soybean meal, 28.49&#x02009;g/L corn powder, and temperature at 30.8&#x000b0;C, was validated with triplicate experiments. The mean maximal value of sublancin 168 production was 129.72&#x02009;mg/L, which agreed with the predicted value (125.88&#x02009;mg/L) well. As a result, the model was considered to be accurate and reliable for predicting the sublancin 168 production by <italic> B. subtilis </italic> 168. However, there is a certain <span class="gene">gap</span> between the <span class="trait">yield</span> and the theoretical value, which may be caused by some factors other than the medium that affect the <span class="trait">yield</span> of sublancin 168 but not investigated in this work. In this study, the <span class="trait">yield</span> of sublancin 168 by <italic> B. subtilis </italic> 168 was improved from low level of no more than 60&#x02009;mg/L up to 129.72&#x02009;mg/L in optimized medium.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparisons of the Effects of Elevated Vapor Pressure Deficit on Gene Expression in Leaves among Two Fast-Wilting and a Slow-Wilting Soybean</h2>
			<h3>PMID:26427064</h3>
				
					<h3>PMCID:4591296</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Devi </surname>  <given-names> Mura Jyostna </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sinclair </surname>  <given-names> Thomas R </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Taliercio </surname>  <given-names> Earl </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref rid="cor001" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 8 </day>  <month> 9 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Eleven genes belonging to the WRKY transcription factor family were identified as up-regulated under high VPD ( <xref rid="pone.0139134.g005" ref-type="fig"> Fig 5 </xref> ). Additionally, the level of four transcripts annotated as encoding VQ-motif proteins that are likely to interact with WRKY transcription factors increased in response to high VPD. WRKY transcription factors are known to be involved in both biotic and abiotic stress responses [ <xref rid="pone.0139134.ref015" ref-type="bibr"> 15 </xref> , <xref rid="pone.0139134.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0139134.ref017" ref-type="bibr"> 17 </xref> ]. The change in expression of several WRKY transcription factor genes in PI 416937 due to high VPD was similar to WRKYs expression pattern in response to other abiotic stresses in numerous plant species. In a study of the soybean genome, 64 <italic> <span class="gene">GmWRKY</span> </italic> genes were identified. These WRKY transcription factors conferred salt, cold and <span class="trait">drought</span> stress tolerance when expressed in Arabidopsis. [ <xref rid="pone.0139134.ref018" ref-type="bibr"> 18 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Fifty-nine genes encoding various proteins such as protein kinases, ubiquitins and receptor-like cytoplasmic kinases were abundant among both up-regulated (28) and down-regulated transcripts (31). Genes such as <span class="gene">AAA</span> type ATPases and subtilase are mostly involved in either post-translational changes or protein degradation. The mostly up-regulated <span class="gene">AAA</span> types of ATPase proteins have chaperon-like activities and play a role in stomatal regulation under dehydration stress [ <xref rid="pone.0139134.ref024" ref-type="bibr"> 24 </xref> ]. Subtilase serine proteases are involved in the mediation of a signal that controls the formation of guard cells and are involved in the increase of stomatal density [ <xref rid="pone.0139134.ref025" ref-type="bibr"> 25 </xref> ]. Down-regulation of subtilase-like serine protease in PI 416937 under high VPD might be a signal to form fewer guard cells and decrease stomatal density to control transpiration rate. Indeed, stomatal density is affected by <span class="trait">drought</span> stress in the perennial grass <italic> Leymus chinensis </italic> [ <xref rid="pone.0139134.ref026" ref-type="bibr"> 26 </xref> ]. Measuring guard cell density and size after prolonged exposure of PI 416937 to high VPD could determine if this slow-wilting genotype&#x02019;s stomatal density is particularly sensitive to high VPD.
					</p>
				
					<p>&nbsp;&nbsp;
						Genes regulating plant hormone signaling pathways were differentially expressed in response to high VPD in PI 416937 (24) ( <xref rid="pone.0139134.g005" ref-type="fig"> Fig 5 </xref> ). Transcript level changes in response to high VPD indicate that some genes in the abscisic acid, gibberellin, auxin and brassinosteroid related pathways are repressed (16). A transcript encoding a protein responsible for the hydroxylation of abscisic acid was used to validate the transcriptome profiling <span class="gene">(Glyma16g08340,</span> <xref rid="pone.0139134.g003" ref-type="fig"> Fig 3A </xref> ). The level of this transcript was substantially increased in PI 416937 under high VPD ( <xref rid="pone.0139134.g005" ref-type="fig"> Fig 5 </xref> ). Genes involved in the jasmonate pathway have a mixed profile of regulation and may be redirecting jasmonate pathways in response to high VPD. Induction of jasmonic acid biosynthesis genes is an early signal of <span class="trait">drought</span> stress in chickpea that conferred better <span class="trait">drought</span> tolerance [ <xref rid="pone.0139134.ref029" ref-type="bibr"> 29 </xref> ].The plant hormone ethylene regulates plant development and responses to various environmental stresses [ <xref rid="pone.0139134.ref030" ref-type="bibr"> 30 </xref> ]. <xref rid="pone.0139134.g005" ref-type="fig"> Fig 5 </xref> shows that the level of a suite of genes involved in the ethylene pathway increases in PI 416937 under high VPD. The level of transcripts encoding an ethylene forming enzyme <span class="gene">(Glyma08g05500)</span> and an ethylene responsive element <span class="gene">(Glyma10g33060)</span> in PI 416937 increased in response to an elevation in VPD ( <xref rid="pone.0139134.g003" ref-type="fig"> Fig 3A </xref> ). It is well known that ethylene and abscisic acid are involved in stomatal regulation [ <xref rid="pone.0139134.ref031" ref-type="bibr"> 31 </xref> , <xref rid="pone.0139134.ref032" ref-type="bibr"> 32 </xref> ]. The response of transcripts associated with phytohormone metabolism in the current study also supports the role of these hormones in the stomatal regulation of PI 416937 exposed to high VPD to control transpiration.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative analysis of endogenous hormones level in two soybean ( <italic> Glycine max </italic> L.) lines differing in waterlogging tolerance</h2>
			<h3>PMID:26442028</h3>
				
					<h3>PMCID:4585003</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Kim </surname>  <given-names> Yoon-Ha </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/218780/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hwang </surname>  <given-names> Sun-Joo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/272683/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Waqas </surname>  <given-names> Muhammad </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/199620/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Khan </surname>  <given-names> Abdul L. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/268421/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Joon-Hee </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/272656/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Jeong-Dong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> In-Jung </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/272655/overview"/>  </contrib>
				<br/> 
				年份：<day> 26 </day>  <month> 8 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						For obtaining maximum yeid, various cultivation factors including soil condition, irrigation, fertilizer supply, protection from fungal and insect attack, environmental condition etc., have to be optimized (Chen et al., <xref rid="B9" ref-type="bibr"> 2011 </xref> ). Cultivation factors such as soil condition, irrigation, fertilizers supplementation, and crop protection from disease can be modulated through human efforts; however, controlling the environmental conditions in a cultivation area is almost impossible (Chen et al., <xref rid="B9" ref-type="bibr"> 2011 </xref> ; <span class="gene">Fan</span> et al., <xref rid="B15" ref-type="bibr"> 2012 </xref> ; Ahmed et al., <xref rid="B1" ref-type="bibr"> 2013 </xref> ). Environmental conditions in cultivation areas are rapidly degrading and mainly attributed to the global climate change. As a result, various meteorological disasters including serious <span class="trait">drought,</span> salinity, unusual high and low temperatures, and freezing and flooding events constantly occur in crop cultivation areas (Fan et al., <xref rid="B15" ref-type="bibr"> 2012 </xref> ). To overcome these poor environmental conditions, agriculture scientists are constantly trying to develop new crop varieties that can adapt to or are tolerant to unfavorable environmental conditions (Ahmed et al., <xref rid="B1" ref-type="bibr"> 2013 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Soybean <span class="trait">yield</span> in relation to distance from the Itaipu reservoir</h2>
			<h3>PMID:26530053</h3>
				
					<h3>PMCID:4927606</h3>
					<h4>期刊:International Journal of Biometeorology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> de Faria </surname>  <given-names> Rog&#x000e9;rio Teixeira </given-names>  </name>  <address>  <email> rogeriofaria@fcav.unesp.br </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Junior </surname>  <given-names> Ruy Cas&#x000e3;o </given-names>  </name>  <address>  <email> ruycasao@iapar.br </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Werner </surname>  <given-names> Simone Silmara </given-names>  </name>  <address>  <email> simoneswerner@gmail.com </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Junior </surname>  <given-names> Luiz Ant&#x000f4;nio Zan&#x000e3;o </given-names>  </name>  <address>  <email> lzanao@iapar.br </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hoogenboom </surname>  <given-names> Gerrit </given-names>  </name>  <address>  <email> gerrit.hoogenboom@wsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff5"/>  </contrib>  <aff id="Aff1">  <label/> Univ. Estadual Paulista, Via de Acesso Prof. Paulo Donato Castellane s/n, Jaboticabal, SP 14884-900 Brazil  </aff>  <aff id="Aff2">  <label/> Fapeagro, Rua Paranagu&#x000e1;, 1672, Londrina, PR 86015-030 Brazil  </aff>  <aff id="Aff3">  <label/> Epagri, Rua Jo&#x000e3;o Jos&#x000e9; Godinho, S/N, Lages, SC 88502970 Brazil  </aff>  <aff id="Aff4">  <label/> Iapar, BR 163, km 188, Santa Tereza do Oeste, PR 85825-000 Brazil  </aff>  <aff id="Aff5">  <label/> Washington State University, 24106 North Bunn Road, Prosser, WA 99350-8694 USA  </aff>
				<br/> 
				年份：<day> 17 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Changes in temperature within a crop&#x02019;s sensitivity range impact the physiological processes and <span class="trait">yield</span> (Board and Kahlon <xref ref-type="bibr" rid="CR4"> 2011 </xref> ). Variations in air temperature mainly affect the duration of phenological stages, carbon balance (photosynthesis and respiration), and growth, abortion, and senescence of plant organs (Boote et al. <xref ref-type="bibr" rid="CR6"> 2010 </xref> ). For soybean, increased temperature affects grain <span class="trait">yield</span> by shortening the phenological stages (Seddigh and Joliff <xref ref-type="bibr" rid="CR30"> 1984 </xref> ), especially the duration of seed development and grain filling stages (Boote <xref ref-type="bibr" rid="CR5"> 2011 </xref> ), resulting in smaller seeds and lower yields (Egli and Wardlaw <xref ref-type="bibr" rid="CR11"> 1980 </xref> ). Temperatures above 40&#x000a0;&#x000b0;C reduce photosynthetic capacity and canopy development, and temperatures below 15&#x000a0;&#x000b0;C can negatively affect vegetative development, flowering, and <span class="trait">yield</span> components (Board and Kahlon <xref ref-type="bibr" rid="CR4"> 2011 </xref> ; Embrapa <xref ref-type="bibr" rid="CR13"> 2011 </xref> ). Diurnal temperature variations between 26 and 36&#x000a0;&#x000b0;C do not cause significant alterations in photosynthesis and are not considered an important factor in defining soybean grain <span class="trait">yield</span> (Campbell et al. <xref ref-type="bibr" rid="CR7"> 1990 </xref> ; Gibson and Mullen <xref ref-type="bibr" rid="CR16"> 1996 </xref> ; Boote <xref ref-type="bibr" rid="CR5"> 2011 </xref> ). In the sites neighboring the Itaipu reservoir, the mean and maximum temperatures during the soybean reproductive phase in January and February are between 25 and 27&#x000a0;&#x000b0;C and 32 and 34&#x000a0;&#x000b0;C, respectively; thus, they are within the temperature range considered to produce a low response to photosynthesis. Growth respiration (growth conversion efficiency) is unaffected by temperatures below the extreme values (Penning de Vries et al. <xref ref-type="bibr" rid="CR25"> 1974 </xref> ); however, maintenance respiration increases with increasing temperature (Mccree <xref ref-type="bibr" rid="CR24"> 1974 </xref> ) and consumes assimilates, which reduces assimilate availability for biomass accumulation and decreases soybean grain <span class="trait">yield</span> (Board and Kahlon <xref ref-type="bibr" rid="CR4"> 2011 </xref> ). Because soybean is a <span class="gene">C3</span> plant, it is less sensitive to increased respiration because of increased nocturnal temperatures (Peters et al. <xref ref-type="bibr" rid="CR26"> 1971 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome-wide functional characterization reveals novel relationships among differentially expressed transcripts in developing soybean embryos</h2>
			<h3>PMID:26572793</h3>
				
					<h3>PMCID:4647491</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Aghamirzaie </surname>  <given-names> Delasa </given-names>  </name>  <address>  <email> delasa@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Batra </surname>  <given-names> Dhruv </given-names>  </name>  <address>  <email> dbatra@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Heath </surname>  <given-names> Lenwood S. </given-names>  </name>  <address>  <email> heath@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Schneider </surname>  <given-names> Andrew </given-names>  </name>  <address>  <email> andrews5@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Grene </surname>  <given-names> Ruth </given-names>  </name>  <address>  <email> grene@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Collakova </surname>  <given-names> Eva </given-names>  </name>  <address>  <email> collakov@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <aff id="Aff1">  <label/> Genetics, Bioinformatics and Computational Biology Program, Virginia Tech, Blacksburg, VA 24061 USA  </aff>  <aff id="Aff2">  <label/> Bradley Department of Electrical and Computer Engineering, Virginia Tech, Blacksburg, VA 24061 USA  </aff>  <aff id="Aff3">  <label/> Department of Computer Science, Virginia Tech, Blacksburg, VA 24061 USA  </aff>  <aff id="Aff4">  <label/> Department of Plant Pathology, Physiology, and Weed Science, Virginia Tech, Blacksburg, VA 24061 USA  </aff>
				<br/> 
				年份：<day> 16 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						GCR1 and CPK11 are two putative regulators of seed development belonging to the group of 318 ABA- and/or FUS3-related transcripts. <italic> GCR1 </italic> and <italic> CPK11 </italic> pre-mRNAs were alternatively spliced in developing soybean embryos and the resulting SVs were present in different super-clusters. The domain composition of each GCR1 and CPK11 SVs was confirmed by using the InterPro database [ <xref ref-type="bibr" rid="CR16"> 16 </xref> ]. In Arabidopsis, GCR1 <span class="gene">(At1g48270)</span> is an ABA-responsive, G-protein-related receptor component distinct from the well-studied RCAR group of receptors [ <xref ref-type="bibr" rid="CR67"> 67 </xref> ]. CPK11 <span class="gene">(At1g35670)</span> is a protein kinase acting as a positive regulator of ABA/FUS3-mediated responses during seed filling [ <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. Guilt-by-association of GCR1 and CPK11 SVs with transcripts of known functions can <span class="trait">yield</span> improved understanding of their regulation and function.
					</p>
				
					<p>&nbsp;&nbsp;
						Interestingly, the novel CPK11 SV was expressed during mid-to-late seed filling and lacked both EFh domains present in the canonical SV. The EFh domains occur as pairs and are responsible for changing protein conformation upon Ca <sup> 2+ </sup> binding to modulate protein activity [ <xref ref-type="bibr" rid="CR100"> 100 </xref> ]. The novel CPK11 variant could phosphorylate its targets during mid-to-late seed filling independently of Ca <sup> 2+ </sup> -mediated signaling. Its direct co-expressers AFP2 and DRIP2 are associated with the B3 network through ABA/ABI3/ABI5 and FUS3 signaling, respectively. DRIP2 ubiquitinates the positive regulator of ABA-independent <span class="trait">drought</span> responses <span class="gene">DREB2A</span> [ <xref ref-type="bibr" rid="CR101"> 101 </xref> &#x02013; <xref ref-type="bibr" rid="CR103"> 103 </xref> ]. Protein-protein interactions of the novel DRIP2 SV are likely compromised due to the absence of the C3HC4-type RING-finger domain and functions of this SV in the B3 network and early seed filling phases remain to be elucidated.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Nutritional value of raw soybeans, extruded soybeans, roasted soybeans and tallow as fat sources in early lactating dairy cows</h2>
			<h3>PMID:26623299</h3>
				
					<h3>PMCID:4655767</h3>
					<h4>期刊:Open Veterinary Journal</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Amanlou </surname>  <given-names> H. </given-names>  </name>  <xref ref-type="aff" rid="aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Maheri-Sis </surname>  <given-names> N. </given-names>  </name>  <xref ref-type="aff" rid="aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bassiri </surname>  <given-names> S. </given-names>  </name>  <xref ref-type="aff" rid="aff2"> 2 </xref>  <xref ref-type="corresp" rid="cor1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mirza-Aghazadeh </surname>  <given-names> A. </given-names>  </name>  <xref ref-type="aff" rid="aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Salamatdust </surname>  <given-names> R. </given-names>  </name>  <xref ref-type="aff" rid="aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Moosavi </surname>  <given-names> A. </given-names>  </name>  <xref ref-type="aff" rid="aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Karimi </surname>  <given-names> V. </given-names>  </name>  <xref ref-type="aff" rid="aff4"> 4 </xref>  </contrib>
				<br/> 
				年份：<day> 29 </day>  <month> 7 </month>  <year> 2012 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Similary, Duske <italic> et al </italic> . (2009) suspected that higher milk fat percentages are due to triacylglycerols being transported by chylomicrons and utilized by the mammary gland. A numerical low content and <span class="trait">yield</span> of milk fat for extruded soybean-fed cows compared with other soybean diets may be related to more rapid release of oil into the rumen, resulting from extrusion rupturing of the fat micelles (Scott <italic> et al. </italic> , 1991). Under certain circumstances, rumen biohydrogenation of linoleice and linolenic acids results in unique fatty acids, such as <italic> trans </italic> -10, <italic> cis </italic> -12 conjugated linolenic acid that are potent inhibitors of <italic> de novo </italic> fatty acids synthesis (Caldari-Torres <italic> et al. </italic> , 2011). However, milk fat <span class="trait">yield</span> for cows fed extruded soybeans was still significantly more ( <italic> P </italic> &#x0003c;0.01) than cows fed the control diet, implying that the decrease in milk <span class="gene">C4</span> to C16 fatty acid <span class="trait">yield</span> was compensated by an increase in C18 fatty acid <span class="trait">yield</span> (Glasser <italic> et al. </italic> , 2008). Moreover, <span class="trait">yield</span> and percentage of milk fat was higher when soybean oil was fed continuously (as in the current experiment) than when fed in two meals (Chouinard <italic> et al. </italic> , 1997b).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Productivity and sustainability of rainfed wheat-soybean system in the North China Plain: results from a long-term experiment and crop modelling</h2>
			<h3>PMID:26627707</h3>
				
					<h3>PMCID:4667244</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Qin </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Daozhong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Xisheng </given-names>  </name>  <xref ref-type="corresp" rid="c3"> c </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Taiming </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oenema </surname>  <given-names> Oene </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Soil and Fertilizer Research Institute, Anhui Academy of Agricultural Sciences </institution> , Hefei, 230031,  <country> China </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Key Laboratory of Nutrient Cycling and Resources Environment of Anhui Province </institution> , Hefei,  <country> China </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Anhui Center of Agricultural Meteorology </institution> , Hefei, 230031,  <country> China </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> Department of Soil Quality, Wageningen UR </institution> , 6700 AA, Wageningen,  <country> the Netherlands </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> Alterra, Wageningen UR </institution> , 6700 AA, Wageningen,  <country> the Netherlands </country>  </aff>
				<br/> 
				年份：<day> 30 </day>  <month> 10 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Model experiment <span class="gene">E1</span> simulated crop growth with two levels of initial soil water content at seeding, i.e. a low level with 60% field capacity (FC) and a high level with 75% FC. The range from 60 to 75% FC largely represented the initial soil water content during the experimental period. Model experiment E2 aimed at testing the effects of straw mulching, and model experiment E3 for the effects of plastic film cover. The effectiveness of straw mulching in reducing soil evaporation was estimated as 50% and that of plastic cover at 90%, which are the default values in the AquaCrop model <xref ref-type="bibr" rid="b44"> 44 </xref> . However, plastic film covers around 80% of the field in practice. Hence, the overall soil evaporation reduction by plastic film was set at 72%. Model experiments E4 tested the effects of a smart irrigation that automatically replenishes the soil water content back to field capacity when 80% of total available soil water is depleted during the crop growing season. We ran these model experiments for the period of 1982&#x02013;2012, assuming no soil <span class="trait">fertility</span> stress and keeping all input data (e.g., climatic and soil information) and crop parameters the same in all simulations.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Expanding Omics Resources for Improvement of Soybean Seed Composition Traits</h2>
			<h3>PMID:26635846</h3>
				
					<h3>PMCID:4657443</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chaudhary </surname>  <given-names> Juhi </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/262086/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant B. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/104588/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sonah </surname>  <given-names> Humira </given-names>  </name>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/162853/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> Rupesh K. </given-names>  </name>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103657/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Vuong </surname>  <given-names> Tri D. </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/162861/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249960/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>
				<br/> 
				年份：<day> 05 </day>  <month> 11 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Due to the advantages of <span class="gene">GS</span> over conventional methods, it has been successfully applied to a variety of crops, such as wheat and maize (Crossa et al., <xref rid="B31" ref-type="bibr"> 2010 </xref> ), pear (Iwata et al., <xref rid="B60" ref-type="bibr"> 2013 </xref> ), sugar beet (W&#x000fc;rschum and Kraft, <xref rid="B155" ref-type="bibr"> 2014 </xref> ), and is constantly being used for other crop species. There are only a few reports for the application of <span class="gene">GS</span> in soybean (Hu et al., <xref rid="B57" ref-type="bibr"> 2011 </xref> ; Shu et al., <xref rid="B129" ref-type="bibr"> 2012 </xref> ; Bao et al., <xref rid="B9" ref-type="bibr"> 2014 </xref> ; Jarqu&#x000ed;n et al., <xref rid="B62" ref-type="bibr"> 2014 </xref> ). Hu et al. ( <xref rid="B57" ref-type="bibr"> 2011 </xref> ) studied primary embryogenesis capacity in soybean using 126 RILs and 80 SSRs and reported a strong correlation ( <italic> r </italic> <sup> 2 </sup> = 0.78) between GEBVs and phenotypic data (Hu et al., <xref rid="B57" ref-type="bibr"> 2011 </xref> ). A study was performed on soybean HSW using 79 sequence-characterized amplified region (SCAR) markers for 288 varieties. This study reported a correlation coefficient of 0.904 amongst the GEBVs and phenotypic values (Shu et al., <xref rid="B129" ref-type="bibr"> 2012 </xref> ). Recently, Bao et al. ( <xref rid="B9" ref-type="bibr"> 2014 </xref> ) investigated soybean cyst nematode <span class="trait">resistance</span> via genotyping of 282 accessions employing the 1536 SNP array and demonstrated a significant prediction accuracy (0.59&#x02013;0.67) for soybean cyst nematode <span class="trait">resistance</span> in soybean. Another study reported a prediction accuracy of 0.64 for grain <span class="trait">yield,</span> which indicates a good potential of <span class="gene">GS</span> utilization strategy in soybean (Jarqu&#x000ed;n et al., <xref rid="B62" ref-type="bibr"> 2014 </xref> ). It is noticeable that implementation of a <span class="gene">GS</span> approach is needed in soybean seed related traits. Moreover, with the declining genotyping costs and increasing phenotyping costs, <span class="gene">GS</span> can be helpful to mitigate many of the selections associated with phenotyping and ultimately speed up the breeding cycles. Hence, the <span class="gene">GS</span> statistical method is more feasible and more emphasis should be given to employ this approach in the improvement of seed composition with available genotypic and phenotypic data.
					</p>
				
					<p>&nbsp;&nbsp;
						The availability of the sequence information provides new avenues for the engineering of soybean seed composition. The new &#x0201c;omics&#x0201d; approaches offers a potential resource to comprehend the metabolic regulatory network governing seed storage compound accumulation. To accommodate the global food demand and growing population needs, several genetic engineering and genome editing strategies need to be employed to obtain the next-generation crop at faster rate than the conventional breeding. Studies have reported improved soybean seed oil content via X-ray, and EMS-induced mutations in the desired genes (Dierking and Bilyeu, <xref rid="B37" ref-type="bibr"> 2009 </xref> ; Pham et al., <xref rid="B106" ref-type="bibr"> 2010 </xref> , <xref rid="B107" ref-type="bibr"> 2011 </xref> ). For example, deletion of a 100 kb sequence on Chr. 10 in a mutant line M23 increased oleic acid content but caused <span class="trait">yield</span> drag (Pham et al., <xref rid="B107" ref-type="bibr"> 2011 </xref> ). Similarly, the <span class="gene">SACPD-C</span> deletion in an A6 mutant line showed elevated stearic acid content; however, simultaneously a reduced seed <span class="trait">yield</span> was also observed (Gillman et al., <xref rid="B45" ref-type="bibr"> 2014 </xref> ). Nevertheless, conventional methods require labor intensive screening procedure to identify germplasm accessions with desired phenotypes and stable performance in multiple generations and environments.
					</p>
				
					<p>&nbsp;&nbsp;
						A number of potential strategies have been outlined, including combining diverse and wild germplasm associated with specific seed traits and studying them in various &#x0201c;omics&#x0201d; dimensions to enhance our knowledge. It is projected that the combined soybean seed content of protein and oil should be increased by 10% by 2025 ( <ext-link ext-link-type="uri" xlink:href="http://unitedsoybean.org/about-usb/strategic-planning/"> http://unitedsoybean.org/about-usb/strategic-planning/ </ext-link> ). Most of the US soybeans have 59&#x02013;62% combined protein and oil content. It is a great challenge to develop soybeans with &#x0003e;70% combined protein and oil. In addition, there is a need to develop soybean germplasm with increased meal protein (48&#x02013;50%) by improving amino acid balance (10% increase in methionine, cysteine, and threonine) and environmental stability without reducing the seed oil or <span class="trait">yield.</span> Therefore, it is essential to identify and validate independent novel QTL for high protein content and to breakdown the negative correlation with <span class="trait">yield</span> and oil content. To meet this challenge, it is necessary to bridge the <span class="gene">gap</span> between biological and computational system by integrating multiple &#x0201c;omics&#x0201d; approaches, statistical genetic models, and bioinformatics tools, such as data pipelines, cloud computing, and user-friendly script developments. Despite the challenges, advances in &#x0201c;omics&#x0201d; technologies offers a promising potential to create next-generation soybeans with the desired seed composition traits. With constant developments in breeding technologies in conjunction with &#x0201c;omics&#x0201d; approaches, it is foreseeable in the future that a high yielding soybean cultivar with balanced amino acid, increased oil and protein content can be subsequently developed. In summary, this review provides a glimpse of advances made in improvement of soybean seed traits using integrated omics approaches. The discussed information would be a useful resource to accelerate desired seed composition improvement and, in turn, meet the global soybean demand in coming years.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide transcriptome analyses of developing seeds from low and normal phytic acid soybean lines</h2>
			<h3>PMID:26678836</h3>
				
					<h3>PMCID:4683714</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Redekar </surname>  <given-names> Neelam R. </given-names>  </name>  <address>  <email> neelamrr@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Biyashev </surname>  <given-names> Ruslan M. </given-names>  </name>  <address>  <email> rbiyashe@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jensen </surname>  <given-names> Roderick V. </given-names>  </name>  <address>  <email> rvjensen@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Helm </surname>  <given-names> Richard F. </given-names>  </name>  <address>  <email> helmrf@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grabau </surname>  <given-names> Elizabeth A. </given-names>  </name>  <address>  <email> egrabau@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Maroof </surname>  <given-names> M. A. Saghai </given-names>  </name>  <address>  <email> smaroof@vt.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Department of Crop and Soil Environmental Sciences, Virginia Tech, 185 AgQuad Lane, 24061 Blacksburg, VA USA  </aff>  <aff id="Aff2">  <label/> Department of Biological Sciences, Virginia Tech, Life Science I building, 24061 Blacksburg, VA USA  </aff>  <aff id="Aff3">  <label/> Department of Biochemistry, Virginia Tech, Life Science I building, 24061 Blacksburg, VA USA  </aff>  <aff id="Aff4">  <label/> Department of Plant Pathology, Physiology, and Weed Science, Virginia Tech, Price Hall, 24061 Blacksburg, VA USA  </aff>
				<br/> 
				年份：<day> 10 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The PA biosynthesis pathway plays a vital role in maintaining homeostasis. Several pathway intermediates, such as <italic> myo </italic> -inositol-1,4,5-trisphosphate, act as secondary messengers in signal transduction and are known to regulate growth and developmental processes, including phosphorus and mineral storage, DNA repair, chromatin remodeling, RNA editing and export, ATP generation, regulation of gene expression, regulation of guard cells, auxin metabolism and cell-wall polysaccharide biosynthesis [ <xref ref-type="bibr" rid="CR17"> 17 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> &#x02013; <xref ref-type="bibr" rid="CR24"> 24 </xref> ]. Numerous studies have reported the effects of <italic> lpa </italic> on plant growth and development. An RNAi-mediated <italic> mips1 </italic> knockdown in soybean was reported to inhibit seed development along with reduced PA content [ <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. Similarly, seed embryo defects were reported for Arabidopsis and common bean ( <italic> Phaseolus vulgaris L. </italic> ) <italic> mips </italic> mutants [ <xref ref-type="bibr" rid="CR26"> 26 </xref> , <xref ref-type="bibr" rid="CR27"> 27 </xref> ]. The <italic> mips </italic> mutation regulates raffinose family oligosaccharide pathway, and mutants exhibit impaired pathogen <span class="trait">resistance,</span> programmed cell death in leaves, and polar auxin transport causing deformed cotyledon development [ <xref ref-type="bibr" rid="CR26"> 26 </xref> , <xref ref-type="bibr" rid="CR28"> 28 </xref> &#x02013; <xref ref-type="bibr" rid="CR33"> 33 </xref> ]. The <italic> <span class="gene">mrp</span> </italic> mutants are known to exhibit <italic> lpa </italic> phenotypes in soybean, rice, maize, and Arabidopsis [ <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR13"> 13 </xref> , <xref ref-type="bibr" rid="CR34"> 34 </xref> , <xref ref-type="bibr" rid="CR35"> 35 </xref> ]. MRP knockout studies in Arabidopsis exhibit phenotypes such as insensitivity to abscisic acid-mediated germination and unresponsive stomata opening, resulting in reduced transpiration rate and increased <span class="trait">drought</span> tolerance; which were rescued by MRP overexpression [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. Finally, <italic> lpa </italic> crops are known to show poor seedling emergence, resulting in reduced crop <span class="trait">yield,</span> which decreases the agronomic value of <italic> lpa </italic> crops [ <xref ref-type="bibr" rid="CR37"> 37 </xref> &#x02013; <xref ref-type="bibr" rid="CR39"> 39 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The defense-related genes identified in this study lie upstream of the apoptotic processes. For example, the LRR and NB-ARC domain-containing disease <span class="trait">resistance</span> genes are involved in initiating the defense response via induction of salicylic acid (SA) and formation of reactive oxygen species (ROS), both of which ultimately leading to cell death. <italic> Arabidopsis thaliana mips1 </italic> mutants were shown to accumulate SA and ceramide, both associated with formation of a cell death lesion area, with the phenotype rescued by treating with either <italic> myo </italic> -inositol or galactinol [ <xref ref-type="bibr" rid="CR26"> 26 </xref> , <xref ref-type="bibr" rid="CR30"> 30 </xref> , <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. Apoptosis in <italic> mips1 </italic> mutants is also regulated by light intensity and ROS such as peroxisomal hydrogen peroxide and the superoxide anion [ <xref ref-type="bibr" rid="CR26"> 26 </xref> , <xref ref-type="bibr" rid="CR30"> 30 </xref> , <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. <italic> Myo </italic> -inositol mitigates SA-dependent apoptosis and defense responses [ <xref ref-type="bibr" rid="CR43"> 43 </xref> ]. Although apoptosis is associated with <italic> mips1 </italic> mutation and not with <italic> <span class="gene">mrp</span> </italic> mutation, induction of the defense-related genes in <italic> 3mlpa </italic> may not be solely due to <italic> mips1 </italic> mutation. The ADR1-L1 genes <span class="gene">(Glyma14g08700</span> and <span class="gene">Glyma17g36420),</span> which belong to a subgroup of CNL-A clade of coiled-coil NBS-LRR gene family, showed two-fold higher expression in <italic> 3mlpa </italic> mutant [ <xref ref-type="bibr" rid="CR44"> 44 </xref> ]. The ADR1 genes are positive regulators of SA-mediated cell death and defense response [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. Enhanced expression of ADR1 gene was previously shown to establish <span class="trait">drought</span> tolerance in presence of SA [ <xref ref-type="bibr" rid="CR46"> 46 </xref> ]. And interestingly, the <italic> mrp5 </italic> mutants of <italic> Arabidopsis </italic> are also known to be <span class="trait">drought</span> tolerant [ <xref ref-type="bibr" rid="CR36"> 36 </xref> ]. Role of MRP genes in regulating defense responses is still unknown, and further studies are required to confirm the association of the <span class="trait">drought</span> tolerant phenotype of the <italic> <span class="gene">mrp</span> </italic> mutants and up regulation of ADR1 genes as in <italic> 3mlpa </italic> . In addition, several WRKY transcription factor family genes that regulate expression of other defense-related genes were also up-regulated in early stages of seed development in <italic> 3mlpa </italic> mutant. In summary, the <italic> lpa </italic> -causing mutations in <italic> 3mlpa </italic> mutant may play a role in initiating defense responses during seed development.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification of <italic> Fusarium virguliforme </italic> FvTox1-Interacting Synthetic Peptides for Enhancing Foliar Sudden Death Syndrome <span class="trait">Resistance</span> in Soybean</h2>
			<h3>PMID:26709700</h3>
				
					<h3>PMCID:4692527</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Bing </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Swaminathan </surname>  <given-names> Sivakumar </given-names>  </name>  <xref ref-type="aff" rid="aff001"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bhattacharyya </surname>  <given-names> Madan K. </given-names>  </name>  <xref rid="cor001" ref-type="corresp"> * </xref>  <xref ref-type="aff" rid="aff001"/>  </contrib>
				<br/> 
				年份：<day> 1 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We added two cysteine residues, one on each side of the nine synthetic peptides, for improving the binding affinities of the nine fusion peptides to FvTox1 ( <xref rid="pone.0145156.g003" ref-type="fig"> Fig 3 </xref> ). It is expected that formation of the sulfhydryl bridges between the added two flanking <span class="gene">cys</span> residues could improve the binding affinities of the FvTox1-interacting peptides to FvTox1. When two flanking <span class="gene">cys</span> residues were added, the <italic> in vivo </italic> binding affinity of FvTox1 with two (Pep5 and Pep8) of the five fusion peptides in yeast was improved as compared to their corresponding original forms with no <span class="gene">cys</span> residues at their flanking sites ( <xref rid="pone.0145156.t004" ref-type="table"> Table 4 </xref> ). On the contrary, reduced interaction strength of FvTox1 with PEP7 with two flanking <span class="gene">cys</span> residues was observed as compared to that of FvTox1 with PEP7 with no <span class="gene">cys</span> residues ( <xref rid="pone.0145156.g003" ref-type="fig"> Fig 3 </xref> ; <xref rid="pone.0145156.t004" ref-type="table"> Table 4 </xref> ). These results suggest that expression of one or more of the 18 FvTox1-interacting peptides could neutralize FvTox1 <italic> in planta </italic> and enhance foliar SDS <span class="trait">resistance</span> in transgenic soybean plants.
					</p>
				
					<p>&nbsp;&nbsp;
						In mammals, antibodies bind to target using six antibody complementarity determining regions [ <xref rid="pone.0145156.ref031" ref-type="bibr"> 31 </xref> ]. The single peptides have weaker affinity compared to antibody. In animals, to increase affinity of binding single peptides to target proteins many copies of peptides are attached to synthetic scaffold such as liposome [ <xref rid="pone.0145156.ref016" ref-type="bibr"> 16 </xref> , <xref rid="pone.0145156.ref025" ref-type="bibr"> 25 </xref> ]. In plants, peptides can be fused to selected proteins as display scaffold for delivering to correct cellular or extracellular spaces [ <xref rid="pone.0145156.ref026" ref-type="bibr"> 26 </xref> ]. FvTox1 is the major toxin that induces foliar SDS. The toxin protein has been localized to chloroplasts (H.K. Brar and M.K. Bhattacharyya, unpublished). Light is essential for FvTox1-induced foliar SDS symptoms [ <xref rid="pone.0145156.ref007" ref-type="bibr"> 7 </xref> , <xref rid="pone.0145156.ref009" ref-type="bibr"> 9 </xref> ]. A thioredoxin protein, <span class="gene">GmTRX2,</span> localized to chloroplasts, has been identified as the candidate FvTox1-interacting target soybean protein (R.N. Pudake, and M.K. Bhattacharyya, unpublished). Targeting the FvTox1-interacting peptides to chloroplasts using a suitable chloroplast protein such as the FvTox1-interacting <span class="gene">GmTRX2</span> as display scaffold could compete with endogenous FvTox1-interacting protein for FvTox1 binding and thereby suppress the foliar SDS development in transgenic soybean plants. If successful, this could be a suitable biotechnological approach for enhancing SDS <span class="trait">resistance</span> in soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Ncl </italic> Synchronously Regulates Na <sup> + </sup> , K <sup> + </sup> , and Cl <sup> <bold> &#x02212; </bold> </sup> in Soybean and Greatly Increases the Grain <span class="trait">Yield</span> in Saline Field Conditions</h2>
			<h3>PMID:26744076</h3>
				
					<h3>PMCID:4705485</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Do </surname>  <given-names> Tuyen Duc </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a2"> 2 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Huatao </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a3"> 3 </xref>  <xref ref-type="author-notes" rid="n1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hien </surname>  <given-names> Vu Thi Thu </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hamwieh </surname>  <given-names> Aladdin </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamada </surname>  <given-names> Tetsuya </given-names>  </name>  <xref ref-type="aff" rid="a6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sato </surname>  <given-names> Tadashi </given-names>  </name>  <xref ref-type="aff" rid="a7"> 7 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Yongliang </given-names>  </name>  <xref ref-type="aff" rid="a8"> 8 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cong </surname>  <given-names> Hua </given-names>  </name>  <xref ref-type="aff" rid="a8"> 8 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shono </surname>  <given-names> Mariko </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Suenaga </surname>  <given-names> Kazuhiro </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Donghe </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Japan International Research Center for Agricultural Sciences (JIRCAS) </institution> , Tsukuba, Ibaraki, 305-8686,  <country> Japan </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Cuulong Delta Rice Research Institute (CLRRI) </institution> , Thoilai, Cantho,  <country> Vietnam </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> Institute of Vegetable Crops, Jiangsu Academy of Agricultural Sciences </institution> , Nanjing, Jiangsu,  <country> China </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> Genetic Engineering Division, Agricultural Genetics Institute </institution> , Hanoi,  <country> Vietnam </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> International Center for Agricultural Research in the Dry Areas (ICARDA) </institution> , Cairo,  <country> Egypt </country>  </aff>  <aff id="a6">  <label> 6 </label>  <institution> Graduate School of Agriculture, Hokkaido University </institution> , Sapporo, Hokkaido,  <country> Japan </country>  </aff>  <aff id="a7">  <label> 7 </label>  <institution> Graduate School of Life Sciences, Tohoku University </institution> , Sendai, Miyagi,  <country> Japan </country>  </aff>  <aff id="a8">  <label> 8 </label>  <institution> Institute of Crop Germplasm Resources, Xinjiang Academy of Agricultural Sciences </institution> , Urumqi, Xinjiang,  <country> China </country>  </aff>
				<br/> 
				年份：<day> 07 </day>  <month> 12 </month>  <year> 2015 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A wild soybean accession W05, from which a salt-tolerant gene ( <italic> <span class="gene">GmCHX1</span> </italic> ) was identified, was proposed to be used be used for soybean breeding for salt tolerance <xref ref-type="bibr" rid="b12"> 12 </xref> . Guan <italic> et al. </italic> also identified the same salt-tolerant gene ( <italic> <span class="gene">GmSALT3</span> </italic> ) from a Chinese commercial cultivar Tiefeng 8 <sup> 20 </sup> . The present study detected many wild and cultivated soybean accessions that harbored the tolerant allele of <italic> Ncl </italic> . Some cultivated soybean cultivars that highly express <italic> Ncl </italic> and show a greater degree of salt tolerance, such as S-100 and FT-Abyrara, were identified. The fact that both wild and cultivated soybeans showed variations in salt tolerance suggested that the loss-of-function mutation in <italic> Ncl </italic> occurred before the domestication of soybean from its ancestral wild species. The close association between <italic> Ncl </italic> expression and salt tolerance also demonstrated that <italic> Ncl </italic> is effective in different genetic backgrounds. In the present study, we transgressed the salt tolerance allele from the wild soybean accession, JWS156-1, to a cultivated soybean cultivar, Jackson. After four cycles of backcrossing, the effect of the salt tolerance gene was confirmed. However, the BC <sub> 4 </sub> F <sub> 3 </sub> introgression lines still generated a lower biomass <span class="trait">yield</span> compared with its recurrent parent, Jackson, in a control condition. Therefore, the cultivated soybean germplasm may be more efficiently used in breeding cultivars with improved salt tolerance compared with that of wild soybean, which generally possesses several undesirable agronomic traits.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Molecular phylogeny and dynamic evolution of disease <span class="trait">resistance</span> genes in the legume family</h2>
			<h3>PMID:27229309</h3>
				
					<h3>PMCID:4881053</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zheng </surname>  <given-names> Fengya </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Wu </surname>  <given-names> Haiyang </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  <xref ref-type="aff" rid="Aff3"/>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Zhang </surname>  <given-names> Rongzhi </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Shiming </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Weiming </given-names>  </name>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wong </surname>  <given-names> Fuk-Ling </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Genying </given-names>  </name>  <xref ref-type="aff" rid="Aff4"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Zhao </surname>  <given-names> Shancen </given-names>  </name>  <address>  <email> zhaoshancen@genomics.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  <xref ref-type="aff" rid="Aff2"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Lam </surname>  <given-names> Hon-Ming </given-names>  </name>  <address>  <email> honming@cuhk.edu.hk </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <label/> Centre for Soybean Research, Partner State Key Laboratory of Agrobiotechnology and School of Life Sciences, The Chinese University of Hong Kong, New Territories, Hong Kong  </aff>  <aff id="Aff2">  <label/> BGI-Shenzhen, Shenzhen, 518083 China  </aff>  <aff id="Aff3">  <label/> HKU-BGI Bioinformatics Laboratory and Department of Computer Science, University of Hong Kong, Pofulam, Hong Kong  </aff>  <aff id="Aff4">  <label/> Crop research institution, Shandong Academy of Agricultural Sciences, Jinan, 250100 China  </aff>
				<br/> 
				年份：<day> 12 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Interestingly, the number of R proteins with the <span class="gene">TIR</span> domain experienced a distinct expansion in dicots when compared to monocots [ <xref ref-type="bibr" rid="CR45"> 45 </xref> ]. In our analysis, R proteins with the <span class="gene">TIR</span> domain make up a higher proportion in legumes than in grape, indicating that the number of R proteins with <span class="gene">TIR</span> domains might have experienced additional expansions after the divergence of legumes from other dicots. The lineage-specific expansion of TIR-domain R proteins might have diversified the <span class="trait">resistance</span> functions in combination with R proteins containing other domains in the legume species.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions</h2>
			<h3>PMID:27247288</h3>
				
					<h3>PMCID:4978888</h3>
					<h4>期刊:G3: Genes|Genomes|Genetics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jarquin </surname>  <given-names> Diego </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Specht </surname>  <given-names> James </given-names>  </name>  <xref ref-type="aff" rid="aff1"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lorenz </surname>  <given-names> Aaron </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> &#x02020; </sup>  </xref>  <xref ref-type="corresp" rid="cor1">  <sup> 1 </sup>  </xref>  </contrib>  <aff id="aff1">  <label> * </label> Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915 </aff>  <aff id="aff2">  <label> &#x02020; </label> Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108 </aff>
				<br/> 
				年份：<day> 23 </day>  <month> 5 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In addition to the phenotypic data routinely collected by the USDA and collaborators on the collection, an independent data set on MGs I&#x02013;V PIs was obtained to serve as an additional validation set. These data were collected by J. E. S. at the University of Nebraska in 2003 and 2004. Briefly, 101 accessions were selected from a larger set of &#x0223c;1500 accessions on the basis of acceptable <span class="trait">lodging,</span> seed shattering, disease <span class="trait">resistance,</span> and overall appearance. Most of these 101 accessions belong to MGs&#x000a0;II and III. They were evaluated in field trials under two water regimes, dryland and full irrigation, at Lincoln, NE. Plots were arranged in a randomized complete block design with four replications per water regime. Replications receiving the same water treatment were blocked together in the field. Plots consisted of two rows 0.76&#x000a0;m apart and 2.90&#x000a0;m long. Plots were machine harvested, and seed <span class="trait">yield</span> was adjusted to 13% seed moisture. Protein and oil concentration were measured using near-infrared reflectance spectroscopy. For use here, the data were divided into four water-regime&#x02013;year combinations. A linear model was fit to each dataset separately to calculate estimates of broad-sense heritability on an entry-mean basis. The linear model included <span class="gene">rep</span> (fixed) and accession (random).
					</p>
				
					<p>&nbsp;&nbsp;
						By ordering the trials by state, it is apparent that the northern locations of MN and IL predicted one another relatively well as compared to be predictive ability between <span class="gene">MS</span> and the northern locations. This pattern was more prominent for <span class="trait">yield</span> ( <ext-link ext-link-type="uri" xlink:href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS3.xlsx"> Table S3 </ext-link> ).
					</p>
				
					<p>&nbsp;&nbsp;
						As expected, a training set including data from the state being predicted ( <italic> One/Group </italic> or <italic> One/All </italic> ) performed much better than training sets not including data from the state being predicted ( <italic> Group/All </italic> ) ( <xref ref-type="fig" rid="fig4"> Figure 4 </xref> and <xref ref-type="table" rid="t4"> Table 4 </xref> ). A key question we wanted to address with this analysis was whether training sets should be created by dividing data among states, or if a universal training set including all data&#x02014;regardless of state&#x02014;would perform just as well or better. Little to no differences were observed between these two CV schemes for any trait and state combination ( <xref ref-type="fig" rid="fig4"> Figure 4 </xref> and <xref ref-type="table" rid="t4"> Table 4 </xref> ). This finding suggests that predictive abilities are not improved by maximizing training set size by combining across states, nor are they reduced by including data from environments as different as <span class="gene">MS</span> when predicting relative performance of early MG accessions in MN. A similar pattern was observed when variation for <span class="trait">lodging</span> and shattering was removed through inclusion of covariates.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Transcriptome sequencing and marker development in winged bean ( <italic> Psophocarpus tetragonolobus </italic> ; Leguminosae)</h2>
			<h3>PMID:27356763</h3>
				
					<h3>PMCID:4928180</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Vatanparast </surname>  <given-names> Mohammad </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shetty </surname>  <given-names> Prateek </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chopra </surname>  <given-names> Ratan </given-names>  </name>  <xref ref-type="aff" rid="a3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Doyle </surname>  <given-names> Jeff J. </given-names>  </name>  <xref ref-type="aff" rid="a4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sathyanarayana </surname>  <given-names> N. </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Egan </surname>  <given-names> Ashley N. </given-names>  </name>  <xref ref-type="corresp" rid="c2"> b </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> US National Herbarium (US), Department of Botany, Smithsonian Institution-NMNH, 10th and Constitution Ave </institution> , Washington DC, 20013,  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Department of Plant Biology, Michigan State University </institution> , 612 Wilson Road, Room 166, East Lansing, MI, 48824,  <country> USA </country>  </aff>  <aff id="a3">  <label> 3 </label>  <institution> United States Department of Agriculture, Agriculture Research Service </institution> , 3810 4th St., Lubbock, TX, 79415,  <country> USA </country>  </aff>  <aff id="a4">  <label> 4 </label>  <institution> Section of Plant Breeding &#x00026; Genetics, School of Integrative Plant Science, Cornell University </institution> , 412 Mann Library, Ithaca, NY, 14853,  <country> USA </country>  </aff>  <aff id="a5">  <label> 5 </label>  <institution> Department of Botany, Sikkim University </institution> , 5th Mile, Tadong, Gangtok, Sikkim, 737102,  <country> India </country>  </aff>
				<br/> 
				年份：<day> 14 </day>  <month> 06 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The unequal distribution of <italic> Psophocarpus </italic> STI sequences across the six subclades may be due to tissue specificity, depth of transcriptome reads, amplification of certain gene subfamilies or gene loss over time. As mentioned earlier, most of the winged bean KTIs currently known were characterized from seeds, yet all of these are present in subclade F, in spite of the fact that the CPP34-7 transcriptome did not include seed transcripts, but was sequenced from young leaves, shoots, and buds. This subclade also includes a <italic> Psophocarpus </italic> nodulin (Ptet_Q43325) expressed in nodules of winged bean, likely as a delayed response of the host plant to <italic> Rhizobium </italic> infection <xref ref-type="bibr" rid="b71"> 71 </xref> . Expression levels of STI genes in winged bean likely differ across plant tissues, as demonstrated in poplar <xref ref-type="bibr" rid="b64"> 64 </xref> , and this may be one explanation for unequal distribution of <italic> Psophocarpus </italic> sequences across the gene tree, although inclusion of such tissue-specific genes as nodulins argues against that. Unfortunately, we cannot determine tissue-specific expression of our winged bean STIs due to the fact that our transcriptomes were sequenced from pooled tissue samples. But, given the roles of KTIs in wound and herbivory defense, radiation of <span class="gene">KTI</span> genes would be evolutionarily beneficial to large-leaved, highly nutritious plants such as winged bean. Deeper sequencing of the transcriptomes across more tissue types would likely <span class="trait">yield</span> other STI gene family members in <italic> Psophocarpus </italic> and provide a more holistic view of STI gene family evolution in the winged bean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenotypic Characterization and Genetic Dissection of Growth Period Traits in Soybean ( <italic> Glycine max </italic> ) Using Association Mapping</h2>
			<h3>PMID:27367048</h3>
				
					<h3>PMCID:4930185</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Liu </surname>  <given-names> Zhangxiong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Li </surname>  <given-names> Huihui </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Fan </surname>  <given-names> Xuhong </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Wen </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Jiyu </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Candong </given-names>  </name>  <xref ref-type="aff" rid="aff005">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wen </surname>  <given-names> Zixiang </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yinghui </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guan </surname>  <given-names> Rongxia </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Ruzhen </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Dechun </given-names>  </name>  <xref ref-type="aff" rid="aff006">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Shuming </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Qiu </surname>  <given-names> Li-Juan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean was grown at a wide range of latitudes, from at least 50&#x000b0;N to 35&#x000b0;S [ <xref rid="pone.0158602.ref004" ref-type="bibr"> 4 </xref> ], however, each soybean cultivar generally has a specific geographic or latitudinal distribution to reach its high <span class="trait">yield.</span> Thirteen soybean maturity groups (MG 000, MG 00, MG 0, and MG I&#x02014;X) have been well developed to estimate the range of adaptability to latitudinal or geographic zones of soybean cultivars in the USA and Canada [ <xref rid="pone.0158602.ref005" ref-type="bibr"> 5 </xref> &#x02013; <xref rid="pone.0158602.ref006" ref-type="bibr"> 6 </xref> ]. The growth period of soybean can be divided into vegetative growth period, from seedling to flowering, and reproductive growth period, from flowering to maturity. Hence, flowering is the transition from the two stages, which is a critical event in the life cycle of plants. Since the genes controlling soybean maturity were first reported by Woodworth (1923) [ <xref rid="pone.0158602.ref007" ref-type="bibr"> 7 </xref> ], a series of loci ( <italic> <span class="gene">E1</span> </italic> to <italic> E9 </italic> , and J) that control flowering time and maturity of soybean were identified, and characterized at the phenotypic and genetic levels by classic methods [ <xref rid="pone.0158602.ref004" ref-type="bibr"> 4 </xref> , <xref rid="pone.0158602.ref008" ref-type="bibr"> 8 </xref> &#x02013; <xref rid="pone.0158602.ref015" ref-type="bibr"> 15 </xref> ]. To date, the molecular mechanism for <italic> <span class="gene">E1</span> </italic> to <italic> E4 </italic> loci has been uncovered [ <xref rid="pone.0158602.ref016" ref-type="bibr"> 16 </xref> &#x02013; <xref rid="pone.0158602.ref019" ref-type="bibr"> 19 </xref> ], while other loci such as <italic> E5 </italic> to <italic> E8 </italic> remain unknown. It was reported that most of the <italic> E </italic> series genes had pleiotropic effects on vegetative growth period, reproductive growth period, and growth period. For example, the dominant alleles of <italic> E2 </italic> to <italic> E4 </italic> prolonged the reproductive period from stages R1 (days to flowering of the first flower) to R8 (full maturity) to some extent [ <xref rid="pone.0158602.ref020" ref-type="bibr"> 20 </xref> &#x02013; <xref rid="pone.0158602.ref021" ref-type="bibr"> 21 </xref> ]. Relationship between maturity groups and genotypes of the <italic> E </italic> loci was inferred by Harosoy or Clark near isogeneic lines (NILs) [ <xref rid="pone.0158602.ref022" ref-type="bibr"> 22 </xref> ]. Besides the cloned maturity genes, two homologs of <italic> Flowering Locus T </italic> ( <italic> FT </italic> ), <italic> <span class="gene">GmFT2a</span> </italic> , and <italic> <span class="gene">GmFT5a</span> </italic> , were found to encode the components of &#x02018;florigen&#x02019;, which is the mobile flowering promotion signal involved in the transition to flowering, and to coordinately control flowering in soybean [ <xref rid="pone.0158602.ref023" ref-type="bibr"> 23 </xref> ]. Recently, the <italic> <span class="gene">GmFT2a</span> </italic> was validated to be gene <italic> E9 </italic> [ <xref rid="pone.0158602.ref024" ref-type="bibr"> 24 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Five QTL, three on chromosome 6 and one on each of chromosomes 11 and 15, had pleotropic effects on more than one trait. Four of them controlled ETM and FTM, and the other one controlled ETM and ETF. None of the QTL had pleotropic effects on ETF and FTM. These results were concordant with the high phenotypic correlations between ETM and FTM, and between ETM and ETF, but low correlations between ETF and FTM ( <xref ref-type="table" rid="pone.0158602.t002"> Table 2 </xref> ). ss245950346 on chromosome 6 was consistently associated with ETM and FTM under two planting densities, explaining an average of 13.16% of the phenotypic variance. It was worth noting that ss245950346 was also in the same region with well characterized gene <italic> <span class="gene">E1</span> </italic> ( <xref ref-type="table" rid="pone.0158602.t004"> Table 4 </xref> ). ss245937498 on chromosome 6 was identified for traits of ETM and FTM, and also reported by other literatures controlling series of traits in interest. Three pleotropic QTL, ss245977002 on chromosome 6, ss247571761 on chromosome 11, and ss248571908 on chromosome 15, had not been reported yet, but were adjacent to candidate genes with functions such as haloacid dehalogenase-like hydrolase superfamily protein, disease <span class="trait">resistance</span> family protein, etc.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Comparative Transcriptome Analysis of Resistant and Susceptible Common Bean Genotypes in Response to Soybean Cyst Nematode Infection</h2>
			<h3>PMID:27441552</h3>
				
					<h3>PMCID:4956322</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jain </surname>  <given-names> Shalu </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chittem </surname>  <given-names> Kishore </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Brueggeman </surname>  <given-names> Robert </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Osorno </surname>  <given-names> Juan M. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Richards </surname>  <given-names> Jonathan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nelson </surname>  <given-names> Berlin D. </given-names>  <suffix> Jr. </suffix>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 30 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean cyst nematode ( <italic> Heterodera glycines </italic> Ichinohe; SCN) is an obligate sedentary endoparasite and the most destructive pathogen of soybean ( <italic> Glycine max </italic> L.) in the world causing losses of approximately $1 billion annually in the USA alone [ <xref rid="pone.0159338.ref003" ref-type="bibr"> 3 </xref> ]. SCN does not cause unique aboveground symptoms and can be confused with symptoms of nutrient deficiency, <span class="trait">drought</span> stress, herbicide injury or soil-borne fungal disease. SCN can spread readily from field to field in soil particles on farm machinery. SCN has been found in every soybean producing state in the United States except New York and West Virginia [ <xref rid="pone.0159338.ref004" ref-type="bibr"> 4 </xref> ]. In North Dakota and northern Minnesota, SCN was first found in soybean fields in 2003. There are differences in virulence within the SCN population with virulent types termed HG types. In North Dakota and northern Minnesota the predominant virulent type is HG 0, previously known as race 3. The primary management of SCN in soybean is the use of resistant cultivars and crop rotation with non-host crops to reduce nematode populations in the soil [ <xref rid="pone.0159338.ref005" ref-type="bibr"> 5 </xref> ]. The soybean genotypes Peking and PI88788 are the major sources of SCN <span class="trait">resistance</span> in commercial soybean varieties. However, due to shifts in virulent HG types, the major source of SCN <span class="trait">resistance</span> from PI88788 used in most commercial soybean cultivars is now less effective in some US production areas since new HG types such as HG 2.5.7 can reproduce on that <span class="trait">resistance</span> source [ <xref rid="pone.0159338.ref006" ref-type="bibr"> 6 </xref> ]. Thus, it is imperative to identify novel <span class="trait">resistance</span> genes from different sources to incorporate more effective genes into cultivars for <span class="trait">resistance</span> to predominant SCN HG types. <span class="trait">Resistance</span> to SCN is quantitative and major QTLs have been mapped on chromosomes 18 ( <italic> rhg1 </italic> ) and 8 ( <italic> <span class="gene">Rhg4</span> </italic> ) in many soybean cultivars [ <xref rid="pone.0159338.ref007" ref-type="bibr"> 7 </xref> ]. The SCN <span class="trait">resistance</span> of <italic> rhg1 </italic> is contributed by multi-copy <underline> <italic> rhg1 </italic> </underline> haplotypes forming two distinct resistant groups i.e. high copy number <italic> rhg1 </italic> accessions and low copy number <italic> rhg1 </italic> accessions [ <xref rid="pone.0159338.ref008" ref-type="bibr"> 8 </xref> ]. High copy number <italic> rhg1 </italic> group (PI 88788-type) contain 7&#x02013;10 copies of a 31 kb <italic> rhg1 </italic> repeat segment while the low copy number <italic> rhg1 </italic> group (Peking-type) contains 3 copies of the <italic> rhg1 </italic> repeat along with a newly identified allele of <span class="gene">Glyma18g02590.</span> Three different genes encoding a predicted amino acid transporter <span class="gene">(Glyma18g02580),</span> an &#x003b1;&#x02013;SNAP protein <span class="gene">(Glyma18g02590)</span> and a protein with wound-inducible protein 12 (WI12) <span class="gene">(Glyma18g02610)</span> are present in 31 kb <italic> rhg1 </italic> repeat segment and contribute to SCN <span class="trait">resistance</span> [ <xref rid="pone.0159338.ref008" ref-type="bibr"> 8 </xref> ]. The SCN <span class="trait">resistance</span> of <italic> <span class="gene">Rhg4</span> </italic> is controlled by a serine hydroxymethyl transferase gene [ <xref rid="pone.0159338.ref009" ref-type="bibr"> 9 </xref> ]. <italic> <span class="gene">Rhg4</span> </italic> exhibits dominant gene action for <span class="trait">resistance</span> to race 3 of SCN [ <xref rid="pone.0159338.ref010" ref-type="bibr"> 10 </xref> ] <ext-link ext-link-type="uri" xlink:href="http://www.nature.com/nature/journal/v492/n7428/full/nature11651.html-ref5"> http://www.nature.com/nature/journal/v492/n7428/full/nature11651.html-ref5 </ext-link> and is one of the major QTLs controlling soybean <span class="trait">resistance</span> to SCN race 3 (HG type 0) in cv. Forrest [ <xref rid="pone.0159338.ref011" ref-type="bibr"> 11 </xref> ]. Roots of plants carrying <italic> Rhg </italic> genes are penetrated by infective juveniles, but feeding cells ultimately degenerate, causing the nematodes to die before reaching adult stages.
					</p>
				
					<p>&nbsp;&nbsp;
						There are substantial reports regarding the role of kinases in stress response in common bean [ <xref rid="pone.0159338.ref044" ref-type="bibr"> 44 </xref> , <xref rid="pone.0159338.ref045" ref-type="bibr"> 45 </xref> ]. Kinase signaling appears to have an important role in the resistance/susceptibility reaction of common bean genotypes in response to pathogens [ <xref rid="pone.0159338.ref046" ref-type="bibr"> 46 </xref> , <xref rid="pone.0159338.ref047" ref-type="bibr"> 47 </xref> ]. In soybean, up-regulation of the MAP3K homolog implicated the role of mitogen-activated protein kinase signaling in the regulation of <span class="trait">resistance</span> to SCN [ <xref rid="pone.0159338.ref039" ref-type="bibr"> 39 </xref> ]. A heat map of differentially expressed kinase genes is shown in <xref ref-type="fig" rid="pone.0159338.g004"> Fig 4 </xref> . Different classes of protein kinases were upregulated and downregulated in the resistant genotype. One of the upregulated DEGs (Phvul.009G180700) was annotated as a leucine-rich repeat protein kinase family protein. On the other hand, 27 protein kinases were induced and 18 protein kinases were down-regulated in the susceptible genotype. One of the transmembrane protein kinase tmk1-like genes (Phvul.003G015800) was upregulated in GTC-900 and has 51% similarity with the <italic> <span class="gene">Rhg4</span> </italic> like gene (GeneBank ID: NP_001236710) in soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Identification and Comparative Analysis of Differential Gene Expression in Soybean Leaf Tissue under <span class="trait">Drought</span> and Flooding Stress Revealed by RNA-Seq</h2>
			<h3>PMID:27486466</h3>
				
					<h3>PMCID:4950259</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Wei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/351363/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yao </surname>  <given-names> Qiuming </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/56071/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Patil </surname>  <given-names> Gunvant B. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/104588/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Agarwal </surname>  <given-names> Gaurav </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/350689/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deshmukh </surname>  <given-names> Rupesh K. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103657/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lin </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/360252/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Biao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/360453/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Yongqin </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/65688/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Prince </surname>  <given-names> Silvas J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/142509/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/103322/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Dong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/27080/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> An </surname>  <given-names> Yongqiang C. </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Valliyodan </surname>  <given-names> Babu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/249960/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Varshney </surname>  <given-names> Rajeev K. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/25772/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nguyen </surname>  <given-names> Henry T. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/190240/overview"/>  </contrib>
				<br/> 
				年份：<day> 04 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The mechanism that overcomes <span class="trait">drought</span> stress in plants is facilitated by <span class="trait">drought</span> avoidance, <span class="trait">drought</span> tolerance, <span class="trait">drought</span> escape, and <span class="trait">drought</span> recovery (Cruz De Carvalho, <xref rid="B14" ref-type="bibr"> 2008 </xref> ). <span class="trait">Drought</span> causes a negative impact on all developmental stages, starting from germination to seed maturation (Valliyodan and Nguyen, <xref rid="B69" ref-type="bibr"> 2006 </xref> ). Flooding also leads to very sensitive responses that affect both soybean growth and <span class="trait">yield</span> (Komatsu et al., <xref rid="B34" ref-type="bibr"> 2009 </xref> ). Stress-specific adaptive mechanisms and various molecular, biochemical, and physiological responses in plants help maintain normal growth and survival under <span class="trait">drought</span> and flooding conditions (Dat et al., <xref rid="B15" ref-type="bibr"> 2004 </xref> ). The perception of the environmental signal and subsequent molecular signaling is the first step in a stress response. In this direction, several signal compounds including abscisic acid (ABA), have been identified and studied for <span class="trait">drought</span> stress. The ABA level is up-regulated under <span class="trait">drought</span> conditions and the downstream responsive network is induced (Xiong and Zhu, <xref rid="B77" ref-type="bibr"> 2002 </xref> ). The initiation of the ABA signaling process involves PYR/PYL/RCAR (Pyrabactin <span class="trait">Resistance</span> /PYR1-Like/Regulatory Components of ABA Receptors) ABA receptors, and two enzymes, protein phosphatase 2c family (PP2Cs) and SNF-related kinase (SnRK2), with opposite functions of phosphatase/kinase (Park et al., <xref rid="B49" ref-type="bibr"> 2009 </xref> ; Kim, <xref rid="B30" ref-type="bibr"> 2014 </xref> ). The binding of ABA to PP2Cs can inhibit PP2C activity, which causes it to lose its ability to inhibit SnRK2s. Then, activated SnRK2s can phosphorylate and activate the downstream ABFs (ABA responsive element binding factor), which can bind to their own targets (or other genes) and activate the downstream response pathways (Park et al., <xref rid="B49" ref-type="bibr"> 2009 </xref> ). This causes plant to express a <span class="trait">drought</span> tolerance phenotype. The ABA level is also increased in flooding treatments in tomato (Else et al., <xref rid="B18" ref-type="bibr"> 1995 </xref> ). However, the ABA level is reduced under flooding treatment in rice seedlings during submergence (Saika et al., <xref rid="B57" ref-type="bibr"> 2007 </xref> ). Among different pathways of stress response signaling, most of the ABA dependent and independent responses involve TFs. These TFs belong to very diverse families representing ~10% of the genes in soybean, which are involved in most of the biotic and abiotic stress responses ( <ext-link ext-link-type="uri" xlink:href="http://planttfdb_v1.cbi.pku.edu.cn:9010/web/index.php?sp=gm"> http://planttfdb_v1.cbi.pku.edu.cn:9010/web/index.php?sp=gm </ext-link> ). In rice, SUBMERGENCE1 ( <italic> <span class="gene">SUB1</span> </italic> ) has been identified as an important TF involved in submergence tolerance (Xu et al., <xref rid="B78" ref-type="bibr"> 2006 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The complex molecular mechanisms, signaling perception, integrated responses, and molecular cross talk activated in response to different abiotic stress are not well-understood in soybean. In this regard, several efforts have been made to elucidate the molecular mechanisms. Genome-wide transcriptome profiling will be helpful in understanding a global view of gene expression under <span class="trait">drought.</span> Recently, drought-responsive candidate genes ( <italic> <span class="gene">GmNAC</span> </italic> ) have been identified using transcriptome profiling of soybean under <span class="trait">drought</span> conditions performed with a 66K Affymetrix microarray platform (Le et al., <xref rid="B39" ref-type="bibr"> 2012 </xref> ). It has been reported that transcription profiles of the cold- and dehydration-responsive genes were similar among Arabidopsis, rice, and soybean, which shows representative up-regulated late embryogenesis abundant (dehydrin/LEA) and down-regulated (photosynthesis-related) genes. This suggests that different species have conserved different stress responses (Maruyama et al., <xref rid="B42" ref-type="bibr"> 2012 </xref> ). Similarly, several studies using microarray platforms have been conducted and expression data for over 5000 soybean samples have been deposited in a public database ( <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/geo/"> http://www.ncbi.nlm.nih.gov/geo/ </ext-link> ). However, the microarray have some drawbacks such as cross-hybridization, non-specific hybridization, and a limited sensitivity. Most importantly, microarray platforms provides the information on only those genes that are available on the microarray, which is a problem for soybean whose gene models are not well-characterized. These limitations can be overcome using recently advanced sequencing-based techniques, such as RNA sequencing (RNA-Seq; Varshney et al., <xref rid="B70" ref-type="bibr"> 2009 </xref> ; Trapnell et al., <xref rid="B65" ref-type="bibr"> 2013 </xref> ). In soybean, RNA-Seq has been deployed to generate an expression atlas for soybean genes using several tissues in seed development stages (Severin et al., <xref rid="B58" ref-type="bibr"> 2010 </xref> ). In another study, variability in commercial and developing cultivars under <span class="trait">drought</span> conditions was identified using single nucleotide polymorphisms (SNPs) in RNA-Seq data (Vidal et al., <xref rid="B72" ref-type="bibr"> 2012 </xref> ). In this study, genome-wide transcriptome profiling of soybean leaf tissue using the RNA-Seq approach was performed to understand the response to <span class="trait">drought</span> and flooding stress. The comparative analysis has enabled us to understand molecular responses against the extreme water availability conditions.
					</p>
				
					<p>&nbsp;&nbsp;
						The photosynthesis related genes were down-regulated under both <span class="trait">drought</span> and flooding conditions. The expression level of three PS-II light harvesting complex (LHC) related genes, <italic> <span class="gene">Glyma16g28030.1,</span> <span class="gene">Glyma16g28070.1</span> </italic> , and <italic> <span class="gene">Glyma10g04630.1</span> </italic> , were repressed significantly under <span class="trait">drought</span> conditions. Similarly, three photosynthesis light reaction related genes, <italic> <span class="gene">Glyma07g21150.1,</span> <span class="gene">Glyma20g24500.1</span> </italic> , and <italic> <span class="gene">Glyma20g24500.1</span> </italic> , and one Calvin cycle related gene, <italic> <span class="gene">Glyma01g34300.1</span> </italic> , were also down-regulated under <span class="trait">drought</span> conditions. Only one gene from the photosynthetic system encoding an electron carrier was induced under the <span class="trait">drought</span> conditions (Supplementary Figure <xref ref-type="supplementary-material" rid="SM6"> 6A </xref> ). However, compared to the <span class="trait">drought,</span> more photosynthesis related genes were found to be down-regulated under flooding conditions (Supplementary Figure <xref ref-type="supplementary-material" rid="SM6"> 6B </xref> ). The relative expression level of 34 genes encoding PS-II LHC and 45 genes encoding PS-II subunits were repressed (Supplementary Figure <xref ref-type="supplementary-material" rid="SM6"> 6B </xref> ). In addition, 41 genes related to the Calvin cycle and eight electron carrier related genes were also repressed. In total, seven genes were down- and one was up-regulated under <span class="trait">drought</span> stress. However, 128 genes were down-regulated under flood treatment (Supplementary Figure <xref ref-type="supplementary-material" rid="SM6"> 6B </xref> ). The dynamic expression of photosynthesis related genes can be correlated with the chlorophyll content under <span class="trait">drought</span> and flood conditions, which concurs with the inhibition of photosynthesis (Supplementary Figures <xref ref-type="supplementary-material" rid="SM3"> 3A,B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Changes in the expression levels of a large number of sucrose and starch metabolism pathway genes were observed under the <span class="trait">drought</span> and flood conditions. Genes encoding sucrose and starch biosynthesis genes, namely, amylose synthase, ADP-Glucose synthase, and starch and maltose synthase, showed a significant decrease in the transcript level in <span class="trait">drought</span> and flooding treatments (Supplementary Figure <xref ref-type="supplementary-material" rid="SM4"> 4 </xref> ). No change in expression was observed for sucrose phosphate synthase genes ( <italic> <span class="gene">Glyma08g42140.1</span> </italic> and <italic> <span class="gene">Glyma06g48200.1</span> </italic> ) in the <span class="trait">drought</span> condition, in contrast to the significant repression in expression of these genes under the flood conditions. However, the genes encoding beta-fructofuranosidase ( <italic> <span class="gene">Glyma13g42530.1</span> </italic> ), fructokinase ( <italic> <span class="gene">Glyma16g32530.1</span> </italic> ), hexokinase ( <italic> <span class="gene">Glyma11g10130.1</span> </italic> ), and three maltose synthase genes ( <italic> <span class="gene">Glyma13g28630.1,</span> <span class="gene">Glyma11g04210.1</span> </italic> , and <italic> <span class="gene">Glyma15g10480.1</span> </italic> ) were up-regulated under the <span class="trait">drought</span> conditions. Overall, transcript abundance for starch biosynthesis encoding genes was reduced more in the flooding stress compared to the <span class="trait">drought</span> stress. This could result in the decrease of starch content under flooding conditions (Supplementary Figure <xref ref-type="supplementary-material" rid="SM4"> 4B </xref> ). Only one maltose synthesis gene, <italic> <span class="gene">Glyma11g04210.1</span> </italic> , was induced during flooding (Supplementary Figures <xref ref-type="supplementary-material" rid="SM4"> 4 </xref> , <xref ref-type="supplementary-material" rid="SM6"> 6 </xref> ). We also identified six genes of the soybean sugar-effluxer (SWEET) family that was differentially expressed under the two extreme water regimes. Three genes, <italic> <span class="gene">Glyma12g36300,</span> <span class="gene">Glyma14g27610</span> </italic> , and <italic> <span class="gene">Glyma06g13110</span> </italic> , were up-regulated under the <span class="trait">drought</span> conditions, and the other three genes, <italic> <span class="gene">Glyma06g17520,</span> <span class="gene">Glyma09g04840</span> </italic> , and <italic> <span class="gene">Glyma13g33950</span> </italic> , were down-regulated under flooding conditions. Interestingly, one SWEET gene, <italic> <span class="gene">Glyma14g27610</span> </italic> , was up-regulated in both <span class="trait">drought</span> and flooding conditions (Supplementary Tables <xref ref-type="supplementary-material" rid="SM8"> 1A,B </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The involvement of TFs was examined in the <span class="trait">drought</span> and flood stress responses through both the ABA dependent and independent pathways. The ABA and other upstream signals can trigger the downstream regulation pathway through the TF regulatory network in both <span class="trait">drought</span> and flood conditions. In this research, ABA related TFs: AP2, bZIP, MYB, and NAC were up-regulated under both <span class="trait">drought</span> and flood conditions (Figure <xref ref-type="fig" rid="F4"> 4 </xref> ). These TF families have been extensively studied in many plants under several different stress conditions (Dubos et al., <xref rid="B17" ref-type="bibr"> 2010 </xref> ; Hao et al., <xref rid="B23" ref-type="bibr"> 2011 </xref> ; Mizoi et al., <xref rid="B43" ref-type="bibr"> 2012 </xref> ; Nakashima et al., <xref rid="B45" ref-type="bibr"> 2012 </xref> ,). In several abiotic stress regulation studies, the TFs identified are initially involved in tolerance against a specific stress and later observed to be important for other stresses. For instance, the <italic> SUB1A </italic> TF was initially identified as a key regulator for the flood responsive pathways in rice, and later transcriptional and functional studies confirmed its important role in <span class="trait">drought</span> tolerance (Xu et al., <xref rid="B78" ref-type="bibr"> 2006 </xref> ; Fukao et al., <xref rid="B21" ref-type="bibr"> 2011 </xref> ). However, the flood response in soybean seems to be independent of the <italic> <span class="gene">SUB1</span> </italic> pathway since no <italic> SUB1A </italic> homolog was found to be differentially expressed in our study. On the other hand, recent studies have shown the involvement of <italic> <span class="gene">SUB1</span> </italic> and core-clock related genes mediated through alternatively spliced forms (Syed et al., <xref rid="B62" ref-type="bibr"> 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The 1-Deoxy-D-xylulose-5-phosphate reductoisomerase (DXR) is an important enzyme involved in the 2-C-methyl-D-erythritol-4-phosphate (MEP) pathway for isoprenoid biosynthesis and was identified among the highly expressed <span class="trait">drought</span> responsive genes (Table <xref ref-type="table" rid="T2"> 2 </xref> ). Xing et al. ( <xref rid="B76" ref-type="bibr"> 2010 </xref> ) reported that <italic> <span class="gene">dxr</span> </italic> mutants with a dwarf phenotype significantly reduced the number of trichomes and abnormal stomatal closure, which was rescued with exogenous application of ABA and gibberellic acids (GAs). Thus, the high expression of DXR under <span class="trait">drought</span> stress implies a plant's response to withstand stress and compensate for compromised photosynthetic capacity and abnormal stomatal conductance. Auxin, ethylene, and ABA were previously identified to be involved in the <span class="trait">drought</span> responsive pathway in soybean (Le et al., <xref rid="B39" ref-type="bibr"> 2012 </xref> ). Additionally, ethylene, ABA, and GA are known to be involved in a regulatory network under flooding stress (Bailey-Serres et al., <xref rid="B5" ref-type="bibr"> 2012 </xref> ). Hormonal cross-talk at different levels of the stress response and the resultant synergetic or antagonist interactions have been found to be important for stress tolerance (Chinnusamy et al., <xref rid="B11" ref-type="bibr"> 2004 </xref> ; Peleg and Blumwald, <xref rid="B53" ref-type="bibr"> 2011 </xref> ). The biosynthesis of jasmonic acid (JA) through the octadecanoic pathway (Vick and Zimmerman, <xref rid="B71" ref-type="bibr"> 1984 </xref> ) with &#x003b1;-linolenic acid (LA) as a precursor, is mediated by 12-oxophytodienoic acid reductase 2 (OPDAR2) ( <italic> <span class="gene">Glyma01g44600</span> </italic> ), which is found to be up-regulated under both <span class="trait">drought</span> and flooding stress (Supplementary Tables <xref ref-type="supplementary-material" rid="SM8"> 1B </xref> , <xref ref-type="supplementary-material" rid="SM9"> 2A </xref> ) but highly expressed under flooding stress (Table <xref ref-type="table" rid="T3"> 3 </xref> ). The high expression of OPDRA2 is correlated with higher JA synthesis and other systemic responses. The enzyme has also been shown to detoxify 2,4,6-trinitro toluene (TNT) in polluted water contaminated by explosives as a result of manufacturing and munitions (Beynon et al., <xref rid="B6" ref-type="bibr"> 2009 </xref> ). The upsurge of OPDAR2 under flooding stress suggests plants' adaptive response against stress and this could also help in detoxification of contaminants such as TNT if any are present in the water. DEGs involved in different hormonal pathways identified here under <span class="trait">drought</span> and flooding stress could be good candidates to manipulate hormonal cross-talk for the effective regulation of stress tolerance in plants.
					</p>
				
					<p>&nbsp;&nbsp;
						Reduced photosynthesis during prolonged abiotic stresses such as <span class="trait">drought</span> and flooding can be correlated with the production of ROS (Klok et al., <xref rid="B31" ref-type="bibr"> 2002 </xref> ). High levels of ROS can damage the cell through peroxidation of the lipids, oxidation of proteins and other pathways, and finally the death of the cell (Clement et al., <xref rid="B13" ref-type="bibr"> 2008 </xref> ; Wrzaczek et al., <xref rid="B75" ref-type="bibr"> 2011 </xref> ). Glutathione S-transferase, one of the important enzyme in the regulation of ROS flux, is featured as a highly enriched gene among the top 50 highly expressed genes under flooding stress in this study, which indicates another adaptive response under the stress conditions (Table <xref ref-type="table" rid="T3"> 3 </xref> ). However, the opposite (high <span class="gene">GST</span> expression in response to <span class="trait">drought</span> compared to flooding stress) was reported by Oh and Komatsu ( <xref rid="B48" ref-type="bibr"> 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Most research on the effect of <span class="trait">drought</span> and flooding on sugar metabolism and starch biosynthesis has demonstrated that the carbohydrate levels are altered in leaves (Lemoine et al., <xref rid="B41" ref-type="bibr"> 2013 </xref> ). Under stress conditions, the increase in hexose amounts is associated with the induction of starch hydrolysis (Pelleschi et al., <xref rid="B54" ref-type="bibr"> 1997 </xref> ). In agreement with these findings, we observed an increased expression of the hexokinase ( <italic> <span class="gene">Glyma11g10130</span> </italic> . <italic> 1 </italic> ) gene in the <span class="trait">drought</span> treatment. The gradual slowing down of starch metabolism is a general mechanism under flooding (Fukao et al., <xref rid="B21" ref-type="bibr"> 2011 </xref> ) and <span class="trait">drought</span> (Burke, <xref rid="B9" ref-type="bibr"> 2007 </xref> ) conditions for rice and cotton, respectively, which reserves the energy source for a prolonged energy supply to maintain cell survival. It is conceivable that starch metabolism during flooding and <span class="trait">drought</span> response may also be controlled/gated by the up- and down-regulation of carbohydrate metabolism genes in soybean. One of the prominent processes under sugar metabolism that gets affected in water stressed plants is the sugar efflux rate, which is carried out by proteins coded by sugar transporter genes. They ensure sugar transport from the source (leaves) to the sink (roots and seeds) to counteract the depleted sugar levels under severe conditions such as <span class="trait">drought</span> and flooding. Utilizing the available resource generated by Patil et al. ( <xref rid="B52" ref-type="bibr"> 2015 </xref> ), six SWEET family genes were featured among the DEGs identified in this study under <span class="trait">drought</span> and flood conditions. The findings suggest that SWEET genes in general are induced in <span class="trait">drought</span> conditions and repressed in flooding conditions, which indicate that sucrose efflux genes are activated under <span class="trait">drought</span> conditions and repressed under flooding conditions. However, one gene ( <italic> <span class="gene">Glyma14g276210</span> </italic> ) was found to be up-regulated under both of the conditions. Syed et al. ( <xref rid="B62" ref-type="bibr"> 2015 </xref> ) also identified the up-regulation of soybean SWEET genes under <span class="trait">drought</span> stress. The activated gene expression under <span class="trait">drought</span> stress could be accounted for by sugar transportation due to an uninterrupted energy supply under such adversity. The down-regulation of three SWEET family genes under flooding stress in the current study is contrary to the findings of Lemoine et al. ( <xref rid="B41" ref-type="bibr"> 2013 </xref> ), where it was suggested that the up-regulated sugar effluxer genes may contribute to sugar homeostasis in a flood tolerant genotype. In this study, the repressed genes might be playing the same role by controlling the efflux of sugar under flooding stress and thus putting a check on normal efflux of sugar. However, this hypothesis needs additional substantiation.
					</p>
				
					<p>&nbsp;&nbsp;
						Glycolytic enzymes along with inducers of heat shock proteins are key factors in the early response to flooding and <span class="trait">drought</span> in soybean (Nanjo et al., <xref rid="B47" ref-type="bibr"> 2011 </xref> ; Oh and Komatsu, <xref rid="B48" ref-type="bibr"> 2015 </xref> ), which suggests the significance of the glycolytic pathway in the adaptation to flooding and <span class="trait">drought</span> conditions. Oh and Komatsu ( <xref rid="B48" ref-type="bibr"> 2015 </xref> ) reported increased expression of all glycolysis related proteins under both <span class="trait">drought</span> and flooding responses. The up-regulated expression of genes coding for the glycolytic pathway enzymes, fructokinase ( <italic> <span class="gene">Glyma16g32530.1</span> </italic> ) and hexokinase ( <italic> <span class="gene">Glyma11g10130.1</span> </italic> ), under <span class="trait">drought</span> stress can be considered an adaptive response of soybean under <span class="trait">drought</span> stress. On the contrary, the transcript levels of glycolytic pathway enzymes were not significantly affected under water deficit conditions in chickpea (Khanna et al., <xref rid="B28" ref-type="bibr"> 2014 </xref> ). This suggests a legume specific adaptive variability under similar stress.
					</p>
				
					<p>&nbsp;&nbsp;
						Plant responses toward abiotic stress are complex and involve several different mechanisms regulated by crosstalk between genes related to hormonal signaling, photosynthesis, respiration, and transcriptional regulations. Comparative transcriptome profiling performed in this study under <span class="trait">drought</span> and flooding has provided an opportunity to evaluate the categorized molecular responses. Besides having contrasting stress conditions, many common molecular mechanisms were observed to be involved under <span class="trait">drought</span> and flooding. A notable example is the ABA related TFs that are up-regulated under both <span class="trait">drought</span> and flood conditions. ABA is considered as the stress related hormone and seems to be a key regulator under both conditions. Unlike rice and Arabidopsis, the ABA mediated stress response in soybean under flooding stress is independent of the <span class="gene">SUB1</span> pathway. Another important aspect highlighted here is the reduced overall metabolic activity under flooding stress, which supports the quiescence rather than the escape mechanism.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-wide association mapping of partial <span class="trait">resistance</span> to <italic> Phytophthora sojae </italic> in soybean plant introductions from the Republic of Korea</h2>
			<h3>PMID:27515508</h3>
				
					<h3>PMCID:4982113</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Schneider </surname>  <given-names> Rhiannon </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rolling </surname>  <given-names> William </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Qijian </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cregan </surname>  <given-names> Perry </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Dorrance </surname>  <given-names> Anne E. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> McHale </surname>  <given-names> Leah K. </given-names>  </name>  <address>  <email> mchale.21@osu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Department of Horticulture and Crop Science, The Ohio State University, Columbus, OH 43210 USA  </aff>  <aff id="Aff2">  <label> 2 </label> Present Address: Pioneer Hi-Bred International Inc., Napoleon, OH 43545 USA  </aff>  <aff id="Aff3">  <label> 3 </label> Center for Applied Plant Sciences, The Ohio State University, Columbus, OH 43210 USA  </aff>  <aff id="Aff4">  <label> 4 </label> US Department of Agriculture, Soybean Genomics and Improvement Laboratory, Agricultural Research Service, Beltsville, MD 20705 USA  </aff>  <aff id="Aff5">  <label> 5 </label> Department of Plant Pathology, The Ohio State University, Wooster, OH 44691 USA  </aff>
				<br/> 
				年份：<day> 7 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The second QTL (3-2) is 238.1&#x000a0;kb downstream of QTL 3-1 and is coincident with the mapped positions of <italic> Rps1d </italic> and <italic> RpsUn1 </italic> [ <xref ref-type="bibr" rid="CR10"> 10 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ] <italic> . </italic> This region does not coincide with any QTL for partial <span class="trait">resistance</span> to <italic> P. sojae </italic> (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3a </xref> ). Thus, QTL 3-2 may represent a novel QTL for partial <span class="trait">resistance</span> to <italic> P. sojae </italic> . QTL 3-2 encompasses 304&#x000a0;kb and 13 predicted genes (Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> : Table S3), seven of which have putative functions relating to disease <span class="trait">resistance</span> or defense. Four of these genes <span class="gene">(Glyma.03G034400,</span> <span class="gene">Glyma.03G034500,</span> <span class="gene">Glyma.03G034800</span> and <span class="gene">Glyma.03G034900)</span> putatively encode a nucleotide binding (NB) domain characteristic of <italic> R </italic> -genes [ <xref ref-type="bibr" rid="CR64"> 64 </xref> ]. In addition, there are two leucine-rich repeat receptor-like kinase (LRR-RLK)encoding genes in this region <span class="gene">(Glyma.03G034200</span> and <span class="gene">Glyma.03G035300),</span> a class of genes involved in basal defense and plant developmental responses [ <xref ref-type="bibr" rid="CR65"> 65 </xref> &#x02013; <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. Finally, <span class="gene">Glyma.03G034100</span> is homologous to a Sec61 protein transporter-encoding gene, which is involved in activation of systemic-acquired <span class="trait">resistance</span> in Arabidopsis [ <xref ref-type="bibr" rid="CR69"> 69 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Located on Chr 13, ss71565031 was significantly associated with IRW (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3b </xref> ). Similar to QTL 3-3, ss71565031 was not located in a haplotype block. Therefore, QTL 13-1 region was defined by the two flanking markers, ss715615924 and ss715615033 and spans 49.2&#x000a0;kb. QTL 13-1 is coincident with <italic> Rps8 </italic> and <italic> Rps3 </italic> [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> ] <italic> , </italic> three previously identified QTL for partial <span class="trait">resistance</span> against <italic> P. sojae </italic> ( <italic> Phytoph 11-21 </italic> , <italic> 9 </italic> - <italic> 3 </italic> , <italic> 12 </italic> - <italic> 1 </italic> [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ]) [ <xref ref-type="bibr" rid="CR43"> 43 </xref> , <xref ref-type="bibr" rid="CR48"> 48 </xref> , <xref ref-type="bibr" rid="CR52"> 52 </xref> ], a QTL for flood tolerance [ <xref ref-type="bibr" rid="CR48"> 48 </xref> ], and QTL for <span class="trait">resistance</span> against <italic> S. sclerotiorum </italic> [ <xref ref-type="bibr" rid="CR62"> 62 </xref> ] <italic> . </italic> The region contains a total of five predicted genes (Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> : Table S3), of which two <span class="gene">(Glyma.13G194100</span> and <span class="gene">Glyma.13G194500)</span> are NB-LRR encoding genes.
					</p>
				
					<p>&nbsp;&nbsp;
						QTL 19-1 spans 149&#x000a0;kb and contains a total of 21 predicted genes (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ; Additional file <xref rid="MOESM2" ref-type="media"> 2 </xref> : Table S3), of which seven are putative defense related genes. The defense genes in the region include <span class="gene">Glyma.19G245400,</span> <span class="gene">Glyma.19G245500</span> and <span class="gene">Glyma.19G245600,</span> which encode putative PR4-related chitin-binding proteins [ <xref ref-type="bibr" rid="CR74"> 74 </xref> ]. <span class="gene">Glyma.19G243800</span> encodes a putative ribose 5-phosphate isomerase. Homologs in <italic> Arabidopsis </italic> have been shown to function in cellulose synthase [ <xref ref-type="bibr" rid="CR75"> 75 </xref> ], which has been shown to be involved in the regulation of jasmonic acid and ethylene [ <xref ref-type="bibr" rid="CR76"> 76 </xref> ] with mutants displaying enhanced <span class="trait">resistance</span> to bacterial and fungal pathogens [ <xref ref-type="bibr" rid="CR77"> 77 </xref> ]. Other genes in the region include <span class="gene">Glyma.19G244000</span> that encodes a putative <span class="gene">MATE</span> efflux protein potentially involved in defense signaling pathways or in transporting toxic compounds from infected cells [ <xref ref-type="bibr" rid="CR78"> 78 </xref> , <xref ref-type="bibr" rid="CR79"> 79 </xref> ] as well as <span class="gene">Glyma.19G245200</span> that encodes a putative auxin responsive gene [ <xref ref-type="bibr" rid="CR70"> 70 </xref> ]. Finally, <span class="gene">Glyma.19G244400</span> encodes a putative ammonium transporter, homologs of which have been implicated in interactions with root endophytes [ <xref ref-type="bibr" rid="CR80"> 80 </xref> &#x02013; <xref ref-type="bibr" rid="CR82"> 82 </xref> ] and can act as a negative regulator of basal defense responses in <italic> Arabidopsis </italic> [ <xref ref-type="bibr" rid="CR83"> 83 </xref> ] <italic> . </italic>
					</p>
				
					<p>&nbsp;&nbsp;
						Mechanisms of partial <span class="trait">resistance</span> associated with developmental or morphological mechanisms are difficult to assess through annotations of co-localized genes because there are a limited number of clearly defined pathways for these complex traits. However, QTLs 3-3 and 19-1 contain candidate genes putatively involved in morphology and development, including cell growth and cellulose production. Evidence for basal defense was found through co-localization of QTLs 3-2 and 19-3 with LRR-RLK-encoding genes. However, while LRR-RLKs are known to be involved in recognition of microbe associated molecular patterns leading to basal defense responses [ <xref ref-type="bibr" rid="CR65"> 65 </xref> &#x02013; <xref ref-type="bibr" rid="CR67"> 67 </xref> ], LRR-RLK-encoding genes have also been implicated in a range of functions, including plant development [ <xref ref-type="bibr" rid="CR68"> 68 </xref> ]. Transport of toxic compounds is a potential function of a <span class="gene">MATE</span> efflux-encoding gene within QTL 19-1, providing a candidate for the chemical warfare hypothesis. QTLs 3-2, 3-3, 19-1 and 19-3 encompass genes involved in defense signal transduction, with putative functions ranging from negative regulators of basal defense to the control of gene silencing. Among the largest class of genes within these QTL are NB-LRR-encoding genes [ <xref ref-type="bibr" rid="CR64"> 64 </xref> ]. In support of the weak <italic> R </italic> -gene hypothesis, QTLs 3-2 and 13-1 are coincident with six NB-LRR-encoding genes and overlap with estimated positions of <italic> Rps </italic> genes [ <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR10"> 10 </xref> , <xref ref-type="bibr" rid="CR12"> 12 </xref> , <xref ref-type="bibr" rid="CR15"> 15 </xref> ,&#x000a0; <xref ref-type="bibr" rid="CR63"> 63 </xref> ]. Finally, QTL 3-1 does not contain any genes normally associated with defense or development pertinent to <italic> P. sojae </italic> <span class="trait">resistance</span> and therefore supports the idea that quantitative defense can be conferred by a yet unknown class of genes [ <xref ref-type="bibr" rid="CR27"> 27 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Potential of marker selection to increase prediction accuracy of genomic selection in soybean ( <italic> Glycine max </italic> L.)</h2>
			<h3>PMID:27524935</h3>
				
					<h3>PMCID:4965486</h3>
					<h4>期刊:Molecular Breeding</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Yansong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reif </surname>  <given-names> Jochen C. </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wen </surname>  <given-names> Zixiang </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Dechun </given-names>  </name>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Zhangxiong </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Yong </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wei </surname>  <given-names> Shuhong </given-names>  </name>  <xref ref-type="aff" rid="Aff7"> 7 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Shuming </given-names>  </name>  <xref ref-type="aff" rid="Aff8"> 8 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Chunming </given-names>  </name>  <xref ref-type="aff" rid="Aff8"> 8 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Huicai </given-names>  </name>  <xref ref-type="aff" rid="Aff9"> 9 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Chunyan </given-names>  </name>  <xref ref-type="aff" rid="Aff10"> 10 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lu </surname>  <given-names> Weiguo </given-names>  </name>  <xref ref-type="aff" rid="Aff11"> 11 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Ran </given-names>  </name>  <xref ref-type="aff" rid="Aff12"> 12 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Rong </given-names>  </name>  <xref ref-type="aff" rid="Aff13"> 13 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Ruizhen </given-names>  </name>  <xref ref-type="aff" rid="Aff14"> 14 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Zudong </given-names>  </name>  <xref ref-type="aff" rid="Aff15"> 15 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Huaizhu </given-names>  </name>  <xref ref-type="aff" rid="Aff15"> 15 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Wanhai </given-names>  </name>  <xref ref-type="aff" rid="Aff16"> 16 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Jian </given-names>  </name>  <xref ref-type="aff" rid="Aff17"> 17 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Guohua </given-names>  </name>  <xref ref-type="aff" rid="Aff18"> 18 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Chunyan </given-names>  </name>  <xref ref-type="aff" rid="Aff18"> 18 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Luan </surname>  <given-names> Xiaoyan </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fu </surname>  <given-names> Yashu </given-names>  </name>  <xref ref-type="aff" rid="Aff19"> 19 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Tai </given-names>  </name>  <xref ref-type="aff" rid="Aff20"> 20 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Tianfu </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Mengchen </given-names>  </name>  <xref ref-type="aff" rid="Aff10"> 10 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Bincheng </given-names>  </name>  <xref ref-type="aff" rid="Aff16"> 16 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Lei </given-names>  </name>  <xref ref-type="aff" rid="Aff21"> 21 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Weiyuan </given-names>  </name>  <xref ref-type="aff" rid="Aff19"> 19 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wu </surname>  <given-names> Cunxiang </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Shi </given-names>  </name>  <xref ref-type="aff" rid="Aff6"> 6 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yuan </surname>  <given-names> Baojun </given-names>  </name>  <xref ref-type="aff" rid="Aff22"> 22 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Xinan </given-names>  </name>  <xref ref-type="aff" rid="Aff13"> 13 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Han </surname>  <given-names> Dezhi </given-names>  </name>  <xref ref-type="aff" rid="Aff17"> 17 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yan </surname>  <given-names> Hongrui </given-names>  </name>  <xref ref-type="aff" rid="Aff17"> 17 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Wenbin </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0003-1746-3653 </contrib-id>  <name>  <surname> Qiu </surname>  <given-names> Lijuan </given-names>  </name>  <address>  <phone> (+086)-010-82105843 </phone>  <email> qiulijuan@caas.cn </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> College of Agriculture, Northeast Agricultural University, Harbin, 150030 China  </aff>  <aff id="Aff2">  <label> 2 </label> The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI), Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China  </aff>  <aff id="Aff3">  <label> 3 </label> Soybean Research Institute, Heilongjiang Academy of Agricultural Sciences, Harbin, 150086 China  </aff>  <aff id="Aff4">  <label> 4 </label> Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), 06466 Gatersleben, Germany  </aff>  <aff id="Aff5">  <label> 5 </label> Department of Plant, Soil and Microbial Sciences, Michigan State University, East Lansing, MI 48824 USA  </aff>  <aff id="Aff6">  <label> 6 </label> Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, 100081 China  </aff>  <aff id="Aff7">  <label> 7 </label> Heilongjiang Academy of Agricultural Sciences, Harbin, 150086 China  </aff>  <aff id="Aff8">  <label> 8 </label> Soybean Research Institute, Jilin Academy of Agricultural Sciences, Changchun, 130033 China  </aff>  <aff id="Aff9">  <label> 9 </label> Chifeng Institute of Agricultural Sciences, Chifeng, 024031 China  </aff>  <aff id="Aff10">  <label> 10 </label> Institution of Cereal and Oil Crops Hebei Academy of Agricultural and Forestry Sciences, Shijiazhuang, 050031 China  </aff>  <aff id="Aff11">  <label> 11 </label> Economic Crops Institute, Henan Academy of Agricultural Sciences, Zhengzhou, 450002 China  </aff>  <aff id="Aff12">  <label> 12 </label> Crop Research Institute, Shandong Academy of Agricultural Sciences, Jinan, 250010 China  </aff>  <aff id="Aff13">  <label> 13 </label> Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Wuhan, 430062 China  </aff>  <aff id="Aff14">  <label> 14 </label> Institute of Crop Sciences, Jiangxi Academy of Agricultural Sciences, Nanchang, 330200 China  </aff>  <aff id="Aff15">  <label> 15 </label> Institute of Economical Crops, Guangxi Academy of Agricultural Sciences, Nanning, 530007 China  </aff>  <aff id="Aff16">  <label> 16 </label> Hulun Buir Institute of Agricultural Sciences, Hulun Buir, 021000 China  </aff>  <aff id="Aff17">  <label> 17 </label> Heihe Branch Institute, Heilongjiang Academy of Agricultural Sciences, Heihe, 164300 China  </aff>  <aff id="Aff18">  <label> 18 </label> The Crop Research and Breeding Center of Land-Reclamation, Harbin, 150090 Heilongjiang China  </aff>  <aff id="Aff19">  <label> 19 </label> Suihua Branch Institute, Heilongjiang Academy of Agricultural Sciences, Suihua, 152052 China  </aff>  <aff id="Aff20">  <label> 20 </label> Jiamusi Branch Institute, Heilongjiang Academy of Agricultural Sciences, Jiamusi, 154007 China  </aff>  <aff id="Aff21">  <label> 21 </label> Crop Institute, Anhui Academy of Agricultural Sciences, Hefei, 230031 Anhui China  </aff>  <aff id="Aff22">  <label> 22 </label> Zhoukou Institute of Agricultural Sciences, Zhoukou, 466001 Henan China  </aff>
				<br/> 
				年份：<day> 2 </day>  <month> 6 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						We used fivefold cross-validation to examine the potential of genome-wide prediction for different soybean traits. The average prediction accuracy was substantially higher for plant height ( <italic> r </italic> <sub> <span class="gene">GS</span> </sub> &#x000a0;=&#x000a0;0.86) compared to <span class="trait">yield</span> per plant ( <italic> r </italic> <sub> <span class="gene">GS</span> </sub> &#x000a0;=&#x000a0;0.47) (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> , Table S2). Moreover, the standard deviation of the prediction accuracies was substantially larger for <span class="trait">yield</span> per plant compared to plant height (Fig.&#x000a0; <xref rid="Fig5" ref-type="fig"> 5 </xref> ). <fig id="Fig5"> <label> Fig.&#x000a0;5 </label> <caption> <p> Box-Whisker plots of cross-validated prediction accuracies of plant height and <span class="trait">yield</span> per plant, with the method of ridge regression best linear unbiased prediction
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Commercial Crop Yields Reveal Strengths and Weaknesses for Organic Agriculture in the United States</h2>
			<h3>PMID:27552217</h3>
				
					<h3>PMCID:4995028</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0003-2551-4959 </contrib-id>  <name>  <surname> Kniss </surname>  <given-names> Andrew R. </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Savage </surname>  <given-names> Steven D. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jabbour </surname>  <given-names> Randa </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 7 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						One of the main criticisms of organic agriculture has consistently been lower crop <span class="trait">yield</span> compared to non-organic systems. Meta-analyses comparing yields of organic and conventionally grown crops have repeatedly demonstrated a <span class="trait">yield</span> <span class="gene">gap</span> between the two systems. Recently published meta-analyses report mean estimates across all crops varying from 19% to 25% lower yields in organic systems [ <xref rid="pone.0161673.ref006" ref-type="bibr"> 6 </xref> &#x02013; <xref rid="pone.0161673.ref008" ref-type="bibr"> 8 </xref> ]. Critics of organic agriculture argue that society cannot justify being less efficient with arable land in the face of a rapidly growing human population. With respect to conservation interests, if more-efficient conventional farmers can match organic yields with 70% of the land, remaining land could be set aside for conservation and other environmental benefits [ <xref rid="pone.0161673.ref009" ref-type="bibr"> 9 </xref> &#x02013; <xref rid="pone.0161673.ref012" ref-type="bibr"> 12 </xref> ]. However, <span class="trait">yield</span> gains have not been clearly linked with increased land set aside for conservation at the global or regional scale, thus the yield/conservation tradeoff is likely a false dichotomy not representative of the socioecological complexity of agricultural systems, with management decisions tied to markets and policy [ <xref rid="pone.0161673.ref013" ref-type="bibr"> 13 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Organic yields were lower than conventional yields for most crops. However, several crops had no significant difference in yields between organic and conventional production, and in a few examples, organic yields surpassed conventional yields. Across all crops and all states, organic <span class="trait">yield</span> averaged 80% of conventional <span class="trait">yield.</span> However, the <span class="trait">yield</span> ratio varied widely among crops, and in some cases, among states within a crop. Without more detail about the farms reporting <span class="trait">yield</span> data, it is impossible to conclude definitively the cause of the organic <span class="trait">yield</span> <span class="gene">gap</span> in any particular crop. The biggest production challenges organic farmers face relative to conventional farmers are with respect to <span class="trait">fertility</span> (especially nitrogen) due to a lack of synthetic fertilizers, and pest management (weeds, insects, and pathogens) due to a lack of synthetic pesticides [ <xref rid="pone.0161673.ref006" ref-type="bibr"> 6 </xref> ]. These production challenges are likely responsible for the organic <span class="trait">yield</span> <span class="gene">gap</span> in most of the crops we analyzed, though the relative contribution of each may differ. Because <span class="trait">yield</span> data is reported and analyzed at the state level, any discussion on the specific cause of <span class="trait">yield</span> differences between organic and conventional production of a particular crop would be speculation. For this reason, we have refrained from delving too deeply into any specific crop in our analysis, and instead focus on broader trends, though some more detailed discussion and data can be found in <xref ref-type="supplementary-material" rid="pone.0161673.s013"> S1 Supplementary Information </xref> .
					</p>
				
					<p>&nbsp;&nbsp;
						Organic crop yields were significantly less than conventional yields for 9 of 13 field and forage crops ( <xref ref-type="fig" rid="pone.0161673.g001"> Fig 1 </xref> ). Organic wheat <span class="trait">yield</span> was significantly less than conventional wheat for both spring and winter types. Combined over types, organic wheat yielded 66% of conventional <span class="trait">yield.</span> Organic soybean yielded 68% of conventional. The organic cereal crops maize and barley yielded 65% and 76% of conventional <span class="trait">yield,</span> respectively. The organic oat ( <italic> Avena sativa </italic> ) <span class="trait">yield</span> <span class="gene">gap</span> was less, but organic still only produced 80% of conventional oat <span class="trait">yield.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Organic crop <span class="trait">yield</span> for all five crops in <xref ref-type="fig" rid="pone.0161673.g003"> Fig 3 </xref> were significantly less than conventional crop <span class="trait">yield</span> in our analysis based on USDA estimates, which is similar to results presented by Seufert et al. [ <xref rid="pone.0161673.ref008" ref-type="bibr"> 8 </xref> ], and with the exception of tomato, also similar to Ponisio&#x02019;s [ <xref rid="pone.0161673.ref007" ref-type="bibr"> 7 </xref> ] meta-analysis. For maize, soybean, and tomato, our analysis of UDSA data shows an organic <span class="trait">yield</span> <span class="gene">gap</span> that is substantially greater than previous estimates; that is, commercial organic yields for these crops are further behind conventional yields than previous analyses suggest. There are, our analysis indicates, still improvements to be made in commercial organic production of maize, tomato, and soybean for these crops to meet the results obtained mostly under experimental conditions. For wheat and barley, USDA <span class="trait">yield</span> estimates from 2014 suggest <span class="trait">yield</span> ratios similar to the estimates from Seufert et al. [ <xref rid="pone.0161673.ref008" ref-type="bibr"> 8 </xref> ] and Ponisio [ <xref rid="pone.0161673.ref007" ref-type="bibr"> 7 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Although our data agree with previous work showing lower yields in organic production systems in general, our data suggest that commercial hay crops produced significantly greater <span class="trait">yield</span> when produced in an organically managed system. This is contrary to Seufert et al. [ <xref rid="pone.0161673.ref008" ref-type="bibr"> 8 </xref> ] and Ponisio et al. [ <xref rid="pone.0161673.ref007" ref-type="bibr"> 7 </xref> ] who did not find evidence for greater <span class="trait">yield</span> under organic management. Seufert suggested that the organic <span class="trait">yield</span> <span class="gene">gap</span> was less for legume and perennial crops compared to non-legume and annual crops, respectively. In contrast, Ponisio et al. concluded there were not major differences between annual vs perennial crops, nor with legume vs non-legume crops with respect to the organic <span class="trait">yield</span> gap. Our analysis agrees more closely with Seufert et al., showing that annuals and non-legumes fared worse under organic management compared to perennials and legumes, since hay crops tend to be primarily perennial and also include legumes ( <xref ref-type="fig" rid="pone.0161673.g001"> Fig 1 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						We conducted a similar analysis to de Ponti et al. [ <xref rid="pone.0161673.ref006" ref-type="bibr"> 6 </xref> ] for the 25 crops with at least seven data pairs, using a weighted regression to determine whether the organic to conventional <span class="trait">yield</span> ratio was related to conventional <span class="trait">yield.</span> Out of the 25 crops we analyzed, eight showed a significant relationship between organic to conventional crop <span class="trait">yield</span> ratio and conventional crop <span class="trait">yield,</span> including soybean and wheat, the two crops that were significant in the de Ponti analysis ( <xref ref-type="fig" rid="pone.0161673.g004"> Fig 4 </xref> ). Of those eight crops, six showed a decreasing trend, similar to that observed by de Ponti et al. However, contrary to de Ponti&#x02019;s hypothesis, soybean and potato showed an increasing trend in our analysis, suggesting that in locations with greater conventional yields, the organic <span class="trait">yield</span> <span class="gene">gap</span> was lowest. If the statistical significance is ignored and only the direction of the slope (increasing or decreasing) is considered, 15 out of 25 crops had negative slopes compared to 10 with positive slopes ( <xref ref-type="table" rid="pone.0161673.t002"> Table 2 </xref> ). The relationship between the organic <span class="trait">yield</span> <span class="gene">gap</span> and conventional <span class="trait">yield</span> potential does not appear to generalize well across different crops, and in fact, can be completely different depending on the crop of interest.
					</p>
				
					<p>&nbsp;&nbsp;
						The reasons for food insecurity around the world are varied and complex, and go far beyond just <span class="trait">yield.</span> Even so, a dramatic, sustained reduction in crop <span class="trait">yield</span> could be devastating to food security, even in developed countries, making a rapid and complete switch to organic agriculture unwise. Unless other inefficiencies in our food systems are corrected (like food waste, food distribution, and meat-intensive diets), we are likely to need continued <span class="trait">yield</span> increases into the future to feed a growing population. Based on our estimates, if all US wheat production were grown organically, an additional 12.4 million hectares (30.6 million acres) would be needed to match 2014 production levels in the U.S., unless the organic <span class="trait">yield</span> <span class="gene">gap</span> can be narrowed. Current annual production of some crops (like wheat, corn, and soybean) are greater than annual domestic consumption in the U.S., allowing for export. Given world population projections and diet trends, maintaining current production levels in developed countries (while continuing to increase production in developing countries) will likely be the minimum required for a food-secure world.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The Role of Climate Covariability on Crop Yields in the Conterminous United States</h2>
			<h3>PMID:27616326</h3>
				
					<h3>PMCID:5018813</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Leng </surname>  <given-names> Guoyong </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xuesong </given-names>  </name>  <xref ref-type="corresp" rid="c1"> a </xref>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Maoyi </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Asrar </surname>  <given-names> Ghassem R. </given-names>  </name>  <xref ref-type="aff" rid="a1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Leung </surname>  <given-names> L. Ruby </given-names>  </name>  <xref ref-type="aff" rid="a2"> 2 </xref>  </contrib>  <aff id="a1">  <label> 1 </label>  <institution> Joint Global Change Research Institute, Pacific Northwest National Laboratory </institution> , College Park MD,  <country> USA </country>  </aff>  <aff id="a2">  <label> 2 </label>  <institution> Earth System Analysis and Modeling Group, Atmospheric Sciences &#x00026; Global Change Division, Pacific Northwest National Laboratory </institution> ,  <country> USA </country>  </aff>
				<br/> 
				年份：<day> 16 </day>  <month> 08 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Dominant climate factor significantly explaining inter-annual ( <bold> a </bold> ) corn and ( <bold> b </bold> ) soybean <span class="trait">yield</span> variability during 1983&#x02013;2012 at county level over US. Dominant climate factors indicate those with significant relations with crop yields at the 90% confidence level. For example, T means only T has significant relations with crop yields while T&#x00026;P means both T and P have significant relations with crop yields at the 90% statistical confidence level. The Grey color indicates the counties where the crop <span class="trait">yield</span> variability cannot be explained by any of the single climate factor at the 90% significant level. Note the relations are obtained after excluding the effects of climate covariability. The insert <span class="gene">bar</span> diagrams are statistics summarizing the percentage of crop growing counties with crop <span class="trait">yield</span> variability dominated by each climate factor(s) and their percentage contributions to the US total productions. Figure was created by NCAR Command Language <xref ref-type="bibr" rid="b58"> 58 </xref> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Phenology and Seed <span class="trait">Yield</span> Performance of Determinate Soybean Cultivars Grown at Elevated Temperatures in a Temperate Region</h2>
			<h3>PMID:27812185</h3>
				
					<h3>PMCID:5094742</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Choi </surname>  <given-names> Doug-Hwan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ban </surname>  <given-names> Ho-Young </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Seo </surname>  <given-names> Beom-Seok </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Kyu-Jong </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0003-2284-1348 </contrib-id>  <name>  <surname> Lee </surname>  <given-names> Byun-Woo </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 20 </day>  <month> 10 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Temperature is an important environmental factor that affects the development, growth, and <span class="trait">yield</span> of soybeans. Development rates accelerate when temperatures increase from below the optimum or decrease from above the optimum temperature [ <xref rid="pone.0165977.ref008" ref-type="bibr"> 8 </xref> ]. Optimum temperatures were reported to be 30.0&#x000b0;C for vegetative development [ <xref rid="pone.0165977.ref009" ref-type="bibr"> 9 </xref> ] and 26.0&#x000b0;C for anthesis [ <xref rid="pone.0165977.ref010" ref-type="bibr"> 10 </xref> ]. In addition, the optimum temperature for reproductive development was reported to vary between 25.0 and 29.0&#x000b0;C, without significant differences between cultivars of differing maturity. However, a slight increasing trend was observed from <span class="gene">V1</span> to R7 [ <xref rid="pone.0165977.ref011" ref-type="bibr"> 11 </xref> ]. The optimum temperatures for soybean growth and total <span class="trait">yield</span> differed from the optimum temperatures for plant development. The maximum vegetative biomass of soybeans was achieved at temperatures ranging from 25.0 to 37.0&#x000b0;C [ <xref rid="pone.0165977.ref012" ref-type="bibr"> 12 </xref> ]. The number of seeds and seed size are two main <span class="trait">yield</span> components of soybeans; the number of seeds is determined by the number of pods and number of seeds per pod [ <xref rid="pone.0165977.ref013" ref-type="bibr"> 13 </xref> ]. Soybean <span class="trait">yield</span> components are negatively correlated and temperature-dependent [ <xref rid="pone.0165977.ref014" ref-type="bibr"> 14 </xref> , <xref rid="pone.0165977.ref015" ref-type="bibr"> 15 </xref> ]. Compared with the optimum temperature for vegetative growth, the optimum temperatures for single seed growth (23.5&#x000b0;C) and seed size (23&#x000b0;C) of soybeans were lower as reported by Egli and Wardlaw [ <xref rid="pone.0165977.ref016" ref-type="bibr"> 16 </xref> ]. Temporary exposure to extreme temperatures can also affect yield-determining processes. Short heat stress (6&#x000b0;C above ambient) only for 3 days reduced photosynthesis and increased oxidative stress that can cause <span class="trait">yield</span> loss in soybean under field condition [ <xref rid="pone.0165977.ref017" ref-type="bibr"> 17 </xref> ]. Pollen viability (seed set) is decreased by exposure to instantaneous temperatures above 30.0&#x000b0;C in vitro [ <xref rid="pone.0165977.ref018" ref-type="bibr"> 18 </xref> ]. Understanding how <span class="trait">yield</span> components respond to elevated temperatures is important for predicting soybean yields.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Assessment of Soybean Flowering and Seed Maturation Time in Different Latitude Regions of Kazakhstan</h2>
			<h3>PMID:27907027</h3>
				
					<h3>PMCID:5132232</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Abugalieva </surname>  <given-names> Saule </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Didorenko </surname>  <given-names> Svetlana </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Anuarbek </surname>  <given-names> Shynar </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Volkova </surname>  <given-names> Lubov </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gerasimova </surname>  <given-names> Yelena </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sidorik </surname>  <given-names> Ivan </given-names>  </name>  <xref ref-type="aff" rid="aff004">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid" authenticated="false"> http://orcid.org/0000-0001-8590-1745 </contrib-id>  <name>  <surname> Turuspekov </surname>  <given-names> Yerlan </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 4 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Soybean is still a minor crop in Kazakhstan despite an increase in planting area from 4,500 to 11,400 km <sup> 2 </sup> between 2006 and 2014. However, the Government&#x02019;s recently accepted crop diversification policy projects the expansion of soybean cultivation area to more than 40,000 km <sup> 2 </sup> by 2020. The policy is targeting significant expansion of soybean production in South-eastern, Eastern, and Northern regions of Kazakhstan. Successful realization of this policy requires a comprehensive characterization of plant growth parameters to identify optimal genotypes with appropriate adaptive phenotypic traits. In this study 120 soybean accessions from different parts of the World, including 18 accessions from Kazakhstan, were field tested in South-eastern, Eastern, and Northern regions of the country. These studies revealed positive correlation of <span class="trait">yield</span> with flowering time in Northern Kazakhstan, with seed maturity time in Eastern Kazakhstan, and with both these growth stages in South-eastern Kazakhstan. It was determined that in South-eastern, Eastern and Northern regions of Kazakhstan the majority of productive genotypes were in maturity groups MGI, MG0, and MG00, respectively. The accessions were genotyped for four major maturity genes ( <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> , and <italic> E4 </italic> ) in order to assess the relationship between <italic> E </italic> loci and agronomic traits. The allele composition of the majority of accessions was <italic> e1-as/e2/E3/E4 </italic> (specific frequencies 57.5%, 91.6%, 65.0%, and 63.3%, respectively). Accessions with dominant alleles in either <italic> E3 </italic> or <italic> E4 </italic> genes showed higher <span class="trait">yield</span> in all three regions, although the specific genotype associated with greatest productivity was different for each site. Genotype-environment interaction studies based on <span class="trait">yield</span> performances suggest that South-east and East regions formed one mega-environment, which was well separated from North Kazakhstan where significantly earlier time to maturation is required. The results provide important insights into the relationship between genetic and phenotypic patterns in new soybean growing territories in Kazakhstan.
					</p>
				
					<p>&nbsp;&nbsp;
						The soybean material consisted of 120 genetically diverse cultivars and lines from the collection of the Kazakh Agricultural Research Institute ( <ext-link ext-link-type="uri" xlink:href="http://www.kiz.kz/"> http://www.kiz.kz </ext-link> , Almaty region, Kazakhstan). The accessions can be obtained by direct request from Dr. Didorenko ( <email> svetl_did@mail.ru </email> ). The collection is currently a prime source for ongoing breeding projects in SEK, EK, and NK regions of Kazakhstan. The research material represents 12 countries from 5 geographic regions, including Western and Eastern Europe, Northern America, Eastern Asia, and Kazakhstan ( <xref ref-type="supplementary-material" rid="pone.0166894.s001"> S1 Table </xref> ). It includes 18 accessions from Kazakhstan containing 5 officially registered cultivars in SEK. The collection was grown in three randomized replicates in breeding stations of SEK, EK, and NK regions of Kazakhstan. The locations and meteorological data of field conditions are shown in <xref ref-type="fig" rid="pone.0166894.g001"> Fig 1 </xref> . SEK was irrigated while EK and NK stations were non-irrigated sites. Plants were grown in 1 meter rows with 30 cm between rows and 5 cm between plants within rows. Times to flowering and maturation were studied at R stages described by Fehr et al [ <xref rid="pone.0166894.ref018" ref-type="bibr"> 18 </xref> ]. The averaged data for triplicated trials were analyzed statistically for assessments of plant growth and <span class="trait">yield</span> components in three studied locations. The accessions were genotyped for <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , <italic> E3 </italic> , and <italic> E4 </italic> alleles according to protocols published elsewhere [ <xref rid="pone.0166894.ref005" ref-type="bibr"> 5 </xref> ], [ <xref rid="pone.0166894.ref019" ref-type="bibr"> 19 </xref> ], [ <xref rid="pone.0166894.ref020" ref-type="bibr"> 20 </xref> ], [ <xref rid="pone.0166894.ref014" ref-type="bibr"> 14 </xref> ], respectively. DNA extraction and agarose gel electrophoresis were performed according to Abugalieva et al [ <xref rid="pone.0166894.ref017" ref-type="bibr"> 17 </xref> ]. Assignment of samples to maturity groups in each region was carried out using thermal time (TT) degrees ( <sup> 0 </sup> Cd) at R8 stage (TT-R8) according to Fehr et al [ <xref rid="pone.0166894.ref018" ref-type="bibr"> 18 </xref> ]. Time from emergence to flowering was designated as VE-R2, from flowering to pods development as R2-R4, from pods development to seed maturation as R4-R8, and time from emergence to maturation as VE-R8. Statistical analyses of data, including Pearson&#x02019;s correlation and <italic> t-test </italic> , were calculated by using GraphPad Prism (version 5.00 for Windows, GraphPad Software, San Diego California USA, <ext-link ext-link-type="uri" xlink:href="http://www.graphpad.com/"> www.graphpad.com </ext-link> ). Genotype-environment interaction patterns, including AMMI (Additive Main Multiplicative Interaction) and GGE Biplot methods, were studied using the GenStat package (16 <sup> th </sup> release, VSN International, Hertfordshire, UK). The symmetric scaling option of both methods and available field data for all three sites were used in estimations.
					</p>
				
					<p>&nbsp;&nbsp;
						The role of each gene in plant performance was examined by comparing field data for each parameter. For the SEK location a two-tailed t-test indicated that alleles at <italic> <span class="gene">E1</span> </italic> and <italic> E4 </italic> were highly significant for plant growth performance ( <xref ref-type="supplementary-material" rid="pone.0166894.s004"> S4 Table </xref> ). Specifically, diversity at <italic> <span class="gene">E1</span> </italic> was statistically significant for time to flowering (P&#x0003c;0.001), while that at <italic> E4 </italic> played an important role for time to maturity and <span class="trait">yield</span> (P&#x0003c;0.01). In pairwise analysis significant allele combinations for maturation and YPP were also recorded for <italic> E1/E4 </italic> (P&#x0003c;0.001). This trend was confirmed for three <italic> E </italic> gene comparisons ( <xref ref-type="supplementary-material" rid="pone.0166894.s004"> S4 Table </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> t-test </italic> was applied in order to determine the significance of each gene and their combinations in the analysis of growth stages and <span class="trait">yield</span> components for all studied 115 accessions in EK ( <xref ref-type="supplementary-material" rid="pone.0166894.s004"> S4 Table </xref> ). It was found that the <italic> <span class="gene">E1</span> </italic> genotype is statistically significant for pre-flowering time (P&#x0003c;0.001), and <italic> E4 </italic> for post-flowering time (P&#x0003c;0.05). The analysis of combinations of alleles for three genes ( <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , <italic> and E4 </italic> ) allowed the identification of 11 different allelic combinations with the largest group being 39 <italic> e1-as/E3/E4 </italic> accessions. However, similar to the MG studies, the most productive genotype was found to be <italic> e1/E3/E4 </italic> (11 accessions). In the study of 115 accessions, <italic> e1/E3/E4 </italic> showed earlier flowering (3.4 days in average) and later maturation time (10.3 days in average) than <italic> e1-as/E3/E4 </italic> . In accessions with an <italic> <span class="gene">e1</span> </italic> background all four combinations of <italic> E3 </italic> and <italic> E4 </italic> alleles showed similar flowering time (30.5&#x02013;30.9 days) but different time to maturation. The earliest maturation (R2-R8) was recorded for <italic> e3/e4 </italic> (57.5 days) followed by <italic> E3/e4 </italic> (65.5), <italic> e3/E4 </italic> (71.3), and <italic> E3/E4 </italic> (74.0). Hence the largest difference between these groups was 16.5 days, which was the biggest factor contributing to the YPP in the region (P&#x0003c;0.0001). It is interesting to note that none of five local promising breeding lines was found to have the <italic> e1/E3/E4 </italic> genotype in EK ( <xref ref-type="supplementary-material" rid="pone.0166894.s001"> S1 Table </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Pairwise comparisons of genes showed that <italic> <span class="gene">E1</span> </italic> and <italic> E4 </italic> showed the highest statistical correlation with time to flowering and maturity and YPP ( <xref ref-type="supplementary-material" rid="pone.0166894.s004"> S4 Table </xref> ). As in the other two environments (SEK and EK), the individual contributions of <italic> E2 </italic> alleles to flowering and maturation in NK were not significant in this study. In three gene analyses the most frequent allelic combination was <italic> e1-as/E3/E4 </italic> (30 accessions), which included five locally bred accessions. In an <italic> <span class="gene">E1</span> </italic> background all three genotypes had the same duration of seed maturation but differed by 4.2 days in flowering time between <italic> E3/e4 </italic> and <italic> e3/E4 </italic> genotypes, suggesting that late flowering is beneficial for higher <span class="trait">yield</span> components in the region. The comparison of <italic> E1/E3/e4 </italic> and <italic> e1-as/E3/E4 </italic> using <italic> t-test </italic> showed statistically significant difference for duration of R2 and R8, and YPP (P&#x0003c;0.01).
					</p>
				
					<p>&nbsp;&nbsp;
						The <italic> E </italic> series of genes were shown to be important factors for plant adaptation in soybean growing regions [ <xref rid="pone.0166894.ref012" ref-type="bibr"> 12 </xref> ], [ <xref rid="pone.0166894.ref014" ref-type="bibr"> 14 </xref> ]. It was recently hypothesized that three of those <italic> E </italic> genes ( <italic> <span class="gene">E1</span> </italic> , <italic> E3 </italic> , <italic> and E4 </italic> ) play larger roles in pre- and post-flowering photoperiod responses [ <xref rid="pone.0166894.ref011" ref-type="bibr"> 11 </xref> ]. The authors underlined the role of <italic> E3 </italic> and <italic> E4 </italic> genes in photoperiod insensitive genotypes for early flowering and seed maturation. In particular, they proposed a gene regulatory network comprising three known maturity genes, a determinate habit gene ( <italic> <span class="gene">Dt1</span> </italic> ) and two unknown factors (X, Y) that regulate flowering under long day conditions. Results in this study suggests that alleles of <italic> <span class="gene">E1</span> </italic> and <italic> E4 </italic> and their interaction, with a contribution from <italic> E1/E3/E4 </italic> interactions, are most important in prediction of flowering time in different latitude regions. The frequency of the <italic> E2 </italic> dominant allele in this study was small and the effect of the gene was statistically insignificant for plant growth parameters in all tested regions. In the SEK region the single accession of <italic> E1/e2/E3/E4 </italic> (SD116, local commercial cultivar &#x0201c;Zara&#x0201d;) had the latest pre- and post-flowering and the highest <span class="trait">yield</span> in comparison with averages of sixteen other genotypes, including <italic> e1-as/e2/E3/E4 </italic> and <italic> e1/e2/E3/E4 </italic> . The duration of VE-R8 stage in <italic> E1/e2/E3/E4 </italic> was lower than in <italic> e1-as/e2/E3/E4 </italic> and <italic> e1/e2/E3/E4 </italic> genotypes for 22.0 and 17.2 days, respectively. Analysis of time to flowering and maturation suggested that SEK was the only studied region where VE-R2 and R2-R8 have significant correlation (P&#x0003c;0.001, <xref ref-type="supplementary-material" rid="pone.0166894.s003"> S3 Table </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In higher latitude regions of EK and NK the patterns of associations between flowering genes and <span class="trait">yield</span> were different to some extent. The highest <span class="trait">yield</span> in the collection in EK was recorded for the <italic> e1/e2/E3/E4 </italic> genotype (11 accessions). Dominant alleles of <italic> E3/E4 </italic> significantly delayed the R2-R8 stage resulting in the longest VE-R8 among 17 genotypes ( <xref ref-type="supplementary-material" rid="pone.0166894.s005"> S5 Table </xref> ). The <italic> e1/e2/E3/E4 </italic> genotype showed 3.4 days earlier flowering and 10.3 days later post-flowering difference in comparison with <italic> e1-as/e2E3/E4 </italic> suggesting that <italic> E3/E4 </italic> is prolonging seed maturation when <italic> <span class="gene">E1</span> </italic> and <italic> E2 </italic> are dysfunctional. The GEI analysis based on <span class="trait">yield</span> performance suggested that early flowering time is essential in SEK and EK, and, therefore, this result effectively separated SEK and EK from NK ( <xref ref-type="fig" rid="pone.0166894.g002"> Fig 2 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						This study is the first attempt to assess soybean adaptation in different latitude regions of Kazakhstan based on <italic> E </italic> gene variation and field trial results. Although <italic> e1-as/e2/E3/E4 </italic> was the most common genotype in this collection, it was not among the top <span class="trait">yield</span> performing genotypes in three studied regions. Specific allele combinations of the four <italic> E </italic> genes and optimal ranges of time to flowering and maturity were proposed for each experimental site based on identification of top genotypes with higher productivity. The results confirmed the high importance of <italic> <span class="gene">E1</span> </italic> for length of flowering, and of <italic> E3/E4 </italic> for time to maturation. The MG analyses based on <span class="trait">yield</span> performance and GGE Biplot method showed that SEK and EK regions have more similarity and comprised one mega-environment while NK region formed another.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:<italic> Bradyrhizobium </italic> Inoculants Enhance Grain Yields of Soybean and Cowpea in Northern Ghana</h2>
			<h3>PMID:27965683</h3>
				
					<h3>PMCID:5126734</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Ulzen </surname>  <given-names> Jacob </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/366159/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Abaidoo </surname>  <given-names> Robert C. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mensah </surname>  <given-names> Nana E. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Masso </surname>  <given-names> Cargele </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/358010/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> AbdelGadir </surname>  <given-names> AbdelAziz H. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/366169/overview"/>  </contrib>
				<br/> 
				年份：<day> 10 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Among the factors for selecting <italic> Rhizobium </italic> strains, effectiveness is of major importance. In recent times, there have been interventions by various non-Governmental Organizations (NGOs) to increase the <span class="trait">yield</span> of soybean and cowpea for smallholder farmers through rhizobia inoculation. The <italic> Bradyrhizobium </italic> strains BR 3267 and BR 3262 are of proven quality in increasing yields of cowpea in Brazil ( <xref rid="B16" ref-type="bibr"> Martins et al., 2003 </xref> ). Biofix and Legumefix have been used in many parts of sub-Saharan Africa to increase <span class="trait">yield</span> of smallholder farmers. There has been a long search for good inoculant formulation for soybean and cowpea production in Ghana. This study; therefore, attempts to bridge the <span class="trait">yield</span> <span class="gene">gap</span> using BR 3267, BR3262, Legumefix, and Biofix as an entry point for smallholder farmers in Ghana and also evaluate the symbiotic effectiveness and economic benefits of selected <italic> Bradyrhizobium </italic> strains for soybean and cowpea.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Allocation of Nitrogen and Carbon Is Regulated by Nodulation and Mycorrhizal Networks in Soybean/Maize Intercropping System</h2>
			<h3>PMID:28018420</h3>
				
					<h3>PMCID:5160927</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Guihua </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/397926/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sheng </surname>  <given-names> Lichao </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/397904/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Dan </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/397910/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sheng </surname>  <given-names> Jiandong </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/396406/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xiurong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/370999/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liao </surname>  <given-names> Hong </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 01 </day>  <month> 12 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						In legume/cereal intercropping systems, legumes are generally weak competitors compared with cereals ( <xref rid="B35" ref-type="bibr"> Willey and Rao, 1980 </xref> ; <xref rid="B7" ref-type="bibr"> Hauggaard-Nielsen and Jensen, 2001 </xref> ), which is often ascribed to differences in the root distributions of legumes and cereals, and the resulting differences in the ability of these crops to compete for soil N ( <xref rid="B5" ref-type="bibr"> <span class="gene">Fan</span> et al., 2006 </xref> ). In the maize/soybean intercropping system, the <span class="trait">yield</span> advantage of maize has been also reported ( <xref rid="B30" ref-type="bibr"> Tang et al., 2005 </xref> ; <xref rid="B22" ref-type="bibr"> Meng et al., 2015 </xref> ). In the present study, we demonstrated that the high competitive ability of maize relative to soybean can be partly ascribed to the contribution of AMF and rhizobia. To our knowledge, this is the first report that the beneficial effects of intercropping on maize growth are due, at least in part, to allocation of carbon and nitrogen regulated by nodulation and mycorrhizal networks in the soybean/maize intercropping system.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Seed-Specific Overexpression of the Pyruvate Transporter <italic> BASS2 </italic> Increases Oil Content in <italic> Arabidopsis </italic> Seeds</h2>
			<h3>PMID:28265278</h3>
				
					<h3>PMCID:5316546</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Eun-Jung </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/390110/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Oh </surname>  <given-names> Minwoo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/396294/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hwang </surname>  <given-names> Jae-Ung </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li-Beisson </surname>  <given-names> Yonghua </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/46110/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nishida </surname>  <given-names> Ikuo </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/44906/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lee </surname>  <given-names> Youngsook </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/186586/overview"/>  </contrib>
				<br/> 
				年份：<day> 31 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Seed oil is an important source of energy, and is in increasing demand for various industrial applications (Dyer et al., <xref rid="B10" ref-type="bibr"> 2008 </xref> ; Hayden et al., <xref rid="B18" ref-type="bibr"> 2011 </xref> ). Thus, methods to increase seed oil <span class="trait">yield</span> are being actively investigated. Many efforts to boost seed oil <span class="trait">yield</span> have involved genetically engineering <italic> Arabidopsis </italic> and other plants to overexpress transcription factors and enzymes involved in fatty acid biosynthesis and lipid production and storage (Napier et al., <xref rid="B33" ref-type="bibr"> 2014 </xref> ). For example, overexpression of the transcription factor <italic> WRINKLED 1 </italic> ( <italic> <span class="gene">WRI1</span> </italic> ), which controls the expression of genes involved in lipid metabolism, including glycolysis and fatty acid biosynthesis, increased seed oil content by 10&#x02013;20% compared to the wild type (Cernac and Benning, <xref rid="B8" ref-type="bibr"> 2004 </xref> ; Baud et al., <xref rid="B5" ref-type="bibr"> 2007 </xref> , <xref rid="B6" ref-type="bibr"> 2009 </xref> ; Maeo et al., <xref rid="B31" ref-type="bibr"> 2009 </xref> ). Furthermore, the seed-specific overexpression of the <italic> Arabidopsis </italic> acyl-CoA:diacylglycerol transferase 1 ( <italic> <span class="gene">DGAT1</span> </italic> ), which catalyzes the formation of triacylglycerol (TAG) from diacylglycerol (DAG) and acyl-CoA, increased oil content by 11&#x02013;28% compared with the control (Jako et al., <xref rid="B21" ref-type="bibr"> 2001 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Evaluation of Exotically-Derived Soybean Breeding Lines for Seed <span class="trait">Yield,</span> Germination, Damage, and Composition under Dryland Production in the Midsouthern USA</h2>
			<h3>PMID:28289420</h3>
				
					<h3>PMCID:5327356</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Bellaloui </surname>  <given-names> Nacer </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/63229/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Smith </surname>  <given-names> James R. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mengistu </surname>  <given-names> Alemu </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/86055/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ray </surname>  <given-names> Jeffery D. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/376867/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Gillen </surname>  <given-names> Anne M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 1 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<bold> Analysis of variance results for soybean seed <span class="trait">yield,</span> seed composition, seed quality, and seed disease measures for 16 soybean genotypes grown under dryland conditions in a 2-year field experiment at Stoneville, <span class="gene">MS</span> in 2012 and 2013 </bold> .
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="trait">Yield</span> showed significant differences between years. <span class="trait">Yield</span> in 2012 was higher than in 2013, except for AG5606 (Table <xref ref-type="table" rid="T3"> 3 </xref> ), which had an 8.6% increase in 2013. For the other genotypes, the difference in <span class="trait">yield</span> between years ranged from a 12.7 to a 58.8% decrease in 2013 over 2012 (Table <xref ref-type="table" rid="T3"> 3 </xref> ). Two breeding lines (one <span class="gene">MS</span> and one IL breeding line) were more stable for <span class="trait">yield,</span> with each line having &#x0003c; 15% difference in <span class="trait">yield</span> between years (Table <xref ref-type="table" rid="T3"> 3 </xref> ). The <span class="gene">MS</span> breeding line, 25-1-1-4-1-1, yielded higher than all other breeding lines and higher than all cultivars except AG3803, AG3905, and DK4866 in 2013, the more stressful year. Although 2013 was generally cooler than 2012 (Figures <xref ref-type="fig" rid="F2"> 2A,B </xref> , <xref ref-type="fig" rid="F3"> 3A,B </xref> ), rainfall was more uniform during the growing season in 2012 than in 2013 and also more rainfall occurred in July through August (seed-fill stage) in 2012 (Figures <xref ref-type="fig" rid="F1"> 1A,B </xref> ). It appears that the higher <span class="trait">yield</span> in 2012 was likely due to rainfall uniformity and higher rainfall during the seed-fill stage. However, this cannot be generalized, as the higher <span class="trait">yield</span> of AG5606 (8.6% higher) in 2013 may be due to its longer maturity (Table <xref ref-type="table" rid="T1"> 1 </xref> ), which may have better utilized the August and September rains occurring in that year.
					</p>
				
					<p>&nbsp;&nbsp;
						This study involved soybean genotypes from three distinct germplasm pools; the nine cultivars typify the current pool available to producers, the three breeding lines from the Illinois exotic pool were derived from PIs 68508, 445837, 361064, 407710, 189930, and 68600 and previously selected for high <span class="trait">yield</span> potential, and the four breeding lines from the Mississippi exotic pool were derived from PIs 587982A and 603756 and previously selected for high germinability under heat stress. Given the distinct nature of the three germplasm pools, some inter-pool comparisons are in order. First, there was no significant difference between the three pools for seed <span class="trait">yield</span> or total seed damage in either year (Table <xref ref-type="table" rid="T11"> 11 </xref> ). The high germinability (MS) pool had significantly lower seed size than the other two pools in 2013, but not in 2012 (Table <xref ref-type="table" rid="T11"> 11 </xref> ). The <span class="gene">MS</span> pool had lower hard seed and higher germination and AA than the cultivar pool in both years (Table <xref ref-type="table" rid="T11"> 11 </xref> ). There was less seed wrinkling in the <span class="gene">MS</span> pool than in the cultivar pool only in 2013, but the <span class="gene">MS</span> pool had lower green seed damage than the other two pools in both years (Table <xref ref-type="table" rid="T11"> 11 </xref> ). In terms of seed constituents, the <span class="gene">MS</span> pool had higher protein than the other two pools in both years, while having lower Ca than the cultivar pool in both years (Table <xref ref-type="table" rid="T11"> 11 </xref> ). The <span class="gene">MS</span> pool had higher oleic acid than the other two pools only in 2012 (Table <xref ref-type="table" rid="T11"> 11 </xref> ). Given the increasing world-wide demand for high quality protein meal, and given the increased likelihood of global warming, the differences between pools for protein content are striking (Tables <xref ref-type="table" rid="T4"> 4 </xref> , <xref ref-type="table" rid="T11"> 11 </xref> ). During the breeding process, the four lines in the high germinability <span class="gene">MS</span> pool were never selected for protein content. Rather, they were repeatedly selected in a pedigree breeding protocol that tested only seed germination from the F <sub> 2 </sub> through F <sub> 5 </sub> generations. Yet, as can be observed in Table <xref ref-type="table" rid="T4"> 4 </xref> , their protein contents are significantly and substantially higher than any other line in either of the other germplasm pools. That all four lines have high protein content is likely due to more than just chance. The potential association between protein content and high germinability needs further investigation.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Quantitative trait loci underlying <span class="trait">resistance</span> to sudden death syndrome (SDS) in MD96-5722 by &#x02018;Spencer&#x02019; recombinant inbred line population of soybean</h2>
			<h3>PMID:28324582</h3>
				
					<h3>PMCID:4362743</h3>
					<h4>期刊:3 Biotech</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Anderson </surname>  <given-names> J. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Akond </surname>  <given-names> M. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kassem </surname>  <given-names> M. A. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Meksem </surname>  <given-names> K. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Kantartzi </surname>  <given-names> S. K. </given-names>  </name>  <address>  <email> kantart@siu.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label> Department of Plant, Soil and Agricultural Systems, Southern Illinois University, Carbondale, IL 62901-4415 USA  </aff>  <aff id="Aff2">  <label> 2 </label> Plant Genomics and Biotechnology Lab, Department of Biological Sciences, Fayetteville State University, Fayetteville, NC 28301-4298 USA  </aff>
				<br/> 
				年份：<day> 21 </day>  <month> 3 </month>  <year> 2014 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Although simple sequence repeat (SSR) markers were used successfully for localizing QTL in soybeans (Hnetkovsky et al. 1995; Kassem et al. <xref ref-type="bibr" rid="CR21"> 2006 </xref> ; Kazi et al. <xref ref-type="bibr" rid="CR24"> 2008 </xref> ), single nucleotide polymorphism (SNP) markers are the most ample genetic markers available for taking research to the next level (Nicod and Largiader <xref ref-type="bibr" rid="CR25"> 2003 </xref> ; <span class="gene">Fan</span> et al. <xref ref-type="bibr" rid="CR10"> 2003 </xref> ; Barbazuk et al. <xref ref-type="bibr" rid="CR4"> 2007 </xref> ; Hsu et al. <xref ref-type="bibr" rid="CR15"> 2008 </xref> ). Currently, very few SNP markers/QTL are available for SDS <span class="trait">resistance</span> in soybean. Recently, Kassem et al. ( <xref ref-type="bibr" rid="CR23"> 2012 </xref> ) mapped 14 significant QTL linked to SDS <span class="trait">resistance</span> through a high-density SNP-based genetic linkage map using PI438489B by &#x02018;Hamilton&#x02019; RIL population of soybean. Another SNP-based high-density genetic linkage map (Akond et al. <xref ref-type="bibr" rid="CR1"> 2013 </xref> ) was constructed using MD 96-5722&#x000a0;&#x000d7;&#x000a0;&#x02018;Spencer&#x02019; RIL population to map QTL for SDS <span class="trait">resistance.</span> Therefore, the objective of this research was to identify QTL conferring SDS <span class="trait">resistance</span> in a population advanced from a cross between soybean line MD 96-5722 and variety &#x02018;Spencer&#x02019;.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Isolation and characterization of a promoter responsive to salt, osmotic and dehydration stresses in soybean</h2>
			<h3>PMID:28350037</h3>
				
					<h3>PMCID:5452143</h3>
					<h4>期刊:Genetics and Molecular Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Conforte </surname>  <given-names> Alessandra Jordano </given-names>  </name>  <xref ref-type="aff" rid="aff1">
				<br/> 
				年份：<day> 21 </day>  <month> 11 </month>  <year> 2016 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The constitutive cauliflower mosaic virus 35S RNA promoter ( <italic> CaMV35S </italic> ) has been broadly used to control the expression of several genes responsive to <span class="trait">drought</span> due to the strong and conspicuous activity presented when introduced in plant genomes ( <xref rid="B34" ref-type="bibr"> Luo <italic> et al. </italic> , 2013 </xref> ; <xref rid="B39" ref-type="bibr"> Nov&#x000e1;k <italic> et al. </italic> , 2013 </xref> ; <xref rid="B7" ref-type="bibr"> Bhauso <italic> et al. </italic> , 2014 </xref> ; <xref rid="B60" ref-type="bibr"> Withanage <italic> et al. </italic> , 2015 </xref> ; <xref rid="B17" ref-type="bibr"> Du <italic> et al. </italic> , 2016 </xref> ). However, the constitutive overexpression of genes can affect plant development and metabolism ( <xref rid="B26" ref-type="bibr"> Hsieh <italic> et al. </italic> , 2002 </xref> ; <xref rid="B24" ref-type="bibr"> Homrich <italic> et al. </italic> , 2012 </xref> ). Indeed, the control of the expression of drought-tolerant genes by stress-responsive and/or tissue-specific promoters has been used as alternative for the elimination of negative effect of constitutive gene expression driven by the <italic> CaMV35S </italic> promoter ( <xref rid="B10" ref-type="bibr"> Chan Ju <italic> et al. </italic> , 2012 </xref> ; <xref rid="B5" ref-type="bibr"> Banerjee <italic> et al. </italic> , 2013 </xref> ; <xref rid="B63" ref-type="bibr"> Yan <italic> et al. </italic> , 2015 </xref> ). Recently, stress-responsive promoters, such as the <italic> PeNAC1 </italic> ( <xref rid="B59" ref-type="bibr"> Wang <italic> et al. </italic> , 2016 </xref> ), <italic> ZmGAPP </italic> ( <xref rid="B25" ref-type="bibr"> Hou <italic> et al. </italic> , 2016 </xref> ), <italic> BBX24 </italic> ( <xref rid="B27" ref-type="bibr"> Imtiaz <italic> et al. </italic> , 2015 </xref> ), and <italic> <span class="gene">GmNCED</span> </italic> 1 and <italic> <span class="gene">GmMYB363P</span> </italic> promoters ( <xref rid="B33" ref-type="bibr"> Li <italic> et al. </italic> , 2014 </xref> ; <xref rid="B64" ref-type="bibr"> Zhang <italic> et al. </italic> , 2014 </xref> ), have been characterized. Among gene promoters previously characterized as drought-responsive, the <italic> AtRD29A </italic> promoter showed stronger activity during water deficit stress in transgenic plants when compared to the control plants ( <xref rid="B61" ref-type="bibr"> Yamaguchi-Shinozaki and Shinozaki, 1993 </xref> ). Therefore, it has been successfully used to drive the expression of drought-resistant genes in different plant species ( <xref rid="B30" ref-type="bibr"> Kasuga <italic> et al. </italic> , 1999 </xref> ; <xref rid="B41" ref-type="bibr"> Polizel <italic> et al. </italic> , 2011 </xref> ; <xref rid="B15" ref-type="bibr"> Datta <italic> et al. </italic> , 2012 </xref> ; <xref rid="B45" ref-type="bibr"> Saint Pierre <italic> et al. </italic> , 2012 </xref> ; <xref rid="B8" ref-type="bibr"> Bihmidine <italic> et al. </italic> , 2013 </xref> ; <xref rid="B19" ref-type="bibr"> Engels <italic> et al. </italic> , 2013 </xref> ). However, the use of the <italic> AtRD29A </italic> promoter is restricted by patent protection ( <xref rid="B47" ref-type="bibr"> Shinozaki <italic> et al. </italic> , 2007 </xref> ). Moreover, soybean promoters could be an alternative to avoid the use of exotic DNA in soybean transgenic plants. Therefore, the isolation and characterization of new drought-responsive promoters from soybean are crucial and will augment the set of tools available for the development of drought-tolerant cultivars.
					</p>
				
					<p>&nbsp;&nbsp;
						The use of the <span class="trait">drought</span> stress-responsive promoter originally isolated from the soybean genome may have advantages as a biotechnological tool to improve <span class="trait">drought</span> stress tolerance in soybean cultivars. Its use may minimize the unexpected promoter activity caused by cryptic <italic> cis </italic> -elements introduced with the use of foreign DNA in soybean transgenic cultivars. In addition to <italic> pGAL </italic> -2kb, two other soybean promoters, <italic> <span class="gene">GmNCED1</span> </italic> and <italic> <span class="gene">GmMYB363</span> </italic> , which present activity in response to abiotic stresses, were recently characterized and patented ( <xref rid="B33" ref-type="bibr"> Li <italic> et al. </italic> , 2014 </xref> ; <xref rid="B64" ref-type="bibr"> Zhang <italic> et al. </italic> , 2014 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						The activity of the <italic> <span class="gene">GmNCED1</span> </italic> promoter was significantly induced by salt stress in the roots of tobacco transgenic plants ( <xref rid="B33" ref-type="bibr"> Li <italic> et al. </italic> , 2014 </xref> ). The <italic> <span class="gene">GmMYB363</span> </italic> promoter activity was induced in soybean transgenic roots under PEG 6000 treatment ( <xref rid="B64" ref-type="bibr"> Zhang <italic> et al. </italic> , 2014 </xref> ). Compared to <italic> pGmNCED1 </italic> and <italic> pGmMYB363 </italic> , the soybean <italic> pGAL </italic> -2kb promoter is active in roots as well as in leaves during <span class="trait">drought</span> stress. The promoter activity in both organs may represent an advantage to provide wide and effective protection under water deficit when driving the expression of drought-tolerant genes in transgenic plants. Nevertheless, <italic> pGmMYB363 </italic> and <italic> pGmNCED1 </italic> have shorter sequences than the <italic> pGAL </italic> -2kb promoter (1,384 and 1,253 bp, respectively), which is an important characteristic. However, we identified a high number of <italic> cis </italic> -elements in the <italic> pGAL </italic> -2kb promoter between 1000 and 1500 bp upstream of the start codon. Furthermore, we did not find <italic> cis </italic> -elements in the distal region between 1,500 and 2,000 bp. Therefore, it is important to evaluate the 1.5-kb sequence of the <italic> pGAL </italic> promoter to improve its use as a biotech tool. In addition, other constructs containing <italic> pGAL </italic> -2kb deletions may also be active as drought-responsive, as efficient as the entire sequence. <xref rid="B27" ref-type="bibr"> Imtiaz <italic> et al. </italic> (2015) </xref> showed that the deletion of the 1,162-bp fragment between &#x02212;2,552 (full promoter) and &#x02212;1,390 significantly reduced promoter activity in leaves and roots, while no significant decrease in promoter activity was observed with further deletions from &#x02212;1,390 to &#x02212;780, &#x02212;780 to 600 and &#x02212;600 to &#x02212;480. Thus, further analyses will aid in better characterizing the use of the <italic> pGAL </italic> -2kb promoter as a new <span class="trait">drought</span> stress-responsive promoter. Here we showed that the full-length <italic> pGAL </italic> -2kb sequence is a potential candidate for use in genetic engineering to induce a response to <span class="trait">drought</span> stress in soybean.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Meteorological fluctuations define long-term crop <span class="trait">yield</span> patterns in conventional and organic production systems</h2>
			<h3>PMID:28386102</h3>
				
					<h3>PMCID:5429610</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <name>  <surname> Teasdale </surname>  <given-names> John R. </given-names>  </name>  <address>  <email> john.teasdale@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cavigelli </surname>  <given-names> Michel A. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <institution-wrap>  <institution-id institution-id-type="GRID"> grid.463419.d </institution-id>  <institution> Sustainable Agricultural Systems Laboratory,  </institution>  <institution> Agricultural Research Service, United States Department of Agriculture,  </institution>  </institution-wrap> 10300 Baltimore Avenue, Beltsville, Maryland 20705 United States  </aff>
				<br/> 
				年份：<day> 14 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Several recent meta-analyses of conventional versus organic farming have found that organic farming provides many benefits over conventional farming in environmental protection, improved biodiversity, higher food nutritional value, and positive economic returns. A summary of 71 publications found that European organic farms had significantly higher soil organic matter content and lower nitrogen leaching and nitrous oxide emissions per unit of field area, however, nitrogen leaching and nitrous oxide emissions per product unit were higher from organic systems <sup> <xref ref-type="bibr" rid="CR9"> 9 </xref> </sup> . This report also summarized 38 publications on biodiversity and found that most studies demonstrated higher species richness and abundance on organic farms. Another meta-analysis summarized 343 publications and demonstrated that concentrations of antioxidants were 19 to 69% higher in organic crops/foods, but the occurrence of pesticide residues was four times higher on conventional crops <sup> <xref ref-type="bibr" rid="CR10"> 10 </xref> </sup> . There was a positive difference in net returns per field area for organic versus conventional production of corn and soybean grown in the US, primarily because of price premiums paid for organic crops <sup> <xref ref-type="bibr" rid="CR11"> 11 </xref> </sup> . On the other hand, crop <span class="trait">yield</span> is often limiting on organic farms. A summary of 66 reports of 316 organic-to-conventional <span class="trait">yield</span> comparisons showed that, on average, yields of organically-produced crops were 25% lower than yields of conventionally-produced crops <sup> <xref ref-type="bibr" rid="CR12"> 12 </xref> </sup> . In a subsequent meta-analysis with upgraded analytical techniques, organic yields were shown to be 19% lower than conventional yields overall, but multi-cropping and crop rotations substantially reduced that <span class="trait">yield</span> <span class="gene">gap</span> to 9% and 8%, respectively, when these methods were applied in only organic systems <sup> <xref ref-type="bibr" rid="CR13"> 13 </xref> </sup> . Several reports suggest that ideal cropping systems should include a combination of high <span class="trait">yield</span> potential and low negative environmental impacts, drawing on the most sustainable techniques from both organic and conventional systems <sup> <xref ref-type="bibr" rid="CR8"> 8 </xref> , <xref ref-type="bibr" rid="CR9"> 9 </xref> , <xref ref-type="bibr" rid="CR12"> 12 </xref> </sup> .
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Characterization of Soybean Genetically Modified for <span class="trait">Drought</span> Tolerance in Field Conditions</h2>
			<h3>PMID:28443101</h3>
				
					<h3>PMCID:5387084</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Fuganti-Pagliarini </surname>  <given-names> Renata </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/425718/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ferreira </surname>  <given-names> Leonardo C. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Rodrigues </surname>  <given-names> Fabiana A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Molinari </surname>  <given-names> Hugo B. C. </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/31336/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marin </surname>  <given-names> Silvana R. R. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Molinari </surname>  <given-names> Mayla D. C. </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/425729/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Marcolino-Gomes </surname>  <given-names> Juliana </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/425739/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mertz-Henning </surname>  <given-names> Liliane M. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/425946/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Farias </surname>  <given-names> Jos&#x000e9; R. B. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/426021/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> de Oliveira </surname>  <given-names> Maria C. N. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/365651/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Neumaier </surname>  <given-names> Norman </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/425749/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kanamori </surname>  <given-names> Norihito </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/366244/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Fujita </surname>  <given-names> Yasunari </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/363989/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mizoi </surname>  <given-names> Junya </given-names>  </name>  <xref ref-type="aff" rid="aff7">  <sup> 7 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nakashima </surname>  <given-names> Kazuo </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yamaguchi-Shinozaki </surname>  <given-names> Kazuko </given-names>  </name>  <xref ref-type="aff" rid="aff7">  <sup> 7 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/124985/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nepomuceno </surname>  <given-names> Alexandre L. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/364174/overview"/>  </contrib>
				<br/> 
				年份：<day> 15 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Since existing evidence demonstrates that <span class="trait">drought</span> response pathways can be both abscisic acid (ABA)-independent and -dependent, TFs also acts through these two systems that govern drought-inducible gene expression. Among these TFs, dehydration responsive element binding (DREB) proteins interact with DRE/CRT by their <span class="gene">AP2</span> DNA-binding domain, thus mediating downstream gene expression in the stress-responsive pathway. In contrast, the ABA-responsive element (ABRE) mainly mediates downstream gene expression in the ABA-signaling pathway ( <xref rid="B84" ref-type="bibr"> Yamaguchi-Shinozaki and Shinozaki, 2005 </xref> ). Insertion of the TF <italic> AtDREB1A </italic> , under the control of the stress-inducible rd29A promoter, successfully improved the <span class="trait">drought</span> tolerance responses in <italic> Arabidopsis thaliana </italic> ( <xref rid="B31" ref-type="bibr"> Gilmour et al., 1998 </xref> ; <xref rid="B38" ref-type="bibr"> Jaglo-Ottosen et al., 1998 </xref> ; <xref rid="B51" ref-type="bibr"> Liu et al., 1998 </xref> ), tobacco ( <xref rid="B42" ref-type="bibr"> Kasuga et al., 2004 </xref> ), rice ( <xref rid="B20" ref-type="bibr"> Dubouzet et al., 2003 </xref> ; <xref rid="B58" ref-type="bibr"> Oh et al., 2005 </xref> ; <xref rid="B37" ref-type="bibr"> Ito et al., 2006 </xref> ), maize ( <xref rid="B65" ref-type="bibr"> Qin et al., 2004 </xref> , <xref rid="B64" ref-type="bibr"> 2007 </xref> ), wheat ( <xref rid="B62" ref-type="bibr"> Pellegrineschi et al., 2004 </xref> ; <xref rid="B30" ref-type="bibr"> Gao et al., 2009 </xref> ) and peanut ( <xref rid="B10" ref-type="bibr"> Bhatnagar-Mathur et al., 2004 </xref> , <xref rid="B9" ref-type="bibr"> 2007 </xref> ; <xref rid="B17" ref-type="bibr"> Devi et al., 2011 </xref> ; <xref rid="B81" ref-type="bibr"> Vadez et al., 2013 </xref> ). Particularly in soybean, in experiments under greenhouse conditions, transgenic lines containing TF <italic> AtDREB1A </italic> , presented both a higher survival rate after a severe water deficit and important physiological responses to water deprivation, such as higher stomatal conductance and the maintenance of photosynthesis and photosynthetic efficiency ( <xref rid="B63" ref-type="bibr"> Polizel et al., 2011 </xref> ; <xref rid="B66" ref-type="bibr"> Rolla et al., 2013 </xref> ). Furthermore, data suggested that the higher survival rates of DREB plants are because of lower water use due to lower transpiration rates under well-watered conditions. In addition to physiological studies, molecular analysis revealed that drought-response genes were highly expressed in DREB1A plants subjected to severe water deficit ( <xref rid="B63" ref-type="bibr"> Polizel et al., 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Another member of the DREB family, <span class="gene">DREB2A</span> protein, has also been used to develop genetically modified drought-tolerant plants. In <italic> Arabidopsis </italic> , the overexpression of a constitutively active (CA) <span class="gene">DREB2A</span> form resulted in significant tolerance to <span class="trait">drought</span> and heat stress ( <xref rid="B68" ref-type="bibr"> Sakuma et al., 2006a </xref> , <xref rid="B69" ref-type="bibr"> b </xref> ). <italic> AtDREB2A </italic> homologous genes were studied in maize ( <xref rid="B64" ref-type="bibr"> Qin et al., 2007 </xref> ), rice ( <xref rid="B20" ref-type="bibr"> Dubouzet et al., 2003 </xref> ), sunflower ( <xref rid="B1" ref-type="bibr"> Almogueva et al., 2009 </xref> ), wheat ( <xref rid="B78" ref-type="bibr"> Terashima and Takumi, 2009 </xref> ) and chrysanthemum ( <xref rid="B50" ref-type="bibr"> Liu et al., 2008 </xref> ). <italic> AtDREB2A </italic> was also successfully introduced in soybean. Molecular analysis conducted under hydroponic experiments showed that transgenic plants exhibited high expression of the transgene, with roots showing the highest expression levels during water deficit. Recently, <xref rid="B54" ref-type="bibr"> Mizoi et al. (2013) </xref> identified a soybean <span class="gene">DREB2</span> gene, <italic> <span class="gene">GmDREB2A</span> </italic> , and showed that its heterologous expression in <italic> Arabidopsis </italic> induced stress-inducible genes such as <italic> RD29A, RD29B, HsfA3 </italic> , and <italic> <span class="gene">HSP70</span> </italic> and improved stress tolerance. These findings indicate that plants overexpressing <italic> AtDREB2A </italic> and <italic> DREB2Alike </italic> proteins have increased tolerance to abiotic stress, <span class="trait">drought</span> and heat, which often occur together under field conditions ( <xref rid="B26" ref-type="bibr"> Engels et al., 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Considering ABA-dependent TFs, the AREB (ABA-responsive element-binding) protein family has showed interesting results conferring <span class="trait">drought</span> tolerance. In <italic> Arabidopsis </italic> , AREB acts as the major TF under abiotic stress ( <xref rid="B46" ref-type="bibr"> Kobayashi et al., 2008 </xref> ; <xref rid="B47" ref-type="bibr"> Lee et al., 2010 </xref> ) and has been reported to regulate environmental stress responses and ABA signaling during the vegetative stage ( <xref rid="B39" ref-type="bibr"> Jakoby et al., 2002 </xref> ; <xref rid="B29" ref-type="bibr"> Fujita et al., 2005 </xref> ; <xref rid="B16" ref-type="bibr"> C&#x000f4;rrea et al., 2008 </xref> ; <xref rid="B86" ref-type="bibr"> Yoshida et al., 2010 </xref> ). The overexpression of <italic> <span class="gene">AREB1</span> </italic> in <italic> A. thaliana </italic> resulted in hypersensitivity to ABA, the induction of drought-responsive genes such as <italic> RD29B </italic> and improved water deficit tolerance. In soybean, the <italic> <span class="gene">AREB1</span> </italic> gene was introduced and overexpressing <italic> AtAREB1FL </italic> lines showed the ability to survive for a period of 5 days without water under greenhouse conditions, exhibiting no leaf damage. These lines also displayed better growth and physiological performance under water-deficit (higher relative rate of shoot length, stomatal conductance, and photosynthesis) when compared to the wild type ( <xref rid="B4" ref-type="bibr"> Barbosa et al., 2012 </xref> ). Particularly, line 1Ea2939 showed <italic> AtAREB1FL </italic> expression and a greater total number of pods and seeds and increased dry matter of seeds. The best performance of line 1Ea2939 relative to BR16 plants (wild type) might be related to the mechanisms of <span class="trait">drought</span> prevention through reduced stomatal conductance and leaf transpiration under control conditions (no water restriction). Such results suggest that the constitutive overexpression of the TF AtAREB1 leads to an improved capacity of the soybean crop to cope with <span class="trait">drought</span> with no <span class="trait">yield</span> losses ( <xref rid="B52" ref-type="bibr"> Marinho et al., 2016 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Expression level of transgenes <italic> AtDREB1A, AtDREB2CA </italic> , and <italic> AtAREB1FL </italic> was assessed using qPCR. Also, based on a search of the available literature, some genes related to <span class="trait">drought</span> responses were selected. The expression level of these genes was quantified under IRR and NIRR conditions. Genes related to <span class="trait">drought</span> response such as stomata overture/closure and osmotic adjustment, photosynthesis, metabolic and hormone pathways such as nitrogen assimilation, proteins related to <span class="trait">drought</span> such as dehydrins and heat shock proteins and water channels were chosen. Thus, selected genes were phosphatase <italic> <span class="gene">GmPP2C</span> </italic> <span class="gene">(Glyma14G195200),</span> alanine aminotransferase <italic> <span class="gene">GmAlaAT</span> </italic> <span class="gene">(Glyma01G026700,</span> and <span class="gene">Glyma07G045900),</span> &#x00394;- 1-pyrroline-5-carboxylate synthetase - <span class="gene">P5CS</span> <span class="gene">(Glyma18G034300),</span> galactinol/Gols <span class="gene">(Glyma10G145300),</span> late embryogenesis abundant/LEA18 <span class="gene">(Glyma17G164200),</span> dehydrins <span class="gene">(Glyma09G185500),</span> heat shock proteins <span class="gene">(Glyma17G072400),</span> putative soybean aquaporin pip1/UDP galactose transporter <span class="gene">(Glyma12G066800),</span> putative soybean aquaporin pip2/aquaporin transporter/glycerol uptake facilitator <span class="gene">(Glyma12G172500),</span> ribulose-1,5-bisphosphate carboxylase/oxygenase &#x02013; small chain <span class="gene">(Glyma13G046200)</span> and chlorophyll a/b binding protein &#x02013; Cab21 <span class="gene">(Glyma16G165800).</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Under <span class="trait">drought,</span> photosynthesis is among the primary processes which are down-regulated. Molecularly, in response to the stress condition, mRNA levels of the light and dark reaction genes (such as <italic> RbcS </italic> , <span class="gene">Glyma13G046200,</span> in lines 1Bb2193 and 1Ea2939, <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ) rapidly reduced, a phenomenon referred to as stress-induced mRNA decay (SMD) ( <xref rid="B60" ref-type="bibr"> Park et al., 2012 </xref> ). Physiologically, as observed, in NIRR conditions line 1Ea2939 decreased gas exchanges (data not presented), by stomatal closure (lower <italic> gs </italic> ) which probably occurs as a mechanism to keep water in the cell, which, however, did not imply losses in the use of the free CO <sub> 2 </sub> , as instantaneous water use efficiency ( <italic> A </italic> / <italic> E </italic> ) was equal to that of the other transgenic lines and WT plants and intrinsic water use efficiency ( <italic> A </italic> / <italic> g <sub> s </sub> </italic> ) was higher ( <bold> Figure <xref ref-type="fig" rid="F1"> 1 </xref> </bold> ). The explanation for stomatal closure during water stress in seed plants relies on the phytohormone ABA, which is seen as a cornerstone of stomatal function, because it has been shown to trigger responses in guard cell membrane channels and transporters that cause a reduction in guard cell turgor, thereby closing stomata ( <xref rid="B6" ref-type="bibr"> Bauer et al., 2013 </xref> ). In the field, few studies show a strong correlation between the level of ABA and <italic> gs </italic> during water deficit. In field-grown grapevine ( <italic> Vitis vinifera </italic> L. cv Cabernet Sauvignon), a correlation between ABA abundance in the xylem sap and <italic> gs </italic> strongly supported the involvement of ABA in stomatal regulation under field conditions. The different irrigation levels significantly altered the &#x003a8; <sub> leaf </sub> and <italic> gs </italic> of the vines across two crop seasons ( <xref rid="B74" ref-type="bibr"> Speirs et al., 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						In the ABA-mediated stomatal closure, the light-harvesting chlorophyll a/b-binding (LHCB) proteins can also be involved ( <xref rid="B83" ref-type="bibr"> Xu et al., 2012 </xref> ). In the present study, <span class="gene">Glyma16G165800,</span> a LHCB protein was up-regulated in lines 1Bb2193 and 1Ea2939 ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ) and ABRE <italic> cis </italic> -elements were found in the promoter region ( <bold> Additional File <xref ref-type="supplementary-material" rid="SM6"> S6 </xref> </bold> ). In higher plants, the superfamily of these proteins is composed of more than 20 members associated with photosystem I (PSI) or photosystem II (PSII). As observed here for soybean, in <italic> Arabidopsis </italic> , LHCBs positively regulate plant <span class="trait">drought</span> tolerance by functioning to positively control stomatal movement, through guard cell signaling, in response to ABA ( <xref rid="B83" ref-type="bibr"> Xu et al., 2012 </xref> ). In date palm ( <italic> Phoenix dactylifera </italic> L.) cultivar &#x0201c;Sagie&#x0201d; subjected to <span class="trait">drought,</span> LHCBs were up-regulated. The accumulation of these proteins in PSI exposed to salt and <span class="trait">drought</span> stress might represent one of the strategies to prevent or lower light stress-induced damage. It was proposed that these proteins might have a protective function within PSII under stress conditions, either by binding free chlorophyll molecules and preventing the formation of free radicals and/or by acting as sinks for excitation energy, because under stress conditions, a mobile pool of these proteins moves from PSII to PSI due to the reversible phosphorylation of these proteins by a thylakoid bound kinase ( <xref rid="B23" ref-type="bibr"> El Rabey et al., 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Considering the obtained molecular and physiological data together, it suggests that <span class="gene">GMs</span> lines 1Bb2193 and 1Ea2939 share some similarities in the <span class="trait">drought</span> response behavior, targeting more than one mechanism to cope with water deficit periods combining modulation of the gene expression profile and physiological responses, as a strategy to conserve more water and protect cells during water starvation.
					</p>
				
					<p>&nbsp;&nbsp;
						However, since <span class="trait">drought</span> response is a complex mechanism and plants, as sessile organisms, cannot avoid abiotic stresses by sheltering, they have evolved a series of mechanisms that alone or combined together overcome <span class="trait">drought</span> conditions. Thus, many other molecules are synthesized to cope with water deficit. Considering alanine aminotransferase (AlaAT) <span class="gene">(Glyma01G026700</span> and <span class="gene">Glyma07G045900)</span> genes, which are evolved in nitrogen metabolism, no differences were identified among the GM lines, which presented slight up-regulation when compared to BR16. This expression profile also occurs for proline (P5CS &#x02013; <span class="gene">Glyma18G034300).</span> In general, AlaAT plays a key role in plant metabolism by linking primary carbon metabolism with the synthesis of amino acids. If considered that under <span class="trait">drought,</span> a decrease in protein synthesis, protein levels and activity of enzymes occurs, thus, in GM lines, AlaATs might be ensuring amino acid and protein synthesis during <span class="trait">drought,</span> which explains a higher proline accumulation in transgenic lines when compared to BR16. This amino acid, under stress conditions, as well as acting as an excellent osmolyte, plays three major roles, as a metal chelator, an antioxidative defense molecule and a signaling molecule. Overproduction of proline in plants imparts stress tolerance by maintaining cell turgor or osmotic balance; stabilizing membranes to prevent electrolyte leakage; and bringing concentrations of reactive oxygen species (ROS) within normal ranges, thus preventing oxidative burst in plants ( <xref rid="B35" ref-type="bibr"> Hayat et al., 2012 </xref> ). In wheat, <span class="trait">drought</span> stress tolerance at a cellular level was associated with the ability to accumulate proline and high water level conservation ( <xref rid="B76" ref-type="bibr"> Sultan et al., 2012 </xref> ). In sugarcane, proline accumulation in transgenic plants under water-deficit stress acts in the cytoplasmatic osmotic adjustment but also as a component of the antioxidative defense system ( <xref rid="B55" ref-type="bibr"> Molinari et al., 2007 </xref> ). In soybean, proline content increased in the leaves and nodules of plants subjected to water deficit during the flowering stage ( <xref rid="B73" ref-type="bibr"> Silvente et al., 2012 </xref> ). Also, a positive and significant correlation among the activity of antioxidant enzymes, ABA and proline content with seed and oil <span class="trait">yield</span> in water deficit stress was observed ( <xref rid="B53" ref-type="bibr"> Masoumi et al., 2011 </xref> ). Higher accumulation of proline was still associated with less injury and a greater amount of water retention in a stress-tolerant soybean genotype ( <xref rid="B3" ref-type="bibr"> Angra et al., 2010 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Dehydrins (DHNs are the group II, D11 family of LEAs &#x02013; Late Embryogenesis Abundant) <span class="gene">(Glyma09G185500)</span> and heat shock proteins (HSPs) <span class="gene">(Glyma17G072400)</span> were also induced under <span class="trait">drought</span> treatment, in line <italic> AtAREB1FL </italic> 1Ea2939, which presented the higher expression among transgenic and BR16 plants ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ). Dehydrins are exclusively found in plants and accumulate in the late stages of embryogenesis, when water content in seeds declines, or in response to various stressors. The presence of these proteins has been observed in several independent studies on <span class="trait">drought</span> and salinity stresses as well as on cold acclimation and after ABA treatment ( <xref rid="B67" ref-type="bibr"> Rorat et al., 2006 </xref> ; <xref rid="B79" ref-type="bibr"> Tripepi et al., 2011 </xref> ). DHNs play critical roles in desiccation tolerance by capturing water, stabilizing and protecting the structure and function of proteins and membranes, as well as acting as molecular chaperons (as heat shock proteins) and hydrophilic solutes to protect cells from the damage of water stress ( <xref rid="B32" ref-type="bibr"> Hand et al., 2011 </xref> ). In <italic> Olea europaea </italic> L. Subsp. <italic> europaea </italic> , var. <italic> sylvestris </italic> , the wild plant of olive, the expression <italic> OesDHN </italic> was induced under mild <span class="trait">drought</span> stress. In addition, <italic> Arabidopsis </italic> transgenic plants showed a better tolerance to osmotic stress suggesting that <italic> OesDHN </italic> expression is induced by <span class="trait">drought</span> stress and is able to confer osmotic stress tolerance ( <xref rid="B14" ref-type="bibr"> Chiappetta et al., 2015 </xref> ). In grapevine, <italic> DHN1 </italic> and <italic> DHN2 </italic> genes were induced by <span class="trait">drought,</span> cold, heat, embryogenesis, as well as the application of ABA, salicylic acid (SA), and methyl jasmonate (MeJA) and a higher number of putative ABRE <italic> cis </italic> -elements was located in <italic> DHN1 </italic> and <italic> DHN2 </italic> promoters ( <xref rid="B85" ref-type="bibr"> Yang et al., 2012 </xref> ), which were also identified here in soybean <span class="gene">(Glyma09G185500,</span> <bold> Additional File <xref ref-type="supplementary-material" rid="SM5"> S5 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						As observed here, lines 1Bb2193 and 1Ea2939, containing ABA-independent and -dependent TFs (DREB2A and AREB1), respectively, showed a similar expression pattern under <span class="trait">drought</span> conditions for some endogenous genes ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ). Interestingly, increasing evidence shows that DRE/CRT can act as a coupling element of the ABRE <italic> cis </italic> -element to regulate downstream gene expression ( <xref rid="B56" ref-type="bibr"> Narusaka et al., 2003 </xref> ). Thus, there exists a comprehensive connection between stress-responsive and ABA-signaling pathways ( <xref rid="B70" ref-type="bibr"> Shinozaki and Yamaguchi-Shinozaki, 2000 </xref> ; <xref rid="B72" ref-type="bibr"> Shinozaki et al., 2003 </xref> ). A few of the DREB-type TFs were found to be involved in the ABA-dependent pathway ( <xref rid="B21" ref-type="bibr"> Egawa et al., 2006 </xref> ). In maize, ZmABI4, a DREB protein that shows ABA-induced expression, binds to CE1 and acts as a coupling element of the ABRE ( <xref rid="B57" ref-type="bibr"> Niu et al., 2002 </xref> ). Consistently, overexpression of DREB1D/CBF4, which is an ABA-responsive gene of <italic> Arabidopsis </italic> , activates the expression of <span class="trait">drought</span> and cold-related downstream genes that contain the DRE/CRT <italic> cis </italic> -element ( <xref rid="B45" ref-type="bibr"> Knight et al., 2004 </xref> ). In rice, three ABRE elements were identified in the promoter region of <italic> ARAG1 </italic> gene, which encodes a DREB-like protein containing the characterized <span class="gene">AP2</span> DNA binding domain ( <xref rid="B87" ref-type="bibr"> Zhao et al., 2010 </xref> ). In <italic> Arabidopsis </italic> , evidence shows that AREB/ABF proteins physically interact with DREB/CBFs including DREB1A, DREB2A, and DREB2C ( <xref rid="B47" ref-type="bibr"> Lee et al., 2010 </xref> ). In <italic> Setaria italica </italic> , the transcript level of <italic> SiARDP </italic> , an abscisic-responsive DREB-binding protein, increased not only after <span class="trait">drought,</span> high salt and low temperature stresses, but also after an ABA treatment in foxtail millet seedlings ( <xref rid="B49" ref-type="bibr"> Li et al., 2014 </xref> ). Furthermore, transient-expression analyses coupled with ChIP (Chromatin Immunoprecipitation) assays have shown AREB/ABFs such as AREB1, AREB2, and ABF3 can bind to the promoter of <span class="gene">DREB2A</span> and thereby induce them in an ABRE-dependent manner ( <xref rid="B43" ref-type="bibr"> Kim et al., 2011 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						As a drought-sensitive genotype, WT BR16 also targets molecular mechanisms in response to water deficit. Thus, expression profile showed that LEA protein <span class="gene">(Glyma17G164200)</span> was highly induced ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ). In the conventional cultivar, this protein, acting as molecular chaperone, might be associated with the osmotic adjustment (OA) mechanism, as galactinol ( <italic> Gols </italic> &#x02013; <span class="gene">Glyma10G145300)</span> and aquaporin <span class="gene">(Glyma12G172500)</span> also presented higher expression levels. In the OA, the accumulation of a variety of organic and inorganic substances increases the concentration in the cytochylema, reducing the osmotic potential and improving cell water retention in response to water stress. In overexpressing <italic> Arabidopsis </italic> , a wheat ( <italic> Triticum durum </italic> ) group 2 LEA protein (DHN-5) improved tolerance to salt and <span class="trait">drought</span> stress through osmotic adjustment. The water potential was more negative in transgenic than in wild type plants and, in addition, these plants have lower water loss rate under water stress ( <xref rid="B12" ref-type="bibr"> Brini et al., 2007 </xref> ). In <italic> Arabidopsis </italic> , sugar analysis showed that <span class="trait">drought,</span> high salinity and cold treated plants accumulate a large amount of raffinose and galactinol, functioning as osmoprotectants in drought-stress tolerance ( <xref rid="B77" ref-type="bibr"> Taji et al., 2002 </xref> ). In poplar ( <italic> Populus </italic> spp.), the osmolytes raffinose and galactinol exhibited increased abundance under <span class="trait">drought</span> stress ( <xref rid="B5" ref-type="bibr"> Barchet et al., 2013 </xref> ). Also, in <italic> Coffea arabica </italic> and <italic> C. canephora, GolS </italic> genes were highly induced under water deficit periods ( <xref rid="B18" ref-type="bibr"> Dos Santos et al., 2011 </xref> , <xref rid="B19" ref-type="bibr"> 2015 </xref> ). It must be emphasized that, although BR16 (WT) plants seemed to have presented OA mechanism under <span class="trait">drought</span> stress, such a response was not more efficient than the mechanisms shown by the transgenic lines, based on physiological, growth and agronomical results ( <bold> Figures <xref ref-type="fig" rid="F1"> 1 </xref> </bold> &#x02013; <bold> <xref ref-type="fig" rid="F3"> 3 </xref> </bold> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Besides these molecules, aquaporins (AQP), which were differentially expressed in BR16 <span class="gene">(Glyma12G066800</span> and Gyma12G172500) ( <bold> Figure <xref ref-type="fig" rid="F5"> 5 </xref> </bold> ), have also been demonstrated to have crucial roles in OA in plant cells. AQPs participate in the rapid transmembrane water flow and when plants are subjected to <span class="trait">drought</span> or salt conditions, increased transport of water across membranes is crucial to maintain a healthy physiological status. In banana ( <italic> Musa acuminata </italic> L.), the aquaporin gene <italic> MaPIP1;1 </italic> was induced in leaves and roots after salinity stress and simulated <span class="trait">drought</span> treatments, and overexpressing <italic> Arabidopsis </italic> displayed better growth, more green leaves, higher survival rates and lower water loss rate compared to WT under <span class="trait">drought</span> conditions. Transgenic lines also experimented less lipid peroxidation and membrane injury, and improved osmotic adjustment under <span class="trait">drought</span> treatment ( <xref rid="B82" ref-type="bibr"> Xu et al., 2014 </xref> ). Once more, these data suggest that soybean conventional cultivar BR16 might be targeting, mainly, the OA mechanism to cope with water deficit, as GM line 1Ab58.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fine Mapping of a <span class="trait">Resistance</span> Gene <italic> RpsHN </italic> that Controls <italic> Phytophthora sojae </italic> Using Recombinant Inbred Lines and Secondary Populations</h2>
			<h3>PMID:28443124</h3>
				
					<h3>PMCID:5387331</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Niu </surname>  <given-names> Jingping </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/428307/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Guo </surname>  <given-names> Na </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Sun </surname>  <given-names> Jutao </given-names>  </name>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/402243/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Lihong </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cao </surname>  <given-names> Yongce </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Shuguang </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Jianli </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Jinming </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Tuanjie </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/345840/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xing </surname>  <given-names> Han </given-names>  </name>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/401939/overview"/>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To our knowledge, 22 race-specific <span class="trait">resistance</span> soybean cultivars containing single genes have been identified and reported: L88-8470 ( <italic> Rps1a </italic> ), L77-1863 ( <italic> Rps1b </italic> ), L75-3735 ( <italic> Rps1c </italic> ), L93-3312 ( <italic> Rps1d </italic> ), L77-1794 ( <italic> Rps1k </italic> ), L76-1988 ( <italic> Rps2 </italic> ), L83-570 ( <italic> Rps3a </italic> ), L91-8347 ( <italic> Rps3b </italic> ), L92-7857 ( <italic> Rps3c </italic> ), L85-2352 ( <italic> Rps4 </italic> ), L85-3059 ( <italic> Rps5 </italic> ), L89-1581 ( <italic> Rps6 </italic> ), L93-3258 ( <italic> Rps7 </italic> ), PI 399073 ( <italic> Rps8 </italic> ), Ludou4 ( <italic> Rps9 </italic> ), Wandou15 ( <italic> Rps10 </italic> ), PI 594527 ( <italic> Rps11 </italic> ), PI 567139B ( <italic> RpsUN1 </italic> and <italic> RpsUN2 </italic> ), Yudou25 ( <italic> RpsYu25 </italic> ), Yudou29 ( <italic> RpsYD29 </italic> ), Waseshiroge (unnamed <italic> Rps </italic> gene), and Nannong 10-1 ( <italic> RpsJS </italic> ) ( <xref rid="B29" ref-type="bibr"> Sugimoto et al., 2011 </xref> , <xref rid="B28" ref-type="bibr"> 2012 </xref> ; <xref rid="B31" ref-type="bibr"> Sun et al., 2011 </xref> , <xref rid="B30" ref-type="bibr"> 2014 </xref> ; <xref rid="B38" ref-type="bibr"> Wu et al., 2011 </xref> ; <xref rid="B41" ref-type="bibr"> Zhang et al., 2013a </xref> , <xref rid="B42" ref-type="bibr"> b </xref> ; <xref rid="B24" ref-type="bibr"> Ping et al., 2015 </xref> ; <xref rid="B18" ref-type="bibr"> Li et al., 2016 </xref> ). Twenty-three genes/alleles identified in the soybean cultivars listed above were located on six chromosomes. The genes/alleles <italic> Rps1 </italic> (including five alleles <italic> Rps1-a, Rps1-b, Rps1-c, Rps1-d </italic> , and <italic> Rps1-k </italic> ), <italic> Rps7, Rps9, RpsUN1, RpsYu25 </italic> and an unnamed <italic> Rps </italic> gene ( <italic> Rps1? </italic> ) on chromosome 3, <italic> Rps3 </italic> (including three alleles <italic> Rps3a, Rps3b, Rps3c </italic> ) and <italic> Rps8 </italic> on chromosome 13, <italic> Rps2 </italic> and <italic> RpsUN2 </italic> on chromosome 16, and <italic> Rps4, Rps5 </italic> and <italic> Rps6 </italic> on chromosome 18 were detected by linkage analysis and genetic mapping ( <xref rid="B16" ref-type="bibr"> Kilen et al., 1974 </xref> ; <xref rid="B23" ref-type="bibr"> Mueller et al., 1978 </xref> ; <xref rid="B3" ref-type="bibr"> Athow et al., 1980 </xref> ; <xref rid="B25" ref-type="bibr"> Ploper et al., 1985 </xref> ; <xref rid="B6" ref-type="bibr"> Buzzell and Anderson, 1992 </xref> ; <xref rid="B7" ref-type="bibr"> Cregan et al., 1999 </xref> ; <xref rid="B13" ref-type="bibr"> Gordon et al., 2006 </xref> ; <xref rid="B29" ref-type="bibr"> Sugimoto et al., 2011 </xref> ; <xref rid="B31" ref-type="bibr"> Sun et al., 2011 </xref> ; <xref rid="B38" ref-type="bibr"> Wu et al., 2011 </xref> ). <italic> RpsYD29 </italic> was mapped to a 204.8-kb region on chromosome 3, and two nucleotide-binding site and leucine-rich repeat (NBS-LRR) type genes <italic> <span class="gene">Glyma03g04030.1</span> </italic> and <italic> <span class="gene">Glyma03g04080.1</span> </italic> were identified ( <xref rid="B42" ref-type="bibr"> Zhang et al., 2013b </xref> ). <italic> Rps1k </italic> has an NBS-LRR structure that is typical of <span class="trait">resistance</span> proteins. However, the physical location of <italic> Rps1k </italic> is unknown in the reference genome of &#x02018;Williams 82&#x02019; ( <xref rid="B12" ref-type="bibr"> Gao et al., 2005 </xref> ; <xref rid="B11" ref-type="bibr"> Gao and Bhattacharyya, 2008 </xref> ). <italic> RpsJS </italic> , a fine mapping gene located in a 138.9-kb region with 14 candidate genes on chromosome 18, and three genes <italic> <span class="gene">Glyma18g51930,</span> <span class="gene">Glyma18g51950</span> </italic> and <italic> <span class="gene">Glyma18g51960</span> </italic> were characterized as NBS-LRR type genes ( <xref rid="B30" ref-type="bibr"> Sun et al., 2014 </xref> ). <italic> Rps11 </italic> mapped to a 225.3-kb region on chromosome 7, and <italic> Rps10 </italic> mapped to a 311-kb region on chromosome 17 ( <xref rid="B41" ref-type="bibr"> Zhang et al., 2013a </xref> ; <xref rid="B24" ref-type="bibr"> Ping et al., 2015 </xref> ). Furthermore, the <italic> Rps10 </italic> mapping region contained two candidate genes, <italic> <span class="gene">Glyma17g28950.1</span> </italic> and <italic> <span class="gene">Glyma17g28970.1</span> </italic> , annotated as serine/threonine (Ser/Thr) protein kinases.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:An integrated RNAseq- <sup> 1 </sup> H NMR metabolomics approach to understand soybean primary metabolism regulation in response to Rhizoctonia foliar blight disease</h2>
			<h3>PMID:28449662</h3>
				
					<h3>PMCID:5408482</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Copley </surname>  <given-names> Tanya R. </given-names>  </name>  <address>  <email> tanya.copley@mail.mcgill.ca </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Aliferis </surname>  <given-names> Konstantinos A. </given-names>  </name>  <address>  <email> konstantinos.aliferis@aua.gr </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kliebenstein </surname>  <given-names> Daniel J. </given-names>  </name>  <address>  <email> kliebenstein@ucdavis.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Jabaji </surname>  <given-names> Suha H. </given-names>  </name>  <address>  <email> suha.jabaji@mcgill.ca </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1936 8649 </institution-id>  <institution-id institution-id-type="GRID"> grid.14709.3b </institution-id>  <institution> Plant Science Department,  </institution>  <institution> McGill University,  </institution>  </institution-wrap> Ste-Anne-de-Bellevue, Quebec, H9X 3V9 Canada  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0794 1186 </institution-id>  <institution-id institution-id-type="GRID"> grid.10985.35 </institution-id>  <institution> Department of Plant Science,  </institution>  <institution> Laboratory of Pesticide Science, Agricultural University of Athens,  </institution>  </institution-wrap> Iera Odos 75, 118 55 Athens, Greece  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1936 9684 </institution-id>  <institution-id institution-id-type="GRID"> grid.27860.3b </institution-id>  <institution> Department of Plant Sciences,  </institution>  <institution> University of California,  </institution>  </institution-wrap> Davis, 95616 USA  </aff>
				<br/> 
				年份：<day> 27 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Rhizoctonia foliar blight (RFB) caused by <italic> Rhizoctonia solani </italic> anastomosis group (AG) 1 intraspecific group IA is a serious disease that causes rapid and severe destruction of soybean ( <italic> Glycine max </italic> L. Merr) [ <xref ref-type="bibr" rid="CR1"> 1 </xref> , <xref ref-type="bibr" rid="CR2"> 2 </xref> ]. Outbreaks of AG1-IA on soybean in Brazil, the southern states of the U.S.A. and China have caused <span class="trait">yield</span> losses of 30&#x02013;60% [ <xref ref-type="bibr" rid="CR3"> 3 </xref> , <xref ref-type="bibr" rid="CR4"> 4 </xref> ]. Protection against RFB is difficult as the use of less-susceptible cultivars is limited due to a lack of availability [ <xref ref-type="bibr" rid="CR1"> 1 </xref> ] resulting in the use of chemical pesticides. Because of the impact of RFB on agriculture, it is important to identify factors that regulate plant <span class="trait">resistance;</span> however, no studies have been published examining the molecular responses of soybean due to <italic> R. solani </italic> AG1-IA infection. Understanding the effect of <italic> R. solani </italic> <span class="gene">AG1</span> on soybean defense pathways and their regulation will greatly assist breeding efforts towards the development of cultivars with improved <span class="trait">resistance</span> to RFB by applying biomarker-assisted selection.
					</p>
				
					<p>&nbsp;&nbsp;
						Phytopathogen infections lead to changes in secondary metabolism (i.e., metabolism not directly involved in maintaining growth, development or reproduction) as well as changes in primary metabolism (i.e., metabolism and its products required in all cells to maintain normal functions) that affect growth and development of the plant. While the regulation of defense responses has been intensively studied [ <xref ref-type="bibr" rid="CR16"> 16 </xref> , <xref ref-type="bibr" rid="CR17"> 17 </xref> ], less is known about the effects of necrotrophic pathogen infection, such as <italic> R. solani </italic> , on primary metabolism. We have recently demonstrated the effectiveness of metabolomics for the analysis of plant primary metabolism during Rhizoctonia root rot and stem canker diseases of soybean hypocotyls and potato sprouts [ <xref ref-type="bibr" rid="CR18"> 18 </xref> , <xref ref-type="bibr" rid="CR19"> 19 </xref> ] with results indicating that host plants reorganize their primary metabolism to mount defense mechanisms, while, pathogens might simultaneously manipulate plant metabolism to promote infection to support replication and spreading within the plant [ <xref ref-type="bibr" rid="CR20"> 20 </xref> , <xref ref-type="bibr" rid="CR21"> 21 </xref> ]. The increased demands of redirection of primary metabolites involved in photosynthesis, carbohydrate and amino acid metaboism, glycolysis and TCA cycles suggest that reprogramming of vital functions in infected tissue occurs and is a necessary defense requirement [ <xref ref-type="bibr" rid="CR22"> 22 </xref> &#x02013; <xref ref-type="bibr" rid="CR25"> 25 </xref> ]. Taken together, examining alterations in soybean primary metabolism to early and late infection stages of the foliar pathogen <italic> R. solani </italic> <span class="gene">AG1</span> can provide insights on how to develop soybean lines with more <span class="trait">resistance</span> to foliar blight disease.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Photosynthesis, Light Use Efficiency, and <span class="trait">Yield</span> of Reduced-Chlorophyll Soybean Mutants in Field Conditions</h2>
			<h3>PMID:28458677</h3>
				
					<h3>PMCID:5394119</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Slattery </surname>  <given-names> Rebecca A. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/387216/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> VanLoocke </surname>  <given-names> Andy </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Bernacchi </surname>  <given-names> Carl J. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/312310/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Xin-Guang </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/13067/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ort </surname>  <given-names> Donald R. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/336005/overview"/>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 3 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Increasing the <span class="trait">yield</span> potential ( <italic> Y </italic> <sub> p </sub> ) of important agronomic crops is imperative for meeting predicted future production needs. <italic> Y </italic> <sub> p </sub> is the maximum possible regional <span class="trait">yield</span> for a given crop in the absence of biotic and abiotic stresses ( <xref rid="B23" ref-type="bibr"> Evans and Fischer, 1999 </xref> ), but as radiation, water, nutrients, etc., become limiting or there is pest/pathogen pressure, realized yields decrease, resulting in a <span class="trait">yield</span> <span class="gene">gap</span> ( <xref rid="B44" ref-type="bibr"> Lobell et al., 2009 </xref> ). <italic> Y </italic> <sub> p </sub> for a given crop during a growing season is the product of several components: the incident solar radiation across the growing season ( <italic> S </italic> <sub> t </sub> ), the proportion of <italic> S </italic> <sub> t </sub> that is photosynthetically active radiation (PAR; estimated as 0.487), the radiation interception efficiency (&#x1d700; <sub> i </sub> ), the conversion efficiency of intercepted radiation into biomass (&#x1d700; <sub> c </sub> ), and the partition efficiency of biomass into harvestable product (&#x1d700; <sub> p </sub> ; modified from <xref rid="B53" ref-type="bibr"> Monteith, 1977 </xref> ). <italic> S </italic> <sub> t </sub> and the proportion that is PAR vary but are largely predetermined by growing season length and location, although there can be substantial interannual variability in <italic> S </italic> <sub> t </sub> at a given location ( <xref rid="B51" ref-type="bibr"> Monteith, 1965 </xref> , <xref rid="B52" ref-type="bibr"> 1972 </xref> ). Of the three efficiencies, plant breeders have already pushed &#x1d700; <sub> i </sub> and &#x1d700; <sub> p </sub> near their theoretical upper limits in highly productive crops in the best years ( <xref rid="B22" ref-type="bibr"> Evans, 1993 </xref> ; <xref rid="B32" ref-type="bibr"> Hay, 1995 </xref> ; <xref rid="B67" ref-type="bibr"> Sinclair, 1998 </xref> ). Breeding in soybean ( <italic> Glycine max </italic> Merr.) has achieved &#x1d700; <sub> i </sub> season averages of approximately 0.60&#x02013;0.75 ( <xref rid="B38" ref-type="bibr"> Koester et al., 2014 </xref> ) with peak midseason &#x1d700; <sub> i </sub> of &#x0003e;0.90 in modern cultivars ( <xref rid="B12" ref-type="bibr"> Dermody et al., 2008 </xref> ; <xref rid="B38" ref-type="bibr"> Koester et al., 2014 </xref> ). &#x1d700; <sub> p </sub> has reached values of 0.60 for soybean ( <xref rid="B12" ref-type="bibr"> Dermody et al., 2008 </xref> ; <xref rid="B38" ref-type="bibr"> Koester et al., 2014 </xref> ), which is at or near the estimated theoretical maximum of &#x0223c;0.55&#x02013;0.67 for major food crops ( <xref rid="B5" ref-type="bibr"> Austin et al., 1980 </xref> ; <xref rid="B9" ref-type="bibr"> Bugbee and Monje, 1992 </xref> ; <xref rid="B36" ref-type="bibr"> Khush, 1995 </xref> ; <xref rid="B71" ref-type="bibr"> Smil, 1999 </xref> ; <xref rid="B33" ref-type="bibr"> Hay and Porter, 2006 </xref> ; <xref rid="B64" ref-type="bibr"> Prasad et al., 2006 </xref> ). These advancements in &#x1d700; <sub> i </sub> and &#x1d700; <sub> p </sub> leave only modest potential for further improvement of <italic> Y </italic> <sub> p </sub> ( <xref rid="B78" ref-type="bibr"> Zhu et al., 2010 </xref> ). However, &#x1d700; <sub> c </sub> operates substantially below the theoretical maxima for <italic> C </italic> <sub> 3 </sub> (0.094) and <italic> C </italic> <sub> 4 </sub> (0.123; <xref rid="B78" ref-type="bibr"> Zhu et al., 2010 </xref> ) crop canopies and therefore limits <span class="trait">yield</span> potential ( <xref rid="B77" ref-type="bibr"> Zhu et al., 2008 </xref> , <xref rid="B78" ref-type="bibr"> 2010 </xref> ), especially in soybean where the maximum realized &#x1d700; <sub> c </sub> (0.028) is estimated at less than a third of the <italic> C </italic> <sub> 3 </sub> theoretical potential ( <xref rid="B70" ref-type="bibr"> Slattery and Ort, 2015 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						A decline in photosynthetic efficiency and capacity in <italic> y9y9 </italic> was correlated with severe reductions in chl. Despite greater <italic> g </italic> <sub> s </sub> and lower SLW, which was evident in <italic> Y11y11 </italic> as well, the <italic> y9y9 </italic> mutant displayed reduced <italic> A </italic> <sub> <span class="gene">sat</span> </sub> and light use efficiency compared to the WT ( <bold> Figures <xref ref-type="fig" rid="F4"> 4A </xref> , <xref ref-type="fig" rid="F5"> 5A </xref> , <xref ref-type="fig" rid="F6"> 6 </xref> , <xref ref-type="fig" rid="F7"> 7A </xref> </bold> and <bold> Table <xref ref-type="table" rid="T3"> 3 </xref> </bold> ) when chl was reduced by more than 85% compared to the WT ( <bold> Figure <xref ref-type="fig" rid="F2"> 2C </xref> </bold> ). Similar responses to severe reductions in chl content (&#x0003e;80%) seen in <italic> y9y9 </italic> have been reported in <italic> Y11y11 </italic> when grown in controlled environment chambers at lower than field light levels ( <xref rid="B75" ref-type="bibr"> Xu et al., 1993 </xref> ; <xref rid="B69" ref-type="bibr"> Slattery et al., 2016 </xref> ). These two mutants are characterized by higher chl <italic> a </italic> / <italic> b </italic> ratios due to greatly reduced PSII-associated light harvesting complexes (LHCII) and to some extent PSI-associated light harvesting complexes (LHCI; <xref rid="B28" ref-type="bibr"> Ghirardi and Melis, 1988 </xref> ). Theoretically, severe deficiencies in LHCII could have negative effects at the leaf level, including greater levels of photoinhibition ( <xref rid="B41" ref-type="bibr"> Leverenz et al., 1992 </xref> ; <xref rid="B75" ref-type="bibr"> Xu et al., 1993 </xref> ), reduced photoprotective capacity ( <xref rid="B57" ref-type="bibr"> Ort, 2001 </xref> ), and lowered connectivity among PSII centers and therefore lower quantum <span class="trait">yield</span> at low light ( <xref rid="B2" ref-type="bibr"> Allen and Forsberg, 2001 </xref> ). In this study, extreme reductions in chl coupled with large reductions in carotenoids (&#x0003e;40%; <bold> Table <xref ref-type="table" rid="T2"> 2 </xref> </bold> ) were correlated with impaired photoprotective mechanisms such as <italic> NPQ </italic> ( <bold> Figure <xref ref-type="fig" rid="F6"> 6 </xref> </bold> ). &#x003d5; <italic> CO </italic> <sub> 2 </sub> was also significantly reduced in <italic> y9y9 </italic> ( <bold> Tables <xref ref-type="table" rid="T2"> 2 </xref> , <xref ref-type="table" rid="T3"> 3 </xref> </bold> ). Thus, these data suggest improving leaf photosynthetic efficiency through lowering chl <italic> a </italic> and chl <italic> b </italic> concentrations proportionally, but not to an extent that impairs photosynthetic and photoprotective capacity.
					</p>
				
					<p>&nbsp;&nbsp;
						Reduced pigmentation was hypothesized to benefit leaf water use along with canopy &#x1d700; <sub> c </sub> , but the greater <italic> g </italic> <sub> s </sub> that accompanied chl reductions in these mutants likely prevented these effects. Increased albedo was associated with lower <italic> T </italic> <sub> leaf </sub> in the mutants as expected ( <bold> Figures <xref ref-type="fig" rid="F5"> 5G,H </xref> </bold> ), but cooler <italic> T </italic> <sub> leaf </sub> may have been, at least in part, an effect of greater <italic> g </italic> <sub> s </sub> and therefore cooling via transpiration in the mutants ( <bold> Figures <xref ref-type="fig" rid="F5"> 5E,F </xref> </bold> ). Greater <italic> g </italic> <sub> s </sub> was most likely a result of the mutations causing chl deficiency. A recent study identified a mutation in the magnesium chelatase subunit-I gene (CHLI) as the cause of the light-green phenotype of <italic> Y11y11 </italic> ( <xref rid="B10" ref-type="bibr"> Campbell et al., 2015 </xref> ). In <italic> A. thaliana </italic> , both <span class="gene">CHLI</span> and <span class="gene">CHLH</span> mutants have demonstrated ABA-insensitivity ( <xref rid="B73" ref-type="bibr"> Tsuzuki et al., 2011 </xref> ; <xref rid="B15" ref-type="bibr"> Du et al., 2012 </xref> ) that is independent of chl biosynthesis ( <xref rid="B15" ref-type="bibr"> Du et al., 2012 </xref> ). Whether the greater <italic> g </italic> <sub> s </sub> in <italic> y9y9 </italic> that occurred in this study is also due to ABA-insensitivity is unknown since the mutation has not yet been identified, but greater <italic> g </italic> <sub> s </sub> has been reported in <italic> y9y9 </italic> compared to WT regardless of water stress ( <xref rid="B47" ref-type="bibr"> Luquez et al., 1997 </xref> ). The increase in <italic> g </italic> <sub> s </sub> resulted in lower mutant iWUE ( <bold> Figures <xref ref-type="fig" rid="F5"> 5C,D </xref> </bold> ) and integrated canopy WUE (Supporting Information Figure <xref ref-type="supplementary-material" rid="SM1"> S4 </xref> ). Lower canopy WUE in the mutants would likely result in greater soil moisture depletion ( <xref rid="B35" ref-type="bibr"> Hussain et al., 2013 </xref> ) and therefore greater susceptibility to <span class="trait">drought,</span> which occurred during both of the growing seasons ( <bold> Figures <xref ref-type="fig" rid="F1"> 1C,D </xref> </bold> ). <span class="trait">Drought</span> stress significantly reduces &#x1d700; <sub> c </sub> ( <xref rid="B68" ref-type="bibr"> Slattery et al., 2013 </xref> ), and the greater susceptibility to <span class="trait">drought</span> in the mutants may have dampened even small benefits of reduced chl on &#x1d700; <sub> c </sub> and <span class="trait">yield.</span> In the previous study where <italic> A </italic> <sub> can </sub> and yields were greater in <italic> Y11y11 </italic> compared to WT, water was not limiting due to irrigation ( <xref rid="B61" ref-type="bibr"> Pettigrew et al., 1989 </xref> ), suggesting that these specific mutations causing chl deficiency may limit productivity in times of even moderate water stress. Although rice chl mutants with greater <italic> g </italic> <sub> s </sub> resulted in higher yields when grown at a higher planting density, the authors do not report any water limitations. Further assessments of chl-deficient crops may need to incorporate tests of efficiency and productivity in non-optimal field conditions while maintaining or improving WUE.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Elevated CO <sub> 2 </sub> alters distribution of nodal leaf area and enhances nitrogen uptake contributing to <span class="trait">yield</span> increase of soybean cultivars grown in Mollisols</h2>
			<h3>PMID:28459840</h3>
				
					<h3>PMCID:5411100</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Jin </surname>  <given-names> Jian </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Yansheng </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xiaobing </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Guanghua </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Caixian </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Zhenhua </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xiaojuan </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Herbert </surname>  <given-names> Stephen J. </given-names>  </name>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 16 </day>  <month> 4 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						At the R8 stage (full maturity), shoots were removed at the soil surface and shoot biomass was recorded. Detailed data of <span class="trait">yield</span> components including pod number, seed number and seed dry weight were recorded according to node position and corresponding branches. Shoots were then separated into stems plus pod walls, and seeds. All plant samples were dried at 70&#x000b0;C for 72 h and then finely ground (&#x02264;0.2 mm). The concentration of N in plant tissues was determined using an Elementar CNS analyser (Vario EL III, Elementar Analysensysteme <span class="gene">GmbH,</span> Germany).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Fungal Diversity in Field Mold-Damaged Soybean Fruits and Pathogenicity Identification Based on High-Throughput rDNA Sequencing</h2>
			<h3>PMID:28515718</h3>
				
					<h3>PMCID:5413577</h3>
					<h4>期刊:Frontiers in Microbiology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Jiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/365608/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Deng </surname>  <given-names> Jun-cai </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/426584/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Cai-qiong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> &#x02020; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/394807/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Ni </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chang </surname>  <given-names> Xiao-li </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Feng </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Wei-guo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/374075/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xiao-chun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yong </surname>  <given-names> Tai-wen </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Du </surname>  <given-names> Jun-bo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/362959/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shu </surname>  <given-names> Kai </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/320922/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Wen-yu </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 18 </day>  <month> 4 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Grain fungi can be divided into two large categories, field and storage molds, based on the location of the outbreak ( <xref rid="B19" ref-type="bibr"> Liu et al., 2016 </xref> ). Storage fungi can grow at low water activity, which occurs readily under a high-temperature (25&#x02013;30&#x000b0;C) and humid environment. In contrast, FM infection typically occurs in the field when the environment is abnormally cold (temperature 13&#x02013;21&#x000b0;C) and humid (humidity 85&#x02013;100%). The ability of fungi to germinate, grow and sporulate on grain depends on the availability of water and temperature as well as on the intergranular gas composition ( <xref rid="B29" ref-type="bibr"> Zhang et al., 2014 </xref> ), and these factors influence grain spoilage by fungi. Different environmental conditions result in different fungal community compositions. For instance, storage mold on <italic> Zea mays </italic> is caused by <italic> A. flavus </italic> and <italic> Penicillium </italic> spp. ( <xref rid="B23" ref-type="bibr"> Pinto and Fonseca, 2006 </xref> ). To date, there have been many in-depth studies on post-harvest fungal ecology, especially with regards to the impact of fungal growth on stored grain. <italic> F. culmorum, A. ochraceus, Penicillium verrucosum </italic> , and <italic> Fusarium </italic> section <italic> Liseola </italic> , as well as their released fungal toxins, are sources of contaminants in stored cereal grains ( <xref rid="B20" ref-type="bibr"> Magan et al., 2003 </xref> ). Abnormally cold and wet climatic conditions occasionally result in FM outbreaks in the field before soybean harvest in Southwest China. FM reduces soybean <span class="trait">yield</span> and quality and has become the main obstacle to soybean production in Southwest China ( <xref rid="B8" ref-type="bibr"> Deng et al., 2015b </xref> ). However, there is scant information to date regarding FM on soybean grains, and the pathogenic fungi have not been elucidated. The current study investigated the fungal diversity among FM-infected soybean grains as well as identifying the pathogenic fungal, filling an important <span class="gene">gap</span> in this research.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Interactions between temperature and <span class="trait">drought</span> in global and regional crop <span class="trait">yield</span> variability during 1961-2014</h2>
			<h3>PMID:28552938</h3>
				
					<h3>PMCID:5446168</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Matiu </surname>  <given-names> Michael </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ankerst </surname>  <given-names> Donna P. </given-names>  </name>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Menzel </surname>  <given-names> Annette </given-names>  </name>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 11 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To quantitatively assess the potential impact of measurement error in FAO <span class="trait">yield</span> data on statistical significance of higher-order effects in the model, such as interactions, a simulation study was performed. Noise was added to the detrended log yields by calculating the standard deviation (sd) of each crop-country time series and then adding normally distributed random noise with mean 0 and sd ranging from 1, 2, 3 &#x02026; to 50% of the initial sd to ensure the same percent relative error across the different crop by country time series. The original models were then refit to the noisy data. The addition of random noise was repeated 100 times for each %-level of added noise, yielding 100 simulations of p-values corresponding to the significance of the highest order term(s), whether they being interaction, quadratic, or linear terms, depending on model. The 100 replications of p-values corresponding to a specific percent relative noise were presented in terms of stacked <span class="gene">bar</span> charts.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The development and use of a molecular model for soybean maturity groups</h2>
			<h3>PMID:28558691</h3>
				
					<h3>PMCID:5450301</h3>
					<h4>期刊:BMC Plant Biology</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Langewisch </surname>  <given-names> Tiffany </given-names>  </name>  <address>  <email> langewischt@missouri.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lenis </surname>  <given-names> Julian </given-names>  </name>  <address>  <email> Lenis@dow.com </email>  </address>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Jiang </surname>  <given-names> Guo-Liang </given-names>  </name>  <address>  <email> gjiang@vsu.edu </email>  </address>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Dechun </given-names>  </name>  <address>  <email> wangdech@msu.edu </email>  </address>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Pantalone </surname>  <given-names> Vince </given-names>  </name>  <address>  <email> vpantalo@utk.edu </email>  </address>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-4141-4790 </contrib-id>  <name>  <surname> Bilyeu </surname>  <given-names> Kristin </given-names>  </name>  <address>  <phone> (573) 884-2234 </phone>  <email> Kristin.Bilyeu@ars.usda.gov </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2162 3504 </institution-id>  <institution-id institution-id-type="GRID"> grid.134936.a </institution-id>  <institution> Plant Genetics Research Unit, United States Department of Agriculture-Agricultural Research Service,  </institution>  <institution> University of Missouri,  </institution>  </institution-wrap> 110 Waters Hall, Columbia, MO 65211 USA  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2179 3263 </institution-id>  <institution-id institution-id-type="GRID"> grid.418574.b </institution-id>  <institution/>  <institution> Dow AgroSciences LLC,  </institution>  </institution-wrap> 454 E 300N Road, Gibson City, IL 60936 USA  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9883 6009 </institution-id>  <institution-id institution-id-type="GRID"> grid.267895.7 </institution-id>  <institution> Agricultural Research Station,  </institution>  <institution> Virginia State University,  </institution>  </institution-wrap> P.O. Box 9061, Petersburg, VA 23806 USA  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2150 1785 </institution-id>  <institution-id institution-id-type="GRID"> grid.17088.36 </institution-id>  <institution> Department of Plant, Soil, and Microbial Sciences,  </institution>  <institution> Michigan State University,  </institution>  </institution-wrap> Plant and Soil Sciences Building, 1066 Bogue St., Room 348E, East Lansing, MI 48824 USA  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2315 1184 </institution-id>  <institution-id institution-id-type="GRID"> grid.411461.7 </institution-id>  <institution> Department of Plant Sciences,  </institution>  <institution> University of Tennessee,  </institution>  </institution-wrap> 2431 Joe Johnson Drive, Knoxville, TN 37996 USA  </aff>
				<br/> 
				年份：<day> 19 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Achieving appropriate maturity in a target environment is essential to maximizing crop <span class="trait">yield</span> potential. In soybean [ <italic> Glycine max </italic> (L.) Merr.], the time to maturity is largely dependent on developmental response to dark periods. Once the critical photoperiod is reached, flowering is initiated and reproductive development proceeds. Therefore, soybean adaptation has been attributed to genetic changes and natural or artificial selection to optimize plant development in specific, narrow latitudinal ranges. In North America, these regions have been classified into twelve maturity groups (MG), with lower MG being shorter season than higher MG. Growing soybean lines not adapted to a particular environment typically results in poor growth and significant <span class="trait">yield</span> reductions. The objective of this study was to develop a molecular model for soybean maturity based on the alleles underlying the major maturity loci: <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , and <italic> E3 </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Our results provide a selection model to account for the vast majority of variation in soybean flowering time and maturity for major US production environments (MGs 0-V). The sum of the results presented here is the ability to target soybean breeding for different maturities in the US using the genotype of <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , and <italic> E3 </italic> , a concept we term the &#x0201c;molecular maturity model&#x0201d; (Table <xref rid="Tab6" ref-type="table"> 6 </xref> ). This model was based mostly on cultivars developed for release in the US. Even though we presented evidence that multiple <italic> E </italic> allele combinations of <italic> <span class="gene">E1</span> </italic> , <italic> E2 </italic> , and <italic> E3 </italic> can be classified into the same MG, typically a MG had one prevalent <italic> E </italic> genotype (Fig. <xref rid="Fig2" ref-type="fig"> 2 </xref> ). We selected the most common <italic> E </italic> genotypes for MGs 0-V for the model to target US MG with high <span class="trait">yield</span> potential. <table-wrap id="Tab6"> <label> Table 6 </label> <caption> <p> Proposed <italic> E </italic> gene maturity model for the US
					</p>
				
					<p>&nbsp;&nbsp;
						Our research revealed a strong signal for artificial selection for the recessive <italic> e1-as </italic> allele in nearly all released cultivars in the US in MG 0-IV. In contrast, cultivars released in the US for MG V and later more closely matched the Eastern Hemisphere soybean landraces and cultivars which had a contrasting strong signal for artificial selection for functional <italic> <span class="gene">E1</span> </italic> alleles. This finding is surprising because the majority of Asian-originating soybeans have been classified with MGs IV or earlier when assessed in US production environments. We determined that the typical soybean originating from Asia has <italic> <span class="gene">E1</span> </italic> and <italic> e2 </italic> alleles. Bernard et al. established that soybean lines with contrasting <italic> <span class="gene">E1</span> </italic> and <italic> E2 </italic> genes ( <italic> <span class="gene">E1</span> e2 </italic> versus <italic> <span class="gene">e1</span> E2 </italic> ) had similar flowering times and very similar maturities of approximately MG III [ <xref ref-type="bibr" rid="CR3"> 3 </xref> ]. US soybean cultivars earlier than MG V excluded the <italic> <span class="gene">E1</span> e2 </italic> genotype, suggesting a negative impact on <span class="trait">yield</span> in the US for the predominant <italic> E </italic> genotype found in all Asian soybeans. This result also suggests that soybean breeding for US MG IV or earlier should include selection for <italic> e1-as </italic> when using Asian-originating soybean germplasm to increase overall genetic diversity or mining for new traits.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Proteomic analysis by iTRAQ-MRM of soybean <span class="trait">resistance</span> to <italic> Lamprosema Indicate </italic></h2>
			<h3>PMID:28587595</h3>
				
					<h3>PMCID:5461738</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Zeng </surname>  <given-names> Weiying </given-names>  </name>  <address>  <email> zengweiying_1981@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author" corresp="yes">  <name>  <surname> Sun </surname>  <given-names> Zudong </given-names>  </name>  <address>  <email> sunzudong639@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cai </surname>  <given-names> Zhaoyan </given-names>  </name>  <address>  <email> caizhaoyan424@163.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Huaizhu </given-names>  </name>  <address>  <email> chhuaizhu@sina.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lai </surname>  <given-names> Zhenguang </given-names>  </name>  <address>  <email> 519229671@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Shouzhen </given-names>  </name>  <address>  <email> 13367811306@189.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tang </surname>  <given-names> Xiangmin </given-names>  </name>  <address>  <email> TXM906@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"/>  </contrib>  <aff id="Aff1">  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 0415 7259 </institution-id>  <institution-id institution-id-type="GRID"> grid.452720.6 </institution-id>  <institution/>  <institution> Guangxi Academy of Agricultural Sciences,  </institution>  </institution-wrap> Nanning, Guangxi 530007 China  </aff>
				<br/> 
				年份：<day> 28 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						One of the key enzymes in the ethylene biosynthesis of plants is 1-aminocyclopropane-1- carboxylate oxidase (ACO). The amount of ethylene production is mainly determined by the activity of the <span class="gene">ACO</span> [ <xref ref-type="bibr" rid="CR49"> 49 </xref> ]. The <span class="gene">ACO</span> is a multigene family encoding protein, which is regulated by the feedback regulation of the ethylene signaling [ <xref ref-type="bibr" rid="CR50"> 50 </xref> ]. Ethylene, as a plant endogenous hormone, regulates the growth and development of plants, defense responses, and the synthesis of secondary metabolites [ <xref ref-type="bibr" rid="CR51"> 51 </xref> ]. It was found that the <span class="gene">ACO</span> would be induced when peach was affected by wounding [ <xref ref-type="bibr" rid="CR52"> 52 </xref> ]. The NtACO1, NtACO2 and NtACO3 in tobacco were induced during salt stress [ <xref ref-type="bibr" rid="CR53"> 53 </xref> , <xref ref-type="bibr" rid="CR54"> 54 </xref> ]. Also, the <span class="gene">ACO</span> in a wild type of <italic> Arabidopsis </italic> was enhanced after being treated for salt <span class="trait">resistance</span> [ <xref ref-type="bibr" rid="CR55"> 55 </xref> ]. In this study, the <span class="gene">ACO</span> in the Wan 82&#x02013;178 was up-regulated compared 48&#x000a0;h with 0&#x000a0;h. It was hypothesized that the pest stress inducted the <span class="gene">ACO</span> expression, and produced a large amount of ethylene in order to improve the <span class="trait">resistance</span> abilities to the <italic> Lamprosema indicate </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						OAS-TL is a key enzyme in the synthesis of Cys, and it has been found to be closely related to the secondary metabolic synthesis of some plants [ <xref ref-type="bibr" rid="CR56"> 56 </xref> ]. Previous studies have shown that the OAS-TL isoenzymes in watermelon [ <xref ref-type="bibr" rid="CR57"> 57 </xref> ], <italic> Leucaena leucocephala </italic> [ <xref ref-type="bibr" rid="CR58"> 58 </xref> ] and <italic> Quisqulis indica </italic> [ <xref ref-type="bibr" rid="CR59"> 59 </xref> ] can synthase toxic or anthelmintic beta heterocyclic rings, which replaced alanine and resisted pests. In this study, the expression level of the <italic> <span class="gene">GmOASTL4</span> </italic> in Gantai-2-2 was found to always be higher than the Wan 82&#x02013;178 at 0&#x000a0;h and 48&#x000a0;h. These results suggested that it may synthase some of the secondary products <span class="trait">resistance</span> to <italic> Lamprosema indicate </italic> feeding, or beta site heterocyclic substituted alanine and other substances. Therefore, it plays a key role in the defense against <italic> Lamprosema indicate </italic> .
					</p>
				
					<p>&nbsp;&nbsp;
						Glutamine synthetase (GS) is the key enzyme of the nitrogen metabolisms in higher plants. It catalyzes the ammonium salt and glutamate to generate glutamine, providing needed nitrogen compounds for proline synthesis, which affects plant growth and development [ <xref ref-type="bibr" rid="CR60"> 60 </xref> ]. <span class="gene">GS</span> is divided into two categories, cytoplasmic or cytosolic types of <span class="gene">GS</span> (GS1), and plastid or chloroplast types of <span class="gene">GS</span> (GS2), in which the GS1 mainly produces glutamine to complete the transport of nitrogen between cells [ <xref ref-type="bibr" rid="CR61"> 61 </xref> ]. Previous studies have shown that the GS1 isozyme is associated with plant disease and stress resistances. The expression of the GS1 isozyme was enhanced when the following plants suffered pathogen infections, <span class="trait">drought,</span> and other stresses: tomato [ <xref ref-type="bibr" rid="CR62"> 62 </xref> , <xref ref-type="bibr" rid="CR63"> 63 </xref> ], tobacco [ <xref ref-type="bibr" rid="CR64"> 64 </xref> ], potato [ <xref ref-type="bibr" rid="CR65"> 65 </xref> ] and soybean [ <xref ref-type="bibr" rid="CR66"> 66 </xref> ]. In this study, the GS1 isoform X1 was all up-regulated in the Gantai-2-2 and Wan 82&#x02013;178, comparing 48&#x000a0;h with 0&#x000a0;h, which may have facilitated proline accumulation, in order to improve the osmotic adjustment abilities in the soybean leaves, as well as to activate the oxygen scavenging capacities against the <italic> Lamprosema indicate </italic> feeding.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Main and epistatic loci studies in soybean for <italic> Sclerotinia sclerotiorum </italic> <span class="trait">resistance</span> reveal multiple modes of <span class="trait">resistance</span> in multi-environments</h2>
			<h3>PMID:28620159</h3>
				
					<h3>PMCID:5472596</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Moellers </surname>  <given-names> Tara C. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Arti </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jiaoping </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Brungardt </surname>  <given-names> Jae </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Kabbage </surname>  <given-names> Mehdi </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mueller </surname>  <given-names> Daren S. </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Grau </surname>  <given-names> Craig R. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ranjan </surname>  <given-names> Ashish </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Smith </surname>  <given-names> Damon L. </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chowda-Reddy </surname>  <given-names> R. V. </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-7522-037X </contrib-id>  <name>  <surname> Singh </surname>  <given-names> Asheesh K. </given-names>  </name>  <address>  <email> singhak@iastate.edu </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1936 7312 </institution-id>  <institution-id institution-id-type="GRID"> grid.34421.30 </institution-id>  <institution> Department of Agronomy,  </institution>  <institution> Iowa State University,  </institution>  </institution-wrap> Ames, Iowa 50011 United States of America  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2167 3675 </institution-id>  <institution-id institution-id-type="GRID"> grid.14003.36 </institution-id>  <institution> Department of Plant Pathology,  </institution>  <institution> University of Wisconsin-Madison,  </institution>  </institution-wrap> Madison, Wisconsin 53706 United States of America  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0004 1936 7312 </institution-id>  <institution-id institution-id-type="GRID"> grid.34421.30 </institution-id>  <institution> Department of Plant Pathology,  </institution>  <institution> Iowa State University,  </institution>  </institution-wrap> Ames, Iowa 50011 United States of America  </aff>
				<br/> 
				年份：<day> 10 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Genome-wide association (GWAS) and epistatic (GWES) studies along with expression studies in soybean [ <italic> Glycine max </italic> (L.) Merr.] were leveraged to dissect the genetics of Sclerotinia stem rot (SSR) [caused by <italic> Sclerotinia sclerotiorum </italic> (Lib.) de Bary], a significant fungal disease causing <span class="trait">yield</span> and quality losses. A large association panel of 466 diverse plant introduction accessions were phenotyped in multiple field and controlled environments to: (1) discover sources of <span class="trait">resistance,</span> (2) identify SNPs associated with <span class="trait">resistance,</span> and (3) determine putative candidate genes to elucidate the mode of <span class="trait">resistance.</span> We report 58 significant main effect loci and 24 significant epistatic interactions associated with SSR <span class="trait">resistance,</span> with candidate genes involved in a wide range of processes including cell wall structure, hormone signaling, and sugar allocation related to plant immunity, revealing the complex nature of SSR <span class="trait">resistance.</span> Putative candidate genes [for example, <italic> PHYTOALEXIN DEFFICIENT 4 </italic> ( <italic> <span class="gene">PAD4</span> </italic> ), <italic> ETHYLENE-INSENSITIVE 3-LIKE 1 </italic> ( <italic> EIL3 </italic> ), and <italic> ETHYLENE RESPONSE FACTOR 1 </italic> ( <italic> ERF1 </italic> )] clustered into salicylic acid (SA), jasmonic acid (JA), and ethylene (ET) pathways suggest the involvement of a complex hormonal network typically activated by both necrotrophic (ET/JA) and biotrophic (SA) pathogens supporting that <italic> S </italic> . <italic> sclerotiorum </italic> is a hemibiotrophic plant pathogen.
					</p>
				
					<p>&nbsp;&nbsp;
						Comparisons for previously reported QTL were done by projecting genetic map locations onto the Glyma. Wm. 82. a2 reference genome physical map <sup> <xref ref-type="bibr" rid="CR12"> 12 </xref> </sup> . The significant main effect loci from GWAS are presented in Supplementary Table&#x000a0; <xref rid="MOESM1" ref-type="media"> S3 </xref> . Significant SNPs for SSR <span class="trait">resistance</span> were identified on <span class="gene">Gm04</span> and <span class="gene">Gm12,</span> where no SSR <span class="trait">resistance</span> loci have been reported. The genetic background used in previous studies could have inhibited the detection of loci on these chromosomes in several ways: lack of segregation in bi-parental populations, low allele frequency in the panels chosen for GWAS, or marker density resulting in lack of LD with the causal loci. This study used 466 diverse accessions to increase historic recombination and decrease LD, with over 35&#x02009;K markers used to capture the genetic variation and increase the genetic resolution. Twenty-one of the 58 identified main effect loci co-localized with previously reported QTL for SSR <span class="trait">resistance.</span> Of these, three loci ( <italic> ss715599948 </italic> , <italic> ss715603406 </italic> , and <italic> ss715603408 </italic> ) located on <span class="gene">Gm08</span> and <span class="gene">Gm09</span> overlapped with multiple QTL previously reported for SSR <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Putative candidate genes with a defense functional category were involved in programmed plant cell death (PCD), both apoptosis and autophagy, or encoded genes related to pathogenesis-related (PR) genes and their products. Many genes in this category could also be dual-classified under signaling. Putative candidate gene <italic> Glyma </italic> . <italic> 12g169300 </italic> , associated with 14FLD, and <italic> Glyma </italic> . <italic> 12g171300 </italic> , associated with 15GHSE-DAI03, are homologous to barley ( <italic> Hordeum vulgare </italic> ) <italic> MILDEW RESISTNACE LOCUS O </italic> ( <italic> <span class="gene">MLO</span> </italic> ). A loss-of-function mutation in <italic> mlo </italic> genes confer <span class="trait">resistance</span> to powdery mildew fungus ( <italic> Erysiphe graminis </italic> ) in barley in a non-race-specific manner through the prevention of fungus penetration <sup> <xref ref-type="bibr" rid="CR42"> 42 </xref> </sup> . Additional mutations to the barley <italic> <span class="gene">MLO</span> </italic> locus show function in PCD and pathogen defense <sup> <xref ref-type="bibr" rid="CR43"> 43 </xref> </sup> .
					</p>
				
					<p>&nbsp;&nbsp;
						<italic> Glyma </italic> . <italic> 13g328100 </italic> , associated with 15FLD, encodes a plant, small ubiquitin-like modifier (SUMO) E3 ligase homologous to <italic> Arabidopsis SIZ1 </italic> , which has been shown to regulate innate immunity <sup> <xref ref-type="bibr" rid="CR44"> 44 </xref> </sup> . Mutant plants demonstrated elevated salicylic acid (SA) accumulation, expression of PR-genes, and increased <span class="trait">resistance</span> to <italic> P </italic> . <italic> syringae </italic> . Double mutants with genes such as <italic> pad4 </italic> suggest the control of <italic> SIZ1 </italic> on SA signaling. Mutations in <italic> Arabidopsis pad4 </italic> gene reduce the expression of <italic> PR-1 </italic> and levels of SA when infected with <italic> P </italic> . <italic> syringae </italic> <sup> <xref ref-type="bibr" rid="CR45"> 45 </xref> </sup> . Together, <italic> <span class="gene">PAD4</span> </italic> and <italic> SIZ1 </italic> work epistatically to regulate PR expression <sup> <xref ref-type="bibr" rid="CR44"> 44 </xref> </sup> . Finally, <italic> Glyma </italic> . <italic> 14g042400 </italic> , also associate with 15FLD, encodes a putative <span class="gene">LRR-RLK</span> transmembrane receptor induced by chitin oligomers homologous to Arabidopsis <italic> RECETPOR LIKE PROTEIN 52 </italic> ( <italic> RLP52 </italic> ). Chitin is a major component of fungal cell walls, and chitin oligomers are known to induce plant defense cascades.
					</p>
				
					<p>&nbsp;&nbsp;
						An RNAseq dataset was mined for all available putative candidate genes detected through GWAS. This was done successfully for 38 putative candidate genes. Of these, 50% were differentially expressed between susceptible and resistant soybean lines. Out of the eight genes associated with DAI03, five were differentially regulated. <italic> Glyma </italic> . <italic> 04g209700 </italic> , a putative candidate gene ( <italic> <span class="gene">PAD4</span> </italic> ) associated with 14GHSE-DAI03, was consistently expressed at higher levels in the resistant line compared to the susceptible line throughout the time course of the experiment, and at significant levels at 24, 48, and 96&#x02009;hours post-inoculation (Fig.&#x000a0; <xref rid="Fig2" ref-type="fig"> 2 </xref> ). <italic> Glyma </italic> . <italic> 04g209700 </italic> encodes a lipase (class 3) protein and is homologous to <italic> Arabidopsis <span class="gene">PAD4</span> </italic> . <italic> <span class="gene">PAD4</span> </italic> is thought to be involved in the regulation of SA and has been shown to affect the activation of subsequent defense responses such as camalexin synthesis and <italic> PR-1 </italic> gene expression <sup> <xref ref-type="bibr" rid="CR45"> 45 </xref> , <xref ref-type="bibr" rid="CR46"> 46 </xref> </sup> . <italic> Glyma </italic> . <italic> 18g039900 </italic> was the putative candidate gene associated with 15GHSE-DAI03. Although the function of this gene is currently unknown, it was highly expressed in the susceptible line, and transcript levels were significantly higher compared to the resistant line (Fig.&#x000a0; <xref rid="Fig3" ref-type="fig"> 3 </xref> ). Its association with SSR <span class="trait">resistance</span> in soybean makes this a gene of interest, and further studies addressing on its function are needed. <fig id="Fig2"> <label> Figure 2 </label> <caption> <p> Regional association plot of <italic> ss715588567 </italic> associated with 14GHSE-DAI03 and expression of the putative candidate gene <italic> Glyma </italic> . <italic> 04g209700 </italic> . ( <bold> a </bold> ) The top panel shows the negative log10-transformed <italic> P </italic> values from a genome-wide scan by using a mixed linear model (MLM) for plant severity taken 3 DAI within the adjacent region of peak SNP <italic> ss715588567 </italic> plotted against base pair positions (Mb) on soybean chromosome 4. The significance threshold line is distinguished (gray dotted line), and each SNP depicts the extent of linkage disequilibrium in the region based on pairwise values with respect to the peak SNP. The values are indicated using the color intensity index shown. The bottom panel shows all putative genes in the region. The putative candidate gene indicated in green. ( <bold> b </bold> ) The expression of <italic> Glyma </italic> . <italic> 04g209700 </italic> , the putative candidate gene for <italic> ss715588567 </italic> , in a modern resistant (blue) and susceptible (orange) cultivar. Error bars shown to indicate significance (p&#x02009;&#x0003c;&#x02009;0.05).
					</p>
				
					<p>&nbsp;&nbsp;
						GWAS and GWES results were combined to propose a putative disease <span class="trait">resistance</span> model based on the <italic> S </italic> . <italic> sclerotiorum </italic> &#x02013; soybean interaction, and the newly presumed hemibiotrophic nature of the fungus <sup> <xref ref-type="bibr" rid="CR65"> 65 </xref> </sup> . The putative candidate genes clustered into SA, JA, and ET pathways, indicating the possible involvement of a complex hormonal network typically activated by both necrotrophic (ET/JA) and biotrophic (SA) pathogens. This approach allowed the annotation of candidate genes with particular interest ( <italic> <span class="gene">PAD4</span> </italic> , <italic> EIL1 </italic> , and <italic> ERF1 </italic> ) identified in this study to the proposed model, giving support to the pathways of defense against biotrophs and necrotrophs for <italic> S </italic> . <italic> sclerotiorum </italic> (Fig.&#x000a0; <xref rid="Fig4" ref-type="fig"> 4 </xref> ). The putative candidate genes, <italic> <span class="gene">PAD4</span> </italic> , <italic> EIL1 </italic> , and <italic> ERF1 </italic> , formed a gene network and included several other candidate genes identified in this study (Supplementary Fig.&#x000a0; <xref rid="MOESM1" ref-type="media"> S7 </xref> ). Twenty-one putative candidate genes interacted at first level when <italic> <span class="gene">PAD4</span> </italic> , <italic> ERF1 </italic> , and <italic> EIL1 </italic> were considered (8 in SA, 9 in JA, and 5 in ET). The <italic> <span class="gene">PAD4</span> </italic> gene, as previously indicated, was significantly upregulated at 24, 48, and 96&#x02009;hours in the RNAseq experiment, and seems to be crucial for the <italic> S </italic> . <italic> sclerotiorum </italic> &#x02013; soybean disease interaction. <fig id="Fig4"> <label> Figure 4 </label> <caption> <p> Proposed phytohormones-dependent disease defense model against <italic> Sclerotinia sclerotiorum </italic> in soybean. SA, salicylic acid; JA, jasmonic acid; ET, ethylene; LRR-RLKs, Leucine-rich repeat receptor-like protein kinases; PAD4, Phytoalexin-dependent4; SIZ1, small ubiquitin-related modifier E3 ligase; PRs, Pathogenesis-related proteins; EIL1, Ethylene-insensitive 3-like 1; ERF1, Ethylene response factor 1.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Construction of high-density genetic map and QTL mapping of yield-related and two quality traits in soybean RILs population by RAD-sequencing</h2>
			<h3>PMID:28629322</h3>
				
					<h3>PMCID:5477377</h3>
					<h4>期刊:BMC Genomics</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Liu </surname>  <given-names> Nianxi </given-names>  </name>  <address>  <email> lnx69@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author" equal-contrib="yes">  <name>  <surname> Li </surname>  <given-names> Mu </given-names>  </name>  <address>  <email> lm928094297@126.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hu </surname>  <given-names> Xiangbao </given-names>  </name>  <address>  <email> 1584519512@qq.com </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Qibin </given-names>  </name>  <address>  <email> maqibin@scau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mu </surname>  <given-names> Yinghui </given-names>  </name>  <address>  <email> youhymoon@scau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Tan </surname>  <given-names> Zhiyuan </given-names>  </name>  <address>  <email> zytan@scau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xia </surname>  <given-names> Qiuju </given-names>  </name>  <address>  <email> xiaqiuju@genomics.cn </email>  </address>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Gengyun </given-names>  </name>  <address>  <email> zhanggengyun@genomics.cn </email>  </address>  <xref ref-type="aff" rid="Aff5"> 5 </xref>  </contrib>  <contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0003-0227-2045 </contrib-id>  <name>  <surname> Nian </surname>  <given-names> Hai </given-names>  </name>  <address>  <email> hnian@scau.edu.cn </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  <xref ref-type="aff" rid="Aff4"> 4 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9546 5767 </institution-id>  <institution-id institution-id-type="GRID"> grid.20561.30 </institution-id>  <institution> The State Key Laboratory for Conservation and Utilization of Subtropical Agro-Bioresources,  </institution>  <institution> South China Agricultural University,  </institution>  </institution-wrap> Guangzhou, 510642 People&#x02019;s Republic of China  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9546 5767 </institution-id>  <institution-id institution-id-type="GRID"> grid.20561.30 </institution-id>  <institution> The Key Laboratory of Plant Molecular Breeding,  </institution>  <institution> South China Agricultural University,  </institution>  </institution-wrap> Guangzhou, 510642 People&#x02019;s Republic of China  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9546 5767 </institution-id>  <institution-id institution-id-type="GRID"> grid.20561.30 </institution-id>  <institution> The Guangdong Subcenter of the National Center for Soybean Improvement, College of Agriculture,  </institution>  <institution> South China Agricultural University,  </institution>  </institution-wrap> Guangzhou, 510642 People&#x02019;s Republic of China  </aff>  <aff id="Aff4">  <label> 4 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0000 9546 5767 </institution-id>  <institution-id institution-id-type="GRID"> grid.20561.30 </institution-id>  <institution> Joint Laboratory of Plant Breeding,  </institution>  <institution> South China Agricultural University-ShanDong Shofine Seed Technology Co. Ltd,  </institution>  </institution-wrap> Guangzhou, 510642 People&#x02019;s Republic of China  </aff>  <aff id="Aff5">  <label> 5 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2034 1839 </institution-id>  <institution-id institution-id-type="GRID"> grid.21155.32 </institution-id>  <institution/>  <institution> Beijing Genome Institute (BGI),  </institution>  </institution-wrap> Shenzhen, 518083 People&#x02019;s Republic of China  </aff>
				<br/> 
				年份：<day> 9 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Based on the predicted function of the five groups, three predicted genes ( <italic> <span class="gene">Glyma19g37910,</span> 37,570, 36,990 </italic> ) were selected as the best candidate genes that may affect both <span class="trait">yield</span> and quality because they are involved in various biological process (Table <xref rid="Tab4" ref-type="table"> 4 </xref> ). <italic> <span class="gene">Glyma19g37910</span> </italic> encodes a member of the basic leucine zipper transcription factor family, involved in <italic> arabidopsis </italic> abscisic acid signalling during seed maturation and germination. GO analysis showed that this gene participated in more than ten biological process, which include seed development, lipid storage, gibberellin biosynthesis, and vegetative to reproductive phase transition of the meristem, etc. <italic> <span class="gene">Glyma19g37570</span> </italic> gene has a domain predicted to encode a serine/threonine protein kinase that could influence cells in various ways. This gene is related to the process of stem cell division, protein phosphorylation, gibberellin biosynthesis and timing of the transition from vegetative to reproductive development. <italic> <span class="gene">Glyma19g36990</span> </italic> encodes a plastidic triose phosphate isomerase, and GO analysis revealed that this gene participates in three catabolic process (glycine, tryptophan, and glycerol) and four biosynthetic process (indoleacetic acid, cysteine, and glyceraldehyde-3-phosphate, isopentenyl diphosphate). Moreover, it also plays a key role in multicellular organism reproduction and primary root development, which may have an effect on the <span class="trait">yield</span> and quality of crops. In general, these three candidates should be investigated in more detail in further studies to increase our understanding regarding the factors involved in the process of improving quality and productivity in soybean. <table-wrap id="Tab4"> <label> Table 4 </label> <caption> <p> The information of three candidates&#x02019; annotations
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Potassium Starvation Limits Soybean Growth More than the Photosynthetic Processes across CO <sub> 2 </sub> Levels</h2>
			<h3>PMID:28642785</h3>
				
					<h3>PMCID:5462980</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Shardendu K. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/347336/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Reddy </surname>  <given-names> Vangimalla R. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/400588/overview"/>  </contrib>
				<br/> 
				年份：<day> 24 </day>  <month> 5 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The regression relationships of the plant processes with tissue K concentration provided insight into the changes in their functional relationship across a gradient of leaf K at different CO <sub> 2 </sub> levels (Figures <xref ref-type="fig" rid="F3"> 3 </xref> , <xref ref-type="fig" rid="F4"> 4 </xref> ). The N accumulation in plants led to the inverse relationship observed between tissue K and N concentrations similarly across CO <sub> 2 </sub> levels but resulted in reduced C/N and increased N/K ratios under K deficiency. The mean N/K ratio in the uppermost fully expanded leaves was below two under control conditions whereas it increased up to 16 in K-deficient plants, indicating K as the primary growth-limiting factor (Lawniczak et al., <xref rid="B29" ref-type="bibr"> 2009 </xref> ). When growth is limited by other nutrients, the excess N might be stored in the form of nitrogenous compounds (e.g., non-functional protein, amino acids) at the expense of other cellular components such as chlorophyll and enzymes (Staswick et al., <xref rid="B56" ref-type="bibr"> 1991 </xref> ; Singh et al., <xref rid="B52" ref-type="bibr"> 2016 </xref> ). In the present study, the accumulation of N was exacerbated by the increase in N supply in the K- deficient treatments. The eCO <sub> 2 </sub> modified the exponential relationship between P <sub> net </sub> and leaf K, with a greater P <sub> net </sub> observed above roughly 8 mg g <sup> &#x02212;1 </sup> tissue K concentration. However, below this (8 mg g <sup> &#x02212;1 </sup> ) leaf K concentration, P <sub> net </sub> did not differ between CO <sub> 2 </sub> levels and sharply declined. A similar reduction in photosynthesis with leaf K concentration has also been observed in other species (Reddy and Zhao, <xref rid="B41" ref-type="bibr"> 2005 </xref> ; Jin et al., <xref rid="B27" ref-type="bibr"> 2011 </xref> ). The DM and seed <span class="trait">yield</span> were also increased in response to increasing leaf K concentration, and productivity was always greater under eCO <sub> 2 </sub> for a given leaf K concentration, except under very low leaf K. Moreover, as compared with the P <sub> net </sub> , the DM and seed <span class="trait">yield</span> began to decline at the higher level of leaf K concentration indicating greater sensitivity to K deficiency. For instance, the DM and seed <span class="trait">yield</span> declined while P <sub> net </sub> remained unaffected roughly near leaf K concentration of 11 mg g <sup> &#x02212;1 </sup> (Figures <xref ref-type="fig" rid="F4"> 4A&#x02013;C </xref> ). However, regression relationships for DM and seed <span class="trait">yield</span> should be treated with caution because of the large <span class="gene">gap</span> between leaf K concentrations of plants supplied with 5.0 vs. 0.5 mM K treatments. The eCO <sub> 2 </sub> did not modify the relationship of root/shoot ratio, individual seed weight, or the concentration of seed K and N to the concentration of leaf K, which was in agreement with a previous study conducted under P nutrition in soybean (Singh et al., <xref rid="B53" ref-type="bibr"> 2014 </xref> ). The inverse relationship between seed N and leaf K concentration suggested that N mobilization to the seed was not restricted and the seed N demand may be lower due to decreased seed <span class="trait">yield</span> under K deficiency (Singh et al., <xref rid="B53" ref-type="bibr"> 2014 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Unmanned Aerial Vehicle Remote Sensing for Field-Based Crop Phenotyping: Current Status and Perspectives</h2>
			<h3>PMID:28713402</h3>
				
					<h3>PMCID:5492853</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Guijun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02021; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Jiangang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn003">  <sup> &#x02020; </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02021; </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/429108/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Chunjiang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Zhenhong </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/430200/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Huang </surname>  <given-names> Yanbo </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Haiyang </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Xu </surname>  <given-names> Bo </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Xiaodong </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhu </surname>  <given-names> Dongmei </given-names>  </name>  <xref ref-type="aff" rid="aff6">  <sup> 6 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Xiaoyan </given-names>  </name>  <xref ref-type="aff" rid="aff7">  <sup> 7 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Ruyang </given-names>  </name>  <xref ref-type="aff" rid="aff8">  <sup> 8 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Feng </surname>  <given-names> Haikuan </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhao </surname>  <given-names> Xiaoqing </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Zhenhai </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Heli </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Hao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 08 </day>  <month> 6 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Field-based phenotyping (FBP) is a critical component of crop improvement through genetics, as it is the ultimate expression of the relative effects of genetic factors, environmental factors, and their interaction on critical production traits, such as <span class="trait">yield</span> potential and tolerance to abiotic/biotic stresses (Araus and Cairns, <xref rid="B3" ref-type="bibr"> 2014 </xref> ; Neilson et al., <xref rid="B89" ref-type="bibr"> 2015 </xref> ). <span class="gene">FBP</span> is increasingly recognized as the only approach capable of delivering the required throughput and an accurate description of trait expression in real-world cropping systems. The performance of breeding programs on crop <span class="trait">yield</span> and productivity must be evaluated under natural conditions (Gonzalez-Recio et al., <xref rid="B45" ref-type="bibr"> 2014 </xref> ; Gonzalez-Dugo et al., <xref rid="B42" ref-type="bibr"> 2015 </xref> ; Rahaman et al., <xref rid="B100" ref-type="bibr"> 2015 </xref> ). Currently, the most commonly field-based phenotyping platforms (FBPPs) use ground wheeled or aerial vehicles deploying multiple types of sensors to measure plant traits on a timescale of a few seconds per plot. For FBPPs based on ground vehicles, the process is time-consuming if there are too many plots needed to collect data (Zhang and Kovacs, <xref rid="B153" ref-type="bibr"> 2012 </xref> ). For example, more than 40 h were required to cover the 20,000 plots with a single vehicle traveling at 2 km per h to measure traits on single rows (White et al., <xref rid="B143" ref-type="bibr"> 2012 </xref> ). Using multiple vehicles and multiple sets of sensors to take measurements in all plots simultaneously would increase the costs (Zhang and Kovacs, <xref rid="B153" ref-type="bibr"> 2012 </xref> ; Cobb et al., <xref rid="B20" ref-type="bibr"> 2013 </xref> ). Moreover, FBPPs with ground vehicles cannot be used for cross-regional work due to the lack of maneuvrability. In the recent years, the cable-suspended field phenotyping platform was developed for rapid and non-destructive estimation of crop traits. For the cable-suspend field-based phenotyping platform, there's advantages of safety, high precision, independent of soil conditions and minimal tactile interference of plants. However, as it has to be located at certain sites, the coverable area of cable-suspend field phenotyping platform is relatively low, which limit its applications for the large-scale phenotyping (Kirchgessner et al., <xref rid="B66" ref-type="bibr"> 2016 </xref> ). Some of these limitations can be addressed using satellite-based or aerial remote sensing approaches.
					</p>
				
					<p>&nbsp;&nbsp;
						<span class="gene">FBP</span> using UAV-RSPs is based on an unmanned aircraft equipped with multiple sensors, using communication technology and GPS positioning technology to rapidly and non-destructively acquire high-resolution images about the crop canopy in the field. Remote sensing retrieval models are then used for phenotyping field trials after data processing (Sugiura et al., <xref rid="B122" ref-type="bibr"> 2005 </xref> ; Li W. et al., <xref rid="B72" ref-type="bibr"> 2016 </xref> ). The typical UAVs used for <span class="gene">FBP</span> include multi-rotors, helicopters, fixed-wing, blimps and flying wing (Table <xref ref-type="table" rid="T2"> 2 </xref> ; Sankaran et al., <xref rid="B115" ref-type="bibr"> 2015b </xref> ) and are selected based on the purpose and budget. Blimps have the advantages of hovering ability, higher effective loads and the ability of vertical take-off and landing; however, they are slow because of the large size, and their stability is poor under windy conditions, making it difficult to obtain accurate information (Liebisch et al., <xref rid="B80" ref-type="bibr"> 2015 </xref> ). Unmanned helicopters have the advantage of being able to take-off and land vertically, fly sideways, and hover. The helicopter payload is larger than that of a multi-rotor UAV and can support large sensors, such as LIDAR. However, the complex operation, lack of free hover, high maintenance cost and noise limit the application of helicopters (Sugiura et al., <xref rid="B122" ref-type="bibr"> 2005 </xref> ; Swain et al., <xref rid="B125" ref-type="bibr"> 2010 </xref> ; Chapman et al., <xref rid="B17" ref-type="bibr"> 2014 </xref> ). The fixed-wing UAV is characterized by fast flying velocity and long flight time; however, the bottleneck for the fixed-wing application of <span class="gene">FBP</span> is the lack of free hover ability and the image blur caused by higher speeds and altitudes (Herwitz et al., <xref rid="B55" ref-type="bibr"> 2004 </xref> ; Link et al., <xref rid="B81" ref-type="bibr"> 2013 </xref> ). Multi-rotor UAVs have the advantages of low cost, the ability to hover, low take-off and landing requirements and are most frequently used for FBP. However, the greatest limitations of multi-rotor UAVs are the relatively short flight time, lower payload and the sensitivity to weather (Zhang and Kovacs, <xref rid="B153" ref-type="bibr"> 2012 </xref> ; Pena et al., <xref rid="B97" ref-type="bibr"> 2013 </xref> ; Uto et al., <xref rid="B133" ref-type="bibr"> 2013 </xref> ; Nasi et al., <xref rid="B88" ref-type="bibr"> 2015 </xref> ). Traditional UAV manufactured body materials are metals, such as aviation steel and aluminum (Colomina and Molina, <xref rid="B21" ref-type="bibr"> 2014 </xref> ; Salami et al., <xref rid="B111" ref-type="bibr"> 2014 </xref> ; Pajares, <xref rid="B95" ref-type="bibr"> 2015 </xref> ). To reduce the UAV weight, enhance the body strength and prolong flight time, a variety of lightweight, high-strength composite materials have been widely used, including glass fiber and carbon fiber, and have become the main alternative materials for the body of UAVs. For the engine, the UAV engines can be divided into two categories: oil and electric engine. The oil engines have the advantages of strong wind <span class="trait">resistance</span> and long working time, while there're disadvantages of being bulky, producing big vibration and having poor reliability, which lead to the image blur (Xiang and Tian, <xref rid="B144" ref-type="bibr"> 2011 </xref> ; Sankaran et al., <xref rid="B115" ref-type="bibr"> 2015b </xref> ). The electric engines have the advantages of safe, small vibration, easy to maintain and low cost, which make it become an important way for crop phenotyping by UAV; however, the short flight endurance time and weak wind <span class="trait">resistance</span> limited its use for crop phenotyping at large scale. A series of propulsion systems with the advantages of small volume, low vibration and new energy sources have become available and have greatly enhanced the UAV payload space and capacity. Hale engines and low-altitude silent propulsion systems are necessary to satisfy the requirements of small- and medium-size UAVs (Verhoeven, <xref rid="B135" ref-type="bibr"> 2009 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Responses of crop <span class="trait">yield</span> growth to global temperature and socioeconomic changes</h2>
			<h3>PMID:28798370</h3>
				
					<h3>PMCID:5552729</h3>
					<h4>期刊:Scientific Reports</h4>
				

				
			<p>
				作者：<contrib contrib-type="author" corresp="yes">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-0611-4637 </contrib-id>  <name>  <surname> Iizumi </surname>  <given-names> Toshichika </given-names>  </name>  <address>  <email> iizumit@affrc.go.jp </email>  </address>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Furuya </surname>  <given-names> Jun </given-names>  </name>  <xref ref-type="aff" rid="Aff2"> 2 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shen </surname>  <given-names> Zhihong </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0002-5316-4135 </contrib-id>  <name>  <surname> Kim </surname>  <given-names> Wonsik </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Okada </surname>  <given-names> Masashi </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0001-7897-1796 </contrib-id>  <name>  <surname> Fujimori </surname>  <given-names> Shinichiro </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id contrib-id-type="orcid"> http://orcid.org/0000-0003-2456-5789 </contrib-id>  <name>  <surname> Hasegawa </surname>  <given-names> Tomoko </given-names>  </name>  <xref ref-type="aff" rid="Aff3"> 3 </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Nishimori </surname>  <given-names> Motoki </given-names>  </name>  <xref ref-type="aff" rid="Aff1"> 1 </xref>  </contrib>  <aff id="Aff1">  <label> 1 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2222 0432 </institution-id>  <institution-id institution-id-type="GRID"> grid.416835.d </institution-id>  <institution> Institute for Agro-Environmental Sciences,  </institution>  <institution> National Agriculture and Food Research Organization,  </institution>  </institution-wrap> Tsukuba, Japan  </aff>  <aff id="Aff2">  <label> 2 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 2107 8171 </institution-id>  <institution-id institution-id-type="GRID"> grid.452611.5 </institution-id>  <institution/>  <institution> Japan International Research Center for Agricultural Science,  </institution>  </institution-wrap> Tsukuba, Japan  </aff>  <aff id="Aff3">  <label> 3 </label>  <institution-wrap>  <institution-id institution-id-type="ISNI"> 0000 0001 0746 5933 </institution-id>  <institution-id institution-id-type="GRID"> grid.140139.e </institution-id>  <institution> Center for Social and Environmental Systems Research,  </institution>  <institution> National Institute for Environmental Studies,  </institution>  </institution-wrap> Tsukuba, Japan  </aff>
				<br/> 
				年份：<day> 7 </day>  <month> 7 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The temperature increase and corresponding mitigation level (RCP) at which the anticipated <span class="trait">yield</span> growth for 2100 (the average for the period 2091&#x02013;2100) was found to be the highest for the four RCPs. Any warming above this level leads to <span class="trait">yield</span> stagnation. The pie diagrams denote percentages of harvested area under the aforementioned warming levels. All data shown in the pie diagrams are normalized to the global harvested area for 2000. The maps presented here were created from Generic Mapping Tools <span class="gene">(GMT)</span> <sup> <xref ref-type="bibr" rid="CR49"> 49 </xref> </sup> version 4.5.12 ( <ext-link ext-link-type="uri" xlink:href="https://www.soest.hawaii.edu/gmt/"> https://www.soest.hawaii.edu/gmt/ </ext-link> ) using data described in the main text.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:The transcriptional landscape of <italic> Rhizoctonia solani </italic> AG1-IA during infection of soybean as defined by RNA-seq</h2>
			<h3>PMID:28877263</h3>
				
					<h3>PMCID:5587340</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Copley </surname>  <given-names> Tanya R. </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Duggavathi </surname>  <given-names> Raj </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-3050-4723 </contrib-id>  <name>  <surname> Jabaji </surname>  <given-names> Suha </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 17 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						The pigment, melanin is required to protect fungi and enhance their survival during adverse conditions, and it is also involved in sclerotial development [ <xref rid="pone.0184095.ref055" ref-type="bibr"> 55 </xref> ]. One possible way for fungi to <span class="trait">yield</span> melanin is by catalyzing the oxidation of tyrosine via the action of tyrosinases [ <xref rid="pone.0184095.ref056" ref-type="bibr"> 56 </xref> ]. In our study, we have identified 5 putative <italic> TYROSINASE </italic> genes that were differentially expressed at both time points ( <bold> <xref ref-type="supplementary-material" rid="pone.0184095.s005"> S5 </xref> and <xref ref-type="supplementary-material" rid="pone.0184095.s006"> S6 </xref> Tables </bold> ) and between time points ( <bold> <xref ref-type="supplementary-material" rid="pone.0184095.s007"> S7 Table </xref> </bold> ) during infection of soybean leaves by <italic> R </italic> . <italic> solani </italic> AG1-IA. Consistent with this result, the EST dataset of <italic> R </italic> . <italic> solani </italic> AG1-IB (isolate 7/3/14), another subgroup of <span class="gene">AG1</span> causing bottom rot of lettuce, also contained two <italic> TYROSINASE </italic> genes during exposure to lettuce exudates [ <xref rid="pone.0184095.ref057" ref-type="bibr"> 57 </xref> ]. Generally, melanin is not known to have a direct role in fungal pathogens; however, some phytopathogenic fungi produce melanized infective structures for efficient pathogenicity [ <xref rid="pone.0184095.ref058" ref-type="bibr"> 58 </xref> ]. The fact that 4 out of 5 <italic> R </italic> . <italic> solani TYROSINASE </italic> genes were down-regulated in our isolate at post-necrotic stages of soybean leaves suggests that tyrosinases may have a role during host penetration although this warrants further investigation.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Effects of reduced nitrogen inputs on crop <span class="trait">yield</span> and nitrogen use efficiency in a long-term maize-soybean relay strip intercropping system</h2>
			<h3>PMID:28910355</h3>
				
					<h3>PMCID:5598979</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Chen </surname>  <given-names> Ping </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Du </surname>  <given-names> Qing </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Xiaoming </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhou </surname>  <given-names> Li </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Hussain </surname>  <given-names> Sajad </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Lei </surname>  <given-names> Lu </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <xref ref-type="aff" rid="aff003">  <sup> 3 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Song </surname>  <given-names> Chun </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Xiaochun </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Weiguo </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Feng </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Shu </surname>  <given-names> Kai </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Jiang </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Du </surname>  <given-names> Junbo </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Wenyu </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-4650-0727 </contrib-id>  <name>  <surname> Yong </surname>  <given-names> Taiwen </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Project administration </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 27 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						However, most studies focus on relay intercropping systems in which the legumes play a secondary role, i.e., the legumes are used as a cover crop, and the legume <span class="trait">yield</span> is not considered. Previous studies show that facilitation and competition coexist in intercropping systems, particularly in legume/non-legume intercropping systems [ <xref rid="pone.0184503.ref002" ref-type="bibr"> 2 </xref> ]. Xia et al. reported that legumes facilitate the root system and grain <span class="trait">yield</span> of maize considerably [ <xref rid="pone.0184503.ref013" ref-type="bibr"> 13 </xref> ], whereas <span class="gene">Fan</span> et al. found that the grain <span class="trait">yield</span> of fava bean decreased in a wheat/fava bean intercropping system in contrast with a monoculture [ <xref rid="pone.0184503.ref002" ref-type="bibr"> 2 </xref> ]. Generally, the architecture of a plant influences the relative competitive ability. Soybean seedlings often grow under the shade of the maize canopy and then are transferred to full sunlight after the harvest of maize [ <xref rid="pone.0184503.ref015" ref-type="bibr"> 15 </xref> ]. Moreover, the distribution of the root system plays a key role in the acquisition of belowground nutrients. The roots of cereals occupy soils both near the surface and in deeper layers, whereas the roots of legumes are distributed in the upper soil layers [ <xref rid="pone.0184503.ref016" ref-type="bibr"> 16 </xref> ]. The competitive ability of cereals for soil N is stronger than that of legumes in an intercropping system [ <xref rid="pone.0184503.ref017" ref-type="bibr"> 17 </xref> ]. Furthermore, the separation of the root ecological niche between component crops is well known to affect the total grain <span class="trait">yield</span> of intercropping systems. In a maize/fava bean system compared with that of a wheat/fava bean system, the total grain <span class="trait">yield</span> was significantly higher [ <xref rid="pone.0184503.ref002" ref-type="bibr"> 2 </xref> ]. Nitrogen recovery efficiency and uptake efficiency can increase significantly through the interaction of roots between crops. Furthermore, the bi-directional N transfer and positive N competition between crops are advantageous to improve to NUE in the wheat/maize/soybean relay intercropping system [ <xref rid="pone.0184503.ref018" ref-type="bibr"> 18 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						The intercropping advantages in grain <span class="trait">yield</span> and N utilization identified in Experiment 1 suggested that belowground facilitation might be responsible for the aboveground growth and efficient use of nutrients. Experiment 2 was conducted with the same three planting patterns and rates of total nitrogen application as in experiment 1. The planting density and fertilization methods and amounts were same as in Experiment 1; however, the row spacing was different, which was 60 cm for all treatments. This experiment was conducted within a rhizo-box, which had a length, width, and height of 1 m, 0.38 m and 1 m, respectively. Each rhizo-box was planted with 2 rows and 2 holes per row, with row spacing and plant spacing of 0.6 m and 0.17 m, respectively. IMS was planted with 2 maize and 4 soybeans; MM was planted with 4 maize plants; and <span class="gene">MS</span> was planted with 8 soybeans.
					</p>
				
					<p>&nbsp;&nbsp;
						In field experiment 1, samples were collected in the middle row of each plot, and the crop grain <span class="trait">yield</span> and straw and root dry matter at physiological maturity were determined. Crop roots were collected from soil blocks, with the length and depth 34 cm (dug in the middle of two plants in a row) and 40 cm, respectively, and the width (dug in the middle of two crop rows) 100 cm for MM, 50 cm for IM, and 50 cm for <span class="gene">MS</span> and IS. After manual identification, root samples were hand-washed and oven-dried at 80&#x000b0;C for 72 h before weighing. Crop straw was oven-dried at 80&#x000b0;C for 72 h before weighing. Samples were weighed, and the N content of straw and grain was determined using the Kjeldahl method [ <xref rid="pone.0184503.ref021" ref-type="bibr"> 21 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Competition ratio and system productivity index are used to assess interspecific competition and intercropping advantages, respectively. As an indicator, competition ratio is used to measure the degree of competition between crops in an intercropping system [ <xref rid="pone.0184503.ref025" ref-type="bibr"> 25 </xref> ] and is calculated with the following formula: <disp-formula id="pone.0184503.e003"> <alternatives> <graphic xlink:href="pone.0184503.e003.jpg" id="pone.0184503.e003g" mimetype="image" position="anchor" orientation="portrait"/> <mml:math id="M3"> <mml:mrow> <mml:mi> C </mml:mi> <mml:msub> <mml:mi> R </mml:mi> <mml:mrow> <mml:mtext> SM </mml:mtext> </mml:mrow> </mml:msub> <mml:mo> = </mml:mo> <mml:mfrac> <mml:mrow> <mml:msub> <mml:mi> Y </mml:mi> <mml:mrow> <mml:mtext> IS </mml:mtext> </mml:mrow> </mml:msub> <mml:mo> / </mml:mo> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mrow> <mml:msub> <mml:mi> Y </mml:mi> <mml:mrow> <mml:mtext> <span class="gene">MS</span> </mml:mtext> </mml:mrow> </mml:msub> <mml:mo> &#x000d7; </mml:mo> <mml:msub> <mml:mi> A </mml:mi> <mml:mtext> S </mml:mtext> </mml:msub> </mml:mrow> <mml:mo> ) </mml:mo> </mml:mrow> </mml:mrow> <mml:mrow> <mml:msub> <mml:mi> Y </mml:mi> <mml:mrow> <mml:mtext> IM </mml:mtext> </mml:mrow> </mml:msub> <mml:mo> / </mml:mo> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mrow> <mml:msub> <mml:mi> Y </mml:mi> <mml:mrow> <mml:mtext> MM </mml:mtext> </mml:mrow> </mml:msub> <mml:mo> &#x000d7; </mml:mo> <mml:msub> <mml:mi> A </mml:mi> <mml:mtext> M </mml:mtext> </mml:msub> </mml:mrow> <mml:mo> ) </mml:mo> </mml:mrow> </mml:mrow> </mml:mfrac> </mml:mrow> </mml:math> </alternatives> <label> (3) </label> </disp-formula> where <italic> CR </italic> <sub> <italic> SM </italic> </sub> is the competitive ratio of maize relative to soybean, <italic> Y </italic> <sub> IS </sub> and <italic> Y </italic> <sub> <span class="gene">MS</span> </sub> are the <span class="trait">yield</span> or nitrogen acquisition per unit area of maize under intercropping and monoculture, respectively, <italic> Y </italic> <sub> IM </sub> and <italic> Y </italic> <sub> MM </sub> are the <span class="trait">yield</span> or nitrogen acquisition per unit area of soybean under intercropping and monoculture, respectively, and <italic> A </italic> <sub> S </sub> and <italic> A </italic> <sub> M </sub> are the ratios of the area occupied by maize and soybean under the intercropping system relative to that of the corresponding monoculture, respectively. In this study, <italic> A </italic> <sub> S </sub> and <italic> A </italic> <sub> M </sub> were the same. A competition ratio greater than 1 indicated the competitive ability of soybean was greater than that of maize in the maize-soybean relay intercropping system. A ratio less than 1 indicated the competitive ability of soybean was less than that of maize in the maize-soybean relay intercropping system. System productivity index, SPI, is another indicator used to assess intercropping that standardizes the <span class="trait">yield</span> of the secondary crop in terms of the primary crop [ <xref rid="pone.0184503.ref026" ref-type="bibr"> 26 </xref> ] and is calculated as follows: <disp-formula id="pone.0184503.e004"> <alternatives> <graphic xlink:href="pone.0184503.e004.jpg" id="pone.0184503.e004g" mimetype="image" position="anchor" orientation="portrait"/> <mml:math id="M4"> <mml:mrow> <mml:mtext> SPI </mml:mtext> <mml:mo> = </mml:mo> <mml:mrow> <mml:mo> ( </mml:mo> <mml:mrow> <mml:mfrac> <mml:mrow> <mml:msub> <mml:mi> S </mml:mi> <mml:mtext> S </mml:mtext> </mml:msub> </mml:mrow> <mml:mrow> <mml:msub> <mml:mi> S </mml:mi> <mml:mtext> M </mml:mtext> </mml:msub> </mml:mrow> </mml:mfrac> </mml:mrow> <mml:mo> ) </mml:mo> </mml:mrow> <mml:mo> &#x000d7; </mml:mo> <mml:msub> <mml:mi> Y </mml:mi> <mml:mtext> M </mml:mtext> </mml:msub> <mml:mo> + </mml:mo> <mml:msub> <mml:mi> Y </mml:mi> <mml:mtext> S </mml:mtext> </mml:msub> </mml:mrow> </mml:math> </alternatives> <label> (4) </label> </disp-formula> where <italic> S </italic> <sub> S </sub> and <italic> S </italic> <sub> M </sub> are the average yields of soybean and maize under monoculture, respectively, and <italic> Y </italic> <sub> S </sub> and <italic> Y </italic> <sub> M </sub> are the average yields of soybean and maize under intercropping, respectively.
					</p>
				
					<p>&nbsp;&nbsp;
						Planting patterns and N application rates had remarkable effects on crop grain <span class="trait">yield</span> ( <xref ref-type="table" rid="pone.0184503.t002"> Table 2 </xref> ). From 2012 to 2014, straw dry matter and grain <span class="trait">yield</span> of IM declined by 3.4% and 3.5%, respectively, compared with those in MM; however, the difference was not significant. Compared with NN in IM, dry matter accumulation of RN and CN was significantly greater by 29.5% and 33.9% for straw dry matter and 19.6% and 26.6% for grain <span class="trait">yield,</span> respectively ( <xref ref-type="table" rid="pone.0184503.t002"> Table 2 </xref> ). Effects of planting pattern and N level on soybean dry matter accumulation differed from those for maize. The straw dry matter and grain <span class="trait">yield</span> of <span class="gene">MS</span> increased and declined, respectively, compared with those in IS. Under different N application rates in MS, the grain <span class="trait">yield</span> of RN was greater than that in CN, whereas the straw dry matter of RN was lower. In IS, grain <span class="trait">yield</span> and straw dry matter of RN were greater than those of CN.
					</p>
				
					<p>&nbsp;&nbsp;
						The belowground benefits contributed to recovery of aboveground growth, thereby improving soybean <span class="trait">yield</span> ( <xref ref-type="table" rid="pone.0184503.t002"> Table 2 </xref> ). The recovery of growth in soybean is responsible for pod formation, which improves the grain <span class="trait">yield</span> of soybean in the relay intercropping system [ <xref rid="pone.0184503.ref015" ref-type="bibr"> 15 </xref> ]. The N competition ratio (NCR) of soybean relative to maize was greater than 1, indicating a positive interspecific facilitation between component crops in the maize-soybean relay intercropping system ( <xref ref-type="fig" rid="pone.0184503.g003"> Fig 3 </xref> ). This result is consistent with that of a previous study in which the NCR of a legume was greater than that of maize, and therefore, interspecific complementation between the component crops in fava bean-maize intercropping increased total grain <span class="trait">yield</span> [ <xref rid="pone.0184503.ref002" ref-type="bibr"> 2 </xref> ]. In the present study, the N uptake and NUE of IMS increased through interspecific facilitation. The loss of N by maize in IMS was compensated by a gain in soybean in IMS (Tables <xref ref-type="table" rid="pone.0184503.t002"> 2 </xref> and <xref ref-type="table" rid="pone.0184503.t003"> 3 </xref> ). The NUE of IMS increased continually and was higher than that of <span class="gene">MS</span> in 2013 and 2014. These results confirmed that after maize harvest, soybean roots recovered growth and proliferated in the soil layer that was previously occupied by maize. Additionally, planting patterns have a large influence on crop root growth and nutrients utilization. For example, the interspecific competition for nutrients is more important than competition for sunlight when the number of maize rows to soybean rows is 1:1 and the distance between soybean and maize is 30 cm [ <xref rid="pone.0184503.ref030" ref-type="bibr"> 30 </xref> ]. The roots of maize extend to soybean rows, whereas the roots of soybean are distributed primarily underneath the soybean plants when the number of maize rows to soybean rows is 1:3 and the distance between soybean and maize is 30 cm [ <xref rid="pone.0184503.ref022" ref-type="bibr"> 22 </xref> ]. With the number of maize rows to soybean rows 2:2 and the distance between soybean and maize 60 cm, interactions belowground become weaker than those in the forward intercropping systems [ <xref rid="pone.0184503.ref031" ref-type="bibr"> 31 </xref> ]. However, no difference is observed in the total grain <span class="trait">yield</span> between maize-soybean relay intercropping and maize-soybean intercropping [ <xref rid="pone.0184503.ref030" ref-type="bibr"> 30 </xref> , <xref rid="pone.0184503.ref031" ref-type="bibr"> 31 </xref> ].
					</p>
				
					<p>&nbsp;&nbsp;
						Aboveground growth is strongly influenced by belowground processes. Crop root systems have the key role in soil nutrient acquisition, with effects on nutrient balance, and improvements in root distribution increase the potential opportunities for nutrient use. Interspecific root interactions in different cropping systems are manifested as competition or facilitation [ <xref rid="pone.0184503.ref039" ref-type="bibr"> 39 </xref> ]. A complementary root distribution is a prerequisite for high-yields in intercropping systems [ <xref rid="pone.0184503.ref002" ref-type="bibr"> 2 </xref> ]. In the present study, the coexistence period between maize and soybean was approximately 8 weeks, with sampling at the grain-filling stage of maize and soybean at approximately the <span class="gene">V3</span> stage, and the results showed that the ecological niche of crop roots was separated (Figs <xref ref-type="fig" rid="pone.0184503.g004"> 4 </xref> and <xref ref-type="fig" rid="pone.0184503.g005"> 5 </xref> ; <xref ref-type="supplementary-material" rid="pone.0184503.s003"> S2 Table </xref> ). In the coexistence period, with the increase in N levels, the roots of maize rapidly proliferated underneath the maize plants and in the middle soil layer (20&#x02013;80 cm), whereas those of soybeans were primarily distributed underneath the soybean plants. Although the soybean seedlings were weak, the seedlings helped to improve soil nutrients and facilitate maize in relay intercropping [ <xref rid="pone.0184503.ref020" ref-type="bibr"> 20 </xref> ], which is in contrast to a previous study in which legume seedlings were too weak to compete or facilitate with cereal in relay intercropping [ <xref rid="pone.0184503.ref011" ref-type="bibr"> 11 </xref> ]. This contrast may be the result of different legumes species. The CR of soybean was stronger than that of maize, and soybean performed better under RN than under CN. In a similar study, the CR of alfalfa was stronger than that of maize, and the <span class="trait">yield</span> advantage was greater in maize-alfalfa intercropping than that in the corresponding monocultures [ <xref rid="pone.0184503.ref047" ref-type="bibr"> 47 </xref> ]. Additionally, N fertilizer input was required in long-term maize-soybean relay intercropping (Tables <xref ref-type="table" rid="pone.0184503.t001"> 1 </xref> and <xref ref-type="table" rid="pone.0184503.t002"> 2 </xref> , <xref ref-type="fig" rid="pone.0184503.g003"> Fig 3 </xref> ). During the coexistence period, the demand for N was high in the maize grain-filling stage, and an insufficient supply of N would result in <span class="trait">yield</span> loss [ <xref rid="pone.0184503.ref041" ref-type="bibr"> 41 </xref> ].
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Biosynthesis of Polyhydroxyalkanoate from Steamed Soybean Wastewater by a Recombinant Strain of <italic> Pseudomonas </italic> sp. 61-3</h2>
			<h3>PMID:28952548</h3>
				
					<h3>PMCID:5615314</h3>
					<h4>期刊:Bioengineering</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Hokamura </surname>  <given-names> Ayaka </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yunoue </surname>  <given-names> Yuko </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Goto </surname>  <given-names> Saki </given-names>  </name>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Matsusaki </surname>  <given-names> Hiromi </given-names>  </name>  <xref rid="c1-bioengineering-04-00068" ref-type="corresp"> * </xref>  </contrib>
				<br/> 
				年份：<day> 04 </day>  <month> 8 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						<italic> Pseudomonas </italic> sp. 61-3 accumulates a blend of poly(3-hydroxybutyrate) [P(3HB)] homopolymer and a random copolymer, poly(3-hydroxybutyrate- <italic> co </italic> -3-hydroxyalkanoate) [P(3HB- <italic> co </italic> -3HA)], consisting of 3HA units of 4&#x02013;12 carbon atoms. <italic> Pseudomonas </italic> sp. 61-3 possesses two types of PHA synthases, PHB synthase (PhbC) and PHA synthases (PhaC1 and PhaC2), encoded by the <italic> phb </italic> and <italic> pha </italic> loci, respectively. The P(94 mol% 3HB- <italic> co </italic> -6 mol% 3HA) copolymer synthesized by the recombinant strain of <italic> Pseudomonas </italic> sp. 61-3 ( <italic> <span class="gene">phbC</span> </italic> :: <italic> tet </italic> ) harboring additional copies of <italic> phaC1 </italic> gene is known to have desirable physical properties and to be a flexible material with moderate toughness, similar to low-density polyethylene. In this study, we focused on the production of the P(3HB- <italic> co </italic> -3HA) copolymer using steamed soybean wastewater, a by-product in brewing <italic> miso </italic> , which is a traditional Japanese seasoning. The steamed soybean wastewater was spray-dried to produce a powder (SWP) and used as the sole nitrogen source for the synthesis of P(3HB- <italic> co </italic> -3HA) by the <italic> Pseudomonas </italic> sp. 61-3 recombinant strain. Hydrolyzed SWP (HSWP) was also used as a carbon and nitrogen source. P(3HB- <italic> co </italic> -3HA)s with relatively high 3HB fractions could be synthesized by a recombinant strain of <italic> Pseudomonas </italic> sp. 61-3 ( <italic> <span class="gene">phbC</span> </italic> :: <italic> tet </italic> ) harboring additional copies of the <italic> phaC1 </italic> gene in the presence of 2% glucose and 10&#x02013;20 g/L SWP as the sole nitrogen source, producing a PHA concentration of 1.0&#x02013;1.4 g/L. When HSWP was added to a nitrogen- and carbon-free medium, the recombinant strain could synthesize PHA without glucose as a carbon source. The recombinant strain accumulated 32 wt% P(3HB- <italic> co </italic> -3HA) containing 80 mol% 3HB and 20 mol% medium-chain-length 3HA with a PHA concentration of 1.0 g/L when 50 g/L of HSWP was used. The PHA production <span class="trait">yield</span> was estimated as 20 mg-PHA/g-HSWP, which equates to approximately 1.0 g-PHA per liter of soybean wastewater.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genetic Architecture of Charcoal Rot ( <italic> Macrophomina phaseolina </italic> ) <span class="trait">Resistance</span> in Soybean Revealed Using a Diverse Panel</h2>
			<h3>PMID:28983305</h3>
				
					<h3>PMCID:5613161</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Coser </surname>  <given-names> Sara M. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn004">  <sup> &#x02020; </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Chowda Reddy </surname>  <given-names> R. V. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Jiaoping </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/383525/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mueller </surname>  <given-names> Daren S. </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/440823/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Mengistu </surname>  <given-names> Alemu </given-names>  </name>  <xref ref-type="aff" rid="aff3">  <sup> 3 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/86055/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wise </surname>  <given-names> Kiersten A. </given-names>  </name>  <xref ref-type="aff" rid="aff4">  <sup> 4 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Allen </surname>  <given-names> Tom W. </given-names>  </name>  <xref ref-type="aff" rid="aff5">  <sup> 5 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Arti </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn002">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/224402/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Singh </surname>  <given-names> Asheesh K. </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/433094/overview"/>  </contrib>
				<br/> 
				年份：<day> 05 </day>  <month> 9 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						A total of 463 soybean genotypes were evaluated including 459 PI lines and four checks, two resistant and two susceptible checks repeated twice per block. The genotypes were arranged in a randomized complete block design (RCBD) with four replications. Two plants of each genotype in cups made for an experimental unit in each block. Plants were grown in 0.24 L styrofoam cups, with holes in the bottom for drainage, filled with soilless mix (Sunshine Mix, LC1; Sun Gro Horticulture Inc., Agawam, MA) and topped with pellets of slow-release fertilizer (Osmocote Plus 19-9-12, 23 g per cup) spread over the surface of each cup. Seeds were over sown, three seeds per cup, in each pot and thinned to two plants 10 days post-emergence. All experiments were conducted in a greenhouse maintained at 30&#x000b0;C day and 22&#x000b0;C night temperatures, and the room was supplemented with high-pressure 400 W sodium lights to ensure the 16-h photo period. Plants were watered manually to avoid plant wilting. The cut-stem inoculation technique was used to classify genotypes for <span class="trait">resistance</span> (Twizeyimana et al., <xref rid="B57" ref-type="bibr"> 2012 </xref> ). Briefly, soybean plants were grown to the <span class="gene">V2</span> growth stage and a razor sharp blade was used to cut 40 mm above the unifoliate node. From the growing margin of a 4-day old culture of <italic> M. phaseolina </italic> on PDA, a mycelial plug was obtained using a 200 &#x003bc;L pipette tip (Fisher Scientific). The pipette tip with mycelial plug was immediately placed over the stem cut by the razor blade and ensuring the agar was in embedded in the stem. Ratings were based on the recorded measurements of lesion length. The details of measurements are provided in Pawlowski et al. ( <xref rid="B40" ref-type="bibr"> 2015 </xref> ) with the modification that the length of cut-stem at the time of inoculation was 40 mm instead of 25 mm used in their publication. Measurements were taken the third day after inoculation (dai), and followed every 3 days, for a total of 5 ratings (3, 6, 9, 12, and 15 dai). The area under the disease progress curve (AUDPC) for each entry was calculated to estimate the disease <span class="trait">resistance</span> and select superior PI lines (Jeger and Viljanen-Rollinson, <xref rid="B24" ref-type="bibr"> 2001 </xref> ). AUDPC was used for GWAS using greenhouse data.
					</p>
				
					<p>&nbsp;&nbsp;
						Field screening identified four putative candidate genes (Table <xref ref-type="table" rid="T3"> 3 </xref> ). The one, <italic> <span class="gene">Glyma.04g053100</span> </italic> , on chromosome 4 was identified at 6.4 kb downstream of <italic> ss715588228 </italic> . <italic> <span class="gene">Glyma.04g053100</span> </italic> belongs to a universal stress protein family (USP) protein and is homologous to the <italic> AT5G12000.1 </italic> that encodes a Leucine-rich repeat receptor-like protein kinase response to stress (Table <xref ref-type="table" rid="T3"> 3 </xref> ). On chromosome 14, the candidate gene <italic> <span class="gene">Glyma.14g002000</span> </italic> was identified at 6.9 kb away from the peak SNP <italic> ss715618004 </italic> . It encodes a multidrug <span class="trait">resistance</span> protein and is homologous to the <italic> AT4G38380.1 </italic> , which encodes a transporter of plant metabolites related to defense signals (Table <xref ref-type="table" rid="T3"> 3 </xref> ). On chromosome 18, two gene candidates, <italic> <span class="gene">Glyma.18g248100</span> </italic> and <italic> <span class="gene">Glyma.18g228600</span> </italic> , were found at the proximity of the peak SNP <italic> ss715631906 </italic> and <italic> ss715631726 </italic> , respectively. The former is homologous to <italic> Arabidopsi </italic> s <italic> AT4G17070.1 </italic> encoding a peptidyl-prolyl cis-trans isomerase and is related to the response to oxidative stress and pathogen infection. The latter produces a protein containing the LysM domain, which is related to cell wall catabolism and fungal pathogen defense in <italic> Arabidopsis </italic> (Table <xref ref-type="table" rid="T3"> 3 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						Greenhouse study identified eight loci across six chromosomes (Table <xref ref-type="table" rid="T3"> 3 </xref> ). On chromosome 6, two candidate genes were identified at 24.5 and 14.8 kb upstream of the peak SNP <italic> ss715593307 </italic> . Both encode a Cytochrome <span class="gene">P450</span> protein involved in biosynthetic reactions including defensive compounds such as terpenoids. On chromosome 8, two loci were identified, each with two candidate genes. <italic> <span class="gene">Glyma.08g306800</span> </italic> and <italic> <span class="gene">Glyma.08g306900</span> </italic> at the proximity of <italic> ss715601990 </italic> encoding a Glutathione S-transferase, proposed to be involved in the synthesis of stress-related proteins and defense components. The second locus was targeted by two SNPs in high LD ( <italic> r </italic> <sup> 2 </sup> &#x0003e; 0.7). Two genes located at 21.3 kb downstream and 35.8 kb upstream of the leading SNP <italic> ss715602087 </italic> were identified. <italic> <span class="gene">Glyma.08g315900</span> </italic> and <italic> <span class="gene">Glyma.08g316500</span> </italic> encode a peptide methionine sulfoxide reductase (PMSR) and a Calmodulin-domain Protein Kinase (CDPK), respectively, and both are related to abiotic and biotic stress responses. On chromosome 9, <italic> <span class="gene">Glyma.09g230300</span> </italic> was identified 15.6 kb downstream of the lead SNP <italic> ss715604575 </italic> . It encodes a leucine rich repeat (LRR) protein that is involved in disease <span class="trait">resistance</span> mechanisms. The candidate gene, <italic> <span class="gene">Glyma.12g006300</span> </italic> , was identified in a region consisting of 16 significant SNPs in high LD ( <italic> r </italic> <sup> 2 </sup> &#x0003e; 0.7) on chromosome 12. It also encodes a LRR type protein and is related to disease defense components and stress related responses. On chromosome 18, putative candidate gene <italic> <span class="gene">Glyma.18g262800</span> </italic> is located 2.9 kb downstream of the peak SNP <italic> ss715632099 </italic> and encodes an <span class="gene">AP2</span> domain-containing inductor of ethylene-responsive element (ERE) that is associated with plant disease defense. Similarly, <italic> <span class="gene">Glyma.20g197000</span> </italic> encoding an <span class="gene">AP2</span> domain-containing protein was found at the close proximity of the peak SNP <italic> ss715638424 </italic> on chromosome 20.
					</p>
				
					<p>&nbsp;&nbsp;
						The lack of correlation between field and greenhouse experiments for disease assessment is commonly reported in literature (Kim and Diers, <xref rid="B26" ref-type="bibr"> 2000 </xref> ; Hoffman et al., <xref rid="B19" ref-type="bibr"> 2002 </xref> ; Hartman et al., <xref rid="B18" ref-type="bibr"> 2014 </xref> ). Environmental factors, including rainfall and temperature, and plant maturity have major effect on the severity of CR and must be considered when correlating greenhouse and field studies that evaluate a diverse maturity set of soybean accessions (Pawlowski et al., <xref rid="B40" ref-type="bibr"> 2015 </xref> ). Several issues with greenhouse and field disease experiments, such as control of environmental conditions, physiological differences between genotypes, <span class="trait">resistance</span> mechanisms, uniform concentrations of inoculum, and inoculation location on the plant should be considered to help standardize and correlate future research efforts. Plants in field tests are indirectly inoculated by placing inoculum with the seed during planting, and the initial inoculum content of soil is generally not quantified. Furthermore, the rating methodologies for field and greenhouse screening may be of influence because one rating is subjective based on visual symptoms (RSS), and the other is directly quantified by measuring the extent of necrosis (cut-stem method). Another issue is related to the plant growth stage and physiology. A plant's response to abiotic and biotic stresses involves complex signaling pathways that depend on numerous genes, proteins, and metabolites, reflecting in differences in the <span class="trait">resistance</span> mechanism, which may also vary across life stages (Radwan et al., <xref rid="B42" ref-type="bibr"> 2013 </xref> ). In the current studies, disease evaluation occurred at the <span class="gene">V2</span> growth stage in the greenhouse screening, and R7 growth stage for the field screening. Response differences due to artificial wounding (in cut stem method) and growth stage may relate to the expression of different genes for disease reaction mechanisms, and environmental conditions for field settings can generate genotype by environment interactions.
					</p>
				
					<p>&nbsp;&nbsp;
						One of the genes identified is a characterized member of the multi antimicrobial extrusion (MATE) protein which has been reported to be involved in many physiological functions. Among these functions are alkaloid accumulation, flavonoid accumulation, and plant hormone signaling, all directly related to defense responses against herbivores and pathogens (Shitan, <xref rid="B48" ref-type="bibr"> 2016 </xref> ). <span class="gene">MATE</span> transporters are associated with plant disease <span class="trait">resistance,</span> transporting compounds such as salicylic acid (SA) that plays an essential role in plant innate immune signaling and in disease <span class="trait">resistance.</span> Initiation of plant antimicrobial defenses in response to attempted microbial infection relies on a molecular dialog between the interacting organisms. <italic> LysMs </italic> genes have been associated in the recognition of carbohydrate patterns commonly related with microbial surfaces and in microbial infection immunity (Willmann et al., <xref rid="B63" ref-type="bibr"> 2011 </xref> ). Wan et al. ( <xref rid="B60" ref-type="bibr"> 2008 </xref> ) demonstrated that plant cells can perceive chitin fragments that lead to gene induction and defense responses by a <italic> LysM receptor </italic> - <italic> like kinase1 </italic> ( <italic> LysM <span class="gene">RLK1</span> </italic> ) in <italic> Arabidopsis thaliana </italic> , indicating that <italic> LysM <span class="gene">RLK1</span> </italic> is essential for chitin signaling and plant innate immunity.
					</p>
				
					<p>&nbsp;&nbsp;
						For the greenhouse screening, eight gene families were related to the mechanism of response to CR <span class="trait">resistance.</span> Among them, a cytochrome <span class="gene">P450</span> was identified on chromosome 6, which is widely reported for its function in plant defense, including soybean cyst nematode and soybean <span class="trait">rust</span> (Irmisch et al., <xref rid="B23" ref-type="bibr"> 2015 </xref> ; Wan et al., <xref rid="B59" ref-type="bibr"> 2015 </xref> ; Langenbach et al., <xref rid="B28" ref-type="bibr"> 2016 </xref> ). Three candidate genes were proposed on chromosome 8. Glutathione S-transferases (GSTs) has been connected in reactions linked to secondary metabolism and response to pathogen, including its importance in regulation of jasmonic acid (Han et al., <xref rid="B16" ref-type="bibr"> 2013 </xref> ) and has been reported in multiple species, for example, <italic> Populus tomentosa </italic> (Liao et al., <xref rid="B31" ref-type="bibr"> 2014 </xref> ) and <italic> Lilium regale </italic> Wilson (Han et al., <xref rid="B15" ref-type="bibr"> 2016 </xref> ). Peptide methionine sulfoxide reductase (PMSR) acts as an antioxidant, repairing proteins damaged from oxidative stress and a novel defensive role against attack by pathogens <italic> Phytophthora capsici </italic> and <italic> P. infestans </italic> on pepper ( <italic> Capsicum annuum </italic> ) via regulation of the cellular levels of reactive oxygen and defense-related genes (Oh et al., <xref rid="B38" ref-type="bibr"> 2010 </xref> ). Calcium-dependent protein kinases (CPKs or CDPKs) are regulated by various external stimuli including temperature stress (heat or cold), water stress, salinity, wounding, and biotic stress. NtCDPK2 has been reported to be activated by race-specific pathogen elicitation (Cf-9/Avr9) and abiotic stress in tobacco (Romeis et al., <xref rid="B44" ref-type="bibr"> 2000 </xref> ). On chromosome 12 a strong association was detected and leucine rich repeat (LRR) and Terpene synthase (TPS) related candidate genes were identified. LRR proteins plays a crucial role in the plant's defense against pathogens, for their &#x0201c;immune&#x0201d; functions and recognition of non-self-molecules, including from GWAS in soybean (Li et al., <xref rid="B30" ref-type="bibr"> 2016 </xref> ). Genome-wide identification and evolutionary analysis of LRR genes in soybean reported 467 putative LRR-receptor like kinase (LRR-RLK) genes in the soybean genome (Zhou et al., <xref rid="B75" ref-type="bibr"> 2016 </xref> ). Terpene synthase (TPS) act as signaling molecules that induce defenses against tissue damage from wounds and microbes infected sites. Recent research demonstrated a terpene synthase 24 (OsTPS24) encoding a jasmonate-responsive monoterpene synthase that produces an antibacterial &#x003b3;-terpinene against pathogens in rice (Yoshitomi et al., <xref rid="B68" ref-type="bibr"> 2016 </xref> ). On chromosome 18 and 20, the APETALA2/ethylene-responsive element binding protein ( <italic> <span class="gene">AP2</span> </italic> / <italic> EREBP </italic> ) plays various roles in plant growth and development, and in response to stresses as pathogen infection, <span class="trait">drought,</span> temperature, and salinity (Zhang et al., <xref rid="B69" ref-type="bibr"> 2013 </xref> ; Tang et al., <xref rid="B56" ref-type="bibr"> 2016 </xref> ). It has been reported that the ethylene pathway was activated in response to the pathogen attack in several crop species (Dong et al., <xref rid="B13" ref-type="bibr"> 2015 </xref> ; Wang et al., <xref rid="B61" ref-type="bibr"> 2015 </xref> ; Agarwal et al., <xref rid="B2" ref-type="bibr"> 2016 </xref> ; Hong et al., <xref rid="B20" ref-type="bibr"> 2016 </xref> ).
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Tolerance analysis of chloroplast <italic> OsCu/Zn-SOD </italic> overexpressing rice under NaCl and NaHCO <sub> 3 </sub> stress</h2>
			<h3>PMID:29020034</h3>
				
					<h3>PMCID:5636109</h3>
					<h4>期刊:PLoS ONE</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Guan </surname>  <given-names> Qingjie </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liao </surname>  <given-names> Xu </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> He </surname>  <given-names> Mingliang </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Formal analysis </role>  <role content-type="http://credit.casrai.org/"> Methodology </role>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Li </surname>  <given-names> Xiufeng </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Resources </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Zhenyu </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Data curation </role>  <role content-type="http://credit.casrai.org/"> Software </role>  <role content-type="http://credit.casrai.org/"> Validation </role>  <xref ref-type="aff" rid="aff002">  <sup> 2 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Ma </surname>  <given-names> Haiyan </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Validation </role>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; review &#x00026; editing </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Song </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Writing &#x02013; original draft </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <contrib-id authenticated="true" contrib-id-type="orcid"> http://orcid.org/0000-0002-9438-1427 </contrib-id>  <name>  <surname> Liu </surname>  <given-names> Shenkui </given-names>  </name>  <role content-type="http://credit.casrai.org/"> Conceptualization </role>  <role content-type="http://credit.casrai.org/"> Funding acquisition </role>  <role content-type="http://credit.casrai.org/"> Investigation </role>  <role content-type="http://credit.casrai.org/"> Supervision </role>  <role content-type="http://credit.casrai.org/"> Visualization </role>  <xref ref-type="aff" rid="aff001">  <sup> 1 </sup>  </xref>  <xref ref-type="corresp" rid="cor001"> * </xref>  </contrib>
				<br/> 
				年份：<day> 25 </day>  <month> 9 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						To improve genetic resources, genetic engineering has been performed by scientific workers in various countries and the study of transgenic plants has expanded. The SOD gene has been cloned from many plant species. Transgenic tobacco with the rice ( <italic> Oryza sativa </italic> ) chloroplast SOD gene has salt, water, and PEG stress tolerance; moreover, the tobacco chloroplast antioxidant system has been optimized [ <xref rid="pone.0186052.ref012" ref-type="bibr"> 12 </xref> ]. Transgenic wheat protoplasts with the tobacco Mn-SOD gene showed lower levels of oxidative damage and higher levels of SOD and <span class="gene">GR</span> activities [ <xref rid="pone.0186052.ref013" ref-type="bibr"> 13 </xref> ]. Transgenic <italic> Populus </italic> L. that overexpression of the <italic> Tamarix androssowii </italic> Mn-SOD gene has enhanced salt tolerance as a result of SOD activity being increased; furthermore, the relative biomass of the transgenic plants increased by 8- to 23-fold [ <xref rid="pone.0186052.ref014" ref-type="bibr"> 14 </xref> ]. Put Cu/Zn-SOD-overexpressing <italic> Arabidopsis thaliana </italic> has enhanced <span class="trait">resistance</span> to saline&#x02013;sodic stress and growth tolerance to H <sub> 2 </sub> O <sub> 2 </sub> stress [ <xref rid="pone.0186052.ref015" ref-type="bibr"> 15 </xref> ]. In Mn-SOD-overexpressing <italic> A </italic> . <italic> thaliana </italic> shows that the tolerance to NaCl stress was increased and the levels of Mn-SOD, Cu/Zn-SOD, Fe-SOD, CAT, and POD expression also have been improved under salt stress [ <xref rid="pone.0186052.ref016" ref-type="bibr"> 16 </xref> ]. The overexpression of tobacco Mn-SOD/APX in transgenic tall fescue plant confers increased tolerance to MV, H <sub> 2 </sub> O <sub> 2 </sub> , Cu, Cd, and as stresses as a result of SOD and APX enzyme activities being increased [ <xref rid="pone.0186052.ref017" ref-type="bibr"> 17 </xref> ]. The increase of SOD, CAT, <span class="gene">GR</span> and APX activities enhances tolerance to sulfur dioxide (SO <sub> 2 </sub> ) stress in transgenic Chinese cabbage plants that overexpress the <italic> A </italic> . <italic> thaliana </italic> Mn-SOD/CAT gene [ <xref rid="pone.0186052.ref018" ref-type="bibr"> 18 </xref> ]. The overexpression of Mn-SOD ( <italic> Nicotiana plumbaginifolia </italic> ) + Fe-SOD ( <italic> A </italic> . <italic> thaliana </italic> ) in transgenic alfalfa enhances <span class="trait">resistance</span> against <span class="trait">drought</span> stress and improves photosynthetic efficiency [ <xref rid="pone.0186052.ref019" ref-type="bibr"> 19 </xref> ]. The overexpression of cytosolic copper/zinc superoxide dismutase (Cu/Zn-SOD) from the mangrove plant <italic> Avicennia marina </italic> in indica rice var Pusa Basmati enhances tolerance to oxidative, salt, and <span class="trait">drought</span> stresses [ <xref rid="pone.0186052.ref020" ref-type="bibr"> 20 </xref> ]. Saline-sodic stress is caused by salt and pH variation. The in vivo accumulation of ROS injures cell membranes via lipid oxidation, and the accumulation of a large amount of membrane lipid peroxidation products, such as MDA is a cause or product of membrane lipid peroxidation, and also is one of the most obvious indicators of oxidative stress [ <xref rid="pone.0186052.ref021" ref-type="bibr"> 21 </xref> ; <xref rid="pone.0186052.ref022" ref-type="bibr"> 22 </xref> ]. So far, there have been few reports on the regulatory effects of CuZn-SOD overexpression on saline&#x02013;sodic stress tolerance in rice.
					</p>
				
			 
			
			
			

			<hr/> 
		

			
			<h2>文献标题:Genome-Wide Association Study Reveals Novel Loci for SC7 <span class="trait">Resistance</span> in a Soybean Mutant Panel</h2>
			<h3>PMID:29075282</h3>
				
					<h3>PMCID:5641574</h3>
					<h4>期刊:Frontiers in Plant Science</h4>
				

				
			<p>
				作者：<contrib contrib-type="author">  <name>  <surname> Che </surname>  <given-names> Zhijun </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/449857/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Liu </surname>  <given-names> Hailun </given-names>  </name>  <xref ref-type="aff" rid="aff2">  <sup> 2 </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/459990/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yi </surname>  <given-names> Fanglei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Cheng </surname>  <given-names> Hao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yang </surname>  <given-names> Yuming </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Li </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Du </surname>  <given-names> Jingyi </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Zhang </surname>  <given-names> Peipei </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Wang </surname>  <given-names> Jiao </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  <uri xlink:type="simple" xlink:href="http://loop.frontiersin.org/people/309333/overview"/>  </contrib>  <contrib contrib-type="author">  <name>  <surname> Yu </surname>  <given-names> Deyue </given-names>  </name>  <xref ref-type="aff" rid="aff1">  <sup> 1 </sup>  </xref>  <xref ref-type="author-notes" rid="fn001">  <sup> * </sup>  </xref>  </contrib>
				<br/> 
				年份：<day> 28 </day>  <month> 9 </month>  <year> 2017 </year>
				<br/>
			</p>
			<br/> 
			
			
			

			
			
				<h2>正文</h2>
				
					<p>&nbsp;&nbsp;
						Genome-wide association study is a powerful technique that enables researchers to exploit natural variations and locate candidate genes in the genome ( <xref rid="B34" ref-type="bibr"> Korte and Farlow, 2013 </xref> ). In recent years, with the development of the technology of sequencing, a large number of SNP markers were developed. Combining high-quality of SNP markers and phenotypic resources, GWAS has been successfully applied in many crops. In rice, a substantial number of loci were identified performed by GWAS for 14 agronomic traits that potentially important for rice production and improvement ( <xref rid="B27" ref-type="bibr"> Huang et al., 2010 </xref> ). In maize, <xref rid="B59" ref-type="bibr"> Wang X. et al. (2016) </xref> used a GWAS of maize <span class="trait">drought</span> tolerance at the seedling stage and identified 83 genetic variants, which were resolved to 42 candidate genes. According to the SNP signal, <italic> ZmVPP1 </italic> was identified contributing to this trait ( <xref rid="B59" ref-type="bibr"> Wang X. et al., 2016 </xref> ). In soybean, <xref rid="B68" ref-type="bibr"> Zhang et al. (2014) </xref> combined genome-wide association analysis and linkage analysis identified <italic> <span class="gene">GmACP1</span> </italic> gene that related to P efficiency. Nineteen SNPs and five haplotypes for soybean <span class="trait">yield</span> components based on 1536 SNPs were identified by GWAS ( <xref rid="B23" ref-type="bibr"> Hao et al., 2012 </xref> ). Moreover, GWAS can provide a comparatively higher resolution in terms of mapping the genomic position of interested genes or QTLs because it can be applied to germplasm collections or naturally occurring populations ( <xref rid="B50" ref-type="bibr"> Shirasawa et al., 2013 </xref> ).
					</p>
				
					<p>&nbsp;&nbsp;
						To annotate the SNPs significantly associated with <span class="trait">resistance</span> to SC7, we searched the 1 Mb (LD decay distance) flanked regions of these SNPs to find putative genes. Then we utilized the soybean genome annotation information and performed BLASTP against <italic> Arabidopsis </italic> genome to annotate the putative genes ( <bold> Table <xref ref-type="table" rid="T3"> 3 </xref> </bold> ). On chromosome 1, <italic> <span class="gene">Glyma.01G043300</span> </italic> was a putative gene which was homologous to <italic> WRKY72 </italic> in <italic> Arabidopsis </italic> . The latter was reported to contribute to the basal immunity and defense against root-knot nematodes ( <xref rid="B4" ref-type="bibr"> Bhattarai et al., 2010 </xref> ). Moreover, <italic> AtWRKY72 </italic> can also utilize SA-independent defense mechanisms ( <xref rid="B4" ref-type="bibr"> Bhattarai et al., 2010 </xref> ). On chromosome 9, <italic> <span class="gene">Glyma.09g063800</span> </italic> was an eukaryotic translation elongation factor EF1B and contained guanine nucleotide exchange domain ( <xref rid="B3" ref-type="bibr"> Andersen et al., 2000 </xref> ) and a <span class="gene">GST</span> C-terminal domain ( <xref rid="B15" ref-type="bibr"> Dixon et al., 2002 </xref> ). <italic> <span class="gene">Glyma.09g063800</span> </italic> was homologous to <italic> eEF1B&#x003b2; </italic> in <italic> Arabidopsis </italic> . <italic> eEF1B&#x003b2; </italic> was reported to facilitate potato virus X infection and interacted with PVX <italic> TGBp1 </italic> ( <xref rid="B28" ref-type="bibr"> Hwang et al., 2015 </xref> ). In soybean, <xref rid="B36" ref-type="bibr"> Luan et al. (2016) </xref> reported that knocked down the four <italic> <span class="gene">GmEF1B</span> </italic> homologs <italic> Glyma.13g07320 </italic> 0, <italic> <span class="gene">Glyma.02g276600,</span> <span class="gene">Glyma.14g039100</span> </italic> , and <italic> <span class="gene">Glyma.04g195100</span> </italic> enhanced <span class="trait">resistance</span> against the virulent G7 strain of SMV. On chromosome 12, we identified a receptor-like protein <italic> <span class="gene">Glyma.12g233700</span> </italic> which was homologous to <italic> AtRLP9 </italic> in <italic> Arabidopsis </italic> . <italic> AtRLP9 </italic> was reported to co-regulate with Rust-Induced Secreted Protein (RISP), which was strong and specific induced by an avirulent strain of <italic> Melampsora larici-populina </italic> ( <xref rid="B48" ref-type="bibr"> Rinaldi et al., 2007 </xref> ). The gene expression analysis in response to downy mildew showed that <italic> AtRLP9 </italic> was distributed in pathology-related group which also contained AtRLP52, AtWRKY72, and other NBS-LRR class disease <span class="trait">resistance</span> protein ( <xref rid="B13" ref-type="bibr"> Coker et al., 2015 </xref> ). <italic> AtRLP52 </italic> was found to be involved in the <span class="trait">resistance</span> response ( <xref rid="B46" ref-type="bibr"> Ramonell et al., 2005 </xref> ). Therefore, these three genes ( <italic> <span class="gene">Glyma.01G043300,</span> <span class="gene">Glyma.09g063800</span> </italic> , and <italic> <span class="gene">Glyma.12g233700</span> </italic> ) were identified as putative genes related to SC7 <span class="trait">resistance.</span>
					</p>
				
					<p>&nbsp;&nbsp;
						Expression of putative genes were analyzed through qRT-PCR at 0, 2, 3, and 4 days after SC7 or mock inoculated in a <span class="trait">resistance</span> (NJAU_M001) accession and a susceptible (NJAU_M082) accession. As shown in <bold> Figure <xref ref-type="fig" rid="F7"> 7 </xref> </bold> , the expression of <italic> <span class="gene">Glyma.01G043300</span> </italic> significantly decreased with SC7 inoculating than mock inoculating, which was consistent in both the <span class="trait">resistance</span> and susceptible accessions. On the contrary, the <italic> <span class="gene">Glyma.09g063800</span> </italic> was significantly up-regulated in the SMV <span class="trait">resistance</span> NJAU_M001 soybean at 2, 3, and 4 days. And the expression level was the highest at 2 days after SC7 inoculating compared with mock inoculating in the same time. Meanwhile, in SMV susceptible soybean NJAU_M082, the expression of <italic> <span class="gene">Glyma.09g063800</span> </italic> increased slightly at 0, 2, and 3 days and significantly increased at 4 days after SC7 inoculating compared with mock inoculating. The expression pattern of <italic> <span class="gene">Glyma.12g233700</span> </italic> was different in the SMV resistant and susceptible soybean accessions. The expression levels of <italic> <span class="gene">Glyma.12g233700</span> </italic> up-regulated in the SMV resistant soybean NJAU_M001, and decreased in SMV susceptible soybean NJAU_M082 after SMV inoculating compared with mock inoculating. These expression analyses indicate the three putative genes were induced by the infection of SC7, which should provide an important basis for further functional study of these SMV putative <span class="trait">resistance</span> genes.
					</p>
				
			 
			
			
			

			<hr/> 
		
	</div>
</body>

</html>